from cobra import Model, Reaction, Metabolite
model = Model('Example')

Cpd1 = Metabolite('Cpd1', formula='', name='trimethylenediaminium [c]', compartment='')
Cpd2 = Metabolite('Cpd2', formula='', name='Water [c]', compartment='')
Cpd3 = Metabolite('Cpd3', formula='', name='O2 [c]', compartment='')
Cpd4 = Metabolite('Cpd4', formula='', name='beta-Aminopropion aldehyde [c]', compartment='')
Cpd5 = Metabolite('Cpd5', formula='', name='Hydrogen peroxide [c]', compartment='')
Cpd6 = Metabolite('Cpd6', formula='', name='Ammonium [c]', compartment='')
Cpd7 = Metabolite('Cpd7', formula='', name='proton [m]', compartment='')
Cpd8 = Metabolite('Cpd8', formula='', name='Nicotinamide adenine dinucleotide phosphate - reduced [m]', compartment='')
Cpd9 = Metabolite('Cpd9', formula='', name='O2 [m]', compartment='')
Cpd10 = Metabolite('Cpd10', formula='', name='25-Hydroxyvitamin D2 [m]', compartment='')
Cpd11 = Metabolite('Cpd11', formula='', name='24R,25-Dihyoxyvitamin D2 [m]', compartment='')
Cpd12 = Metabolite('Cpd12', formula='', name='Water [m]', compartment='')
Cpd13 = Metabolite('Cpd13', formula='', name='Nicotinamide adenine dinucleotide phosphate [m]', compartment='')
Cpd14 = Metabolite('Cpd14', formula='', name='calcidiol [m]', compartment='')
Cpd15 = Metabolite('Cpd15', formula='', name='(24R)-24,25-dihydroxycalciol [m]', compartment='')
Cpd16 = Metabolite('Cpd16', formula='', name='calcidiol [e]', compartment='')
Cpd17 = Metabolite('Cpd17', formula='', name='calcidiol [c]', compartment='')
Cpd18 = Metabolite('Cpd18', formula='', name='1-alpha,25-Dihydroxyvitamin D2 [m]', compartment='')
Cpd19 = Metabolite('Cpd19', formula='', name='calcitriol [m]', compartment='')
Cpd20 = Metabolite('Cpd20', formula='', name='2-aminoacrylic acid [c]', compartment='')
Cpd21 = Metabolite('Cpd21', formula='', name='pyruvate [c]', compartment='')
Cpd22 = Metabolite('Cpd22', formula='', name='Nicotinamide adenine dinucleotide phosphate - reduced [c]', compartment='')
Cpd23 = Metabolite('Cpd23', formula='', name='(2R,3S,4R,5R)-5-(6-amino-9H-purin-9-yl)-4-hydroxy-2-({[(sulfonatooxy)phosphinato]oxy}methyl)oxolan-3-yl phosphate [c]', compartment='')
Cpd24 = Metabolite('Cpd24', formula='', name='L-cysteate(1-) [c]', compartment='')
Cpd25 = Metabolite('Cpd25', formula='', name='Nicotinamide adenine dinucleotide phosphate [c]', compartment='')
Cpd26 = Metabolite('Cpd26', formula='', name='adenosine 3,5-bismonophosphate(4-) [c]', compartment='')
Cpd27 = Metabolite('Cpd27', formula='', name='L-2-aminoadipate(1-) [c]', compartment='')
Cpd28 = Metabolite('Cpd28', formula='', name='2-Oxoglutarate [m]', compartment='')
Cpd29 = Metabolite('Cpd29', formula='', name='L-2-aminoadipate(1-) [m]', compartment='')
Cpd30 = Metabolite('Cpd30', formula='', name='2-Oxoglutarate [c]', compartment='')
Cpd31 = Metabolite('Cpd31', formula='', name='2-Hydroxybutyrate [c]', compartment='')
Cpd32 = Metabolite('Cpd32', formula='', name='Nicotinamide adenine dinucleotide [c]', compartment='')
Cpd33 = Metabolite('Cpd33', formula='', name='proton [c]', compartment='')
Cpd34 = Metabolite('Cpd34', formula='', name='2-Oxobutanoate [c]', compartment='')
Cpd35 = Metabolite('Cpd35', formula='', name='Nicotinamide adenine dinucleotide - reduced [c]', compartment='')
Cpd36 = Metabolite('Cpd36', formula='', name='2-Hydroxybutyrate [e]', compartment='')
Cpd37 = Metabolite('Cpd37', formula='', name='proton [e]', compartment='')
Cpd38 = Metabolite('Cpd38', formula='', name='Bicarbonate [e]', compartment='')
Cpd39 = Metabolite('Cpd39', formula='', name='Sodium [e]', compartment='')
Cpd40 = Metabolite('Cpd40', formula='', name='Bicarbonate [c]', compartment='')
Cpd41 = Metabolite('Cpd41', formula='', name='Sodium [c]', compartment='')
Cpd42 = Metabolite('Cpd42', formula='', name='2-oxoadipate(2-) [m]', compartment='')
Cpd43 = Metabolite('Cpd43', formula='', name='Coenzyme A [m]', compartment='')
Cpd44 = Metabolite('Cpd44', formula='', name='Nicotinamide adenine dinucleotide [m]', compartment='')
Cpd45 = Metabolite('Cpd45', formula='', name='carbon dioxide [m]', compartment='')
Cpd46 = Metabolite('Cpd46', formula='', name='glutaryl-CoA(5-) [m]', compartment='')
Cpd47 = Metabolite('Cpd47', formula='', name='Nicotinamide adenine dinucleotide - reduced [m]', compartment='')
Cpd48 = Metabolite('Cpd48', formula='', name='2-oxoadipate(2-) [c]', compartment='')
Cpd49 = Metabolite('Cpd49', formula='', name='3,4-Dihydroxymandelaldehyde [c]', compartment='')
Cpd50 = Metabolite('Cpd50', formula='', name='3,4-Dihydroxyphenylethyleneglycol [c]', compartment='')
Cpd51 = Metabolite('Cpd51', formula='', name='(3,4-dihydroxyphenyl)acetate [c]', compartment='')
Cpd52 = Metabolite('Cpd52', formula='', name='S-Adenosyl-L-methionine [c]', compartment='')
Cpd53 = Metabolite('Cpd53', formula='', name='S-Adenosyl-L-homocysteine [c]', compartment='')
Cpd54 = Metabolite('Cpd54', formula='', name='Homovanillate [c]', compartment='')
Cpd55 = Metabolite('Cpd55', formula='', name='L-dopa [e]', compartment='')
Cpd56 = Metabolite('Cpd56', formula='', name='L-dopa [c]', compartment='')
Cpd57 = Metabolite('Cpd57', formula='', name='3,4-Dihydroxyphenylacetaldehyde [c]', compartment='')
Cpd58 = Metabolite('Cpd58', formula='', name='3,4-Dihydroxymandelate [c]', compartment='')
Cpd59 = Metabolite('Cpd59', formula='', name='3-(4-hydroxyphenyl)pyruvate [c]', compartment='')
Cpd60 = Metabolite('Cpd60', formula='', name='carbon dioxide [c]', compartment='')
Cpd61 = Metabolite('Cpd61', formula='', name='homogentisate [c]', compartment='')
Cpd62 = Metabolite('Cpd62', formula='', name='2-methyl-3-oxopropanoate [m]', compartment='')
Cpd63 = Metabolite('Cpd63', formula='', name='L-glutamate(1-) [m]', compartment='')
Cpd64 = Metabolite('Cpd64', formula='', name='(S)-3-aminoisobutyric acid [m]', compartment='')
Cpd65 = Metabolite('Cpd65', formula='', name='(S)-3-aminoisobutyric acid [c]', compartment='')
Cpd66 = Metabolite('Cpd66', formula='', name='(2S)-1-hydroxy-3-oxooctadecan-2-aminium [c]', compartment='')
Cpd67 = Metabolite('Cpd67', formula='', name='sphinganine(1+) [c]', compartment='')
Cpd68 = Metabolite('Cpd68', formula='', name='3-hydroxyanthranilate [c]', compartment='')
Cpd69 = Metabolite('Cpd69', formula='', name='cis,cis-2-amino-3-(3-oxoprop-1-enyl)but-2-enedioate [c]', compartment='')
Cpd70 = Metabolite('Cpd70', formula='', name='crotonoyl-CoA(4-) [m]', compartment='')
Cpd71 = Metabolite('Cpd71', formula='', name='(R)-3-hydroxybutanoyl-CoA(4-) [m]', compartment='')
Cpd72 = Metabolite('Cpd72', formula='', name='3-hydroxy-L-kynurenine [c]', compartment='')
Cpd73 = Metabolite('Cpd73', formula='', name='4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate [c]', compartment='')
Cpd74 = Metabolite('Cpd74', formula='', name='L-glutamate(1-) [c]', compartment='')
Cpd75 = Metabolite('Cpd75', formula='', name='dopaminium(1+) [c]', compartment='')
Cpd76 = Metabolite('Cpd76', formula='', name='3-hydroxypropanoyl-CoA(4-) [c]', compartment='')
Cpd77 = Metabolite('Cpd77', formula='', name='3-hydroxypropionate [c]', compartment='')
Cpd78 = Metabolite('Cpd78', formula='', name='Coenzyme A [c]', compartment='')
Cpd79 = Metabolite('Cpd79', formula='', name='3-Hydroxykynurenamine [c]', compartment='')
Cpd80 = Metabolite('Cpd80', formula='', name='quinoline-4,8-diol [c]', compartment='')
Cpd81 = Metabolite('Cpd81', formula='', name='(4-hydroxy-3-methoxyphenyl)acetaldehyde [c]', compartment='')
Cpd82 = Metabolite('Cpd82', formula='', name='3-Methoxy-4-hydroxyphenylglycolaldehyde [c]', compartment='')
Cpd83 = Metabolite('Cpd83', formula='', name='3-Methoxy-4-hydroxymandelate [c]', compartment='')
Cpd84 = Metabolite('Cpd84', formula='', name='3-Methoxytyramine [c]', compartment='')
Cpd85 = Metabolite('Cpd85', formula='', name='3-AMP(2-) [l]', compartment='')
Cpd86 = Metabolite('Cpd86', formula='', name='Water [l]', compartment='')
Cpd87 = Metabolite('Cpd87', formula='', name='Adenosine [l]', compartment='')
Cpd88 = Metabolite('Cpd88', formula='', name='hydrogenphosphate [l]', compartment='')
Cpd89 = Metabolite('Cpd89', formula='', name='3-Sulfino-L-alanine [m]', compartment='')
Cpd90 = Metabolite('Cpd90', formula='', name='L-aspartate(1-) [c]', compartment='')
Cpd91 = Metabolite('Cpd91', formula='', name='3-Sulfino-L-alanine [c]', compartment='')
Cpd92 = Metabolite('Cpd92', formula='', name='L-aspartate(1-) [m]', compartment='')
Cpd93 = Metabolite('Cpd93', formula='', name='Hypotaurine [c]', compartment='')
Cpd94 = Metabolite('Cpd94', formula='', name='3-Sulfinopyruvate [c]', compartment='')
Cpd95 = Metabolite('Cpd95', formula='', name='3-Sulfinopyruvate [m]', compartment='')
Cpd96 = Metabolite('Cpd96', formula='', name='Adrenaline [c]', compartment='')
Cpd97 = Metabolite('Cpd97', formula='', name='methylammonium [c]', compartment='')
Cpd98 = Metabolite('Cpd98', formula='', name='Norepinephrine [c]', compartment='')
Cpd99 = Metabolite('Cpd99', formula='', name='L-4-hydroxyglutamic semialdehyde [m]', compartment='')
Cpd100 = Metabolite('Cpd100', formula='', name='erythro-4-hydroxy-L-glutamate(1-) [m]', compartment='')
Cpd101 = Metabolite('Cpd101', formula='', name='(4-hydroxyphenyl)acetaldehyde [c]', compartment='')
Cpd102 = Metabolite('Cpd102', formula='', name='4-hydroxyphenylacetate [c]', compartment='')
Cpd103 = Metabolite('Cpd103', formula='', name='4-nitrophenolate [c]', compartment='')
Cpd104 = Metabolite('Cpd104', formula='', name='4-Nitrophenyl sulfate [c]', compartment='')
Cpd105 = Metabolite('Cpd105', formula='', name='ATP [c]', compartment='')
Cpd106 = Metabolite('Cpd106', formula='', name='5alpha-Dihydrotestosterone glucuronide [c]', compartment='')
Cpd107 = Metabolite('Cpd107', formula='', name='ADP [c]', compartment='')
Cpd108 = Metabolite('Cpd108', formula='', name='hydrogenphosphate [c]', compartment='')
Cpd109 = Metabolite('Cpd109', formula='', name='5alpha-Dihydrotestosterone glucuronide [e]', compartment='')
Cpd110 = Metabolite('Cpd110', formula='', name='17beta-hydroxy-5alpha-androstan-3-one [c]', compartment='')
Cpd111 = Metabolite('Cpd111', formula='', name='5alpha-Dihydrotestosterone sulfate [c]', compartment='')
Cpd112 = Metabolite('Cpd112', formula='', name='5-formyltetrahydrofolate(2-) [e]', compartment='')
Cpd113 = Metabolite('Cpd113', formula='', name='5-formyltetrahydrofolate(2-) [c]', compartment='')
Cpd114 = Metabolite('Cpd114', formula='', name='Water [e]', compartment='')
Cpd115 = Metabolite('Cpd115', formula='', name='5-hydroxy-L-tryptophan [c]', compartment='')
Cpd116 = Metabolite('Cpd116', formula='', name='serotonin(1+) [c]', compartment='')
Cpd117 = Metabolite('Cpd117', formula='', name='(5-hydroxyindol-3-yl)acetaldehyde [c]', compartment='')
Cpd118 = Metabolite('Cpd118', formula='', name='5-Hydroxyindoleacetate [c]', compartment='')
Cpd119 = Metabolite('Cpd119', formula='', name='(5-hydroxyindol-3-yl)acetaldehyde [m]', compartment='')
Cpd120 = Metabolite('Cpd120', formula='', name='5-Hydroxyindoleacetate [m]', compartment='')
Cpd121 = Metabolite('Cpd121', formula='', name='5-hydroxy-N-formylkynurenine [c]', compartment='')
Cpd122 = Metabolite('Cpd122', formula='', name='5-hydroxy-L-tryptophan [e]', compartment='')
Cpd123 = Metabolite('Cpd123', formula='', name='5-hydroxykynurenine [c]', compartment='')
Cpd124 = Metabolite('Cpd124', formula='', name='5-Hydroxykynurenamine [c]', compartment='')
Cpd125 = Metabolite('Cpd125', formula='', name='4,6-Dihydroxyquinoline [c]', compartment='')
Cpd126 = Metabolite('Cpd126', formula='', name='5-Methyltetrahydrofolate [e]', compartment='')
Cpd127 = Metabolite('Cpd127', formula='', name='5-Methyltetrahydrofolate [c]', compartment='')
Cpd128 = Metabolite('Cpd128', formula='', name='(alpha-D-mannosyl)2-beta-D-mannosyl-N-acetylglucosamine [c]', compartment='')
Cpd129 = Metabolite('Cpd129', formula='', name='D-Mannose [c]', compartment='')
Cpd130 = Metabolite('Cpd130', formula='', name='beta-1,4-mannose-N-acetylglucosamine [c]', compartment='')
Cpd131 = Metabolite('Cpd131', formula='', name='(alpha-D-mannosyl)2-beta-D-mannosyl-N-acetylglucosamine [l]', compartment='')
Cpd132 = Metabolite('Cpd132', formula='', name='D-Mannose [l]', compartment='')
Cpd133 = Metabolite('Cpd133', formula='', name='beta-1,4-mannose-N-acetylglucosamine [l]', compartment='')
Cpd134 = Metabolite('Cpd134', formula='', name='galactosyl glucosyl ceramide [c]', compartment='')
Cpd135 = Metabolite('Cpd135', formula='', name='UDP-D-galactose(2-) [c]', compartment='')
Cpd136 = Metabolite('Cpd136', formula='', name='trihexosyl ceramide [c]', compartment='')
Cpd137 = Metabolite('Cpd137', formula='', name='UDP [c]', compartment='')
Cpd138 = Metabolite('Cpd138', formula='', name='galactosyl glucosyl ceramide [g]', compartment='')
Cpd139 = Metabolite('Cpd139', formula='', name='UDP-D-galactose(2-) [g]', compartment='')
Cpd140 = Metabolite('Cpd140', formula='', name='proton [g]', compartment='')
Cpd141 = Metabolite('Cpd141', formula='', name='trihexosyl ceramide [g]', compartment='')
Cpd142 = Metabolite('Cpd142', formula='', name='UDP [g]', compartment='')
Cpd143 = Metabolite('Cpd143', formula='', name='T antigen (core 1) [g]', compartment='')
Cpd144 = Metabolite('Cpd144', formula='', name='UDP-N-acetyl-alpha-D-glucosamine(2-) [g]', compartment='')
Cpd145 = Metabolite('Cpd145', formula='', name='GlcNAc-alpha-1,4-Core 1 [g]', compartment='')
Cpd146 = Metabolite('Cpd146', formula='', name='Core 2 [g]', compartment='')
Cpd147 = Metabolite('Cpd147', formula='', name='GlcNAc-alpha-1,4-Core 2 [g]', compartment='')
Cpd148 = Metabolite('Cpd148', formula='', name='acetoacetate [c]', compartment='')
Cpd149 = Metabolite('Cpd149', formula='', name='Acetoacetyl-CoA [c]', compartment='')
Cpd150 = Metabolite('Cpd150', formula='', name='AMP [c]', compartment='')
Cpd151 = Metabolite('Cpd151', formula='', name='Diphosphate [c]', compartment='')
Cpd152 = Metabolite('Cpd152', formula='', name='ammonioacetone [c]', compartment='')
Cpd153 = Metabolite('Cpd153', formula='', name='methylglyoxal [c]', compartment='')
Cpd154 = Metabolite('Cpd154', formula='', name='Type IH glycolipid [g]', compartment='')
Cpd155 = Metabolite('Cpd155', formula='', name='UDP-N-acetyl-D-galactosamine(2-) [g]', compartment='')
Cpd156 = Metabolite('Cpd156', formula='', name='Type IA glycolipid [g]', compartment='')
Cpd157 = Metabolite('Cpd157', formula='', name='Type IB glycolipid [g]', compartment='')
Cpd158 = Metabolite('Cpd158', formula='', name='Type IIH glycolipid [g]', compartment='')
Cpd159 = Metabolite('Cpd159', formula='', name='(Gal)3 (Glc)1 (GlcNAc)1 (LFuc)1 (Cer)1 [g]', compartment='')
Cpd160 = Metabolite('Cpd160', formula='', name='(Gal)2 (GalNAc)1 (Glc)1 (GlcNAc)1 (LFuc)1 (Cer)1 [g]', compartment='')
Cpd161 = Metabolite('Cpd161', formula='', name='Type IIIH glycolipid [g]', compartment='')
Cpd162 = Metabolite('Cpd162', formula='', name='Type IIIA glycolipid [g]', compartment='')
Cpd163 = Metabolite('Cpd163', formula='', name='VI2Fuc-nLc6 [g]', compartment='')
Cpd164 = Metabolite('Cpd164', formula='', name='(Gal)3 (GalNAc)1 (Glc)1 (GlcNAc)2 (LFuc)1 (Cer)1 [g]', compartment='')
Cpd165 = Metabolite('Cpd165', formula='', name='(Gal)4 (GalNAc)1 (Glc)1 (GlcNAc)2 (LFuc)2 (Cer)1 [g]', compartment='')
Cpd166 = Metabolite('Cpd166', formula='', name='Type IIIAb [g]', compartment='')
Cpd167 = Metabolite('Cpd167', formula='', name='(Gal)4 (Glc)1 (GlcNAc)2 (LFuc)1 (Cer)1 [g]', compartment='')
Cpd168 = Metabolite('Cpd168', formula='', name='(Gal)4 (Glc)1 (GlcNAc)3 (LFuc)2 (Cer)1 [g]', compartment='')
Cpd169 = Metabolite('Cpd169', formula='', name='(Gal)6 (Glc)1 (GlcNAc)3 (LFuc)2 (Cer)1 [g]', compartment='')
Cpd170 = Metabolite('Cpd170', formula='', name='4-Aminobutanoate [m]', compartment='')
Cpd171 = Metabolite('Cpd171', formula='', name='Succinic semialdehyde [m]', compartment='')
Cpd172 = Metabolite('Cpd172', formula='', name='4-Aminobutanal [c]', compartment='')
Cpd173 = Metabolite('Cpd173', formula='', name='4-Aminobutanoate [c]', compartment='')
Cpd174 = Metabolite('Cpd174', formula='', name='4-Aminobutanoate [e]', compartment='')
Cpd175 = Metabolite('Cpd175', formula='', name='4-Aminobutanoate [l]', compartment='')
Cpd176 = Metabolite('Cpd176', formula='', name='proton [l]', compartment='')
Cpd177 = Metabolite('Cpd177', formula='', name='2-methylacetoacetyl-CoA(4-) [m]', compartment='')
Cpd178 = Metabolite('Cpd178', formula='', name='Acetyl-CoA [m]', compartment='')
Cpd179 = Metabolite('Cpd179', formula='', name='Propanoyl-CoA [m]', compartment='')
Cpd180 = Metabolite('Cpd180', formula='', name='Acetyl-CoA [c]', compartment='')
Cpd181 = Metabolite('Cpd181', formula='', name='Acetoacetyl-CoA [m]', compartment='')
Cpd182 = Metabolite('Cpd182', formula='', name='Acetyl-CoA [x]', compartment='')
Cpd183 = Metabolite('Cpd183', formula='', name='Acetoacetyl-CoA [x]', compartment='')
Cpd184 = Metabolite('Cpd184', formula='', name='Coenzyme A [x]', compartment='')
Cpd185 = Metabolite('Cpd185', formula='', name='acetoacetate [e]', compartment='')
Cpd186 = Metabolite('Cpd186', formula='', name='acetoacetate [m]', compartment='')
Cpd187 = Metabolite('Cpd187', formula='', name='3-Oxodecanoyl-CoA [x]', compartment='')
Cpd188 = Metabolite('Cpd188', formula='', name='Octanoyl-CoA (n-C8:0CoA) [x]', compartment='')
Cpd189 = Metabolite('Cpd189', formula='', name='3-Oxododecanoyl-CoA [x]', compartment='')
Cpd190 = Metabolite('Cpd190', formula='', name='Decanoyl-CoA (n-C10:0CoA) [x]', compartment='')
Cpd191 = Metabolite('Cpd191', formula='', name='3-Oxotetradecanoyl-CoA [x]', compartment='')
Cpd192 = Metabolite('Cpd192', formula='', name='Dodecanoyl-CoA (n-C12:0CoA) [x]', compartment='')
Cpd193 = Metabolite('Cpd193', formula='', name='3-oxopalmitoyl-CoA(4-) [x]', compartment='')
Cpd194 = Metabolite('Cpd194', formula='', name='Tetradecanoyl-CoA (n-C14:0CoA) [x]', compartment='')
Cpd195 = Metabolite('Cpd195', formula='', name='3-Oxooctadecanoyl-CoA [x]', compartment='')
Cpd196 = Metabolite('Cpd196', formula='', name='Palmitoyl-CoA (n-C16:0CoA) [x]', compartment='')
Cpd197 = Metabolite('Cpd197', formula='', name='3-Oxohexacosyl-CoA [x]', compartment='')
Cpd198 = Metabolite('Cpd198', formula='', name='tetracosanoyl-CoA  (n-C24:0CoA) [x]', compartment='')
Cpd199 = Metabolite('Cpd199', formula='', name='Malonyl-CoA [c]', compartment='')
Cpd200 = Metabolite('Cpd200', formula='', name='ATP [m]', compartment='')
Cpd201 = Metabolite('Cpd201', formula='', name='Bicarbonate [m]', compartment='')
Cpd202 = Metabolite('Cpd202', formula='', name='ADP [m]', compartment='')
Cpd203 = Metabolite('Cpd203', formula='', name='Malonyl-CoA [m]', compartment='')
Cpd204 = Metabolite('Cpd204', formula='', name='hydrogenphosphate [m]', compartment='')
Cpd205 = Metabolite('Cpd205', formula='', name='Acetyl-CoA [g]', compartment='')
Cpd206 = Metabolite('Cpd206', formula='', name='propionate [c]', compartment='')
Cpd207 = Metabolite('Cpd207', formula='', name='Propanoyl-CoA [c]', compartment='')
Cpd208 = Metabolite('Cpd208', formula='', name='propionate [m]', compartment='')
Cpd209 = Metabolite('Cpd209', formula='', name='AMP [m]', compartment='')
Cpd210 = Metabolite('Cpd210', formula='', name='Diphosphate [m]', compartment='')
Cpd211 = Metabolite('Cpd211', formula='', name='Acetyl-CoA [r]', compartment='')
Cpd212 = Metabolite('Cpd212', formula='', name='acetone [e]', compartment='')
Cpd213 = Metabolite('Cpd213', formula='', name='acetone [c]', compartment='')
Cpd214 = Metabolite('Cpd214', formula='', name='acetone [m]', compartment='')
Cpd215 = Metabolite('Cpd215', formula='', name='N-acetyl-D-glucosamine [c]', compartment='')
Cpd216 = Metabolite('Cpd216', formula='', name='N-acetyl-D-mannosamine [c]', compartment='')
Cpd217 = Metabolite('Cpd217', formula='', name='D-Glucosamine 6-phosphate [c]', compartment='')
Cpd218 = Metabolite('Cpd218', formula='', name='N-acetyl-D-glucosamine 6-phosphate(2-) [c]', compartment='')
Cpd219 = Metabolite('Cpd219', formula='', name='N-Acetyl-D-glucosamine 1-phosphate [c]', compartment='')
Cpd220 = Metabolite('Cpd220', formula='', name='6-(N-acetyl-alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol(1-) [c]', compartment='')
Cpd221 = Metabolite('Cpd221', formula='', name='acetate [c]', compartment='')
Cpd222 = Metabolite('Cpd222', formula='', name='6-(alpha-D-glucosaminyl)-1-phosphatidyl-1D-myo-inositol [c]', compartment='')
Cpd223 = Metabolite('Cpd223', formula='', name='N-acetyl-L-glutamate(2-) [m]', compartment='')
Cpd224 = Metabolite('Cpd224', formula='', name='acetylcholine [e]', compartment='')
Cpd225 = Metabolite('Cpd225', formula='', name='acetate [e]', compartment='')
Cpd226 = Metabolite('Cpd226', formula='', name='Choline [e]', compartment='')
Cpd227 = Metabolite('Cpd227', formula='', name='acetylcholine [c]', compartment='')
Cpd228 = Metabolite('Cpd228', formula='', name='Citrate [c]', compartment='')
Cpd229 = Metabolite('Cpd229', formula='', name='Oxaloacetate [c]', compartment='')
Cpd230 = Metabolite('Cpd230', formula='', name='N-acetyl-D-mannosamine 6-phosphate(2-) [c]', compartment='')
Cpd231 = Metabolite('Cpd231', formula='', name='Phosphoenolpyruvate [c]', compartment='')
Cpd232 = Metabolite('Cpd232', formula='', name='N-Acetylneuraminate 9-phosphate [c]', compartment='')
Cpd233 = Metabolite('Cpd233', formula='', name='D-Mannose 6-phosphate [c]', compartment='')
Cpd234 = Metabolite('Cpd234', formula='', name='2-keto-3-deoxy-D-glycero-D-galactononic acid 9-phosphate [c]', compartment='')
Cpd235 = Metabolite('Cpd235', formula='', name='N-acetylneuraminate [c]', compartment='')
Cpd236 = Metabolite('Cpd236', formula='', name='N-acetylneuraminate [l]', compartment='')
Cpd237 = Metabolite('Cpd237', formula='', name='2-methylbutanoyl-CoA(4-) [m]', compartment='')
Cpd238 = Metabolite('Cpd238', formula='', name='Flavin adenine dinucleotide oxidized [m]', compartment='')
Cpd239 = Metabolite('Cpd239', formula='', name='2-methylcrotonoyl-CoA(4-) [m]', compartment='')
Cpd240 = Metabolite('Cpd240', formula='', name='Flavin adenine dinucleotide reduced [m]', compartment='')
Cpd241 = Metabolite('Cpd241', formula='', name='Butanoyl-CoA [m]', compartment='')
Cpd242 = Metabolite('Cpd242', formula='', name='Isovaleryl-CoA [m]', compartment='')
Cpd243 = Metabolite('Cpd243', formula='', name='3-methylbut-2-enoyl-CoA(4-) [m]', compartment='')
Cpd244 = Metabolite('Cpd244', formula='', name='isobutyryl-CoA(4-) [m]', compartment='')
Cpd245 = Metabolite('Cpd245', formula='', name='2-Methylprop-2-enoyl-CoA [m]', compartment='')
Cpd246 = Metabolite('Cpd246', formula='', name='O2 [x]', compartment='')
Cpd247 = Metabolite('Cpd247', formula='', name='Hydrogen peroxide [x]', compartment='')
Cpd248 = Metabolite('Cpd248', formula='', name='trans-Hexadec-2-enoyl-CoA [x]', compartment='')
Cpd249 = Metabolite('Cpd249', formula='', name='acyl carrier protein [c]', compartment='')
Cpd250 = Metabolite('Cpd250', formula='', name='Acetyl-ACP [c]', compartment='')
Cpd251 = Metabolite('Cpd251', formula='', name='N(2)-acetyl-L-ornithine [c]', compartment='')
Cpd252 = Metabolite('Cpd252', formula='', name='Ornithine [c]', compartment='')
Cpd253 = Metabolite('Cpd253', formula='', name='Isocitrate [c]', compartment='')
Cpd254 = Metabolite('Cpd254', formula='', name='Citrate [m]', compartment='')
Cpd255 = Metabolite('Cpd255', formula='', name='Isocitrate [m]', compartment='')
Cpd256 = Metabolite('Cpd256', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholest-24-enoyl-CoA [x]', compartment='')
Cpd257 = Metabolite('Cpd257', formula='', name='Flavin adenine dinucleotide reduced [x]', compartment='')
Cpd258 = Metabolite('Cpd258', formula='', name='Flavin adenine dinucleotide oxidized [x]', compartment='')
Cpd259 = Metabolite('Cpd259', formula='', name='3alpha,7alpha,24-Trihydroxy-5beta-cholestanoyl-CoA [x]', compartment='')
Cpd260 = Metabolite('Cpd260', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA(S) [x]', compartment='')
Cpd261 = Metabolite('Cpd261', formula='', name='Water [x]', compartment='')
Cpd262 = Metabolite('Cpd262', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholest-24-enoyl-CoA [x]', compartment='')
Cpd263 = Metabolite('Cpd263', formula='', name='FMN [c]', compartment='')
Cpd264 = Metabolite('Cpd264', formula='', name='Riboflavin [c]', compartment='')
Cpd265 = Metabolite('Cpd265', formula='', name='O-acetylcarnitine [c]', compartment='')
Cpd266 = Metabolite('Cpd266', formula='', name='O-acetylcarnitine [m]', compartment='')
Cpd267 = Metabolite('Cpd267', formula='', name='acetate [m]', compartment='')
Cpd268 = Metabolite('Cpd268', formula='', name='5-Methoxyindoleacetate [c]', compartment='')
Cpd269 = Metabolite('Cpd269', formula='', name='N-acetylserotonin [c]', compartment='')
Cpd270 = Metabolite('Cpd270', formula='', name='melatonin [c]', compartment='')
Cpd271 = Metabolite('Cpd271', formula='', name='hydroxyacetone [c]', compartment='')
Cpd272 = Metabolite('Cpd272', formula='', name='3-Phospho-D-glyceroyl phosphate [c]', compartment='')
Cpd273 = Metabolite('Cpd273', formula='', name='3-Phospho-D-glycerate [c]', compartment='')
Cpd274 = Metabolite('Cpd274', formula='', name='Adenosine [c]', compartment='')
Cpd275 = Metabolite('Cpd275', formula='', name='Inosine [c]', compartment='')
Cpd276 = Metabolite('Cpd276', formula='', name='Adenosine [e]', compartment='')
Cpd277 = Metabolite('Cpd277', formula='', name='Inosine [e]', compartment='')
Cpd278 = Metabolite('Cpd278', formula='', name='Ammonium [e]', compartment='')
Cpd279 = Metabolite('Cpd279', formula='', name='Adenine [e]', compartment='')
Cpd280 = Metabolite('Cpd280', formula='', name='Adenine [c]', compartment='')
Cpd281 = Metabolite('Cpd281', formula='', name='Adenine [l]', compartment='')
Cpd282 = Metabolite('Cpd282', formula='', name='GTP [c]', compartment='')
Cpd283 = Metabolite('Cpd283', formula='', name='GDP [c]', compartment='')
Cpd284 = Metabolite('Cpd284', formula='', name='GTP [m]', compartment='')
Cpd285 = Metabolite('Cpd285', formula='', name='GDP [m]', compartment='')
Cpd286 = Metabolite('Cpd286', formula='', name='dAMP(2-) [c]', compartment='')
Cpd287 = Metabolite('Cpd287', formula='', name='dATP [c]', compartment='')
Cpd288 = Metabolite('Cpd288', formula='', name='dADP [c]', compartment='')
Cpd289 = Metabolite('Cpd289', formula='', name='S-Adenosylmethioninamine [c]', compartment='')
Cpd290 = Metabolite('Cpd290', formula='', name='3,5-cyclic AMP(1-) [c]', compartment='')
Cpd291 = Metabolite('Cpd291', formula='', name='Adenosine [m]', compartment='')
Cpd292 = Metabolite('Cpd292', formula='', name='ADP alpha-D-glucoside(2-) [c]', compartment='')
Cpd293 = Metabolite('Cpd293', formula='', name='D-Glucose 1-phosphate [c]', compartment='')
Cpd294 = Metabolite('Cpd294', formula='', name='ADPmannose [c]', compartment='')
Cpd295 = Metabolite('Cpd295', formula='', name='D-Mannose 1-phosphate [c]', compartment='')
Cpd296 = Metabolite('Cpd296', formula='', name='ADPribose [c]', compartment='')
Cpd297 = Metabolite('Cpd297', formula='', name='alpha-D-Ribose 5-phosphate [c]', compartment='')
Cpd298 = Metabolite('Cpd298', formula='', name='ADPribose [m]', compartment='')
Cpd299 = Metabolite('Cpd299', formula='', name='alpha-D-Ribose 5-phosphate [m]', compartment='')
Cpd300 = Metabolite('Cpd300', formula='', name='ADPribose [e]', compartment='')
Cpd301 = Metabolite('Cpd301', formula='', name='5-Phospho-alpha-D-ribose 1-diphosphate [c]', compartment='')
Cpd302 = Metabolite('Cpd302', formula='', name='adrenyl coenzyme A [c]', compartment='')
Cpd303 = Metabolite('Cpd303', formula='', name='L-Carnitine [c]', compartment='')
Cpd304 = Metabolite('Cpd304', formula='', name='Adrenyl carnitine [c]', compartment='')
Cpd305 = Metabolite('Cpd305', formula='', name='Adrenyl carnitine [m]', compartment='')
Cpd306 = Metabolite('Cpd306', formula='', name='adrenyl coenzyme A [m]', compartment='')
Cpd307 = Metabolite('Cpd307', formula='', name='L-Carnitine [m]', compartment='')
Cpd308 = Metabolite('Cpd308', formula='', name='Adrenaline [e]', compartment='')
Cpd309 = Metabolite('Cpd309', formula='', name='Adenylylselenate [c]', compartment='')
Cpd310 = Metabolite('Cpd310', formula='', name='3-Phosphoadenylylselenate [c]', compartment='')
Cpd311 = Metabolite('Cpd311', formula='', name='5-adenylyl sulfate(2-) [c]', compartment='')
Cpd312 = Metabolite('Cpd312', formula='', name='N6-(1,2-Dicarboxyethyl)-AMP [c]', compartment='')
Cpd313 = Metabolite('Cpd313', formula='', name='Fumarate [c]', compartment='')
Cpd314 = Metabolite('Cpd314', formula='', name='(S)-2-[5-Amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxamido]succinate [c]', compartment='')
Cpd315 = Metabolite('Cpd315', formula='', name='5-Amino-1-(5-Phospho-D-ribosyl)imidazole-4-carboxamide [c]', compartment='')
Cpd316 = Metabolite('Cpd316', formula='', name='IMP [c]', compartment='')
Cpd317 = Metabolite('Cpd317', formula='', name='keratan sulfate I biosynthesis, precursor 18 [g]', compartment='')
Cpd318 = Metabolite('Cpd318', formula='', name='keratan sulfate I biosynthesis, precursor 19 [g]', compartment='')
Cpd319 = Metabolite('Cpd319', formula='', name='keratan sulfate I biosynthesis, precursor 21 [g]', compartment='')
Cpd320 = Metabolite('Cpd320', formula='', name='keratan sulfate I biosynthesis, precursor 22 [g]', compartment='')
Cpd321 = Metabolite('Cpd321', formula='', name='keratan sulfate I biosynthesis, precursor 24 [g]', compartment='')
Cpd322 = Metabolite('Cpd322', formula='', name='keratan sulfate I biosynthesis, precursor 25 [g]', compartment='')
Cpd323 = Metabolite('Cpd323', formula='', name='keratan sulfate I biosynthesis, precursor 27 [g]', compartment='')
Cpd324 = Metabolite('Cpd324', formula='', name='keratan sulfate I biosynthesis, precursor 28 [g]', compartment='')
Cpd325 = Metabolite('Cpd325', formula='', name='keratan sulfate I biosynthesis, precursor 30 [g]', compartment='')
Cpd326 = Metabolite('Cpd326', formula='', name='keratan sulfate I biosynthesis, precursor 31 [g]', compartment='')
Cpd327 = Metabolite('Cpd327', formula='', name='keratan sulfate I biosynthesis, precursor 33 [g]', compartment='')
Cpd328 = Metabolite('Cpd328', formula='', name='keratan sulfate I biosynthesis, precursor 34 [g]', compartment='')
Cpd329 = Metabolite('Cpd329', formula='', name='keratan sulfate II boisynthesis, precursor 2 [g]', compartment='')
Cpd330 = Metabolite('Cpd330', formula='', name='keratan sulfate II biosynthesis, precursor 3 [g]', compartment='')
Cpd331 = Metabolite('Cpd331', formula='', name='keratan sulfate II biosynthesis, precursor 4 [g]', compartment='')
Cpd332 = Metabolite('Cpd332', formula='', name='keratan sulfate II biosynthesis, precursor 5 [g]', compartment='')
Cpd333 = Metabolite('Cpd333', formula='', name='keratan sulfate II biosynthesis, precursor 7 [g]', compartment='')
Cpd334 = Metabolite('Cpd334', formula='', name='keratan sulfate II biosynthesis, precursor 8 [g]', compartment='')
Cpd335 = Metabolite('Cpd335', formula='', name='keratan sulfate II biosynthesis, precursor 2 [g]', compartment='')
Cpd336 = Metabolite('Cpd336', formula='', name='keratan sulfate I biosynthesis, precursor 1 [g]', compartment='')
Cpd337 = Metabolite('Cpd337', formula='', name='keratan sulfate I biosynthesis, precursor 2 [g]', compartment='')
Cpd338 = Metabolite('Cpd338', formula='', name='keratan sulfate I biosynthesis, precursor 3 [g]', compartment='')
Cpd339 = Metabolite('Cpd339', formula='', name='keratan sulfate I biosynthesis, precursor 4 [g]', compartment='')
Cpd340 = Metabolite('Cpd340', formula='', name='keratan sulfate I biosynthesis, precursor 6 [g]', compartment='')
Cpd341 = Metabolite('Cpd341', formula='', name='keratan sulfate I biosynthesis, precursor 7 [g]', compartment='')
Cpd342 = Metabolite('Cpd342', formula='', name='keratan sulfate I biosynthesis, precursor 9 [g]', compartment='')
Cpd343 = Metabolite('Cpd343', formula='', name='keratan sulfate I biosynthesis, precursor 10 [g]', compartment='')
Cpd344 = Metabolite('Cpd344', formula='', name='keratan sulfate I biosynthesis, precursor 12 [g]', compartment='')
Cpd345 = Metabolite('Cpd345', formula='', name='keratan sulfate I biosynthesis, precursor 13 [g]', compartment='')
Cpd346 = Metabolite('Cpd346', formula='', name='keratan sulfate I biosynthesis, precursor 15 [g]', compartment='')
Cpd347 = Metabolite('Cpd347', formula='', name='keratan sulfate I biosynthesis, precursor 16 [g]', compartment='')
Cpd348 = Metabolite('Cpd348', formula='', name='agmatinium(2+) [m]', compartment='')
Cpd349 = Metabolite('Cpd349', formula='', name='Putrescine [m]', compartment='')
Cpd350 = Metabolite('Cpd350', formula='', name='Urea [m]', compartment='')
Cpd351 = Metabolite('Cpd351', formula='', name='R total 2 coenzyme A [c]', compartment='')
Cpd352 = Metabolite('Cpd352', formula='', name='lysophosphatidic acid [c]', compartment='')
Cpd353 = Metabolite('Cpd353', formula='', name='phosphatidate(2-) [c]', compartment='')
Cpd354 = Metabolite('Cpd354', formula='', name='N-acetyl-L-gamma-glutamyl phosphate(3-) [m]', compartment='')
Cpd355 = Metabolite('Cpd355', formula='', name='2-acetamido-5-oxopentanoate [m]', compartment='')
Cpd356 = Metabolite('Cpd356', formula='', name='acyl-dihydroxyacetone phosphate [x]', compartment='')
Cpd357 = Metabolite('Cpd357', formula='', name='hydroxy alkyl chain [x]', compartment='')
Cpd358 = Metabolite('Cpd358', formula='', name='proton [x]', compartment='')
Cpd359 = Metabolite('Cpd359', formula='', name='1-alkyldihydroxyacetone phosphate [x]', compartment='')
Cpd360 = Metabolite('Cpd360', formula='', name='R total [x]', compartment='')
Cpd361 = Metabolite('Cpd361', formula='', name='L-alanine [m]', compartment='')
Cpd362 = Metabolite('Cpd362', formula='', name='glyoxylate [m]', compartment='')
Cpd363 = Metabolite('Cpd363', formula='', name='pyruvate [m]', compartment='')
Cpd364 = Metabolite('Cpd364', formula='', name='Glycine [m]', compartment='')
Cpd365 = Metabolite('Cpd365', formula='', name='L-alanine [x]', compartment='')
Cpd366 = Metabolite('Cpd366', formula='', name='glyoxylate [x]', compartment='')
Cpd367 = Metabolite('Cpd367', formula='', name='Glycine [x]', compartment='')
Cpd368 = Metabolite('Cpd368', formula='', name='pyruvate [x]', compartment='')
Cpd369 = Metabolite('Cpd369', formula='', name='3alpha-hydroxy-5beta-androstan-17-one 3-glucosiduronic acid [c]', compartment='')
Cpd370 = Metabolite('Cpd370', formula='', name='3alpha-hydroxy-5beta-androstan-17-one 3-glucosiduronic acid [e]', compartment='')
Cpd371 = Metabolite('Cpd371', formula='', name='L-homocysteine [c]', compartment='')
Cpd372 = Metabolite('Cpd372', formula='', name='reducing GlcNAc removed form of n2m2nmasn (w/o peptide) [l]', compartment='')
Cpd373 = Metabolite('Cpd373', formula='', name='N-acetyl-D-glucosamine [l]', compartment='')
Cpd374 = Metabolite('Cpd374', formula='', name='de-Fuc, reducing GlcNAc removed, de-Sia, de-Gal form of PA6 (w/o peptide linkage) [l]', compartment='')
Cpd375 = Metabolite('Cpd375', formula='', name='10-Formyltetrahydrofolate [c]', compartment='')
Cpd376 = Metabolite('Cpd376', formula='', name='5-Formamido-1-(5-phospho-D-ribosyl)imidazole-4-carboxamide [c]', compartment='')
Cpd377 = Metabolite('Cpd377', formula='', name='5,6,7,8-Tetrahydrofolate [c]', compartment='')
Cpd378 = Metabolite('Cpd378', formula='', name='5-amino-1-(5-phospho-D-ribosyl)imidazole [c]', compartment='')
Cpd379 = Metabolite('Cpd379', formula='', name='5-amino-1-(5-phospho-D-ribosyl)imidazole-4-carboxylate [c]', compartment='')
Cpd380 = Metabolite('Cpd380', formula='', name='Succinyl-CoA [m]', compartment='')
Cpd381 = Metabolite('Cpd381', formula='', name='(S)-malate(2-) [c]', compartment='')
Cpd382 = Metabolite('Cpd382', formula='', name='(S)-malate(2-) [m]', compartment='')
Cpd383 = Metabolite('Cpd383', formula='', name='2-Oxoglutarate [e]', compartment='')
Cpd384 = Metabolite('Cpd384', formula='', name='progesterone [c]', compartment='')
Cpd385 = Metabolite('Cpd385', formula='', name='(20S)-20-hydroxypregn-4-en-3-one [c]', compartment='')
Cpd386 = Metabolite('Cpd386', formula='', name='7alpha-hydroxy-5beta-cholestan-3-one [c]', compartment='')
Cpd387 = Metabolite('Cpd387', formula='', name='5beta-cholestane-3alpha,7alpha-diol [c]', compartment='')
Cpd388 = Metabolite('Cpd388', formula='', name='7alpha,12alpha-dihydroxy-5beta-cholestan-3-one [c]', compartment='')
Cpd389 = Metabolite('Cpd389', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha-triol [c]', compartment='')
Cpd390 = Metabolite('Cpd390', formula='', name='7alpha-hydroxycholest-4-en-3-one [c]', compartment='')
Cpd391 = Metabolite('Cpd391', formula='', name='7alpha,12alpha-Dihydroxycholest-4-en-3-one [c]', compartment='')
Cpd392 = Metabolite('Cpd392', formula='', name='L-alanine [e]', compartment='')
Cpd393 = Metabolite('Cpd393', formula='', name='L-asparagine [c]', compartment='')
Cpd394 = Metabolite('Cpd394', formula='', name='L-alanine [c]', compartment='')
Cpd395 = Metabolite('Cpd395', formula='', name='L-asparagine [e]', compartment='')
Cpd396 = Metabolite('Cpd396', formula='', name='L-cysteine [c]', compartment='')
Cpd397 = Metabolite('Cpd397', formula='', name='L-cysteine [e]', compartment='')
Cpd398 = Metabolite('Cpd398', formula='', name='D-alanine [e]', compartment='')
Cpd399 = Metabolite('Cpd399', formula='', name='L-glutamine [c]', compartment='')
Cpd400 = Metabolite('Cpd400', formula='', name='D-alanine [c]', compartment='')
Cpd401 = Metabolite('Cpd401', formula='', name='L-glutamine [e]', compartment='')
Cpd402 = Metabolite('Cpd402', formula='', name='Glycine [c]', compartment='')
Cpd403 = Metabolite('Cpd403', formula='', name='Glycine [e]', compartment='')
Cpd404 = Metabolite('Cpd404', formula='', name='L-serine [c]', compartment='')
Cpd405 = Metabolite('Cpd405', formula='', name='L-serine [e]', compartment='')
Cpd406 = Metabolite('Cpd406', formula='', name='5-Amino-4-oxopentanoate [m]', compartment='')
Cpd407 = Metabolite('Cpd407', formula='', name='L-alanine [l]', compartment='')
Cpd408 = Metabolite('Cpd408', formula='', name='L-threonine [c]', compartment='')
Cpd409 = Metabolite('Cpd409', formula='', name='L-threonine [e]', compartment='')
Cpd410 = Metabolite('Cpd410', formula='', name='methanol [c]', compartment='')
Cpd411 = Metabolite('Cpd411', formula='', name='formaldehyde [c]', compartment='')
Cpd412 = Metabolite('Cpd412', formula='', name='(R)-propane-1,2-diol [c]', compartment='')
Cpd413 = Metabolite('Cpd413', formula='', name='(R)-lactaldehyde [c]', compartment='')
Cpd414 = Metabolite('Cpd414', formula='', name='(S)-propane-1,2-diol [c]', compartment='')
Cpd415 = Metabolite('Cpd415', formula='', name='(S)-lactaldehyde [c]', compartment='')
Cpd416 = Metabolite('Cpd416', formula='', name='ethanol [c]', compartment='')
Cpd417 = Metabolite('Cpd417', formula='', name='acetaldehyde [c]', compartment='')
Cpd418 = Metabolite('Cpd418', formula='', name='indol-3-ylacetaldehyde [c]', compartment='')
Cpd419 = Metabolite('Cpd419', formula='', name='indole-3-acetate [c]', compartment='')
Cpd420 = Metabolite('Cpd420', formula='', name='indol-3-ylacetaldehyde [m]', compartment='')
Cpd421 = Metabolite('Cpd421', formula='', name='indole-3-acetate [m]', compartment='')
Cpd422 = Metabolite('Cpd422', formula='', name='pristanal [c]', compartment='')
Cpd423 = Metabolite('Cpd423', formula='', name='pristanic acid [c]', compartment='')
Cpd424 = Metabolite('Cpd424', formula='', name='acetaldehyde [m]', compartment='')
Cpd425 = Metabolite('Cpd425', formula='', name='Dihydroxyacetone phosphate [c]', compartment='')
Cpd426 = Metabolite('Cpd426', formula='', name='glycerone [c]', compartment='')
Cpd427 = Metabolite('Cpd427', formula='', name='arachidonate [c]', compartment='')
Cpd428 = Metabolite('Cpd428', formula='', name='12-Hydroperoxyeicosa-5,8,10,14-tetraenoate [c]', compartment='')
Cpd429 = Metabolite('Cpd429', formula='', name='12R-Hydroperoxyeicosatetraenoate [c]', compartment='')
Cpd430 = Metabolite('Cpd430', formula='', name='15(S)-HPETE(1-) [c]', compartment='')
Cpd431 = Metabolite('Cpd431', formula='', name='5(S)-HPETE(1-) [c]', compartment='')
Cpd432 = Metabolite('Cpd432', formula='', name='leukotriene A4(1-) [c]', compartment='')
Cpd433 = Metabolite('Cpd433', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholestanoyl-CoA [x]', compartment='')
Cpd434 = Metabolite('Cpd434', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholestanoyl-CoA [r]', compartment='')
Cpd435 = Metabolite('Cpd435', formula='', name='O2 [r]', compartment='')
Cpd436 = Metabolite('Cpd436', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholest-24-enoyl-CoA [r]', compartment='')
Cpd437 = Metabolite('Cpd437', formula='', name='Water [r]', compartment='')
Cpd438 = Metabolite('Cpd438', formula='', name='(25R)-3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-26-oyl-CoA(4-) [x]', compartment='')
Cpd439 = Metabolite('Cpd439', formula='', name='(25R)-3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-26-oyl-CoA(4-) [r]', compartment='')
Cpd440 = Metabolite('Cpd440', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoyl-CoA(S) [r]', compartment='')
Cpd441 = Metabolite('Cpd441', formula='', name='S-Adenosyl-L-homocysteine [m]', compartment='')
Cpd442 = Metabolite('Cpd442', formula='', name='S-Adenosyl-L-methionine [m]', compartment='')
Cpd443 = Metabolite('Cpd443', formula='', name='L-cysteinylglycine [c]', compartment='')
Cpd444 = Metabolite('Cpd444', formula='', name='L-cysteinylglycine [e]', compartment='')
Cpd445 = Metabolite('Cpd445', formula='', name='starch, structure 1 (1,6-{7[1,4-Glc], 4[1,4-Glc]}) [e]', compartment='')
Cpd446 = Metabolite('Cpd446', formula='', name='D-glucose [e]', compartment='')
Cpd447 = Metabolite('Cpd447', formula='', name='starch, structure 2 (1,6-{2[1,4-Glc], [1,4-Glc]}) [e]', compartment='')
Cpd448 = Metabolite('Cpd448', formula='', name='glycogen, structure 2 (glycogenin-1,6-{7[1,4-Glc], 4[1,4-Glc]}) [e]', compartment='')
Cpd449 = Metabolite('Cpd449', formula='', name='glycogen, structure 4 (glycogenin-1,6-{2[1,4-Glc], [1,4-Glc]}) [e]', compartment='')
Cpd450 = Metabolite('Cpd450', formula='', name='androsterone 3-glucosiduronic acid [c]', compartment='')
Cpd451 = Metabolite('Cpd451', formula='', name='androsterone 3-glucosiduronic acid [e]', compartment='')
Cpd452 = Metabolite('Cpd452', formula='', name='P(1),P(4)-bis(5-adenosyl) tetraphosphate(4-) [c]', compartment='')
Cpd453 = Metabolite('Cpd453', formula='', name='beta-alanine [m]', compartment='')
Cpd454 = Metabolite('Cpd454', formula='', name='3-oxopropanoate [m]', compartment='')
Cpd455 = Metabolite('Cpd455', formula='', name='N-acetylputrescinium [c]', compartment='')
Cpd456 = Metabolite('Cpd456', formula='', name='4-acetamidobutanal [c]', compartment='')
Cpd457 = Metabolite('Cpd457', formula='', name='L-arabinose [c]', compartment='')
Cpd458 = Metabolite('Cpd458', formula='', name='L-arabinitol [c]', compartment='')
Cpd459 = Metabolite('Cpd459', formula='', name='arachidyl coenzyme A [c]', compartment='')
Cpd460 = Metabolite('Cpd460', formula='', name='arachidyl carnitine [c]', compartment='')
Cpd461 = Metabolite('Cpd461', formula='', name='arachidyl carnitine [m]', compartment='')
Cpd462 = Metabolite('Cpd462', formula='', name='arachidyl coenzyme A [m]', compartment='')
Cpd463 = Metabolite('Cpd463', formula='', name='arachidonate [e]', compartment='')
Cpd464 = Metabolite('Cpd464', formula='', name='L-argininium(1+) [m]', compartment='')
Cpd465 = Metabolite('Cpd465', formula='', name='L-lysinium(1+) [c]', compartment='')
Cpd466 = Metabolite('Cpd466', formula='', name='L-argininium(1+) [e]', compartment='')
Cpd467 = Metabolite('Cpd467', formula='', name='L-argininium(1+) [c]', compartment='')
Cpd468 = Metabolite('Cpd468', formula='', name='L-lysinium(1+) [e]', compartment='')
Cpd469 = Metabolite('Cpd469', formula='', name='Urea [c]', compartment='')
Cpd470 = Metabolite('Cpd470', formula='', name='Ornithine [m]', compartment='')
Cpd471 = Metabolite('Cpd471', formula='', name='L-citrulline [c]', compartment='')
Cpd472 = Metabolite('Cpd472', formula='', name='N(omega)-(L-Arginino)succinate [c]', compartment='')
Cpd473 = Metabolite('Cpd473', formula='', name='sulfatide galactocerebroside [l]', compartment='')
Cpd474 = Metabolite('Cpd474', formula='', name='D-galactosyl-N-acylsphingosine [l]', compartment='')
Cpd475 = Metabolite('Cpd475', formula='', name='sulfate [l]', compartment='')
Cpd476 = Metabolite('Cpd476', formula='', name='N-acylsphingosine [l]', compartment='')
Cpd477 = Metabolite('Cpd477', formula='', name='R total [l]', compartment='')
Cpd478 = Metabolite('Cpd478', formula='', name='sphingosine(1+) [l]', compartment='')
Cpd479 = Metabolite('Cpd479', formula='', name='L-ascorbate [e]', compartment='')
Cpd480 = Metabolite('Cpd480', formula='', name='L-ascorbate [c]', compartment='')
Cpd481 = Metabolite('Cpd481', formula='', name='L-asparagine [m]', compartment='')
Cpd482 = Metabolite('Cpd482', formula='', name='Ammonium [m]', compartment='')
Cpd483 = Metabolite('Cpd483', formula='', name='beta-alanine [c]', compartment='')
Cpd484 = Metabolite('Cpd484', formula='', name='Carbamoyl phosphate [c]', compartment='')
Cpd485 = Metabolite('Cpd485', formula='', name='N-Carbamoyl-L-aspartate [c]', compartment='')
Cpd486 = Metabolite('Cpd486', formula='', name='D-aspartate(1-) [e]', compartment='')
Cpd487 = Metabolite('Cpd487', formula='', name='potassium [c]', compartment='')
Cpd488 = Metabolite('Cpd488', formula='', name='D-aspartate(1-) [c]', compartment='')
Cpd489 = Metabolite('Cpd489', formula='', name='potassium [e]', compartment='')
Cpd490 = Metabolite('Cpd490', formula='', name='L-aspartate(1-) [e]', compartment='')
Cpd491 = Metabolite('Cpd491', formula='', name='Oxaloacetate [m]', compartment='')
Cpd492 = Metabolite('Cpd492', formula='', name='ATP [e]', compartment='')
Cpd493 = Metabolite('Cpd493', formula='', name='AMP [e]', compartment='')
Cpd494 = Metabolite('Cpd494', formula='', name='hydrogenphosphate [e]', compartment='')
Cpd495 = Metabolite('Cpd495', formula='', name='ADP [e]', compartment='')
Cpd496 = Metabolite('Cpd496', formula='', name='(+)-alpha-tocopherol [e]', compartment='')
Cpd497 = Metabolite('Cpd497', formula='', name='(+)-alpha-tocopherol [c]', compartment='')
Cpd498 = Metabolite('Cpd498', formula='', name='globoside [g]', compartment='')
Cpd499 = Metabolite('Cpd499', formula='', name='GA2 [g]', compartment='')
Cpd500 = Metabolite('Cpd500', formula='', name='gibberellin A1(1-) [g]', compartment='')
Cpd501 = Metabolite('Cpd501', formula='', name='N-acetyl-beta-D-galactosaminyl-(1->4)-[alpha-N-acetylneuraminosyl-(2->3)]-beta-D-galactosyl-beta-D-glucosyl-(1<->1)-ceramide anion [g]', compartment='')
Cpd502 = Metabolite('Cpd502', formula='', name='ganglioside GM1 [g]', compartment='')
Cpd503 = Metabolite('Cpd503', formula='', name='GD2 [g]', compartment='')
Cpd504 = Metabolite('Cpd504', formula='', name='GD1b [g]', compartment='')
Cpd505 = Metabolite('Cpd505', formula='', name='GT2 [g]', compartment='')
Cpd506 = Metabolite('Cpd506', formula='', name='GT1c [g]', compartment='')
Cpd507 = Metabolite('Cpd507', formula='', name='galactosylgloboside [g]', compartment='')
Cpd508 = Metabolite('Cpd508', formula='', name='Lactoneotetraosylceramide [g]', compartment='')
Cpd509 = Metabolite('Cpd509', formula='', name='nLc5Cer [g]', compartment='')
Cpd510 = Metabolite('Cpd510', formula='', name='i-antigen [g]', compartment='')
Cpd511 = Metabolite('Cpd511', formula='', name='nLc7Cer [g]', compartment='')
Cpd512 = Metabolite('Cpd512', formula='', name='N-acetyl-beta-D-glucosaminyl-(1->3)-beta-D-galactosyl-(1->4)-beta-D-glucosylceramide [g]', compartment='')
Cpd513 = Metabolite('Cpd513', formula='', name='(Gal)3 (GalNAc)1 (Glc)1 (GlcNAc)1 (LFuc)1 (Cer)1 [g]', compartment='')
Cpd514 = Metabolite('Cpd514', formula='', name='(Gal)4 (GalNAc)1 (Glc)1 (GlcNAc)2 (LFuc)1 (Cer)1 [g]', compartment='')
Cpd515 = Metabolite('Cpd515', formula='', name='(Gal)3 (Glc)1 (GlcNAc)3 (Cer)1 [g]', compartment='')
Cpd516 = Metabolite('Cpd516', formula='', name='I-antigen [g]', compartment='')
Cpd517 = Metabolite('Cpd517', formula='', name='nLc8Cer [g]', compartment='')
Cpd518 = Metabolite('Cpd518', formula='', name='Gal-Gal-Gal-Glc-Cer [g]', compartment='')
Cpd519 = Metabolite('Cpd519', formula='', name='Gal-Gal-Gal-Gal-Glc-Cer [g]', compartment='')
Cpd520 = Metabolite('Cpd520', formula='', name='Gal-Gal-Gal-Gal-Gal-Glc-Cer [g]', compartment='')
Cpd521 = Metabolite('Cpd521', formula='', name='GlcNAc-Gal globoside [g]', compartment='')
Cpd522 = Metabolite('Cpd522', formula='', name='Gal-GlcNAc-Gal globoside [g]', compartment='')
Cpd523 = Metabolite('Cpd523', formula='', name='N-acetyl-D-galactosamine [g]', compartment='')
Cpd524 = Metabolite('Cpd524', formula='', name='beta GalNAc globoside [g]', compartment='')
Cpd525 = Metabolite('Cpd525', formula='', name='Water [g]', compartment='')
Cpd526 = Metabolite('Cpd526', formula='', name='GM2alpha [g]', compartment='')
Cpd527 = Metabolite('Cpd527', formula='', name='GM1alpha [g]', compartment='')
Cpd528 = Metabolite('Cpd528', formula='', name='D-glucosyl-N-acylsphingosine [g]', compartment='')
Cpd529 = Metabolite('Cpd529', formula='', name='choloyl-CoA(4-) [x]', compartment='')
Cpd530 = Metabolite('Cpd530', formula='', name='glycocholate [x]', compartment='')
Cpd531 = Metabolite('Cpd531', formula='', name='Taurine [x]', compartment='')
Cpd532 = Metabolite('Cpd532', formula='', name='taurocholate [x]', compartment='')
Cpd533 = Metabolite('Cpd533', formula='', name='Chenodeoxyglycocholoyl-CoA [x]', compartment='')
Cpd534 = Metabolite('Cpd534', formula='', name='glycochenodeoxycholate [x]', compartment='')
Cpd535 = Metabolite('Cpd535', formula='', name='chenodeoxycholoyl coenzyme a [x]', compartment='')
Cpd536 = Metabolite('Cpd536', formula='', name='Biotin [c]', compartment='')
Cpd537 = Metabolite('Cpd537', formula='', name='Biotinyl-5-AMP [c]', compartment='')
Cpd538 = Metabolite('Cpd538', formula='', name='Biotin [m]', compartment='')
Cpd539 = Metabolite('Cpd539', formula='', name='Biotinyl-5-AMP [m]', compartment='')
Cpd540 = Metabolite('Cpd540', formula='', name='beta-alanine [e]', compartment='')
Cpd541 = Metabolite('Cpd541', formula='', name='beta-Aminopropion aldehyde [m]', compartment='')
Cpd542 = Metabolite('Cpd542', formula='', name='4-(trimethylammonio)butanoate [c]', compartment='')
Cpd543 = Metabolite('Cpd543', formula='', name='Succinate [c]', compartment='')
Cpd544 = Metabolite('Cpd544', formula='', name='beta-carotene [c]', compartment='')
Cpd545 = Metabolite('Cpd545', formula='', name='retinal [c]', compartment='')
Cpd546 = Metabolite('Cpd546', formula='', name='(R)-3-hydroxybutyrate [m]', compartment='')
Cpd547 = Metabolite('Cpd547', formula='', name='N,N-Diacetylchitobiosyldiphosphodolichol, human liver homolog [c]', compartment='')
Cpd548 = Metabolite('Cpd548', formula='', name='GDP-alpha-D-mannose(2-) [c]', compartment='')
Cpd549 = Metabolite('Cpd549', formula='', name='beta-D-Mannosyldiacetylchitobiosyldiphosphodolichol, human liver homolog [c]', compartment='')
Cpd550 = Metabolite('Cpd550', formula='', name='N,N-Diacetylchitobiosyldiphosphodolichol, human uterine homolog [c]', compartment='')
Cpd551 = Metabolite('Cpd551', formula='', name='beta-D-Mannosyldiacetylchitobiosyldiphosphodolichol, human uterine homolog [c]', compartment='')
Cpd552 = Metabolite('Cpd552', formula='', name='glycine betaine [c]', compartment='')
Cpd553 = Metabolite('Cpd553', formula='', name='N,N-dimethylglycine [c]', compartment='')
Cpd554 = Metabolite('Cpd554', formula='', name='L-methionine [c]', compartment='')
Cpd555 = Metabolite('Cpd555', formula='', name='bis(beta-glucosyluronate)bilirubin [e]', compartment='')
Cpd556 = Metabolite('Cpd556', formula='', name='bis(beta-glucosyluronate)bilirubin [c]', compartment='')
Cpd557 = Metabolite('Cpd557', formula='', name='mono(glucosyluronic acid)bilirubin(2-) [e]', compartment='')
Cpd558 = Metabolite('Cpd558', formula='', name='mono(glucosyluronic acid)bilirubin(2-) [c]', compartment='')
Cpd559 = Metabolite('Cpd559', formula='', name='biliverdin(2-) [c]', compartment='')
Cpd560 = Metabolite('Cpd560', formula='', name='bilirubin(2-) [c]', compartment='')
Cpd561 = Metabolite('Cpd561', formula='', name='bilirubin(2-) [e]', compartment='')
Cpd562 = Metabolite('Cpd562', formula='', name='Dolichyl phosphate D-mannose, human liver homolog [r]', compartment='')
Cpd563 = Metabolite('Cpd563', formula='', name='(mannosyl),(phosphoethanolaminyl)-mannosyl-glucosylaminyl-acylphosphatidylinositol (B) [r]', compartment='')
Cpd564 = Metabolite('Cpd564', formula='', name='dolichyl phosphate(2-) [r]', compartment='')
Cpd565 = Metabolite('Cpd565', formula='', name='proton [r]', compartment='')
Cpd566 = Metabolite('Cpd566', formula='', name='(dimannosyl),(phosphoethanolaminyl)-mannosyl-glucosylaminyl-acylphosphatidylinositol (H6) [r]', compartment='')
Cpd567 = Metabolite('Cpd567', formula='', name='Dolichyl phosphate D-mannose, human uterine homolog [r]', compartment='')
Cpd568 = Metabolite('Cpd568', formula='', name='Biocytin [c]', compartment='')
Cpd569 = Metabolite('Cpd569', formula='', name='Biocytin [n]', compartment='')
Cpd570 = Metabolite('Cpd570', formula='', name='Water [n]', compartment='')
Cpd571 = Metabolite('Cpd571', formula='', name='Biotin [n]', compartment='')
Cpd572 = Metabolite('Cpd572', formula='', name='L-lysinium(1+) [n]', compartment='')
Cpd573 = Metabolite('Cpd573', formula='', name='Biocytin [e]', compartment='')
Cpd574 = Metabolite('Cpd574', formula='', name='Biotin [e]', compartment='')
Cpd575 = Metabolite('Cpd575', formula='', name='Biocytin [m]', compartment='')
Cpd576 = Metabolite('Cpd576', formula='', name='L-lysinium(1+) [m]', compartment='')
Cpd577 = Metabolite('Cpd577', formula='', name='Apocarboxylase (Lys residue) [c]', compartment='')
Cpd578 = Metabolite('Cpd578', formula='', name='Holocarboxylase (biotin covalent bound to Lys residue of apoC) [c]', compartment='')
Cpd579 = Metabolite('Cpd579', formula='', name='Apocarboxylase (Lys residue) [m]', compartment='')
Cpd580 = Metabolite('Cpd580', formula='', name='Holocarboxylase (biotin covalent bound to Lys residue of apoC) [m]', compartment='')
Cpd581 = Metabolite('Cpd581', formula='', name='3-Ureidoisobutyrate [c]', compartment='')
Cpd582 = Metabolite('Cpd582', formula='', name='D-3-Amino-isobutanoate [c]', compartment='')
Cpd583 = Metabolite('Cpd583', formula='', name='butyrate [c]', compartment='')
Cpd584 = Metabolite('Cpd584', formula='', name='butyrate [m]', compartment='')
Cpd585 = Metabolite('Cpd585', formula='', name='butyrate [e]', compartment='')
Cpd586 = Metabolite('Cpd586', formula='', name='4,4-dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol [r]', compartment='')
Cpd587 = Metabolite('Cpd587', formula='', name='Nicotinamide adenine dinucleotide phosphate - reduced [r]', compartment='')
Cpd588 = Metabolite('Cpd588', formula='', name='14-demethyllanosterol [r]', compartment='')
Cpd589 = Metabolite('Cpd589', formula='', name='Nicotinamide adenine dinucleotide phosphate [r]', compartment='')
Cpd590 = Metabolite('Cpd590', formula='', name='Palmitoyl-CoA (n-C16:0CoA) [c]', compartment='')
Cpd591 = Metabolite('Cpd591', formula='', name='L-palmitoylcarnitine [c]', compartment='')
Cpd592 = Metabolite('Cpd592', formula='', name='L-palmitoylcarnitine [m]', compartment='')
Cpd593 = Metabolite('Cpd593', formula='', name='Palmitoyl-CoA (n-C16:0CoA) [m]', compartment='')
Cpd594 = Metabolite('Cpd594', formula='', name='Hexadecenoyl-CoA (n-C16:1CoA) [c]', compartment='')
Cpd595 = Metabolite('Cpd595', formula='', name='Hexadecenoyl carnitine [c]', compartment='')
Cpd596 = Metabolite('Cpd596', formula='', name='trans-Hexadec-2-enoyl-CoA [c]', compartment='')
Cpd597 = Metabolite('Cpd597', formula='', name='trans-Hexadec-2-enoyl carnitine [c]', compartment='')
Cpd598 = Metabolite('Cpd598', formula='', name='Hexadecenoyl carnitine [m]', compartment='')
Cpd599 = Metabolite('Cpd599', formula='', name='Hexadecenoyl-CoA (n-C16:1CoA) [m]', compartment='')
Cpd600 = Metabolite('Cpd600', formula='', name='trans-Hexadec-2-enoyl carnitine [m]', compartment='')
Cpd601 = Metabolite('Cpd601', formula='', name='trans-Hexadec-2-enoyl-CoA [m]', compartment='')
Cpd602 = Metabolite('Cpd602', formula='', name='Stearoyl-CoA (n-C18:0CoA) [c]', compartment='')
Cpd603 = Metabolite('Cpd603', formula='', name='stearoylcarnitine [c]', compartment='')
Cpd604 = Metabolite('Cpd604', formula='', name='stearoylcarnitine [m]', compartment='')
Cpd605 = Metabolite('Cpd605', formula='', name='Stearoyl-CoA (n-C18:0CoA) [m]', compartment='')
Cpd606 = Metabolite('Cpd606', formula='', name='Octadecenoyl-CoA (n-C18:1CoA) [c]', compartment='')
Cpd607 = Metabolite('Cpd607', formula='', name='octadecenoyl carnitine [c]', compartment='')
Cpd608 = Metabolite('Cpd608', formula='', name='octadecenoyl carnitine [m]', compartment='')
Cpd609 = Metabolite('Cpd609', formula='', name='Octadecenoyl-CoA (n-C18:1CoA) [m]', compartment='')
Cpd610 = Metabolite('Cpd610', formula='', name='arachidonoyl-CoA(4-) [c]', compartment='')
Cpd611 = Metabolite('Cpd611', formula='', name='C20:4 carnitine [c]', compartment='')
Cpd612 = Metabolite('Cpd612', formula='', name='C20:4 carnitine [m]', compartment='')
Cpd613 = Metabolite('Cpd613', formula='', name='arachidonoyl-CoA(4-) [m]', compartment='')
Cpd614 = Metabolite('Cpd614', formula='', name='cervonyl coenzyme A [c]', compartment='')
Cpd615 = Metabolite('Cpd615', formula='', name='cervonyl carnitine [c]', compartment='')
Cpd616 = Metabolite('Cpd616', formula='', name='cervonyl carnitine [m]', compartment='')
Cpd617 = Metabolite('Cpd617', formula='', name='cervonyl coenzyme A [m]', compartment='')
Cpd618 = Metabolite('Cpd618', formula='', name='cis-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA [m]', compartment='')
Cpd619 = Metabolite('Cpd619', formula='', name='3-Hydroxy-2,6-dimethyl-5-methylene-heptanoyl-CoA [m]', compartment='')
Cpd620 = Metabolite('Cpd620', formula='', name='cis-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA [x]', compartment='')
Cpd621 = Metabolite('Cpd621', formula='', name='3-Hydroxy-2,6-dimethyl-5-methylene-heptanoyl-CoA [x]', compartment='')
Cpd622 = Metabolite('Cpd622', formula='', name='4-Methylzymosterol intermediate 1 [r]', compartment='')
Cpd623 = Metabolite('Cpd623', formula='', name='4-Methylzymosterol intermediate 2 [r]', compartment='')
Cpd624 = Metabolite('Cpd624', formula='', name='carbon dioxide [r]', compartment='')
Cpd625 = Metabolite('Cpd625', formula='', name='Nicotinamide adenine dinucleotide [r]', compartment='')
Cpd626 = Metabolite('Cpd626', formula='', name='Nicotinamide adenine dinucleotide - reduced [r]', compartment='')
Cpd627 = Metabolite('Cpd627', formula='', name='zymosterol intermediate 2 [r]', compartment='')
Cpd628 = Metabolite('Cpd628', formula='', name='zymosterol [r]', compartment='')
Cpd629 = Metabolite('Cpd629', formula='', name='calcium(2+) [c]', compartment='')
Cpd630 = Metabolite('Cpd630', formula='', name='calcium(2+) [e]', compartment='')
Cpd631 = Metabolite('Cpd631', formula='', name='3,5-cyclic AMP(1-) [e]', compartment='')
Cpd632 = Metabolite('Cpd632', formula='', name='ethanol [x]', compartment='')
Cpd633 = Metabolite('Cpd633', formula='', name='acetaldehyde [x]', compartment='')
Cpd634 = Metabolite('Cpd634', formula='', name='Hydrogen peroxide [m]', compartment='')
Cpd635 = Metabolite('Cpd635', formula='', name='Cob(II)alamin [c]', compartment='')
Cpd636 = Metabolite('Cpd636', formula='', name='Cob(II)alamin [m]', compartment='')
Cpd637 = Metabolite('Cpd637', formula='', name='Cob(I)alamin [m]', compartment='')
Cpd638 = Metabolite('Cpd638', formula='', name='Adenosylcobalamin [m]', compartment='')
Cpd639 = Metabolite('Cpd639', formula='', name='triphosphate(5-) [m]', compartment='')
Cpd640 = Metabolite('Cpd640', formula='', name='Carbamoyl phosphate [r]', compartment='')
Cpd641 = Metabolite('Cpd641', formula='', name='D-glucose [r]', compartment='')
Cpd642 = Metabolite('Cpd642', formula='', name='D-Glucose 6-phosphate [r]', compartment='')
Cpd643 = Metabolite('Cpd643', formula='', name='Ammonium [r]', compartment='')
Cpd644 = Metabolite('Cpd644', formula='', name='Carbamoyl phosphate [m]', compartment='')
Cpd645 = Metabolite('Cpd645', formula='', name='15-dehydro-prostaglandin E1(1-) [c]', compartment='')
Cpd646 = Metabolite('Cpd646', formula='', name='prostaglandin E1(1-) [c]', compartment='')
Cpd647 = Metabolite('Cpd647', formula='', name='prostaglandin E2(1-) [c]', compartment='')
Cpd648 = Metabolite('Cpd648', formula='', name='prostaglandin F2alpha(1-) [c]', compartment='')
Cpd649 = Metabolite('Cpd649', formula='', name='CDP-diacylglycerol(2-) [c]', compartment='')
Cpd650 = Metabolite('Cpd650', formula='', name='myo-inositol [c]', compartment='')
Cpd651 = Metabolite('Cpd651', formula='', name='CMP [c]', compartment='')
Cpd652 = Metabolite('Cpd652', formula='', name='1-phosphatidyl-1D-myo-inositol(1-) [c]', compartment='')
Cpd653 = Metabolite('Cpd653', formula='', name='CTP [c]', compartment='')
Cpd654 = Metabolite('Cpd654', formula='', name='CTP [m]', compartment='')
Cpd655 = Metabolite('Cpd655', formula='', name='phosphatidate(2-) [m]', compartment='')
Cpd656 = Metabolite('Cpd656', formula='', name='CDP-diacylglycerol(2-) [m]', compartment='')
Cpd657 = Metabolite('Cpd657', formula='', name='CDPcholine [c]', compartment='')
Cpd658 = Metabolite('Cpd658', formula='', name='diglyceride [c]', compartment='')
Cpd659 = Metabolite('Cpd659', formula='', name='Phosphatidylcholine [c]', compartment='')
Cpd660 = Metabolite('Cpd660', formula='', name='CDP-ethanolamine(1-) [c]', compartment='')
Cpd661 = Metabolite('Cpd661', formula='', name='phosphatidylethanolamine [c]', compartment='')
Cpd662 = Metabolite('Cpd662', formula='', name='N-acylsphingosine [c]', compartment='')
Cpd663 = Metabolite('Cpd663', formula='', name='ceramide 1-phosphate(2-) [c]', compartment='')
Cpd664 = Metabolite('Cpd664', formula='', name='N-acylsphingosine [g]', compartment='')
Cpd665 = Metabolite('Cpd665', formula='', name='N-acylsphingosine [r]', compartment='')
Cpd666 = Metabolite('Cpd666', formula='', name='D-glucosyl-N-acylsphingosine [c]', compartment='')
Cpd667 = Metabolite('Cpd667', formula='', name='D-glucosyl-N-acylsphingosine [r]', compartment='')
Cpd668 = Metabolite('Cpd668', formula='', name='3,5-cyclic GMP(1-) [c]', compartment='')
Cpd669 = Metabolite('Cpd669', formula='', name='3,5-cyclic GMP(1-) [e]', compartment='')
Cpd670 = Metabolite('Cpd670', formula='', name='cholesterol [r]', compartment='')
Cpd671 = Metabolite('Cpd671', formula='', name='25-hydroxycholesterol [r]', compartment='')
Cpd672 = Metabolite('Cpd672', formula='', name='Choline [c]', compartment='')
Cpd673 = Metabolite('Cpd673', formula='', name='Acetyl-CoA [n]', compartment='')
Cpd674 = Metabolite('Cpd674', formula='', name='Choline [n]', compartment='')
Cpd675 = Metabolite('Cpd675', formula='', name='acetylcholine [n]', compartment='')
Cpd676 = Metabolite('Cpd676', formula='', name='Coenzyme A [n]', compartment='')
Cpd677 = Metabolite('Cpd677', formula='', name='choline phosphate(1-) [c]', compartment='')
Cpd678 = Metabolite('Cpd678', formula='', name='Cholate [e]', compartment='')
Cpd679 = Metabolite('Cpd679', formula='', name='Cholate [c]', compartment='')
Cpd680 = Metabolite('Cpd680', formula='', name='Choline [m]', compartment='')
Cpd681 = Metabolite('Cpd681', formula='', name='betaine aldehyde [m]', compartment='')
Cpd682 = Metabolite('Cpd682', formula='', name='cholesterol [c]', compartment='')
Cpd683 = Metabolite('Cpd683', formula='', name='cholesterol sulfate [c]', compartment='')
Cpd684 = Metabolite('Cpd684', formula='', name='cholesterol [e]', compartment='')
Cpd685 = Metabolite('Cpd685', formula='', name='cholesterol [l]', compartment='')
Cpd686 = Metabolite('Cpd686', formula='', name='cholesterol [m]', compartment='')
Cpd687 = Metabolite('Cpd687', formula='', name='cholesterol [g]', compartment='')
Cpd688 = Metabolite('Cpd688', formula='', name='chitin [c]', compartment='')
Cpd689 = Metabolite('Cpd689', formula='', name='chitin [e]', compartment='')
Cpd690 = Metabolite('Cpd690', formula='', name='N-acetyl-D-glucosamine [e]', compartment='')
Cpd691 = Metabolite('Cpd691', formula='', name='L-citrulline [m]', compartment='')
Cpd692 = Metabolite('Cpd692', formula='', name='Citrate [e]', compartment='')
Cpd693 = Metabolite('Cpd693', formula='', name='Phosphoenolpyruvate [m]', compartment='')
Cpd694 = Metabolite('Cpd694', formula='', name='creatine [m]', compartment='')
Cpd695 = Metabolite('Cpd695', formula='', name='N-phosphocreatinate(2-) [m]', compartment='')
Cpd696 = Metabolite('Cpd696', formula='', name='creatine [c]', compartment='')
Cpd697 = Metabolite('Cpd697', formula='', name='N-phosphocreatinate(2-) [c]', compartment='')
Cpd698 = Metabolite('Cpd698', formula='', name='Chloride [e]', compartment='')
Cpd699 = Metabolite('Cpd699', formula='', name='Formate [c]', compartment='')
Cpd700 = Metabolite('Cpd700', formula='', name='Chloride [c]', compartment='')
Cpd701 = Metabolite('Cpd701', formula='', name='Formate [e]', compartment='')
Cpd702 = Metabolite('Cpd702', formula='', name='iodide [c]', compartment='')
Cpd703 = Metabolite('Cpd703', formula='', name='iodide [e]', compartment='')
Cpd704 = Metabolite('Cpd704', formula='', name='oxalate(2-) [c]', compartment='')
Cpd705 = Metabolite('Cpd705', formula='', name='oxalate(2-) [e]', compartment='')
Cpd706 = Metabolite('Cpd706', formula='', name='clupanodonyl CoA [c]', compartment='')
Cpd707 = Metabolite('Cpd707', formula='', name='clupanodonyl carnitine [c]', compartment='')
Cpd708 = Metabolite('Cpd708', formula='', name='clupanodonyl carnitine [m]', compartment='')
Cpd709 = Metabolite('Cpd709', formula='', name='clupanodonyl CoA [m]', compartment='')
Cpd710 = Metabolite('Cpd710', formula='', name='phosphatidylglycerol(1-) [c]', compartment='')
Cpd711 = Metabolite('Cpd711', formula='', name='cardiolipin [c]', compartment='')
Cpd712 = Metabolite('Cpd712', formula='', name='CMP [g]', compartment='')
Cpd713 = Metabolite('Cpd713', formula='', name='CMP-N-acetyl-beta-neuraminate(2-) [c]', compartment='')
Cpd714 = Metabolite('Cpd714', formula='', name='CMP-N-acetyl-beta-neuraminate(2-) [g]', compartment='')
Cpd715 = Metabolite('Cpd715', formula='', name='N-acetylneuraminate [n]', compartment='')
Cpd716 = Metabolite('Cpd716', formula='', name='CTP [n]', compartment='')
Cpd717 = Metabolite('Cpd717', formula='', name='CMP-N-acetyl-beta-neuraminate(2-) [n]', compartment='')
Cpd718 = Metabolite('Cpd718', formula='', name='Diphosphate [n]', compartment='')
Cpd719 = Metabolite('Cpd719', formula='', name='cocaine(1+) [r]', compartment='')
Cpd720 = Metabolite('Cpd720', formula='', name='benzoate [r]', compartment='')
Cpd721 = Metabolite('Cpd721', formula='', name='ecgoninium methyl ester(1+) [r]', compartment='')
Cpd722 = Metabolite('Cpd722', formula='', name='2-decaprenyl-5-hydroxy-6-methoxy-3-methyl-1,4-benzoquinone [m]', compartment='')
Cpd723 = Metabolite('Cpd723', formula='', name='ubiquinone-10 [m]', compartment='')
Cpd724 = Metabolite('Cpd724', formula='', name='2-Decaprenyl-6-methoxyphenol [m]', compartment='')
Cpd725 = Metabolite('Cpd725', formula='', name='2-decaprenyl-6-methoxy-1,4-benzoquinone [m]', compartment='')
Cpd726 = Metabolite('Cpd726', formula='', name='2-decaprenyl-6-methoxy-3-methyl-1,4-benzoquinone [m]', compartment='')
Cpd727 = Metabolite('Cpd727', formula='', name='Tn antigen [g]', compartment='')
Cpd728 = Metabolite('Cpd728', formula='', name='Core 3 [g]', compartment='')
Cpd729 = Metabolite('Cpd729', formula='', name='Core 4 [g]', compartment='')
Cpd730 = Metabolite('Cpd730', formula='', name='Core 6 [g]', compartment='')
Cpd731 = Metabolite('Cpd731', formula='', name='N-Trimethyl-2-aminoethylphosphonate [c]', compartment='')
Cpd732 = Metabolite('Cpd732', formula='', name='CMP-N-trimethyl-2-aminoethylphosphonate [c]', compartment='')
Cpd733 = Metabolite('Cpd733', formula='', name='Coproporphyrinogen III [c]', compartment='')
Cpd734 = Metabolite('Cpd734', formula='', name='Protoporphyrinogen IX [c]', compartment='')
Cpd735 = Metabolite('Cpd735', formula='', name='creatine [e]', compartment='')
Cpd736 = Metabolite('Cpd736', formula='', name='O-acetylcarnitine [x]', compartment='')
Cpd737 = Metabolite('Cpd737', formula='', name='L-Carnitine [x]', compartment='')
Cpd738 = Metabolite('Cpd738', formula='', name='L-Carnitine [e]', compartment='')
Cpd739 = Metabolite('Cpd739', formula='', name='propionyl-carnitine [m]', compartment='')
Cpd740 = Metabolite('Cpd740', formula='', name='4,8 dimethylnonanoyl-CoA [x]', compartment='')
Cpd741 = Metabolite('Cpd741', formula='', name='4,8-dimethylnonanoyl carnitine [x]', compartment='')
Cpd742 = Metabolite('Cpd742', formula='', name='Propanoyl-CoA [x]', compartment='')
Cpd743 = Metabolite('Cpd743', formula='', name='propionyl-carnitine [x]', compartment='')
Cpd744 = Metabolite('Cpd744', formula='', name='L-Carnitine [r]', compartment='')
Cpd745 = Metabolite('Cpd745', formula='', name='Coenzyme A [r]', compartment='')
Cpd746 = Metabolite('Cpd746', formula='', name='O-acetylcarnitine [r]', compartment='')
Cpd747 = Metabolite('Cpd747', formula='', name='cytosine [e]', compartment='')
Cpd748 = Metabolite('Cpd748', formula='', name='cytosine [c]', compartment='')
Cpd749 = Metabolite('Cpd749', formula='', name='UTP [c]', compartment='')
Cpd750 = Metabolite('Cpd750', formula='', name='Ferrocytochrome C [m]', compartment='')
Cpd751 = Metabolite('Cpd751', formula='', name='Ferricytochrome c [m]', compartment='')
Cpd752 = Metabolite('Cpd752', formula='', name='Ubiquinol-10 [m]', compartment='')
Cpd753 = Metabolite('Cpd753', formula='', name='3-mercaptopyruvate [c]', compartment='')
Cpd754 = Metabolite('Cpd754', formula='', name='L-cysteine [m]', compartment='')
Cpd755 = Metabolite('Cpd755', formula='', name='3-mercaptopyruvate [m]', compartment='')
Cpd756 = Metabolite('Cpd756', formula='', name='L-cystathionine [c]', compartment='')
Cpd757 = Metabolite('Cpd757', formula='', name='L-cystine [e]', compartment='')
Cpd758 = Metabolite('Cpd758', formula='', name='L-cystine [c]', compartment='')
Cpd759 = Metabolite('Cpd759', formula='', name='L-glutamate(1-) [e]', compartment='')
Cpd760 = Metabolite('Cpd760', formula='', name='Cytidine [c]', compartment='')
Cpd761 = Metabolite('Cpd761', formula='', name='Uridine [c]', compartment='')
Cpd762 = Metabolite('Cpd762', formula='', name='Cytidine [n]', compartment='')
Cpd763 = Metabolite('Cpd763', formula='', name='proton [n]', compartment='')
Cpd764 = Metabolite('Cpd764', formula='', name='Ammonium [n]', compartment='')
Cpd765 = Metabolite('Cpd765', formula='', name='Uridine [n]', compartment='')
Cpd766 = Metabolite('Cpd766', formula='', name='Cytidine [e]', compartment='')
Cpd767 = Metabolite('Cpd767', formula='', name='Cytidine [l]', compartment='')
Cpd768 = Metabolite('Cpd768', formula='', name='Cytidine [m]', compartment='')
Cpd769 = Metabolite('Cpd769', formula='', name='CDP [c]', compartment='')
Cpd770 = Metabolite('Cpd770', formula='', name='dGTP [c]', compartment='')
Cpd771 = Metabolite('Cpd771', formula='', name='dGDP [c]', compartment='')
Cpd772 = Metabolite('Cpd772', formula='', name='CMP [n]', compartment='')
Cpd773 = Metabolite('Cpd773', formula='', name='dGTP [n]', compartment='')
Cpd774 = Metabolite('Cpd774', formula='', name='CDP [n]', compartment='')
Cpd775 = Metabolite('Cpd775', formula='', name='dGDP [n]', compartment='')
Cpd776 = Metabolite('Cpd776', formula='', name='dCMP [c]', compartment='')
Cpd777 = Metabolite('Cpd777', formula='', name='dCDP [c]', compartment='')
Cpd778 = Metabolite('Cpd778', formula='', name='dCMP [n]', compartment='')
Cpd779 = Metabolite('Cpd779', formula='', name='dCDP [n]', compartment='')
Cpd780 = Metabolite('Cpd780', formula='', name='dCTP [c]', compartment='')
Cpd781 = Metabolite('Cpd781', formula='', name='dCTP [n]', compartment='')
Cpd782 = Metabolite('Cpd782', formula='', name='dATP [n]', compartment='')
Cpd783 = Metabolite('Cpd783', formula='', name='dADP [n]', compartment='')
Cpd784 = Metabolite('Cpd784', formula='', name='UTP [n]', compartment='')
Cpd785 = Metabolite('Cpd785', formula='', name='UDP [n]', compartment='')
Cpd786 = Metabolite('Cpd786', formula='', name='CMP [m]', compartment='')
Cpd787 = Metabolite('Cpd787', formula='', name='CDP [m]', compartment='')
Cpd788 = Metabolite('Cpd788', formula='', name='ATP [n]', compartment='')
Cpd789 = Metabolite('Cpd789', formula='', name='ADP [n]', compartment='')
Cpd790 = Metabolite('Cpd790', formula='', name='GTP [n]', compartment='')
Cpd791 = Metabolite('Cpd791', formula='', name='GDP [n]', compartment='')
Cpd792 = Metabolite('Cpd792', formula='', name='D-3-Amino-isobutanoate [m]', compartment='')
Cpd793 = Metabolite('Cpd793', formula='', name='2-deoxyadenosine [c]', compartment='')
Cpd794 = Metabolite('Cpd794', formula='', name='Deoxyinosine [c]', compartment='')
Cpd795 = Metabolite('Cpd795', formula='', name='2-deoxyadenosine [e]', compartment='')
Cpd796 = Metabolite('Cpd796', formula='', name='Deoxyinosine [e]', compartment='')
Cpd797 = Metabolite('Cpd797', formula='', name='diglyceride [n]', compartment='')
Cpd798 = Metabolite('Cpd798', formula='', name='phosphatidate(2-) [n]', compartment='')
Cpd799 = Metabolite('Cpd799', formula='', name='D-alanine [x]', compartment='')
Cpd800 = Metabolite('Cpd800', formula='', name='Ammonium [x]', compartment='')
Cpd801 = Metabolite('Cpd801', formula='', name='D-alanine [l]', compartment='')
Cpd802 = Metabolite('Cpd802', formula='', name='D-argininium(1+) [x]', compartment='')
Cpd803 = Metabolite('Cpd803', formula='', name='5-guanidino-2-oxopentanoic acid [x]', compartment='')
Cpd804 = Metabolite('Cpd804', formula='', name='dehydroascorbide(1-) [c]', compartment='')
Cpd805 = Metabolite('Cpd805', formula='', name='D-aspartate(1-) [x]', compartment='')
Cpd806 = Metabolite('Cpd806', formula='', name='Oxaloacetate [x]', compartment='')
Cpd807 = Metabolite('Cpd807', formula='', name='trans-Dodec-2-enoyl-CoA [m]', compartment='')
Cpd808 = Metabolite('Cpd808', formula='', name='cis-dodec-3-enoyl-CoA(4-) [m]', compartment='')
Cpd809 = Metabolite('Cpd809', formula='', name='Deoxycytidine [n]', compartment='')
Cpd810 = Metabolite('Cpd810', formula='', name='dUMP [c]', compartment='')
Cpd811 = Metabolite('Cpd811', formula='', name='docosa-4,7,10,13,16-pentaenoyl coenzyme A [c]', compartment='')
Cpd812 = Metabolite('Cpd812', formula='', name='docosa-4,7,10,13,16-pentaenoyl carnitine [c]', compartment='')
Cpd813 = Metabolite('Cpd813', formula='', name='docosa-4,7,10,13,16-pentaenoyl carnitine [m]', compartment='')
Cpd814 = Metabolite('Cpd814', formula='', name='docosa-4,7,10,13,16-pentaenoyl coenzyme A [m]', compartment='')
Cpd815 = Metabolite('Cpd815', formula='', name='L-dopachromate [c]', compartment='')
Cpd816 = Metabolite('Cpd816', formula='', name='5,6-dihydroxyindole-2-carboxylate [c]', compartment='')
Cpd817 = Metabolite('Cpd817', formula='', name='Deoxycytidine [c]', compartment='')
Cpd818 = Metabolite('Cpd818', formula='', name='Deoxyuridine [c]', compartment='')
Cpd819 = Metabolite('Cpd819', formula='', name='Deoxyuridine [n]', compartment='')
Cpd820 = Metabolite('Cpd820', formula='', name='Deoxycytidine [e]', compartment='')
Cpd821 = Metabolite('Cpd821', formula='', name='debrisoquine [e]', compartment='')
Cpd822 = Metabolite('Cpd822', formula='', name='debrisoquine [c]', compartment='')
Cpd823 = Metabolite('Cpd823', formula='', name='alpha-Linolenoyl-CoA [c]', compartment='')
Cpd824 = Metabolite('Cpd824', formula='', name='stearidonyl coenzyme A [c]', compartment='')
Cpd825 = Metabolite('Cpd825', formula='', name='vaccenyl coenzyme A [c]', compartment='')
Cpd826 = Metabolite('Cpd826', formula='', name='trans-Octadec-2-enoyl-CoA [c]', compartment='')
Cpd827 = Metabolite('Cpd827', formula='', name='linoelaidyl coenzyme A [c]', compartment='')
Cpd828 = Metabolite('Cpd828', formula='', name='linoleic coenzyme A [c]', compartment='')
Cpd829 = Metabolite('Cpd829', formula='', name='gamma-linolenoyl-CoA(4-) [c]', compartment='')
Cpd830 = Metabolite('Cpd830', formula='', name='dihomo-gamma-linolenyl coenzyme A [c]', compartment='')
Cpd831 = Metabolite('Cpd831', formula='', name='eicosatetranoyl coenzyme A [c]', compartment='')
Cpd832 = Metabolite('Cpd832', formula='', name='timnodonyl coenzyme A [c]', compartment='')
Cpd833 = Metabolite('Cpd833', formula='', name='tetracosapentaenoyl coenzyme A, n-3 [c]', compartment='')
Cpd834 = Metabolite('Cpd834', formula='', name='tetracosahexaenoyl coenzyme A [c]', compartment='')
Cpd835 = Metabolite('Cpd835', formula='', name='R total 3 Coenzyme A [c]', compartment='')
Cpd836 = Metabolite('Cpd836', formula='', name='triglyceride [c]', compartment='')
Cpd837 = Metabolite('Cpd837', formula='', name='dGMP [c]', compartment='')
Cpd838 = Metabolite('Cpd838', formula='', name='Deoxyguanosine [m]', compartment='')
Cpd839 = Metabolite('Cpd839', formula='', name='dGMP [m]', compartment='')
Cpd840 = Metabolite('Cpd840', formula='', name='Deoxyguanosine [e]', compartment='')
Cpd841 = Metabolite('Cpd841', formula='', name='Deoxyguanosine [c]', compartment='')
Cpd842 = Metabolite('Cpd842', formula='', name='dehydroascorbide(1-) [e]', compartment='')
Cpd843 = Metabolite('Cpd843', formula='', name='R total Coenzyme A [c]', compartment='')
Cpd844 = Metabolite('Cpd844', formula='', name='acyl-dihydroxyacetone phosphate [c]', compartment='')
Cpd845 = Metabolite('Cpd845', formula='', name='R total Coenzyme A [x]', compartment='')
Cpd846 = Metabolite('Cpd846', formula='', name='Dihydroxyacetone phosphate [x]', compartment='')
Cpd847 = Metabolite('Cpd847', formula='', name='Flavin adenine dinucleotide reduced [r]', compartment='')
Cpd848 = Metabolite('Cpd848', formula='', name='Flavin adenine dinucleotide oxidized [r]', compartment='')
Cpd849 = Metabolite('Cpd849', formula='', name='5alpha-cholest-8-en-3beta-ol [r]', compartment='')
Cpd850 = Metabolite('Cpd850', formula='', name='5alpha-cholesta-7,24-dien-3beta-ol [r]', compartment='')
Cpd851 = Metabolite('Cpd851', formula='', name='5alpha-cholest-7-en-3beta-ol [r]', compartment='')
Cpd852 = Metabolite('Cpd852', formula='', name='desmosterol [r]', compartment='')
Cpd853 = Metabolite('Cpd853', formula='', name='7-dehydrodesmosterol [r]', compartment='')
Cpd854 = Metabolite('Cpd854', formula='', name='cholesta-5,7-dien-3beta-ol [r]', compartment='')
Cpd855 = Metabolite('Cpd855', formula='', name='dihydroceramide [c]', compartment='')
Cpd856 = Metabolite('Cpd856', formula='', name='Flavin adenine dinucleotide oxidized [c]', compartment='')
Cpd857 = Metabolite('Cpd857', formula='', name='Flavin adenine dinucleotide reduced [c]', compartment='')
Cpd858 = Metabolite('Cpd858', formula='', name='3-Decaprenyl-4,5-dihdydroxybenzoate [m]', compartment='')
Cpd859 = Metabolite('Cpd859', formula='', name='3-Decaprenyl-4-hydroxy-5-methoxybenzoate [m]', compartment='')
Cpd860 = Metabolite('Cpd860', formula='', name='3beta-hydroxyandrost-5-en-17-one 3-sulfate(1-) [e]', compartment='')
Cpd861 = Metabolite('Cpd861', formula='', name='3beta-hydroxyandrost-5-en-17-one 3-sulfate(1-) [c]', compartment='')
Cpd862 = Metabolite('Cpd862', formula='', name='3beta-hydroxyandrost-5-en-17-one [c]', compartment='')
Cpd863 = Metabolite('Cpd863', formula='', name='7,8-Dihydrofolate [c]', compartment='')
Cpd864 = Metabolite('Cpd864', formula='', name='(S)-dihydroorotate [c]', compartment='')
Cpd865 = Metabolite('Cpd865', formula='', name='Orotate [c]', compartment='')
Cpd866 = Metabolite('Cpd866', formula='', name='5,6-dihydrouracil [c]', compartment='')
Cpd867 = Metabolite('Cpd867', formula='', name='N-carbamoyl-beta-alaninate [c]', compartment='')
Cpd868 = Metabolite('Cpd868', formula='', name='5,6-dihydrothymine [c]', compartment='')
Cpd869 = Metabolite('Cpd869', formula='', name='6,7-dihydrobiopterin [c]', compartment='')
Cpd870 = Metabolite('Cpd870', formula='', name='5,6,7,8-tetrahydrobiopterin [c]', compartment='')
Cpd871 = Metabolite('Cpd871', formula='', name='(R)-lactate [e]', compartment='')
Cpd872 = Metabolite('Cpd872', formula='', name='(R)-lactate [c]', compartment='')
Cpd873 = Metabolite('Cpd873', formula='', name='(R)-lactate [m]', compartment='')
Cpd874 = Metabolite('Cpd874', formula='', name='dihomo-gamma-linolenyl carnitine [c]', compartment='')
Cpd875 = Metabolite('Cpd875', formula='', name='dihomo-gamma-linolenyl carnitine [m]', compartment='')
Cpd876 = Metabolite('Cpd876', formula='', name='dihomo-gamma-linolenyl coenzyme A [m]', compartment='')
Cpd877 = Metabolite('Cpd877', formula='', name='dihomo-gamma-linolenic acid (n-6) [e]', compartment='')
Cpd878 = Metabolite('Cpd878', formula='', name='dihomo-gamma-linolenic acid (n-6) [c]', compartment='')
Cpd879 = Metabolite('Cpd879', formula='', name='Dimethylallyl diphosphate [c]', compartment='')
Cpd880 = Metabolite('Cpd880', formula='', name='Isopentenyl diphosphate [c]', compartment='')
Cpd881 = Metabolite('Cpd881', formula='', name='Geranyl diphosphate [c]', compartment='')
Cpd882 = Metabolite('Cpd882', formula='', name='Dimethylallyl diphosphate [x]', compartment='')
Cpd883 = Metabolite('Cpd883', formula='', name='Isopentenyl diphosphate [x]', compartment='')
Cpd884 = Metabolite('Cpd884', formula='', name='Geranyl diphosphate [x]', compartment='')
Cpd885 = Metabolite('Cpd885', formula='', name='Diphosphate [x]', compartment='')
Cpd886 = Metabolite('Cpd886', formula='', name='N,N-dimethylglycine [m]', compartment='')
Cpd887 = Metabolite('Cpd887', formula='', name='formaldehyde [m]', compartment='')
Cpd888 = Metabolite('Cpd888', formula='', name='Sarcosine [m]', compartment='')
Cpd889 = Metabolite('Cpd889', formula='', name='S-Adenosyl-L-methionine [n]', compartment='')
Cpd890 = Metabolite('Cpd890', formula='', name='DNA [n]', compartment='')
Cpd891 = Metabolite('Cpd891', formula='', name='S-Adenosyl-L-homocysteine [n]', compartment='')
Cpd892 = Metabolite('Cpd892', formula='', name='DNA 5-methylcytosine [n]', compartment='')
Cpd893 = Metabolite('Cpd893', formula='', name='Se-Adenosylselenomethionine [n]', compartment='')
Cpd894 = Metabolite('Cpd894', formula='', name='Se-Adenosylselenohomocysteine [n]', compartment='')
Cpd895 = Metabolite('Cpd895', formula='', name='dCDP [m]', compartment='')
Cpd896 = Metabolite('Cpd896', formula='', name='dADP [m]', compartment='')
Cpd897 = Metabolite('Cpd897', formula='', name='dGDP [m]', compartment='')
Cpd898 = Metabolite('Cpd898', formula='', name='dTDP [m]', compartment='')
Cpd899 = Metabolite('Cpd899', formula='', name='dUDP [c]', compartment='')
Cpd900 = Metabolite('Cpd900', formula='', name='dTDP [c]', compartment='')
Cpd901 = Metabolite('Cpd901', formula='', name='dUDP [m]', compartment='')
Cpd902 = Metabolite('Cpd902', formula='', name='dATP [m]', compartment='')
Cpd903 = Metabolite('Cpd903', formula='', name='dUTP [c]', compartment='')
Cpd904 = Metabolite('Cpd904', formula='', name='dUTP [m]', compartment='')
Cpd905 = Metabolite('Cpd905', formula='', name='dTTP [c]', compartment='')
Cpd906 = Metabolite('Cpd906', formula='', name='dTTP [m]', compartment='')
Cpd907 = Metabolite('Cpd907', formula='', name='dCTP [m]', compartment='')
Cpd908 = Metabolite('Cpd908', formula='', name='dGTP [m]', compartment='')
Cpd909 = Metabolite('Cpd909', formula='', name='protein-linked asparagine residue (N-glycosylation site) [r]', compartment='')
Cpd910 = Metabolite('Cpd910', formula='', name='(alpha-D-Glucosyl)3-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human liver homolog [r]', compartment='')
Cpd911 = Metabolite('Cpd911', formula='', name='Dolichol diphosphate, human liver homolog [r]', compartment='')
Cpd912 = Metabolite('Cpd912', formula='', name='(alpha-D-Glucosyl)3-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [r]', compartment='')
Cpd913 = Metabolite('Cpd913', formula='', name='(alpha-D-Glucosyl)3-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human uterine homolog [r]', compartment='')
Cpd914 = Metabolite('Cpd914', formula='', name='Dolichol diphosphate, human uterine homolog [r]', compartment='')
Cpd915 = Metabolite('Cpd915', formula='', name='hydrogenphosphate [r]', compartment='')
Cpd916 = Metabolite('Cpd916', formula='', name='Dolichyl beta-D-glucosyl phosphate, human liver homolog [r]', compartment='')
Cpd917 = Metabolite('Cpd917', formula='', name='alpha-D-Glucosyl-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human liver homolog [r]', compartment='')
Cpd918 = Metabolite('Cpd918', formula='', name='(alpha-D-Glucosyl)2-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human liver homolog [r]', compartment='')
Cpd919 = Metabolite('Cpd919', formula='', name='Dolichyl beta-D-glucosyl phosphate, human uterine homolog [r]', compartment='')
Cpd920 = Metabolite('Cpd920', formula='', name='alpha-D-Glucosyl-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human uterine homolog [r]', compartment='')
Cpd921 = Metabolite('Cpd921', formula='', name='(alpha-D-Glucosyl)2-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human uterine homolog [r]', compartment='')
Cpd922 = Metabolite('Cpd922', formula='', name='Dolichol, human liver homolog [r]', compartment='')
Cpd923 = Metabolite('Cpd923', formula='', name='Dolichol, human uterine homolog [r]', compartment='')
Cpd924 = Metabolite('Cpd924', formula='', name='(alpha-D-Mannosyl)5-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human liver homolog [r]', compartment='')
Cpd925 = Metabolite('Cpd925', formula='', name='(alpha-D-Mannosyl)6-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human liver homolog [r]', compartment='')
Cpd926 = Metabolite('Cpd926', formula='', name='(alpha-D-Mannosyl)5-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human uterine homolog [r]', compartment='')
Cpd927 = Metabolite('Cpd927', formula='', name='(alpha-D-Mannosyl)6-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human uterine homolog [r]', compartment='')
Cpd928 = Metabolite('Cpd928', formula='', name='(alpha-D-Mannosyl)7-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human liver homolog [r]', compartment='')
Cpd929 = Metabolite('Cpd929', formula='', name='(alpha-D-Mannosyl)7-beta-D-mannosyl-diacetylchitobiosyldiphosphodolichol, human uterine homolog [r]', compartment='')
Cpd930 = Metabolite('Cpd930', formula='', name='dolichyl phosphate(2-) [c]', compartment='')
Cpd931 = Metabolite('Cpd931', formula='', name='Dolichyl phosphate D-mannose, human liver homolog [c]', compartment='')
Cpd932 = Metabolite('Cpd932', formula='', name='Dolichyl phosphate D-mannose, human uterine homolog [c]', compartment='')
Cpd933 = Metabolite('Cpd933', formula='', name='2-Carboxy-2,3-dihydro-5,6-dihydroxyindole [c]', compartment='')
Cpd934 = Metabolite('Cpd934', formula='', name='dopamine 3-O-sulfate [c]', compartment='')
Cpd935 = Metabolite('Cpd935', formula='', name='dopaminium(1+) [e]', compartment='')
Cpd936 = Metabolite('Cpd936', formula='', name='D-ornithinium(1+) [x]', compartment='')
Cpd937 = Metabolite('Cpd937', formula='', name='5-amino-2-oxopentanoic acid [x]', compartment='')
Cpd938 = Metabolite('Cpd938', formula='', name='Dephospho-CoA [c]', compartment='')
Cpd939 = Metabolite('Cpd939', formula='', name='2,3-bisphosphonato-D-glycerate(5-) [c]', compartment='')
Cpd940 = Metabolite('Cpd940', formula='', name='ATP [x]', compartment='')
Cpd941 = Metabolite('Cpd941', formula='', name='(R)-5-Diphosphomevalonate [x]', compartment='')
Cpd942 = Metabolite('Cpd942', formula='', name='ADP [x]', compartment='')
Cpd943 = Metabolite('Cpd943', formula='', name='carbon dioxide [x]', compartment='')
Cpd944 = Metabolite('Cpd944', formula='', name='hydrogenphosphate [x]', compartment='')
Cpd945 = Metabolite('Cpd945', formula='', name='D-proline [x]', compartment='')
Cpd946 = Metabolite('Cpd946', formula='', name='1-pyrroline-2-carboxylate [x]', compartment='')
Cpd947 = Metabolite('Cpd947', formula='', name='2-deoxy-D-ribose [c]', compartment='')
Cpd948 = Metabolite('Cpd948', formula='', name='2-deoxy-D-ribose 5-phosphate(2-) [c]', compartment='')
Cpd949 = Metabolite('Cpd949', formula='', name='Glyceraldehyde 3-phosphate [c]', compartment='')
Cpd950 = Metabolite('Cpd950', formula='', name='dTMP [c]', compartment='')
Cpd951 = Metabolite('Cpd951', formula='', name='uracil [c]', compartment='')
Cpd952 = Metabolite('Cpd952', formula='', name='Thymine [c]', compartment='')
Cpd953 = Metabolite('Cpd953', formula='', name='Deoxyuridine [m]', compartment='')
Cpd954 = Metabolite('Cpd954', formula='', name='dUMP [m]', compartment='')
Cpd955 = Metabolite('Cpd955', formula='', name='2-Deoxy-D-ribose 1-phosphate [c]', compartment='')
Cpd956 = Metabolite('Cpd956', formula='', name='Deoxyuridine [e]', compartment='')
Cpd957 = Metabolite('Cpd957', formula='', name='dUTP [n]', compartment='')
Cpd958 = Metabolite('Cpd958', formula='', name='dUMP [n]', compartment='')
Cpd959 = Metabolite('Cpd959', formula='', name='(S)-3-Hydroxyisobutyryl-CoA [m]', compartment='')
Cpd960 = Metabolite('Cpd960', formula='', name='(S)-3-hydroxybutanoyl-CoA(4-) [m]', compartment='')
Cpd961 = Metabolite('Cpd961', formula='', name='(S)-3-hydroxybutanoyl-CoA(4-) [x]', compartment='')
Cpd962 = Metabolite('Cpd962', formula='', name='crotonoyl-CoA(4-) [x]', compartment='')
Cpd963 = Metabolite('Cpd963', formula='', name='(2S,3S)-3-hydroxy-2-methylbutanoyl-CoA(4-) [m]', compartment='')
Cpd964 = Metabolite('Cpd964', formula='', name='ecgonine [r]', compartment='')
Cpd965 = Metabolite('Cpd965', formula='', name='methanol [r]', compartment='')
Cpd966 = Metabolite('Cpd966', formula='', name='erythro-4-hydroxy-L-glutamate(1-) [c]', compartment='')
Cpd967 = Metabolite('Cpd967', formula='', name='4-hydroxy-2-oxoglutarate(2-) [c]', compartment='')
Cpd968 = Metabolite('Cpd968', formula='', name='4-hydroxy-2-oxoglutarate(2-) [m]', compartment='')
Cpd969 = Metabolite('Cpd969', formula='', name='Eicosatetranoyl carnitine [c]', compartment='')
Cpd970 = Metabolite('Cpd970', formula='', name='Eicosatetranoyl carnitine [m]', compartment='')
Cpd971 = Metabolite('Cpd971', formula='', name='eicosatetranoyl coenzyme A [m]', compartment='')
Cpd972 = Metabolite('Cpd972', formula='', name='Elaidic carnitine [c]', compartment='')
Cpd973 = Metabolite('Cpd973', formula='', name='Elaidic carnitine [m]', compartment='')
Cpd974 = Metabolite('Cpd974', formula='', name='trans-Octadec-2-enoyl-CoA [m]', compartment='')
Cpd975 = Metabolite('Cpd975', formula='', name='elaidate [e]', compartment='')
Cpd976 = Metabolite('Cpd976', formula='', name='elaidate [c]', compartment='')
Cpd977 = Metabolite('Cpd977', formula='', name='de-Fuc form of PA6 (w/o peptide linkage) [c]', compartment='')
Cpd978 = Metabolite('Cpd978', formula='', name='de-Fuc, reducing GlcNAc removed form of PA6 (w/o peptide linkage) [c]', compartment='')
Cpd979 = Metabolite('Cpd979', formula='', name='n2m2nmasn (w/o peptide linkage) [c]', compartment='')
Cpd980 = Metabolite('Cpd980', formula='', name='reducing GlcNAc removed form of n2m2nmasn (w/o peptide) [c]', compartment='')
Cpd981 = Metabolite('Cpd981', formula='', name='D-Glycerate 2-phosphate [c]', compartment='')
Cpd982 = Metabolite('Cpd982', formula='', name='(2-Aminoethyl)phosphonate [c]', compartment='')
Cpd983 = Metabolite('Cpd983', formula='', name='CMP-2-aminoethylphosphonate [c]', compartment='')
Cpd984 = Metabolite('Cpd984', formula='', name='17beta-estradiol 3-glucosiduronic acid [e]', compartment='')
Cpd985 = Metabolite('Cpd985', formula='', name='17beta-estradiol 3-glucosiduronic acid [c]', compartment='')
Cpd986 = Metabolite('Cpd986', formula='', name='16-Glucuronide-estriol [c]', compartment='')
Cpd987 = Metabolite('Cpd987', formula='', name='16-Glucuronide-estriol [e]', compartment='')
Cpd988 = Metabolite('Cpd988', formula='', name='estrone 3-glucosiduronic acid [c]', compartment='')
Cpd989 = Metabolite('Cpd989', formula='', name='estrone 3-glucosiduronic acid [e]', compartment='')
Cpd990 = Metabolite('Cpd990', formula='', name='estrone 3-sulfate(1-) [e]', compartment='')
Cpd991 = Metabolite('Cpd991', formula='', name='estrone 3-sulfate(1-) [c]', compartment='')
Cpd992 = Metabolite('Cpd992', formula='', name='estrone [c]', compartment='')
Cpd993 = Metabolite('Cpd993', formula='', name='Electron transfer flavoprotein oxidized [m]', compartment='')
Cpd994 = Metabolite('Cpd994', formula='', name='Electron transfer flavoprotein reduced [m]', compartment='')
Cpd995 = Metabolite('Cpd995', formula='', name='etha[c] [c]', compartment='')
Cpd996 = Metabolite('Cpd996', formula='', name='O-phosphonatoethanaminium(1-) [c]', compartment='')
Cpd997 = Metabolite('Cpd997', formula='', name='L-Fucose 1-phosphate [c]', compartment='')
Cpd998 = Metabolite('Cpd998', formula='', name='GDP-L-fucose(2-) [c]', compartment='')
Cpd999 = Metabolite('Cpd999', formula='', name='GDP-L-fucose(2-) [g]', compartment='')
Cpd1000 = Metabolite('Cpd1000', formula='', name='((N-acetyl-D-glucosaminyl)2-(alpha-D-mannosyl)2-beta-D-mannosyl-diacetylchitobiosyl)-L-asparagine (protein) [g]', compartment='')
Cpd1001 = Metabolite('Cpd1001', formula='', name='N4-{N-Acetyl-beta-D-glucosaminyl-(1,2)-alpha-D-mannosyl-(1,3)-[N-acetyl-beta-D-glucosaminyl-(1,2)-alpha-D-mannosyl-(1,6)]-beta-D-mannosyl-(1,4)-N-acetyl-beta-D-glucosaminyl-(1,4)-[alpha-L-fucosyl-(1,6)]-N-acetyl-beta-D-glucosaminyl}asparagine [g]', compartment='')
Cpd1002 = Metabolite('Cpd1002', formula='', name='GDP [g]', compartment='')
Cpd1003 = Metabolite('Cpd1003', formula='', name='Dodecanoyl-ACP (n-C12:0ACP) [c]', compartment='')
Cpd1004 = Metabolite('Cpd1004', formula='', name='laurate [c]', compartment='')
Cpd1005 = Metabolite('Cpd1005', formula='', name='Myristoyl-ACP (n-C14:0ACP) [c]', compartment='')
Cpd1006 = Metabolite('Cpd1006', formula='', name='tetradecanoate (n-C14:0) [c]', compartment='')
Cpd1007 = Metabolite('Cpd1007', formula='', name='cis-tetradec-7-enoyl-[acyl-carrier protein] (n-C14:1) [c]', compartment='')
Cpd1008 = Metabolite('Cpd1008', formula='', name='tetradecenoate (n-C14:1) [c]', compartment='')
Cpd1009 = Metabolite('Cpd1009', formula='', name='Palmitoyl-ACP (n-C16:0ACP) [c]', compartment='')
Cpd1010 = Metabolite('Cpd1010', formula='', name='Hexadecanoate (n-C16:0) [c]', compartment='')
Cpd1011 = Metabolite('Cpd1011', formula='', name='cis-hexadec-9-enoyl-[acyl-carrier protein] (n-C16:1) [c]', compartment='')
Cpd1012 = Metabolite('Cpd1012', formula='', name='Hexadecenoate (n-C16:1) [c]', compartment='')
Cpd1013 = Metabolite('Cpd1013', formula='', name='Octadecanoyl-ACP (n-C18:0ACP) [c]', compartment='')
Cpd1014 = Metabolite('Cpd1014', formula='', name='octadecanoate (n-C18:0) [c]', compartment='')
Cpd1015 = Metabolite('Cpd1015', formula='', name='cis-octadec-11-enoyl-[acyl-carrier protein] (n-C18:1) [c]', compartment='')
Cpd1016 = Metabolite('Cpd1016', formula='', name='octadecenoate (n-C18:1) [c]', compartment='')
Cpd1017 = Metabolite('Cpd1017', formula='', name='Octadecynoyl-ACP (n-C18:2ACP) [c]', compartment='')
Cpd1018 = Metabolite('Cpd1018', formula='', name='octadecadienoate (n-C18:2) [c]', compartment='')
Cpd1019 = Metabolite('Cpd1019', formula='', name='Tetradecanoyl-CoA (n-C14:0CoA) [c]', compartment='')
Cpd1020 = Metabolite('Cpd1020', formula='', name='pentadecanoate [c]', compartment='')
Cpd1021 = Metabolite('Cpd1021', formula='', name='pentadecanoyl Coenzyme A [c]', compartment='')
Cpd1022 = Metabolite('Cpd1022', formula='', name='margarate [c]', compartment='')
Cpd1023 = Metabolite('Cpd1023', formula='', name='heptadecanoyl coa [c]', compartment='')
Cpd1024 = Metabolite('Cpd1024', formula='', name='trans-vaccenate [c]', compartment='')
Cpd1025 = Metabolite('Cpd1025', formula='', name='linoleate [c]', compartment='')
Cpd1026 = Metabolite('Cpd1026', formula='', name='linoelaidic acid (all trans C18:2) [c]', compartment='')
Cpd1027 = Metabolite('Cpd1027', formula='', name='gamma-linolenate [c]', compartment='')
Cpd1028 = Metabolite('Cpd1028', formula='', name='alpha-linolenate [c]', compartment='')
Cpd1029 = Metabolite('Cpd1029', formula='', name='stearidonic acid C18:4, n-3 [c]', compartment='')
Cpd1030 = Metabolite('Cpd1030', formula='', name='pristanoyl coa [c]', compartment='')
Cpd1031 = Metabolite('Cpd1031', formula='', name='arachidate [c]', compartment='')
Cpd1032 = Metabolite('Cpd1032', formula='', name='eicosatetranoic acid [c]', compartment='')
Cpd1033 = Metabolite('Cpd1033', formula='', name='timnodonic acid C20:5, n-3 [c]', compartment='')
Cpd1034 = Metabolite('Cpd1034', formula='', name='phytanate [c]', compartment='')
Cpd1035 = Metabolite('Cpd1035', formula='', name='phytanoyl-CoA(4-) [c]', compartment='')
Cpd1036 = Metabolite('Cpd1036', formula='', name='adrenic acid [c]', compartment='')
Cpd1037 = Metabolite('Cpd1037', formula='', name='docosa-4,7,10,13,16-pentaenoic acid [c]', compartment='')
Cpd1038 = Metabolite('Cpd1038', formula='', name='clupanodonic acid [c]', compartment='')
Cpd1039 = Metabolite('Cpd1039', formula='', name='cervonic acid, C22:6 n-3 [c]', compartment='')
Cpd1040 = Metabolite('Cpd1040', formula='', name='lignocerate [c]', compartment='')
Cpd1041 = Metabolite('Cpd1041', formula='', name='lignocericyl coenzyme A [c]', compartment='')
Cpd1042 = Metabolite('Cpd1042', formula='', name='nervonate [c]', compartment='')
Cpd1043 = Metabolite('Cpd1043', formula='', name='nervonyl coenzyme A [c]', compartment='')
Cpd1044 = Metabolite('Cpd1044', formula='', name='tetracosatetraenoic acid n-6 [c]', compartment='')
Cpd1045 = Metabolite('Cpd1045', formula='', name='tetracosatetraenoyl coenzyme A [c]', compartment='')
Cpd1046 = Metabolite('Cpd1046', formula='', name='tetracosapentaenoic acid, n-6 [c]', compartment='')
Cpd1047 = Metabolite('Cpd1047', formula='', name='tetracosapentaenoyl coenzyme A, n-6 [c]', compartment='')
Cpd1048 = Metabolite('Cpd1048', formula='', name='tetracosapentaenoic acid, n-3 [c]', compartment='')
Cpd1049 = Metabolite('Cpd1049', formula='', name='tetracosahexaenoic acid, n-3 [c]', compartment='')
Cpd1050 = Metabolite('Cpd1050', formula='', name='cerotate [c]', compartment='')
Cpd1051 = Metabolite('Cpd1051', formula='', name='Hexacosanoyl-CoA (n-C26:0CoA) [c]', compartment='')
Cpd1052 = Metabolite('Cpd1052', formula='', name='octanoate (n-C8:0) [c]', compartment='')
Cpd1053 = Metabolite('Cpd1053', formula='', name='Octanoyl-CoA (n-C8:0CoA) [c]', compartment='')
Cpd1054 = Metabolite('Cpd1054', formula='', name='omega hydroxy dodecanoate (n-C12:0) [c]', compartment='')
Cpd1055 = Metabolite('Cpd1055', formula='', name='omega hydroxy tetradecanoate (n-C14:0) [c]', compartment='')
Cpd1056 = Metabolite('Cpd1056', formula='', name='omega hydroxy hexadecanoate (n-C16:0) [c]', compartment='')
Cpd1057 = Metabolite('Cpd1057', formula='', name='Reduced glutathione [c]', compartment='')
Cpd1058 = Metabolite('Cpd1058', formula='', name='S-formylglutathionate(1-) [c]', compartment='')
Cpd1059 = Metabolite('Cpd1059', formula='', name='4,8 dimethylnonanoyl-CoA [m]', compartment='')
Cpd1060 = Metabolite('Cpd1060', formula='', name='2,6 dimethylheptanoyl-CoA [m]', compartment='')
Cpd1061 = Metabolite('Cpd1061', formula='', name='Tetradecanoyl-CoA (n-C14:0CoA) [m]', compartment='')
Cpd1062 = Metabolite('Cpd1062', formula='', name='pentadecanoyl Coenzyme A [m]', compartment='')
Cpd1063 = Metabolite('Cpd1063', formula='', name='Octanoyl-CoA (n-C8:0CoA) [m]', compartment='')
Cpd1064 = Metabolite('Cpd1064', formula='', name='Nicotinamide adenine dinucleotide [x]', compartment='')
Cpd1065 = Metabolite('Cpd1065', formula='', name='Nicotinamide adenine dinucleotide - reduced [x]', compartment='')
Cpd1066 = Metabolite('Cpd1066', formula='', name='heptadecanoyl coa [m]', compartment='')
Cpd1067 = Metabolite('Cpd1067', formula='', name='Stearoyl-CoA (n-C18:0CoA) [x]', compartment='')
Cpd1068 = Metabolite('Cpd1068', formula='', name='vaccenyl coenzyme A [m]', compartment='')
Cpd1069 = Metabolite('Cpd1069', formula='', name='linoleic coenzyme A [m]', compartment='')
Cpd1070 = Metabolite('Cpd1070', formula='', name='linoelaidyl coenzyme A [m]', compartment='')
Cpd1071 = Metabolite('Cpd1071', formula='', name='alpha-Linolenoyl-CoA [m]', compartment='')
Cpd1072 = Metabolite('Cpd1072', formula='', name='gamma-linolenoyl-CoA(4-) [m]', compartment='')
Cpd1073 = Metabolite('Cpd1073', formula='', name='gamma-linolenoyl-CoA(4-) [x]', compartment='')
Cpd1074 = Metabolite('Cpd1074', formula='', name='stearidonyl coenzyme A [m]', compartment='')
Cpd1075 = Metabolite('Cpd1075', formula='', name='stearidonyl coenzyme A [x]', compartment='')
Cpd1076 = Metabolite('Cpd1076', formula='', name='arachidyl coenzyme A [x]', compartment='')
Cpd1077 = Metabolite('Cpd1077', formula='', name='timnodonyl coenzyme A [m]', compartment='')
Cpd1078 = Metabolite('Cpd1078', formula='', name='timnodonyl coenzyme A [x]', compartment='')
Cpd1079 = Metabolite('Cpd1079', formula='', name='adrenyl coenzyme A [x]', compartment='')
Cpd1080 = Metabolite('Cpd1080', formula='', name='arachidonoyl-CoA(4-) [x]', compartment='')
Cpd1081 = Metabolite('Cpd1081', formula='', name='docosa-4,7,10,13,16-pentaenoyl coenzyme A [x]', compartment='')
Cpd1082 = Metabolite('Cpd1082', formula='', name='cervonyl coenzyme A [x]', compartment='')
Cpd1083 = Metabolite('Cpd1083', formula='', name='lignocericyl coenzyme A [x]', compartment='')
Cpd1084 = Metabolite('Cpd1084', formula='', name='nervonyl coenzyme A [x]', compartment='')
Cpd1085 = Metabolite('Cpd1085', formula='', name='Octadecenoyl-CoA (n-C18:1CoA) [x]', compartment='')
Cpd1086 = Metabolite('Cpd1086', formula='', name='tetracosatetraenoyl coenzyme A [x]', compartment='')
Cpd1087 = Metabolite('Cpd1087', formula='', name='tetracosapentaenoyl coenzyme A, n-3 [x]', compartment='')
Cpd1088 = Metabolite('Cpd1088', formula='', name='clupanodonyl CoA [x]', compartment='')
Cpd1089 = Metabolite('Cpd1089', formula='', name='tetracosapentaenoyl coenzyme A, n-6 [x]', compartment='')
Cpd1090 = Metabolite('Cpd1090', formula='', name='tetracosahexaenoyl coenzyme A [x]', compartment='')
Cpd1091 = Metabolite('Cpd1091', formula='', name='Hexacosanoyl-CoA (n-C26:0CoA) [x]', compartment='')
Cpd1092 = Metabolite('Cpd1092', formula='', name='Decanoyl-CoA (n-C10:0CoA) [c]', compartment='')
Cpd1093 = Metabolite('Cpd1093', formula='', name='Dodecanoyl-CoA (n-C12:0CoA) [c]', compartment='')
Cpd1094 = Metabolite('Cpd1094', formula='', name='Hexadecanoate (n-C16:0) [e]', compartment='')
Cpd1095 = Metabolite('Cpd1095', formula='', name='octadecenoate (n-C18:1) [e]', compartment='')
Cpd1096 = Metabolite('Cpd1096', formula='', name='octadecanoate (n-C18:0) [e]', compartment='')
Cpd1097 = Metabolite('Cpd1097', formula='', name='arachidate [e]', compartment='')
Cpd1098 = Metabolite('Cpd1098', formula='', name='adrenic acid [e]', compartment='')
Cpd1099 = Metabolite('Cpd1099', formula='', name='cervonic acid, C22:6 n-3 [e]', compartment='')
Cpd1100 = Metabolite('Cpd1100', formula='', name='lignocerate [e]', compartment='')
Cpd1101 = Metabolite('Cpd1101', formula='', name='nervonate [e]', compartment='')
Cpd1102 = Metabolite('Cpd1102', formula='', name='D-Fructose 1,6-bisphosphate [c]', compartment='')
Cpd1103 = Metabolite('Cpd1103', formula='', name='D-Fructose 1-phosphate [c]', compartment='')
Cpd1104 = Metabolite('Cpd1104', formula='', name='D-Glyceraldehyde [c]', compartment='')
Cpd1105 = Metabolite('Cpd1105', formula='', name='D-Xylulose 1-phosphate [c]', compartment='')
Cpd1106 = Metabolite('Cpd1106', formula='', name='glycolaldehyde [c]', compartment='')
Cpd1107 = Metabolite('Cpd1107', formula='', name='D-Tagatose 1-phosphate [c]', compartment='')
Cpd1108 = Metabolite('Cpd1108', formula='', name='D-Fructose 6-phosphate [c]', compartment='')
Cpd1109 = Metabolite('Cpd1109', formula='', name='beta-D-fructofuranose 2,6-bisphosphate(4-) [c]', compartment='')
Cpd1110 = Metabolite('Cpd1110', formula='', name='Fe2+ [m]', compartment='')
Cpd1111 = Metabolite('Cpd1111', formula='', name='Protoporphyrin [m]', compartment='')
Cpd1112 = Metabolite('Cpd1112', formula='', name='Protoheme [m]', compartment='')
Cpd1113 = Metabolite('Cpd1113', formula='', name='Fe3+ [e]', compartment='')
Cpd1114 = Metabolite('Cpd1114', formula='', name='Fe2+ [e]', compartment='')
Cpd1115 = Metabolite('Cpd1115', formula='', name='L-fucose [c]', compartment='')
Cpd1116 = Metabolite('Cpd1116', formula='', name='N-formyl-L-kynurenine [c]', compartment='')
Cpd1117 = Metabolite('Cpd1117', formula='', name='L-kynurenine [c]', compartment='')
Cpd1118 = Metabolite('Cpd1118', formula='', name='Folate [c]', compartment='')
Cpd1119 = Metabolite('Cpd1119', formula='', name='Folate [e]', compartment='')
Cpd1120 = Metabolite('Cpd1120', formula='', name='pentaglutamyl folate (THF) [c]', compartment='')
Cpd1121 = Metabolite('Cpd1121', formula='', name='hexaglutamyl folate  (THF) [c]', compartment='')
Cpd1122 = Metabolite('Cpd1122', formula='', name='pentaglutamyl folate (THF) [m]', compartment='')
Cpd1123 = Metabolite('Cpd1123', formula='', name='hexaglutamyl folate  (THF) [m]', compartment='')
Cpd1124 = Metabolite('Cpd1124', formula='', name='heptaglutamyl folate (THF) [c]', compartment='')
Cpd1125 = Metabolite('Cpd1125', formula='', name='heptaglutamyl folate (THF) [m]', compartment='')
Cpd1126 = Metabolite('Cpd1126', formula='', name='pentaglutamyl folate (DHF) [c]', compartment='')
Cpd1127 = Metabolite('Cpd1127', formula='', name='7,8-Dihydrofolate [m]', compartment='')
Cpd1128 = Metabolite('Cpd1128', formula='', name='pentaglutamyl folate (DHF) [m]', compartment='')
Cpd1129 = Metabolite('Cpd1129', formula='', name='hexaglutamyl folate (DHF) [c]', compartment='')
Cpd1130 = Metabolite('Cpd1130', formula='', name='hexaglutamyl folate (DHF) [m]', compartment='')
Cpd1131 = Metabolite('Cpd1131', formula='', name='heptaglutamyl folate (DHF) [c]', compartment='')
Cpd1132 = Metabolite('Cpd1132', formula='', name='heptaglutamyl folate (DHF) [m]', compartment='')
Cpd1133 = Metabolite('Cpd1133', formula='', name='10-formyltetrahydrofolate-[Glu](5) [c]', compartment='')
Cpd1134 = Metabolite('Cpd1134', formula='', name='10-Formyltetrahydrofolate [m]', compartment='')
Cpd1135 = Metabolite('Cpd1135', formula='', name='10-formyltetrahydrofolate-[Glu](5) [m]', compartment='')
Cpd1136 = Metabolite('Cpd1136', formula='', name='10-formyltetrahydrofolate-[Glu](6) [c]', compartment='')
Cpd1137 = Metabolite('Cpd1137', formula='', name='10-formyltetrahydrofolate-[Glu](6) [m]', compartment='')
Cpd1138 = Metabolite('Cpd1138', formula='', name='10-formyltetrahydrofolate-[Glu](7) [c]', compartment='')
Cpd1139 = Metabolite('Cpd1139', formula='', name='10-formyltetrahydrofolate-[Glu](7) [m]', compartment='')
Cpd1140 = Metabolite('Cpd1140', formula='', name='5,6,7,8-Tetrahydrofolate [m]', compartment='')
Cpd1141 = Metabolite('Cpd1141', formula='', name='Farnesyl diphosphate [c]', compartment='')
Cpd1142 = Metabolite('Cpd1142', formula='', name='2-trans,6-trans,10-trans-geranylgeranyl diphosphate(3-) [c]', compartment='')
Cpd1143 = Metabolite('Cpd1143', formula='', name='D-Fructose [e]', compartment='')
Cpd1144 = Metabolite('Cpd1144', formula='', name='D-Fructose [c]', compartment='')
Cpd1145 = Metabolite('Cpd1145', formula='', name='5-Formiminotetrahydrofolate [c]', compartment='')
Cpd1146 = Metabolite('Cpd1146', formula='', name='5,10-Methenyltetrahydrofolate [c]', compartment='')
Cpd1147 = Metabolite('Cpd1147', formula='', name='Formate [m]', compartment='')
Cpd1148 = Metabolite('Cpd1148', formula='', name='keratan sulfate I [e]', compartment='')
Cpd1149 = Metabolite('Cpd1149', formula='', name='L-fucose [e]', compartment='')
Cpd1150 = Metabolite('Cpd1150', formula='', name='keratan sulfate I, degradation product 1 [e]', compartment='')
Cpd1151 = Metabolite('Cpd1151', formula='', name='keratan sulfate I [l]', compartment='')
Cpd1152 = Metabolite('Cpd1152', formula='', name='L-fucose [l]', compartment='')
Cpd1153 = Metabolite('Cpd1153', formula='', name='keratan sulfate I, degradation product 1 [l]', compartment='')
Cpd1154 = Metabolite('Cpd1154', formula='', name='PA6 [e]', compartment='')
Cpd1155 = Metabolite('Cpd1155', formula='', name='de-Fuc form of PA6 [e]', compartment='')
Cpd1156 = Metabolite('Cpd1156', formula='', name='PA6 [l]', compartment='')
Cpd1157 = Metabolite('Cpd1157', formula='', name='de-Fuc form of PA6 [l]', compartment='')
Cpd1158 = Metabolite('Cpd1158', formula='', name='4-fumarylacetoacetate(2-) [c]', compartment='')
Cpd1159 = Metabolite('Cpd1159', formula='', name='Fumarate [m]', compartment='')
Cpd1160 = Metabolite('Cpd1160', formula='', name='Sulfite [m]', compartment='')
Cpd1161 = Metabolite('Cpd1161', formula='', name='Sulfite [c]', compartment='')
Cpd1162 = Metabolite('Cpd1162', formula='', name='sulfate [m]', compartment='')
Cpd1163 = Metabolite('Cpd1163', formula='', name='sulfate [c]', compartment='')
Cpd1164 = Metabolite('Cpd1164', formula='', name='thiosulfate(2-) [m]', compartment='')
Cpd1165 = Metabolite('Cpd1165', formula='', name='thiosulfate(2-) [c]', compartment='')
Cpd1166 = Metabolite('Cpd1166', formula='', name='fucosyl galactosylgloboside [g]', compartment='')
Cpd1167 = Metabolite('Cpd1167', formula='', name='Lc4Cer [g]', compartment='')
Cpd1168 = Metabolite('Cpd1168', formula='', name='(Gal)4 (Glc)1 (GlcNAc)3 (LFuc)1 (Cer)1 [g]', compartment='')
Cpd1169 = Metabolite('Cpd1169', formula='', name='(Gal)4 (Glc)1 (GlcNAc)3 (LFuc)3 (Cer)1 [g]', compartment='')
Cpd1170 = Metabolite('Cpd1170', formula='', name='Lacto-N-fucopentaosyl III ceramide [g]', compartment='')
Cpd1171 = Metabolite('Cpd1171', formula='', name='Sialyl-3-paragloboside [g]', compartment='')
Cpd1172 = Metabolite('Cpd1172', formula='', name='IV3-a-NeuAc,III3-a-Fuc-nLc4Cer [g]', compartment='')
Cpd1173 = Metabolite('Cpd1173', formula='', name='Ley glycolipid [g]', compartment='')
Cpd1174 = Metabolite('Cpd1174', formula='', name='(Gal)3 (Glc)1 (GlcNAc)2 (LFuc)2 (Cer)1 [g]', compartment='')
Cpd1175 = Metabolite('Cpd1175', formula='', name='(Gal)3 (Glc)1 (GlcNAc)2 (LFuc)3 (Cer)1 [g]', compartment='')
Cpd1176 = Metabolite('Cpd1176', formula='', name='V3Fuc-nLc6Cer [g]', compartment='')
Cpd1177 = Metabolite('Cpd1177', formula='', name='V3Fuc,III3Fuc-nLc6Cer [g]', compartment='')
Cpd1178 = Metabolite('Cpd1178', formula='', name='III3Fuc-nLc6Cer [g]', compartment='')
Cpd1179 = Metabolite('Cpd1179', formula='', name='alpha-D-mannosyl-beta-D-mannosyl-diacylchitobiosyldiphosphodolichol, human liver homolog [c]', compartment='')
Cpd1180 = Metabolite('Cpd1180', formula='', name='alpha-D-mannosyl-beta-D-mannosyl-diacylchitobiosyldiphosphodolichol, human uterine homolog [c]', compartment='')
Cpd1181 = Metabolite('Cpd1181', formula='', name='keratan sulfate I biosynthesis, precursor 14 [g]', compartment='')
Cpd1182 = Metabolite('Cpd1182', formula='', name='keratan sulfate I biosynthesis, precursor 17 [g]', compartment='')
Cpd1183 = Metabolite('Cpd1183', formula='', name='keratan sulfate I biosynthesis, precursor 20 [g]', compartment='')
Cpd1184 = Metabolite('Cpd1184', formula='', name='keratan sulfate I biosynthesis, precursor 23 [g]', compartment='')
Cpd1185 = Metabolite('Cpd1185', formula='', name='keratan sulfate I biosynthesis, precursor 26 [g]', compartment='')
Cpd1186 = Metabolite('Cpd1186', formula='', name='keratan sulfate I biosynthesis, precursor 29 [g]', compartment='')
Cpd1187 = Metabolite('Cpd1187', formula='', name='keratan sulfate I biosynthesis, precursor 32 [g]', compartment='')
Cpd1188 = Metabolite('Cpd1188', formula='', name='keratan sulfate I biosynthesis, precursor 35 [g]', compartment='')
Cpd1189 = Metabolite('Cpd1189', formula='', name='keratan sulfate I biosynthesis, precursor 36 [g]', compartment='')
Cpd1190 = Metabolite('Cpd1190', formula='', name='keratan sulfate II biosynthesis, precursor 1 [g]', compartment='')
Cpd1191 = Metabolite('Cpd1191', formula='', name='keratan sulfate II biosynthesis, precursor 6 [g]', compartment='')
Cpd1192 = Metabolite('Cpd1192', formula='', name='keratan sulfate II biosynthesis, precursor 9 [g]', compartment='')
Cpd1193 = Metabolite('Cpd1193', formula='', name='keratan sulfate II biosynthesis, precursor 10 [g]', compartment='')
Cpd1194 = Metabolite('Cpd1194', formula='', name='keratan sulfate I biosynthesis, precursor 5 [g]', compartment='')
Cpd1195 = Metabolite('Cpd1195', formula='', name='keratan sulfate I biosynthesis, precursor 8 [g]', compartment='')
Cpd1196 = Metabolite('Cpd1196', formula='', name='keratan sulfate I biosynthesis, precursor 11 [g]', compartment='')
Cpd1197 = Metabolite('Cpd1197', formula='', name='{[galactosyl-(1,4)-N-Acetyl-glucosaminyl-(1,2)-mannosyl-(1,3)-],[N-acetyl-glucosaminyl-(1,2)-mannosyl-(1,6)-]mannosyl-(1,4)-N-acetyl-glucosaminyl-(1,4)-[fucosyl-(1,6)]-N-acetyl-glucosaminyl}asparagine [g]', compartment='')
Cpd1198 = Metabolite('Cpd1198', formula='', name='Glycerol 3-phosphate [c]', compartment='')
Cpd1199 = Metabolite('Cpd1199', formula='', name='L-Glutamate 5-phosphate [m]', compartment='')
Cpd1200 = Metabolite('Cpd1200', formula='', name='L-Glutamate 5-semialdehyde [m]', compartment='')
Cpd1201 = Metabolite('Cpd1201', formula='', name='6-phospho-D-glucono-1,5-lactone [r]', compartment='')
Cpd1202 = Metabolite('Cpd1202', formula='', name='D-Glucose 6-phosphate [c]', compartment='')
Cpd1203 = Metabolite('Cpd1203', formula='', name='6-phospho-D-glucono-1,5-lactone [c]', compartment='')
Cpd1204 = Metabolite('Cpd1204', formula='', name='guanidinoacetic acid [c]', compartment='')
Cpd1205 = Metabolite('Cpd1205', formula='', name='D-galactosyl-N-acylsphingosine [g]', compartment='')
Cpd1206 = Metabolite('Cpd1206', formula='', name='(2R,3S,4R,5R)-5-(6-amino-9H-purin-9-yl)-4-hydroxy-2-({[(sulfonatooxy)phosphinato]oxy}methyl)oxolan-3-yl phosphate [g]', compartment='')
Cpd1207 = Metabolite('Cpd1207', formula='', name='adenosine 3,5-bismonophosphate(4-) [g]', compartment='')
Cpd1208 = Metabolite('Cpd1208', formula='', name='sulfatide galactocerebroside [g]', compartment='')
Cpd1209 = Metabolite('Cpd1209', formula='', name='Digalactosylceramide [g]', compartment='')
Cpd1210 = Metabolite('Cpd1210', formula='', name='Digalactosylceramidesulfate [g]', compartment='')
Cpd1211 = Metabolite('Cpd1211', formula='', name='keratan sulfate I, degradation product 26 [l]', compartment='')
Cpd1212 = Metabolite('Cpd1212', formula='', name='D-Galactose [l]', compartment='')
Cpd1213 = Metabolite('Cpd1213', formula='', name='keratan sulfate I, degradation product 27 [l]', compartment='')
Cpd1214 = Metabolite('Cpd1214', formula='', name='keratan sulfate I, degradation product 29 [l]', compartment='')
Cpd1215 = Metabolite('Cpd1215', formula='', name='keratan sulfate I, degradation product 30 [l]', compartment='')
Cpd1216 = Metabolite('Cpd1216', formula='', name='keratan sulfate I, degradation product 32 [l]', compartment='')
Cpd1217 = Metabolite('Cpd1217', formula='', name='keratan sulfate I, degradation product 33 [l]', compartment='')
Cpd1218 = Metabolite('Cpd1218', formula='', name='keratan sulfate I, degradation product 35 [l]', compartment='')
Cpd1219 = Metabolite('Cpd1219', formula='', name='keratan sulfate I, degradation product 36 [l]', compartment='')
Cpd1220 = Metabolite('Cpd1220', formula='', name='keratan sulfate I, degradation product 38 [l]', compartment='')
Cpd1221 = Metabolite('Cpd1221', formula='', name='keratan sulfate I, degradation product 39 [l]', compartment='')
Cpd1222 = Metabolite('Cpd1222', formula='', name='keratan sulfate I, degradation product 40 [l]', compartment='')
Cpd1223 = Metabolite('Cpd1223', formula='', name='keratan sulfate I, degradation product 41 [l]', compartment='')
Cpd1224 = Metabolite('Cpd1224', formula='', name='keratan sulfate II (core 2-linked), degradation product 2 [l]', compartment='')
Cpd1225 = Metabolite('Cpd1225', formula='', name='keratan sulfate II (core 2-linked), degradation product 3 [l]', compartment='')
Cpd1226 = Metabolite('Cpd1226', formula='', name='keratan sulfate II (core 2-linked), degradation product 5 [l]', compartment='')
Cpd1227 = Metabolite('Cpd1227', formula='', name='keratan sulfate II (core 2-linked), degradation product 6 [l]', compartment='')
Cpd1228 = Metabolite('Cpd1228', formula='', name='keratan sulfate II (core 2-linked), degradation product 8 [l]', compartment='')
Cpd1229 = Metabolite('Cpd1229', formula='', name='keratan sulfate II (core 2-linked), degradation product 9 [l]', compartment='')
Cpd1230 = Metabolite('Cpd1230', formula='', name='F1alpha [l]', compartment='')
Cpd1231 = Metabolite('Cpd1231', formula='', name='Core 6 [l]', compartment='')
Cpd1232 = Metabolite('Cpd1232', formula='', name='de-Fuc, reducing GlcNAc removed, de-Sia form of PA6 (w/o peptide linkage) [l]', compartment='')
Cpd1233 = Metabolite('Cpd1233', formula='', name='keratan sulfate II (core 4-linked), degradation product 2 [l]', compartment='')
Cpd1234 = Metabolite('Cpd1234', formula='', name='keratan sulfate II (core 4-linked), degradation product 3 [l]', compartment='')
Cpd1235 = Metabolite('Cpd1235', formula='', name='keratan sulfate I, degradation product 5 [l]', compartment='')
Cpd1236 = Metabolite('Cpd1236', formula='', name='keratan sulfate I, degradation product 6 [l]', compartment='')
Cpd1237 = Metabolite('Cpd1237', formula='', name='keratan sulfate I, degradation product 8 [l]', compartment='')
Cpd1238 = Metabolite('Cpd1238', formula='', name='keratan sulfate I, degradation product 9 [l]', compartment='')
Cpd1239 = Metabolite('Cpd1239', formula='', name='keratan sulfate I, degradation product 11 [l]', compartment='')
Cpd1240 = Metabolite('Cpd1240', formula='', name='keratan sulfate I, degradation product 12 [l]', compartment='')
Cpd1241 = Metabolite('Cpd1241', formula='', name='keratan sulfate I, degradation product 14 [l]', compartment='')
Cpd1242 = Metabolite('Cpd1242', formula='', name='keratan sulfate I, degradation product 15 [l]', compartment='')
Cpd1243 = Metabolite('Cpd1243', formula='', name='keratan sulfate I, degradation product 17 [l]', compartment='')
Cpd1244 = Metabolite('Cpd1244', formula='', name='keratan sulfate I, degradation product 18 [l]', compartment='')
Cpd1245 = Metabolite('Cpd1245', formula='', name='keratan sulfate I, degradation product 20 [l]', compartment='')
Cpd1246 = Metabolite('Cpd1246', formula='', name='keratan sulfate I, degradation product 21 [l]', compartment='')
Cpd1247 = Metabolite('Cpd1247', formula='', name='keratan sulfate I, degradation product 23 [l]', compartment='')
Cpd1248 = Metabolite('Cpd1248', formula='', name='keratan sulfate I, degradation product 24 [l]', compartment='')
Cpd1249 = Metabolite('Cpd1249', formula='', name='alpha-N-acetylneuraminosyl-(2->3)-beta-D-galactosyl-(1->4)-beta-D-glucosylceramide(1-) [g]', compartment='')
Cpd1250 = Metabolite('Cpd1250', formula='', name='GD3 [g]', compartment='')
Cpd1251 = Metabolite('Cpd1251', formula='', name='GT3 [g]', compartment='')
Cpd1252 = Metabolite('Cpd1252', formula='', name='D-Galactose [c]', compartment='')
Cpd1253 = Metabolite('Cpd1253', formula='', name='(2R,3R,4S,5R,6R)-3,4,5-trihydroxy-6-(hydroxymethyl)oxan-2-yl phosphate [c]', compartment='')
Cpd1254 = Metabolite('Cpd1254', formula='', name='chondroitin sulfate/heparan sulfate linkage region (GlcA-(Gal)2-Xyl-L-Ser (protein)) [g]', compartment='')
Cpd1255 = Metabolite('Cpd1255', formula='', name='chondroitin sulfate precursor (GalNAc-GlcA-(Gal)2-Xyl-L-Ser (protein)) [g]', compartment='')
Cpd1256 = Metabolite('Cpd1256', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA) and B (IdoA2S-GalNAc4S), precursor 2 [g]', compartment='')
Cpd1257 = Metabolite('Cpd1257', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA) and B (IdoA2S-GalNAc4S), precursor 3 [g]', compartment='')
Cpd1258 = Metabolite('Cpd1258', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), precursor 2 [g]', compartment='')
Cpd1259 = Metabolite('Cpd1259', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), precursor 3 [g]', compartment='')
Cpd1260 = Metabolite('Cpd1260', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), precursor 3 [g]', compartment='')
Cpd1261 = Metabolite('Cpd1261', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), precursor 4 [g]', compartment='')
Cpd1262 = Metabolite('Cpd1262', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), precursor 3 [g]', compartment='')
Cpd1263 = Metabolite('Cpd1263', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), precursor 4 [g]', compartment='')
Cpd1264 = Metabolite('Cpd1264', formula='', name='protein-linked serine or threonine residue (O-glycosylation site) [g]', compartment='')
Cpd1265 = Metabolite('Cpd1265', formula='', name='galactitol [c]', compartment='')
Cpd1266 = Metabolite('Cpd1266', formula='', name='D-Galactose [e]', compartment='')
Cpd1267 = Metabolite('Cpd1267', formula='', name='Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1268 = Metabolite('Cpd1268', formula='', name='(Gal)2-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1269 = Metabolite('Cpd1269', formula='', name='Xyl-L-Ser (protein) [g]', compartment='')
Cpd1270 = Metabolite('Cpd1270', formula='', name='UDPglucose [c]', compartment='')
Cpd1271 = Metabolite('Cpd1271', formula='', name='D-Glucosamine [e]', compartment='')
Cpd1272 = Metabolite('Cpd1272', formula='', name='D-Glucosamine [c]', compartment='')
Cpd1273 = Metabolite('Cpd1273', formula='', name='glycogen, structure 5 (glycogenin-2[1,4-Glc]) [e]', compartment='')
Cpd1274 = Metabolite('Cpd1274', formula='', name='Maltose [e]', compartment='')
Cpd1275 = Metabolite('Cpd1275', formula='', name='Tyr-194 of apo-glycogenin protein (primer for glycogen synthesis) [e]', compartment='')
Cpd1276 = Metabolite('Cpd1276', formula='', name='GD3 [c]', compartment='')
Cpd1277 = Metabolite('Cpd1277', formula='', name='9-O-Acetylated GD3 [c]', compartment='')
Cpd1278 = Metabolite('Cpd1278', formula='', name='Coenzyme A [g]', compartment='')
Cpd1279 = Metabolite('Cpd1279', formula='', name='9-O-Acetylated GD3 [g]', compartment='')
Cpd1280 = Metabolite('Cpd1280', formula='', name='GT3 [c]', compartment='')
Cpd1281 = Metabolite('Cpd1281', formula='', name='9-O-Acetylated GT3 [c]', compartment='')
Cpd1282 = Metabolite('Cpd1282', formula='', name='9-O-Acetylated GT3 [g]', compartment='')
Cpd1283 = Metabolite('Cpd1283', formula='', name='N1-(5-Phospho-D-ribosyl)glycinamide [c]', compartment='')
Cpd1284 = Metabolite('Cpd1284', formula='', name='N2-Formyl-N1-(5-phospho-D-ribosyl)glycinamide [c]', compartment='')
Cpd1285 = Metabolite('Cpd1285', formula='', name='N-Acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,3-(N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,6)-(N-acetyl-beta-D-glucosaminyl-1,4)-beta-D-mannosyl-1,4-N-acetyl-beta-D-glucosaminyl-R [l]', compartment='')
Cpd1286 = Metabolite('Cpd1286', formula='', name='protein-linked asparagine residue (N-glycosylation site) [l]', compartment='')
Cpd1287 = Metabolite('Cpd1287', formula='', name='n2m2nmasn (w/o peptide linkage) [l]', compartment='')
Cpd1288 = Metabolite('Cpd1288', formula='', name='keratan sulfate I, degradation product 2 [l]', compartment='')
Cpd1289 = Metabolite('Cpd1289', formula='', name='de-Fuc form of PA6 (w/o peptide linkage) [l]', compartment='')
Cpd1290 = Metabolite('Cpd1290', formula='', name='D-glucose [c]', compartment='')
Cpd1291 = Metabolite('Cpd1291', formula='', name='D-glucosyl-N-acylsphingosine [l]', compartment='')
Cpd1292 = Metabolite('Cpd1292', formula='', name='D-glucose [l]', compartment='')
Cpd1293 = Metabolite('Cpd1293', formula='', name='alpha GalNAc globoside [g]', compartment='')
Cpd1294 = Metabolite('Cpd1294', formula='', name='glycolate [c]', compartment='')
Cpd1295 = Metabolite('Cpd1295', formula='', name='glycolaldehyde [m]', compartment='')
Cpd1296 = Metabolite('Cpd1296', formula='', name='glycolate [m]', compartment='')
Cpd1297 = Metabolite('Cpd1297', formula='', name='lipoamide [m]', compartment='')
Cpd1298 = Metabolite('Cpd1298', formula='', name='S-aminomethyldihydrolipoamide [m]', compartment='')
Cpd1299 = Metabolite('Cpd1299', formula='', name='dihydrolipoamide [m]', compartment='')
Cpd1300 = Metabolite('Cpd1300', formula='', name='5,10-Methylenetetrahydrofolate [m]', compartment='')
Cpd1301 = Metabolite('Cpd1301', formula='', name='Lipoylprotein [m]', compartment='')
Cpd1302 = Metabolite('Cpd1302', formula='', name='S-Aminomethyldihydrolipoylprotein [m]', compartment='')
Cpd1303 = Metabolite('Cpd1303', formula='', name='Dihydrolipolprotein [m]', compartment='')
Cpd1304 = Metabolite('Cpd1304', formula='', name='glycocholate [e]', compartment='')
Cpd1305 = Metabolite('Cpd1305', formula='', name='glycocholate [c]', compartment='')
Cpd1306 = Metabolite('Cpd1306', formula='', name='GMP [g]', compartment='')
Cpd1307 = Metabolite('Cpd1307', formula='', name='GMP [c]', compartment='')
Cpd1308 = Metabolite('Cpd1308', formula='', name='GDP-4-dehydro-6-deoxy-alpha-D-mannose(2-) [c]', compartment='')
Cpd1309 = Metabolite('Cpd1309', formula='', name='10-formyltetrahydrofolate-[Glu](5) [e]', compartment='')
Cpd1310 = Metabolite('Cpd1310', formula='', name='10-Formyltetrahydrofolate [e]', compartment='')
Cpd1311 = Metabolite('Cpd1311', formula='', name='10-formyltetrahydrofolate-[Glu](5) [l]', compartment='')
Cpd1312 = Metabolite('Cpd1312', formula='', name='10-Formyltetrahydrofolate [l]', compartment='')
Cpd1313 = Metabolite('Cpd1313', formula='', name='L-glutamate(1-) [l]', compartment='')
Cpd1314 = Metabolite('Cpd1314', formula='', name='10-formyltetrahydrofolate-[Glu](6) [e]', compartment='')
Cpd1315 = Metabolite('Cpd1315', formula='', name='10-formyltetrahydrofolate-[Glu](6) [l]', compartment='')
Cpd1316 = Metabolite('Cpd1316', formula='', name='10-formyltetrahydrofolate-[Glu](7) [e]', compartment='')
Cpd1317 = Metabolite('Cpd1317', formula='', name='10-formyltetrahydrofolate-[Glu](7) [l]', compartment='')
Cpd1318 = Metabolite('Cpd1318', formula='', name='pentaglutamyl folate (DHF) [e]', compartment='')
Cpd1319 = Metabolite('Cpd1319', formula='', name='7,8-Dihydrofolate [e]', compartment='')
Cpd1320 = Metabolite('Cpd1320', formula='', name='pentaglutamyl folate (DHF) [l]', compartment='')
Cpd1321 = Metabolite('Cpd1321', formula='', name='7,8-Dihydrofolate [l]', compartment='')
Cpd1322 = Metabolite('Cpd1322', formula='', name='pentaglutamyl folate (THF) [e]', compartment='')
Cpd1323 = Metabolite('Cpd1323', formula='', name='5,6,7,8-Tetrahydrofolate [e]', compartment='')
Cpd1324 = Metabolite('Cpd1324', formula='', name='pentaglutamyl folate (THF) [l]', compartment='')
Cpd1325 = Metabolite('Cpd1325', formula='', name='5,6,7,8-Tetrahydrofolate [l]', compartment='')
Cpd1326 = Metabolite('Cpd1326', formula='', name='hexaglutamyl folate (DHF) [e]', compartment='')
Cpd1327 = Metabolite('Cpd1327', formula='', name='hexaglutamyl folate (DHF) [l]', compartment='')
Cpd1328 = Metabolite('Cpd1328', formula='', name='hexaglutamyl folate  (THF) [e]', compartment='')
Cpd1329 = Metabolite('Cpd1329', formula='', name='hexaglutamyl folate  (THF) [l]', compartment='')
Cpd1330 = Metabolite('Cpd1330', formula='', name='heptaglutamyl folate (DHF) [e]', compartment='')
Cpd1331 = Metabolite('Cpd1331', formula='', name='heptaglutamyl folate (DHF) [l]', compartment='')
Cpd1332 = Metabolite('Cpd1332', formula='', name='heptaglutamyl folate (THF) [e]', compartment='')
Cpd1333 = Metabolite('Cpd1333', formula='', name='heptaglutamyl folate (THF) [l]', compartment='')
Cpd1334 = Metabolite('Cpd1334', formula='', name='5-L-Glutamyl-L-alanine [e]', compartment='')
Cpd1335 = Metabolite('Cpd1335', formula='', name='5-oxoprolinate [c]', compartment='')
Cpd1336 = Metabolite('Cpd1336', formula='', name='Tyr-194 of apo-glycogenin protein (primer for glycogen synthesis) [c]', compartment='')
Cpd1337 = Metabolite('Cpd1337', formula='', name='primed glycogenin (glycogenin-8[1,4-Glc]) [c]', compartment='')
Cpd1338 = Metabolite('Cpd1338', formula='', name='leukotriene C4(2-) [r]', compartment='')
Cpd1339 = Metabolite('Cpd1339', formula='', name='L-glutamate(1-) [r]', compartment='')
Cpd1340 = Metabolite('Cpd1340', formula='', name='Leukotriene D4 [r]', compartment='')
Cpd1341 = Metabolite('Cpd1341', formula='', name='5,10-Methylenetetrahydrofolate [c]', compartment='')
Cpd1342 = Metabolite('Cpd1342', formula='', name='L-serine [m]', compartment='')
Cpd1343 = Metabolite('Cpd1343', formula='', name='3-Hydroxy-N6,N6,N6-trimethyl-L-lysine [c]', compartment='')
Cpd1344 = Metabolite('Cpd1344', formula='', name='4-trimethylammoniobutanal [c]', compartment='')
Cpd1345 = Metabolite('Cpd1345', formula='', name='3-Hydroxy-N6,N6,N6-trimethyl-L-lysine [m]', compartment='')
Cpd1346 = Metabolite('Cpd1346', formula='', name='4-trimethylammoniobutanal [m]', compartment='')
Cpd1347 = Metabolite('Cpd1347', formula='', name='GMP [m]', compartment='')
Cpd1348 = Metabolite('Cpd1348', formula='', name='D-glucurono-6,3-lactone [c]', compartment='')
Cpd1349 = Metabolite('Cpd1349', formula='', name='D-glucarate(2-) [c]', compartment='')
Cpd1350 = Metabolite('Cpd1350', formula='', name='D-glucurono-6,3-lactone [m]', compartment='')
Cpd1351 = Metabolite('Cpd1351', formula='', name='D-glucarate(2-) [m]', compartment='')
Cpd1352 = Metabolite('Cpd1352', formula='', name='Digalactosylceramide [l]', compartment='')
Cpd1353 = Metabolite('Cpd1353', formula='', name='galactosyl glucosyl ceramide [l]', compartment='')
Cpd1354 = Metabolite('Cpd1354', formula='', name='glycogen, structure 1 (glycogenin-11[1,4-Glc]) [c]', compartment='')
Cpd1355 = Metabolite('Cpd1355', formula='', name='glycogen, structure 2 (glycogenin-1,6-{7[1,4-Glc], 4[1,4-Glc]}) [c]', compartment='')
Cpd1356 = Metabolite('Cpd1356', formula='', name='heparan sulfate, degradation product 9 [l]', compartment='')
Cpd1357 = Metabolite('Cpd1357', formula='', name='D-glucuronate [l]', compartment='')
Cpd1358 = Metabolite('Cpd1358', formula='', name='heparan sulfate, degradation product 10 [l]', compartment='')
Cpd1359 = Metabolite('Cpd1359', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA), degradation product 2 [l]', compartment='')
Cpd1360 = Metabolite('Cpd1360', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA), degradation product 3 [l]', compartment='')
Cpd1361 = Metabolite('Cpd1361', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), degradation product 2 [l]', compartment='')
Cpd1362 = Metabolite('Cpd1362', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), degradation product 3 [l]', compartment='')
Cpd1363 = Metabolite('Cpd1363', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), degradation product 3 [l]', compartment='')
Cpd1364 = Metabolite('Cpd1364', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), degradation product 4 [l]', compartment='')
Cpd1365 = Metabolite('Cpd1365', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), degradation product 3 [l]', compartment='')
Cpd1366 = Metabolite('Cpd1366', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), degradation product 4 [l]', compartment='')
Cpd1367 = Metabolite('Cpd1367', formula='', name='hyaluronan [l]', compartment='')
Cpd1368 = Metabolite('Cpd1368', formula='', name='hyaluronan degradation product 1 [l]', compartment='')
Cpd1369 = Metabolite('Cpd1369', formula='', name='hyaluronan biosynthesis, precursor 1 [l]', compartment='')
Cpd1370 = Metabolite('Cpd1370', formula='', name='(GlcN2S-GlcA)4-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1371 = Metabolite('Cpd1371', formula='', name='(GlcN2S-IdoA-GlcN2S-GlcA)2-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1372 = Metabolite('Cpd1372', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA), B (IdoA2S-GalNAc4S), and E (GalNAc4,6diS-GlcA), precursor 1 [g]', compartment='')
Cpd1373 = Metabolite('Cpd1373', formula='', name='UDP-D-glucuronate [g]', compartment='')
Cpd1374 = Metabolite('Cpd1374', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA) and D (GlcNAc6S-GlcA2S), precursor 1 [g]', compartment='')
Cpd1375 = Metabolite('Cpd1375', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S) precursor 2 [g]', compartment='')
Cpd1376 = Metabolite('Cpd1376', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), precursor 2 [g]', compartment='')
Cpd1377 = Metabolite('Cpd1377', formula='', name='heparan sulfate, precursor 1 [g]', compartment='')
Cpd1378 = Metabolite('Cpd1378', formula='', name='heparan sulfate, precursor 2 [g]', compartment='')
Cpd1379 = Metabolite('Cpd1379', formula='', name='GlcNAc-GlcA-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1380 = Metabolite('Cpd1380', formula='', name='GlcA-GlcNAc-GlcA-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1381 = Metabolite('Cpd1381', formula='', name='(GlcNAc-GlcA)2-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1382 = Metabolite('Cpd1382', formula='', name='GlcA-(GlcNAc-GlcA)2-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1383 = Metabolite('Cpd1383', formula='', name='(GlcNAc-GlcA)3-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1384 = Metabolite('Cpd1384', formula='', name='GlcA-(GlcNAc-GlcA)3-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1385 = Metabolite('Cpd1385', formula='', name='heparan sulfate, degradation product 3 [l]', compartment='')
Cpd1386 = Metabolite('Cpd1386', formula='', name='heparan sulfate, degradation product 4 [l]', compartment='')
Cpd1387 = Metabolite('Cpd1387', formula='', name='heparan sulfate, degradation product 8 [l]', compartment='')
Cpd1388 = Metabolite('Cpd1388', formula='', name='heparan sulfate, degradation product 14 [l]', compartment='')
Cpd1389 = Metabolite('Cpd1389', formula='', name='heparan sulfate, degradation product 15 [l]', compartment='')
Cpd1390 = Metabolite('Cpd1390', formula='', name='heparan sulfate, degradation product 20 [l]', compartment='')
Cpd1391 = Metabolite('Cpd1391', formula='', name='IdoA2S-beta-1,4-GlcNAc(3S)-alpha-1,4-GlcA-beta-1,3-Gal-beta-1,3-Gal-beta-1,4-Xylose [l]', compartment='')
Cpd1392 = Metabolite('Cpd1392', formula='', name='GlcNAc-GlcA-(Gal)2-Xyl [l]', compartment='')
Cpd1393 = Metabolite('Cpd1393', formula='', name='GlcA-beta-1,3-Gal-beta-1,3-Gal-beta-1,4-Xylose [l]', compartment='')
Cpd1394 = Metabolite('Cpd1394', formula='', name='(GlcNAc-GlcA)4-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd1395 = Metabolite('Cpd1395', formula='', name='acetate [g]', compartment='')
Cpd1396 = Metabolite('Cpd1396', formula='', name='UDP-N-acetyl-alpha-D-glucosamine(2-) [c]', compartment='')
Cpd1397 = Metabolite('Cpd1397', formula='', name='N-Acetyl-D-glucosaminyldiphosphodolichol, human liver homolog [c]', compartment='')
Cpd1398 = Metabolite('Cpd1398', formula='', name='UMP [c]', compartment='')
Cpd1399 = Metabolite('Cpd1399', formula='', name='N-Acetyl-D-glucosaminyldiphosphodolichol, human uterine homolog [c]', compartment='')
Cpd1400 = Metabolite('Cpd1400', formula='', name='D-glucose [g]', compartment='')
Cpd1401 = Metabolite('Cpd1401', formula='', name='D-glucuronate [c]', compartment='')
Cpd1402 = Metabolite('Cpd1402', formula='', name='phosphorylase-limit dextrin (glycogenin-1,6{4[1,4-Glc], 4[1,4-Glc]}) [c]', compartment='')
Cpd1403 = Metabolite('Cpd1403', formula='', name='glycogen, structure 3 (glycogenin-7[1,4-Glc]) [c]', compartment='')
Cpd1404 = Metabolite('Cpd1404', formula='', name='gamma-L-Glutamyl-L-cysteine [c]', compartment='')
Cpd1405 = Metabolite('Cpd1405', formula='', name='N-Formimidoyl-L-glutamate [c]', compartment='')
Cpd1406 = Metabolite('Cpd1406', formula='', name='L-glutamine [m]', compartment='')
Cpd1407 = Metabolite('Cpd1407', formula='', name='5-Phospho-beta-D-ribosylamine [c]', compartment='')
Cpd1408 = Metabolite('Cpd1408', formula='', name='glyoxylate [c]', compartment='')
Cpd1409 = Metabolite('Cpd1409', formula='', name='oxalate(2-) [x]', compartment='')
Cpd1410 = Metabolite('Cpd1410', formula='', name='L-2-amino-3-oxobutanoic acid [m]', compartment='')
Cpd1411 = Metabolite('Cpd1411', formula='', name='glycine betaine [e]', compartment='')
Cpd1412 = Metabolite('Cpd1412', formula='', name='glycolate [x]', compartment='')
Cpd1413 = Metabolite('Cpd1413', formula='', name='Glycerol [c]', compartment='')
Cpd1414 = Metabolite('Cpd1414', formula='', name='Glycerol [m]', compartment='')
Cpd1415 = Metabolite('Cpd1415', formula='', name='Glycerol 3-phosphate [m]', compartment='')
Cpd1416 = Metabolite('Cpd1416', formula='', name='(R)-S-lactoylglutathionate(1-) [c]', compartment='')
Cpd1417 = Metabolite('Cpd1417', formula='', name='(R)-S-lactoylglutathionate(1-) [m]', compartment='')
Cpd1418 = Metabolite('Cpd1418', formula='', name='Reduced glutathione [m]', compartment='')
Cpd1419 = Metabolite('Cpd1419', formula='', name='Glycine [l]', compartment='')
Cpd1420 = Metabolite('Cpd1420', formula='', name='Xanthosine 5-phosphate [c]', compartment='')
Cpd1421 = Metabolite('Cpd1421', formula='', name='6-Phospho-D-gluconate [c]', compartment='')
Cpd1422 = Metabolite('Cpd1422', formula='', name='D-ribulose 5-phosphate(2-) [c]', compartment='')
Cpd1423 = Metabolite('Cpd1423', formula='', name='6-Phospho-D-gluconate [r]', compartment='')
Cpd1424 = Metabolite('Cpd1424', formula='', name='D-ribulose 5-phosphate(2-) [r]', compartment='')
Cpd1425 = Metabolite('Cpd1425', formula='', name='Sarcosine [c]', compartment='')
Cpd1426 = Metabolite('Cpd1426', formula='', name='R total Coenzyme A [m]', compartment='')
Cpd1427 = Metabolite('Cpd1427', formula='', name='lysophosphatidic acid [m]', compartment='')
Cpd1428 = Metabolite('Cpd1428', formula='', name='glucosaminyl-acylphosphatidylinositol [c]', compartment='')
Cpd1429 = Metabolite('Cpd1429', formula='', name='(phosphoethanolaminyl-dimannosyl),(phosphoethanolaminyl)-mannosyl-glucosaminyl-acylphosphatidylinositol [r]', compartment='')
Cpd1430 = Metabolite('Cpd1430', formula='', name='deacylated-(phosphoethanolaminyl-dimannosyl),(phosphoethanolaminyl)-mannosyl-glucosaminyl-acylphosphatidylinositol [r]', compartment='')
Cpd1431 = Metabolite('Cpd1431', formula='', name='Hexadecanoate (n-C16:0) [r]', compartment='')
Cpd1432 = Metabolite('Cpd1432', formula='', name='{[(Prot-PE-Man),(PE)-Man],[PE]}-Man-GlcN-acylPI [r]', compartment='')
Cpd1433 = Metabolite('Cpd1433', formula='', name='deacylated-glycophosphatidylinositol (GPI)-anchored protein [r]', compartment='')
Cpd1434 = Metabolite('Cpd1434', formula='', name='glucosaminyl-acylphosphatidylinositol [r]', compartment='')
Cpd1435 = Metabolite('Cpd1435', formula='', name='mannosyl-glucosaminyl-acylphosphatidylinositiol (H2) [r]', compartment='')
Cpd1436 = Metabolite('Cpd1436', formula='', name='Farnesyl diphosphate [x]', compartment='')
Cpd1437 = Metabolite('Cpd1437', formula='', name='Guanosine [e]', compartment='')
Cpd1438 = Metabolite('Cpd1438', formula='', name='Guanosine [c]', compartment='')
Cpd1439 = Metabolite('Cpd1439', formula='', name='Guanosine [l]', compartment='')
Cpd1440 = Metabolite('Cpd1440', formula='', name='Guanosine [m]', compartment='')
Cpd1441 = Metabolite('Cpd1441', formula='', name='Oxidized glutathione [c]', compartment='')
Cpd1442 = Metabolite('Cpd1442', formula='', name='Oxidized glutathione [m]', compartment='')
Cpd1443 = Metabolite('Cpd1443', formula='', name='Reduced glutathione [e]', compartment='')
Cpd1444 = Metabolite('Cpd1444', formula='', name='Hydrogen peroxide [e]', compartment='')
Cpd1445 = Metabolite('Cpd1445', formula='', name='Oxidized glutathione [e]', compartment='')
Cpd1446 = Metabolite('Cpd1446', formula='', name='7,8-dihydroneopterin 3-triphosphate(4-) [c]', compartment='')
Cpd1447 = Metabolite('Cpd1447', formula='', name='Formate [n]', compartment='')
Cpd1448 = Metabolite('Cpd1448', formula='', name='7,8-dihydroneopterin 3-triphosphate(4-) [n]', compartment='')
Cpd1449 = Metabolite('Cpd1449', formula='', name='Guanine [c]', compartment='')
Cpd1450 = Metabolite('Cpd1450', formula='', name='Xanthine [c]', compartment='')
Cpd1451 = Metabolite('Cpd1451', formula='', name='Guanine [e]', compartment='')
Cpd1452 = Metabolite('Cpd1452', formula='', name='phosphatidylethanolamine [r]', compartment='')
Cpd1453 = Metabolite('Cpd1453', formula='', name='diglyceride [r]', compartment='')
Cpd1454 = Metabolite('Cpd1454', formula='', name='phosphoethanolaminyl-mannosyl-glucosylaminyl-acylphosphatidylinositol (H5) [r]', compartment='')
Cpd1455 = Metabolite('Cpd1455', formula='', name='dimannosyl-glucosaminyl-acylphosphatidylinositol (H3) [r]', compartment='')
Cpd1456 = Metabolite('Cpd1456', formula='', name='trimannosyl-glucosaminyl-acylphosphatidylinositol (H4) [r]', compartment='')
Cpd1457 = Metabolite('Cpd1457', formula='', name='phosphoethanolaminyl-trimannosyl-glucosaminyl-acylphosphatidylinositol (H6) [r]', compartment='')
Cpd1458 = Metabolite('Cpd1458', formula='', name='(phosphoethanolaminyl-mannosyl),(phosphoethanolaminyl)-dimannosyl-glucosaminyl-acylphosphatidylinositol (H7) [r]', compartment='')
Cpd1459 = Metabolite('Cpd1459', formula='', name='(phosphoethanolaminyl-dimannosyl),(phosphoethanolaminyl)-mannosyl-glucosaminyl-acylphosphatidylinositol (H7) [r]', compartment='')
Cpd1460 = Metabolite('Cpd1460', formula='', name='(trimannosyl),(phosphoethanolaminyl)-mannosyl-glucosaminyl-acylphosphatidylinositol (M4C) [r]', compartment='')
Cpd1461 = Metabolite('Cpd1461', formula='', name='{[PE-Man-],[PE-]Man},{PE-}Man-GlcN-acylPI [r]', compartment='')
Cpd1462 = Metabolite('Cpd1462', formula='', name='({[(mannosyl),(phosphoethanolaminyl)]-dimannosyl},{phosphoethanolaminyl})-mannosyl-glucosaminyl-acylphosphatidylinositol (M4B) [r]', compartment='')
Cpd1463 = Metabolite('Cpd1463', formula='', name='glycophosphatidylinositol (GPI)-anchored protein precursor [r]', compartment='')
Cpd1464 = Metabolite('Cpd1464', formula='', name='glycophosphatidylinositol (GPI) signal sequence (C-terminal peptide) [r]', compartment='')
Cpd1465 = Metabolite('Cpd1465', formula='', name='mannosyl-3-(phosphoethanolaminyl-mannosyl)-glucosaminyl-acylphosphatidylinositol (M4A) [r]', compartment='')
Cpd1466 = Metabolite('Cpd1466', formula='', name='10fthf5glu[c] [r]', compartment='')
Cpd1467 = Metabolite('Cpd1467', formula='', name='UDP-D-glucuronate [c]', compartment='')
Cpd1468 = Metabolite('Cpd1468', formula='', name='hyaluronan biosynthesis, precursor 1 [e]', compartment='')
Cpd1469 = Metabolite('Cpd1469', formula='', name='hyaluronan [e]', compartment='')
Cpd1470 = Metabolite('Cpd1470', formula='', name='4-hydroxybenzoate [m]', compartment='')
Cpd1471 = Metabolite('Cpd1471', formula='', name='all trans Decaprenyl diphosphate [m]', compartment='')
Cpd1472 = Metabolite('Cpd1472', formula='', name='3-Decaprenyl-4-hydroxybenzoate [m]', compartment='')
Cpd1473 = Metabolite('Cpd1473', formula='', name='Hexadecenoate (n-C16:1) [e]', compartment='')
Cpd1474 = Metabolite('Cpd1474', formula='', name='hexacosanoyl carnitine [c]', compartment='')
Cpd1475 = Metabolite('Cpd1475', formula='', name='hexacosanoyl carnitine [m]', compartment='')
Cpd1476 = Metabolite('Cpd1476', formula='', name='Hexacosanoyl-CoA (n-C26:0CoA) [m]', compartment='')
Cpd1477 = Metabolite('Cpd1477', formula='', name='4-maleylacetoacetate [c]', compartment='')
Cpd1478 = Metabolite('Cpd1478', formula='', name='3-hydroxy-2-methylpropanoate [m]', compartment='')
Cpd1479 = Metabolite('Cpd1479', formula='', name='L-histidine [c]', compartment='')
Cpd1480 = Metabolite('Cpd1480', formula='', name='Urocanate [c]', compartment='')
Cpd1481 = Metabolite('Cpd1481', formula='', name='histamium [c]', compartment='')
Cpd1482 = Metabolite('Cpd1482', formula='', name='L-histidine [e]', compartment='')
Cpd1483 = Metabolite('Cpd1483', formula='', name='imidazole-4-acetaldehyde [c]', compartment='')
Cpd1484 = Metabolite('Cpd1484', formula='', name='histamium [e]', compartment='')
Cpd1485 = Metabolite('Cpd1485', formula='', name='Porphobilinogen [c]', compartment='')
Cpd1486 = Metabolite('Cpd1486', formula='', name='Hydroxymethylbilane [c]', compartment='')
Cpd1487 = Metabolite('Cpd1487', formula='', name='Hydroxymethylglutaryl-CoA [r]', compartment='')
Cpd1488 = Metabolite('Cpd1488', formula='', name='(R)-mevalonate [r]', compartment='')
Cpd1489 = Metabolite('Cpd1489', formula='', name='Hydroxymethylglutaryl-CoA [c]', compartment='')
Cpd1490 = Metabolite('Cpd1490', formula='', name='Hydroxymethylglutaryl-CoA [m]', compartment='')
Cpd1491 = Metabolite('Cpd1491', formula='', name='Hydroxymethylglutaryl-CoA [x]', compartment='')
Cpd1492 = Metabolite('Cpd1492', formula='', name='acetoacetate [x]', compartment='')
Cpd1493 = Metabolite('Cpd1493', formula='', name='L-homoserine [e]', compartment='')
Cpd1494 = Metabolite('Cpd1494', formula='', name='L-homoserine [c]', compartment='')
Cpd1495 = Metabolite('Cpd1495', formula='', name='Protoheme [c]', compartment='')
Cpd1496 = Metabolite('Cpd1496', formula='', name='carbon monoxide [c]', compartment='')
Cpd1497 = Metabolite('Cpd1497', formula='', name='Fe2+ [c]', compartment='')
Cpd1498 = Metabolite('Cpd1498', formula='', name='heptadecanoyl carnitine [c]', compartment='')
Cpd1499 = Metabolite('Cpd1499', formula='', name='heptadecanoyl carnitine [m]', compartment='')
Cpd1500 = Metabolite('Cpd1500', formula='', name='3-hydroxypyruvate [c]', compartment='')
Cpd1501 = Metabolite('Cpd1501', formula='', name='(S)-Glycerate [c]', compartment='')
Cpd1502 = Metabolite('Cpd1502', formula='', name='D-glycerate [c]', compartment='')
Cpd1503 = Metabolite('Cpd1503', formula='', name='heparan sulfate, degradation product 1 [l]', compartment='')
Cpd1504 = Metabolite('Cpd1504', formula='', name='heparan sulfate, degradation product 2 [l]', compartment='')
Cpd1505 = Metabolite('Cpd1505', formula='', name='heparan sulfate, degradation product 6 [l]', compartment='')
Cpd1506 = Metabolite('Cpd1506', formula='', name='heparan sulfate, degradation product 7 [l]', compartment='')
Cpd1507 = Metabolite('Cpd1507', formula='', name='heparan sulfate, degradation product 12 [l]', compartment='')
Cpd1508 = Metabolite('Cpd1508', formula='', name='heparan sulfate, degradation product 13 [l]', compartment='')
Cpd1509 = Metabolite('Cpd1509', formula='', name='heparan sulfate, degradation product 18 [l]', compartment='')
Cpd1510 = Metabolite('Cpd1510', formula='', name='heparan sulfate, degradation product 19 [l]', compartment='')
Cpd1511 = Metabolite('Cpd1511', formula='', name='cortisone [r]', compartment='')
Cpd1512 = Metabolite('Cpd1512', formula='', name='cortisol [r]', compartment='')
Cpd1513 = Metabolite('Cpd1513', formula='', name='estradiol [c]', compartment='')
Cpd1514 = Metabolite('Cpd1514', formula='', name='androst-4-ene-3,17-dione [r]', compartment='')
Cpd1515 = Metabolite('Cpd1515', formula='', name='testosterone [r]', compartment='')
Cpd1516 = Metabolite('Cpd1516', formula='', name='Nicotinamide adenine dinucleotide phosphate - reduced [x]', compartment='')
Cpd1517 = Metabolite('Cpd1517', formula='', name='Nicotinamide adenine dinucleotide phosphate [x]', compartment='')
Cpd1518 = Metabolite('Cpd1518', formula='', name='3alpha,7alpha,12alpha-trihydroxy-24-oxo-5beta-cholestan-26-oyl-CoA(4-) [x]', compartment='')
Cpd1519 = Metabolite('Cpd1519', formula='', name='estrone [r]', compartment='')
Cpd1520 = Metabolite('Cpd1520', formula='', name='estradiol [r]', compartment='')
Cpd1521 = Metabolite('Cpd1521', formula='', name='androsterone [r]', compartment='')
Cpd1522 = Metabolite('Cpd1522', formula='', name='16alpha-hydroxydehydroepiandrosterone [r]', compartment='')
Cpd1523 = Metabolite('Cpd1523', formula='', name='5alpha-androstane-3,17-dione [r]', compartment='')
Cpd1524 = Metabolite('Cpd1524', formula='', name='pregnenolone [c]', compartment='')
Cpd1525 = Metabolite('Cpd1525', formula='', name='pregnenolone [r]', compartment='')
Cpd1526 = Metabolite('Cpd1526', formula='', name='progesterone [r]', compartment='')
Cpd1527 = Metabolite('Cpd1527', formula='', name='3beta-hydroxyandrost-5-en-17-one [r]', compartment='')
Cpd1528 = Metabolite('Cpd1528', formula='', name='17alpha-hydroxypregnenolone [c]', compartment='')
Cpd1529 = Metabolite('Cpd1529', formula='', name='17alpha-hydroxyprogesterone [c]', compartment='')
Cpd1530 = Metabolite('Cpd1530', formula='', name='17alpha-hydroxypregnenolone [r]', compartment='')
Cpd1531 = Metabolite('Cpd1531', formula='', name='17alpha-hydroxyprogesterone [r]', compartment='')
Cpd1532 = Metabolite('Cpd1532', formula='', name='20-hydroxycholesterol [r]', compartment='')
Cpd1533 = Metabolite('Cpd1533', formula='', name='7alpha-hydroxycholest-4-en-3-one [r]', compartment='')
Cpd1534 = Metabolite('Cpd1534', formula='', name='Hypoxanthine [l]', compartment='')
Cpd1535 = Metabolite('Cpd1535', formula='', name='Hypoxanthine [c]', compartment='')
Cpd1536 = Metabolite('Cpd1536', formula='', name='Hypoxanthine [e]', compartment='')
Cpd1537 = Metabolite('Cpd1537', formula='', name='Isocitrate [x]', compartment='')
Cpd1538 = Metabolite('Cpd1538', formula='', name='2-Oxoglutarate [x]', compartment='')
Cpd1539 = Metabolite('Cpd1539', formula='', name='3-iodo-L-tyrosine [c]', compartment='')
Cpd1540 = Metabolite('Cpd1540', formula='', name='Iodine [c]', compartment='')
Cpd1541 = Metabolite('Cpd1541', formula='', name='3,5-diiodo-L-tyrosine [c]', compartment='')
Cpd1542 = Metabolite('Cpd1542', formula='', name='3,3,5-triiodo-L-thyronine [c]', compartment='')
Cpd1543 = Metabolite('Cpd1543', formula='', name='L-thyroxine [c]', compartment='')
Cpd1544 = Metabolite('Cpd1544', formula='', name='L-tyrosine [c]', compartment='')
Cpd1545 = Metabolite('Cpd1545', formula='', name='heparan sulfate, degradation product 5 [l]', compartment='')
Cpd1546 = Metabolite('Cpd1546', formula='', name='L-iduronate [l]', compartment='')
Cpd1547 = Metabolite('Cpd1547', formula='', name='heparan sulfate, degradation product 16 [l]', compartment='')
Cpd1548 = Metabolite('Cpd1548', formula='', name='heparan sulfate, degradation product 17 [l]', compartment='')
Cpd1549 = Metabolite('Cpd1549', formula='', name='IdoA-beta-1,4-GlcNAc(3S)-alpha-1,4-GlcA-beta-1,3-Gal-beta-1,3-Gal-beta-1,4-Xylose [l]', compartment='')
Cpd1550 = Metabolite('Cpd1550', formula='', name='GlcNAc(3S)-alpha-1,4-GlcA-beta-1,3-Gal-beta-1,3-Gal-beta-1,4-Xylose [l]', compartment='')
Cpd1551 = Metabolite('Cpd1551', formula='', name='chondroitin sulfate B / dermatan sulfate (IdoA2S-GalNAc4S), degradation product 3 [l]', compartment='')
Cpd1552 = Metabolite('Cpd1552', formula='', name='L-iduronate [c]', compartment='')
Cpd1553 = Metabolite('Cpd1553', formula='', name='L-isoleucine [e]', compartment='')
Cpd1554 = Metabolite('Cpd1554', formula='', name='L-isoleucine [c]', compartment='')
Cpd1555 = Metabolite('Cpd1555', formula='', name='3-methyl-2-oxopentanoate [c]', compartment='')
Cpd1556 = Metabolite('Cpd1556', formula='', name='L-isoleucine [m]', compartment='')
Cpd1557 = Metabolite('Cpd1557', formula='', name='3-methyl-2-oxopentanoate [m]', compartment='')
Cpd1558 = Metabolite('Cpd1558', formula='', name='imidazol-4-ylacetate [c]', compartment='')
Cpd1559 = Metabolite('Cpd1559', formula='', name='imidazole-4-acetaldehyde [m]', compartment='')
Cpd1560 = Metabolite('Cpd1560', formula='', name='imidazol-4-ylacetate [m]', compartment='')
Cpd1561 = Metabolite('Cpd1561', formula='', name='Inosine [l]', compartment='')
Cpd1562 = Metabolite('Cpd1562', formula='', name='Inosine [m]', compartment='')
Cpd1563 = Metabolite('Cpd1563', formula='', name='myo-inositol [e]', compartment='')
Cpd1564 = Metabolite('Cpd1564', formula='', name='itaconate(2-) [m]', compartment='')
Cpd1565 = Metabolite('Cpd1565', formula='', name='itaconyl-CoA(5-) [m]', compartment='')
Cpd1566 = Metabolite('Cpd1566', formula='', name='4-Imidazolone-5-propanoate [c]', compartment='')
Cpd1567 = Metabolite('Cpd1567', formula='', name='D-xylulose [c]', compartment='')
Cpd1568 = Metabolite('Cpd1568', formula='', name='D-tagatose [c]', compartment='')
Cpd1569 = Metabolite('Cpd1569', formula='', name='potassium [g]', compartment='')
Cpd1570 = Metabolite('Cpd1570', formula='', name='anthranilate [c]', compartment='')
Cpd1571 = Metabolite('Cpd1571', formula='', name='4-(2-aminophenyl)-2,4-dioxobutanoate [c]', compartment='')
Cpd1572 = Metabolite('Cpd1572', formula='', name='Lactose [e]', compartment='')
Cpd1573 = Metabolite('Cpd1573', formula='', name='Lactose [l]', compartment='')
Cpd1574 = Metabolite('Cpd1574', formula='', name='Coenzyme A [l]', compartment='')
Cpd1575 = Metabolite('Cpd1575', formula='', name='Dephospho-CoA [l]', compartment='')
Cpd1576 = Metabolite('Cpd1576', formula='', name='(S)-lactate [c]', compartment='')
Cpd1577 = Metabolite('Cpd1577', formula='', name='(R)-lactaldehyde [m]', compartment='')
Cpd1578 = Metabolite('Cpd1578', formula='', name='(S)-lactaldehyde [m]', compartment='')
Cpd1579 = Metabolite('Cpd1579', formula='', name='(S)-lactate [m]', compartment='')
Cpd1580 = Metabolite('Cpd1580', formula='', name='Phosphatidylcholine [e]', compartment='')
Cpd1581 = Metabolite('Cpd1581', formula='', name='lysophosphatidylcholine [e]', compartment='')
Cpd1582 = Metabolite('Cpd1582', formula='', name='cholesterol ester [e]', compartment='')
Cpd1583 = Metabolite('Cpd1583', formula='', name='3-sulfonatopyruvate(2-) [c]', compartment='')
Cpd1584 = Metabolite('Cpd1584', formula='', name='L-cysteate(1-) [m]', compartment='')
Cpd1585 = Metabolite('Cpd1585', formula='', name='3-sulfonatopyruvate(2-) [m]', compartment='')
Cpd1586 = Metabolite('Cpd1586', formula='', name='Taurine [c]', compartment='')
Cpd1587 = Metabolite('Cpd1587', formula='', name='leukotriene C4(2-) [e]', compartment='')
Cpd1588 = Metabolite('Cpd1588', formula='', name='leukotriene C4(2-) [c]', compartment='')
Cpd1589 = Metabolite('Cpd1589', formula='', name='L-leucine [e]', compartment='')
Cpd1590 = Metabolite('Cpd1590', formula='', name='L-leucine [c]', compartment='')
Cpd1591 = Metabolite('Cpd1591', formula='', name='4-methyl-2-oxopentanoate [c]', compartment='')
Cpd1592 = Metabolite('Cpd1592', formula='', name='L-leucine [m]', compartment='')
Cpd1593 = Metabolite('Cpd1593', formula='', name='4-methyl-2-oxopentanoate [m]', compartment='')
Cpd1594 = Metabolite('Cpd1594', formula='', name='N-formylanthranilate [c]', compartment='')
Cpd1595 = Metabolite('Cpd1595', formula='', name='lignoceryl carnitine [c]', compartment='')
Cpd1596 = Metabolite('Cpd1596', formula='', name='lignoceryl carnitine [m]', compartment='')
Cpd1597 = Metabolite('Cpd1597', formula='', name='lignocericyl coenzyme A [m]', compartment='')
Cpd1598 = Metabolite('Cpd1598', formula='', name='lipoate [e]', compartment='')
Cpd1599 = Metabolite('Cpd1599', formula='', name='lipoate [c]', compartment='')
Cpd1600 = Metabolite('Cpd1600', formula='', name='(S)-lactate [e]', compartment='')
Cpd1601 = Metabolite('Cpd1601', formula='', name='Linoelaidyl carnitine [c]', compartment='')
Cpd1602 = Metabolite('Cpd1602', formula='', name='Linoelaidyl carnitine [m]', compartment='')
Cpd1603 = Metabolite('Cpd1603', formula='', name='Linoleyl carnitine [c]', compartment='')
Cpd1604 = Metabolite('Cpd1604', formula='', name='Linoleyl carnitine [m]', compartment='')
Cpd1605 = Metabolite('Cpd1605', formula='', name='linoleate [e]', compartment='')
Cpd1606 = Metabolite('Cpd1606', formula='', name='alpha-linolenyl carnitine [c]', compartment='')
Cpd1607 = Metabolite('Cpd1607', formula='', name='alpha-linolenyl carnitine [m]', compartment='')
Cpd1608 = Metabolite('Cpd1608', formula='', name='alpha-linolenate [e]', compartment='')
Cpd1609 = Metabolite('Cpd1609', formula='', name='gamma-linolenyl carnitine [c]', compartment='')
Cpd1610 = Metabolite('Cpd1610', formula='', name='gamma-linolenyl carnitine [m]', compartment='')
Cpd1611 = Metabolite('Cpd1611', formula='', name='gamma-linolenate [e]', compartment='')
Cpd1612 = Metabolite('Cpd1612', formula='', name='lanosterol [c]', compartment='')
Cpd1613 = Metabolite('Cpd1613', formula='', name='4,4-dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol [c]', compartment='')
Cpd1614 = Metabolite('Cpd1614', formula='', name='(S)-2,3-epoxysqualene [r]', compartment='')
Cpd1615 = Metabolite('Cpd1615', formula='', name='lanosterol [r]', compartment='')
Cpd1616 = Metabolite('Cpd1616', formula='', name='lysophosphatidylcholine [c]', compartment='')
Cpd1617 = Metabolite('Cpd1617', formula='', name='R total [c]', compartment='')
Cpd1618 = Metabolite('Cpd1618', formula='', name='sn-Glycero-3-phosphocholine [c]', compartment='')
Cpd1619 = Metabolite('Cpd1619', formula='', name='L-pipecolic acid [x]', compartment='')
Cpd1620 = Metabolite('Cpd1620', formula='', name='1-piperideine-6-carboxylate [x]', compartment='')
Cpd1621 = Metabolite('Cpd1621', formula='', name='R total 3 position [c]', compartment='')
Cpd1622 = Metabolite('Cpd1622', formula='', name='monoacylglycerol 2 [c]', compartment='')
Cpd1623 = Metabolite('Cpd1623', formula='', name='diglyceride [e]', compartment='')
Cpd1624 = Metabolite('Cpd1624', formula='', name='monoacylglycerol 2 [e]', compartment='')
Cpd1625 = Metabolite('Cpd1625', formula='', name='R total [e]', compartment='')
Cpd1626 = Metabolite('Cpd1626', formula='', name='R total 2 position [c]', compartment='')
Cpd1627 = Metabolite('Cpd1627', formula='', name='Glycerol [e]', compartment='')
Cpd1628 = Metabolite('Cpd1628', formula='', name='R total 2 position [e]', compartment='')
Cpd1629 = Metabolite('Cpd1629', formula='', name='phosphatidylglycerol(1-) [e]', compartment='')
Cpd1630 = Metabolite('Cpd1630', formula='', name='1 acyl phosphoglycerol [e]', compartment='')
Cpd1631 = Metabolite('Cpd1631', formula='', name='triglyceride [e]', compartment='')
Cpd1632 = Metabolite('Cpd1632', formula='', name='R total 3 position [e]', compartment='')
Cpd1633 = Metabolite('Cpd1633', formula='', name='retinol [c]', compartment='')
Cpd1634 = Metabolite('Cpd1634', formula='', name='fatty acid retinol [c]', compartment='')
Cpd1635 = Metabolite('Cpd1635', formula='', name='leukotriene B4(1-) [c]', compartment='')
Cpd1636 = Metabolite('Cpd1636', formula='', name='Reduced glutathione [r]', compartment='')
Cpd1637 = Metabolite('Cpd1637', formula='', name='leukotriene A4(1-) [r]', compartment='')
Cpd1638 = Metabolite('Cpd1638', formula='', name='L-tryptophan [c]', compartment='')
Cpd1639 = Metabolite('Cpd1639', formula='', name='tryptaminium [c]', compartment='')
Cpd1640 = Metabolite('Cpd1640', formula='', name='Peptidyl-L-lysine [n]', compartment='')
Cpd1641 = Metabolite('Cpd1641', formula='', name='Histone N6-methyl-L-lysine [n]', compartment='')
Cpd1642 = Metabolite('Cpd1642', formula='', name='Protein N6,N6-dimethyl-L-lysine [n]', compartment='')
Cpd1643 = Metabolite('Cpd1643', formula='', name='Protein N6,N6,N6-trimethyl-L-lysine [n]', compartment='')
Cpd1644 = Metabolite('Cpd1644', formula='', name='(N-acetyl-D-glucosaminyl-(alpha-D-mannosyl)4-beta-D-mannosyl-diacetylchitobiosyl)-L-asparagine (protein) [g]', compartment='')
Cpd1645 = Metabolite('Cpd1645', formula='', name='D-Mannose [g]', compartment='')
Cpd1646 = Metabolite('Cpd1646', formula='', name='(N-acetyl-D-glucosaminyl-(alpha-D-mannosyl)2-beta-D-mannosyl-diacetylchitobiosyl)-L-asparagine (protein) [g]', compartment='')
Cpd1647 = Metabolite('Cpd1647', formula='', name='(alpha-D-mannosyl)4-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [g]', compartment='')
Cpd1648 = Metabolite('Cpd1648', formula='', name='((N-acetyl-D-glucosaminyl)3-(alpha-D-mannosyl)2-beta-D-mannosyl-diacetylchitobiosyl)-L-asparagine (protein) [g]', compartment='')
Cpd1649 = Metabolite('Cpd1649', formula='', name='N-Acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,3-(N-acetyl-beta-D-glucosaminyl-1,2-alpha-D-mannosyl-1,6)-(N-acetyl-beta-D-glucosaminyl-1,4)-beta-D-mannosyl-1,4-N-acetyl-beta-D-glucosaminyl-R [g]', compartment='')
Cpd1650 = Metabolite('Cpd1650', formula='', name='((N-acetyl-D-glucosaminyl)4-(alpha-D-mannosyl)2-beta-D-mannosyl-diacetylchitobiosyl)-L-asparagine (protein) [g]', compartment='')
Cpd1651 = Metabolite('Cpd1651', formula='', name='mannosyl-3-(phosphoethanolaminyl-mannosyl)-glucosaminyl-acylphosphatidylinositol (M4A)) [r]', compartment='')
Cpd1652 = Metabolite('Cpd1652', formula='', name='mannosyl-3-(phosphoethanolaminyl-mannosyl)-glucosaminyl-acylphosphatidylinositol-Protein (M4A) [r]', compartment='')
Cpd1653 = Metabolite('Cpd1653', formula='', name='({[(mannosyl),(phosphoethanolaminyl)]-dimannosyl},{phosphoethanolaminyl})-mannosyl-glucosaminyl-acylphosphatidylinositol-Protein (M4B) [r]', compartment='')
Cpd1654 = Metabolite('Cpd1654', formula='', name='1-methylimidazole-4-acetaldehyde [c]', compartment='')
Cpd1655 = Metabolite('Cpd1655', formula='', name='3-Methylimidazoleacetic acid [c]', compartment='')
Cpd1656 = Metabolite('Cpd1656', formula='', name='Maltose [c]', compartment='')
Cpd1657 = Metabolite('Cpd1657', formula='', name='Maltose [l]', compartment='')
Cpd1658 = Metabolite('Cpd1658', formula='', name='glucosyl-(alpha-D-mannosyl)7-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine, isoform C (protein) [r]', compartment='')
Cpd1659 = Metabolite('Cpd1659', formula='', name='glucosyl-(alpha-D-mannosyl)6-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [r]', compartment='')
Cpd1660 = Metabolite('Cpd1660', formula='', name='D-Mannose [r]', compartment='')
Cpd1661 = Metabolite('Cpd1661', formula='', name='(alpha-D-Glucosyl)-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [r]', compartment='')
Cpd1662 = Metabolite('Cpd1662', formula='', name='glucosyl-(alpha-D-mannosyl)7-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine, isoform B (protein) [r]', compartment='')
Cpd1663 = Metabolite('Cpd1663', formula='', name='D-Mannose [e]', compartment='')
Cpd1664 = Metabolite('Cpd1664', formula='', name='L-Normetanephrine [c]', compartment='')
Cpd1665 = Metabolite('Cpd1665', formula='', name='trans-3-methylglutaconyl-CoA(5-) [m]', compartment='')
Cpd1666 = Metabolite('Cpd1666', formula='', name='Malonyl-CoA [x]', compartment='')
Cpd1667 = Metabolite('Cpd1667', formula='', name='3-Mercaptolactate [c]', compartment='')
Cpd1668 = Metabolite('Cpd1668', formula='', name='Malonyl-[acyl-carrier protein] [c]', compartment='')
Cpd1669 = Metabolite('Cpd1669', formula='', name='acyl carrier protein [m]', compartment='')
Cpd1670 = Metabolite('Cpd1670', formula='', name='Malonyl-[acyl-carrier protein] [m]', compartment='')
Cpd1671 = Metabolite('Cpd1671', formula='', name='hydrogen cyanide [c]', compartment='')
Cpd1672 = Metabolite('Cpd1672', formula='', name='thiocyanate [c]', compartment='')
Cpd1673 = Metabolite('Cpd1673', formula='', name='Mesaconate [m]', compartment='')
Cpd1674 = Metabolite('Cpd1674', formula='', name='Mesaconyl-CoA [m]', compartment='')
Cpd1675 = Metabolite('Cpd1675', formula='', name='Formyl-N-acetyl-5-methoxykynurenamine [c]', compartment='')
Cpd1676 = Metabolite('Cpd1676', formula='', name='6-Hydroxymelatonin [c]', compartment='')
Cpd1677 = Metabolite('Cpd1677', formula='', name='L-methionine [e]', compartment='')
Cpd1678 = Metabolite('Cpd1678', formula='', name='(R)-mevalonate [x]', compartment='')
Cpd1679 = Metabolite('Cpd1679', formula='', name='(R)-5-phosphonatomevalonate(3-) [x]', compartment='')
Cpd1680 = Metabolite('Cpd1680', formula='', name='(alpha-D-Glucosyl)2-(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [r]', compartment='')
Cpd1681 = Metabolite('Cpd1681', formula='', name='N(tele)-methylhistaminium [c]', compartment='')
Cpd1682 = Metabolite('Cpd1682', formula='', name='1D-myo-inositol 1,3,4,5-tetrakisphosphate(8-) [n]', compartment='')
Cpd1683 = Metabolite('Cpd1683', formula='', name='myo-inositol 1,3,4,5,6-pentakisphosphate(10-) [n]', compartment='')
Cpd1684 = Metabolite('Cpd1684', formula='', name='1D-myo-inositol 1,3,4,5-tetrakisphosphate(8-) [c]', compartment='')
Cpd1685 = Metabolite('Cpd1685', formula='', name='1D-myo-inositol 1,3,4-trisphosphate(6-) [c]', compartment='')
Cpd1686 = Metabolite('Cpd1686', formula='', name='myo-inositol 1,3,4,6-tetrakisphosphate(8-) [n]', compartment='')
Cpd1687 = Metabolite('Cpd1687', formula='', name='1D-myo-Inositol 3,4-bisphosphate [c]', compartment='')
Cpd1688 = Metabolite('Cpd1688', formula='', name='1D-myo-inositol 1,4,5,6-tetrakisphosphate(8-) [n]', compartment='')
Cpd1689 = Metabolite('Cpd1689', formula='', name='1D-myo-inositol 1,4,5-trisphosphate(6-) [n]', compartment='')
Cpd1690 = Metabolite('Cpd1690', formula='', name='1D-myo-inositol 1,4,5-trisphosphate(6-) [c]', compartment='')
Cpd1691 = Metabolite('Cpd1691', formula='', name='(1R,2R,3R,4R,5R,6S)-2,3,5,6-tetrahydroxy-4-(phosphonatooxy)cyclohexyl phosphate [c]', compartment='')
Cpd1692 = Metabolite('Cpd1692', formula='', name='1D-myo-inositol 4-phosphate(2-) [c]', compartment='')
Cpd1693 = Metabolite('Cpd1693', formula='', name='1D-myo-inositol 1-phosphate(2-) [c]', compartment='')
Cpd1694 = Metabolite('Cpd1694', formula='', name='1D-myo-inositol 3-phosphate(2-) [c]', compartment='')
Cpd1695 = Metabolite('Cpd1695', formula='', name='maltotriose [c]', compartment='')
Cpd1696 = Metabolite('Cpd1696', formula='', name='maltotriose [e]', compartment='')
Cpd1697 = Metabolite('Cpd1697', formula='', name='maltotriose [l]', compartment='')
Cpd1698 = Metabolite('Cpd1698', formula='', name='(alpha-D-mannosyl)5-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [g]', compartment='')
Cpd1699 = Metabolite('Cpd1699', formula='', name='(alpha-D-mannosyl)6-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [g]', compartment='')
Cpd1700 = Metabolite('Cpd1700', formula='', name='(alpha-D-mannosyl)7-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine, isoform A (protein) [g]', compartment='')
Cpd1701 = Metabolite('Cpd1701', formula='', name='(alpha-D-mannosyl)7-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine, isoform B (protein) [g]', compartment='')
Cpd1702 = Metabolite('Cpd1702', formula='', name='(alpha-D-mannosyl)7-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine, isoform C (protein) [g]', compartment='')
Cpd1703 = Metabolite('Cpd1703', formula='', name='(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [g]', compartment='')
Cpd1704 = Metabolite('Cpd1704', formula='', name='(alpha-D-mannosyl)8-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine (protein) [r]', compartment='')
Cpd1705 = Metabolite('Cpd1705', formula='', name='(alpha-D-mannosyl)7-beta-D-mannosyl-diacetylchitobiosyl-L-asparagine, isoform B (protein) [r]', compartment='')
Cpd1706 = Metabolite('Cpd1706', formula='', name='(S)-methylmalonyl-CoA(5-) [c]', compartment='')
Cpd1707 = Metabolite('Cpd1707', formula='', name='(S)-methylmalonyl-CoA(5-) [m]', compartment='')
Cpd1708 = Metabolite('Cpd1708', formula='', name='(S)-methylmalonyl-CoA(5-) [x]', compartment='')
Cpd1709 = Metabolite('Cpd1709', formula='', name='(R)-methylmalonyl-CoA(5-) [m]', compartment='')
Cpd1710 = Metabolite('Cpd1710', formula='', name='5-Methylthioadenosine [c]', compartment='')
Cpd1711 = Metabolite('Cpd1711', formula='', name='5-Methylthio-5-deoxy-D-ribose 1-phosphate [c]', compartment='')
Cpd1712 = Metabolite('Cpd1712', formula='', name='5,10-Methenyltetrahydrofolate [m]', compartment='')
Cpd1713 = Metabolite('Cpd1713', formula='', name='4-acetamidobutanoate [c]', compartment='')
Cpd1714 = Metabolite('Cpd1714', formula='', name='4-acetamidobutanal [m]', compartment='')
Cpd1715 = Metabolite('Cpd1715', formula='', name='4-acetamidobutanoate [m]', compartment='')
Cpd1716 = Metabolite('Cpd1716', formula='', name='N-acetyl-L-aspartate(2-) [c]', compartment='')
Cpd1717 = Metabolite('Cpd1717', formula='', name='keratan sulfate I, degradation product 7 [l]', compartment='')
Cpd1718 = Metabolite('Cpd1718', formula='', name='keratan sulfate I, degradation product 10 [l]', compartment='')
Cpd1719 = Metabolite('Cpd1719', formula='', name='keratan sulfate I, degradation product 13 [l]', compartment='')
Cpd1720 = Metabolite('Cpd1720', formula='', name='keratan sulfate I, degradation product 16 [l]', compartment='')
Cpd1721 = Metabolite('Cpd1721', formula='', name='keratan sulfate I, degradation product 19 [l]', compartment='')
Cpd1722 = Metabolite('Cpd1722', formula='', name='keratan sulfate I, degradation product 22 [l]', compartment='')
Cpd1723 = Metabolite('Cpd1723', formula='', name='keratan sulfate I, degradation product 25 [l]', compartment='')
Cpd1724 = Metabolite('Cpd1724', formula='', name='keratan sulfate I, degradation product 28 [l]', compartment='')
Cpd1725 = Metabolite('Cpd1725', formula='', name='keratan sulfate I, degradation product 31 [l]', compartment='')
Cpd1726 = Metabolite('Cpd1726', formula='', name='keratan sulfate I, degradation product 34 [l]', compartment='')
Cpd1727 = Metabolite('Cpd1727', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA), degradation product 1 [l]', compartment='')
Cpd1728 = Metabolite('Cpd1728', formula='', name='N-acetyl-D-galactosamine [l]', compartment='')
Cpd1729 = Metabolite('Cpd1729', formula='', name='keratan sulfate I, degradation product 37 [l]', compartment='')
Cpd1730 = Metabolite('Cpd1730', formula='', name='keratan sulfate II (core 2-linked), degradation product 4 [l]', compartment='')
Cpd1731 = Metabolite('Cpd1731', formula='', name='keratan sulfate II (core 2-linked), degradation product 7 [l]', compartment='')
Cpd1732 = Metabolite('Cpd1732', formula='', name='keratan sulfate II (core 4-linked), degradation product 4 [l]', compartment='')
Cpd1733 = Metabolite('Cpd1733', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA), degradation product 4 [l]', compartment='')
Cpd1734 = Metabolite('Cpd1734', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA), degradation product 5 [l]', compartment='')
Cpd1735 = Metabolite('Cpd1735', formula='', name='chondroitin sulfate B / dermatan sulfate (IdoA2S-GalNAc4S), degradation product 1 [l]', compartment='')
Cpd1736 = Metabolite('Cpd1736', formula='', name='chondroitin sulfate B / dermatan sulfate (IdoA2S-GalNAc4S), degradation product 2 [l]', compartment='')
Cpd1737 = Metabolite('Cpd1737', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), degradation product 1 [l]', compartment='')
Cpd1738 = Metabolite('Cpd1738', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), degradation product 4 [l]', compartment='')
Cpd1739 = Metabolite('Cpd1739', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), degradation product 5 [l]', compartment='')
Cpd1740 = Metabolite('Cpd1740', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), degradation product 1 [l]', compartment='')
Cpd1741 = Metabolite('Cpd1741', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), degradation product 2 [l]', compartment='')
Cpd1742 = Metabolite('Cpd1742', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), degradation product 5 [l]', compartment='')
Cpd1743 = Metabolite('Cpd1743', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), degradation product 6 [l]', compartment='')
Cpd1744 = Metabolite('Cpd1744', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), degradation product 2 [l]', compartment='')
Cpd1745 = Metabolite('Cpd1745', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), degradation product 6 [l]', compartment='')
Cpd1746 = Metabolite('Cpd1746', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), degradation product 7 [l]', compartment='')
Cpd1747 = Metabolite('Cpd1747', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA), free chain [l]', compartment='')
Cpd1748 = Metabolite('Cpd1748', formula='', name='chondroitin sulfate B / dermatan sulfate (IdoA2S-GalNAc4S), free chain [l]', compartment='')
Cpd1749 = Metabolite('Cpd1749', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA), free chain [l]', compartment='')
Cpd1750 = Metabolite('Cpd1750', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), free chain [l]', compartment='')
Cpd1751 = Metabolite('Cpd1751', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), free chain [l]', compartment='')
Cpd1752 = Metabolite('Cpd1752', formula='', name='Nicotinamide adenine dinucleotide [e]', compartment='')
Cpd1753 = Metabolite('Cpd1753', formula='', name='Nicotinamide [e]', compartment='')
Cpd1754 = Metabolite('Cpd1754', formula='', name='Nicotinamide adenine dinucleotide phosphate [e]', compartment='')
Cpd1755 = Metabolite('Cpd1755', formula='', name='ADP-D-ribose 2-phosphate(4-) [e]', compartment='')
Cpd1756 = Metabolite('Cpd1756', formula='', name='Deamino-NAD+ [c]', compartment='')
Cpd1757 = Metabolite('Cpd1757', formula='', name='Tn antigen [l]', compartment='')
Cpd1758 = Metabolite('Cpd1758', formula='', name='UDP [l]', compartment='')
Cpd1759 = Metabolite('Cpd1759', formula='', name='protein-linked serine or threonine residue (O-glycosylation site) [l]', compartment='')
Cpd1760 = Metabolite('Cpd1760', formula='', name='UDP-N-acetyl-D-galactosamine(2-) [l]', compartment='')
Cpd1761 = Metabolite('Cpd1761', formula='', name='alpha GalNAc globoside [l]', compartment='')
Cpd1762 = Metabolite('Cpd1762', formula='', name='globoside [l]', compartment='')
Cpd1763 = Metabolite('Cpd1763', formula='', name='Sodium [g]', compartment='')
Cpd1764 = Metabolite('Cpd1764', formula='', name='carnosine [c]', compartment='')
Cpd1765 = Metabolite('Cpd1765', formula='', name='IDP(3-) [e]', compartment='')
Cpd1766 = Metabolite('Cpd1766', formula='', name='IMP [e]', compartment='')
Cpd1767 = Metabolite('Cpd1767', formula='', name='GDP [e]', compartment='')
Cpd1768 = Metabolite('Cpd1768', formula='', name='GMP [e]', compartment='')
Cpd1769 = Metabolite('Cpd1769', formula='', name='UDP [r]', compartment='')
Cpd1770 = Metabolite('Cpd1770', formula='', name='UMP [r]', compartment='')
Cpd1771 = Metabolite('Cpd1771', formula='', name='UDP [e]', compartment='')
Cpd1772 = Metabolite('Cpd1772', formula='', name='UMP [e]', compartment='')
Cpd1773 = Metabolite('Cpd1773', formula='', name='hydrogenphosphate [g]', compartment='')
Cpd1774 = Metabolite('Cpd1774', formula='', name='UMP [g]', compartment='')
Cpd1775 = Metabolite('Cpd1775', formula='', name='UTP [e]', compartment='')
Cpd1776 = Metabolite('Cpd1776', formula='', name='dIDP [c]', compartment='')
Cpd1777 = Metabolite('Cpd1777', formula='', name='dITP(4-) [c]', compartment='')
Cpd1778 = Metabolite('Cpd1778', formula='', name='dIDP [m]', compartment='')
Cpd1779 = Metabolite('Cpd1779', formula='', name='dITP(4-) [m]', compartment='')
Cpd1780 = Metabolite('Cpd1780', formula='', name='dIDP [n]', compartment='')
Cpd1781 = Metabolite('Cpd1781', formula='', name='dITP(4-) [n]', compartment='')
Cpd1782 = Metabolite('Cpd1782', formula='', name='UDP [m]', compartment='')
Cpd1783 = Metabolite('Cpd1783', formula='', name='UTP [m]', compartment='')
Cpd1784 = Metabolite('Cpd1784', formula='', name='dTDP [n]', compartment='')
Cpd1785 = Metabolite('Cpd1785', formula='', name='dTTP [n]', compartment='')
Cpd1786 = Metabolite('Cpd1786', formula='', name='dUDP [n]', compartment='')
Cpd1787 = Metabolite('Cpd1787', formula='', name='IDP(3-) [c]', compartment='')
Cpd1788 = Metabolite('Cpd1788', formula='', name='ITP(3-) [c]', compartment='')
Cpd1789 = Metabolite('Cpd1789', formula='', name='IDP(3-) [m]', compartment='')
Cpd1790 = Metabolite('Cpd1790', formula='', name='ITP(3-) [m]', compartment='')
Cpd1791 = Metabolite('Cpd1791', formula='', name='IDP(3-) [n]', compartment='')
Cpd1792 = Metabolite('Cpd1792', formula='', name='ITP(3-) [n]', compartment='')
Cpd1793 = Metabolite('Cpd1793', formula='', name='NMN [m]', compartment='')
Cpd1794 = Metabolite('Cpd1794', formula='', name='NMN [n]', compartment='')
Cpd1795 = Metabolite('Cpd1795', formula='', name='Nicotinamide adenine dinucleotide [n]', compartment='')
Cpd1796 = Metabolite('Cpd1796', formula='', name='NMN [c]', compartment='')
Cpd1797 = Metabolite('Cpd1797', formula='', name='Nicotinamide [c]', compartment='')
Cpd1798 = Metabolite('Cpd1798', formula='', name='N-methylputrescinium(2+) [c]', compartment='')
Cpd1799 = Metabolite('Cpd1799', formula='', name='1-methylpyrrolinium [c]', compartment='')
Cpd1800 = Metabolite('Cpd1800', formula='', name='Nicotinate D-ribonucleotide [m]', compartment='')
Cpd1801 = Metabolite('Cpd1801', formula='', name='Deamino-NAD+ [m]', compartment='')
Cpd1802 = Metabolite('Cpd1802', formula='', name='Nicotinate D-ribonucleotide [n]', compartment='')
Cpd1803 = Metabolite('Cpd1803', formula='', name='Deamino-NAD+ [n]', compartment='')
Cpd1804 = Metabolite('Cpd1804', formula='', name='Nicotinate D-ribonucleotide [c]', compartment='')
Cpd1805 = Metabolite('Cpd1805', formula='', name='1-methylnicotinamide [c]', compartment='')
Cpd1806 = Metabolite('Cpd1806', formula='', name='N-(omega)-Hydroxyarginine [c]', compartment='')
Cpd1807 = Metabolite('Cpd1807', formula='', name='Nitric oxide [c]', compartment='')
Cpd1808 = Metabolite('Cpd1808', formula='', name='Nicotinate [c]', compartment='')
Cpd1809 = Metabolite('Cpd1809', formula='', name='alpha-D-Ribose 1-phosphate [c]', compartment='')
Cpd1810 = Metabolite('Cpd1810', formula='', name='Nicotinate D-ribonucleoside [c]', compartment='')
Cpd1811 = Metabolite('Cpd1811', formula='', name='Sulfate derivative of norepinephrine [c]', compartment='')
Cpd1812 = Metabolite('Cpd1812', formula='', name='Norepinephrine [e]', compartment='')
Cpd1813 = Metabolite('Cpd1813', formula='', name='Nervonyl carnitine [c]', compartment='')
Cpd1814 = Metabolite('Cpd1814', formula='', name='Nervonyl carnitine [m]', compartment='')
Cpd1815 = Metabolite('Cpd1815', formula='', name='nervonyl coenzyme A [m]', compartment='')
Cpd1816 = Metabolite('Cpd1816', formula='', name='sialyl-T antigen [g]', compartment='')
Cpd1817 = Metabolite('Cpd1817', formula='', name='disialyl-T antigen [g]', compartment='')
Cpd1818 = Metabolite('Cpd1818', formula='', name='sialyl-Tn antigen [g]', compartment='')
Cpd1819 = Metabolite('Cpd1819', formula='', name='xanthosine [c]', compartment='')
Cpd1820 = Metabolite('Cpd1820', formula='', name='2-deoxyinosine 5-phosphate(2-) [c]', compartment='')
Cpd1821 = Metabolite('Cpd1821', formula='', name='Uridine [e]', compartment='')
Cpd1822 = Metabolite('Cpd1822', formula='', name='UMP [l]', compartment='')
Cpd1823 = Metabolite('Cpd1823', formula='', name='Uridine [l]', compartment='')
Cpd1824 = Metabolite('Cpd1824', formula='', name='UMP [m]', compartment='')
Cpd1825 = Metabolite('Cpd1825', formula='', name='Uridine [m]', compartment='')
Cpd1826 = Metabolite('Cpd1826', formula='', name='dCMP [l]', compartment='')
Cpd1827 = Metabolite('Cpd1827', formula='', name='Deoxycytidine [l]', compartment='')
Cpd1828 = Metabolite('Cpd1828', formula='', name='CMP [e]', compartment='')
Cpd1829 = Metabolite('Cpd1829', formula='', name='CMP [l]', compartment='')
Cpd1830 = Metabolite('Cpd1830', formula='', name='Thymidine [c]', compartment='')
Cpd1831 = Metabolite('Cpd1831', formula='', name='dTMP [l]', compartment='')
Cpd1832 = Metabolite('Cpd1832', formula='', name='Thymidine [l]', compartment='')
Cpd1833 = Metabolite('Cpd1833', formula='', name='dTMP [m]', compartment='')
Cpd1834 = Metabolite('Cpd1834', formula='', name='Thymidine [m]', compartment='')
Cpd1835 = Metabolite('Cpd1835', formula='', name='dAMP(2-) [l]', compartment='')
Cpd1836 = Metabolite('Cpd1836', formula='', name='2-deoxyadenosine [l]', compartment='')
Cpd1837 = Metabolite('Cpd1837', formula='', name='AMP [l]', compartment='')
Cpd1838 = Metabolite('Cpd1838', formula='', name='dGMP [l]', compartment='')
Cpd1839 = Metabolite('Cpd1839', formula='', name='Deoxyguanosine [l]', compartment='')
Cpd1840 = Metabolite('Cpd1840', formula='', name='GMP [l]', compartment='')
Cpd1841 = Metabolite('Cpd1841', formula='', name='GTP [e]', compartment='')
Cpd1842 = Metabolite('Cpd1842', formula='', name='XTP [c]', compartment='')
Cpd1843 = Metabolite('Cpd1843', formula='', name='Succinate [m]', compartment='')
Cpd1844 = Metabolite('Cpd1844', formula='', name='3-methyl-2-oxobutanoate [m]', compartment='')
Cpd1845 = Metabolite('Cpd1845', formula='', name='Orotidine 5-phosphate [c]', compartment='')
Cpd1846 = Metabolite('Cpd1846', formula='', name='Putrescine [c]', compartment='')
Cpd1847 = Metabolite('Cpd1847', formula='', name='Ornithine [e]', compartment='')
Cpd1848 = Metabolite('Cpd1848', formula='', name='4-methylpentanal [m]', compartment='')
Cpd1849 = Metabolite('Cpd1849', formula='', name='pregnenolone [m]', compartment='')
Cpd1850 = Metabolite('Cpd1850', formula='', name='11-deoxycorticosterone [m]', compartment='')
Cpd1851 = Metabolite('Cpd1851', formula='', name='corticosterone [m]', compartment='')
Cpd1852 = Metabolite('Cpd1852', formula='', name='11-deoxycortisol [m]', compartment='')
Cpd1853 = Metabolite('Cpd1853', formula='', name='cortisol [m]', compartment='')
Cpd1854 = Metabolite('Cpd1854', formula='', name='aldosterone [m]', compartment='')
Cpd1855 = Metabolite('Cpd1855', formula='', name='acetaldehyde [r]', compartment='')
Cpd1856 = Metabolite('Cpd1856', formula='', name='Formate [r]', compartment='')
Cpd1857 = Metabolite('Cpd1857', formula='', name='4,17 dihydroxy estradiol [r]', compartment='')
Cpd1858 = Metabolite('Cpd1858', formula='', name='11-deoxycorticosterone [r]', compartment='')
Cpd1859 = Metabolite('Cpd1859', formula='', name='11-deoxycortisol [r]', compartment='')
Cpd1860 = Metabolite('Cpd1860', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha-triol [m]', compartment='')
Cpd1861 = Metabolite('Cpd1861', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,26-tetrol [m]', compartment='')
Cpd1862 = Metabolite('Cpd1862', formula='', name='3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-26-al [m]', compartment='')
Cpd1863 = Metabolite('Cpd1863', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate [m]', compartment='')
Cpd1864 = Metabolite('Cpd1864', formula='', name='5beta-cholestane-3alpha,7alpha-diol [m]', compartment='')
Cpd1865 = Metabolite('Cpd1865', formula='', name='5beta-cholestane-3alpha,7alpha,26-triol [m]', compartment='')
Cpd1866 = Metabolite('Cpd1866', formula='', name='3alpha,7alpha-dihydroxy-5beta-cholestan-26-al [m]', compartment='')
Cpd1867 = Metabolite('Cpd1867', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholestanate [m]', compartment='')
Cpd1868 = Metabolite('Cpd1868', formula='', name='26-hydroxycholesterol [m]', compartment='')
Cpd1869 = Metabolite('Cpd1869', formula='', name='coumarin [c]', compartment='')
Cpd1870 = Metabolite('Cpd1870', formula='', name='umbelliferone [c]', compartment='')
Cpd1871 = Metabolite('Cpd1871', formula='', name='antipyrine [c]', compartment='')
Cpd1872 = Metabolite('Cpd1872', formula='', name='demethylated antipyrine [c]', compartment='')
Cpd1873 = Metabolite('Cpd1873', formula='', name='4 hydroxy debrisoquine [c]', compartment='')
Cpd1874 = Metabolite('Cpd1874', formula='', name='2-hydroxy-4-nitrophenolate [c]', compartment='')
Cpd1875 = Metabolite('Cpd1875', formula='', name='naphthalene [c]', compartment='')
Cpd1876 = Metabolite('Cpd1876', formula='', name='naphthalene epoxide [c]', compartment='')
Cpd1877 = Metabolite('Cpd1877', formula='', name='(24S)-24-hydroxycholesterol [r]', compartment='')
Cpd1878 = Metabolite('Cpd1878', formula='', name='(24S)-7alpha,24-dihydroxycholesterol [r]', compartment='')
Cpd1879 = Metabolite('Cpd1879', formula='', name='nifedipine [c]', compartment='')
Cpd1880 = Metabolite('Cpd1880', formula='', name='hydroxy nifedipine [c]', compartment='')
Cpd1881 = Metabolite('Cpd1881', formula='', name='w hydroxy testosterone [r]', compartment='')
Cpd1882 = Metabolite('Cpd1882', formula='', name='aflatoxin B1 [c]', compartment='')
Cpd1883 = Metabolite('Cpd1883', formula='', name='aflatoxin B1 exo-8,9-epoxide [c]', compartment='')
Cpd1884 = Metabolite('Cpd1884', formula='', name='6 beta hydroxy testosterone [r]', compartment='')
Cpd1885 = Metabolite('Cpd1885', formula='', name='arachidonate [r]', compartment='')
Cpd1886 = Metabolite('Cpd1886', formula='', name='12 hydroxy arachidonic acid [r]', compartment='')
Cpd1887 = Metabolite('Cpd1887', formula='', name='w-hydroxyl arachidonic acid [r]', compartment='')
Cpd1888 = Metabolite('Cpd1888', formula='', name='leukotriene B4(1-) [r]', compartment='')
Cpd1889 = Metabolite('Cpd1889', formula='', name='w-hydroxyl leukotriene B4 [r]', compartment='')
Cpd1890 = Metabolite('Cpd1890', formula='', name='ebastine [r]', compartment='')
Cpd1891 = Metabolite('Cpd1891', formula='', name='hydroxylated ebastine [r]', compartment='')
Cpd1892 = Metabolite('Cpd1892', formula='', name='18 hydroxy arachidonic acid [r]', compartment='')
Cpd1893 = Metabolite('Cpd1893', formula='', name='7alpha,25-dihydroxycholesterol [r]', compartment='')
Cpd1894 = Metabolite('Cpd1894', formula='', name='26-hydroxycholesterol [r]', compartment='')
Cpd1895 = Metabolite('Cpd1895', formula='', name='7alpha,26-dihydroxycholesterol [r]', compartment='')
Cpd1896 = Metabolite('Cpd1896', formula='', name='7alpha,12alpha-Dihydroxycholest-4-en-3-one [r]', compartment='')
Cpd1897 = Metabolite('Cpd1897', formula='', name='1-Pyrroline-5-carboxylate [m]', compartment='')
Cpd1898 = Metabolite('Cpd1898', formula='', name='L-proline [m]', compartment='')
Cpd1899 = Metabolite('Cpd1899', formula='', name='1-alkyl 2-acteylglycerol 3-phosphocholine [c]', compartment='')
Cpd1900 = Metabolite('Cpd1900', formula='', name='1-alkyl 2-lysoglycerol 3-phosphocholine [c]', compartment='')
Cpd1901 = Metabolite('Cpd1901', formula='', name='1-alkyl 2-acteylglycerol 3-phosphocholine [e]', compartment='')
Cpd1902 = Metabolite('Cpd1902', formula='', name='1-alkyl 2-lysoglycerol 3-phosphocholine [e]', compartment='')
Cpd1903 = Metabolite('Cpd1903', formula='', name='Phosphatidylcholine [m]', compartment='')
Cpd1904 = Metabolite('Cpd1904', formula='', name='Phosphatidylcholine [g]', compartment='')
Cpd1905 = Metabolite('Cpd1905', formula='', name='Choline [g]', compartment='')
Cpd1906 = Metabolite('Cpd1906', formula='', name='phosphatidate(2-) [g]', compartment='')
Cpd1907 = Metabolite('Cpd1907', formula='', name='Phosphatidylcholine [r]', compartment='')
Cpd1908 = Metabolite('Cpd1908', formula='', name='Choline [r]', compartment='')
Cpd1909 = Metabolite('Cpd1909', formula='', name='phosphatidate(2-) [r]', compartment='')
Cpd1910 = Metabolite('Cpd1910', formula='', name='2-aminomuconate 6-semialdehyde(1-) [c]', compartment='')
Cpd1911 = Metabolite('Cpd1911', formula='', name='Procollagen L-lysine [c]', compartment='')
Cpd1912 = Metabolite('Cpd1912', formula='', name='erythro-5-hydroxy-L-lysinium(1+) [c]', compartment='')
Cpd1913 = Metabolite('Cpd1913', formula='', name='propionyl-carnitine [c]', compartment='')
Cpd1914 = Metabolite('Cpd1914', formula='', name='3,5-cyclic AMP(1-) [g]', compartment='')
Cpd1915 = Metabolite('Cpd1915', formula='', name='AMP [g]', compartment='')
Cpd1916 = Metabolite('Cpd1916', formula='', name='3,5-cyclic GMP(1-) [g]', compartment='')
Cpd1917 = Metabolite('Cpd1917', formula='', name='3,5-cyclic GMP(1-) [n]', compartment='')
Cpd1918 = Metabolite('Cpd1918', formula='', name='GMP [n]', compartment='')
Cpd1919 = Metabolite('Cpd1919', formula='', name='Pyridoxine 5-phosphate [c]', compartment='')
Cpd1920 = Metabolite('Cpd1920', formula='', name='Pyridoxal 5-phosphate [c]', compartment='')
Cpd1921 = Metabolite('Cpd1921', formula='', name='2-phenylethanaminium [c]', compartment='')
Cpd1922 = Metabolite('Cpd1922', formula='', name='phenylacetaldehyde [c]', compartment='')
Cpd1923 = Metabolite('Cpd1923', formula='', name='phosphatidylethanolamine [e]', compartment='')
Cpd1924 = Metabolite('Cpd1924', formula='', name='phosphatidylethanolamine [m]', compartment='')
Cpd1925 = Metabolite('Cpd1925', formula='', name='pristanoyl coa [x]', compartment='')
Cpd1926 = Metabolite('Cpd1926', formula='', name='S-Adenosyl-L-methionine [r]', compartment='')
Cpd1927 = Metabolite('Cpd1927', formula='', name='S-Adenosyl-L-homocysteine [r]', compartment='')
Cpd1928 = Metabolite('Cpd1928', formula='', name='3-phosphonatooxypyruvate(3-) [c]', compartment='')
Cpd1929 = Metabolite('Cpd1929', formula='', name='prostaglandin H2(1-) [c]', compartment='')
Cpd1930 = Metabolite('Cpd1930', formula='', name='prostaglandin D2(1-) [c]', compartment='')
Cpd1931 = Metabolite('Cpd1931', formula='', name='prostaglandin H2(1-) [r]', compartment='')
Cpd1932 = Metabolite('Cpd1932', formula='', name='prostaglandin D2(1-) [r]', compartment='')
Cpd1933 = Metabolite('Cpd1933', formula='', name='prostaglandin E2(1-) [r]', compartment='')
Cpd1934 = Metabolite('Cpd1934', formula='', name='prostaglandin I2(1-) [r]', compartment='')
Cpd1935 = Metabolite('Cpd1935', formula='', name='2-phosphonatoglycolate(3-) [c]', compartment='')
Cpd1936 = Metabolite('Cpd1936', formula='', name='phosphatidyl glycerol phosphate [c]', compartment='')
Cpd1937 = Metabolite('Cpd1937', formula='', name='L-1-Pyrroline-3-hydroxy-5-carboxylate [m]', compartment='')
Cpd1938 = Metabolite('Cpd1938', formula='', name='Protoheme [e]', compartment='')
Cpd1939 = Metabolite('Cpd1939', formula='', name='L-phenylalanine [e]', compartment='')
Cpd1940 = Metabolite('Cpd1940', formula='', name='L-phenylalanine [c]', compartment='')
Cpd1941 = Metabolite('Cpd1941', formula='', name='keto-phenylpyruvate [c]', compartment='')
Cpd1942 = Metabolite('Cpd1942', formula='', name='L-phenylalanine [m]', compartment='')
Cpd1943 = Metabolite('Cpd1943', formula='', name='keto-phenylpyruvate [m]', compartment='')
Cpd1944 = Metabolite('Cpd1944', formula='', name='4a-hydroxytetrahydrobiopterin [c]', compartment='')
Cpd1945 = Metabolite('Cpd1945', formula='', name='phytanoyl-CoA(4-) [x]', compartment='')
Cpd1946 = Metabolite('Cpd1946', formula='', name='2-hydroxyphytanoyl-CoA(4-) [x]', compartment='')
Cpd1947 = Metabolite('Cpd1947', formula='', name='Succinate [x]', compartment='')
Cpd1948 = Metabolite('Cpd1948', formula='', name='1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate(7-) [c]', compartment='')
Cpd1949 = Metabolite('Cpd1949', formula='', name='phosphatidylinositol 4,5-bisphosphate [c]', compartment='')
Cpd1950 = Metabolite('Cpd1950', formula='', name='1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate(7-) [n]', compartment='')
Cpd1951 = Metabolite('Cpd1951', formula='', name='phosphatidylinositol 4,5-bisphosphate [n]', compartment='')
Cpd1952 = Metabolite('Cpd1952', formula='', name='hydrogenphosphate [n]', compartment='')
Cpd1953 = Metabolite('Cpd1953', formula='', name='1-phosphatidyl-1D-myo-inositol 3,4-bisphosphate(5-) [c]', compartment='')
Cpd1954 = Metabolite('Cpd1954', formula='', name='1-phosphatidyl-1D-myo-inositol 3-phosphate(3-) [c]', compartment='')
Cpd1955 = Metabolite('Cpd1955', formula='', name='1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate(5-) [c]', compartment='')
Cpd1956 = Metabolite('Cpd1956', formula='', name='1-phosphatidyl-1D-myo-inositol 4-phosphate(3-) [c]', compartment='')
Cpd1957 = Metabolite('Cpd1957', formula='', name='ATP [r]', compartment='')
Cpd1958 = Metabolite('Cpd1958', formula='', name='1-phosphatidyl-1D-myo-inositol 4-phosphate(3-) [r]', compartment='')
Cpd1959 = Metabolite('Cpd1959', formula='', name='ADP [r]', compartment='')
Cpd1960 = Metabolite('Cpd1960', formula='', name='1-phosphatidyl-1D-myo-inositol 3,4-bisphosphate(5-) [r]', compartment='')
Cpd1961 = Metabolite('Cpd1961', formula='', name='1-phosphatidyl-1D-myo-inositol 4-phosphate(3-) [n]', compartment='')
Cpd1962 = Metabolite('Cpd1962', formula='', name='(1R,2R,3R,4R,5R,6S)-2,3,5,6-tetrahydroxy-4-(phosphonatooxy)cyclohexyl phosphate [n]', compartment='')
Cpd1963 = Metabolite('Cpd1963', formula='', name='1-phosphatidyl-1D-myo-inositol 5-phosphate(3-) [c]', compartment='')
Cpd1964 = Metabolite('Cpd1964', formula='', name='1-phosphatidyl-1D-myo-inositol 5-phosphate(3-) [r]', compartment='')
Cpd1965 = Metabolite('Cpd1965', formula='', name='1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate(5-) [r]', compartment='')
Cpd1966 = Metabolite('Cpd1966', formula='', name='1-phosphatidyl-1D-myo-inositol(1-) [r]', compartment='')
Cpd1967 = Metabolite('Cpd1967', formula='', name='1-phosphatidyl-1D-myo-inositol 3-phosphate(3-) [r]', compartment='')
Cpd1968 = Metabolite('Cpd1968', formula='', name='1-phosphatidyl-1D-myo-inositol(1-) [n]', compartment='')
Cpd1969 = Metabolite('Cpd1969', formula='', name='1D-myo-inositol 1-phosphate(2-) [n]', compartment='')
Cpd1970 = Metabolite('Cpd1970', formula='', name='Protein N6,N6,N6-trimethyl-L-lysine [r]', compartment='')
Cpd1971 = Metabolite('Cpd1971', formula='', name='peptide sans lysine [r]', compartment='')
Cpd1972 = Metabolite('Cpd1972', formula='', name='N6,N6,N6-Trimethyl-L-lysine [r]', compartment='')
Cpd1973 = Metabolite('Cpd1973', formula='', name='5-Diphosphoinositol pentakisphosphate [c]', compartment='')
Cpd1974 = Metabolite('Cpd1974', formula='', name='myo-inositol hexakisphosphate(12-) [c]', compartment='')
Cpd1975 = Metabolite('Cpd1975', formula='', name='5-Diphosphoinositol pentakisphosphate [n]', compartment='')
Cpd1976 = Metabolite('Cpd1976', formula='', name='myo-inositol hexakisphosphate(12-) [n]', compartment='')
Cpd1977 = Metabolite('Cpd1977', formula='', name='Diphosphoinositol tetrakisphosphate [c]', compartment='')
Cpd1978 = Metabolite('Cpd1978', formula='', name='myo-inositol 1,3,4,5,6-pentakisphosphate(10-) [c]', compartment='')
Cpd1979 = Metabolite('Cpd1979', formula='', name='Diphosphoinositol tetrakisphosphate [n]', compartment='')
Cpd1980 = Metabolite('Cpd1980', formula='', name='N-Ribosylnicotinamide [c]', compartment='')
Cpd1981 = Metabolite('Cpd1981', formula='', name='pantetheine [c]', compartment='')
Cpd1982 = Metabolite('Cpd1982', formula='', name='cysteaminium [c]', compartment='')
Cpd1983 = Metabolite('Cpd1983', formula='', name='(R)-Pantothenate [c]', compartment='')
Cpd1984 = Metabolite('Cpd1984', formula='', name='D-4-Phosphopantothenate [c]', compartment='')
Cpd1985 = Metabolite('Cpd1985', formula='', name='(R)-Pantothenate [m]', compartment='')
Cpd1986 = Metabolite('Cpd1986', formula='', name='D-4-Phosphopantothenate [m]', compartment='')
Cpd1987 = Metabolite('Cpd1987', formula='', name='(R)-Pantothenate [e]', compartment='')
Cpd1988 = Metabolite('Cpd1988', formula='', name='Diphosphate [r]', compartment='')
Cpd1989 = Metabolite('Cpd1989', formula='', name='5-Amino-4-oxopentanoate [c]', compartment='')
Cpd1990 = Metabolite('Cpd1990', formula='', name='N-((R)-4-Phosphopantothenoyl)-L-cysteine [c]', compartment='')
Cpd1991 = Metabolite('Cpd1991', formula='', name='Pantetheine 4-phosphate [c]', compartment='')
Cpd1992 = Metabolite('Cpd1992', formula='', name='acryloyl-CoA(4-) [m]', compartment='')
Cpd1993 = Metabolite('Cpd1993', formula='', name='2-Hydroxyphenylacetate [c]', compartment='')
Cpd1994 = Metabolite('Cpd1994', formula='', name='Protoporphyrinogen IX [m]', compartment='')
Cpd1995 = Metabolite('Cpd1995', formula='', name='Hydrogen peroxide [l]', compartment='')
Cpd1996 = Metabolite('Cpd1996', formula='', name='methanol [l]', compartment='')
Cpd1997 = Metabolite('Cpd1997', formula='', name='formaldehyde [l]', compartment='')
Cpd1998 = Metabolite('Cpd1998', formula='', name='2-(Formamido)-N1-(5-phospho-D-ribosyl)acetamidine [c]', compartment='')
Cpd1999 = Metabolite('Cpd1999', formula='', name='pregnenolone sulfate [c]', compartment='')
Cpd2000 = Metabolite('Cpd2000', formula='', name='2-Oxoglutarate [r]', compartment='')
Cpd2001 = Metabolite('Cpd2001', formula='', name='L-proline [r]', compartment='')
Cpd2002 = Metabolite('Cpd2002', formula='', name='trans-4-hydroxy-L-proline [r]', compartment='')
Cpd2003 = Metabolite('Cpd2003', formula='', name='Succinate [r]', compartment='')
Cpd2004 = Metabolite('Cpd2004', formula='', name='L-proline [c]', compartment='')
Cpd2005 = Metabolite('Cpd2005', formula='', name='1-Pyrroline-5-carboxylate [c]', compartment='')
Cpd2006 = Metabolite('Cpd2006', formula='', name='D-proline [e]', compartment='')
Cpd2007 = Metabolite('Cpd2007', formula='', name='D-proline [c]', compartment='')
Cpd2008 = Metabolite('Cpd2008', formula='', name='D-proline [l]', compartment='')
Cpd2009 = Metabolite('Cpd2009', formula='', name='prostaglandin D2(1-) [e]', compartment='')
Cpd2010 = Metabolite('Cpd2010', formula='', name='prostaglandin E1(1-) [e]', compartment='')
Cpd2011 = Metabolite('Cpd2011', formula='', name='prostaglandin E2(1-) [e]', compartment='')
Cpd2012 = Metabolite('Cpd2012', formula='', name='prostaglandin F2alpha(1-) [e]', compartment='')
Cpd2013 = Metabolite('Cpd2013', formula='', name='L-proline [e]', compartment='')
Cpd2014 = Metabolite('Cpd2014', formula='', name='L-proline [l]', compartment='')
Cpd2015 = Metabolite('Cpd2015', formula='', name='3-hydroxypropanoyl-CoA(4-) [m]', compartment='')
Cpd2016 = Metabolite('Cpd2016', formula='', name='acryloyl-CoA(4-) [x]', compartment='')
Cpd2017 = Metabolite('Cpd2017', formula='', name='3-hydroxypropanoyl-CoA(4-) [x]', compartment='')
Cpd2018 = Metabolite('Cpd2018', formula='', name='O-Phospho-L-serine [c]', compartment='')
Cpd2019 = Metabolite('Cpd2019', formula='', name='phosphatidylserine [e]', compartment='')
Cpd2020 = Metabolite('Cpd2020', formula='', name='phosphatidylserine [c]', compartment='')
Cpd2021 = Metabolite('Cpd2021', formula='', name='phosphatidylserine [m]', compartment='')
Cpd2022 = Metabolite('Cpd2022', formula='', name='pendtadenoyl carnitine [c]', compartment='')
Cpd2023 = Metabolite('Cpd2023', formula='', name='pendtadenoyl carnitine [m]', compartment='')
Cpd2024 = Metabolite('Cpd2024', formula='', name='Hexadecanoate (n-C16:0) [x]', compartment='')
Cpd2025 = Metabolite('Cpd2025', formula='', name='pristanic acid [x]', compartment='')
Cpd2026 = Metabolite('Cpd2026', formula='', name='phytanate [x]', compartment='')
Cpd2027 = Metabolite('Cpd2027', formula='', name='adrenic acid [x]', compartment='')
Cpd2028 = Metabolite('Cpd2028', formula='', name='triphosphate(5-) [c]', compartment='')
Cpd2029 = Metabolite('Cpd2029', formula='', name='6-pyruvoyl-5,6,7,8-tetrahydropterin [c]', compartment='')
Cpd2030 = Metabolite('Cpd2030', formula='', name='triphosphate(5-) [n]', compartment='')
Cpd2031 = Metabolite('Cpd2031', formula='', name='6-pyruvoyl-5,6,7,8-tetrahydropterin [n]', compartment='')
Cpd2032 = Metabolite('Cpd2032', formula='', name='Pyridoxamine 5-phosphate [c]', compartment='')
Cpd2033 = Metabolite('Cpd2033', formula='', name='Pyridoxamine [c]', compartment='')
Cpd2034 = Metabolite('Cpd2034', formula='', name='Pyridoxal [c]', compartment='')
Cpd2035 = Metabolite('Cpd2035', formula='', name='4-pyridoxate [c]', compartment='')
Cpd2036 = Metabolite('Cpd2036', formula='', name='Pyridoxine [c]', compartment='')
Cpd2037 = Metabolite('Cpd2037', formula='', name='perillyl aldehyde [c]', compartment='')
Cpd2038 = Metabolite('Cpd2038', formula='', name='Perillic acid [c]', compartment='')
Cpd2039 = Metabolite('Cpd2039', formula='', name='perillyl aldehyde [m]', compartment='')
Cpd2040 = Metabolite('Cpd2040', formula='', name='Perillic acid [m]', compartment='')
Cpd2041 = Metabolite('Cpd2041', formula='', name='pyruvate [e]', compartment='')
Cpd2042 = Metabolite('Cpd2042', formula='', name='D-ribose [c]', compartment='')
Cpd2043 = Metabolite('Cpd2043', formula='', name='D-ribulose [c]', compartment='')
Cpd2044 = Metabolite('Cpd2044', formula='', name='9-cis-retinol [c]', compartment='')
Cpd2045 = Metabolite('Cpd2045', formula='', name='all-trans-retinal [c]', compartment='')
Cpd2046 = Metabolite('Cpd2046', formula='', name='11-cis-retinol [c]', compartment='')
Cpd2047 = Metabolite('Cpd2047', formula='', name='11-cis-retinal [c]', compartment='')
Cpd2048 = Metabolite('Cpd2048', formula='', name='13-cis-retinol [c]', compartment='')
Cpd2049 = Metabolite('Cpd2049', formula='', name='13-cis-retinal [c]', compartment='')
Cpd2050 = Metabolite('Cpd2050', formula='', name='retinol [e]', compartment='')
Cpd2051 = Metabolite('Cpd2051', formula='', name='Riboflavin [e]', compartment='')
Cpd2052 = Metabolite('Cpd2052', formula='', name='Reduced thioredoxin [c]', compartment='')
Cpd2053 = Metabolite('Cpd2053', formula='', name='Oxidized thioredoxin [c]', compartment='')
Cpd2054 = Metabolite('Cpd2054', formula='', name='D-xylulose 5-phosphate(2-) [c]', compartment='')
Cpd2055 = Metabolite('Cpd2055', formula='', name='R total 2 carnitine [c]', compartment='')
Cpd2056 = Metabolite('Cpd2056', formula='', name='R total 2 carnitine [m]', compartment='')
Cpd2057 = Metabolite('Cpd2057', formula='', name='R total 2 coenzyme A [m]', compartment='')
Cpd2058 = Metabolite('Cpd2058', formula='', name='R total 3 carnitine [c]', compartment='')
Cpd2059 = Metabolite('Cpd2059', formula='', name='R total 3 carnitine [m]', compartment='')
Cpd2060 = Metabolite('Cpd2060', formula='', name='R total 3 Coenzyme A [m]', compartment='')
Cpd2061 = Metabolite('Cpd2061', formula='', name='R total carnitine [c]', compartment='')
Cpd2062 = Metabolite('Cpd2062', formula='', name='R total carnitine [m]', compartment='')
Cpd2063 = Metabolite('Cpd2063', formula='', name='PA6 [g]', compartment='')
Cpd2064 = Metabolite('Cpd2064', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S), precursor 5 [g]', compartment='')
Cpd2065 = Metabolite('Cpd2065', formula='', name='chondroitin sulfate D (GlcNAc6S-GlcA2S) proteoglycan [g]', compartment='')
Cpd2066 = Metabolite('Cpd2066', formula='', name='chondroitin sulfate B (IdoA2S-GalNAc4s), precursor 5 [g]', compartment='')
Cpd2067 = Metabolite('Cpd2067', formula='', name='chondroitin sulfate B / dermatan sulfate (IdoA2S-GalNAc4S) proteoglycan [g]', compartment='')
Cpd2068 = Metabolite('Cpd2068', formula='', name='GlcN2S-IdoA-GlcN2S-GlcA2S-GlcN2S-IdoA2S-GlcN2S-GlcA-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd2069 = Metabolite('Cpd2069', formula='', name='GlcN2S-IdoA-GlcN2,6diS-GlcA2S-GlcN2,6diS-IdoA2S-GlcN2,6diS-GlcA-GlcNAc-GlcA-Gal-Gal-Xyl-L-Ser (protein) [g]', compartment='')
Cpd2070 = Metabolite('Cpd2070', formula='', name='heparan sulfate, precursor 14 [g]', compartment='')
Cpd2071 = Metabolite('Cpd2071', formula='', name='heparan sulfate, precursor 15 [g]', compartment='')
Cpd2072 = Metabolite('Cpd2072', formula='', name='heparan sulfate proteoglycan [g]', compartment='')
Cpd2073 = Metabolite('Cpd2073', formula='', name='chondroitin sulfate A (GalNAc4S-GlcA) proteoglycan [g]', compartment='')
Cpd2074 = Metabolite('Cpd2074', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), precursor 5a [g]', compartment='')
Cpd2075 = Metabolite('Cpd2075', formula='', name='chondroitin sulfate B (IdoA2S-GalNAc4S), precursor 4 [g]', compartment='')
Cpd2076 = Metabolite('Cpd2076', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), degradation product 1 [l]', compartment='')
Cpd2077 = Metabolite('Cpd2077', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), degradation product 5 [l]', compartment='')
Cpd2078 = Metabolite('Cpd2078', formula='', name='keratan sulfate I [g]', compartment='')
Cpd2079 = Metabolite('Cpd2079', formula='', name='keratan sulfate II (core 4-linked) [g]', compartment='')
Cpd2080 = Metabolite('Cpd2080', formula='', name='chondroitin sulfate C (GalNAc6S-GlcA) proteoglycan [g]', compartment='')
Cpd2081 = Metabolite('Cpd2081', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA) proteoglycan [g]', compartment='')
Cpd2082 = Metabolite('Cpd2082', formula='', name='chondroitin sulfate E (GalNAc4,6diS-GlcA), precursor 5b [g]', compartment='')
Cpd2083 = Metabolite('Cpd2083', formula='', name='keratan sulfate II (core 2-linked) [g]', compartment='')
Cpd2084 = Metabolite('Cpd2084', formula='', name='keratan sulfate I, degradation product 4 [l]', compartment='')
Cpd2085 = Metabolite('Cpd2085', formula='', name='heparan sulfate, free chain [l]', compartment='')
Cpd2086 = Metabolite('Cpd2086', formula='', name='keratan sulfate II (core 2-linked), degradation product 1 [l]', compartment='')
Cpd2087 = Metabolite('Cpd2087', formula='', name='keratan sulfate II (core 4-linked), degradation product 1 [l]', compartment='')
Cpd2088 = Metabolite('Cpd2088', formula='', name='heparan sulfate, degradation product 11 [l]', compartment='')
Cpd2089 = Metabolite('Cpd2089', formula='', name='L-saccharopinate(1-) [m]', compartment='')
Cpd2090 = Metabolite('Cpd2090', formula='', name='L-allysine [m]', compartment='')
Cpd2091 = Metabolite('Cpd2091', formula='', name='Metanephrine [c]', compartment='')
Cpd2092 = Metabolite('Cpd2092', formula='', name='Sarcosine [x]', compartment='')
Cpd2093 = Metabolite('Cpd2093', formula='', name='formaldehyde [x]', compartment='')
Cpd2094 = Metabolite('Cpd2094', formula='', name='(2S,3R)-2-azaniumyl-3-hydroxyoctadecyl phosphate [r]', compartment='')
Cpd2095 = Metabolite('Cpd2095', formula='', name='sphinganine(1+) [r]', compartment='')
Cpd2096 = Metabolite('Cpd2096', formula='', name='sphingosine 1-phosphate(1-) [r]', compartment='')
Cpd2097 = Metabolite('Cpd2097', formula='', name='sphingosine(1+) [r]', compartment='')
Cpd2098 = Metabolite('Cpd2098', formula='', name='D-glucitol [c]', compartment='')
Cpd2099 = Metabolite('Cpd2099', formula='', name='4,8 dimethylnonanoyl-CoA [c]', compartment='')
Cpd2100 = Metabolite('Cpd2100', formula='', name='Se-Adenosylselenohomocysteine [c]', compartment='')
Cpd2101 = Metabolite('Cpd2101', formula='', name='Selenohomocysteine [c]', compartment='')
Cpd2102 = Metabolite('Cpd2102', formula='', name='selenate [c]', compartment='')
Cpd2103 = Metabolite('Cpd2103', formula='', name='1,4-dithiothreitol [c]', compartment='')
Cpd2104 = Metabolite('Cpd2104', formula='', name='selenocysteine [c]', compartment='')
Cpd2105 = Metabolite('Cpd2105', formula='', name='trans-1,2-dithiane-4,5-diol [c]', compartment='')
Cpd2106 = Metabolite('Cpd2106', formula='', name='Selenide [c]', compartment='')
Cpd2107 = Metabolite('Cpd2107', formula='', name='Selenocystathionine [c]', compartment='')
Cpd2108 = Metabolite('Cpd2108', formula='', name='selenomethionine [c]', compartment='')
Cpd2109 = Metabolite('Cpd2109', formula='', name='Se-Adenosylselenomethionine [c]', compartment='')
Cpd2110 = Metabolite('Cpd2110', formula='', name='Selenophosphate [c]', compartment='')
Cpd2111 = Metabolite('Cpd2111', formula='', name='selenate [e]', compartment='')
Cpd2112 = Metabolite('Cpd2112', formula='', name='D-serine [e]', compartment='')
Cpd2113 = Metabolite('Cpd2113', formula='', name='D-serine [c]', compartment='')
Cpd2114 = Metabolite('Cpd2114', formula='', name='O-phosphonatoethanaminium(1-) [r]', compartment='')
Cpd2115 = Metabolite('Cpd2115', formula='', name='hexadecanal [r]', compartment='')
Cpd2116 = Metabolite('Cpd2116', formula='', name='de-Fuc, reducing GlcNAc removed, de-Sia form of PA6 (w/o peptide linkage) [c]', compartment='')
Cpd2117 = Metabolite('Cpd2117', formula='', name='keratan sulfate I, degradation product 3 [l]', compartment='')
Cpd2118 = Metabolite('Cpd2118', formula='', name='keratan sulfate II (core 2-linked) [l]', compartment='')
Cpd2119 = Metabolite('Cpd2119', formula='', name='keratan sulfate II (core 4-linked) [l]', compartment='')
Cpd2120 = Metabolite('Cpd2120', formula='', name='de-Fuc, reducing GlcNAc removed form of PA6 (w/o peptide linkage) [l]', compartment='')
Cpd2121 = Metabolite('Cpd2121', formula='', name='sialyl galactosylgloboside [g]', compartment='')
Cpd2122 = Metabolite('Cpd2122', formula='', name='(2S,3R)-2-azaniumyl-3-hydroxyoctadecyl phosphate [c]', compartment='')
Cpd2123 = Metabolite('Cpd2123', formula='', name='(S)-3-sulfonatolactate(2-) [c]', compartment='')
Cpd2124 = Metabolite('Cpd2124', formula='', name='(S)-3-sulfonatolactate(2-) [m]', compartment='')
Cpd2125 = Metabolite('Cpd2125', formula='', name='sphingomyelin betaine [g]', compartment='')
Cpd2126 = Metabolite('Cpd2126', formula='', name='choline phosphate(1-) [g]', compartment='')
Cpd2127 = Metabolite('Cpd2127', formula='', name='sphingomyelin betaine [l]', compartment='')
Cpd2128 = Metabolite('Cpd2128', formula='', name='choline phosphate(1-) [l]', compartment='')
Cpd2129 = Metabolite('Cpd2129', formula='', name='sphingosylphosphorylcholine [c]', compartment='')
Cpd2130 = Metabolite('Cpd2130', formula='', name='sphingosine(1+) [c]', compartment='')
Cpd2131 = Metabolite('Cpd2131', formula='', name='sphingomyelin betaine [c]', compartment='')
Cpd2132 = Metabolite('Cpd2132', formula='', name='sulfate [e]', compartment='')
Cpd2133 = Metabolite('Cpd2133', formula='', name='R group 1 Coenzyme A [c]', compartment='')
Cpd2134 = Metabolite('Cpd2134', formula='', name='cholesterol ester [c]', compartment='')
Cpd2135 = Metabolite('Cpd2135', formula='', name='R group 1 Coenzyme A [r]', compartment='')
Cpd2136 = Metabolite('Cpd2136', formula='', name='cholesterol ester [r]', compartment='')
Cpd2137 = Metabolite('Cpd2137', formula='', name='R group 2 Coenzyme A [c]', compartment='')
Cpd2138 = Metabolite('Cpd2138', formula='', name='R group 2 Coenzyme A [r]', compartment='')
Cpd2139 = Metabolite('Cpd2139', formula='', name='sphingosine 1-phosphate(1-) [c]', compartment='')
Cpd2140 = Metabolite('Cpd2140', formula='', name='Spermidine [c]', compartment='')
Cpd2141 = Metabolite('Cpd2141', formula='', name='superoxide [c]', compartment='')
Cpd2142 = Metabolite('Cpd2142', formula='', name='superoxide [e]', compartment='')
Cpd2143 = Metabolite('Cpd2143', formula='', name='O2 [e]', compartment='')
Cpd2144 = Metabolite('Cpd2144', formula='', name='superoxide [m]', compartment='')
Cpd2145 = Metabolite('Cpd2145', formula='', name='superoxide [n]', compartment='')
Cpd2146 = Metabolite('Cpd2146', formula='', name='Hydrogen peroxide [n]', compartment='')
Cpd2147 = Metabolite('Cpd2147', formula='', name='O2 [n]', compartment='')
Cpd2148 = Metabolite('Cpd2148', formula='', name='superoxide [x]', compartment='')
Cpd2149 = Metabolite('Cpd2149', formula='', name='spermine(4+) [c]', compartment='')
Cpd2150 = Metabolite('Cpd2150', formula='', name='Nicotinamide adenine dinucleotide phosphate - reduced [n]', compartment='')
Cpd2151 = Metabolite('Cpd2151', formula='', name='Nicotinamide adenine dinucleotide phosphate [n]', compartment='')
Cpd2152 = Metabolite('Cpd2152', formula='', name='5,6,7,8-tetrahydrobiopterin [n]', compartment='')
Cpd2153 = Metabolite('Cpd2153', formula='', name='L-serine [x]', compartment='')
Cpd2154 = Metabolite('Cpd2154', formula='', name='3-hydroxypyruvate [x]', compartment='')
Cpd2155 = Metabolite('Cpd2155', formula='', name='squalene [r]', compartment='')
Cpd2156 = Metabolite('Cpd2156', formula='', name='Farnesyl diphosphate [r]', compartment='')
Cpd2157 = Metabolite('Cpd2157', formula='', name='17beta-hydroxy-5alpha-androstan-3-one [r]', compartment='')
Cpd2158 = Metabolite('Cpd2158', formula='', name='Formyl-5-hydroxykynurenamine [c]', compartment='')
Cpd2159 = Metabolite('Cpd2159', formula='', name='N-Methylserotonin [c]', compartment='')
Cpd2160 = Metabolite('Cpd2160', formula='', name='serotonin(1+) [e]', compartment='')
Cpd2161 = Metabolite('Cpd2161', formula='', name='GM1b [g]', compartment='')
Cpd2162 = Metabolite('Cpd2162', formula='', name='N-acetylneuraminosyl-D-galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminosyl)-D-galactosyl-D-glucosylceramide dianion [g]', compartment='')
Cpd2163 = Metabolite('Cpd2163', formula='', name='GT1b [g]', compartment='')
Cpd2164 = Metabolite('Cpd2164', formula='', name='(Gal)3 (Glc)1 (GlcNAc)1 (Cer)1 [g]', compartment='')
Cpd2165 = Metabolite('Cpd2165', formula='', name='VI3NeuAc-nLc6Cer [g]', compartment='')
Cpd2166 = Metabolite('Cpd2166', formula='', name='sialyl (1,3) sialyl (2,6) galactosylgloboside [g]', compartment='')
Cpd2167 = Metabolite('Cpd2167', formula='', name='sialyl (2,3) sialyl (2,6) galactosylgloboside [g]', compartment='')
Cpd2168 = Metabolite('Cpd2168', formula='', name='GD1beta [g]', compartment='')
Cpd2169 = Metabolite('Cpd2169', formula='', name='GT1aalpha [g]', compartment='')
Cpd2170 = Metabolite('Cpd2170', formula='', name='GQ1balpha [g]', compartment='')
Cpd2171 = Metabolite('Cpd2171', formula='', name='GQ1c [g]', compartment='')
Cpd2172 = Metabolite('Cpd2172', formula='', name='GP1c [g]', compartment='')
Cpd2173 = Metabolite('Cpd2173', formula='', name='disialyl galactosylgloboside [g]', compartment='')
Cpd2174 = Metabolite('Cpd2174', formula='', name='GQ1b [g]', compartment='')
Cpd2175 = Metabolite('Cpd2175', formula='', name='GT1a [g]', compartment='')
Cpd2176 = Metabolite('Cpd2176', formula='', name='GD1c [g]', compartment='')
Cpd2177 = Metabolite('Cpd2177', formula='', name='GP1c alpha [g]', compartment='')
Cpd2178 = Metabolite('Cpd2178', formula='', name='3,8-LD1 [g]', compartment='')
Cpd2179 = Metabolite('Cpd2179', formula='', name='stearidonyl carnitine [c]', compartment='')
Cpd2180 = Metabolite('Cpd2180', formula='', name='stearidonyl carnitine [m]', compartment='')
Cpd2181 = Metabolite('Cpd2181', formula='', name='Succinate [e]', compartment='')
Cpd2182 = Metabolite('Cpd2182', formula='', name='Sucrose [e]', compartment='')
Cpd2183 = Metabolite('Cpd2183', formula='', name='trans-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA [m]', compartment='')
Cpd2184 = Metabolite('Cpd2184', formula='', name='trans-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA [x]', compartment='')
Cpd2185 = Metabolite('Cpd2185', formula='', name='trans-4-coumarate [c]', compartment='')
Cpd2186 = Metabolite('Cpd2186', formula='', name='trans-caffeate [c]', compartment='')
Cpd2187 = Metabolite('Cpd2187', formula='', name='Sedoheptulose 7-phosphate [c]', compartment='')
Cpd2188 = Metabolite('Cpd2188', formula='', name='D-Erythrose 4-phosphate [c]', compartment='')
Cpd2189 = Metabolite('Cpd2189', formula='', name='Taurine [e]', compartment='')
Cpd2190 = Metabolite('Cpd2190', formula='', name='Sodium [x]', compartment='')
Cpd2191 = Metabolite('Cpd2191', formula='', name='taurocholate [e]', compartment='')
Cpd2192 = Metabolite('Cpd2192', formula='', name='taurocholate [c]', compartment='')
Cpd2193 = Metabolite('Cpd2193', formula='', name='dTDP-glucose(2-) [c]', compartment='')
Cpd2194 = Metabolite('Cpd2194', formula='', name='dTDP-4-dehydro-6-deoxy-alpha-D-glucose(2-) [c]', compartment='')
Cpd2195 = Metabolite('Cpd2195', formula='', name='tetracosapentaenoyl carnitine [c]', compartment='')
Cpd2196 = Metabolite('Cpd2196', formula='', name='tetracosapentaenoyl carnitine [m]', compartment='')
Cpd2197 = Metabolite('Cpd2197', formula='', name='tetracosapentaenoyl coenzyme A, n-3 [m]', compartment='')
Cpd2198 = Metabolite('Cpd2198', formula='', name='tetracosapentaenoyl coenzyme A, n-6 [m]', compartment='')
Cpd2199 = Metabolite('Cpd2199', formula='', name='Tetracosatetraenoyl carnitine [c]', compartment='')
Cpd2200 = Metabolite('Cpd2200', formula='', name='Tetracosatetraenoyl carnitine [m]', compartment='')
Cpd2201 = Metabolite('Cpd2201', formula='', name='tetracosatetraenoyl coenzyme A [m]', compartment='')
Cpd2202 = Metabolite('Cpd2202', formula='', name='Thymidine [e]', compartment='')
Cpd2203 = Metabolite('Cpd2203', formula='', name='Thiamin monophosphate [e]', compartment='')
Cpd2204 = Metabolite('Cpd2204', formula='', name='Thiamin monophosphate [c]', compartment='')
Cpd2205 = Metabolite('Cpd2205', formula='', name='Thiamin [e]', compartment='')
Cpd2206 = Metabolite('Cpd2206', formula='', name='Thiamin [c]', compartment='')
Cpd2207 = Metabolite('Cpd2207', formula='', name='thiamine(1+) triphosphate(4-) [c]', compartment='')
Cpd2208 = Metabolite('Cpd2208', formula='', name='Thiamine diphosphate [c]', compartment='')
Cpd2209 = Metabolite('Cpd2209', formula='', name='thiamine(1+) triphosphate(4-) [e]', compartment='')
Cpd2210 = Metabolite('Cpd2210', formula='', name='O-phosphonato-L-homoserine(2-) [c]', compartment='')
Cpd2211 = Metabolite('Cpd2211', formula='', name='Thymine [e]', compartment='')
Cpd2212 = Metabolite('Cpd2212', formula='', name='L-thyroxine [e]', compartment='')
Cpd2213 = Metabolite('Cpd2213', formula='', name='N6,N6,N6-Trimethyl-L-lysine [c]', compartment='')
Cpd2214 = Metabolite('Cpd2214', formula='', name='Timnodonyl carnitine [c]', compartment='')
Cpd2215 = Metabolite('Cpd2215', formula='', name='Timnodonyl carnitine [m]', compartment='')
Cpd2216 = Metabolite('Cpd2216', formula='', name='ubiquinone-10 [c]', compartment='')
Cpd2217 = Metabolite('Cpd2217', formula='', name='Ubiquinol-10 [c]', compartment='')
Cpd2218 = Metabolite('Cpd2218', formula='', name='Oxidized thioredoxin [m]', compartment='')
Cpd2219 = Metabolite('Cpd2219', formula='', name='Reduced thioredoxin [m]', compartment='')
Cpd2220 = Metabolite('Cpd2220', formula='', name='Trehalose [c]', compartment='')
Cpd2221 = Metabolite('Cpd2221', formula='', name='Trehalose [e]', compartment='')
Cpd2222 = Metabolite('Cpd2222', formula='', name='Triiodothyronine sulfate [c]', compartment='')
Cpd2223 = Metabolite('Cpd2223', formula='', name='3,3,5-triiodo-L-thyronine [e]', compartment='')
Cpd2224 = Metabolite('Cpd2224', formula='', name='L-tryptophan [e]', compartment='')
Cpd2225 = Metabolite('Cpd2225', formula='', name='testosterone 3-glucosiduronic acid [c]', compartment='')
Cpd2226 = Metabolite('Cpd2226', formula='', name='testosterone 3-glucosiduronic acid [e]', compartment='')
Cpd2227 = Metabolite('Cpd2227', formula='', name='testosterone [c]', compartment='')
Cpd2228 = Metabolite('Cpd2228', formula='', name='testosterone sulfate [c]', compartment='')
Cpd2229 = Metabolite('Cpd2229', formula='', name='thiosulfate(2-) [e]', compartment='')
Cpd2230 = Metabolite('Cpd2230', formula='', name='tetradecanoate (n-C14:0) [e]', compartment='')
Cpd2231 = Metabolite('Cpd2231', formula='', name='tetradecanoyl carnitine [c]', compartment='')
Cpd2232 = Metabolite('Cpd2232', formula='', name='tetradecanoyl carnitine [m]', compartment='')
Cpd2233 = Metabolite('Cpd2233', formula='', name='thromboxane A2(1-) [r]', compartment='')
Cpd2234 = Metabolite('Cpd2234', formula='', name='tyraminium [c]', compartment='')
Cpd2235 = Metabolite('Cpd2235', formula='', name='Tyramine O-sulfate [c]', compartment='')
Cpd2236 = Metabolite('Cpd2236', formula='', name='Melanin [c]', compartment='')
Cpd2237 = Metabolite('Cpd2237', formula='', name='L-dopaquinone [c]', compartment='')
Cpd2238 = Metabolite('Cpd2238', formula='', name='L-tyrosine [e]', compartment='')
Cpd2239 = Metabolite('Cpd2239', formula='', name='L-tyrosine [m]', compartment='')
Cpd2240 = Metabolite('Cpd2240', formula='', name='3-(4-hydroxyphenyl)pyruvate [m]', compartment='')
Cpd2241 = Metabolite('Cpd2241', formula='', name='UDP-N-acetyl-D-galactosamine(2-) [c]', compartment='')
Cpd2242 = Metabolite('Cpd2242', formula='', name='Dolichyl beta-D-glucosyl phosphate, human liver homolog [c]', compartment='')
Cpd2243 = Metabolite('Cpd2243', formula='', name='Dolichyl beta-D-glucosyl phosphate, human uterine homolog [c]', compartment='')
Cpd2244 = Metabolite('Cpd2244', formula='', name='UDP-D-glucuronate [r]', compartment='')
Cpd2245 = Metabolite('Cpd2245', formula='', name='UDPglucose [g]', compartment='')
Cpd2246 = Metabolite('Cpd2246', formula='', name='carbon dioxide [g]', compartment='')
Cpd2247 = Metabolite('Cpd2247', formula='', name='UDP-alpha-D-xylose(2-) [g]', compartment='')
Cpd2248 = Metabolite('Cpd2248', formula='', name='UDP-alpha-D-xylose(2-) [c]', compartment='')
Cpd2249 = Metabolite('Cpd2249', formula='', name='Lactose [g]', compartment='')
Cpd2250 = Metabolite('Cpd2250', formula='', name='UDP-N-acetyl-D-galactosamine(2-) [r]', compartment='')
Cpd2251 = Metabolite('Cpd2251', formula='', name='UDPglucose [r]', compartment='')
Cpd2252 = Metabolite('Cpd2252', formula='', name='bilirubin(2-) [r]', compartment='')
Cpd2253 = Metabolite('Cpd2253', formula='', name='mono(glucosyluronic acid)bilirubin(2-) [r]', compartment='')
Cpd2254 = Metabolite('Cpd2254', formula='', name='estrone 3-glucosiduronic acid [r]', compartment='')
Cpd2255 = Metabolite('Cpd2255', formula='', name='bis(beta-glucosyluronate)bilirubin [r]', compartment='')
Cpd2256 = Metabolite('Cpd2256', formula='', name='androsterone 3-glucosiduronic acid [r]', compartment='')
Cpd2257 = Metabolite('Cpd2257', formula='', name='testosterone 3-glucosiduronic acid [r]', compartment='')
Cpd2258 = Metabolite('Cpd2258', formula='', name='all-trans-retinoate [r]', compartment='')
Cpd2259 = Metabolite('Cpd2259', formula='', name='retinoyl glucuronide [r]', compartment='')
Cpd2260 = Metabolite('Cpd2260', formula='', name='13-cis-retinoate [r]', compartment='')
Cpd2261 = Metabolite('Cpd2261', formula='', name='13-cis-retinoyl glucuronide [r]', compartment='')
Cpd2262 = Metabolite('Cpd2262', formula='', name='17beta-estradiol 3-glucosiduronic acid [r]', compartment='')
Cpd2263 = Metabolite('Cpd2263', formula='', name='3alpha-hydroxy-5beta-androstan-17-one [r]', compartment='')
Cpd2264 = Metabolite('Cpd2264', formula='', name='3alpha-hydroxy-5beta-androstan-17-one 3-glucosiduronic acid [r]', compartment='')
Cpd2265 = Metabolite('Cpd2265', formula='', name='estriol [r]', compartment='')
Cpd2266 = Metabolite('Cpd2266', formula='', name='16-Glucuronide-estriol [r]', compartment='')
Cpd2267 = Metabolite('Cpd2267', formula='', name='5alpha-Dihydrotestosterone glucuronide [r]', compartment='')
Cpd2268 = Metabolite('Cpd2268', formula='', name='UMP [n]', compartment='')
Cpd2269 = Metabolite('Cpd2269', formula='', name='Uroporphyrinogen III [c]', compartment='')
Cpd2270 = Metabolite('Cpd2270', formula='', name='uracil [e]', compartment='')
Cpd2271 = Metabolite('Cpd2271', formula='', name='Urea [e]', compartment='')
Cpd2272 = Metabolite('Cpd2272', formula='', name='Vaccenyl carnitine [c]', compartment='')
Cpd2273 = Metabolite('Cpd2273', formula='', name='Vaccenyl carnitine [m]', compartment='')
Cpd2274 = Metabolite('Cpd2274', formula='', name='L-valine [e]', compartment='')
Cpd2275 = Metabolite('Cpd2275', formula='', name='L-valine [c]', compartment='')
Cpd2276 = Metabolite('Cpd2276', formula='', name='3-methyl-2-oxobutanoate [c]', compartment='')
Cpd2277 = Metabolite('Cpd2277', formula='', name='L-valine [m]', compartment='')
Cpd2278 = Metabolite('Cpd2278', formula='', name='vitamin D2 [m]', compartment='')
Cpd2279 = Metabolite('Cpd2279', formula='', name='calciol [m]', compartment='')
Cpd2280 = Metabolite('Cpd2280', formula='', name='calciol [e]', compartment='')
Cpd2281 = Metabolite('Cpd2281', formula='', name='calciol [c]', compartment='')
Cpd2282 = Metabolite('Cpd2282', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholestanate [x]', compartment='')
Cpd2283 = Metabolite('Cpd2283', formula='', name='AMP [x]', compartment='')
Cpd2284 = Metabolite('Cpd2284', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholestanate [r]', compartment='')
Cpd2285 = Metabolite('Cpd2285', formula='', name='AMP [r]', compartment='')
Cpd2286 = Metabolite('Cpd2286', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate [x]', compartment='')
Cpd2287 = Metabolite('Cpd2287', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate [r]', compartment='')
Cpd2288 = Metabolite('Cpd2288', formula='', name='Xanthine [x]', compartment='')
Cpd2289 = Metabolite('Cpd2289', formula='', name='Urate [x]', compartment='')
Cpd2290 = Metabolite('Cpd2290', formula='', name='Hypoxanthine [x]', compartment='')
Cpd2291 = Metabolite('Cpd2291', formula='', name='aldehydo-D-xylose [e]', compartment='')
Cpd2292 = Metabolite('Cpd2292', formula='', name='aldehydo-D-xylose [c]', compartment='')
Cpd2293 = Metabolite('Cpd2293', formula='', name='UDP-alpha-D-xylose(2-) [r]', compartment='')
Cpd2294 = Metabolite('Cpd2294', formula='', name='protein-linked serine residue (glycosaminoglycan attachment site) [r]', compartment='')
Cpd2295 = Metabolite('Cpd2295', formula='', name='Xyl-L-Ser (protein) [r]', compartment='')
Cpd2296 = Metabolite('Cpd2296', formula='', name='L-xylulose [c]', compartment='')
Cpd2297 = Metabolite('Cpd2297', formula='', name='xylitol [c]', compartment='')
Cpd2298 = Metabolite('Cpd2298', formula='', name='Chitobiose [l]', compartment='')
Cpd2299 = Metabolite('Cpd2299', formula='', name='Ferricytochrome b5 [c]', compartment='')
Cpd2300 = Metabolite('Cpd2300', formula='', name='Ferrocytochrome b5 [c]', compartment='')
Cpd2301 = Metabolite('Cpd2301', formula='', name='Dephospho-CoA [m]', compartment='')
Cpd2302 = Metabolite('Cpd2302', formula='', name='Diphosphate [e]', compartment='')
Cpd2303 = Metabolite('Cpd2303', formula='', name='5-Phospho-alpha-D-ribose 1-diphosphate [e]', compartment='')
Cpd2304 = Metabolite('Cpd2304', formula='', name='Acetyl adenylate [c]', compartment='')
Cpd2305 = Metabolite('Cpd2305', formula='', name='oxalatosuccinate(3-) [c]', compartment='')
Cpd2306 = Metabolite('Cpd2306', formula='', name='oxalatosuccinate(3-) [m]', compartment='')
Cpd2307 = Metabolite('Cpd2307', formula='', name='oxalatosuccinate(3-) [x]', compartment='')
Cpd2308 = Metabolite('Cpd2308', formula='', name='2-oxoglutaramate [c]', compartment='')
Cpd2309 = Metabolite('Cpd2309', formula='', name='2-oxoglutaramate [m]', compartment='')
Cpd2310 = Metabolite('Cpd2310', formula='', name='UDPglucose [e]', compartment='')
Cpd2311 = Metabolite('Cpd2311', formula='', name='D-Glucose 1-phosphate [e]', compartment='')
Cpd2312 = Metabolite('Cpd2312', formula='', name='UDP-N-acetyl-alpha-D-glucosamine(2-) [r]', compartment='')
Cpd2313 = Metabolite('Cpd2313', formula='', name='N-acetyl-D-mannosamine [r]', compartment='')
Cpd2314 = Metabolite('Cpd2314', formula='', name='Formamidopyrimidine nucleoside triphosphate [c]', compartment='')
Cpd2315 = Metabolite('Cpd2315', formula='', name='Formamidopyrimidine nucleoside triphosphate [n]', compartment='')
Cpd2316 = Metabolite('Cpd2316', formula='', name='CDP [e]', compartment='')
Cpd2317 = Metabolite('Cpd2317', formula='', name='CTP [e]', compartment='')
Cpd2318 = Metabolite('Cpd2318', formula='', name='Thiamine diphosphate [m]', compartment='')
Cpd2319 = Metabolite('Cpd2319', formula='', name='3-Carboxy-1-hydroxypropyl-ThPP [m]', compartment='')
Cpd2320 = Metabolite('Cpd2320', formula='', name='presqualene diphosphate(3-) [r]', compartment='')
Cpd2321 = Metabolite('Cpd2321', formula='', name='(S)-lactate [x]', compartment='')
Cpd2322 = Metabolite('Cpd2322', formula='', name='ITP(3-) [e]', compartment='')
Cpd2323 = Metabolite('Cpd2323', formula='', name='hydrosulfide [c]', compartment='')
Cpd2324 = Metabolite('Cpd2324', formula='', name='2-Amino-3-oxoadipate [m]', compartment='')
Cpd2325 = Metabolite('Cpd2325', formula='', name='Glycerol 3-phosphate [x]', compartment='')
Cpd2326 = Metabolite('Cpd2326', formula='', name='Propinol adenylate [c]', compartment='')
Cpd2327 = Metabolite('Cpd2327', formula='', name='Propinol adenylate [m]', compartment='')
Cpd2328 = Metabolite('Cpd2328', formula='', name='Galactose-1P [c]', compartment='')
Cpd2329 = Metabolite('Cpd2329', formula='', name='Dihydroxyacetone phosphate [e]', compartment='')
Cpd2330 = Metabolite('Cpd2330', formula='', name='glycerone [e]', compartment='')
Cpd2331 = Metabolite('Cpd2331', formula='', name='D-Glyceraldehyde [m]', compartment='')
Cpd2332 = Metabolite('Cpd2332', formula='', name='alpha-D-Ribose 5-phosphate [r]', compartment='')
Cpd2333 = Metabolite('Cpd2333', formula='', name='Putrescine [e]', compartment='')
Cpd2334 = Metabolite('Cpd2334', formula='', name='4-Aminobutanal [e]', compartment='')
Cpd2335 = Metabolite('Cpd2335', formula='', name='3-Oxohexanoyl-CoA [m]', compartment='')
Cpd2336 = Metabolite('Cpd2336', formula='', name='arachidonate [x]', compartment='')
Cpd2337 = Metabolite('Cpd2337', formula='', name='Palmitoyl-CoA (n-C16:0CoA) [r]', compartment='')
Cpd2338 = Metabolite('Cpd2338', formula='', name='cis-aconitate(3-) [m]', compartment='')
Cpd2339 = Metabolite('Cpd2339', formula='', name='D-glycerate [m]', compartment='')
Cpd2340 = Metabolite('Cpd2340', formula='', name='3-Phospho-D-glycerate [m]', compartment='')
Cpd2341 = Metabolite('Cpd2341', formula='', name='Apo-ACP [c]', compartment='')
Cpd2342 = Metabolite('Cpd2342', formula='', name='Lactose-6P [e]', compartment='')
Cpd2343 = Metabolite('Cpd2343', formula='', name='S-Acetyldihydrolipoamide [m]', compartment='')
Cpd2344 = Metabolite('Cpd2344', formula='', name='S-Succinyldihydrolipoamide [m]', compartment='')
Cpd2345 = Metabolite('Cpd2345', formula='', name='S-(2-Methylpropanoyl)-dihydrolipoamide [m]', compartment='')
Cpd2346 = Metabolite('Cpd2346', formula='', name='S-(3-Methylbutanoyl)-dihydrolipoamide [m]', compartment='')
Cpd2347 = Metabolite('Cpd2347', formula='', name='Isomaltose [e]', compartment='')
Cpd2348 = Metabolite('Cpd2348', formula='', name='6-Lactoyl-5,6,7,8-tetrahydropterin [c]', compartment='')
Cpd2349 = Metabolite('Cpd2349', formula='', name='sedoheptulose 1,7-bisphosphate(4-) [c]', compartment='')
Cpd2350 = Metabolite('Cpd2350', formula='', name='dGTP [e]', compartment='')
Cpd2351 = Metabolite('Cpd2351', formula='', name='dGMP [e]', compartment='')
Cpd2352 = Metabolite('Cpd2352', formula='', name='Malonyl-CoA [r]', compartment='')
Cpd2353 = Metabolite('Cpd2353', formula='', name='Malonyl-Carnitin [r]', compartment='')
Cpd2354 = Metabolite('Cpd2354', formula='', name='Malonyl-Carnitin [c]', compartment='')
Cpd2355 = Metabolite('Cpd2355', formula='', name='Malonyl-Carnitin [m]', compartment='')
Cpd2356 = Metabolite('Cpd2356', formula='', name='Malonyl-Carnitin [x]', compartment='')
Cpd2357 = Metabolite('Cpd2357', formula='', name='L-palmitoylcarnitine [r]', compartment='')
Cpd2358 = Metabolite('Cpd2358', formula='', name='L-palmitoylcarnitine [x]', compartment='')
Cpd2359 = Metabolite('Cpd2359', formula='', name='linoleic coenzyme A [r]', compartment='')
Cpd2360 = Metabolite('Cpd2360', formula='', name='linoleic-Carnitine [r]', compartment='')
Cpd2361 = Metabolite('Cpd2361', formula='', name='linoleic coenzyme A [x]', compartment='')
Cpd2362 = Metabolite('Cpd2362', formula='', name='linoleic-Carnitine [x]', compartment='')
Cpd2363 = Metabolite('Cpd2363', formula='', name='arachidonoyl-CoA(4-) [r]', compartment='')
Cpd2364 = Metabolite('Cpd2364', formula='', name='C20:4 carnitine [r]', compartment='')
Cpd2365 = Metabolite('Cpd2365', formula='', name='C20:4 carnitine [x]', compartment='')
Cpd2366 = Metabolite('Cpd2366', formula='', name='palmitoleyl-CoA(4-) [r]', compartment='')
Cpd2367 = Metabolite('Cpd2367', formula='', name='palmitoleoyl-Carnitine [r]', compartment='')
Cpd2368 = Metabolite('Cpd2368', formula='', name='palmitoleyl-CoA(4-) [c]', compartment='')
Cpd2369 = Metabolite('Cpd2369', formula='', name='palmitoleoyl-Carnitine [c]', compartment='')
Cpd2370 = Metabolite('Cpd2370', formula='', name='palmitoleyl-CoA(4-) [m]', compartment='')
Cpd2371 = Metabolite('Cpd2371', formula='', name='palmitoleoyl-Carnitine [m]', compartment='')
Cpd2372 = Metabolite('Cpd2372', formula='', name='S-Glutaryldihydrolipoamide [m]', compartment='')
Cpd2373 = Metabolite('Cpd2373', formula='', name='4-Aminobutanal [m]', compartment='')
Cpd2374 = Metabolite('Cpd2374', formula='', name='Homocarnosine [c]', compartment='')
Cpd2375 = Metabolite('Cpd2375', formula='', name='(R)-mevalonate [c]', compartment='')
Cpd2376 = Metabolite('Cpd2376', formula='', name='dTDP [e]', compartment='')
Cpd2377 = Metabolite('Cpd2377', formula='', name='dTMP [e]', compartment='')
Cpd2378 = Metabolite('Cpd2378', formula='', name='dTTP [e]', compartment='')
Cpd2379 = Metabolite('Cpd2379', formula='', name='Urate [c]', compartment='')
Cpd2380 = Metabolite('Cpd2380', formula='', name='Folate [m]', compartment='')
Cpd2381 = Metabolite('Cpd2381', formula='', name='5-formyltetrahydrofolate(2-) [m]', compartment='')
Cpd2382 = Metabolite('Cpd2382', formula='', name='hexadecanal [c]', compartment='')
Cpd2383 = Metabolite('Cpd2383', formula='', name='phenylacetate [c]', compartment='')
Cpd2384 = Metabolite('Cpd2384', formula='', name='phenylacetaldehyde [m]', compartment='')
Cpd2385 = Metabolite('Cpd2385', formula='', name='phenylacetate [m]', compartment='')
Cpd2386 = Metabolite('Cpd2386', formula='', name='betaine aldehyde [c]', compartment='')
Cpd2387 = Metabolite('Cpd2387', formula='', name='2,5-dihydroxybenzoate [c]', compartment='')
Cpd2388 = Metabolite('Cpd2388', formula='', name='Gentisate aldehyde [c]', compartment='')
Cpd2389 = Metabolite('Cpd2389', formula='', name='Cholate [r]', compartment='')
Cpd2390 = Metabolite('Cpd2390', formula='', name='choloyl-CoA(4-) [r]', compartment='')
Cpd2391 = Metabolite('Cpd2391', formula='', name='pantetheine [m]', compartment='')
Cpd2392 = Metabolite('Cpd2392', formula='', name='Pantetheine 4-phosphate [m]', compartment='')
Cpd2393 = Metabolite('Cpd2393', formula='', name='4-nitrophenyl phosphate(2-) [e]', compartment='')
Cpd2394 = Metabolite('Cpd2394', formula='', name='4-nitrophenolate [e]', compartment='')
Cpd2395 = Metabolite('Cpd2395', formula='', name='L-allysine [c]', compartment='')
Cpd2396 = Metabolite('Cpd2396', formula='', name='hydroxidodioxidosulfidosulfate(1-) [c]', compartment='')
Cpd2397 = Metabolite('Cpd2397', formula='', name='3-hydroxypropionate [m]', compartment='')
Cpd2398 = Metabolite('Cpd2398', formula='', name='S-(2-Methylbutanoyl)-dihydrolipoamide [m]', compartment='')
Cpd2399 = Metabolite('Cpd2399', formula='', name='D-tagatofuranose 6-phosphate(2-) [c]', compartment='')
Cpd2400 = Metabolite('Cpd2400', formula='', name='D-tagatofuranose 1,6-bisphosphate(4-) [c]', compartment='')
Cpd2401 = Metabolite('Cpd2401', formula='', name='N-((R)-4-Phosphopantothenoyl)-L-cysteine [m]', compartment='')
Cpd2402 = Metabolite('Cpd2402', formula='', name='trans-4-hydroxy-L-proline [c]', compartment='')
Cpd2403 = Metabolite('Cpd2403', formula='', name='L-1-Pyrroline-3-hydroxy-5-carboxylate [c]', compartment='')
Cpd2404 = Metabolite('Cpd2404', formula='', name='trans-4-hydroxy-L-proline [m]', compartment='')
Cpd2405 = Metabolite('Cpd2405', formula='', name='Galactosylglycerol [e]', compartment='')
Cpd2406 = Metabolite('Cpd2406', formula='', name='choloyl-CoA(4-) [c]', compartment='')
Cpd2407 = Metabolite('Cpd2407', formula='', name='(2E)-Octenoyl-CoA [m]', compartment='')
Cpd2408 = Metabolite('Cpd2408', formula='', name='3-Oxodecanoyl-CoA [m]', compartment='')
Cpd2409 = Metabolite('Cpd2409', formula='', name='L-Octanoylcarnitine [m]', compartment='')
Cpd2410 = Metabolite('Cpd2410', formula='', name='L-Octanoylcarnitine [x]', compartment='')
Cpd2411 = Metabolite('Cpd2411', formula='', name='Dodecanoyl-CoA (n-C12:0CoA) [m]', compartment='')
Cpd2412 = Metabolite('Cpd2412', formula='', name='3-Oxotetradecanoyl-CoA [m]', compartment='')
Cpd2413 = Metabolite('Cpd2413', formula='', name='Leukotriene D4 [c]', compartment='')
Cpd2414 = Metabolite('Cpd2414', formula='', name='2-methyl-3-oxopropanoate [c]', compartment='')
Cpd2415 = Metabolite('Cpd2415', formula='', name='Methylmalonate [c]', compartment='')
Cpd2416 = Metabolite('Cpd2416', formula='', name='Methylmalonate [m]', compartment='')
Cpd2417 = Metabolite('Cpd2417', formula='', name='L-3-Cyanoalanine [e]', compartment='')
Cpd2418 = Metabolite('Cpd2418', formula='', name='gamma-Glutamyl-beta-cyanoalanine [e]', compartment='')
Cpd2419 = Metabolite('Cpd2419', formula='', name='carbon dioxide [e]', compartment='')
Cpd2420 = Metabolite('Cpd2420', formula='', name='gamma-Glutamyl-3-aminopropiononitrile [e]', compartment='')
Cpd2421 = Metabolite('Cpd2421', formula='', name='chenodeoxycholate [r]', compartment='')
Cpd2422 = Metabolite('Cpd2422', formula='', name='chenodeoxycholoyl coenzyme a [r]', compartment='')
Cpd2423 = Metabolite('Cpd2423', formula='', name='chenodeoxycholate [c]', compartment='')
Cpd2424 = Metabolite('Cpd2424', formula='', name='chenodeoxycholoyl coenzyme a [c]', compartment='')
Cpd2425 = Metabolite('Cpd2425', formula='', name='trans-tetradec-2-enoyl-CoA(4-) [m]', compartment='')
Cpd2426 = Metabolite('Cpd2426', formula='', name='3-oxopalmitoyl-CoA(4-) [m]', compartment='')
Cpd2427 = Metabolite('Cpd2427', formula='', name='(S)-3-Hydroxydodecanoyl-CoA [m]', compartment='')
Cpd2428 = Metabolite('Cpd2428', formula='', name='trans-Dodec-2-enoyl-CoA [x]', compartment='')
Cpd2429 = Metabolite('Cpd2429', formula='', name='(S)-3-Hydroxydodecanoyl-CoA [x]', compartment='')
Cpd2430 = Metabolite('Cpd2430', formula='', name='NeuNGc [c]', compartment='')
Cpd2431 = Metabolite('Cpd2431', formula='', name='(2R,4S,5R,6R)-2-[({[(2R,3S,4R,5R)-5-(4-amino-2-oxo-1,2-dihydropyrimidin-1-yl)-3,4-dihydroxyoxolan-2-yl]methoxy}phosphinato)oxy]-4-hydroxy-5-(2-hydroxyacetamido)-6-[(1R,2R)-1,2,3-trihydroxypropyl]oxane-2-carboxylate [c]', compartment='')
Cpd2432 = Metabolite('Cpd2432', formula='', name='3-hydroxy-2-methylpropanoyl-CoA(4-) [m]', compartment='')
Cpd2433 = Metabolite('Cpd2433', formula='', name='Acetoacetyl-ACP [c]', compartment='')
Cpd2434 = Metabolite('Cpd2434', formula='', name='N-[(R)-pantothenoyl]-L-cysteinate [c]', compartment='')
Cpd2435 = Metabolite('Cpd2435', formula='', name='N-[(R)-pantothenoyl]-L-cysteinate [m]', compartment='')
Cpd2436 = Metabolite('Cpd2436', formula='', name='(R)-3-Hydroxybutanoyl-ACP [c]', compartment='')
Cpd2437 = Metabolite('Cpd2437', formula='', name='But-2-enoyl-ACP [c]', compartment='')
Cpd2438 = Metabolite('Cpd2438', formula='', name='Butyryl-ACP [c]', compartment='')
Cpd2439 = Metabolite('Cpd2439', formula='', name='3alpha,7alpha-dihydroxy-5beta-cholestan-26-al [c]', compartment='')
Cpd2440 = Metabolite('Cpd2440', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholestanate [c]', compartment='')
Cpd2441 = Metabolite('Cpd2441', formula='', name='(R)-3-Hydroxydecanoyl-ACP [c]', compartment='')
Cpd2442 = Metabolite('Cpd2442', formula='', name='3-Oxodecanoyl-ACP [c]', compartment='')
Cpd2443 = Metabolite('Cpd2443', formula='', name='(2E)-Decenoyl-ACP [c]', compartment='')
Cpd2444 = Metabolite('Cpd2444', formula='', name='(R)-3-Hydroxyoctanoyl-ACP [c]', compartment='')
Cpd2445 = Metabolite('Cpd2445', formula='', name='3-Oxooctanoyl-ACP [c]', compartment='')
Cpd2446 = Metabolite('Cpd2446', formula='', name='(2E)-Octenoyl-ACP [c]', compartment='')
Cpd2447 = Metabolite('Cpd2447', formula='', name='(R)-3-Hydroxypalmitoyl-ACP [c]', compartment='')
Cpd2448 = Metabolite('Cpd2448', formula='', name='3-Oxohexadecanoyl-ACP [c]', compartment='')
Cpd2449 = Metabolite('Cpd2449', formula='', name='(2E)-Hexadecenoyl-ACP [c]', compartment='')
Cpd2450 = Metabolite('Cpd2450', formula='', name='10fthf5glu[c] [c]', compartment='')
Cpd2451 = Metabolite('Cpd2451', formula='', name='3-Oxotetradecanoyl-ACP [c]', compartment='')
Cpd2452 = Metabolite('Cpd2452', formula='', name='(2E)-Tetradecenoyl-ACP [c]', compartment='')
Cpd2453 = Metabolite('Cpd2453', formula='', name='7,8-dihydroneopterin 3-triphosphate(4-) [e]', compartment='')
Cpd2454 = Metabolite('Cpd2454', formula='', name='Dihydroneopterin [e]', compartment='')
Cpd2455 = Metabolite('Cpd2455', formula='', name='2,5-Diamino-6-(5-triphosphoryl-3,4-trihydroxy-2-oxopentyl)- amino-4-oxopyrimidine [c]', compartment='')
Cpd2456 = Metabolite('Cpd2456', formula='', name='2,5-Diamino-6-(5-triphosphoryl-3,4-trihydroxy-2-oxopentyl)- amino-4-oxopyrimidine [n]', compartment='')
Cpd2457 = Metabolite('Cpd2457', formula='', name='(2E)-Dodecenoyl-ACP [c]', compartment='')
Cpd2458 = Metabolite('Cpd2458', formula='', name='(S)-3-Hydroxyhexadecanoyl-CoA [m]', compartment='')
Cpd2459 = Metabolite('Cpd2459', formula='', name='(S)-3-Hydroxyhexadecanoyl-CoA [x]', compartment='')
Cpd2460 = Metabolite('Cpd2460', formula='', name='(S)-3-Hydroxytetradecanoyl-CoA [m]', compartment='')
Cpd2461 = Metabolite('Cpd2461', formula='', name='(S)-3-Hydroxytetradecanoyl-CoA [x]', compartment='')
Cpd2462 = Metabolite('Cpd2462', formula='', name='trans-tetradec-2-enoyl-CoA(4-) [x]', compartment='')
Cpd2463 = Metabolite('Cpd2463', formula='', name='3-Oxododecanoyl-CoA [m]', compartment='')
Cpd2464 = Metabolite('Cpd2464', formula='', name='Decanoyl-CoA (n-C10:0CoA) [m]', compartment='')
Cpd2465 = Metabolite('Cpd2465', formula='', name='(S)-Hydroxydecanoyl-CoA [m]', compartment='')
Cpd2466 = Metabolite('Cpd2466', formula='', name='(S)-Hydroxydecanoyl-CoA [x]', compartment='')
Cpd2467 = Metabolite('Cpd2467', formula='', name='trans-Dec-2-enoyl-CoA [m]', compartment='')
Cpd2468 = Metabolite('Cpd2468', formula='', name='trans-dec-2-enoyl-CoA(4-) [x]', compartment='')
Cpd2469 = Metabolite('Cpd2469', formula='', name='(S)-Hydroxyoctanoyl-CoA [m]', compartment='')
Cpd2470 = Metabolite('Cpd2470', formula='', name='3-Oxooctanoyl-CoA [m]', compartment='')
Cpd2471 = Metabolite('Cpd2471', formula='', name='Hexanoyl-CoA [m]', compartment='')
Cpd2472 = Metabolite('Cpd2472', formula='', name='(S)-Hydroxyhexanoyl-CoA [m]', compartment='')
Cpd2473 = Metabolite('Cpd2473', formula='', name='(2E)-Hexenoyl-CoA [m]', compartment='')
Cpd2474 = Metabolite('Cpd2474', formula='', name='3-Ketolactose [e]', compartment='')
Cpd2475 = Metabolite('Cpd2475', formula='', name='3-Keto-beta-D-galactose [e]', compartment='')
Cpd2476 = Metabolite('Cpd2476', formula='', name='5beta-cholestane-3alpha,7alpha,26-triol [c]', compartment='')
Cpd2477 = Metabolite('Cpd2477', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,26-tetrol [c]', compartment='')
Cpd2478 = Metabolite('Cpd2478', formula='', name='3alpha,7alpha,12alpha,24-Tetrahydroxy-5beta-cholestanoyl-CoA [x]', compartment='')
Cpd2479 = Metabolite('Cpd2479', formula='', name='3,4-Dihydroxymandelaldehyde [m]', compartment='')
Cpd2480 = Metabolite('Cpd2480', formula='', name='3,4-Dihydroxymandelate [m]', compartment='')
Cpd2481 = Metabolite('Cpd2481', formula='', name='(4-hydroxy-3-methoxyphenyl)acetaldehyde [m]', compartment='')
Cpd2482 = Metabolite('Cpd2482', formula='', name='Homovanillate [m]', compartment='')
Cpd2483 = Metabolite('Cpd2483', formula='', name='3-Methoxy-4-hydroxyphenylglycolaldehyde [m]', compartment='')
Cpd2484 = Metabolite('Cpd2484', formula='', name='3-Methoxy-4-hydroxymandelate [m]', compartment='')
Cpd2485 = Metabolite('Cpd2485', formula='', name='3-Oxohexanoyl-ACP [c]', compartment='')
Cpd2486 = Metabolite('Cpd2486', formula='', name='D-3-Hydroxyhexanoyl-ACP [c]', compartment='')
Cpd2487 = Metabolite('Cpd2487', formula='', name='(2E)-Hexenoyl-ACP [c]', compartment='')
Cpd2488 = Metabolite('Cpd2488', formula='', name='Hexanoyl-ACP [c]', compartment='')
Cpd2489 = Metabolite('Cpd2489', formula='', name='Octanoyl-ACP [c]', compartment='')
Cpd2490 = Metabolite('Cpd2490', formula='', name='Decanoyl-ACP [c]', compartment='')
Cpd2491 = Metabolite('Cpd2491', formula='', name='3-Oxododecanoyl-ACP [c]', compartment='')
Cpd2492 = Metabolite('Cpd2492', formula='', name='D-3-Hydroxydodecanoyl-ACP [c]', compartment='')
Cpd2493 = Metabolite('Cpd2493', formula='', name='UroporphyrinogenI [e]', compartment='')
Cpd2494 = Metabolite('Cpd2494', formula='', name='CoproporphyrinogenI [e]', compartment='')
Cpd2495 = Metabolite('Cpd2495', formula='', name='2,5-Diaminopyrimidine nucleoside triphosphate [c]', compartment='')
Cpd2496 = Metabolite('Cpd2496', formula='', name='2,5-Diaminopyrimidine nucleoside triphosphate [n]', compartment='')
Cpd2497 = Metabolite('Cpd2497', formula='', name='Hexadecenal [c]', compartment='')
Cpd2498 = Metabolite('Cpd2498', formula='', name='glycochenodeoxycholate [c]', compartment='')
Cpd2499 = Metabolite('Cpd2499', formula='', name='L-citrulline [e]', compartment='')
Cpd2500 = Metabolite('Cpd2500', formula='', name='Malonate [c]', compartment='')
Cpd2501 = Metabolite('Cpd2501', formula='', name='Malonate [m]', compartment='')
Cpd2502 = Metabolite('Cpd2502', formula='', name='Sodium [r]', compartment='')
Cpd2503 = Metabolite('Cpd2503', formula='', name='stearidonic acid C18:4, n-3 [l]', compartment='')
Cpd2504 = Metabolite('Cpd2504', formula='', name='stearidonic acid C18:4, n-3 [r]', compartment='')
Cpd2505 = Metabolite('Cpd2505', formula='', name='Lepidimoide [e]', compartment='')
Cpd2506 = Metabolite('Cpd2506', formula='', name='Lepidimoide [c]', compartment='')
Cpd2507 = Metabolite('Cpd2507', formula='', name='chenodeoxycholate [e]', compartment='')
Cpd2508 = Metabolite('Cpd2508', formula='', name='L-Octanoylcarnitine [c]', compartment='')
Cpd2509 = Metabolite('Cpd2509', formula='', name='glycochenodeoxycholate [e]', compartment='')
Cpd2510 = Metabolite('Cpd2510', formula='', name='taurochenodeoxycholate [c]', compartment='')
Cpd2511 = Metabolite('Cpd2511', formula='', name='taurochenodeoxycholate [e]', compartment='')
Cpd2512 = Metabolite('Cpd2512', formula='', name='L-argininium(1+) [l]', compartment='')
Cpd2513 = Metabolite('Cpd2513', formula='', name='Albumin [c]', compartment='')
Cpd2514 = Metabolite('Cpd2514', formula='', name='Antichymotrypsin [c]', compartment='')
Cpd2515 = Metabolite('Cpd2515', formula='', name='Antitrypsin [c]', compartment='')
Cpd2516 = Metabolite('Cpd2516', formula='', name='ApoB100 [c]', compartment='')
Cpd2517 = Metabolite('Cpd2517', formula='', name='L-methionine [m]', compartment='')
Cpd2518 = Metabolite('Cpd2518', formula='', name='L-histidine [m]', compartment='')
Cpd2519 = Metabolite('Cpd2519', formula='', name='L-threonine [m]', compartment='')
Cpd2520 = Metabolite('Cpd2520', formula='', name='mitoApo-ACP [m]', compartment='')
Cpd2521 = Metabolite('Cpd2521', formula='', name='ApoC1 [c]', compartment='')
Cpd2522 = Metabolite('Cpd2522', formula='', name='ApoC2 [c]', compartment='')
Cpd2523 = Metabolite('Cpd2523', formula='', name='ApoC3 [c]', compartment='')
Cpd2524 = Metabolite('Cpd2524', formula='', name='Fibrinogen [c]', compartment='')
Cpd2525 = Metabolite('Cpd2525', formula='', name='Plasminogen [c]', compartment='')
Cpd2526 = Metabolite('Cpd2526', formula='', name='Prothrombin [c]', compartment='')
Cpd2527 = Metabolite('Cpd2527', formula='', name='ApoTransferin [c]', compartment='')
Cpd2528 = Metabolite('Cpd2528', formula='', name='ApoE [c]', compartment='')
Cpd2529 = Metabolite('Cpd2529', formula='', name='ApoA1 [c]', compartment='')
Cpd2530 = Metabolite('Cpd2530', formula='', name='3-Keto-4-methylzymosterol [r]', compartment='')
Cpd2531 = Metabolite('Cpd2531', formula='', name='4alpha-Methylzymosterol [r]', compartment='')
Cpd2532 = Metabolite('Cpd2532', formula='', name='2-Oxobutanoate [m]', compartment='')
Cpd2533 = Metabolite('Cpd2533', formula='', name='Cholesterol-ester-palm [r]', compartment='')
Cpd2534 = Metabolite('Cpd2534', formula='', name='trans-Hexadec-2-enoyl-CoA [r]', compartment='')
Cpd2535 = Metabolite('Cpd2535', formula='', name='Cholesterol-ester-palmn [r]', compartment='')
Cpd2536 = Metabolite('Cpd2536', formula='', name='Stearoyl-CoA (n-C18:0CoA) [r]', compartment='')
Cpd2537 = Metabolite('Cpd2537', formula='', name='Cholesterol-ester-stea [r]', compartment='')
Cpd2538 = Metabolite('Cpd2538', formula='', name='Octadecenoyl-CoA (n-C18:1CoA) [r]', compartment='')
Cpd2539 = Metabolite('Cpd2539', formula='', name='Cholesterol-ester-ol [r]', compartment='')
Cpd2540 = Metabolite('Cpd2540', formula='', name='Cholesterol-ester-lin [r]', compartment='')
Cpd2541 = Metabolite('Cpd2541', formula='', name='gamma-linolenoyl-CoA(4-) [r]', compartment='')
Cpd2542 = Metabolite('Cpd2542', formula='', name='Cholesterol-ester-ala [r]', compartment='')
Cpd2543 = Metabolite('Cpd2543', formula='', name='Cholesterol-ester-gla [r]', compartment='')
Cpd2544 = Metabolite('Cpd2544', formula='', name='Cholesterol-ester-arach [r]', compartment='')
Cpd2545 = Metabolite('Cpd2545', formula='', name='Cholesterol-ester-palm [l]', compartment='')
Cpd2546 = Metabolite('Cpd2546', formula='', name='Hexadecanoate (n-C16:0) [l]', compartment='')
Cpd2547 = Metabolite('Cpd2547', formula='', name='Hexadecenoate (n-C16:1) [r]', compartment='')
Cpd2548 = Metabolite('Cpd2548', formula='', name='Cholesterol-ester-stea [l]', compartment='')
Cpd2549 = Metabolite('Cpd2549', formula='', name='octadecenoate (n-C18:1) [r]', compartment='')
Cpd2550 = Metabolite('Cpd2550', formula='', name='Cholesterol-ester-ol [l]', compartment='')
Cpd2551 = Metabolite('Cpd2551', formula='', name='octadecenoate (n-C18:1) [l]', compartment='')
Cpd2552 = Metabolite('Cpd2552', formula='', name='linoleate [r]', compartment='')
Cpd2553 = Metabolite('Cpd2553', formula='', name='Cholesterol-ester-lin [l]', compartment='')
Cpd2554 = Metabolite('Cpd2554', formula='', name='linoleate [l]', compartment='')
Cpd2555 = Metabolite('Cpd2555', formula='', name='alpha-linolenate [r]', compartment='')
Cpd2556 = Metabolite('Cpd2556', formula='', name='gamma-linolenate [r]', compartment='')
Cpd2557 = Metabolite('Cpd2557', formula='', name='Cholesterol-ester-arach [l]', compartment='')
Cpd2558 = Metabolite('Cpd2558', formula='', name='arachidonate [l]', compartment='')
Cpd2559 = Metabolite('Cpd2559', formula='', name='1-Acylglycerol-3P-palm [c]', compartment='')
Cpd2560 = Metabolite('Cpd2560', formula='', name='1-Acylglycerol-3P-palmn [c]', compartment='')
Cpd2561 = Metabolite('Cpd2561', formula='', name='1-Acylglycerol-3P-stea [c]', compartment='')
Cpd2562 = Metabolite('Cpd2562', formula='', name='1-Acylglycerol-3P-ol [c]', compartment='')
Cpd2563 = Metabolite('Cpd2563', formula='', name='1-Acylglycerol-3P-lin [c]', compartment='')
Cpd2564 = Metabolite('Cpd2564', formula='', name='1-Acylglycerol-3P-arach [c]', compartment='')
Cpd2565 = Metabolite('Cpd2565', formula='', name='3-Oxostearoyl-ACP [c]', compartment='')
Cpd2566 = Metabolite('Cpd2566', formula='', name='3-Hydroxystearoyl-ACP [c]', compartment='')
Cpd2567 = Metabolite('Cpd2567', formula='', name='(2E)-Octadecenoyl-ACP [c]', compartment='')
Cpd2568 = Metabolite('Cpd2568', formula='', name='Stearoyl-ACP [c]', compartment='')
Cpd2569 = Metabolite('Cpd2569', formula='', name='Chitobiose [e]', compartment='')
Cpd2570 = Metabolite('Cpd2570', formula='', name='S-(hydroxymethyl)glutathione(1-) [c]', compartment='')
Cpd2571 = Metabolite('Cpd2571', formula='', name='THF-hexaglutamate [c]', compartment='')
Cpd2572 = Metabolite('Cpd2572', formula='', name='Glycogenin-G7G1 [c]', compartment='')
Cpd2573 = Metabolite('Cpd2573', formula='', name='GM1-pool [e]', compartment='')
Cpd2574 = Metabolite('Cpd2574', formula='', name='GM2-pool [e]', compartment='')
Cpd2575 = Metabolite('Cpd2575', formula='', name='trans,cis-octadeca-2,9-dienoyl-CoA [m]', compartment='')
Cpd2576 = Metabolite('Cpd2576', formula='', name='(S)-3-hydroxyoleyleoyl-CoA [m]', compartment='')
Cpd2577 = Metabolite('Cpd2577', formula='', name='3-oxooleoyl-CoA [m]', compartment='')
Cpd2578 = Metabolite('Cpd2578', formula='', name='3-oxolaur-cis-5-enoyl-CoA [m]', compartment='')
Cpd2579 = Metabolite('Cpd2579', formula='', name='taurolithocholate [e]', compartment='')
Cpd2580 = Metabolite('Cpd2580', formula='', name='taurolithocholate [c]', compartment='')
Cpd2581 = Metabolite('Cpd2581', formula='', name='glycolithocholate [e]', compartment='')
Cpd2582 = Metabolite('Cpd2582', formula='', name='glycolithocholate [c]', compartment='')
Cpd2583 = Metabolite('Cpd2583', formula='', name='tauroursodeoxycholate [e]', compartment='')
Cpd2584 = Metabolite('Cpd2584', formula='', name='tauroursodeoxycholate [c]', compartment='')
Cpd2585 = Metabolite('Cpd2585', formula='', name='glycoursodeoxycholate [e]', compartment='')
Cpd2586 = Metabolite('Cpd2586', formula='', name='glycoursodeoxycholate [c]', compartment='')
Cpd2587 = Metabolite('Cpd2587', formula='', name='sulfochenodeoxycholate [e]', compartment='')
Cpd2588 = Metabolite('Cpd2588', formula='', name='sulfochenodeoxycholate [c]', compartment='')
Cpd2589 = Metabolite('Cpd2589', formula='', name='stearidonic acid C18:4, n-3 [e]', compartment='')
Cpd2590 = Metabolite('Cpd2590', formula='', name='GM4-pool [c]', compartment='')
Cpd2591 = Metabolite('Cpd2591', formula='', name='GM4-pool [e]', compartment='')
Cpd2592 = Metabolite('Cpd2592', formula='', name='caprate [c]', compartment='')
Cpd2593 = Metabolite('Cpd2593', formula='', name='caprate [e]', compartment='')
Cpd2594 = Metabolite('Cpd2594', formula='', name='glutathionyl-leuc4 [c]', compartment='')
Cpd2595 = Metabolite('Cpd2595', formula='', name='glutathionyl-leuc4 [e]', compartment='')
Cpd2596 = Metabolite('Cpd2596', formula='', name='S-glutathionyl-2-4-dinitrobenzene [c]', compartment='')
Cpd2597 = Metabolite('Cpd2597', formula='', name='S-glutathionyl-2-4-dinitrobenzene [e]', compartment='')
Cpd2598 = Metabolite('Cpd2598', formula='', name='S-glutathionyl-ethacrynic-acid [c]', compartment='')
Cpd2599 = Metabolite('Cpd2599', formula='', name='S-glutathionyl-ethacrynic-acid [e]', compartment='')
Cpd2600 = Metabolite('Cpd2600', formula='', name='Urate [e]', compartment='')
Cpd2601 = Metabolite('Cpd2601', formula='', name='Zinc [e]', compartment='')
Cpd2602 = Metabolite('Cpd2602', formula='', name='Zinc [c]', compartment='')
Cpd2603 = Metabolite('Cpd2603', formula='', name='propionate [e]', compartment='')
Cpd2604 = Metabolite('Cpd2604', formula='', name='all-trans-retinoate [e]', compartment='')
Cpd2605 = Metabolite('Cpd2605', formula='', name='all-trans-retinoate [c]', compartment='')
Cpd2606 = Metabolite('Cpd2606', formula='', name='(R)-3-hydroxybutyrate [c]', compartment='')
Cpd2607 = Metabolite('Cpd2607', formula='', name='(R)-3-hydroxybutyrate [e]', compartment='')
Cpd2608 = Metabolite('Cpd2608', formula='', name='lithocholate [e]', compartment='')
Cpd2609 = Metabolite('Cpd2609', formula='', name='lithocholate [c]', compartment='')
Cpd2610 = Metabolite('Cpd2610', formula='', name='ursodeoxycholate [e]', compartment='')
Cpd2611 = Metabolite('Cpd2611', formula='', name='ursodeoxycholate [c]', compartment='')
Cpd2612 = Metabolite('Cpd2612', formula='', name='sulfoglycolithocholate(2-) [e]', compartment='')
Cpd2613 = Metabolite('Cpd2613', formula='', name='sulfoglycolithocholate(2-) [c]', compartment='')
Cpd2614 = Metabolite('Cpd2614', formula='', name='2-[(4R)-4-[(1S,2S,5R,7R,10R,11S,14R,15R)-2,15-dimethyl-5-(sulfonatooxy)tetracyclo[8.7.0.0^{2,7}.0^{11,15}]heptadecan-14-yl]pentanamido]ethane-1-sulfonate [e]', compartment='')
Cpd2615 = Metabolite('Cpd2615', formula='', name='2-[(4R)-4-[(1S,2S,5R,7R,10R,11S,14R,15R)-2,15-dimethyl-5-(sulfonatooxy)tetracyclo[8.7.0.0^{2,7}.0^{11,15}]heptadecan-14-yl]pentanamido]ethane-1-sulfonate [c]', compartment='')
Cpd2616 = Metabolite('Cpd2616', formula='', name='reverse-triiodthyronine [e]', compartment='')
Cpd2617 = Metabolite('Cpd2617', formula='', name='reverse-triiodthyronine [c]', compartment='')
Cpd2618 = Metabolite('Cpd2618', formula='', name='thromboxane-b2 [e]', compartment='')
Cpd2619 = Metabolite('Cpd2619', formula='', name='thromboxane-b2 [c]', compartment='')
Cpd2620 = Metabolite('Cpd2620', formula='', name='leukotriene E4(1-) [e]', compartment='')
Cpd2621 = Metabolite('Cpd2621', formula='', name='leukotriene E4(1-) [c]', compartment='')
Cpd2622 = Metabolite('Cpd2622', formula='', name='Leukotriene D4 [e]', compartment='')
Cpd2623 = Metabolite('Cpd2623', formula='', name='20-hydroxy-arachidonate [e]', compartment='')
Cpd2624 = Metabolite('Cpd2624', formula='', name='20-hydroxy-arachidonate [c]', compartment='')
Cpd2625 = Metabolite('Cpd2625', formula='', name='leukotriene A4(1-) [e]', compartment='')
Cpd2626 = Metabolite('Cpd2626', formula='', name='leukotriene B4(1-) [e]', compartment='')
Cpd2627 = Metabolite('Cpd2627', formula='', name='leukotriene F4 [e]', compartment='')
Cpd2628 = Metabolite('Cpd2628', formula='', name='leukotriene F4 [c]', compartment='')
Cpd2629 = Metabolite('Cpd2629', formula='', name='prostaglandin A1(1-) [e]', compartment='')
Cpd2630 = Metabolite('Cpd2630', formula='', name='prostaglandin A1(1-) [c]', compartment='')
Cpd2631 = Metabolite('Cpd2631', formula='', name='prostaglandin-a2 [e]', compartment='')
Cpd2632 = Metabolite('Cpd2632', formula='', name='prostaglandin-a2 [c]', compartment='')
Cpd2633 = Metabolite('Cpd2633', formula='', name='prostaglandin-b1 [e]', compartment='')
Cpd2634 = Metabolite('Cpd2634', formula='', name='prostaglandin-b1 [c]', compartment='')
Cpd2635 = Metabolite('Cpd2635', formula='', name='prostaglandin-b2 [e]', compartment='')
Cpd2636 = Metabolite('Cpd2636', formula='', name='prostaglandin-b2 [c]', compartment='')
Cpd2637 = Metabolite('Cpd2637', formula='', name='prostaglandin C1(1-) [e]', compartment='')
Cpd2638 = Metabolite('Cpd2638', formula='', name='prostaglandin C1(1-) [c]', compartment='')
Cpd2639 = Metabolite('Cpd2639', formula='', name='prostaglandin-c2 [e]', compartment='')
Cpd2640 = Metabolite('Cpd2640', formula='', name='prostaglandin-c2 [c]', compartment='')
Cpd2641 = Metabolite('Cpd2641', formula='', name='prostaglandin-d1 [e]', compartment='')
Cpd2642 = Metabolite('Cpd2642', formula='', name='prostaglandin-d1 [c]', compartment='')
Cpd2643 = Metabolite('Cpd2643', formula='', name='prostaglandin-d3 [e]', compartment='')
Cpd2644 = Metabolite('Cpd2644', formula='', name='prostaglandin-d3 [c]', compartment='')
Cpd2645 = Metabolite('Cpd2645', formula='', name='prostaglandin-e3 [e]', compartment='')
Cpd2646 = Metabolite('Cpd2646', formula='', name='prostaglandin-e3 [c]', compartment='')
Cpd2647 = Metabolite('Cpd2647', formula='', name='prostaglandin-f1alpha [e]', compartment='')
Cpd2648 = Metabolite('Cpd2648', formula='', name='prostaglandin-f1alpha [c]', compartment='')
Cpd2649 = Metabolite('Cpd2649', formula='', name='prostaglandin-f2beta [e]', compartment='')
Cpd2650 = Metabolite('Cpd2650', formula='', name='prostaglandin-f2beta [c]', compartment='')
Cpd2651 = Metabolite('Cpd2651', formula='', name='prostaglandin-g2 [e]', compartment='')
Cpd2652 = Metabolite('Cpd2652', formula='', name='prostaglandin-g2 [c]', compartment='')
Cpd2653 = Metabolite('Cpd2653', formula='', name='prostaglandin H2(1-) [e]', compartment='')
Cpd2654 = Metabolite('Cpd2654', formula='', name='prostaglandin I2(1-) [e]', compartment='')
Cpd2655 = Metabolite('Cpd2655', formula='', name='prostaglandin I2(1-) [c]', compartment='')
Cpd2656 = Metabolite('Cpd2656', formula='', name='cis-aconitate(3-) [c]', compartment='')
Cpd2657 = Metabolite('Cpd2657', formula='', name='oxalate(2-) [m]', compartment='')
Cpd2658 = Metabolite('Cpd2658', formula='', name='L-Oleoylcarnitine [c]', compartment='')
Cpd2659 = Metabolite('Cpd2659', formula='', name='L-Oleoylcarnitine [m]', compartment='')
Cpd2660 = Metabolite('Cpd2660', formula='', name='O-Propanoylcarnitine [c]', compartment='')
Cpd2661 = Metabolite('Cpd2661', formula='', name='O-Propanoylcarnitine [m]', compartment='')
Cpd2662 = Metabolite('Cpd2662', formula='', name='O-Butanoylcarnitine [c]', compartment='')
Cpd2663 = Metabolite('Cpd2663', formula='', name='O-Butanoylcarnitine [m]', compartment='')
Cpd2664 = Metabolite('Cpd2664', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestanoate [c]', compartment='')
Cpd2665 = Metabolite('Cpd2665', formula='', name='Phosphatidyl-N-methylethanolamine [m]', compartment='')
Cpd2666 = Metabolite('Cpd2666', formula='', name='Phosphatidyl-N-dimethylethanolamine [m]', compartment='')
Cpd2667 = Metabolite('Cpd2667', formula='', name='Phosphatidyl-N-methylethanolamine [r]', compartment='')
Cpd2668 = Metabolite('Cpd2668', formula='', name='Phosphatidyl-N-dimethylethanolamine [r]', compartment='')
Cpd2669 = Metabolite('Cpd2669', formula='', name='octadecanoate (n-C18:0) [m]', compartment='')
Cpd2670 = Metabolite('Cpd2670', formula='', name='octadecanoate (n-C18:0) [x]', compartment='')
Cpd2671 = Metabolite('Cpd2671', formula='', name='L-gulonate [c]', compartment='')
Cpd2672 = Metabolite('Cpd2672', formula='', name='Reduced thioredoxin [n]', compartment='')
Cpd2673 = Metabolite('Cpd2673', formula='', name='Oxidized thioredoxin [n]', compartment='')
Cpd2674 = Metabolite('Cpd2674', formula='', name='(3S)-3-hydroxylinoleoyl-CoA [m]', compartment='')
Cpd2675 = Metabolite('Cpd2675', formula='', name='3-oxolinoleoyl-CoA [m]', compartment='')
Cpd2676 = Metabolite('Cpd2676', formula='', name='(3S)-3-hydroxylinoleoyl-CoA [x]', compartment='')
Cpd2677 = Metabolite('Cpd2677', formula='', name='3-oxolinoleoyl-CoA [x]', compartment='')
Cpd2678 = Metabolite('Cpd2678', formula='', name='3-oxoeicosanoyl-CoA [c]', compartment='')
Cpd2679 = Metabolite('Cpd2679', formula='', name='trans-eicos-2-enoyl-CoA [c]', compartment='')
Cpd2680 = Metabolite('Cpd2680', formula='', name='3-oxodocosanoyl-CoA [c]', compartment='')
Cpd2681 = Metabolite('Cpd2681', formula='', name='Malonyl-CoA [e]', compartment='')
Cpd2682 = Metabolite('Cpd2682', formula='', name='arachidyl coenzyme A [e]', compartment='')
Cpd2683 = Metabolite('Cpd2683', formula='', name='Coenzyme A [e]', compartment='')
Cpd2684 = Metabolite('Cpd2684', formula='', name='3-oxodocosanoyl-CoA [e]', compartment='')
Cpd2685 = Metabolite('Cpd2685', formula='', name='trans-docos-2-enoyl-CoA [n]', compartment='')
Cpd2686 = Metabolite('Cpd2686', formula='', name='docosanoyl-CoA [n]', compartment='')
Cpd2687 = Metabolite('Cpd2687', formula='', name='Malonyl-CoA [n]', compartment='')
Cpd2688 = Metabolite('Cpd2688', formula='', name='carbon dioxide [n]', compartment='')
Cpd2689 = Metabolite('Cpd2689', formula='', name='3-oxotetracosanoyl-CoA [n]', compartment='')
Cpd2690 = Metabolite('Cpd2690', formula='', name='arachidate [m]', compartment='')
Cpd2691 = Metabolite('Cpd2691', formula='', name='arachidate [x]', compartment='')
Cpd2692 = Metabolite('Cpd2692', formula='', name='docosanoyl-CoA [c]', compartment='')
Cpd2693 = Metabolite('Cpd2693', formula='', name='behenate [c]', compartment='')
Cpd2694 = Metabolite('Cpd2694', formula='', name='docosanoyl-CoA [m]', compartment='')
Cpd2695 = Metabolite('Cpd2695', formula='', name='behenate [m]', compartment='')
Cpd2696 = Metabolite('Cpd2696', formula='', name='docosanoyl-CoA [x]', compartment='')
Cpd2697 = Metabolite('Cpd2697', formula='', name='behenate [x]', compartment='')
Cpd2698 = Metabolite('Cpd2698', formula='', name='tetracosanoyl-CoA [c]', compartment='')
Cpd2699 = Metabolite('Cpd2699', formula='', name='tetracosanoyl-CoA [m]', compartment='')
Cpd2700 = Metabolite('Cpd2700', formula='', name='lignocerate [m]', compartment='')
Cpd2701 = Metabolite('Cpd2701', formula='', name='lignocerate [x]', compartment='')
Cpd2702 = Metabolite('Cpd2702', formula='', name='Palmitoyl-CoA (n-C16:0CoA) [l]', compartment='')
Cpd2703 = Metabolite('Cpd2703', formula='', name='Malonyl-CoA [l]', compartment='')
Cpd2704 = Metabolite('Cpd2704', formula='', name='carbon dioxide [l]', compartment='')
Cpd2705 = Metabolite('Cpd2705', formula='', name='3-Oxooctadecanoyl-CoA [l]', compartment='')
Cpd2706 = Metabolite('Cpd2706', formula='', name='3-Oxooctadecanoyl-CoA [r]', compartment='')
Cpd2707 = Metabolite('Cpd2707', formula='', name='3-hydroxyoctadecanoyl-CoA [r]', compartment='')
Cpd2708 = Metabolite('Cpd2708', formula='', name='3-hydroxyoctadecanoyl-CoA [n]', compartment='')
Cpd2709 = Metabolite('Cpd2709', formula='', name='trans-Octadec-2-enoyl-CoA [n]', compartment='')
Cpd2710 = Metabolite('Cpd2710', formula='', name='Stearoyl-CoA (n-C18:0CoA) [n]', compartment='')
Cpd2711 = Metabolite('Cpd2711', formula='', name='spermine monoaldehyde [c]', compartment='')
Cpd2712 = Metabolite('Cpd2712', formula='', name='spermine(4+) [e]', compartment='')
Cpd2713 = Metabolite('Cpd2713', formula='', name='spermine monoaldehyde [e]', compartment='')
Cpd2714 = Metabolite('Cpd2714', formula='', name='spermine(4+) [x]', compartment='')
Cpd2715 = Metabolite('Cpd2715', formula='', name='spermine monoaldehyde [x]', compartment='')
Cpd2716 = Metabolite('Cpd2716', formula='', name='spermidine monoaldehyde 2 [c]', compartment='')
Cpd2717 = Metabolite('Cpd2717', formula='', name='Spermidine [e]', compartment='')
Cpd2718 = Metabolite('Cpd2718', formula='', name='spermidine monoaldehyde 2 [e]', compartment='')
Cpd2719 = Metabolite('Cpd2719', formula='', name='Spermidine [x]', compartment='')
Cpd2720 = Metabolite('Cpd2720', formula='', name='spermidine monoaldehyde 2 [x]', compartment='')
Cpd2721 = Metabolite('Cpd2721', formula='', name='spermidine dialdehyde [c]', compartment='')
Cpd2722 = Metabolite('Cpd2722', formula='', name='spermidine dialdehyde [e]', compartment='')
Cpd2723 = Metabolite('Cpd2723', formula='', name='spermidine dialdehyde [x]', compartment='')
Cpd2724 = Metabolite('Cpd2724', formula='', name='hypothiocyanite [c]', compartment='')
Cpd2725 = Metabolite('Cpd2725', formula='', name='thiocyanate [e]', compartment='')
Cpd2726 = Metabolite('Cpd2726', formula='', name='hypothiocyanite [e]', compartment='')
Cpd2727 = Metabolite('Cpd2727', formula='', name='thiocyanate [l]', compartment='')
Cpd2728 = Metabolite('Cpd2728', formula='', name='hypothiocyanite [l]', compartment='')
Cpd2729 = Metabolite('Cpd2729', formula='', name='thiocyanate [m]', compartment='')
Cpd2730 = Metabolite('Cpd2730', formula='', name='hypothiocyanite [m]', compartment='')
Cpd2731 = Metabolite('Cpd2731', formula='', name='thiocyanate [n]', compartment='')
Cpd2732 = Metabolite('Cpd2732', formula='', name='hypothiocyanite [n]', compartment='')
Cpd2733 = Metabolite('Cpd2733', formula='', name='spermine dialdehyde [c]', compartment='')
Cpd2734 = Metabolite('Cpd2734', formula='', name='spermine dialdehyde [e]', compartment='')
Cpd2735 = Metabolite('Cpd2735', formula='', name='spermine dialdehyde [x]', compartment='')
Cpd2736 = Metabolite('Cpd2736', formula='', name='spermidine monoaldehyde 1 [c]', compartment='')
Cpd2737 = Metabolite('Cpd2737', formula='', name='spermidine monoaldehyde 1 [e]', compartment='')
Cpd2738 = Metabolite('Cpd2738', formula='', name='spermidine monoaldehyde 1 [x]', compartment='')
Cpd2739 = Metabolite('Cpd2739', formula='', name='4a-hydroxytetrahydrobiopterin [n]', compartment='')
Cpd2740 = Metabolite('Cpd2740', formula='', name='quinonoid dihydrobiopterin [n]', compartment='')
Cpd2741 = Metabolite('Cpd2741', formula='', name='3,3-diiodo-L-thyronine [c]', compartment='')
Cpd2742 = Metabolite('Cpd2742', formula='', name='3,3-diiodo-L-thyronine 4-O-sulfate [c]', compartment='')
Cpd2743 = Metabolite('Cpd2743', formula='', name='Maltotetraose [e]', compartment='')
Cpd2744 = Metabolite('Cpd2744', formula='', name='3,5-diiodo-L-thyronine [c]', compartment='')
Cpd2745 = Metabolite('Cpd2745', formula='', name='3,5-diiodo-L-thyronine-beta-D-glucuronoside [c]', compartment='')
Cpd2746 = Metabolite('Cpd2746', formula='', name='3,5-diiodo-L-thyronine [r]', compartment='')
Cpd2747 = Metabolite('Cpd2747', formula='', name='3,5-diiodo-L-thyronine-beta-D-glucuronoside [r]', compartment='')
Cpd2748 = Metabolite('Cpd2748', formula='', name='3,5,3,5-tetraiodo-L-thyronine-beta-D-glucuronoside [c]', compartment='')
Cpd2749 = Metabolite('Cpd2749', formula='', name='L-thyroxine [r]', compartment='')
Cpd2750 = Metabolite('Cpd2750', formula='', name='3,5,3,5-tetraiodo-L-thyronine-beta-D-glucuronoside [r]', compartment='')
Cpd2751 = Metabolite('Cpd2751', formula='', name='3,3,5 - triiodothyronine [c]', compartment='')
Cpd2752 = Metabolite('Cpd2752', formula='', name='3,3,5-triiodo-L-thyronine-beta-D-glucuronoside [c]', compartment='')
Cpd2753 = Metabolite('Cpd2753', formula='', name='3,3,5 - triiodothyronine [r]', compartment='')
Cpd2754 = Metabolite('Cpd2754', formula='', name='3,3,5-triiodo-L-thyronine-beta-D-glucuronoside [r]', compartment='')
Cpd2755 = Metabolite('Cpd2755', formula='', name='3,5,3-triiodo-L-thyronine-beta-D-glucuronoside [c]', compartment='')
Cpd2756 = Metabolite('Cpd2756', formula='', name='3,3,5-triiodo-L-thyronine [r]', compartment='')
Cpd2757 = Metabolite('Cpd2757', formula='', name='3,5,3-triiodo-L-thyronine-beta-D-glucuronoside [r]', compartment='')
Cpd2758 = Metabolite('Cpd2758', formula='', name='3,3-diiodo-L-thyronine-beta-D-glucuronoside [c]', compartment='')
Cpd2759 = Metabolite('Cpd2759', formula='', name='3,3-diiodo-L-thyronine [r]', compartment='')
Cpd2760 = Metabolite('Cpd2760', formula='', name='3,3-diiodo-L-thyronine-beta-D-glucuronoside [r]', compartment='')
Cpd2761 = Metabolite('Cpd2761', formula='', name='3,5,3,5-tetraiodothyroacetate [c]', compartment='')
Cpd2762 = Metabolite('Cpd2762', formula='', name='3,5,3,5-tetraiodothyroacetate-beta-D-glucuronoside [c]', compartment='')
Cpd2763 = Metabolite('Cpd2763', formula='', name='3,5,3,5-tetraiodothyroacetate [r]', compartment='')
Cpd2764 = Metabolite('Cpd2764', formula='', name='3,5,3,5-tetraiodothyroacetate-beta-D-glucuronoside [r]', compartment='')
Cpd2765 = Metabolite('Cpd2765', formula='', name='3,3,5-triiodothyroacetate [c]', compartment='')
Cpd2766 = Metabolite('Cpd2766', formula='', name='3,3,5-triiodothyroacetate-beta-D-glucuronoside [c]', compartment='')
Cpd2767 = Metabolite('Cpd2767', formula='', name='3,3,5-triiodothyroacetate [r]', compartment='')
Cpd2768 = Metabolite('Cpd2768', formula='', name='3,3,5-triiodothyroacetate-beta-D-glucuronoside [r]', compartment='')
Cpd2769 = Metabolite('Cpd2769', formula='', name='maltopentaose [e]', compartment='')
Cpd2770 = Metabolite('Cpd2770', formula='', name='3,5,3-triiodothyroacetate [c]', compartment='')
Cpd2771 = Metabolite('Cpd2771', formula='', name='3,5,3-triiodothyroacetate-beta-D-glucuronoside [c]', compartment='')
Cpd2772 = Metabolite('Cpd2772', formula='', name='3,5,3-triiodothyroacetate [r]', compartment='')
Cpd2773 = Metabolite('Cpd2773', formula='', name='3,5,3-triiodothyroacetate-beta-D-glucuronoside [r]', compartment='')
Cpd2774 = Metabolite('Cpd2774', formula='', name='3,5-diiodothyroacetate [c]', compartment='')
Cpd2775 = Metabolite('Cpd2775', formula='', name='3,5-diiodothyroacetate-beta-D-glucuronoside [c]', compartment='')
Cpd2776 = Metabolite('Cpd2776', formula='', name='3,5-diiodothyroacetate [r]', compartment='')
Cpd2777 = Metabolite('Cpd2777', formula='', name='3,5-diiodothyroacetate-beta-D-glucuronoside [r]', compartment='')
Cpd2778 = Metabolite('Cpd2778', formula='', name='Maltohexaose [e]', compartment='')
Cpd2779 = Metabolite('Cpd2779', formula='', name='beta-casomorphin [c]', compartment='')
Cpd2780 = Metabolite('Cpd2780', formula='', name='beta-casomorphin (1-6) [c]', compartment='')
Cpd2781 = Metabolite('Cpd2781', formula='', name='beta-casomorphin [e]', compartment='')
Cpd2782 = Metabolite('Cpd2782', formula='', name='beta-casomorphin (1-6) [e]', compartment='')
Cpd2783 = Metabolite('Cpd2783', formula='', name='neocasomorphin [c]', compartment='')
Cpd2784 = Metabolite('Cpd2784', formula='', name='neocasomorphin (1-5) [c]', compartment='')
Cpd2785 = Metabolite('Cpd2785', formula='', name='neocasomorphin [e]', compartment='')
Cpd2786 = Metabolite('Cpd2786', formula='', name='neocasomorphin (1-5) [e]', compartment='')
Cpd2787 = Metabolite('Cpd2787', formula='', name='apelin-13 [c]', compartment='')
Cpd2788 = Metabolite('Cpd2788', formula='', name='apelin (1-12) [c]', compartment='')
Cpd2789 = Metabolite('Cpd2789', formula='', name='apelin-13 [e]', compartment='')
Cpd2790 = Metabolite('Cpd2790', formula='', name='apelin (1-12) [e]', compartment='')
Cpd2791 = Metabolite('Cpd2791', formula='', name='maltoheptaose [e]', compartment='')
Cpd2792 = Metabolite('Cpd2792', formula='', name='maltodecaose [e]', compartment='')
Cpd2793 = Metabolite('Cpd2793', formula='', name='maltononaose [e]', compartment='')
Cpd2794 = Metabolite('Cpd2794', formula='', name='cyanosulfurous acid anion [c]', compartment='')
Cpd2795 = Metabolite('Cpd2795', formula='', name='Cyanate [c]', compartment='')
Cpd2796 = Metabolite('Cpd2796', formula='', name='cyanosulfurous acid anion [e]', compartment='')
Cpd2797 = Metabolite('Cpd2797', formula='', name='Sulfite [e]', compartment='')
Cpd2798 = Metabolite('Cpd2798', formula='', name='Cyanate [e]', compartment='')
Cpd2799 = Metabolite('Cpd2799', formula='', name='cyanosulfurous acid anion [l]', compartment='')
Cpd2800 = Metabolite('Cpd2800', formula='', name='Cyanate [l]', compartment='')
Cpd2801 = Metabolite('Cpd2801', formula='', name='Sulfite [l]', compartment='')
Cpd2802 = Metabolite('Cpd2802', formula='', name='cyanosulfurous acid anion [n]', compartment='')
Cpd2803 = Metabolite('Cpd2803', formula='', name='Cyanate [n]', compartment='')
Cpd2804 = Metabolite('Cpd2804', formula='', name='Sulfite [n]', compartment='')
Cpd2805 = Metabolite('Cpd2805', formula='', name='Neurotensin [m]', compartment='')
Cpd2806 = Metabolite('Cpd2806', formula='', name='neurotensin 1-10 [m]', compartment='')
Cpd2807 = Metabolite('Cpd2807', formula='', name='neurotensin 11-13 [m]', compartment='')
Cpd2808 = Metabolite('Cpd2808', formula='', name='17alpha-hydroxyprogesterone [m]', compartment='')
Cpd2809 = Metabolite('Cpd2809', formula='', name='androst-4-ene-3,17-dione [m]', compartment='')
Cpd2810 = Metabolite('Cpd2810', formula='', name='acetate [r]', compartment='')
Cpd2811 = Metabolite('Cpd2811', formula='', name='17alpha-hydroxypregnenolone [m]', compartment='')
Cpd2812 = Metabolite('Cpd2812', formula='', name='3beta-hydroxyandrost-5-en-17-one [m]', compartment='')
Cpd2813 = Metabolite('Cpd2813', formula='', name='kynurenate [c]', compartment='')
Cpd2814 = Metabolite('Cpd2814', formula='', name='L-kynurenine [m]', compartment='')
Cpd2815 = Metabolite('Cpd2815', formula='', name='kynurenate [m]', compartment='')
Cpd2816 = Metabolite('Cpd2816', formula='', name='23S,25-dihydroxyvitamin D3 [m]', compartment='')
Cpd2817 = Metabolite('Cpd2817', formula='', name='23S,25,26-trihydroxyvitamin D3 [m]', compartment='')
Cpd2818 = Metabolite('Cpd2818', formula='', name='25-hydroxyvitamin D3-26,23-lactol [m]', compartment='')
Cpd2819 = Metabolite('Cpd2819', formula='', name='25-hydroxyvitamin D3-26,23-lactone [m]', compartment='')
Cpd2820 = Metabolite('Cpd2820', formula='', name='1-phosphatidyl-1D-myo-inositol(1-) [g]', compartment='')
Cpd2821 = Metabolite('Cpd2821', formula='', name='ATP [g]', compartment='')
Cpd2822 = Metabolite('Cpd2822', formula='', name='1-phosphatidyl-1D-myo-inositol 5-phosphate(3-) [g]', compartment='')
Cpd2823 = Metabolite('Cpd2823', formula='', name='ADP [g]', compartment='')
Cpd2824 = Metabolite('Cpd2824', formula='', name='phosphatidylinositol 4,5-bisphosphate [m]', compartment='')
Cpd2825 = Metabolite('Cpd2825', formula='', name='1-phosphatidyl-1D-myo-inositol 5-phosphate(3-) [m]', compartment='')
Cpd2826 = Metabolite('Cpd2826', formula='', name='linoleate [m]', compartment='')
Cpd2827 = Metabolite('Cpd2827', formula='', name='linoleate [x]', compartment='')
Cpd2828 = Metabolite('Cpd2828', formula='', name='trans,cis,cis-2,9,12-octadecatrienoyl-CoA [m]', compartment='')
Cpd2829 = Metabolite('Cpd2829', formula='', name='trans,cis,cis-2,9,12-octadecatrienoyl-CoA [x]', compartment='')
Cpd2830 = Metabolite('Cpd2830', formula='', name='cis,cis-palmito-7,10-dienoyl-CoA [m]', compartment='')
Cpd2831 = Metabolite('Cpd2831', formula='', name='trans,cis,cis-2,7,10-hexadecatrienoyl-CoA [m]', compartment='')
Cpd2832 = Metabolite('Cpd2832', formula='', name='cis,cis-palmito-7,10-dienoyl-CoA [x]', compartment='')
Cpd2833 = Metabolite('Cpd2833', formula='', name='trans,cis,cis-2,7,10-hexadecatrienoyl-CoA [x]', compartment='')
Cpd2834 = Metabolite('Cpd2834', formula='', name='cis,cis-myristo-5,8-dienoyl-CoA [m]', compartment='')
Cpd2835 = Metabolite('Cpd2835', formula='', name='trans-2-cis,cis-5,8-tetradecatrienoyl-CoA [m]', compartment='')
Cpd2836 = Metabolite('Cpd2836', formula='', name='cis,cis-myristo-5,8-dienoyl-CoA [x]', compartment='')
Cpd2837 = Metabolite('Cpd2837', formula='', name='trans-2-cis,cis-5,8-tetradecatrienoyl-CoA [x]', compartment='')
Cpd2838 = Metabolite('Cpd2838', formula='', name='cis-deca-4-enoyl-CoA [m]', compartment='')
Cpd2839 = Metabolite('Cpd2839', formula='', name='trans,cis-deca-2,4-dienoyl-CoA [m]', compartment='')
Cpd2840 = Metabolite('Cpd2840', formula='', name='cis-deca-4-enoyl-CoA [x]', compartment='')
Cpd2841 = Metabolite('Cpd2841', formula='', name='trans,cis-deca-2,4-dienoyl-CoA [x]', compartment='')
Cpd2842 = Metabolite('Cpd2842', formula='', name='trans,cis-Lauro-2,6-dienoyl-CoA [m]', compartment='')
Cpd2843 = Metabolite('Cpd2843', formula='', name='(3S)-3-hydroxydodec-cis-6-enoyl-CoA [m]', compartment='')
Cpd2844 = Metabolite('Cpd2844', formula='', name='trans,cis-Lauro-2,6-dienoyl-CoA [x]', compartment='')
Cpd2845 = Metabolite('Cpd2845', formula='', name='(3S)-3-hydroxydodec-cis-6-enoyl-CoA [x]', compartment='')
Cpd2846 = Metabolite('Cpd2846', formula='', name='3(S)-hydroxy-5Z,8Z-tetradecadienoyl-CoA [m]', compartment='')
Cpd2847 = Metabolite('Cpd2847', formula='', name='3(S)-hydroxy-5Z,8Z-tetradecadienoyl-CoA [x]', compartment='')
Cpd2848 = Metabolite('Cpd2848', formula='', name='(3S)-3-hydroxy-cis,cis-palmito-7,10-dienoyl-CoA [m]', compartment='')
Cpd2849 = Metabolite('Cpd2849', formula='', name='(3S)-3-hydroxy-cis,cis-palmito-7,10-dienoyl-CoA [x]', compartment='')
Cpd2850 = Metabolite('Cpd2850', formula='', name='3-oxo-cis,cis-7,10-hexadecadienoyl-CoA [m]', compartment='')
Cpd2851 = Metabolite('Cpd2851', formula='', name='3-oxo-cis,cis-7,10-hexadecadienoyl-CoA [x]', compartment='')
Cpd2852 = Metabolite('Cpd2852', formula='', name='3-oxo-cis,cis-5,8-tetradecadienoyl-CoA [m]', compartment='')
Cpd2853 = Metabolite('Cpd2853', formula='', name='3-oxo-cis,cis-5,8-tetradecadienoyl-CoA [x]', compartment='')
Cpd2854 = Metabolite('Cpd2854', formula='', name='3-oxolaur-6-cis-enoyl-CoA [m]', compartment='')
Cpd2855 = Metabolite('Cpd2855', formula='', name='3-oxolaur-6-cis-enoyl-CoA [x]', compartment='')
Cpd2856 = Metabolite('Cpd2856', formula='', name='cis,cis-3,6-Dodecadienoyl-CoA [m]', compartment='')
Cpd2857 = Metabolite('Cpd2857', formula='', name='cis,cis-3,6-Dodecadienoyl-CoA [x]', compartment='')
Cpd2858 = Metabolite('Cpd2858', formula='', name='N(1)-acetylspermidinium(2+) [x]', compartment='')
Cpd2859 = Metabolite('Cpd2859', formula='', name='Putrescine [x]', compartment='')
Cpd2860 = Metabolite('Cpd2860', formula='', name='acetamidopropanal [x]', compartment='')
Cpd2861 = Metabolite('Cpd2861', formula='', name='N(1),N(12)-diacetylsperminium(2+) [x]', compartment='')
Cpd2862 = Metabolite('Cpd2862', formula='', name='N(1)-acetylsperminium(3+) [x]', compartment='')
Cpd2863 = Metabolite('Cpd2863', formula='', name='4-hydroxyestrone [r]', compartment='')
Cpd2864 = Metabolite('Cpd2864', formula='', name='2-Hydroxyestrone [r]', compartment='')
Cpd2865 = Metabolite('Cpd2865', formula='', name='4-hydroxy-17beta-estradiol [r]', compartment='')
Cpd2866 = Metabolite('Cpd2866', formula='', name='17beta-estradiol-2,3-quinone [m]', compartment='')
Cpd2867 = Metabolite('Cpd2867', formula='', name='2-hydroxy-17beta-estradiol-4-S-glutathione [m]', compartment='')
Cpd2868 = Metabolite('Cpd2868', formula='', name='17beta-estradiol-2,3-quinone [r]', compartment='')
Cpd2869 = Metabolite('Cpd2869', formula='', name='2-hydroxy-17beta-estradiol-4-S-glutathione [r]', compartment='')
Cpd2870 = Metabolite('Cpd2870', formula='', name='17beta-estradiol-2,3-quinone [x]', compartment='')
Cpd2871 = Metabolite('Cpd2871', formula='', name='Reduced glutathione [x]', compartment='')
Cpd2872 = Metabolite('Cpd2872', formula='', name='2-hydroxy-17beta-estradiol-4-S-glutathione [x]', compartment='')
Cpd2873 = Metabolite('Cpd2873', formula='', name='hydrogen cyanide [m]', compartment='')
Cpd2874 = Metabolite('Cpd2874', formula='', name='O2-4a-cyclic-tetrahydrobiopterin [n]', compartment='')
Cpd2875 = Metabolite('Cpd2875', formula='', name='5-hydroxytryptophol [m]', compartment='')
Cpd2876 = Metabolite('Cpd2876', formula='', name='20-hydroxycholesterol [m]', compartment='')
Cpd2877 = Metabolite('Cpd2877', formula='', name='7alpha-hydroxycholest-4-en-3-one [m]', compartment='')
Cpd2878 = Metabolite('Cpd2878', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha-triol [r]', compartment='')
Cpd2879 = Metabolite('Cpd2879', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,25-tetrol [r]', compartment='')
Cpd2880 = Metabolite('Cpd2880', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,23,25-pentol [r]', compartment='')
Cpd2881 = Metabolite('Cpd2881', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,24R,25-pentol [r]', compartment='')
Cpd2882 = Metabolite('Cpd2882', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,25,27-pentol [r]', compartment='')
Cpd2883 = Metabolite('Cpd2883', formula='', name='estrone-2,3-quinone [m]', compartment='')
Cpd2884 = Metabolite('Cpd2884', formula='', name='2-hydroxyestrone-1-S-glutathione [m]', compartment='')
Cpd2885 = Metabolite('Cpd2885', formula='', name='estrone-2,3-quinone [r]', compartment='')
Cpd2886 = Metabolite('Cpd2886', formula='', name='2-hydroxyestrone-1-S-glutathione [r]', compartment='')
Cpd2887 = Metabolite('Cpd2887', formula='', name='estrone-2,3-quinone [x]', compartment='')
Cpd2888 = Metabolite('Cpd2888', formula='', name='2-hydroxyestrone-1-S-glutathione [x]', compartment='')
Cpd2889 = Metabolite('Cpd2889', formula='', name='17beta-estradiol-3,4-quinone [m]', compartment='')
Cpd2890 = Metabolite('Cpd2890', formula='', name='4-hydroxy-17beta-estradiol-2-S-glutathione [m]', compartment='')
Cpd2891 = Metabolite('Cpd2891', formula='', name='17beta-estradiol-3,4-quinone [r]', compartment='')
Cpd2892 = Metabolite('Cpd2892', formula='', name='4-hydroxy-17beta-estradiol-2-S-glutathione [r]', compartment='')
Cpd2893 = Metabolite('Cpd2893', formula='', name='17beta-estradiol-3,4-quinone [x]', compartment='')
Cpd2894 = Metabolite('Cpd2894', formula='', name='4-hydroxy-17beta-estradiol-2-S-glutathione [x]', compartment='')
Cpd2895 = Metabolite('Cpd2895', formula='', name='estrone-3,4-quinone [m]', compartment='')
Cpd2896 = Metabolite('Cpd2896', formula='', name='4-hydroxyestrone-2-S-glutathione [m]', compartment='')
Cpd2897 = Metabolite('Cpd2897', formula='', name='estrone-3,4-quinone [r]', compartment='')
Cpd2898 = Metabolite('Cpd2898', formula='', name='4-hydroxyestrone-2-S-glutathione [r]', compartment='')
Cpd2899 = Metabolite('Cpd2899', formula='', name='estrone-3,4-quinone [x]', compartment='')
Cpd2900 = Metabolite('Cpd2900', formula='', name='4-hydroxyestrone-2-S-glutathione [x]', compartment='')
Cpd2901 = Metabolite('Cpd2901', formula='', name='12-oxo-leukotriene B4 [m]', compartment='')
Cpd2902 = Metabolite('Cpd2902', formula='', name='12-oxo-c-LTB3 [m]', compartment='')
Cpd2903 = Metabolite('Cpd2903', formula='', name='12-oxo-leukotriene B4 [r]', compartment='')
Cpd2904 = Metabolite('Cpd2904', formula='', name='12-oxo-c-LTB3 [r]', compartment='')
Cpd2905 = Metabolite('Cpd2905', formula='', name='12-oxo-leukotriene B4 [x]', compartment='')
Cpd2906 = Metabolite('Cpd2906', formula='', name='12-oxo-c-LTB3 [x]', compartment='')
Cpd2907 = Metabolite('Cpd2907', formula='', name='12-oxo-c-LTB3 [c]', compartment='')
Cpd2908 = Metabolite('Cpd2908', formula='', name='10,11-dihydro-12R-hydroxy-leukotriene C4 [c]', compartment='')
Cpd2909 = Metabolite('Cpd2909', formula='', name='10,11-dihydro-12R-hydroxy-leukotriene C4 [m]', compartment='')
Cpd2910 = Metabolite('Cpd2910', formula='', name='10,11-dihydro-12R-hydroxy-leukotriene C4 [x]', compartment='')
Cpd2911 = Metabolite('Cpd2911', formula='', name='5beta-cholestane-3alpha,7alpha,27-triol [m]', compartment='')
Cpd2912 = Metabolite('Cpd2912', formula='', name='3beta-hydroxy-5-cholestenal [m]', compartment='')
Cpd2913 = Metabolite('Cpd2913', formula='', name='3beta-hydroxy-5-cholestene-27-oate [m]', compartment='')
Cpd2914 = Metabolite('Cpd2914', formula='', name='3beta,7alpha-dihydroxy-5-cholestenoate [m]', compartment='')
Cpd2915 = Metabolite('Cpd2915', formula='', name='7alpha-hydroxy-3-oxo-4-cholestenoic acid anion [m]', compartment='')
Cpd2916 = Metabolite('Cpd2916', formula='', name='choloyl-CoA(4-) [m]', compartment='')
Cpd2917 = Metabolite('Cpd2917', formula='', name='Cholate [m]', compartment='')
Cpd2918 = Metabolite('Cpd2918', formula='', name='Cholate [x]', compartment='')
Cpd2919 = Metabolite('Cpd2919', formula='', name='chenodeoxycholoyl coenzyme a [m]', compartment='')
Cpd2920 = Metabolite('Cpd2920', formula='', name='chenodeoxycholate [m]', compartment='')
Cpd2921 = Metabolite('Cpd2921', formula='', name='chenodeoxycholate [x]', compartment='')
Cpd2922 = Metabolite('Cpd2922', formula='', name='3alpha,7alpha-Dihydroxy-5beta-cholestanoyl-CoA [m]', compartment='')
Cpd2923 = Metabolite('Cpd2923', formula='', name='taurochenodeoxycholate [x]', compartment='')
Cpd2924 = Metabolite('Cpd2924', formula='', name='deoxycholoyl-CoA [c]', compartment='')
Cpd2925 = Metabolite('Cpd2925', formula='', name='taurodeoxycholate [c]', compartment='')
Cpd2926 = Metabolite('Cpd2926', formula='', name='deoxycholoyl-CoA [x]', compartment='')
Cpd2927 = Metabolite('Cpd2927', formula='', name='taurodeoxycholate [x]', compartment='')
Cpd2928 = Metabolite('Cpd2928', formula='', name='2,3-cyclic UMP(1-) [e]', compartment='')
Cpd2929 = Metabolite('Cpd2929', formula='', name='3-UMP(2-) [e]', compartment='')
Cpd2930 = Metabolite('Cpd2930', formula='', name='13-cis-retinal [r]', compartment='')
Cpd2931 = Metabolite('Cpd2931', formula='', name='4-OH-13-cis-retinal [r]', compartment='')
Cpd2932 = Metabolite('Cpd2932', formula='', name='all-trans-retinal [r]', compartment='')
Cpd2933 = Metabolite('Cpd2933', formula='', name='4-OH-9-cis-retinal [r]', compartment='')
Cpd2934 = Metabolite('Cpd2934', formula='', name='retinal [r]', compartment='')
Cpd2935 = Metabolite('Cpd2935', formula='', name='4-OH-retinal [r]', compartment='')
Cpd2936 = Metabolite('Cpd2936', formula='', name='L-dopaquinone [x]', compartment='')
Cpd2937 = Metabolite('Cpd2937', formula='', name='5-S-glutathionyl-L-DOPA [x]', compartment='')
Cpd2938 = Metabolite('Cpd2938', formula='', name='retinol [r]', compartment='')
Cpd2939 = Metabolite('Cpd2939', formula='', name='4-hydroxyvitamin A1 [r]', compartment='')
Cpd2940 = Metabolite('Cpd2940', formula='', name='9-cis-retinoate [r]', compartment='')
Cpd2941 = Metabolite('Cpd2941', formula='', name='4-oxo-9-cis-retinoate [r]', compartment='')
Cpd2942 = Metabolite('Cpd2942', formula='', name='4-oxo-13-cis-retinoate [r]', compartment='')
Cpd2943 = Metabolite('Cpd2943', formula='', name='18-hydroxy-all-trans-retinoate [r]', compartment='')
Cpd2944 = Metabolite('Cpd2944', formula='', name='5(S),6(S)-epoxy-15(R)-hydroxyeicosatetraenoate [c]', compartment='')
Cpd2945 = Metabolite('Cpd2945', formula='', name='15-epi-lipoxin A4 [c]', compartment='')
Cpd2946 = Metabolite('Cpd2946', formula='', name='5(S),6(S)-epoxy-15(R)-hydroxyeicosatetraenoate [r]', compartment='')
Cpd2947 = Metabolite('Cpd2947', formula='', name='15-epi-lipoxin A4 [r]', compartment='')
Cpd2948 = Metabolite('Cpd2948', formula='', name='5(S),6(S)-epoxy-15(R)-hydroxyeicosatetraenoate [x]', compartment='')
Cpd2949 = Metabolite('Cpd2949', formula='', name='15-epi-lipoxin A4 [x]', compartment='')
Cpd2950 = Metabolite('Cpd2950', formula='', name='15-epi-lipoxin B4 [c]', compartment='')
Cpd2951 = Metabolite('Cpd2951', formula='', name='15-epi-lipoxin B4 [r]', compartment='')
Cpd2952 = Metabolite('Cpd2952', formula='', name='15-epi-lipoxin B4 [x]', compartment='')
Cpd2953 = Metabolite('Cpd2953', formula='', name='1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate(5-) [g]', compartment='')
Cpd2954 = Metabolite('Cpd2954', formula='', name='1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate(7-) [g]', compartment='')
Cpd2955 = Metabolite('Cpd2955', formula='', name='1-phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate(7-) [r]', compartment='')
Cpd2956 = Metabolite('Cpd2956', formula='', name='N-Acetylmethionine [m]', compartment='')
Cpd2957 = Metabolite('Cpd2957', formula='', name='N-acetyl-L-alanine [m]', compartment='')
Cpd2958 = Metabolite('Cpd2958', formula='', name='N-acetyl-L-asparagine [m]', compartment='')
Cpd2959 = Metabolite('Cpd2959', formula='', name='15(R)-Hydroxy-(5Z,8Z,11Z,13E)-eicosatetraenoate [n]', compartment='')
Cpd2960 = Metabolite('Cpd2960', formula='', name='5(S),6(S)-epoxy-15(R)-hydroxyeicosatetraenoate [n]', compartment='')
Cpd2961 = Metabolite('Cpd2961', formula='', name='2-Arachidonoylglycerol [r]', compartment='')
Cpd2962 = Metabolite('Cpd2962', formula='', name='12-hydroperoxyeicosatetraenoate glyceryl ester [r]', compartment='')
Cpd2963 = Metabolite('Cpd2963', formula='', name='1-lyso-2-arachidonoyl-phosphatidate [c]', compartment='')
Cpd2964 = Metabolite('Cpd2964', formula='', name='2-Arachidonoylglycerol [c]', compartment='')
Cpd2965 = Metabolite('Cpd2965', formula='', name='1-lyso-2-arachidonoyl-phosphatidate [g]', compartment='')
Cpd2966 = Metabolite('Cpd2966', formula='', name='2-Arachidonoylglycerol [g]', compartment='')
Cpd2967 = Metabolite('Cpd2967', formula='', name='1-lyso-2-arachidonoyl-phosphatidate [r]', compartment='')
Cpd2968 = Metabolite('Cpd2968', formula='', name='3-methylcrotonoylglycine [m]', compartment='')
Cpd2969 = Metabolite('Cpd2969', formula='', name='deoxycholate [r]', compartment='')
Cpd2970 = Metabolite('Cpd2970', formula='', name='N-arachidonoylglycinate [m]', compartment='')
Cpd2971 = Metabolite('Cpd2971', formula='', name='rac-5,6-epoxy-retinoate [c]', compartment='')
Cpd2972 = Metabolite('Cpd2972', formula='', name='rac-5,6-epoxy-retinoyl-beta-D-glucuronide [c]', compartment='')
Cpd2973 = Metabolite('Cpd2973', formula='', name='rac-5,6-epoxy-retinoate [r]', compartment='')
Cpd2974 = Metabolite('Cpd2974', formula='', name='rac-5,6-epoxy-retinoyl-beta-D-glucuronide [r]', compartment='')
Cpd2975 = Metabolite('Cpd2975', formula='', name='13-cis-retinoate [c]', compartment='')
Cpd2976 = Metabolite('Cpd2976', formula='', name='13-cis-retinoyl glucuronide [c]', compartment='')
Cpd2977 = Metabolite('Cpd2977', formula='', name='13-cis-retinoyl-beta-D-glucuronide [r]', compartment='')
Cpd2978 = Metabolite('Cpd2978', formula='', name='4-hydroxy-all-trans-retinyl acetate [c]', compartment='')
Cpd2979 = Metabolite('Cpd2979', formula='', name='rac-4-hydroxy-4-O-(beta-D-glucuronide)-all-trans-retinyl acetate [c]', compartment='')
Cpd2980 = Metabolite('Cpd2980', formula='', name='4-hydroxy-all-trans-retinyl acetate [r]', compartment='')
Cpd2981 = Metabolite('Cpd2981', formula='', name='rac-4-hydroxy-4-O-(beta-D-glucuronide)-all-trans-retinyl acetate [r]', compartment='')
Cpd2982 = Metabolite('Cpd2982', formula='', name='9-cis-retinoate [c]', compartment='')
Cpd2983 = Metabolite('Cpd2983', formula='', name='9-cis-retinoyl-beta-D-glucuronide [c]', compartment='')
Cpd2984 = Metabolite('Cpd2984', formula='', name='9-cis-retinoyl-beta-D-glucuronide [r]', compartment='')
Cpd2985 = Metabolite('Cpd2985', formula='', name='4-oxo-9-cis-retinoate [c]', compartment='')
Cpd2986 = Metabolite('Cpd2986', formula='', name='4-oxo-9-cis-retinoyl-beta-glucuronide [c]', compartment='')
Cpd2987 = Metabolite('Cpd2987', formula='', name='4-oxo-9-cis-retinoyl-beta-glucuronide [r]', compartment='')
Cpd2988 = Metabolite('Cpd2988', formula='', name='allopregnanolone [r]', compartment='')
Cpd2989 = Metabolite('Cpd2989', formula='', name='21-hydroxyallopregnanolone [r]', compartment='')
Cpd2990 = Metabolite('Cpd2990', formula='', name='L-cystathionine [m]', compartment='')
Cpd2991 = Metabolite('Cpd2991', formula='', name='N-acetyl-L-cystathionine [m]', compartment='')
Cpd2992 = Metabolite('Cpd2992', formula='', name='8-dehydrocholesterol [m]', compartment='')
Cpd2993 = Metabolite('Cpd2993', formula='', name='27alpha-hydroxy-8-dehydrocholesterol [m]', compartment='')
Cpd2994 = Metabolite('Cpd2994', formula='', name='N-acetyl-L-cysteine [m]', compartment='')
Cpd2995 = Metabolite('Cpd2995', formula='', name='16alpha-Hydroxyestrone [r]', compartment='')
Cpd2996 = Metabolite('Cpd2996', formula='', name='kinetensin [e]', compartment='')
Cpd2997 = Metabolite('Cpd2997', formula='', name='kinetensin 1-7 [e]', compartment='')
Cpd2998 = Metabolite('Cpd2998', formula='', name='kinetensin 1-8 [e]', compartment='')
Cpd2999 = Metabolite('Cpd2999', formula='', name='neuromedin B [l]', compartment='')
Cpd3000 = Metabolite('Cpd3000', formula='', name='neuromedin B (1-3) [l]', compartment='')
Cpd3001 = Metabolite('Cpd3001', formula='', name='neuromedin B (4-10) [l]', compartment='')
Cpd3002 = Metabolite('Cpd3002', formula='', name='neuromedin N [e]', compartment='')
Cpd3003 = Metabolite('Cpd3003', formula='', name='neuromedin N (1-4) [e]', compartment='')
Cpd3004 = Metabolite('Cpd3004', formula='', name='dopamine o-quinone [x]', compartment='')
Cpd3005 = Metabolite('Cpd3005', formula='', name='5-S-glutathionyl-dopamine [x]', compartment='')
Cpd3006 = Metabolite('Cpd3006', formula='', name='kinetensin 1-3 [e]', compartment='')
Cpd3007 = Metabolite('Cpd3007', formula='', name='kinetensin 4-8 [e]', compartment='')
Cpd3008 = Metabolite('Cpd3008', formula='', name='Thyrotropin releasing hormone [c]', compartment='')
Cpd3009 = Metabolite('Cpd3009', formula='', name='histidyl-prolinamide [c]', compartment='')
Cpd3010 = Metabolite('Cpd3010', formula='', name='2-hydroxy-17beta-estradiol-1-S-glutathione [m]', compartment='')
Cpd3011 = Metabolite('Cpd3011', formula='', name='2-hydroxy-17beta-estradiol-1-S-glutathione [r]', compartment='')
Cpd3012 = Metabolite('Cpd3012', formula='', name='2-hydroxy-17beta-estradiol-1-S-glutathione [x]', compartment='')
Cpd3013 = Metabolite('Cpd3013', formula='', name='2-hydroxyestrone-4-S-glutathione [m]', compartment='')
Cpd3014 = Metabolite('Cpd3014', formula='', name='2-hydroxyestrone-4-S-glutathione [r]', compartment='')
Cpd3015 = Metabolite('Cpd3015', formula='', name='2-hydroxyestrone-4-S-glutathione [x]', compartment='')
Cpd3016 = Metabolite('Cpd3016', formula='', name='Xanthurenic acid [c]', compartment='')
Cpd3017 = Metabolite('Cpd3017', formula='', name='3-hydroxy-L-kynurenine [m]', compartment='')
Cpd3018 = Metabolite('Cpd3018', formula='', name='Xanthurenic acid [m]', compartment='')
Cpd3019 = Metabolite('Cpd3019', formula='', name='leukotriene E4(1-) [n]', compartment='')
Cpd3020 = Metabolite('Cpd3020', formula='', name='N-acetyl-leukotriene E4 [n]', compartment='')
Cpd3021 = Metabolite('Cpd3021', formula='', name='(+)-gama-tocopherol [r]', compartment='')
Cpd3022 = Metabolite('Cpd3022', formula='', name='13-hydroxy-gama-tocopherol [r]', compartment='')
Cpd3023 = Metabolite('Cpd3023', formula='', name='13-hydroxy-alpha-tocopherol [r]', compartment='')
Cpd3024 = Metabolite('Cpd3024', formula='', name='13-carboxy-alpha-tocopherol [r]', compartment='')
Cpd3025 = Metabolite('Cpd3025', formula='', name='(+)-alpha-tocopherol [r]', compartment='')
Cpd3026 = Metabolite('Cpd3026', formula='', name='13-carboxy-gama-tocopherol [r]', compartment='')
Cpd3027 = Metabolite('Cpd3027', formula='', name='3-carboxy-alpha-chromanol [c]', compartment='')
Cpd3028 = Metabolite('Cpd3028', formula='', name='alpha-CEHC-glucuronide [c]', compartment='')
Cpd3029 = Metabolite('Cpd3029', formula='', name='3-carboxy-alpha-chromanol [r]', compartment='')
Cpd3030 = Metabolite('Cpd3030', formula='', name='alpha-CEHC-glucuronide [r]', compartment='')
Cpd3031 = Metabolite('Cpd3031', formula='', name='gama-carboxyethyl-hydroxychroman [c]', compartment='')
Cpd3032 = Metabolite('Cpd3032', formula='', name='gama-CEHC-glucuronide [c]', compartment='')
Cpd3033 = Metabolite('Cpd3033', formula='', name='gama-carboxyethyl-hydroxychroman [r]', compartment='')
Cpd3034 = Metabolite('Cpd3034', formula='', name='gama-CEHC-glucuronide [r]', compartment='')
Cpd3035 = Metabolite('Cpd3035', formula='', name='cholesta-5,7,24-trien-3beta-ol [n]', compartment='')
Cpd3036 = Metabolite('Cpd3036', formula='', name='desmosterol [n]', compartment='')
Cpd3037 = Metabolite('Cpd3037', formula='', name='isovalerylglycine [m]', compartment='')
Cpd3038 = Metabolite('Cpd3038', formula='', name='2-methylbutyrylglycine [m]', compartment='')
Cpd3039 = Metabolite('Cpd3039', formula='', name='isobutyrylglycine [m]', compartment='')
Cpd3040 = Metabolite('Cpd3040', formula='', name='3-hydroxykynurenine-O-beta-D-glucoside [m]', compartment='')
Cpd3041 = Metabolite('Cpd3041', formula='', name='glutathionyl-3-hydroxykynurenine glucoside [m]', compartment='')
Cpd3042 = Metabolite('Cpd3042', formula='', name='N-acetyl-seryl-aspartyl-lysyl-proline [c]', compartment='')
Cpd3043 = Metabolite('Cpd3043', formula='', name='N-acetyl-seryl-aspartate [c]', compartment='')
Cpd3044 = Metabolite('Cpd3044', formula='', name='lysyl-proline [c]', compartment='')
Cpd3045 = Metabolite('Cpd3045', formula='', name='N-acetyl-seryl-aspartyl-lysyl-proline [e]', compartment='')
Cpd3046 = Metabolite('Cpd3046', formula='', name='N-acetyl-seryl-aspartate [e]', compartment='')
Cpd3047 = Metabolite('Cpd3047', formula='', name='lysyl-proline [e]', compartment='')
Cpd3048 = Metabolite('Cpd3048', formula='', name='2-methylglutaconyl-CoA [m]', compartment='')
Cpd3049 = Metabolite('Cpd3049', formula='', name='2-methylglutaconic acid [m]', compartment='')
Cpd3050 = Metabolite('Cpd3050', formula='', name='Diazepam [r]', compartment='')
Cpd3051 = Metabolite('Cpd3051', formula='', name='formaldehyde [r]', compartment='')
Cpd3052 = Metabolite('Cpd3052', formula='', name='Nordazepam [r]', compartment='')
Cpd3053 = Metabolite('Cpd3053', formula='', name='hypochlorous acid [c]', compartment='')
Cpd3054 = Metabolite('Cpd3054', formula='', name='hypochlorous acid [e]', compartment='')
Cpd3055 = Metabolite('Cpd3055', formula='', name='Chloride [l]', compartment='')
Cpd3056 = Metabolite('Cpd3056', formula='', name='hypochlorous acid [l]', compartment='')
Cpd3057 = Metabolite('Cpd3057', formula='', name='Chloride [n]', compartment='')
Cpd3058 = Metabolite('Cpd3058', formula='', name='hypochlorous acid [n]', compartment='')
Cpd3059 = Metabolite('Cpd3059', formula='', name='Nitrite [c]', compartment='')
Cpd3060 = Metabolite('Cpd3060', formula='', name='nitryl chloride [c]', compartment='')
Cpd3061 = Metabolite('Cpd3061', formula='', name='Nitrite [e]', compartment='')
Cpd3062 = Metabolite('Cpd3062', formula='', name='nitryl chloride [e]', compartment='')
Cpd3063 = Metabolite('Cpd3063', formula='', name='Nitrite [l]', compartment='')
Cpd3064 = Metabolite('Cpd3064', formula='', name='nitryl chloride [l]', compartment='')
Cpd3065 = Metabolite('Cpd3065', formula='', name='Nitrite [n]', compartment='')
Cpd3066 = Metabolite('Cpd3066', formula='', name='nitryl chloride [n]', compartment='')
Cpd3067 = Metabolite('Cpd3067', formula='', name='adrenochrome [x]', compartment='')
Cpd3068 = Metabolite('Cpd3068', formula='', name='5-S-glutathionyl-adrenochrome hydroquinone [x]', compartment='')
Cpd3069 = Metabolite('Cpd3069', formula='', name='noradrenochrome [x]', compartment='')
Cpd3070 = Metabolite('Cpd3070', formula='', name='5-S-glutathionyl-noradrenochrome hydroquinone [x]', compartment='')
Cpd3071 = Metabolite('Cpd3071', formula='', name='L-dopachromate [x]', compartment='')
Cpd3072 = Metabolite('Cpd3072', formula='', name='5-S-glutathionyl-dopachrome hydroquinone [x]', compartment='')
Cpd3073 = Metabolite('Cpd3073', formula='', name='dopaminochrome [x]', compartment='')
Cpd3074 = Metabolite('Cpd3074', formula='', name='5-S-glutathionyl-aminochrome reduced [x]', compartment='')
Cpd3075 = Metabolite('Cpd3075', formula='', name='gama-CEHC-glucuronide [e]', compartment='')
Cpd3076 = Metabolite('Cpd3076', formula='', name='D-glucuronate [e]', compartment='')
Cpd3077 = Metabolite('Cpd3077', formula='', name='gama-carboxyethyl-hydroxychroman [e]', compartment='')
Cpd3078 = Metabolite('Cpd3078', formula='', name='gama-CEHC-glucuronide [l]', compartment='')
Cpd3079 = Metabolite('Cpd3079', formula='', name='gama-carboxyethyl-hydroxychroman [l]', compartment='')
Cpd3080 = Metabolite('Cpd3080', formula='', name='2-hydroxyamino-3,8-dimethylimidazo[4,5-f]quinoxaline [c]', compartment='')
Cpd3081 = Metabolite('Cpd3081', formula='', name='2-hydroxyamino-MeIQx-N2-(beta-1-glucosiduronyl)-2-hydroxyamino-3,8-dimethylimidazo[4,5-f]quinoxaline [c]', compartment='')
Cpd3082 = Metabolite('Cpd3082', formula='', name='2-hydroxyamino-3,8-dimethylimidazo[4,5-f]quinoxaline [r]', compartment='')
Cpd3083 = Metabolite('Cpd3083', formula='', name='2-hydroxyamino-MeIQx-N2-(beta-1-glucosiduronyl)-2-hydroxyamino-3,8-dimethylimidazo[4,5-f]quinoxaline [r]', compartment='')
Cpd3084 = Metabolite('Cpd3084', formula='', name='25(S)-trihydroxycoprostanoyl-CoA [m]', compartment='')
Cpd3085 = Metabolite('Cpd3085', formula='', name='25(R)-trihydroxycoprostanoyl-CoA [m]', compartment='')
Cpd3086 = Metabolite('Cpd3086', formula='', name='25(S)-trihydroxycoprostanoyl-CoA [x]', compartment='')
Cpd3087 = Metabolite('Cpd3087', formula='', name='25(R)-trihydroxycoprostanoyl-CoA [x]', compartment='')
Cpd3088 = Metabolite('Cpd3088', formula='', name='3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al [m]', compartment='')
Cpd3089 = Metabolite('Cpd3089', formula='', name='7alpha,27-dihydroxycholesterol [m]', compartment='')
Cpd3090 = Metabolite('Cpd3090', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,24S,25-pentol [r]', compartment='')
Cpd3091 = Metabolite('Cpd3091', formula='', name='3alpha,7alpha,12alpha,25-tetrahydroxy-5beta-cholestane-24-one [r]', compartment='')
Cpd3092 = Metabolite('Cpd3092', formula='', name='deoxycholoyl-CoA [r]', compartment='')
Cpd3093 = Metabolite('Cpd3093', formula='', name='lithocholate [r]', compartment='')
Cpd3094 = Metabolite('Cpd3094', formula='', name='lithocholate [x]', compartment='')
Cpd3095 = Metabolite('Cpd3095', formula='', name='Taurolithocholate [x]', compartment='')
Cpd3096 = Metabolite('Cpd3096', formula='', name='glycolithocholate [x]', compartment='')
Cpd3097 = Metabolite('Cpd3097', formula='', name='N-Acetylmethionine [c]', compartment='')
Cpd3098 = Metabolite('Cpd3098', formula='', name='N-acetyl-L-asparagine [c]', compartment='')
Cpd3099 = Metabolite('Cpd3099', formula='', name='propionate [x]', compartment='')
Cpd3100 = Metabolite('Cpd3100', formula='', name='UDP-D-galactose(2-) [r]', compartment='')
Cpd3101 = Metabolite('Cpd3101', formula='', name='D-galactosyl-N-acylsphingosine [r]', compartment='')
Cpd3102 = Metabolite('Cpd3102', formula='', name='beta-D-Galactosyl-1,4-beta-D-glucosylceramide [g]', compartment='')
Cpd3103 = Metabolite('Cpd3103', formula='', name='psychosine(1+) [l]', compartment='')
Cpd3104 = Metabolite('Cpd3104', formula='', name='psychosine(1+) [c]', compartment='')
Cpd3105 = Metabolite('Cpd3105', formula='', name='Psychosine sulfate [c]', compartment='')
Cpd3106 = Metabolite('Cpd3106', formula='', name='psychosine(1+) [g]', compartment='')
Cpd3107 = Metabolite('Cpd3107', formula='', name='Psychosine sulfate [g]', compartment='')
Cpd3108 = Metabolite('Cpd3108', formula='', name='beta-D-Galactosyl-1,4-beta-D-glucosylceramide [c]', compartment='')
Cpd3109 = Metabolite('Cpd3109', formula='', name='Lactosylceramide sulfate [c]', compartment='')
Cpd3110 = Metabolite('Cpd3110', formula='', name='Lactosylceramide sulfate [g]', compartment='')
Cpd3111 = Metabolite('Cpd3111', formula='', name='(2S)-pristanoyl-CoA [x]', compartment='')
Cpd3112 = Metabolite('Cpd3112', formula='', name='(2R)-pristanoyl-CoA [x]', compartment='')
Cpd3113 = Metabolite('Cpd3113', formula='', name='androst-4-ene-3,17-dione [c]', compartment='')
Cpd3114 = Metabolite('Cpd3114', formula='', name='11beta-Hydroxyandrost-4-ene-3,17-dione [m]', compartment='')
Cpd3115 = Metabolite('Cpd3115', formula='', name='11beta-Hydroxyandrost-4-ene-3,17-dione [r]', compartment='')
Cpd3116 = Metabolite('Cpd3116', formula='', name='alloxan [c]', compartment='')
Cpd3117 = Metabolite('Cpd3117', formula='', name='alloxan [e]', compartment='')
Cpd3118 = Metabolite('Cpd3118', formula='', name='Urate [n]', compartment='')
Cpd3119 = Metabolite('Cpd3119', formula='', name='Urea [n]', compartment='')
Cpd3120 = Metabolite('Cpd3120', formula='', name='alloxan [n]', compartment='')
Cpd3121 = Metabolite('Cpd3121', formula='', name='(S)-3-hydroxypalmitoleoyl-CoA [m]', compartment='')
Cpd3122 = Metabolite('Cpd3122', formula='', name='3-oxopalmitoleoyl-CoA [m]', compartment='')
Cpd3123 = Metabolite('Cpd3123', formula='', name='(S)-3-hydroxypalmitoleoyl-CoA [x]', compartment='')
Cpd3124 = Metabolite('Cpd3124', formula='', name='3-oxopalmitoleoyl-CoA [x]', compartment='')
Cpd3125 = Metabolite('Cpd3125', formula='', name='cis-(3S)-hydroxytetradec-7-enoyl-CoA [m]', compartment='')
Cpd3126 = Metabolite('Cpd3126', formula='', name='3-oxomyrist-7-enoyl-CoA [m]', compartment='')
Cpd3127 = Metabolite('Cpd3127', formula='', name='cis-(3S)-hydroxytetradec-7-enoyl-CoA [x]', compartment='')
Cpd3128 = Metabolite('Cpd3128', formula='', name='3-oxomyrist-7-enoyl-CoA [x]', compartment='')
Cpd3129 = Metabolite('Cpd3129', formula='', name='3(S)-3-hydroxydodecen-(5Z)-oyl-CoA [m]', compartment='')
Cpd3130 = Metabolite('Cpd3130', formula='', name='3(S)-3-hydroxydodecen-(5Z)-oyl-CoA [x]', compartment='')
Cpd3131 = Metabolite('Cpd3131', formula='', name='3-oxolaur-cis-5-enoyl-CoA [x]', compartment='')
Cpd3132 = Metabolite('Cpd3132', formula='', name='1-acylglycerone 3-phosphate(2-) [c]', compartment='')
Cpd3133 = Metabolite('Cpd3133', formula='', name='1-acyl-sn-glycerol 3-phosphate(2-) [c]', compartment='')
Cpd3134 = Metabolite('Cpd3134', formula='', name='Hexadecenoate (n-C16:1) [m]', compartment='')
Cpd3135 = Metabolite('Cpd3135', formula='', name='Hexadecenoate (n-C16:1) [x]', compartment='')
Cpd3136 = Metabolite('Cpd3136', formula='', name='palmitoleyl-CoA(4-) [x]', compartment='')
Cpd3137 = Metabolite('Cpd3137', formula='', name='trans,cis-hexadeca-2,9-dienoyl-CoA [m]', compartment='')
Cpd3138 = Metabolite('Cpd3138', formula='', name='trans,cis-hexadeca-2,9-dienoyl-CoA [x]', compartment='')
Cpd3139 = Metabolite('Cpd3139', formula='', name='cis-myrist-7-enoyl-CoA [m]', compartment='')
Cpd3140 = Metabolite('Cpd3140', formula='', name='trans,cis-myristo-2,7-dienoyl-CoA [m]', compartment='')
Cpd3141 = Metabolite('Cpd3141', formula='', name='cis-myrist-7-enoyl-CoA [x]', compartment='')
Cpd3142 = Metabolite('Cpd3142', formula='', name='trans,cis-myristo-2,7-dienoyl-CoA [x]', compartment='')
Cpd3143 = Metabolite('Cpd3143', formula='', name='trans,cis-dodeca-2,5-dienoyl-CoA [m]', compartment='')
Cpd3144 = Metabolite('Cpd3144', formula='', name='cis-laur-5-enoyl-CoA [m]', compartment='')
Cpd3145 = Metabolite('Cpd3145', formula='', name='trans,cis-dodeca-2,5-dienoyl-CoA [x]', compartment='')
Cpd3146 = Metabolite('Cpd3146', formula='', name='cis-laur-5-enoyl-CoA [x]', compartment='')
Cpd3147 = Metabolite('Cpd3147', formula='', name='phosphatidylinositol 4,5-bisphosphate [g]', compartment='')
Cpd3148 = Metabolite('Cpd3148', formula='', name='phosphatidylinositol 4,5-bisphosphate [r]', compartment='')
Cpd3149 = Metabolite('Cpd3149', formula='', name='1-phosphatidyl-1D-myo-inositol 5-phosphate(3-) [n]', compartment='')
Cpd3150 = Metabolite('Cpd3150', formula='', name='1-phosphatidyl-1D-myo-inositol 3,5-bisphosphate(5-) [n]', compartment='')
Cpd3151 = Metabolite('Cpd3151', formula='', name='1-phosphatidyl-1D-myo-inositol 3-phosphate(3-) [g]', compartment='')
Cpd3152 = Metabolite('Cpd3152', formula='', name='24-oxo-1alpha,25-dihydroxyvitamin D3 [m]', compartment='')
Cpd3153 = Metabolite('Cpd3153', formula='', name='24-oxo-1alpha,23,25-trihydroxyvitamin D3 [m]', compartment='')
Cpd3154 = Metabolite('Cpd3154', formula='', name='O-alkylglycerone phosphate(2-) [c]', compartment='')
Cpd3155 = Metabolite('Cpd3155', formula='', name='1-alkyl-sn-glycerol 3-phosphate(1-) [c]', compartment='')
Cpd3156 = Metabolite('Cpd3156', formula='', name='2-trans-cis,cis,cis,cis-4,8,11,14-eicosapentaenoyl-CoA [m]', compartment='')
Cpd3157 = Metabolite('Cpd3157', formula='', name='2-trans-cis,cis,cis,cis-4,8,11,14-eicosapentaenoyl-CoA [x]', compartment='')
Cpd3158 = Metabolite('Cpd3158', formula='', name='trans-2-cis,cis,cis-8,11,14-eicosatetraenoyl-CoA [x]', compartment='')
Cpd3159 = Metabolite('Cpd3159', formula='', name='3(S)-hydroxy-dihomo-gama-linolenoyl-CoA [x]', compartment='')
Cpd3160 = Metabolite('Cpd3160', formula='', name='3-oxo-dihomo-gama-linolenoyl-CoA [x]', compartment='')
Cpd3161 = Metabolite('Cpd3161', formula='', name='3-oxo-6Z,9Z,12Z-octadecatrienoyl-CoA [x]', compartment='')
Cpd3162 = Metabolite('Cpd3162', formula='', name='4Z,7Z,10Z-hexadecatrienoyl-CoA [x]', compartment='')
Cpd3163 = Metabolite('Cpd3163', formula='', name='2E,6Z,9Z,12Z-octadecatetraenoyl-CoA [x]', compartment='')
Cpd3164 = Metabolite('Cpd3164', formula='', name='3(S)-hydroxy-6Z,9Z,12Z-octadecatrienoyl-CoA [x]', compartment='')
Cpd3165 = Metabolite('Cpd3165', formula='', name='3Z,7Z,10Z-hexadecatrienoyl-CoA [m]', compartment='')
Cpd3166 = Metabolite('Cpd3166', formula='', name='3Z,7Z,10Z-hexadecatrienoyl-CoA [x]', compartment='')
Cpd3167 = Metabolite('Cpd3167', formula='', name='2E,4Z,7Z,10Z-hexadecatetraenoyl-CoA [x]', compartment='')
Cpd3168 = Metabolite('Cpd3168', formula='', name='trans-3-cis-5,8,11,14-eicosapentaenoyl-CoA [m]', compartment='')
Cpd3169 = Metabolite('Cpd3169', formula='', name='trans-3-cis-5,8,11,14-eicosapentaenoyl-CoA [x]', compartment='')
Cpd3170 = Metabolite('Cpd3170', formula='', name='trans-3-cis-8,11,14-eicosatetraenoyl-CoA [x]', compartment='')
Cpd3171 = Metabolite('Cpd3171', formula='', name='(3E,5Z,8Z)-tetradecatrienoyl-CoA [m]', compartment='')
Cpd3172 = Metabolite('Cpd3172', formula='', name='trans-2-cis,cis-4,8-tetradecatrienoyl-CoA [m]', compartment='')
Cpd3173 = Metabolite('Cpd3173', formula='', name='trans-3-cis-8-tetradecadienoyl-CoA [m]', compartment='')
Cpd3174 = Metabolite('Cpd3174', formula='', name='trans-2-cis-8-tetradecadienoyl-CoA [m]', compartment='')
Cpd3175 = Metabolite('Cpd3175', formula='', name='(3S)-3-hydroxy-cis-8-tetradecenoyl-CoA [m]', compartment='')
Cpd3176 = Metabolite('Cpd3176', formula='', name='3-oxo-cis-8-tetradecenoyl-CoA [m]', compartment='')
Cpd3177 = Metabolite('Cpd3177', formula='', name='cis-6-dodecenoyl-CoA [m]', compartment='')
Cpd3178 = Metabolite('Cpd3178', formula='', name='12-oxo-leukotriene B4 [c]', compartment='')
Cpd3179 = Metabolite('Cpd3179', formula='', name='leukotriene B4(1-) [m]', compartment='')
Cpd3180 = Metabolite('Cpd3180', formula='', name='leukotriene B4(1-) [x]', compartment='')
Cpd3181 = Metabolite('Cpd3181', formula='', name='20-hydroxy-leukotriene B4(1-) [m]', compartment='')
Cpd3182 = Metabolite('Cpd3182', formula='', name='20-oxo-leukotriene B4 [m]', compartment='')
Cpd3183 = Metabolite('Cpd3183', formula='', name='20-oxo-leukotriene B4 [c]', compartment='')
Cpd3184 = Metabolite('Cpd3184', formula='', name='20-carboxy-leukotriene-B4 [c]', compartment='')
Cpd3185 = Metabolite('Cpd3185', formula='', name='20-carboxy-leukotriene-B4 [m]', compartment='')
Cpd3186 = Metabolite('Cpd3186', formula='', name='20-oxo-leukotriene B4 [r]', compartment='')
Cpd3187 = Metabolite('Cpd3187', formula='', name='20-carboxy-leukotriene-B4 [r]', compartment='')
Cpd3188 = Metabolite('Cpd3188', formula='', name='2-Hydroxyestradiol-17beta [r]', compartment='')
Cpd3189 = Metabolite('Cpd3189', formula='', name='20-hydroxy-leukotriene B4(1-) [r]', compartment='')
Cpd3190 = Metabolite('Cpd3190', formula='', name='20-dihydroxyleukotriene B4 [r]', compartment='')
Cpd3191 = Metabolite('Cpd3191', formula='', name='20-trihydroxy-leukotriene-B4 [r]', compartment='')
Cpd3192 = Metabolite('Cpd3192', formula='', name='12-oxo-20-hydroxy-leukotriene B4 [r]', compartment='')
Cpd3193 = Metabolite('Cpd3193', formula='', name='12-oxo-20-dihydroxy-leukotriene B4 [r]', compartment='')
Cpd3194 = Metabolite('Cpd3194', formula='', name='12-oxo-20-trihydroxy-leukotriene B4 [r]', compartment='')
Cpd3195 = Metabolite('Cpd3195', formula='', name='15(S)-HPETE(1-) [n]', compartment='')
Cpd3196 = Metabolite('Cpd3196', formula='', name='5(S),6(S)-epoxy-15(S)-hydroxy-7E,9E,11Z,13E-eicosatetraenoic acid anion [n]', compartment='')
Cpd3197 = Metabolite('Cpd3197', formula='', name='14,15-DiHETE [c]', compartment='')
Cpd3198 = Metabolite('Cpd3198', formula='', name='5(S),6(S)-epoxy-15(S)-hydroxy-7E,9E,11Z,13E-eicosatetraenoic acid anion [c]', compartment='')
Cpd3199 = Metabolite('Cpd3199', formula='', name='Lipoxin B4 [c]', compartment='')
Cpd3200 = Metabolite('Cpd3200', formula='', name='5(S),6(S)-epoxy-15(S)-hydroxy-7E,9E,11Z,13E-eicosatetraenoic acid anion [r]', compartment='')
Cpd3201 = Metabolite('Cpd3201', formula='', name='Lipoxin B4 [r]', compartment='')
Cpd3202 = Metabolite('Cpd3202', formula='', name='5(S),6(S)-epoxy-15(S)-hydroxy-7E,9E,11Z,13E-eicosatetraenoic acid anion [x]', compartment='')
Cpd3203 = Metabolite('Cpd3203', formula='', name='Lipoxin B4 [x]', compartment='')
Cpd3204 = Metabolite('Cpd3204', formula='', name='Lipoxin A4 [c]', compartment='')
Cpd3205 = Metabolite('Cpd3205', formula='', name='Lipoxin A4 [r]', compartment='')
Cpd3206 = Metabolite('Cpd3206', formula='', name='Lipoxin A4 [x]', compartment='')
Cpd3207 = Metabolite('Cpd3207', formula='', name='leukotriene A4(1-) [n]', compartment='')
Cpd3208 = Metabolite('Cpd3208', formula='', name='14,15-DiHETE [n]', compartment='')
Cpd3209 = Metabolite('Cpd3209', formula='', name='Lipoxin B4 [n]', compartment='')
Cpd3210 = Metabolite('Cpd3210', formula='', name='3(S)-phytanoyl-CoA [x]', compartment='')
Cpd3211 = Metabolite('Cpd3211', formula='', name='3(S)-2-hydroxyphytanoyl-CoA [x]', compartment='')
Cpd3212 = Metabolite('Cpd3212', formula='', name='formyl-CoA(4-) [x]', compartment='')
Cpd3213 = Metabolite('Cpd3213', formula='', name='pristanal [x]', compartment='')
Cpd3214 = Metabolite('Cpd3214', formula='', name='(2R,6S,10S)-pristanate [x]', compartment='')
Cpd3215 = Metabolite('Cpd3215', formula='', name='(3S,7R,11R)-phytanate [x]', compartment='')
Cpd3216 = Metabolite('Cpd3216', formula='', name='trans-2,3-dehydropristanoyl-CoA [x]', compartment='')
Cpd3217 = Metabolite('Cpd3217', formula='', name='3(R)-hydroxy-pristanoyl-CoA [x]', compartment='')
Cpd3218 = Metabolite('Cpd3218', formula='', name='3-oxopristanoyl-CoA(4-) [x]', compartment='')
Cpd3219 = Metabolite('Cpd3219', formula='', name='(2S,6R,10R)-trimethyl-hendecanoyl-CoA [x]', compartment='')
Cpd3220 = Metabolite('Cpd3220', formula='', name='(2R,6R,10R)-trimethyl-hendecanoyl-CoA [x]', compartment='')
Cpd3221 = Metabolite('Cpd3221', formula='', name='4,8,12-trimethyltridecanoyl-CoA(4-) [x]', compartment='')
Cpd3222 = Metabolite('Cpd3222', formula='', name='(4R,8R,12R)-trimethyl-2E-tridecenoyl-CoA [x]', compartment='')
Cpd3223 = Metabolite('Cpd3223', formula='', name='3(R)-hydroxy-(4R,8R,12R)-trimethyl-tridecanoyl-CoA [x]', compartment='')
Cpd3224 = Metabolite('Cpd3224', formula='', name='3-oxo-(4R,8R,12R)-trimethyl-tridecanoyl-CoA [x]', compartment='')
Cpd3225 = Metabolite('Cpd3225', formula='', name='(2S,6R,10R)-trimethyl-2E-hendecenoyl-CoA [x]', compartment='')
Cpd3226 = Metabolite('Cpd3226', formula='', name='3-oxo-(2S,6R,10R)-trimethyl-hendecanoyl-CoA [x]', compartment='')
Cpd3227 = Metabolite('Cpd3227', formula='', name='4(R),8-dimethyl-nonanoyl-CoA [x]', compartment='')
Cpd3228 = Metabolite('Cpd3228', formula='', name='3(R)-hydroxy-(2S,6R,10)-trimethyl-hendecanoyl-CoA [x]', compartment='')
Cpd3229 = Metabolite('Cpd3229', formula='', name='dopamine o-quinone [c]', compartment='')
Cpd3230 = Metabolite('Cpd3230', formula='', name='dopaminium(1+) [l]', compartment='')
Cpd3231 = Metabolite('Cpd3231', formula='', name='dopamine o-quinone [l]', compartment='')
Cpd3232 = Metabolite('Cpd3232', formula='', name='3-oxo-dihomo-gama-linolenoyl-CoA [c]', compartment='')
Cpd3233 = Metabolite('Cpd3233', formula='', name='3-oxo-dihomo-gama-linolenoyl-CoA [r]', compartment='')
Cpd3234 = Metabolite('Cpd3234', formula='', name='trans-2-cis,cis,cis-8,11,14-eicosatetraenoyl-CoA [r]', compartment='')
Cpd3235 = Metabolite('Cpd3235', formula='', name='dihomo-gamma-linolenyl coenzyme A [r]', compartment='')
Cpd3236 = Metabolite('Cpd3236', formula='', name='eicosa-2E,8Z,11Z,14Z,17Z-pentaenoyl-CoA [r]', compartment='')
Cpd3237 = Metabolite('Cpd3237', formula='', name='(8Z,11Z,14Z,17Z)-eicosatetraenoyl-CoA [r]', compartment='')
Cpd3238 = Metabolite('Cpd3238', formula='', name='stearidonoyl-CoA [c]', compartment='')
Cpd3239 = Metabolite('Cpd3239', formula='', name='3-keto-eicosa-8,11,14,17-all-cis-tetraenoyl-CoA [c]', compartment='')
Cpd3240 = Metabolite('Cpd3240', formula='', name='stearidonoyl-CoA [r]', compartment='')
Cpd3241 = Metabolite('Cpd3241', formula='', name='3-keto-eicosa-8,11,14,17-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3242 = Metabolite('Cpd3242', formula='', name='Androst-5-ene-3beta,17beta-diol [m]', compartment='')
Cpd3243 = Metabolite('Cpd3243', formula='', name='17beta-hydroxy-5alpha-androstan-3-one [m]', compartment='')
Cpd3244 = Metabolite('Cpd3244', formula='', name='Androst-5-ene-3beta,17beta-diol [r]', compartment='')
Cpd3245 = Metabolite('Cpd3245', formula='', name='3(S)-hydroxy-all-cis-8,11,14,17-eicosatetraenoyl-CoA [r]', compartment='')
Cpd3246 = Metabolite('Cpd3246', formula='', name='alpha-Linolenoyl-CoA [r]', compartment='')
Cpd3247 = Metabolite('Cpd3247', formula='', name='2-trans-7,10,13,16,19-all-cis-docosahexaenoyl-CoA [r]', compartment='')
Cpd3248 = Metabolite('Cpd3248', formula='', name='docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA [r]', compartment='')
Cpd3249 = Metabolite('Cpd3249', formula='', name='eicosa-5,8,11,14,17-all-cis-pentaenoyl-CoA [c]', compartment='')
Cpd3250 = Metabolite('Cpd3250', formula='', name='3-oxo-docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA [c]', compartment='')
Cpd3251 = Metabolite('Cpd3251', formula='', name='eicosa-5,8,11,14,17-all-cis-pentaenoyl-CoA [r]', compartment='')
Cpd3252 = Metabolite('Cpd3252', formula='', name='3-oxo-docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA [r]', compartment='')
Cpd3253 = Metabolite('Cpd3253', formula='', name='3(S)-hydroxy-docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA [r]', compartment='')
Cpd3254 = Metabolite('Cpd3254', formula='', name='2-trans-9,12,15,18,21-all-cis-tetracosahexaenoyl-CoA [r]', compartment='')
Cpd3255 = Metabolite('Cpd3255', formula='', name='tetracosa-9,12,15,18,21-all-cis-pentaenoyl-CoA [r]', compartment='')
Cpd3256 = Metabolite('Cpd3256', formula='', name='docosa-7,10,13,16,19-all-cis-pentaenoyl-CoA [c]', compartment='')
Cpd3257 = Metabolite('Cpd3257', formula='', name='3-oxo-all-cis-6,9,12,15,18-tetracosapentaenoyl-CoA [c]', compartment='')
Cpd3258 = Metabolite('Cpd3258', formula='', name='3-oxo-all-cis-6,9,12,15,18-tetracosapentaenoyl-CoA [r]', compartment='')
Cpd3259 = Metabolite('Cpd3259', formula='', name='3(S)-hydroxy-tetracosa-9,12,15,18,21-all-cis-pentaenoyl-CoA [r]', compartment='')
Cpd3260 = Metabolite('Cpd3260', formula='', name='timnodonate [c]', compartment='')
Cpd3261 = Metabolite('Cpd3261', formula='', name='timnodonate [r]', compartment='')
Cpd3262 = Metabolite('Cpd3262', formula='', name='4,4-dimethylcholesta-8(9),14-dien-3beta-ol [c]', compartment='')
Cpd3263 = Metabolite('Cpd3263', formula='', name='4,4-dimethylcholesta-8(9)-en-3beta-ol [c]', compartment='')
Cpd3264 = Metabolite('Cpd3264', formula='', name='4,4-dimethylcholesta-8(9),14-dien-3beta-ol [r]', compartment='')
Cpd3265 = Metabolite('Cpd3265', formula='', name='4,4-dimethylcholesta-8(9)-en-3beta-ol [r]', compartment='')
Cpd3266 = Metabolite('Cpd3266', formula='', name='cholesta-8(9)-en-3beta-ol [c]', compartment='')
Cpd3267 = Metabolite('Cpd3267', formula='', name='5alpha-cholest-7-en-3beta-ol [c]', compartment='')
Cpd3268 = Metabolite('Cpd3268', formula='', name='3-oxo-all-cis-6,9,12,15,18,21-tetracosahexaenoyl-CoA [x]', compartment='')
Cpd3269 = Metabolite('Cpd3269', formula='', name='docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA [x]', compartment='')
Cpd3270 = Metabolite('Cpd3270', formula='', name='trans-2-all-cis-6,9,12,15,18,21-tetracosaheptaenoyl-CoA [x]', compartment='')
Cpd3271 = Metabolite('Cpd3271', formula='', name='(3S)-hydroxy-tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA [x]', compartment='')
Cpd3272 = Metabolite('Cpd3272', formula='', name='24,25,26,27-tetranor-23-oxo-hydroxyvitamin D3 [m]', compartment='')
Cpd3273 = Metabolite('Cpd3273', formula='', name='1alpha-hydroxy-23-carboxy-24,25,26,27-tetranorvitamin D3 [m]', compartment='')
Cpd3274 = Metabolite('Cpd3274', formula='', name='tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA [x]', compartment='')
Cpd3275 = Metabolite('Cpd3275', formula='', name='nisinate [c]', compartment='')
Cpd3276 = Metabolite('Cpd3276', formula='', name='tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA [c]', compartment='')
Cpd3277 = Metabolite('Cpd3277', formula='', name='nisinate [r]', compartment='')
Cpd3278 = Metabolite('Cpd3278', formula='', name='tetracosa-6,9,12,15,18,21-all-cis-hexaenoyl-CoA [r]', compartment='')
Cpd3279 = Metabolite('Cpd3279', formula='', name='docosa-4,7,10,13,16,19-all-cis-hexaenoyl-CoA [r]', compartment='')
Cpd3280 = Metabolite('Cpd3280', formula='', name='adrenoyl-CoA [r]', compartment='')
Cpd3281 = Metabolite('Cpd3281', formula='', name='adrenoyl-CoA [c]', compartment='')
Cpd3282 = Metabolite('Cpd3282', formula='', name='2-trans-9,12,15,18-all-cis-tetracosapentaenoyl-CoA [r]', compartment='')
Cpd3283 = Metabolite('Cpd3283', formula='', name='tetracosa-9,12,15,18-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3284 = Metabolite('Cpd3284', formula='', name='3-oxo-docosa-7,10,13,16-all-cis-tetraenoyl-CoA [c]', compartment='')
Cpd3285 = Metabolite('Cpd3285', formula='', name='3-oxo-docosa-7,10,13,16-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3286 = Metabolite('Cpd3286', formula='', name='3(S)-hydroxy-docosa-7,10,13,16-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3287 = Metabolite('Cpd3287', formula='', name='2E,7Z,10Z,13Z,16Z-docosapentaenoyl-CoA [r]', compartment='')
Cpd3288 = Metabolite('Cpd3288', formula='', name='3-oxo-tetracosa-9,12,15,18-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3289 = Metabolite('Cpd3289', formula='', name='3(S)-hydroxy-tetracosa-12,15,18,21-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3290 = Metabolite('Cpd3290', formula='', name='3(S)-hydroxy-tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA [x]', compartment='')
Cpd3291 = Metabolite('Cpd3291', formula='', name='3-oxo-all-cis-6,9,12,15,18-tetracosapentaenoyl-CoA [x]', compartment='')
Cpd3292 = Metabolite('Cpd3292', formula='', name='tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA [x]', compartment='')
Cpd3293 = Metabolite('Cpd3293', formula='', name='trans-2-all-cis-6,9,12,15,18-tetracosahexaenoyl-CoA [x]', compartment='')
Cpd3294 = Metabolite('Cpd3294', formula='', name='adrenoyl-CoA [x]', compartment='')
Cpd3295 = Metabolite('Cpd3295', formula='', name='6,9,12,15,18-all-cis-tetracosapentaenoate [c]', compartment='')
Cpd3296 = Metabolite('Cpd3296', formula='', name='tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA [c]', compartment='')
Cpd3297 = Metabolite('Cpd3297', formula='', name='6,9,12,15,18-all-cis-tetracosapentaenoate [r]', compartment='')
Cpd3298 = Metabolite('Cpd3298', formula='', name='tetracosa-6,9,12,15,18-all-cis-pentaenoyl-CoA [r]', compartment='')
Cpd3299 = Metabolite('Cpd3299', formula='', name='3-oxoeicosa-cis,cis-11,14-dienoyl-CoA [c]', compartment='')
Cpd3300 = Metabolite('Cpd3300', formula='', name='3-oxoeicosa-cis,cis-11,14-dienoyl-CoA [r]', compartment='')
Cpd3301 = Metabolite('Cpd3301', formula='', name='(3S)-hydroxy-eicosa-cis,cis-11,14-dienoyl-CoA [r]', compartment='')
Cpd3302 = Metabolite('Cpd3302', formula='', name='trans,cis,cis-2,11,14-eicosatrienoyl-CoA [r]', compartment='')
Cpd3303 = Metabolite('Cpd3303', formula='', name='cis,cis-11,14-eicosadienoyl-CoA [r]', compartment='')
Cpd3304 = Metabolite('Cpd3304', formula='', name='3-oxo-docosa-cis,cis,cis-10,13,16-trienoyl-CoA [c]', compartment='')
Cpd3305 = Metabolite('Cpd3305', formula='', name='3-oxo-docosa-cis,cis,cis-10,13,16-trienoyl-CoA [r]', compartment='')
Cpd3306 = Metabolite('Cpd3306', formula='', name='3(S)-hydroxy-10,13,16-all-cis-docosatrienoyl-CoA [r]', compartment='')
Cpd3307 = Metabolite('Cpd3307', formula='', name='trans,cis,cis,cis-2,10,13,16-docosatetraenoyl-CoA [r]', compartment='')
Cpd3308 = Metabolite('Cpd3308', formula='', name='cis,cis,cis-10,13,16-docosatrienoyl-CoA [r]', compartment='')
Cpd3309 = Metabolite('Cpd3309', formula='', name='trans,cis,cis,cis,cis-2,10,13,16,19-docosapentaenoyl-CoA [r]', compartment='')
Cpd3310 = Metabolite('Cpd3310', formula='', name='all-cis-10,13,16,19-docosatetraenoyl-CoA [r]', compartment='')
Cpd3311 = Metabolite('Cpd3311', formula='', name='(8Z,11Z,14Z,17Z)-eicosatetraenoyl-CoA [c]', compartment='')
Cpd3312 = Metabolite('Cpd3312', formula='', name='3-oxo-docosa-10,13,16,19-all-cis-tetraenoyl-CoA [c]', compartment='')
Cpd3313 = Metabolite('Cpd3313', formula='', name='3-oxo-docosa-10,13,16,19-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3314 = Metabolite('Cpd3314', formula='', name='3(S)-hydroxy-docosa-10,13,16,19-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3315 = Metabolite('Cpd3315', formula='', name='trans,cis,cis,cis,cis-2,12,15,18,21-tetracosapentaenoyl-CoA [r]', compartment='')
Cpd3316 = Metabolite('Cpd3316', formula='', name='all-cis-12,15,18,21-tetracosatetraenoyl-CoA [r]', compartment='')
Cpd3317 = Metabolite('Cpd3317', formula='', name='all-cis-10,13,16,19-docosatetraenoyl-CoA [c]', compartment='')
Cpd3318 = Metabolite('Cpd3318', formula='', name='3-oxo-tetracosa-12,15,18,21-all-cis-tetraenoyl-CoA [c]', compartment='')
Cpd3319 = Metabolite('Cpd3319', formula='', name='3-oxo-tetracosa-12,15,18,21-all-cis-tetraenoyl-CoA [r]', compartment='')
Cpd3320 = Metabolite('Cpd3320', formula='', name='3(S)-hydroxy-4(R),8-dimethyl-nonanoyl-CoA [m]', compartment='')
Cpd3321 = Metabolite('Cpd3321', formula='', name='3-oxo-4(R),8-dimethyl-nonanoyl-CoA [m]', compartment='')
Cpd3322 = Metabolite('Cpd3322', formula='', name='3(S)-hydroxy-2(S),6-dimethyl-heptanoyl-CoA [m]', compartment='')
Cpd3323 = Metabolite('Cpd3323', formula='', name='3-oxo-(2S)-methylisocapryloyl-CoA [m]', compartment='')
Cpd3324 = Metabolite('Cpd3324', formula='', name='3(S)-hydroxy-4-methyl-pentanoyl-CoA [m]', compartment='')
Cpd3325 = Metabolite('Cpd3325', formula='', name='3-oxo-4-methyl-pentanoyl-CoA [m]', compartment='')
Cpd3326 = Metabolite('Cpd3326', formula='', name='4(R),8-dimethyl-nonanoyl-CoA [m]', compartment='')
Cpd3327 = Metabolite('Cpd3327', formula='', name='4(R),8-dimethyl-trans-2-nonenoyl-CoA [m]', compartment='')
Cpd3328 = Metabolite('Cpd3328', formula='', name='2(S),6-dimethyl-heptanoyl-CoA [m]', compartment='')
Cpd3329 = Metabolite('Cpd3329', formula='', name='2,6-dimethyl-trans-2-heptenoyl-CoA [m]', compartment='')
Cpd3330 = Metabolite('Cpd3330', formula='', name='4-methyl-pentanoyl-CoA [m]', compartment='')
Cpd3331 = Metabolite('Cpd3331', formula='', name='4-methyl-trans-2-pentenoyl-CoA [m]', compartment='')
Cpd3332 = Metabolite('Cpd3332', formula='', name='2(R),6-dimethyl-heptanoyl-CoA [m]', compartment='')
Cpd3333 = Metabolite('Cpd3333', formula='', name='3-oxo-11-cis-eicosenoyl-CoA [c]', compartment='')
Cpd3334 = Metabolite('Cpd3334', formula='', name='3-oxo-11-cis-eicosenoyl-CoA [r]', compartment='')
Cpd3335 = Metabolite('Cpd3335', formula='', name='11-cis-eicosenoyl-CoA [c]', compartment='')
Cpd3336 = Metabolite('Cpd3336', formula='', name='3-oxo-13cis-docosenoyl-CoA [c]', compartment='')
Cpd3337 = Metabolite('Cpd3337', formula='', name='11-cis-eicosenoyl-CoA [r]', compartment='')
Cpd3338 = Metabolite('Cpd3338', formula='', name='3-oxo-13cis-docosenoyl-CoA [r]', compartment='')
Cpd3339 = Metabolite('Cpd3339', formula='', name='cis-13-docosenoyl-CoA [c]', compartment='')
Cpd3340 = Metabolite('Cpd3340', formula='', name='3-oxo-cis-15-tetracosaenoyl-CoA [c]', compartment='')
Cpd3341 = Metabolite('Cpd3341', formula='', name='cis-13-docosenoyl-CoA [r]', compartment='')
Cpd3342 = Metabolite('Cpd3342', formula='', name='3-oxo-cis-15-tetracosaenoyl-CoA [r]', compartment='')
Cpd3343 = Metabolite('Cpd3343', formula='', name='3(S)-hydroxy-11-cis-eicosenoyl-CoA [r]', compartment='')
Cpd3344 = Metabolite('Cpd3344', formula='', name='trans,cis-2,11-eicosadienoyl-CoA [r]', compartment='')
Cpd3345 = Metabolite('Cpd3345', formula='', name='3(S)-hydroxy-13cis-docosenoyl-CoA [r]', compartment='')
Cpd3346 = Metabolite('Cpd3346', formula='', name='trans,cis-2,13-docosadienoyl-CoA [r]', compartment='')
Cpd3347 = Metabolite('Cpd3347', formula='', name='Hordenine [g]', compartment='')
Cpd3348 = Metabolite('Cpd3348', formula='', name='O2 [g]', compartment='')
Cpd3349 = Metabolite('Cpd3349', formula='', name='N,N-dimethyldopaminequinone [g]', compartment='')
Cpd3350 = Metabolite('Cpd3350', formula='', name='O2 [l]', compartment='')
Cpd3351 = Metabolite('Cpd3351', formula='', name='Hordenine [l]', compartment='')
Cpd3352 = Metabolite('Cpd3352', formula='', name='N,N-dimethyldopaminequinone [l]', compartment='')
Cpd3353 = Metabolite('Cpd3353', formula='', name='3(S)-hydroxy-cis-15-tetracosaenoyl-CoA [r]', compartment='')
Cpd3354 = Metabolite('Cpd3354', formula='', name='trans,cis-2,15-tetracosadienoyl-CoA [r]', compartment='')
Cpd3355 = Metabolite('Cpd3355', formula='', name='cis-15-tetracosenoyl-CoA [r]', compartment='')
Cpd3356 = Metabolite('Cpd3356', formula='', name='3-oxo-cis-9-octadecenoyl-CoA [c]', compartment='')
Cpd3357 = Metabolite('Cpd3357', formula='', name='3-oxo-cis-9-octadecenoyl-CoA [r]', compartment='')
Cpd3358 = Metabolite('Cpd3358', formula='', name='3(S)-hydroxy-cis-9-octadecenoyl-CoA [r]', compartment='')
Cpd3359 = Metabolite('Cpd3359', formula='', name='trans,cis-2,9-octadecadienoyl-CoA [r]', compartment='')
Cpd3360 = Metabolite('Cpd3360', formula='', name='3alpha,7alpha,12alpha-Trihydroxy-5beta-cholest-24-enoyl-CoA [m]', compartment='')
Cpd3361 = Metabolite('Cpd3361', formula='', name='24(R),25(R)-varanoyl-CoA [m]', compartment='')
Cpd3362 = Metabolite('Cpd3362', formula='', name='24(R),25(R)-varanoyl-CoA [x]', compartment='')
Cpd3363 = Metabolite('Cpd3363', formula='', name='24-oxo-25(R)-trihydroxycoprostanoyl-CoA [m]', compartment='')
Cpd3364 = Metabolite('Cpd3364', formula='', name='24-oxo-25(R)-trihydroxycoprostanoyl-CoA [x]', compartment='')
Cpd3365 = Metabolite('Cpd3365', formula='', name='5beta-cholestane-3alpha,7alpha,12alpha,27,27-pentaol [m]', compartment='')
Cpd3366 = Metabolite('Cpd3366', formula='', name='12(13)-EpOME [c]', compartment='')
Cpd3367 = Metabolite('Cpd3367', formula='', name='12,13-hydroxyoctadec-9(Z)-enoate [c]', compartment='')
Cpd3368 = Metabolite('Cpd3368', formula='', name='12(13)-EpOME [r]', compartment='')
Cpd3369 = Metabolite('Cpd3369', formula='', name='12,13-hydroxyoctadec-9(Z)-enoate [r]', compartment='')
Cpd3370 = Metabolite('Cpd3370', formula='', name='9(10)-EpOME [c]', compartment='')
Cpd3371 = Metabolite('Cpd3371', formula='', name='9,10-hydroxyoctadec-12(Z)-enoate [c]', compartment='')
Cpd3372 = Metabolite('Cpd3372', formula='', name='9(10)-EpOME [r]', compartment='')
Cpd3373 = Metabolite('Cpd3373', formula='', name='9,10-hydroxyoctadec-12(Z)-enoate [r]', compartment='')
Cpd3374 = Metabolite('Cpd3374', formula='', name='12-hydroxy-13-O-D-glucuronoside-octadec-9Z-enoate [c]', compartment='')
Cpd3375 = Metabolite('Cpd3375', formula='', name='12-hydroxy-13-O-D-glucuronoside-octadec-9Z-enoate [r]', compartment='')
Cpd3376 = Metabolite('Cpd3376', formula='', name='9-hydroxy-10-O-D-glucuronoside-12Z-octadecenoate [c]', compartment='')
Cpd3377 = Metabolite('Cpd3377', formula='', name='9-hydroxy-10-O-D-glucuronoside-12Z-octadecenoate [r]', compartment='')
Cpd3378 = Metabolite('Cpd3378', formula='', name='12-O-D-glucuronoside-13-hydroxyoctadec-9Z-enoate [c]', compartment='')
Cpd3379 = Metabolite('Cpd3379', formula='', name='12-O-D-glucuronoside-13-hydroxyoctadec-9Z-enoate [r]', compartment='')
Cpd3380 = Metabolite('Cpd3380', formula='', name='10-hydroxy-octadec-12Z-enoate-9-beta-D-glucuronide [c]', compartment='')
Cpd3381 = Metabolite('Cpd3381', formula='', name='10-hydroxy-octadec-12Z-enoate-9-beta-D-glucuronide [r]', compartment='')
Cpd3382 = Metabolite('Cpd3382', formula='', name='CDP-ethanolamine(1-) [g]', compartment='')
Cpd3383 = Metabolite('Cpd3383', formula='', name='1,2-Diacyl-sn-glycerol (didodecanoyl, n-C12:0) [g]', compartment='')
Cpd3384 = Metabolite('Cpd3384', formula='', name='phosphatidylethanolamine [g]', compartment='')
Cpd3385 = Metabolite('Cpd3385', formula='', name='CDP-ethanolamine(1-) [n]', compartment='')
Cpd3386 = Metabolite('Cpd3386', formula='', name='1,2-Diacyl-sn-glycerol (didodecanoyl, n-C12:0) [n]', compartment='')
Cpd3387 = Metabolite('Cpd3387', formula='', name='phosphatidylethanolamine [n]', compartment='')
Cpd3388 = Metabolite('Cpd3388', formula='', name='CDP-ethanolamine(1-) [r]', compartment='')
Cpd3389 = Metabolite('Cpd3389', formula='', name='1,2-Diacyl-sn-glycerol (didodecanoyl, n-C12:0) [r]', compartment='')
Cpd3390 = Metabolite('Cpd3390', formula='', name='CMP [r]', compartment='')
Cpd3391 = Metabolite('Cpd3391', formula='', name='myo-inositol [g]', compartment='')
Cpd3392 = Metabolite('Cpd3392', formula='', name='myo-inositol [r]', compartment='')
Cpd3393 = Metabolite('Cpd3393', formula='', name='5,6-EET [r]', compartment='')
Cpd3394 = Metabolite('Cpd3394', formula='', name='11,12-EET [r]', compartment='')
Cpd3395 = Metabolite('Cpd3395', formula='', name='8,9-EET [r]', compartment='')
Cpd3396 = Metabolite('Cpd3396', formula='', name='14,15-EET [r]', compartment='')
Cpd3397 = Metabolite('Cpd3397', formula='', name='phosphatidylserine [g]', compartment='')
Cpd3398 = Metabolite('Cpd3398', formula='', name='L-serine [g]', compartment='')
Cpd3399 = Metabolite('Cpd3399', formula='', name='phosphatidylserine [r]', compartment='')
Cpd3400 = Metabolite('Cpd3400', formula='', name='L-serine [r]', compartment='')
Cpd3401 = Metabolite('Cpd3401', formula='', name='6-trans-leukotriene B4 [c]', compartment='')
Cpd3402 = Metabolite('Cpd3402', formula='', name='5-oxo-6-trans-leukotriene B4 [c]', compartment='')
Cpd3403 = Metabolite('Cpd3403', formula='', name='6E-12-epi-leukotriene B4 [m]', compartment='')
Cpd3404 = Metabolite('Cpd3404', formula='', name='5-oxo-6E-12-epi-leukotriene B4 [m]', compartment='')
Cpd3405 = Metabolite('Cpd3405', formula='', name='6E-12-epi-leukotriene B4 [x]', compartment='')
Cpd3406 = Metabolite('Cpd3406', formula='', name='5-oxo-6E-12-epi-leukotriene B4 [x]', compartment='')
Cpd3407 = Metabolite('Cpd3407', formula='', name='5-oxo-6E-12-epi-leukotriene B4 [r]', compartment='')
Cpd3408 = Metabolite('Cpd3408', formula='', name='6,7-dihydro-5-oxo-12-epi-leukotriene B [r]', compartment='')
Cpd3409 = Metabolite('Cpd3409', formula='', name='6,7-dihydro-leukotriene B4 [r]', compartment='')
Cpd3410 = Metabolite('Cpd3410', formula='', name='6,7-dihydro-5-oxo-leukotriene B4 [r]', compartment='')
Cpd3411 = Metabolite('Cpd3411', formula='', name='5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [m]', compartment='')
Cpd3412 = Metabolite('Cpd3412', formula='', name='5-oxo-12(R)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA [m]', compartment='')
Cpd3413 = Metabolite('Cpd3413', formula='', name='5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [x]', compartment='')
Cpd3414 = Metabolite('Cpd3414', formula='', name='5-oxo-12(R)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA [x]', compartment='')
Cpd3415 = Metabolite('Cpd3415', formula='', name='6,7-dihydro-5-oxo-leukotriene B4 [c]', compartment='')
Cpd3416 = Metabolite('Cpd3416', formula='', name='5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [c]', compartment='')
Cpd3417 = Metabolite('Cpd3417', formula='', name='6,7-dihydro-5-oxo-12-epi-leukotriene B [m]', compartment='')
Cpd3418 = Metabolite('Cpd3418', formula='', name='5-oxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [r]', compartment='')
Cpd3419 = Metabolite('Cpd3419', formula='', name='6,7-dihydro-5-oxo-12-epi-leukotriene B [x]', compartment='')
Cpd3420 = Metabolite('Cpd3420', formula='', name='3,5-dioxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [m]', compartment='')
Cpd3421 = Metabolite('Cpd3421', formula='', name='3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA [m]', compartment='')
Cpd3422 = Metabolite('Cpd3422', formula='', name='3,5-dioxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [x]', compartment='')
Cpd3423 = Metabolite('Cpd3423', formula='', name='3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA [x]', compartment='')
Cpd3424 = Metabolite('Cpd3424', formula='', name='3(S),12(R)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA [m]', compartment='')
Cpd3425 = Metabolite('Cpd3425', formula='', name='3(S),12(R)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA [x]', compartment='')
Cpd3426 = Metabolite('Cpd3426', formula='', name='3(S),12(R)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA [c]', compartment='')
Cpd3427 = Metabolite('Cpd3427', formula='', name='3,5-dioxo-12(R)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [c]', compartment='')
Cpd3428 = Metabolite('Cpd3428', formula='', name='3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA [c]', compartment='')
Cpd3429 = Metabolite('Cpd3429', formula='', name='3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoate [c]', compartment='')
Cpd3430 = Metabolite('Cpd3430', formula='', name='3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoate [m]', compartment='')
Cpd3431 = Metabolite('Cpd3431', formula='', name='3-oxo-10(R)-hydroxy-octadeca-6E,8E,12Z-trienoate [x]', compartment='')
Cpd3432 = Metabolite('Cpd3432', formula='', name='3(S),12(S)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA [c]', compartment='')
Cpd3433 = Metabolite('Cpd3433', formula='', name='3,5-dioxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [c]', compartment='')
Cpd3434 = Metabolite('Cpd3434', formula='', name='3(S),12(S)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA [m]', compartment='')
Cpd3435 = Metabolite('Cpd3435', formula='', name='3,5-dioxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [m]', compartment='')
Cpd3436 = Metabolite('Cpd3436', formula='', name='3(S),12(S)-dihydroxy-5-oxo-eicosa-8E,10E,14Z-trienoyl-CoA [x]', compartment='')
Cpd3437 = Metabolite('Cpd3437', formula='', name='3,5-dioxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [x]', compartment='')
Cpd3438 = Metabolite('Cpd3438', formula='', name='5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [m]', compartment='')
Cpd3439 = Metabolite('Cpd3439', formula='', name='5-oxo-12(S)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA [m]', compartment='')
Cpd3440 = Metabolite('Cpd3440', formula='', name='5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [x]', compartment='')
Cpd3441 = Metabolite('Cpd3441', formula='', name='5-oxo-12(S)-hydroxy-eicosa-2E,8E,10E,14Z-tetraenoyl-CoA [x]', compartment='')
Cpd3442 = Metabolite('Cpd3442', formula='', name='3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA [m]', compartment='')
Cpd3443 = Metabolite('Cpd3443', formula='', name='3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA [x]', compartment='')
Cpd3444 = Metabolite('Cpd3444', formula='', name='3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoyl-CoA [c]', compartment='')
Cpd3445 = Metabolite('Cpd3445', formula='', name='3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoate [c]', compartment='')
Cpd3446 = Metabolite('Cpd3446', formula='', name='3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoate [m]', compartment='')
Cpd3447 = Metabolite('Cpd3447', formula='', name='3-oxo-10(S)-hydroxy-octadeca-6E,8E,12Z-trienoate [x]', compartment='')
Cpd3448 = Metabolite('Cpd3448', formula='', name='6,7-dihydro-5-oxo-12-epi-leukotriene B [c]', compartment='')
Cpd3449 = Metabolite('Cpd3449', formula='', name='5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [c]', compartment='')
Cpd3450 = Metabolite('Cpd3450', formula='', name='5-oxo-12(S)-hydroxy-eicosa-8E,10E,14Z-trienoyl-CoA [r]', compartment='')
Cpd3451 = Metabolite('Cpd3451', formula='', name='3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al [c]', compartment='')
Cpd3452 = Metabolite('Cpd3452', formula='', name='3alpha,7alpha,12alpha-trihydroxy-5beta-cholestan-27-al [r]', compartment='')
Cpd3453 = Metabolite('Cpd3453', formula='', name='13-hydroxy-alpha-tocotrienol [r]', compartment='')
Cpd3454 = Metabolite('Cpd3454', formula='', name='13-carboxy-alpha-tocotrienol [r]', compartment='')
Cpd3455 = Metabolite('Cpd3455', formula='', name='alpha-CEHC-glucuronide [e]', compartment='')
Cpd3456 = Metabolite('Cpd3456', formula='', name='3-carboxy-alpha-chromanol [e]', compartment='')
Cpd3457 = Metabolite('Cpd3457', formula='', name='alpha-CEHC-glucuronide [l]', compartment='')
Cpd3458 = Metabolite('Cpd3458', formula='', name='3-carboxy-alpha-chromanol [l]', compartment='')
Cpd3459 = Metabolite('Cpd3459', formula='', name='8(R)-hydroxy-hexadeca-4E,6E,10Z-trienoate [m]', compartment='')
Cpd3460 = Metabolite('Cpd3460', formula='', name='3-oxo-8(R)-hydroxy-hexadeca-6E,10Z-dienoate [m]', compartment='')
Cpd3461 = Metabolite('Cpd3461', formula='', name='6(R)-hydroxy-tetradeca-4E,8Z-dienoate [m]', compartment='')
Cpd3462 = Metabolite('Cpd3462', formula='', name='8(R)-hydroxy-hexadeca-2E,4E,6E,10Z-tetraenoate [m]', compartment='')
Cpd3463 = Metabolite('Cpd3463', formula='', name='8(R)-hydroxy-hexadeca-2E,6E,10Z-trienoate [m]', compartment='')
Cpd3464 = Metabolite('Cpd3464', formula='', name='3(S),8(R)-dihydroxy-6E,10Z-hexadecadienoate [m]', compartment='')
Cpd3465 = Metabolite('Cpd3465', formula='', name='3-oxo-6(R)-hydroxy-tetradec-8-cis-enoate [m]', compartment='')
Cpd3466 = Metabolite('Cpd3466', formula='', name='4(R)-hydroxy-dodec-6Z-enoate [m]', compartment='')
Cpd3467 = Metabolite('Cpd3467', formula='', name='6(R)-hydroxy-tetradeca-2E,4E,8Z-trienoate [m]', compartment='')
Cpd3468 = Metabolite('Cpd3468', formula='', name='6(R)-hydroxy-tetradeca-2E,8Z-dienoate [m]', compartment='')
Cpd3469 = Metabolite('Cpd3469', formula='', name='3(S),6(R)-dihydroxy-tetradec-8Z-enoate [m]', compartment='')
Cpd3470 = Metabolite('Cpd3470', formula='', name='3-oxo-8(S)-hydroxy-hexadeca-6E,10Z-dienoate [m]', compartment='')
Cpd3471 = Metabolite('Cpd3471', formula='', name='6(S)-hydroxy-tetradeca-4E,8Z-dienoate [m]', compartment='')
Cpd3472 = Metabolite('Cpd3472', formula='', name='6(S)-hydroxy-tetradeca-2E,4E,8Z-trienoate [m]', compartment='')
Cpd3473 = Metabolite('Cpd3473', formula='', name='8(S)-hydroxy-hexadeca-4E,6E-10Z-trienoate [m]', compartment='')
Cpd3474 = Metabolite('Cpd3474', formula='', name='8(S)-hydroxy-hexadeca-2E,4E,6E,10Z-tetraenoate [m]', compartment='')
Cpd3475 = Metabolite('Cpd3475', formula='', name='8(S)-hydroxy-hexadeca-2E,6E,10Z-trienoate [m]', compartment='')
Cpd3476 = Metabolite('Cpd3476', formula='', name='3(S),8(S)-dihydroxy-6E,10Z-hexadecadienoate [m]', compartment='')
Cpd3477 = Metabolite('Cpd3477', formula='', name='3-oxo-6(S)-hydroxy-tetradec-8Z-enoate [m]', compartment='')
Cpd3478 = Metabolite('Cpd3478', formula='', name='4(S)-hydroxy-dodec-6Z-enoate [m]', compartment='')
Cpd3479 = Metabolite('Cpd3479', formula='', name='6(S)-hydroxy-tetradeca-2E,8Z-dienoate [m]', compartment='')
Cpd3480 = Metabolite('Cpd3480', formula='', name='3(S),6(S)-dihydroxy-tetradec-8Z-enoate [m]', compartment='')
Cpd3481 = Metabolite('Cpd3481', formula='', name='10,11-dihydro-12-oxo-LTB4 [c]', compartment='')
Cpd3482 = Metabolite('Cpd3482', formula='', name='10,11-dihydro-leukotriene B4 [c]', compartment='')
Cpd3483 = Metabolite('Cpd3483', formula='', name='10,11-dihydro-12-oxo-LTB4 [m]', compartment='')
Cpd3484 = Metabolite('Cpd3484', formula='', name='10,11-dihydro-12-epi-leukotriene B4 [m]', compartment='')
Cpd3485 = Metabolite('Cpd3485', formula='', name='10,11-dihydro-12-oxo-LTB4 [x]', compartment='')
Cpd3486 = Metabolite('Cpd3486', formula='', name='10,11-dihydro-12-epi-leukotriene B4 [x]', compartment='')
Cpd3487 = Metabolite('Cpd3487', formula='', name='10,11-dihydro-12-epi-leukotriene B4 [r]', compartment='')
Cpd3488 = Metabolite('Cpd3488', formula='', name='20-OH-10,11-dihydro-leukotriene B4 [r]', compartment='')
Cpd3489 = Metabolite('Cpd3489', formula='', name='10,11-dihydro-20-dihydroxy-LTB4 [r]', compartment='')
Cpd3490 = Metabolite('Cpd3490', formula='', name='10,11-dihydro-20-trihydroxy-leukotriene B4 [r]', compartment='')
Cpd3491 = Metabolite('Cpd3491', formula='', name='13-hydroxy-gama-tocotrienol [r]', compartment='')
Cpd3492 = Metabolite('Cpd3492', formula='', name='13-carboxy-gama-tocotrienol [r]', compartment='')
Cpd3493 = Metabolite('Cpd3493', formula='', name='3-oxo-5(S),12(R)-dihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA [m]', compartment='')
Cpd3494 = Metabolite('Cpd3494', formula='', name='3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA [m]', compartment='')
Cpd3495 = Metabolite('Cpd3495', formula='', name='3-oxo-5(S),12(R)-dihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA [x]', compartment='')
Cpd3496 = Metabolite('Cpd3496', formula='', name='3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA [x]', compartment='')
Cpd3497 = Metabolite('Cpd3497', formula='', name='3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoyl-CoA [c]', compartment='')
Cpd3498 = Metabolite('Cpd3498', formula='', name='3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoate [c]', compartment='')
Cpd3499 = Metabolite('Cpd3499', formula='', name='3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoate [m]', compartment='')
Cpd3500 = Metabolite('Cpd3500', formula='', name='3(S),10(R)-OH-octadeca-6-trans-4,12-cis-trienoate [x]', compartment='')
Cpd3501 = Metabolite('Cpd3501', formula='', name='10,11-dihydro-LTB4-CoA [m]', compartment='')
Cpd3502 = Metabolite('Cpd3502', formula='', name='5(S),12(R)-dihydroxy-eicosa-2,8-trans-6,14-cis-tetraenoyl-CoA [m]', compartment='')
Cpd3503 = Metabolite('Cpd3503', formula='', name='10,11-dihydro-LTB4-CoA [x]', compartment='')
Cpd3504 = Metabolite('Cpd3504', formula='', name='5(S),12(R)-dihydroxy-eicosa-2,8-trans-6,14-cis-tetraenoyl-CoA [x]', compartment='')
Cpd3505 = Metabolite('Cpd3505', formula='', name='10,11-dihydro-LTB4-CoA [c]', compartment='')
Cpd3506 = Metabolite('Cpd3506', formula='', name='10,11-dihydro-LTB4-CoA [r]', compartment='')
Cpd3507 = Metabolite('Cpd3507', formula='', name='3(S),5(S),12(R)-trihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA [m]', compartment='')
Cpd3508 = Metabolite('Cpd3508', formula='', name='3(S),5(S),12(R)-trihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA [x]', compartment='')
Cpd3509 = Metabolite('Cpd3509', formula='', name='3(S),5(S),12(R)-trihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA [c]', compartment='')
Cpd3510 = Metabolite('Cpd3510', formula='', name='3-oxo-5(S),12(R)-dihydroxy-eicosa-8-trans-6,14-cis-trienoyl-CoA [c]', compartment='')
Cpd3511 = Metabolite('Cpd3511', formula='', name='alpha-tocotrienol [r]', compartment='')
Cpd3512 = Metabolite('Cpd3512', formula='', name='5-oxo-12(S)-hydroxy-eicosa-6E,8Z,10E,14Z-tetraenoate [c]', compartment='')
Cpd3513 = Metabolite('Cpd3513', formula='', name='5-oxo-12(S)-hydroxy-eicosa-6E,8Z,10E,14Z-tetraenoate [m]', compartment='')
Cpd3514 = Metabolite('Cpd3514', formula='', name='5-oxo-12(S)-hydroxy-eicosa-6E,8Z,10E,14Z-tetraenoate [x]', compartment='')
Cpd3515 = Metabolite('Cpd3515', formula='', name='5(S)-HETE [r]', compartment='')
Cpd3516 = Metabolite('Cpd3516', formula='', name='20-hydroxy-5S-HETE [r]', compartment='')
Cpd3517 = Metabolite('Cpd3517', formula='', name='5(S)-HETE [n]', compartment='')
Cpd3518 = Metabolite('Cpd3518', formula='', name='5,15-DiHETE [n]', compartment='')
Cpd3519 = Metabolite('Cpd3519', formula='', name='5(S)-HETE [c]', compartment='')
Cpd3520 = Metabolite('Cpd3520', formula='', name='5-oxo-(6E,8Z,11Z,14Z)-eicosatetraenoic acid [c]', compartment='')
Cpd3521 = Metabolite('Cpd3521', formula='', name='5(S)-HETE [m]', compartment='')
Cpd3522 = Metabolite('Cpd3522', formula='', name='5-oxo-(6E,8Z,11Z,14Z)-eicosatetraenoic acid [m]', compartment='')
Cpd3523 = Metabolite('Cpd3523', formula='', name='5(S)-HETE [x]', compartment='')
Cpd3524 = Metabolite('Cpd3524', formula='', name='5-oxo-(6E,8Z,11Z,14Z)-eicosatetraenoic acid [x]', compartment='')
Cpd3525 = Metabolite('Cpd3525', formula='', name='timnodonate [n]', compartment='')
Cpd3526 = Metabolite('Cpd3526', formula='', name='5-hydroperoxy-EPA [n]', compartment='')
Cpd3527 = Metabolite('Cpd3527', formula='', name='15(R)-hydroperoxy-EPE [n]', compartment='')
Cpd3528 = Metabolite('Cpd3528', formula='', name='15(R)-hydroperoxy-EPE [r]', compartment='')
Cpd3529 = Metabolite('Cpd3529', formula='', name='timnodonate [x]', compartment='')
Cpd3530 = Metabolite('Cpd3530', formula='', name='15(R)-hydroperoxy-EPE [x]', compartment='')
Cpd3531 = Metabolite('Cpd3531', formula='', name='PGH3 [n]', compartment='')
Cpd3532 = Metabolite('Cpd3532', formula='', name='Prostaglandin E3 [n]', compartment='')
Cpd3533 = Metabolite('Cpd3533', formula='', name='15(R)-HEPE [n]', compartment='')
Cpd3534 = Metabolite('Cpd3534', formula='', name='5(S),6(S)-epoxy-15(R)-HEPE [n]', compartment='')
Cpd3535 = Metabolite('Cpd3535', formula='', name='5(S),6(S)-epoxy-15(R)-HEPE [c]', compartment='')
Cpd3536 = Metabolite('Cpd3536', formula='', name='15-epi-lipoxin B5 [c]', compartment='')
Cpd3537 = Metabolite('Cpd3537', formula='', name='5(S),6(S)-epoxy-15(R)-HEPE [r]', compartment='')
Cpd3538 = Metabolite('Cpd3538', formula='', name='15-epi-lipoxin B5 [r]', compartment='')
Cpd3539 = Metabolite('Cpd3539', formula='', name='5(S),6(S)-epoxy-15(R)-HEPE [x]', compartment='')
Cpd3540 = Metabolite('Cpd3540', formula='', name='15-epi-lipoxin B5 [x]', compartment='')
Cpd3541 = Metabolite('Cpd3541', formula='', name='15-epi-lipoxin A5 [c]', compartment='')
Cpd3542 = Metabolite('Cpd3542', formula='', name='15-epi-lipoxin A5 [r]', compartment='')
Cpd3543 = Metabolite('Cpd3543', formula='', name='15-epi-lipoxin A5 [x]', compartment='')
Cpd3544 = Metabolite('Cpd3544', formula='', name='5-HEPE [c]', compartment='')
Cpd3545 = Metabolite('Cpd3545', formula='', name='5-oxo-EPE [c]', compartment='')
Cpd3546 = Metabolite('Cpd3546', formula='', name='5-HEPE [x]', compartment='')
Cpd3547 = Metabolite('Cpd3547', formula='', name='5-oxo-EPE [x]', compartment='')
Cpd3548 = Metabolite('Cpd3548', formula='', name='18R-HEPE [n]', compartment='')
Cpd3549 = Metabolite('Cpd3549', formula='', name='5,6-epoxy,18R-HEPE [n]', compartment='')
Cpd3550 = Metabolite('Cpd3550', formula='', name='leukotriene B5 [r]', compartment='')
Cpd3551 = Metabolite('Cpd3551', formula='', name='20-hydroxy-leukotriene B5 [r]', compartment='')
Cpd3552 = Metabolite('Cpd3552', formula='', name='12-hydroxyeicosatetraenoate [n]', compartment='')
Cpd3553 = Metabolite('Cpd3553', formula='', name='leukotriene B4(1-) [n]', compartment='')
Cpd3554 = Metabolite('Cpd3554', formula='', name='5,12,20-TriHETE [r]', compartment='')
Cpd3555 = Metabolite('Cpd3555', formula='', name='(5Z,9E,14Z)-(8xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate [r]', compartment='')
Cpd3556 = Metabolite('Cpd3556', formula='', name='20-OH-hepoxilin A3 [r]', compartment='')
Cpd3557 = Metabolite('Cpd3557', formula='', name='(5Z,9E,14Z)-(8xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate [c]', compartment='')
Cpd3558 = Metabolite('Cpd3558', formula='', name='(5Z,9E,12S,14Z)-8,11,12-trihydroxyicosa-5,9,14-trienoate [c]', compartment='')
Cpd3559 = Metabolite('Cpd3559', formula='', name='(5Z,9E,12S,14Z)-8,11,12-trihydroxyicosa-5,9,14-trienoate [r]', compartment='')
Cpd3560 = Metabolite('Cpd3560', formula='', name='(5Z,9E,14Z)-(8xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate [x]', compartment='')
Cpd3561 = Metabolite('Cpd3561', formula='', name='(5Z,9E,12S,14Z)-8,11,12-trihydroxyicosa-5,9,14-trienoate [x]', compartment='')
Cpd3562 = Metabolite('Cpd3562', formula='', name='12(S)-HPETE(1-) [c]', compartment='')
Cpd3563 = Metabolite('Cpd3563', formula='', name='12(S)-HPETE(1-) [e]', compartment='')
Cpd3564 = Metabolite('Cpd3564', formula='', name='(5Z,9E,14Z)-(8xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate [e]', compartment='')
Cpd3565 = Metabolite('Cpd3565', formula='', name='12(S)-HPETE(1-) [m]', compartment='')
Cpd3566 = Metabolite('Cpd3566', formula='', name='(5Z,9E,14Z)-(8xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate [m]', compartment='')
Cpd3567 = Metabolite('Cpd3567', formula='', name='12(S)-HPETE(1-) [n]', compartment='')
Cpd3568 = Metabolite('Cpd3568', formula='', name='(5Z,9E,14Z)-(8xi,11R,12S)-11,12-epoxy-8-hydroxyicosa-5,9,14-trienoate [n]', compartment='')
Cpd3569 = Metabolite('Cpd3569', formula='', name='hepoxilin A3-C [m]', compartment='')
Cpd3570 = Metabolite('Cpd3570', formula='', name='hepoxilin A3-C [r]', compartment='')
Cpd3571 = Metabolite('Cpd3571', formula='', name='hepoxilin A3-C [x]', compartment='')
Cpd3572 = Metabolite('Cpd3572', formula='', name='12-hydroxyeicosatetraenoate [r]', compartment='')
Cpd3573 = Metabolite('Cpd3573', formula='', name='12,20-DiHETE [r]', compartment='')
Cpd3574 = Metabolite('Cpd3574', formula='', name='9-deoxy-delta12-PGD2 [m]', compartment='')
Cpd3575 = Metabolite('Cpd3575', formula='', name='S-(9-deoxy-delta12-PGD2)-glutathione [m]', compartment='')
Cpd3576 = Metabolite('Cpd3576', formula='', name='9-deoxy-delta12-PGD2 [r]', compartment='')
Cpd3577 = Metabolite('Cpd3577', formula='', name='S-(9-deoxy-delta12-PGD2)-glutathione [r]', compartment='')
Cpd3578 = Metabolite('Cpd3578', formula='', name='Prostaglandin J2 [m]', compartment='')
Cpd3579 = Metabolite('Cpd3579', formula='', name='S-(PGJ2)-glutathione [m]', compartment='')
Cpd3580 = Metabolite('Cpd3580', formula='', name='Prostaglandin J2 [r]', compartment='')
Cpd3581 = Metabolite('Cpd3581', formula='', name='S-(PGJ2)-glutathione [r]', compartment='')
Cpd3582 = Metabolite('Cpd3582', formula='', name='delta12-Prostaglandin J2 [m]', compartment='')
Cpd3583 = Metabolite('Cpd3583', formula='', name='S-(9-deoxy-delta9,12-PGD2)-glutathione [m]', compartment='')
Cpd3584 = Metabolite('Cpd3584', formula='', name='delta12-Prostaglandin J2 [r]', compartment='')
Cpd3585 = Metabolite('Cpd3585', formula='', name='S-(9-deoxy-delta9,12-PGD2)-glutathione [r]', compartment='')
Cpd3586 = Metabolite('Cpd3586', formula='', name='Prostaglandin A2 [m]', compartment='')
Cpd3587 = Metabolite('Cpd3587', formula='', name='S-(PGA2)-glutathione [m]', compartment='')
Cpd3588 = Metabolite('Cpd3588', formula='', name='Prostaglandin A2 [r]', compartment='')
Cpd3589 = Metabolite('Cpd3589', formula='', name='S-(PGA2)-glutathione [r]', compartment='')
Cpd3590 = Metabolite('Cpd3590', formula='', name='prostaglandin A1(1-) [m]', compartment='')
Cpd3591 = Metabolite('Cpd3591', formula='', name='S-(PGA1)-glutathione [m]', compartment='')
Cpd3592 = Metabolite('Cpd3592', formula='', name='prostaglandin A1(1-) [r]', compartment='')
Cpd3593 = Metabolite('Cpd3593', formula='', name='S-(PGA1)-glutathione [r]', compartment='')
Cpd3594 = Metabolite('Cpd3594', formula='', name='18,20-dioxo-20-CoA-leukotriene B4 [m]', compartment='')
Cpd3595 = Metabolite('Cpd3595', formula='', name='18-CoA-18-oxo-dinorleukotriene B4 [m]', compartment='')
Cpd3596 = Metabolite('Cpd3596', formula='', name='18,20-dioxo-20-CoA-leukotriene B4 [x]', compartment='')
Cpd3597 = Metabolite('Cpd3597', formula='', name='18-CoA-18-oxo-dinorleukotriene B4 [x]', compartment='')
Cpd3598 = Metabolite('Cpd3598', formula='', name='18-CoA-18-oxo-dinorleukotriene B4 [c]', compartment='')
Cpd3599 = Metabolite('Cpd3599', formula='', name='omega-carboxy-trinor-leukotriene B4 [c]', compartment='')
Cpd3600 = Metabolite('Cpd3600', formula='', name='omega-carboxy-trinor-leukotriene B4 [m]', compartment='')
Cpd3601 = Metabolite('Cpd3601', formula='', name='omega-carboxy-trinor-leukotriene B4 [x]', compartment='')
Cpd3602 = Metabolite('Cpd3602', formula='', name='20-CoA-20-oxo-leukotriene B4 [m]', compartment='')
Cpd3603 = Metabolite('Cpd3603', formula='', name='18E-20-oxo-20-CoA-LTB4 [m]', compartment='')
Cpd3604 = Metabolite('Cpd3604', formula='', name='20-CoA-20-oxo-leukotriene B4 [x]', compartment='')
Cpd3605 = Metabolite('Cpd3605', formula='', name='18E-20-oxo-20-CoA-LTB4 [x]', compartment='')
Cpd3606 = Metabolite('Cpd3606', formula='', name='20-CoA-20-oxo-leukotriene B4 [c]', compartment='')
Cpd3607 = Metabolite('Cpd3607', formula='', name='20-CoA-20-oxo-leukotriene B4 [r]', compartment='')
Cpd3608 = Metabolite('Cpd3608', formula='', name='20-carboxy-leukotriene-B4 [x]', compartment='')
Cpd3609 = Metabolite('Cpd3609', formula='', name='20-CoA-20-oxo-18R-hydroxyleucotriene B4 [m]', compartment='')
Cpd3610 = Metabolite('Cpd3610', formula='', name='20-CoA-20-oxo-18R-hydroxyleucotriene B4 [x]', compartment='')
Cpd3611 = Metabolite('Cpd3611', formula='', name='20-CoA-20-oxo-18R-hydroxyleucotriene B4 [c]', compartment='')
Cpd3612 = Metabolite('Cpd3612', formula='', name='18,20-dioxo-20-CoA-leukotriene B4 [c]', compartment='')
Cpd3613 = Metabolite('Cpd3613', formula='', name='(8Z,11Z,14Z)-eicosatrienoic acid [r]', compartment='')
Cpd3614 = Metabolite('Cpd3614', formula='', name='prostaglandin G1 [r]', compartment='')
Cpd3615 = Metabolite('Cpd3615', formula='', name='CoA-20-COOH-18-oxo-LTE4 [x]', compartment='')
Cpd3616 = Metabolite('Cpd3616', formula='', name='18(R)-hydroxy-20-oxo-20-CoA-LTE4 [x]', compartment='')
Cpd3617 = Metabolite('Cpd3617', formula='', name='16,18-oxo-18-CoA-dinor-LTE4 [x]', compartment='')
Cpd3618 = Metabolite('Cpd3618', formula='', name='16(S)-hydroxy-18-oxo-18-CoA-LTE4 [x]', compartment='')
Cpd3619 = Metabolite('Cpd3619', formula='', name='CoA-omega-COOH-dinor-LTE4 [x]', compartment='')
Cpd3620 = Metabolite('Cpd3620', formula='', name='18-carboxy-dinor-LTE4 [x]', compartment='')
Cpd3621 = Metabolite('Cpd3621', formula='', name='CoA-20-COOH-LTE4 [x]', compartment='')
Cpd3622 = Metabolite('Cpd3622', formula='', name='18E-20-oxo-20-CoA-LTE4 [x]', compartment='')
Cpd3623 = Metabolite('Cpd3623', formula='', name='20-COOH-leukotriene E4 [x]', compartment='')
Cpd3624 = Metabolite('Cpd3624', formula='', name='CoA-18-COOH-16E-dinor-LTE5 [x]', compartment='')
Cpd3625 = Metabolite('Cpd3625', formula='', name='CoA-18-COOH-15E-dinor-LTE4 [x]', compartment='')
Cpd3626 = Metabolite('Cpd3626', formula='', name='16E-18-oxo-18-CoA-dinor-LTE4 [x]', compartment='')
Cpd3627 = Metabolite('Cpd3627', formula='', name='13E-tetranor-16-oxo-16-CoA-LTE4 [x]', compartment='')
Cpd3628 = Metabolite('Cpd3628', formula='', name='13E-tetranor-16-carboxy-LTE4 [x]', compartment='')
Cpd3629 = Metabolite('Cpd3629', formula='', name='prostaglandin H1 [n]', compartment='')
Cpd3630 = Metabolite('Cpd3630', formula='', name='prostaglandin E1(1-) [n]', compartment='')
Cpd3631 = Metabolite('Cpd3631', formula='', name='12-oxo-20-carboxy-leukotriene B4 [c]', compartment='')
Cpd3632 = Metabolite('Cpd3632', formula='', name='12-oxo-10,11-dihydro-20-COOH-LTB4 [c]', compartment='')
Cpd3633 = Metabolite('Cpd3633', formula='', name='12-oxo-20-carboxy-leukotriene B4 [m]', compartment='')
Cpd3634 = Metabolite('Cpd3634', formula='', name='12-oxo-10,11-dihydro-20-COOH-LTB4 [m]', compartment='')
Cpd3635 = Metabolite('Cpd3635', formula='', name='12-oxo-20-carboxy-leukotriene B4 [x]', compartment='')
Cpd3636 = Metabolite('Cpd3636', formula='', name='12-oxo-10,11-dihydro-20-COOH-LTB4 [x]', compartment='')
Cpd3637 = Metabolite('Cpd3637', formula='', name='20-COOH-10,11-dihydro-LTB4 [c]', compartment='')
Cpd3638 = Metabolite('Cpd3638', formula='', name='20-COOH-10,11-dihydro-LTB4 [m]', compartment='')
Cpd3639 = Metabolite('Cpd3639', formula='', name='20-COOH-10,11-dihydro-LTB4 [x]', compartment='')
Cpd3640 = Metabolite('Cpd3640', formula='', name='cis-6-dodecenoyl-CoA [x]', compartment='')
Cpd3641 = Metabolite('Cpd3641', formula='', name='trans,cis-deca-2,4-dienoyl-CoA [c]', compartment='')
Cpd3642 = Metabolite('Cpd3642', formula='', name='trans-3-decenoyl-CoA [c]', compartment='')
Cpd3643 = Metabolite('Cpd3643', formula='', name='trans-3-decenoyl-CoA [m]', compartment='')
Cpd3644 = Metabolite('Cpd3644', formula='', name='trans-3-decenoyl-CoA [x]', compartment='')
Cpd3645 = Metabolite('Cpd3645', formula='', name='selenomethionine [r]', compartment='')
Cpd3646 = Metabolite('Cpd3646', formula='', name='Selenomethionine se-oxide [r]', compartment='')
Cpd3647 = Metabolite('Cpd3647', formula='', name='Benzpyrene [r]', compartment='')
Cpd3648 = Metabolite('Cpd3648', formula='', name='Benzo[a]pyrene-2,3-oxide [r]', compartment='')
Cpd3649 = Metabolite('Cpd3649', formula='', name='Benzo[a]pyrene-2,3-oxide [c]', compartment='')
Cpd3650 = Metabolite('Cpd3650', formula='', name='Benzo[a]pyrene-2,3-diol [c]', compartment='')
Cpd3651 = Metabolite('Cpd3651', formula='', name='Benzo[a]pyrene-2,3-diol [r]', compartment='')
Cpd3652 = Metabolite('Cpd3652', formula='', name='Benzo[a]pyrene-2,3-oxide [x]', compartment='')
Cpd3653 = Metabolite('Cpd3653', formula='', name='Benzo[a]pyrene-2,3-diol [x]', compartment='')
Cpd3654 = Metabolite('Cpd3654', formula='', name='Benzo[a]pyrene-4,5-oxide [c]', compartment='')
Cpd3655 = Metabolite('Cpd3655', formula='', name='Benzo[a]pyrene-4,5-diol [c]', compartment='')
Cpd3656 = Metabolite('Cpd3656', formula='', name='Benzo[a]pyrene-4,5-oxide [r]', compartment='')
Cpd3657 = Metabolite('Cpd3657', formula='', name='Benzo[a]pyrene-4,5-diol [r]', compartment='')
Cpd3658 = Metabolite('Cpd3658', formula='', name='Benzo[a]pyrene-4,5-oxide [x]', compartment='')
Cpd3659 = Metabolite('Cpd3659', formula='', name='Benzo[a]pyrene-4,5-diol [x]', compartment='')
Cpd3660 = Metabolite('Cpd3660', formula='', name='Benzo[a]pyrene-9,10-oxide [c]', compartment='')
Cpd3661 = Metabolite('Cpd3661', formula='', name='Benzo[a]pyrene-9,10-diol [c]', compartment='')
Cpd3662 = Metabolite('Cpd3662', formula='', name='Benzo[a]pyrene-9,10-oxide [r]', compartment='')
Cpd3663 = Metabolite('Cpd3663', formula='', name='Benzo[a]pyrene-9,10-diol [r]', compartment='')
Cpd3664 = Metabolite('Cpd3664', formula='', name='Benzo[a]pyrene-9,10-oxide [x]', compartment='')
Cpd3665 = Metabolite('Cpd3665', formula='', name='Benzo[a]pyrene-9,10-diol [x]', compartment='')
Cpd3666 = Metabolite('Cpd3666', formula='', name='7,12-Dimethylbenz[a]anthracene [r]', compartment='')
Cpd3667 = Metabolite('Cpd3667', formula='', name='7,12-Dimethylbenz[a]anthracene-3,4-oxide [r]', compartment='')
Cpd3668 = Metabolite('Cpd3668', formula='', name='7,12-Dimethylbenz[a]anthracene-3,4-oxide [c]', compartment='')
Cpd3669 = Metabolite('Cpd3669', formula='', name='7,12-Dimethylbenz[a]anthracene-3,4-diol [c]', compartment='')
Cpd3670 = Metabolite('Cpd3670', formula='', name='7,12-Dimethylbenz[a]anthracene-3,4-diol [r]', compartment='')
Cpd3671 = Metabolite('Cpd3671', formula='', name='7,12-Dimethylbenz[a]anthracene-3,4-oxide [x]', compartment='')
Cpd3672 = Metabolite('Cpd3672', formula='', name='7,12-Dimethylbenz[a]anthracene-3,4-diol [x]', compartment='')
Cpd3673 = Metabolite('Cpd3673', formula='', name='7,12-Dimethylbenz[a]anthracene-3,4-diol-1,2-epoxide [r]', compartment='')
Cpd3674 = Metabolite('Cpd3674', formula='', name='Dibenzo[a,l]pyrene [r]', compartment='')
Cpd3675 = Metabolite('Cpd3675', formula='', name='Dibenzo[a,l]pyrene-11,12-epoxide [r]', compartment='')
Cpd3676 = Metabolite('Cpd3676', formula='', name='Dibenzo[a,l]pyrene-11,12-epoxide [c]', compartment='')
Cpd3677 = Metabolite('Cpd3677', formula='', name='Dibenzo[a,l]pyrene-11,12-diol [c]', compartment='')
Cpd3678 = Metabolite('Cpd3678', formula='', name='Dibenzo[a,l]pyrene-11,12-diol [r]', compartment='')
Cpd3679 = Metabolite('Cpd3679', formula='', name='Dibenzo[a,l]pyrene-11,12-epoxide [x]', compartment='')
Cpd3680 = Metabolite('Cpd3680', formula='', name='Dibenzo[a,l]pyrene-11,12-diol [x]', compartment='')
Cpd3681 = Metabolite('Cpd3681', formula='', name='Dibenzo[a,l]pyrene-11,12-diol-13,14-epoxide [r]', compartment='')
Cpd3682 = Metabolite('Cpd3682', formula='', name='2-methylcrotonoyl-CoA(4-) [c]', compartment='')
Cpd3683 = Metabolite('Cpd3683', formula='', name='(R)-3-hydroxybutanoyl-CoA(4-) [c]', compartment='')
Cpd3684 = Metabolite('Cpd3684', formula='', name='adipoylcoa [x]', compartment='')
Cpd3685 = Metabolite('Cpd3685', formula='', name='adipoyl carnitine [x]', compartment='')
Cpd3686 = Metabolite('Cpd3686', formula='', name='adipic acid [x]', compartment='')
Cpd3687 = Metabolite('Cpd3687', formula='', name='decanoyl carnitine [c]', compartment='')
Cpd3688 = Metabolite('Cpd3688', formula='', name='decenoylcoa [c]', compartment='')
Cpd3689 = Metabolite('Cpd3689', formula='', name='decenoyl carnitine [c]', compartment='')
Cpd3690 = Metabolite('Cpd3690', formula='', name='decadienoylcoa [c]', compartment='')
Cpd3691 = Metabolite('Cpd3691', formula='', name='decadienoyl carnitine [c]', compartment='')
Cpd3692 = Metabolite('Cpd3692', formula='', name='sebacoyl carnitine [x]', compartment='')
Cpd3693 = Metabolite('Cpd3693', formula='', name='sebacoyl carnitine [c]', compartment='')
Cpd3694 = Metabolite('Cpd3694', formula='', name='sebacoylcoa [c]', compartment='')
Cpd3695 = Metabolite('Cpd3695', formula='', name='(S)-3-Hydroxydecanoyl-CoA [c]', compartment='')
Cpd3696 = Metabolite('Cpd3696', formula='', name='3-hydroxydecanoylcarnitine [c]', compartment='')
Cpd3697 = Metabolite('Cpd3697', formula='', name='lauroyl carnitine [c]', compartment='')
Cpd3698 = Metabolite('Cpd3698', formula='', name='trans-Dodec-2-enoyl-CoA [c]', compartment='')
Cpd3699 = Metabolite('Cpd3699', formula='', name='dodecenoyl carnitine [c]', compartment='')
Cpd3700 = Metabolite('Cpd3700', formula='', name='dodecanedioylcoa [x]', compartment='')
Cpd3701 = Metabolite('Cpd3701', formula='', name='Dodecanedioic acid [x]', compartment='')
Cpd3702 = Metabolite('Cpd3702', formula='', name='(S)-3-Hydroxydodecanoyl-CoA [c]', compartment='')
Cpd3703 = Metabolite('Cpd3703', formula='', name='3-hydroxydodecanoylcarnitine [c]', compartment='')
Cpd3704 = Metabolite('Cpd3704', formula='', name='myristoyl carnitine [c]', compartment='')
Cpd3705 = Metabolite('Cpd3705', formula='', name='3-hydroxy tetradecenoyl-7-coa [m]', compartment='')
Cpd3706 = Metabolite('Cpd3706', formula='', name='3-hydroxy tetradecenoyl-7-coa [c]', compartment='')
Cpd3707 = Metabolite('Cpd3707', formula='', name='tetradecenoylcoa [c]', compartment='')
Cpd3708 = Metabolite('Cpd3708', formula='', name='tetradecenoyl carnitine [c]', compartment='')
Cpd3709 = Metabolite('Cpd3709', formula='', name='3-hydroxy tetradecenoyl-7-carnitine [c]', compartment='')
Cpd3710 = Metabolite('Cpd3710', formula='', name='3-hydroxy trans5,8tetradecadienoyl coa [m]', compartment='')
Cpd3711 = Metabolite('Cpd3711', formula='', name='3-hydroxy trans5,8tetradecadienoyl coa [c]', compartment='')
Cpd3712 = Metabolite('Cpd3712', formula='', name='tetradecadienoylcoa [c]', compartment='')
Cpd3713 = Metabolite('Cpd3713', formula='', name='tetradecadienoyl carnitine [c]', compartment='')
Cpd3714 = Metabolite('Cpd3714', formula='', name='3-hydroxy trans5,8tetradecadienoyl carnitine [c]', compartment='')
Cpd3715 = Metabolite('Cpd3715', formula='', name='(S)-3-Hydroxytetradecanoyl-CoA [c]', compartment='')
Cpd3716 = Metabolite('Cpd3716', formula='', name='3-hydroxy-tetradecanoyl carnitine [c]', compartment='')
Cpd3717 = Metabolite('Cpd3717', formula='', name='3-hydroxyhexadecenoylcoa [c]', compartment='')
Cpd3718 = Metabolite('Cpd3718', formula='', name='3-hydroxyhexadecenoylcarnitine [c]', compartment='')
Cpd3719 = Metabolite('Cpd3719', formula='', name='3-hydroxy trans7,10-hexadecadienoylcoa [m]', compartment='')
Cpd3720 = Metabolite('Cpd3720', formula='', name='3-hydroxy trans7,10-hexadecadienoylcoa [c]', compartment='')
Cpd3721 = Metabolite('Cpd3721', formula='', name='3-hydroxy trans7,10-hexadecadienoyl carnitine [c]', compartment='')
Cpd3722 = Metabolite('Cpd3722', formula='', name='hexadecanedioylcoa [c]', compartment='')
Cpd3723 = Metabolite('Cpd3723', formula='', name='Hexadecanedioic acid mono-L-carnitine ester [c]', compartment='')
Cpd3724 = Metabolite('Cpd3724', formula='', name='3-hydroxyhexadecanoylcoa [c]', compartment='')
Cpd3725 = Metabolite('Cpd3725', formula='', name='3-hydroxyhexadecanoylcarnitine [c]', compartment='')
Cpd3726 = Metabolite('Cpd3726', formula='', name='3-hydroxyoctadecenoylcoa [c]', compartment='')
Cpd3727 = Metabolite('Cpd3727', formula='', name='3-hydroxy-octadecenoyl carnitine [c]', compartment='')
Cpd3728 = Metabolite('Cpd3728', formula='', name='3-hydroxyoctadecadienoylcoa [c]', compartment='')
Cpd3729 = Metabolite('Cpd3729', formula='', name='3-hydroxyoctadecadienoylcarnitine [c]', compartment='')
Cpd3730 = Metabolite('Cpd3730', formula='', name='(S)-3-Hydroxyoctadecanoyl-CoA [c]', compartment='')
Cpd3731 = Metabolite('Cpd3731', formula='', name='3-hydroxyoctadecanoyl carnitine [c]', compartment='')
Cpd3732 = Metabolite('Cpd3732', formula='', name='Butanoyl-CoA [c]', compartment='')
Cpd3733 = Metabolite('Cpd3733', formula='', name='butyryl carnitine [c]', compartment='')
Cpd3734 = Metabolite('Cpd3734', formula='', name='butyryl carnitine [m]', compartment='')
Cpd3735 = Metabolite('Cpd3735', formula='', name='succinyl carnitine [x]', compartment='')
Cpd3736 = Metabolite('Cpd3736', formula='', name='succinyl carnitine [c]', compartment='')
Cpd3737 = Metabolite('Cpd3737', formula='', name='3-hydroxy butyryl carnitine [c]', compartment='')
Cpd3738 = Metabolite('Cpd3738', formula='', name='butyryl carnitine [x]', compartment='')
Cpd3739 = Metabolite('Cpd3739', formula='', name='Butanoyl-CoA [x]', compartment='')
Cpd3740 = Metabolite('Cpd3740', formula='', name='Isovaleryl-CoA [c]', compartment='')
Cpd3741 = Metabolite('Cpd3741', formula='', name='isovaleryl carnitine [c]', compartment='')
Cpd3742 = Metabolite('Cpd3742', formula='', name='tiglyl carnitine [c]', compartment='')
Cpd3743 = Metabolite('Cpd3743', formula='', name='Hexanoyl-CoA (n-C6:0CoA) [c]', compartment='')
Cpd3744 = Metabolite('Cpd3744', formula='', name='hexanoyl carnitine [c]', compartment='')
Cpd3745 = Metabolite('Cpd3745', formula='', name='Hexanoyl-CoA (n-C6:0CoA) [x]', compartment='')
Cpd3746 = Metabolite('Cpd3746', formula='', name='hexanoyl carnitine [x]', compartment='')
Cpd3747 = Metabolite('Cpd3747', formula='', name='adipoyl carnitine [c]', compartment='')
Cpd3748 = Metabolite('Cpd3748', formula='', name='adipoylcoa [c]', compartment='')
Cpd3749 = Metabolite('Cpd3749', formula='', name='octanoyl carnitine [c]', compartment='')
Cpd3750 = Metabolite('Cpd3750', formula='', name='octenoylcoa [c]', compartment='')
Cpd3751 = Metabolite('Cpd3751', formula='', name='octenoyl carnitine [c]', compartment='')
Cpd3752 = Metabolite('Cpd3752', formula='', name='suberylcoa [c]', compartment='')
Cpd3753 = Metabolite('Cpd3753', formula='', name='suberyl carnitine [c]', compartment='')
Cpd3754 = Metabolite('Cpd3754', formula='', name='4,8-dimethylnonanoyl carnitine [c]', compartment='')
Cpd3755 = Metabolite('Cpd3755', formula='', name='Decanoate (n-C10:0) [e]', compartment='')
Cpd3756 = Metabolite('Cpd3756', formula='', name='Decanoate (n-C10:0) [c]', compartment='')
Cpd3757 = Metabolite('Cpd3757', formula='', name='13-docosenoylcoa [c]', compartment='')
Cpd3758 = Metabolite('Cpd3758', formula='', name='13-docosenoylcoa [x]', compartment='')
Cpd3759 = Metabolite('Cpd3759', formula='', name='docosenoic acid (C22:1) [e]', compartment='')
Cpd3760 = Metabolite('Cpd3760', formula='', name='Behenic acid [c]', compartment='')
Cpd3761 = Metabolite('Cpd3761', formula='', name='docosanoylcoa [c]', compartment='')
Cpd3762 = Metabolite('Cpd3762', formula='', name='docosanoylcoa [x]', compartment='')
Cpd3763 = Metabolite('Cpd3763', formula='', name='4-decenoylcoa [m]', compartment='')
Cpd3764 = Metabolite('Cpd3764', formula='', name='2,4-decadienoylcoa [m]', compartment='')
Cpd3765 = Metabolite('Cpd3765', formula='', name='4-decenoylcoa [x]', compartment='')
Cpd3766 = Metabolite('Cpd3766', formula='', name='2,4-decadienoylcoa [x]', compartment='')
Cpd3767 = Metabolite('Cpd3767', formula='', name='trans-Dec-2-enoyl-CoA [x]', compartment='')
Cpd3768 = Metabolite('Cpd3768', formula='', name='3-decenoylcoa [m]', compartment='')
Cpd3769 = Metabolite('Cpd3769', formula='', name='3-decenoylcoa [x]', compartment='')
Cpd3770 = Metabolite('Cpd3770', formula='', name='4,7-decadienoylcoa [m]', compartment='')
Cpd3771 = Metabolite('Cpd3771', formula='', name='2,4,7-decatrienoylcoa [m]', compartment='')
Cpd3772 = Metabolite('Cpd3772', formula='', name='4,7-decadienoylcoa [x]', compartment='')
Cpd3773 = Metabolite('Cpd3773', formula='', name='2,4,7-decatrienoylcoa [x]', compartment='')
Cpd3774 = Metabolite('Cpd3774', formula='', name='2,7-decadienoylcoa [m]', compartment='')
Cpd3775 = Metabolite('Cpd3775', formula='', name='5-octenoylcoa [m]', compartment='')
Cpd3776 = Metabolite('Cpd3776', formula='', name='2,7-decadienoylcoa [x]', compartment='')
Cpd3777 = Metabolite('Cpd3777', formula='', name='5-octenoylcoa [x]', compartment='')
Cpd3778 = Metabolite('Cpd3778', formula='', name='3,7-decadienoylcoa [m]', compartment='')
Cpd3779 = Metabolite('Cpd3779', formula='', name='3,7-decadienoylcoa [x]', compartment='')
Cpd3780 = Metabolite('Cpd3780', formula='', name='(S)-3-Hydroxydecanoyl-CoA [m]', compartment='')
Cpd3781 = Metabolite('Cpd3781', formula='', name='sebacoylcoa [x]', compartment='')
Cpd3782 = Metabolite('Cpd3782', formula='', name='suberylcoa [x]', compartment='')
Cpd3783 = Metabolite('Cpd3783', formula='', name='2,6,10-trimethyl undecanoylcoa [x]', compartment='')
Cpd3784 = Metabolite('Cpd3784', formula='', name='undecanoylcoa [m]', compartment='')
Cpd3785 = Metabolite('Cpd3785', formula='', name='nonanoylcoa [m]', compartment='')
Cpd3786 = Metabolite('Cpd3786', formula='', name='5-dodecenoylcoa [m]', compartment='')
Cpd3787 = Metabolite('Cpd3787', formula='', name='cis-dodec-3-enoyl-CoA(4-) [x]', compartment='')
Cpd3788 = Metabolite('Cpd3788', formula='', name='2,6-dodecadienoylcoa [m]', compartment='')
Cpd3789 = Metabolite('Cpd3789', formula='', name='3,6-dodecadienoylcoa [m]', compartment='')
Cpd3790 = Metabolite('Cpd3790', formula='', name='3,6-dodecadienoylcoa [x]', compartment='')
Cpd3791 = Metabolite('Cpd3791', formula='', name='2,6-dodecadienoylcoa [x]', compartment='')
Cpd3792 = Metabolite('Cpd3792', formula='', name='2,6,9-dodecatrienoylcoa [m]', compartment='')
Cpd3793 = Metabolite('Cpd3793', formula='', name='2,6,9-dodecatrienoylcoa [x]', compartment='')
Cpd3794 = Metabolite('Cpd3794', formula='', name='3,6,9-dodecatrienoylcoa [m]', compartment='')
Cpd3795 = Metabolite('Cpd3795', formula='', name='3,6,9-dodecatrienoylcoa [x]', compartment='')
Cpd3796 = Metabolite('Cpd3796', formula='', name='dodecanedioyl carnitine [x]', compartment='')
Cpd3797 = Metabolite('Cpd3797', formula='', name='dodecanedioyl carnitine [c]', compartment='')
Cpd3798 = Metabolite('Cpd3798', formula='', name='dodecanedioylcoa [c]', compartment='')
Cpd3799 = Metabolite('Cpd3799', formula='', name='4,8,12-trimethyl tridecanoylcoa [x]', compartment='')
Cpd3800 = Metabolite('Cpd3800', formula='', name='tridecanoylcoa [m]', compartment='')
Cpd3801 = Metabolite('Cpd3801', formula='', name='7-tetradecenoylcoa [m]', compartment='')
Cpd3802 = Metabolite('Cpd3802', formula='', name='5-tetradecenoylcoa [x]', compartment='')
Cpd3803 = Metabolite('Cpd3803', formula='', name='5,8-tetradecadienoylcoa [m]', compartment='')
Cpd3804 = Metabolite('Cpd3804', formula='', name='5,8-tetradecadienoylcoa [x]', compartment='')
Cpd3805 = Metabolite('Cpd3805', formula='', name='trans5,8tetradecadienoylcoa [m]', compartment='')
Cpd3806 = Metabolite('Cpd3806', formula='', name='5,8,11-tetradecatrienoylcoa [m]', compartment='')
Cpd3807 = Metabolite('Cpd3807', formula='', name='5,8,11-tetradecatrienoylcoa [x]', compartment='')
Cpd3808 = Metabolite('Cpd3808', formula='', name='tetradecanedioylcoa [x]', compartment='')
Cpd3809 = Metabolite('Cpd3809', formula='', name='7-hexadecenoylcoa [x]', compartment='')
Cpd3810 = Metabolite('Cpd3810', formula='', name='3-hydroxyhexadecenoylcoa [m]', compartment='')
Cpd3811 = Metabolite('Cpd3811', formula='', name='7,10-hexadecadienoylcoa [m]', compartment='')
Cpd3812 = Metabolite('Cpd3812', formula='', name='trans7,10-hexadecadienoylcoa [m]', compartment='')
Cpd3813 = Metabolite('Cpd3813', formula='', name='2,7,10-hexadecatrienoylcoa [x]', compartment='')
Cpd3814 = Metabolite('Cpd3814', formula='', name='7,10,13-hexadecatrienoylcoa [m]', compartment='')
Cpd3815 = Metabolite('Cpd3815', formula='', name='4,7,10-hexadecatrienoylcoa [m]', compartment='')
Cpd3816 = Metabolite('Cpd3816', formula='', name='2,4,7,10-hexadecatetraenoylcoa [m]', compartment='')
Cpd3817 = Metabolite('Cpd3817', formula='', name='4,7,10-hexadecatrienoylcoa [x]', compartment='')
Cpd3818 = Metabolite('Cpd3818', formula='', name='2,4,7,10-hexadecatetraenoylcoa [x]', compartment='')
Cpd3819 = Metabolite('Cpd3819', formula='', name='2,7,10-hexadecatrienoylcoa [m]', compartment='')
Cpd3820 = Metabolite('Cpd3820', formula='', name='3,7,10-hexadecatrienoylcoa [m]', compartment='')
Cpd3821 = Metabolite('Cpd3821', formula='', name='3,7,10-hexadecatrienoylcoa [x]', compartment='')
Cpd3822 = Metabolite('Cpd3822', formula='', name='2,7,10,13-hexadecatetraenoylcoa [m]', compartment='')
Cpd3823 = Metabolite('Cpd3823', formula='', name='2,7,10,13-hexadecatetraenoylcoa [x]', compartment='')
Cpd3824 = Metabolite('Cpd3824', formula='', name='4,7,10,13-hexadecatetraenoylcoa [m]', compartment='')
Cpd3825 = Metabolite('Cpd3825', formula='', name='2,4,7,10,13-hexadecapentenoylcoa [m]', compartment='')
Cpd3826 = Metabolite('Cpd3826', formula='', name='4,7,10,13-hexadecatetraenoylcoa [x]', compartment='')
Cpd3827 = Metabolite('Cpd3827', formula='', name='2,4,7,10,13-hexadecapentaenoylcoa [x]', compartment='')
Cpd3828 = Metabolite('Cpd3828', formula='', name='3,7,10,13-hexadecatetraenoylcoa [m]', compartment='')
Cpd3829 = Metabolite('Cpd3829', formula='', name='3,7,10,13-hexadecatetraenoylcoa [x]', compartment='')
Cpd3830 = Metabolite('Cpd3830', formula='', name='3-hydroxyhexadecanoylcoa [m]', compartment='')
Cpd3831 = Metabolite('Cpd3831', formula='', name='hexadecanedioylcoa [x]', compartment='')
Cpd3832 = Metabolite('Cpd3832', formula='', name='hexadecanediocacid [r]', compartment='')
Cpd3833 = Metabolite('Cpd3833', formula='', name='hexadecanedioylcoa [r]', compartment='')
Cpd3834 = Metabolite('Cpd3834', formula='', name='w-hydroxypalmiticacid [r]', compartment='')
Cpd3835 = Metabolite('Cpd3835', formula='', name='11-octadecenoylcoa [m]', compartment='')
Cpd3836 = Metabolite('Cpd3836', formula='', name='9-octadecenoylcoa [x]', compartment='')
Cpd3837 = Metabolite('Cpd3837', formula='', name='octadecenoylcoa [m]', compartment='')
Cpd3838 = Metabolite('Cpd3838', formula='', name='3-hydroxyoctadecenoylcoa [m]', compartment='')
Cpd3839 = Metabolite('Cpd3839', formula='', name='3-hydroxyoctadecadienoylcoa [m]', compartment='')
Cpd3840 = Metabolite('Cpd3840', formula='', name='2,6,9,12-octadecatetraenoylcoa [m]', compartment='')
Cpd3841 = Metabolite('Cpd3841', formula='', name='2,6,9,12-octadecatetraenoylcoa [x]', compartment='')
Cpd3842 = Metabolite('Cpd3842', formula='', name='3,6,9,12-octadecatetraenoylcoa [m]', compartment='')
Cpd3843 = Metabolite('Cpd3843', formula='', name='3,6,9,12-octadecatetraenoylcoa [x]', compartment='')
Cpd3844 = Metabolite('Cpd3844', formula='', name='2,6,9,12,15-octadecapentenoylcoa [m]', compartment='')
Cpd3845 = Metabolite('Cpd3845', formula='', name='3,6,9,12,15-octadecapentenoylcoa [m]', compartment='')
Cpd3846 = Metabolite('Cpd3846', formula='', name='(S)-3-Hydroxyoctadecanoyl-CoA [m]', compartment='')
Cpd3847 = Metabolite('Cpd3847', formula='', name='11-eicosenoylcoa [x]', compartment='')
Cpd3848 = Metabolite('Cpd3848', formula='', name='5,8,11,14-eicosatetraenoylcoa [m]', compartment='')
Cpd3849 = Metabolite('Cpd3849', formula='', name='5,8,11,14-eicosatetraenoylcoa [x]', compartment='')
Cpd3850 = Metabolite('Cpd3850', formula='', name='2,5,8,11,14-eicosapentaenoylcoa [x]', compartment='')
Cpd3851 = Metabolite('Cpd3851', formula='', name='5,8,11,14,17-eicosapentenoylcoa [m]', compartment='')
Cpd3852 = Metabolite('Cpd3852', formula='', name='2,7,10,13,16-docosapentenoylcoa [m]', compartment='')
Cpd3853 = Metabolite('Cpd3853', formula='', name='2,7,10,13,16-docosapentenoylcoa [x]', compartment='')
Cpd3854 = Metabolite('Cpd3854', formula='', name='2,4,7,10,13,16-docosahexenoylcoa [m]', compartment='')
Cpd3855 = Metabolite('Cpd3855', formula='', name='2,4,7,10,13,16-docosahexenoylcoa [x]', compartment='')
Cpd3856 = Metabolite('Cpd3856', formula='', name='3,7,10,13,16-docosapentenoylcoa [m]', compartment='')
Cpd3857 = Metabolite('Cpd3857', formula='', name='3,7,10,13,16-docosapentenoylcoa [x]', compartment='')
Cpd3858 = Metabolite('Cpd3858', formula='', name='2,7,10,13,16,19-docosahexenoylcoa [m]', compartment='')
Cpd3859 = Metabolite('Cpd3859', formula='', name='2,4,7,10,13,16,19-docosaheptenoylcoa [m]', compartment='')
Cpd3860 = Metabolite('Cpd3860', formula='', name='3,7,10,13,16,19-docosahexenoylcoa [m]', compartment='')
Cpd3861 = Metabolite('Cpd3861', formula='', name='Behenic acid [r]', compartment='')
Cpd3862 = Metabolite('Cpd3862', formula='', name='docosanedioicacid [r]', compartment='')
Cpd3863 = Metabolite('Cpd3863', formula='', name='w-hydroxydocosanoicacid [r]', compartment='')
Cpd3864 = Metabolite('Cpd3864', formula='', name='malonyl carnitine [c]', compartment='')
Cpd3865 = Metabolite('Cpd3865', formula='', name='Succinyl-CoA [c]', compartment='')
Cpd3866 = Metabolite('Cpd3866', formula='', name='pentanoylcoa [m]', compartment='')
Cpd3867 = Metabolite('Cpd3867', formula='', name='glutaryl-CoA(5-) [c]', compartment='')
Cpd3868 = Metabolite('Cpd3868', formula='', name='glutaryl carnitine [c]', compartment='')
Cpd3869 = Metabolite('Cpd3869', formula='', name='3-hydroxyisovalerylcoa [c]', compartment='')
Cpd3870 = Metabolite('Cpd3870', formula='', name='3-hydroxy-isovaleryl carnitine [c]', compartment='')
Cpd3871 = Metabolite('Cpd3871', formula='', name='trans-Hex-2-enoyl-CoA [m]', compartment='')
Cpd3872 = Metabolite('Cpd3872', formula='', name='trans-Hex-2-enoyl-CoA [x]', compartment='')
Cpd3873 = Metabolite('Cpd3873', formula='', name='3-hexenoylcoa [m]', compartment='')
Cpd3874 = Metabolite('Cpd3874', formula='', name='3-hexenoylcoa [x]', compartment='')
Cpd3875 = Metabolite('Cpd3875', formula='', name='Hexanoyl-CoA (n-C6:0CoA) [m]', compartment='')
Cpd3876 = Metabolite('Cpd3876', formula='', name='Succinyl-CoA [x]', compartment='')
Cpd3877 = Metabolite('Cpd3877', formula='', name='heptanoylcoa [m]', compartment='')
Cpd3878 = Metabolite('Cpd3878', formula='', name='Decanoate (n-C10:0) [r]', compartment='')
Cpd3879 = Metabolite('Cpd3879', formula='', name='w-hydroxydecanoicacid [r]', compartment='')
Cpd3880 = Metabolite('Cpd3880', formula='', name='w-hydroxypalmiticacid [c]', compartment='')
Cpd3881 = Metabolite('Cpd3881', formula='', name='hexadecanediocacid [c]', compartment='')
Cpd3882 = Metabolite('Cpd3882', formula='', name='w-hydroxydocosanoicacid [c]', compartment='')
Cpd3883 = Metabolite('Cpd3883', formula='', name='docosanedioicacid [c]', compartment='')
Cpd3884 = Metabolite('Cpd3884', formula='', name='w-hydroxydecanoicacid [c]', compartment='')
Cpd3885 = Metabolite('Cpd3885', formula='', name='sebacicacid [c]', compartment='')
Cpd3886 = Metabolite('Cpd3886', formula='', name='trans4decenoylcoa [m]', compartment='')
Cpd3887 = Metabolite('Cpd3887', formula='', name='cis2trans4decadienoylcoa [m]', compartment='')
Cpd3888 = Metabolite('Cpd3888', formula='', name='trans2,6dodecadienoylcoa [m]', compartment='')
Cpd3889 = Metabolite('Cpd3889', formula='', name='trans3,6dodecadienoylcoa [m]', compartment='')
Cpd3890 = Metabolite('Cpd3890', formula='', name='3-hydroxyisovalerylcoa [m]', compartment='')
Cpd3891 = Metabolite('Cpd3891', formula='', name='11-octadecenoylcoa [c]', compartment='')
Cpd3892 = Metabolite('Cpd3892', formula='', name='11-octadecenoyl carnitine [c]', compartment='')
Cpd3893 = Metabolite('Cpd3893', formula='', name='11-octadecenoyl carnitine [m]', compartment='')
Cpd3894 = Metabolite('Cpd3894', formula='', name='trans-vaccenate [e]', compartment='')
Cpd3895 = Metabolite('Cpd3895', formula='', name='octadecenoyl carnitine (hypothetical, position of double bond unknown) [c]', compartment='')
Cpd3896 = Metabolite('Cpd3896', formula='', name='octadecenoyl carnitine (hypothetical, position of double bond unknown) [m]', compartment='')
Cpd3897 = Metabolite('Cpd3897', formula='', name='octadecenoylcoa [c]', compartment='')
Cpd3898 = Metabolite('Cpd3898', formula='', name='suberic acid [x]', compartment='')
Cpd3899 = Metabolite('Cpd3899', formula='', name='sebacicacid [x]', compartment='')
Cpd3900 = Metabolite('Cpd3900', formula='', name='suberyl carnitine [x]', compartment='')
Cpd3901 = Metabolite('Cpd3901', formula='', name='trans-4-hydroxy-L-proline [e]', compartment='')
Cpd3902 = Metabolite('Cpd3902', formula='', name='D-Alanyl-D-alanine [c]', compartment='')
Cpd3903 = Metabolite('Cpd3903', formula='', name='D-Alanyl-D-alanine [e]', compartment='')
Cpd3904 = Metabolite('Cpd3904', formula='', name='carnosine [e]', compartment='')
Cpd3905 = Metabolite('Cpd3905', formula='', name='carnosine [l]', compartment='')
Cpd3906 = Metabolite('Cpd3906', formula='', name='Adenosylcobalamin [e]', compartment='')
Cpd3907 = Metabolite('Cpd3907', formula='', name='Adenosylcobalamin [c]', compartment='')
Cpd3908 = Metabolite('Cpd3908', formula='', name='Glycyl-glycine [c]', compartment='')
Cpd3909 = Metabolite('Cpd3909', formula='', name='Glycyl-glycine [e]', compartment='')
Cpd3910 = Metabolite('Cpd3910', formula='', name='Glycylleucine [c]', compartment='')
Cpd3911 = Metabolite('Cpd3911', formula='', name='Glycylleucine [e]', compartment='')
Cpd3912 = Metabolite('Cpd3912', formula='', name='Glycylphenylalanine [e]', compartment='')
Cpd3913 = Metabolite('Cpd3913', formula='', name='Glycylphenylalanine [c]', compartment='')
Cpd3914 = Metabolite('Cpd3914', formula='', name='Glycylproline [e]', compartment='')
Cpd3915 = Metabolite('Cpd3915', formula='', name='Glycylproline [c]', compartment='')
Cpd3916 = Metabolite('Cpd3916', formula='', name='Glycylsarcosine [c]', compartment='')
Cpd3917 = Metabolite('Cpd3917', formula='', name='Glycylsarcosine [e]', compartment='')
Cpd3918 = Metabolite('Cpd3918', formula='', name='L-histidine [l]', compartment='')
Cpd3919 = Metabolite('Cpd3919', formula='', name='5(S)-HPETE(1-) [r]', compartment='')
Cpd3920 = Metabolite('Cpd3920', formula='', name='Leucylglycine [e]', compartment='')
Cpd3921 = Metabolite('Cpd3921', formula='', name='Leucylglycine [c]', compartment='')
Cpd3922 = Metabolite('Cpd3922', formula='', name='Leucylleucine [c]', compartment='')
Cpd3923 = Metabolite('Cpd3923', formula='', name='Leucylleucine [e]', compartment='')
Cpd3924 = Metabolite('Cpd3924', formula='', name='Nicotinate [e]', compartment='')
Cpd3925 = Metabolite('Cpd3925', formula='', name='L-Prolinylglycine [e]', compartment='')
Cpd3926 = Metabolite('Cpd3926', formula='', name='L-Prolinylglycine [c]', compartment='')
Cpd3927 = Metabolite('Cpd3927', formula='', name='calcitriol [c]', compartment='')
Cpd3928 = Metabolite('Cpd3928', formula='', name='(R)-5-Diphosphomevalonate [c]', compartment='')
Cpd3929 = Metabolite('Cpd3929', formula='', name='Flavin adenine dinucleotide oxidized [e]', compartment='')
Cpd3930 = Metabolite('Cpd3930', formula='', name='FMN [e]', compartment='')
Cpd3931 = Metabolite('Cpd3931', formula='', name='Fe3+ [c]', compartment='')
Cpd3932 = Metabolite('Cpd3932', formula='', name='(R)-5-phosphonatomevalonate(3-) [c]', compartment='')
Cpd3933 = Metabolite('Cpd3933', formula='', name='Orotate [e]', compartment='')


model = Model('Example')

reaction = Reaction('13DAMPPOX')
reaction.name = '1,3-Diaminopropane:oxygen oxidoreductase (deaminating)'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1: -1.0, Cpd2: -1.0, Cpd3: -1.0, Cpd4: 1.0, Cpd5: 1.0, Cpd6: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('24_25VITD2Hm')
reaction.name = '24R-Vitamin D-25-hydroxylase (D2)'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd10: -1.0, Cpd11: 1.0, Cpd12: 1.0, Cpd13: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('24_25VITD3Hm')
reaction.name = '24R-Vitamin D-25-hydroxylase (D3)'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd14: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd15: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('25HVITD3tin')
reaction.name = '25-hydroxyvitamin D3 transport in cytoplasm'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd16: -1.0, Cpd17: 1.0})
reaction.gene_reaction_rule = '( CUBN or GC or LRP2 )'
model.add_reactions([reaction])
reaction = Reaction('25VITD2Hm')
reaction.name = '1-alpha-Vitamin D-25-hydroxylase (D2)'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd10: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd18: 1.0})
reaction.gene_reaction_rule = '( CYP27B1 )'
model.add_reactions([reaction])
reaction = Reaction('25VITD3Hm')
reaction.name = '1-alpha-Vitamin D-25-hydroxylase (D3)'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd14: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd19: 1.0})
reaction.gene_reaction_rule = '( CYP27B1 )'
model.add_reactions([reaction])
reaction = Reaction('2AMACHYD')
reaction.name = '2-Aminoacrylate hydrolysis'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd20: -1.0, Cpd6: 1.0, Cpd21: 1.0})
reaction.gene_reaction_rule = '( SDS )'
model.add_reactions([reaction])
reaction = Reaction('2AMACSULT')
reaction.name = '2-Aminoacrylate sulfotransferase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd20: -1.0, Cpd22: -1.0, Cpd23: -1.0, Cpd24: 1.0, Cpd25: 1.0, Cpd26: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('2AMADPTm')
reaction.name = 'L-2-aminoadipate shuttle (cytosol/mitochondria)'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd27: -1.0, Cpd28: -1.0, Cpd29: 1.0, Cpd30: 1.0})
reaction.gene_reaction_rule = '( SLC25A21 )'
model.add_reactions([reaction])
reaction = Reaction('2HBO')
reaction.name = '2-Hydroxybutyrate:NAD+ oxidoreductase'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd32: -1.0, Cpd33: 1.0, Cpd34: 1.0, Cpd35: 1.0})
reaction.gene_reaction_rule = '( LDHA or LDHAL6B or LDHB or LDHC or UEVLD )'
model.add_reactions([reaction])
reaction = Reaction('2HBt2')
reaction.name = '2-hydroxybutyrate cotransport with proton'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd36: -1.0, Cpd37: -1.0, Cpd33: 1.0, Cpd31: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A3 or SLC16A7 )'
model.add_reactions([reaction])
reaction = Reaction('2HCO3_NAt')
reaction.name = 'bicarbonate transport (Na/HCO3 1:2 cotransport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -2.0, Cpd39: -1.0, Cpd40: 2.0, Cpd41: 1.0})
reaction.gene_reaction_rule = '( SLC4A4 or SLC4A5 )'
model.add_reactions([reaction])
reaction = Reaction('2OXOADOXm')
reaction.name = '2-Oxoadipate:lipoamde 2-oxidoreductase(decarboxylating and acceptor-succinylating) (mitochondria)'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd42: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd45: 1.0, Cpd46: 1.0, Cpd47: 1.0})
reaction.gene_reaction_rule = '( DLD or DLST or OGDH or PDHX )'
model.add_reactions([reaction])
reaction = Reaction('2OXOADPTm')
reaction.name = '2-oxoadipate shuttle (cytosol/mitochondria)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd48: -1.0, Cpd30: 1.0, Cpd42: 1.0})
reaction.gene_reaction_rule = '( SLC25A21 )'
model.add_reactions([reaction])
reaction = Reaction('34DHOXPEGOX')
reaction.name = '3,4-Dihydroxyphenylethyleneglycol:NAD+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd49: -1.0, Cpd32: 1.0, Cpd50: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 or ADHFE1 )'
model.add_reactions([reaction])
reaction = Reaction('34DHPHAMT')
reaction.name = '3,4-Dihydroxyphenylacetate:amet O-methyltransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd51: -1.0, Cpd52: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd54: 1.0})
reaction.gene_reaction_rule = '( COMT )'
model.add_reactions([reaction])
reaction = Reaction('34DHPHEt')
reaction.name = '3,4-Dihydroxy-L-phenylalanine transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd55: -1.0, Cpd56: 1.0})
reaction.gene_reaction_rule = '( SLC16A10 )'
model.add_reactions([reaction])
reaction = Reaction('34DHPLACOX')
reaction.name = '3,4-Dihydroxyphenylacetaldehyde:NAD+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd57: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd51: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('34DHPLACOX_NADP_')
reaction.name = '3,4-Dihydroxyphenylacetaldehyde:NADP+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd25: -1.0, Cpd57: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd51: 1.0})
reaction.gene_reaction_rule = '( ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('34DHXMANDACOX')
reaction.name = '3,4-Dihydroxymandelaldehyde:NAD+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd49: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd58: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 )'
model.add_reactions([reaction])
reaction = Reaction('34DHXMANDACOX_NADP_')
reaction.name = '3,4-Dihydroxymandelaldehyde:NADP+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd25: -1.0, Cpd49: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd58: 1.0})
reaction.gene_reaction_rule = '( ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('34HPPOR')
reaction.name = '4-Hydroxyphenylpyruvate:oxygen oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd59: -1.0, Cpd60: 1.0, Cpd61: 1.0})
reaction.gene_reaction_rule = '( HPD )'
model.add_reactions([reaction])
reaction = Reaction('3AIBTm')
reaction.name = 'L-3-aminoisobutyrate transaminase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd62: -1.0, Cpd63: -1.0, Cpd28: 1.0, Cpd64: 1.0})
reaction.gene_reaction_rule = '( ABAT )'
model.add_reactions([reaction])
reaction = Reaction('3AIBtm')
reaction.name = '3-amino-isobutyrate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd64: -1.0, Cpd65: 1.0})
reaction.gene_reaction_rule = '( ABAT )'
model.add_reactions([reaction])
reaction = Reaction('3DSPHR')
reaction.name = '3-Dehydrosphinganine reductase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd66: -1.0, Cpd25: 1.0, Cpd67: 1.0})
reaction.gene_reaction_rule = '( KDSR )'
model.add_reactions([reaction])
reaction = Reaction('3HAO')
reaction.name = '3-hydroxyanthranilate 3,4-dioxygenase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd68: -1.0, Cpd33: 1.0, Cpd69: 1.0})
reaction.gene_reaction_rule = '( HAAO )'
model.add_reactions([reaction])
reaction = Reaction('3HBCDm')
reaction.name = '3-hydroxybutyryl-CoA dehydratase, mitochondrial'
reaction.subsystem = 'Butanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd70: -1.0, Cpd71: 1.0})
reaction.gene_reaction_rule = '( ECHDC2 )'
model.add_reactions([reaction])
reaction = Reaction('3HKYNAKGAT')
reaction.name = '3-Hydroxy-L-kynurenine:2-oxoglutarate aminotransferase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd72: -1.0, Cpd73: 1.0, Cpd74: 1.0})
reaction.gene_reaction_rule = '( AADAT or KYAT1 )'
model.add_reactions([reaction])
reaction = Reaction('3HLYTCL')
reaction.name = '3-Hydroxy-L-tyrosine carboxy-lyase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd56: -1.0, Cpd60: 1.0, Cpd75: 1.0})
reaction.gene_reaction_rule = '( DDC )'
model.add_reactions([reaction])
reaction = Reaction('3HPCOAHYD')
reaction.name = '3-hydroxyisobutyryl-Coenzyme A hydrolase'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd76: -1.0, Cpd33: 1.0, Cpd77: 1.0, Cpd78: 1.0})
reaction.gene_reaction_rule = '( HIBCH )'
model.add_reactions([reaction])
reaction = Reaction('3HXKYNDCL')
reaction.name = '3-Hydroxykynurenamine decarboxy-lyase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd72: -1.0, Cpd60: 1.0, Cpd79: 1.0})
reaction.gene_reaction_rule = '( DDC )'
model.add_reactions([reaction])
reaction = Reaction('3HXKYNOXDA')
reaction.name = '3-Hydroxykynurenamine:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd79: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd80: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('3M4HDXPAC')
reaction.name = '3-Methoxy-4-hydroxyphenylacetaldehyde:NAD+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd81: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd54: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('3MOX4HOXPGALDOX')
reaction.name = '3-Methoxy-4-hydroxyphenylglycolaldehyde:NAD+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd82: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd83: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('3MOX4HOXPGALDOX_NADP_')
reaction.name = '3-Methoxy-4-hydroxyphenylglycolaldehyde:NADP+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd25: -1.0, Cpd82: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd83: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('3MOXTYROX')
reaction.name = '3-Methoxytyramine:oxygen oxidoreductase (deaminating)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd84: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd81: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('3NTD7l')
reaction.name = '3-nucleotidase (AMP), lysosome'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd85: -1.0, Cpd86: -1.0, Cpd87: 1.0, Cpd88: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('3SALAASPm')
reaction.name = 'cysteinesulfinate-aspartate mitochondrial shuttle'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd89: -1.0, Cpd90: -1.0, Cpd91: 1.0, Cpd92: 1.0})
reaction.gene_reaction_rule = '( SLC25A12 or SLC25A13 )'
model.add_reactions([reaction])
reaction = Reaction('3SALACBOXL')
reaction.name = '3-Sulfino-L-alanine carboxy-lyase'
reaction.subsystem = 'Taurine and hypotaurine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd91: -1.0, Cpd60: 1.0, Cpd93: 1.0})
reaction.gene_reaction_rule = '( CSAD or GAD1 or GAD2 )'
model.add_reactions([reaction])
reaction = Reaction('3SALATAi')
reaction.name = '3-sulfino-alanine transaminase (irreversible)'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd30: -1.0, Cpd91: -1.0, Cpd74: 1.0, Cpd94: 1.0})
reaction.gene_reaction_rule = '( GOT1 )'
model.add_reactions([reaction])
reaction = Reaction('3SALATAim')
reaction.name = '3-sulfino-alanine transaminase (irreversible), mitochondrial'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd28: -1.0, Cpd89: -1.0, Cpd63: 1.0, Cpd95: 1.0})
reaction.gene_reaction_rule = '( GOT2 )'
model.add_reactions([reaction])
reaction = Reaction('41R1H2MAE12BOOX')
reaction.name = '4-[(1R)-1-Hydroxy-2-(methylamino)ethyl]-1,2-benzenediol:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd96: -1.0, Cpd5: 1.0, Cpd49: 1.0, Cpd97: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('41R2A1H12BOOX')
reaction.name = '4-[(1R)-2-Amino-1-hydroxyethyl]-1,2-benzenediol:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd98: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd49: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('42A12BOOX')
reaction.name = '4-(2-Aminoethyl)-1,2-benzenediol:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd75: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd57: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or AOC3 or MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('4HGLSDm')
reaction.name = 'L-4-hydroxyglutamate semialdehyde dehydrogenase, mitochondrial'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd99: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd100: 1.0})
reaction.gene_reaction_rule = '( ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('4HOXPACDOX_NADP_')
reaction.name = '4-Hydroxyphenylacetaldehyde:NADP+ oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd25: -1.0, Cpd101: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd102: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('4NPHSULT')
reaction.name = '4-Nitrophenol Sulfotransferase'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd103: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd104: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('5ADTSTSTERONEGLCte')
reaction.name = 'glucuronidated compound transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd106: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd109: 1.0})
reaction.gene_reaction_rule = '( ABCC1 )'
model.add_reactions([reaction])
reaction = Reaction('5ADTSTSTERONESULT')
reaction.name = '5alpha-Dihydrotestosterone sulfotransferase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd110: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd111: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 or SULT2A1 )'
model.add_reactions([reaction])
reaction = Reaction('5FTHFt2')
reaction.name = '5-formyltetrahydrofolate transport via anion exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd112: -1.0, Cpd113: 1.0, Cpd114: 1.0})
reaction.gene_reaction_rule = '( SLC19A1 )'
model.add_reactions([reaction])
reaction = Reaction('5HLTDL')
reaction.name = '5-Hydroxy-L-tryptophan decarboxy-lyase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd115: -1.0, Cpd60: 1.0, Cpd116: 1.0})
reaction.gene_reaction_rule = '( DDC )'
model.add_reactions([reaction])
reaction = Reaction('5HOXINDACTO2OX')
reaction.name = '5-Hydroxyindoleacetaldehyde:oxygen oxidoreductase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd117: -1.0, Cpd5: 1.0, Cpd33: 1.0, Cpd118: 1.0})
reaction.gene_reaction_rule = '( AOX1 )'
model.add_reactions([reaction])
reaction = Reaction('5HOXINDACTOXm')
reaction.name = '5-Hydroxyindoleacetaldehyde:NAD+ oxidoreductase (mito)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd119: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd120: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('5HOXINOXDA')
reaction.name = '5-Hydroxytryptamine:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd116: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd117: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('5HTRPDOX')
reaction.name = '5-Hydroxy-L-tryptophan:oxygen 2,3-dioxygenase (indole-decyclizing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd115: -1.0, Cpd121: 1.0})
reaction.gene_reaction_rule = '( IDO1 )'
model.add_reactions([reaction])
reaction = Reaction('5HTRPVESSEC')
reaction.name = '5-hydroxy-L-tryptophan secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd115: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd122: 1.0})
reaction.gene_reaction_rule = '( SLC18A1 or SLC18A2 )'
model.add_reactions([reaction])
reaction = Reaction('5HXKYNDCL')
reaction.name = '5-Hydroxykynurenamine decarboxy-lyase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd123: -1.0, Cpd60: 1.0, Cpd124: 1.0})
reaction.gene_reaction_rule = '( DDC )'
model.add_reactions([reaction])
reaction = Reaction('5HXKYNOXDA')
reaction.name = '5-Hydroxykynurenamine:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd124: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd125: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('5MTHFt')
reaction.name = '5-Methyltetrahydrofolate transport via receptor binding and protolysis'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd126: -1.0, Cpd127: 1.0})
reaction.gene_reaction_rule = '( FOLR1 )'
model.add_reactions([reaction])
reaction = Reaction('5MTHFt2')
reaction.name = '5-methyltetrahydrofolate transport via anion exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd126: -1.0, Cpd114: 1.0, Cpd127: 1.0})
reaction.gene_reaction_rule = '( SLC19A1 )'
model.add_reactions([reaction])
reaction = Reaction('A_MANASE')
reaction.name = 'alpha-mannosidase'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2.0, Cpd128: -1.0, Cpd129: 2.0, Cpd130: 1.0})
reaction.gene_reaction_rule = '( MAN2C1 )'
model.add_reactions([reaction])
reaction = Reaction('A_MANASEly')
reaction.name = 'alpha-mannosidase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd131: -1.0, Cpd132: 2.0, Cpd133: 1.0})
reaction.gene_reaction_rule = '( MAN2B1 )'
model.add_reactions([reaction])
reaction = Reaction('A4GALTc')
reaction.name = 'Lactosylceramide 4-alpha-galactosyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd134: -1.0, Cpd135: -1.0, Cpd33: 1.0, Cpd136: 1.0, Cpd137: 1.0})
reaction.gene_reaction_rule = '( A4GALT )'
model.add_reactions([reaction])
reaction = Reaction('A4GALTg')
reaction.name = 'Lactosylceramide 4-alpha-galactosyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd138: -1.0, Cpd139: -1.0, Cpd140: 1.0, Cpd141: 1.0, Cpd142: 1.0})
reaction.gene_reaction_rule = '( A4GALT )'
model.add_reactions([reaction])
reaction = Reaction('A4GNT1g')
reaction.name = 'alpha 1,4-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd143: -1.0, Cpd144: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd145: 1.0})
reaction.gene_reaction_rule = '( A4GNT )'
model.add_reactions([reaction])
reaction = Reaction('A4GNT2g')
reaction.name = 'alpha 1,4-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd146: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd147: 1.0})
reaction.gene_reaction_rule = '( A4GNT )'
model.add_reactions([reaction])
reaction = Reaction('AACOAT')
reaction.name = 'Acetoacetyl-CoA:acetate CoA-transferase'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd148: -1.0, Cpd149: 1.0, Cpd150: 1.0, Cpd151: 1.0})
reaction.gene_reaction_rule = '( AACS )'
model.add_reactions([reaction])
reaction = Reaction('AACTOOR')
reaction.name = 'Aminoacetone:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd152: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd153: 1.0})
reaction.gene_reaction_rule = '( AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('AATAi')
reaction.name = '2-aminoadipate transaminase, irreversible'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd27: -1.0, Cpd30: -1.0, Cpd48: 1.0, Cpd74: 1.0})
reaction.gene_reaction_rule = '( AADAT )'
model.add_reactions([reaction])
reaction = Reaction('ABO1g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd154: -1.0, Cpd155: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd156: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO2g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd154: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd157: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO3g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd158: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd159: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO4g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd158: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd160: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO5g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd161: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd162: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO6g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd163: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd164: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO7g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd165: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd166: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO8g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd163: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd167: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABO9g')
reaction.name = 'ABO blood group (transferase A, alpha 1-3-N-acetylgalactosaminyltransferase; transferase B, alpha 1-3-galactosyltransferase)'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -2.0, Cpd168: -1.0, Cpd140: 2.0, Cpd142: 2.0, Cpd169: 1.0})
reaction.gene_reaction_rule = '( ABO )'
model.add_reactions([reaction])
reaction = Reaction('ABTArm')
reaction.name = '4-aminobutyrate transaminase, reversible (mitochondrial)'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd170: -1.0, Cpd63: 1.0, Cpd171: 1.0})
reaction.gene_reaction_rule = '( ABAT )'
model.add_reactions([reaction])
reaction = Reaction('ABUTD')
reaction.name = 'Aminobutyraldehyde dehydrogenase'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd172: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd173: 1.0})
reaction.gene_reaction_rule = '( ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('ABUTt2r')
reaction.name = '4-aminobutyrate reversible transport in via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd174: -1.0, Cpd33: 1.0, Cpd173: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('ABUTt2rL')
reaction.name = '4-aminobutyrate reversible transport in via proton symport (lysosome)'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd175: -1.0, Cpd176: -1.0, Cpd33: 1.0, Cpd173: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('ABUTt4_2_r')
reaction.name = '4-aminobutyrate reversible transport in via sodium symport (1:2)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd174: -1.0, Cpd41: 2.0, Cpd173: 1.0})
reaction.gene_reaction_rule = '( SLC6A1 or SLC6A11 or SLC6A12 or SLC6A13 )'
model.add_reactions([reaction])
reaction = Reaction('ACACT10m')
reaction.name = 'acetyl-CoA C-acetyltransferase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd177: -1.0, Cpd178: 1.0, Cpd179: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACAT1 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('ACACT1r')
reaction.name = 'acetyl-CoA C-acetyltransferase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -2.0, Cpd78: 1.0, Cpd149: 1.0})
reaction.gene_reaction_rule = '( ACAT3 )'
model.add_reactions([reaction])
reaction = Reaction('ACACT1rm')
reaction.name = 'acetyl-CoA C-acetyltransferase, mitochondrial'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -2.0, Cpd43: 1.0, Cpd181: 1.0})
reaction.gene_reaction_rule = '( ACAT1 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('ACACT1x')
reaction.name = 'acetyl-CoA C-acetyltransferase, mitochondrial'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd182: -2.0, Cpd183: 1.0, Cpd184: 1.0})
reaction.gene_reaction_rule = '( ACAT3 )'
model.add_reactions([reaction])
reaction = Reaction('ACACt2')
reaction.name = 'acetoacetate transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd185: -1.0, Cpd33: 1.0, Cpd148: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A7 )'
model.add_reactions([reaction])
reaction = Reaction('ACACt2m')
reaction.name = 'Acetoacetate mitochondrial transport via H+ symport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd148: -1.0, Cpd7: 1.0, Cpd186: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('ACACT4p')
reaction.name = 'acetyl-CoA C-acetyltransferase (octanoyl-CoA), peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd187: -1.0, Cpd182: 1.0, Cpd188: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('ACACT5p')
reaction.name = 'acetyl-CoA C-acyltransferase (decanoyl-CoA), peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd189: -1.0, Cpd182: 1.0, Cpd190: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('ACACT6p')
reaction.name = 'acetyl-CoA C-acetyltransferase (dodecanoyl), peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd191: -1.0, Cpd182: 1.0, Cpd192: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('ACACT7p')
reaction.name = 'acetyl-CoA acyltransferase (tetradecanoyl-CoA), peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd193: -1.0, Cpd182: 1.0, Cpd194: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('ACACT8p')
reaction.name = 'acetyl-CoA acyltransferase (hexadecanoyl-CoA), peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd195: -1.0, Cpd182: 1.0, Cpd196: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('ACACT9p')
reaction.name = 'acetyl-CoA acyltransferase (tetracosanoyl-CoA), peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd197: -1.0, Cpd182: 1.0, Cpd198: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('ACCOAC')
reaction.name = 'acetyl-CoA carboxylase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd40: -1.0, Cpd180: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd199: 1.0})
reaction.gene_reaction_rule = '( ACACA or ACACB )'
model.add_reactions([reaction])
reaction = Reaction('ACCOACm')
reaction.name = 'Acetyl-Coa carboxylase, beta isoform'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd200: -1.0, Cpd201: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd203: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( ACACB )'
model.add_reactions([reaction])
reaction = Reaction('ACCOAgt')
reaction.name = 'acetyl-coa transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd205: 1.0})
reaction.gene_reaction_rule = '( SLC33A1 )'
model.add_reactions([reaction])
reaction = Reaction('ACCOAL')
reaction.name = 'acetate-CoA ligase (ADP-forming)'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd206: -1.0, Cpd107: 1.0, Cpd108: 1.0, Cpd207: 1.0})
reaction.gene_reaction_rule = '( ACSS2 )'
model.add_reactions([reaction])
reaction = Reaction('ACCOALm')
reaction.name = 'acetate-CoA ligase (AMP-forming)'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd208: -1.0, Cpd179: 1.0, Cpd209: 1.0, Cpd210: 1.0})
reaction.gene_reaction_rule = '( ACSS1 or ACSS3 )'
model.add_reactions([reaction])
reaction = Reaction('ACCOAtr')
reaction.name = 'acetyl-coa transport'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd211: 1.0})
reaction.gene_reaction_rule = '( SLC33A1 )'
model.add_reactions([reaction])
reaction = Reaction('ACETONEt2')
reaction.name = 'acetone transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd212: -1.0, Cpd33: 1.0, Cpd213: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A3 or SLC16A7 )'
model.add_reactions([reaction])
reaction = Reaction('ACETONEt2m')
reaction.name = 'acetone mitochondrial transport via proton symport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd213: -1.0, Cpd7: 1.0, Cpd214: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('ACGAM2E')
reaction.name = 'N-acetylglucosamine 2-epimerase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd215: -1.0, Cpd216: 1.0})
reaction.gene_reaction_rule = '( RENBP )'
model.add_reactions([reaction])
reaction = Reaction('ACGAM6PSi')
reaction.name = 'N-acetylglucosamine-6-phosphate synthase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd217: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd218: 1.0})
reaction.gene_reaction_rule = '( GNPNAT1 )'
model.add_reactions([reaction])
reaction = Reaction('ACGAMK')
reaction.name = 'N-acetylglucosamine kinase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd215: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd218: 1.0})
reaction.gene_reaction_rule = '( NAGK )'
model.add_reactions([reaction])
reaction = Reaction('ACGAMPM')
reaction.name = 'phosphoacetylglucosamine mutase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd218: -1.0, Cpd219: 1.0})
reaction.gene_reaction_rule = '( PGM3 )'
model.add_reactions([reaction])
reaction = Reaction('ACGPID')
reaction.name = 'N-acetylglucosaminylphosphatidylinositol deacetylase'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd220: -1.0, Cpd221: 1.0, Cpd222: 1.0})
reaction.gene_reaction_rule = '( PIGL )'
model.add_reactions([reaction])
reaction = Reaction('ACGSm')
reaction.name = 'N-acteylglutamate synthase, mitochondrial'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -1.0, Cpd178: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd223: 1.0})
reaction.gene_reaction_rule = '( NAGS )'
model.add_reactions([reaction])
reaction = Reaction('ACHEe')
reaction.name = 'Acetylcholinesterase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd224: -1.0, Cpd37: 1.0, Cpd225: 1.0, Cpd226: 1.0})
reaction.gene_reaction_rule = '( ACHE )'
model.add_reactions([reaction])
reaction = Reaction('ACHVESSEC')
reaction.name = 'Acetylcholine secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd227: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd224: 1.0})
reaction.gene_reaction_rule = '( SLC18A3 )'
model.add_reactions([reaction])
reaction = Reaction('ACITL')
reaction.name = 'ATP-Citrate lyase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd228: -1.0, Cpd107: 1.0, Cpd108: 1.0, Cpd180: 1.0, Cpd229: 1.0})
reaction.gene_reaction_rule = '( ACLY )'
model.add_reactions([reaction])
reaction = Reaction('ACNAM9PL')
reaction.name = 'N-Acetylneuraminate 9-phosphate pyruvate-lyase (pyruvate-phosphorylating)'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd230: -1.0, Cpd231: -1.0, Cpd108: 1.0, Cpd232: 1.0})
reaction.gene_reaction_rule = '( NANS )'
model.add_reactions([reaction])
reaction = Reaction('ACNAM9PL2')
reaction.name = 'N-Acetylneuraminate 9-phosphate pyruvate-lyase (pyruvate-phosphorylating)'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd231: -1.0, Cpd233: -1.0, Cpd108: 1.0, Cpd234: 1.0})
reaction.gene_reaction_rule = '( NANS )'
model.add_reactions([reaction])
reaction = Reaction('ACNAMlt')
reaction.name = 'N-acetylneuraminate transport into lysososme'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd235: -1.0, Cpd176: 1.0, Cpd236: 1.0})
reaction.gene_reaction_rule = '( SLC17A5 )'
model.add_reactions([reaction])
reaction = Reaction('ACNAMPH')
reaction.name = 'N-Acetylneuraminate 9-phosphate phosphohydrolase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd232: -1.0, Cpd108: 1.0, Cpd235: 1.0})
reaction.gene_reaction_rule = '( ACP5 or ACP6 or NANP )'
model.add_reactions([reaction])
reaction = Reaction('ACOAD10m')
reaction.name = 'acyl-CoA dehydrogenase (2-methylbutanoyl-CoA), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd237: -1.0, Cpd238: -1.0, Cpd239: 1.0, Cpd240: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADSB )'
model.add_reactions([reaction])
reaction = Reaction('ACOAD1fm')
reaction.name = 'acyl-CoA dehydrogenase (butanoyl-CoA), mitochondrial'
reaction.subsystem = 'Butanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd241: -1.0, Cpd70: 1.0, Cpd240: 1.0})
reaction.gene_reaction_rule = '( ACAD11 or ACAD12 or ACAD8 or ACAD9 or ACADM or ACADS or ACADSB )'
model.add_reactions([reaction])
reaction = Reaction('ACOAD8m')
reaction.name = 'acyl-CoA dehydrogenase (isovaleryl-CoA), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd242: -1.0, Cpd240: 1.0, Cpd243: 1.0})
reaction.gene_reaction_rule = '( IVD )'
model.add_reactions([reaction])
reaction = Reaction('ACOAD9m')
reaction.name = 'acyl-CoA dehydrogenase (isobutyryl-CoA), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd244: -1.0, Cpd240: 1.0, Cpd245: 1.0})
reaction.gene_reaction_rule = '( ACAD8 or ACADM )'
model.add_reactions([reaction])
reaction = Reaction('ACOAHi')
reaction.name = 'acetyl-CoA hydrolase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd180: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd221: 1.0})
reaction.gene_reaction_rule = '( ACOT12 )'
model.add_reactions([reaction])
reaction = Reaction('ACOAO7p')
reaction.name = 'acyl-CoA oxidase (hexadecanoyl-CoA), peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd196: -1.0, Cpd246: -1.0, Cpd247: 1.0, Cpd248: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('ACOATA')
reaction.name = 'Acetyl-CoA ACP transacylase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd249: -1.0, Cpd78: 1.0, Cpd250: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('ACODA')
reaction.name = 'acetylornithine deacetylase'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd251: -1.0, Cpd221: 1.0, Cpd252: 1.0})
reaction.gene_reaction_rule = '( ACY1 )'
model.add_reactions([reaction])
reaction = Reaction('ACONT')
reaction.name = 'aconitase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd228: -1.0, Cpd253: 1.0})
reaction.gene_reaction_rule = '( ACO1 or IREB2 )'
model.add_reactions([reaction])
reaction = Reaction('ACONTm')
reaction.name = 'Aconitate hydratase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd254: -1.0, Cpd255: 1.0})
reaction.gene_reaction_rule = '( ACO1 or ACO2 )'
model.add_reactions([reaction])
reaction = Reaction('ACOX22x')
reaction.name = 'acyl-Coenzyme A oxidase 2, branched chain'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -0.5, Cpd256: -1.0, Cpd257: -1.0, Cpd258: 1.0, Cpd259: 1.0})
reaction.gene_reaction_rule = '( ACOX2 )'
model.add_reactions([reaction])
reaction = Reaction('ACOX2x')
reaction.name = 'acyl-Coenzyme A oxidase 2, branched chain'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd257: -1.0, Cpd260: -1.0, Cpd258: 1.0, Cpd261: 2.0, Cpd262: 1.0})
reaction.gene_reaction_rule = '( ACOX2 )'
model.add_reactions([reaction])
reaction = Reaction('ACP1_FMN_')
reaction.name = 'acid phosphatase (FMN)'
reaction.subsystem = 'Vitamin B2 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd263: -1.0, Cpd108: 1.0, Cpd264: 1.0})
reaction.gene_reaction_rule = '( ACP1 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('ACRNtm')
reaction.name = 'O-acetylcarnintine transport into mitochondria via diffusion'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd265: -1.0, Cpd266: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('ACS')
reaction.name = 'acetyl-CoA synthetase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd221: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd180: 1.0})
reaction.gene_reaction_rule = '( AACS or ACSS2 )'
model.add_reactions([reaction])
reaction = Reaction('ACS2')
reaction.name = 'acetyl-CoA synthase (propionate)'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd206: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd207: 1.0})
reaction.gene_reaction_rule = '( AACS or ACSS2 )'
model.add_reactions([reaction])
reaction = Reaction('ACSm')
reaction.name = 'acetyl-CoA synthetase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd267: -1.0, Cpd178: 1.0, Cpd209: 1.0, Cpd210: 1.0})
reaction.gene_reaction_rule = '( ACSS1 )'
model.add_reactions([reaction])
reaction = Reaction('ACSOMT')
reaction.name = 'S-Adenosyl-L-methionine:N-acetylserotonin O-methyltransferase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd118: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd268: 1.0})
reaction.gene_reaction_rule = '( ASMT )'
model.add_reactions([reaction])
reaction = Reaction('ACSRTNMT')
reaction.name = 'S-Adenosyl-L-homocysteine:N-acetylserotonin O-methyltransferase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd269: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd270: 1.0})
reaction.gene_reaction_rule = '( ASMT )'
model.add_reactions([reaction])
reaction = Reaction('ACTLMO')
reaction.name = 'acetol monooxygenase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd271: -1.0, Cpd2: 2.0, Cpd25: 1.0, Cpd153: 1.0})
reaction.gene_reaction_rule = '( CYP2E1 )'
model.add_reactions([reaction])
reaction = Reaction('ACTNMO')
reaction.name = 'acetone monooxygenase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd213: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd271: 1.0})
reaction.gene_reaction_rule = '( CYP2E1 )'
model.add_reactions([reaction])
reaction = Reaction('ACYP')
reaction.name = 'acylphosphatase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd272: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd273: 1.0})
reaction.gene_reaction_rule = '( ACYP1 or ACYP2 or RWDD2A )'
model.add_reactions([reaction])
reaction = Reaction('ADA')
reaction.name = 'Adenosine deaminase'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd274: -1.0, Cpd6: 1.0, Cpd275: 1.0})
reaction.gene_reaction_rule = '( ADA )'
model.add_reactions([reaction])
reaction = Reaction('ADAe')
reaction.name = 'Adenosine deaminase, extracellular'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd114: -1.0, Cpd276: -1.0, Cpd277: 1.0, Cpd278: 1.0})
reaction.gene_reaction_rule = '( ADA )'
model.add_reactions([reaction])
reaction = Reaction('ADEt')
reaction.name = 'adenine reversible transport, cytosol'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd279: -1.0, Cpd280: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('ADEtl')
reaction.name = 'adenine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd281: -1.0, Cpd280: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('ADK1')
reaction.name = 'adenylate kinase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd150: -1.0, Cpd107: 2.0})
reaction.gene_reaction_rule = '( AK1 or AK2 or AK5 or AK7 )'
model.add_reactions([reaction])
reaction = Reaction('ADK1m')
reaction.name = 'adenylate kinase, mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd209: -1.0, Cpd202: 2.0})
reaction.gene_reaction_rule = '( AK2 or AK4 )'
model.add_reactions([reaction])
reaction = Reaction('ADK3')
reaction.name = 'adentylate kinase (GTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd150: -1.0, Cpd282: -1.0, Cpd107: 1.0, Cpd283: 1.0})
reaction.gene_reaction_rule = '( AK5 )'
model.add_reactions([reaction])
reaction = Reaction('ADK3m')
reaction.name = 'adentylate kinase (GTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd209: -1.0, Cpd284: -1.0, Cpd202: 1.0, Cpd285: 1.0})
reaction.gene_reaction_rule = '( AK3 )'
model.add_reactions([reaction])
reaction = Reaction('ADKd')
reaction.name = 'adenylate kinase (d form)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd286: -1.0, Cpd287: -1.0, Cpd288: 2.0})
reaction.gene_reaction_rule = '( AK5 )'
model.add_reactions([reaction])
reaction = Reaction('ADMDC')
reaction.name = 'adenosylmethionine decarboxylase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd52: -1.0, Cpd60: 1.0, Cpd289: 1.0})
reaction.gene_reaction_rule = '( AMD1 )'
model.add_reactions([reaction])
reaction = Reaction('ADNCYC')
reaction.name = 'adenylate cyclase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd151: 1.0, Cpd290: 1.0})
reaction.gene_reaction_rule = '( ADCY1 or ADCY10 or ADCY2 or ADCY3 or ADCY4 or ADCY5 or ADCY6 or ADCY7 or ADCY8 or ADCY9 )'
model.add_reactions([reaction])
reaction = Reaction('ADNK1')
reaction.name = 'adenosine kinase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd274: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd150: 1.0})
reaction.gene_reaction_rule = '( ADK )'
model.add_reactions([reaction])
reaction = Reaction('ADNt')
reaction.name = 'adenosine facilated transport in cytosol'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd276: -1.0, Cpd274: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 or SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('ADNt4')
reaction.name = 'adenosine transport (Na/Adn cotransport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd276: -1.0, Cpd41: 1.0, Cpd274: 1.0})
reaction.gene_reaction_rule = '( SLC28A1 or SLC28A2 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('ADNtl')
reaction.name = 'adenosine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd87: -1.0, Cpd274: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('ADNtm')
reaction.name = 'adenosine facilated transport in mitochondria'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd274: -1.0, Cpd291: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 )'
model.add_reactions([reaction])
reaction = Reaction('ADPGLC')
reaction.name = 'ADPglucose diphosphatase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd292: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd293: 1.0})
reaction.gene_reaction_rule = '( NUDT5 )'
model.add_reactions([reaction])
reaction = Reaction('ADPMAN')
reaction.name = 'ADPmannose diphosphatase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd294: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd295: 1.0})
reaction.gene_reaction_rule = '( NUDT5 )'
model.add_reactions([reaction])
reaction = Reaction('ADPRDP')
reaction.name = 'ADPribose diphosphatase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd296: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd297: 1.0})
reaction.gene_reaction_rule = '( NUDT5 )'
model.add_reactions([reaction])
reaction = Reaction('ADPRDPm')
reaction.name = 'ADPribose diphosphatase (mitochondrial)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd298: -1.0, Cpd7: 2.0, Cpd209: 1.0, Cpd299: 1.0})
reaction.gene_reaction_rule = '( NUDT9 )'
model.add_reactions([reaction])
reaction = Reaction('ADPRIBt')
reaction.name = 'ADPribose transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd300: -1.0, Cpd296: 1.0})
reaction.gene_reaction_rule = '( CD38 )'
model.add_reactions([reaction])
reaction = Reaction('ADPT')
reaction.name = 'adenine phosphoribosyltransferase'
reaction.subsystem = 'Nucleotide salvage pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd280: -1.0, Cpd301: -1.0, Cpd150: 1.0, Cpd151: 1.0})
reaction.gene_reaction_rule = '( APRT )'
model.add_reactions([reaction])
reaction = Reaction('ADRNCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd302: -1.0, Cpd303: -1.0, Cpd78: 1.0, Cpd304: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('ADRNCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd305: -1.0, Cpd306: 1.0, Cpd307: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('ADRNCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd304: -1.0, Cpd305: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('ADRNLPVESSEC')
reaction.name = 'Adrenaline secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd96: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd308: 1.0})
reaction.gene_reaction_rule = '( SLC18A1 or SLC18A2 )'
model.add_reactions([reaction])
reaction = Reaction('ADSELK')
reaction.name = 'adenylyl-selenate kinase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd309: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd310: 1.0})
reaction.gene_reaction_rule = '( PAPSS1 or PAPSS2 )'
model.add_reactions([reaction])
reaction = Reaction('ADSK')
reaction.name = 'adenylyl-sulfate kinase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd311: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd23: 1.0})
reaction.gene_reaction_rule = '( PAPSS1 or PAPSS2 )'
model.add_reactions([reaction])
reaction = Reaction('ADSL1')
reaction.name = 'adenylosuccinate lyase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd312: -1.0, Cpd150: 1.0, Cpd313: 1.0})
reaction.gene_reaction_rule = '( ADSL )'
model.add_reactions([reaction])
reaction = Reaction('ADSL2')
reaction.name = 'adenylosuccinate lyase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd314: -1.0, Cpd313: 1.0, Cpd315: 1.0})
reaction.gene_reaction_rule = '( ADSL )'
model.add_reactions([reaction])
reaction = Reaction('ADSS')
reaction.name = 'adenylosuccinate synthase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd90: -1.0, Cpd282: -1.0, Cpd316: -1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd283: 1.0, Cpd312: 1.0})
reaction.gene_reaction_rule = '( ADSS or ADSSL1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T10g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd317: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd318: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T11g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd319: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd320: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T12g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd321: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd322: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T13g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd323: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd324: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T14g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd325: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd326: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T15g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd327: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd328: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T16g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd329: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd330: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T17g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd331: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd332: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T18g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd333: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd334: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T1g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd335: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd330: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T2g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd331: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd332: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T3g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd333: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd334: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T4g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd336: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd337: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T5g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd338: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd339: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T6g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd340: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd341: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T7g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd342: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd343: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T8g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd344: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd345: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AG13T9g')
reaction.name = 'N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd346: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd347: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 or B3GNT3 or B3GNT4 or B3GNT7 or B3GNT8 or B3GNTL1 or B4GAT1 )'
model.add_reactions([reaction])
reaction = Reaction('AGDC')
reaction.name = 'N-acetylglucosamine-6-phosphate deacetylase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd218: -1.0, Cpd217: 1.0, Cpd221: 1.0})
reaction.gene_reaction_rule = '( AMDHD2 )'
model.add_reactions([reaction])
reaction = Reaction('AGMTm')
reaction.name = 'agmatinase (m)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd348: -1.0, Cpd349: 1.0, Cpd350: 1.0})
reaction.gene_reaction_rule = '( AGMAT )'
model.add_reactions([reaction])
reaction = Reaction('AGPAT1')
reaction.name = '1-acylglycerol-3-phosphate O-acyltransferase 1'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd351: -1.0, Cpd352: -1.0, Cpd78: 1.0, Cpd353: 1.0})
reaction.gene_reaction_rule = '( AGPAT1 or AGPAT2 or AGPAT3 or AGPAT4 or AGPAT5 or GPAT4 or MBOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('AGPRim')
reaction.name = 'N-acetyl-g-glutamyl-phosphate reductase, irreversible, mitochondrial'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd354: -1.0, Cpd13: 1.0, Cpd204: 1.0, Cpd355: 1.0})
reaction.gene_reaction_rule = '( ALDH18A1 )'
model.add_reactions([reaction])
reaction = Reaction('AGPSx')
reaction.name = 'alkylglycerone phosphate synthase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd356: -1.0, Cpd357: -1.0, Cpd358: 1.0, Cpd359: 1.0, Cpd360: 1.0})
reaction.gene_reaction_rule = '( AGPS )'
model.add_reactions([reaction])
reaction = Reaction('AGTim')
reaction.name = 'alanine-glyoxylate transaminase (irreversible), mitochondrial'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd361: -1.0, Cpd362: -1.0, Cpd363: 1.0, Cpd364: 1.0})
reaction.gene_reaction_rule = '( AGXT2 or ETNPPL )'
model.add_reactions([reaction])
reaction = Reaction('AGTix')
reaction.name = 'alanine-glyoxylate transaminase (irreversible), (peroxisomal)'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd365: -1.0, Cpd366: -1.0, Cpd367: 1.0, Cpd368: 1.0})
reaction.gene_reaction_rule = '( AGXT )'
model.add_reactions([reaction])
reaction = Reaction('AHANDROSTANGLCte')
reaction.name = 'glucuronidated compound transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd369: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd370: 1.0})
reaction.gene_reaction_rule = '( ABCC1 )'
model.add_reactions([reaction])
reaction = Reaction('AHC')
reaction.name = 'adenosylhomocysteinase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd53: -1.0, Cpd274: 1.0, Cpd371: 1.0})
reaction.gene_reaction_rule = '( AHCYL1 or GM4737 )'
model.add_reactions([reaction])
reaction = Reaction('AHEXASE2ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -3.0, Cpd372: -1.0, Cpd131: 1.0, Cpd373: 3.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('AHEXASEly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd374: -1.0, Cpd131: 1.0, Cpd373: 2.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('AICART')
reaction.name = 'phosphoribosylaminoimidazolecarboxamide formyltransferase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd375: -1.0, Cpd315: -1.0, Cpd376: 1.0, Cpd377: 1.0})
reaction.gene_reaction_rule = '( ATIC )'
model.add_reactions([reaction])
reaction = Reaction('AIRCr')
reaction.name = 'phosphoribosylaminoimidazole carboxylase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd60: -1.0, Cpd378: -1.0, Cpd33: 1.0, Cpd379: 1.0})
reaction.gene_reaction_rule = '( PAICS )'
model.add_reactions([reaction])
reaction = Reaction('AKGDm')
reaction.name = '2-oxoglutarate dehydrogenase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd380: 1.0})
reaction.gene_reaction_rule = '( DLD or DLST or OGDH or PDHX )'
model.add_reactions([reaction])
reaction = Reaction('AKGMALtm')
reaction.name = 'alpha-ketoglutarate/malate transporter'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd381: -1.0, Cpd30: 1.0, Cpd382: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 or SLC25A11 )'
model.add_reactions([reaction])
reaction = Reaction('AKGt4_3')
reaction.name = 'AKG transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -3.0, Cpd383: -1.0, Cpd30: 1.0, Cpd41: 3.0})
reaction.gene_reaction_rule = '( SLC13A3 )'
model.add_reactions([reaction])
reaction = Reaction('AKR1C1')
reaction.name = 'aldo-keto reductase family 1, member C1 (chlordecone reductase; 3-alpha hydroxysteroid dehydrogenase, type I; dihydrodiol dehydrogenase 4)'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd384: -1.0, Cpd25: 1.0, Cpd385: 1.0})
reaction.gene_reaction_rule = '( AKR1C6 )'
model.add_reactions([reaction])
reaction = Reaction('AKR1C41')
reaction.name = 'aldo-keto reductase family 1, member C4 (chlordecone reductase; 3-alpha hydroxysteroid dehydrogenase, type I; dihydrodiol dehydrogenase 4)'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd386: -1.0, Cpd32: 1.0, Cpd387: 1.0})
reaction.gene_reaction_rule = '( AKR1C6 )'
model.add_reactions([reaction])
reaction = Reaction('AKR1C42')
reaction.name = 'aldo-keto reductase family 1, member C4 (chlordecone reductase; 3-alpha hydroxysteroid dehydrogenase, type I; dihydrodiol dehydrogenase 4)'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd388: -1.0, Cpd32: 1.0, Cpd389: 1.0})
reaction.gene_reaction_rule = '( AKR1C6 )'
model.add_reactions([reaction])
reaction = Reaction('AKR1D')
reaction.name = 'aldo-keto reductase family 1, member D1 (delta 4-3-ketosteroid-5-beta-reductase)'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd390: -1.0, Cpd25: 1.0, Cpd386: 1.0})
reaction.gene_reaction_rule = '( AKR1D1 )'
model.add_reactions([reaction])
reaction = Reaction('AKR1D2')
reaction.name = 'aldo-keto reductase family 1, member D1 (delta 4-3-ketosteroid-5-beta-reductase)'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd391: -1.0, Cpd25: 1.0, Cpd388: 1.0})
reaction.gene_reaction_rule = '( AKR1D1 )'
model.add_reactions([reaction])
reaction = Reaction('ALAASNNaEx')
reaction.name = 'L-alanine/L-asparagine Na-dependent exchange (Ala-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd393: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd395: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ALACYSNaEx')
reaction.name = 'L-alanine/L-cysteine Na-dependent exchange (Ala-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd396: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd397: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ALADGLNexR')
reaction.name = 'L-alanine/glutamine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd398: -1.0, Cpd399: -1.0, Cpd400: 1.0, Cpd401: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('ALADGLYexR')
reaction.name = 'D-alanine/glycine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd398: -1.0, Cpd402: -1.0, Cpd400: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('ALAGLNexR')
reaction.name = 'L-alanine/glutamine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd399: -1.0, Cpd394: 1.0, Cpd401: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('ALAGLNNaEx')
reaction.name = 'L-alanine/L-glutamine Na-dependent exchange (Ala-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd399: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd401: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ALAGLYexR')
reaction.name = 'L-alanine/glycine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd402: -1.0, Cpd394: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('ALAR')
reaction.name = 'alanine racemase'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd400: 1.0})
reaction.gene_reaction_rule = '( PROSC )'
model.add_reactions([reaction])
reaction = Reaction('ALASERNaEx')
reaction.name = 'L-alanine/L-serine Na-dependent exchange (Ala-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd404: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd405: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ALASm')
reaction.name = '5-aminolevulinate synthase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd364: -1.0, Cpd380: -1.0, Cpd43: 1.0, Cpd45: 1.0, Cpd406: 1.0})
reaction.gene_reaction_rule = '( ALAS1 or ALAS2 or GCAT )'
model.add_reactions([reaction])
reaction = Reaction('ALAt2r')
reaction.name = 'L-alanine reversible transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd392: -1.0, Cpd33: 1.0, Cpd394: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 or SLC36A2 )'
model.add_reactions([reaction])
reaction = Reaction('ALAt2rL')
reaction.name = 'L-alanine reversible transport via proton symport (lysosome)'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd407: -1.0, Cpd33: 1.0, Cpd394: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('ALAt4')
reaction.name = 'Alanine-Sodium symporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd41: 1.0, Cpd394: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC38A4 or SLC6A14 or SLC6A19 or SLC7A1 or SLC7A2 or SLC7A3 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('ALATA_L')
reaction.name = 'L-alanine transaminase'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd394: -1.0, Cpd21: 1.0, Cpd74: 1.0})
reaction.gene_reaction_rule = '( GPT or GPT2 )'
model.add_reactions([reaction])
reaction = Reaction('ALATHRNaEx')
reaction.name = 'L-alanine/L-threonine Na-dependent exchange (Ala-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd408: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ALAtN1')
reaction.name = 'Alanine transport (Na, H coupled)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -2.0, Cpd392: -1.0, Cpd37: 1.0, Cpd41: 2.0, Cpd394: 1.0})
reaction.gene_reaction_rule = '( SLC38A3 or SLC38A5 )'
model.add_reactions([reaction])
reaction = Reaction('ALCD1')
reaction.name = 'alcohol dehydrogenase (methanol)'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd410: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd411: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 or ADHFE1 or ZADH2 )'
model.add_reactions([reaction])
reaction = Reaction('ALCD21_D')
reaction.name = 'alcohol dehydrogenase (D-1,2-propanediol)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd412: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd413: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 or ADHFE1 or ZADH2 )'
model.add_reactions([reaction])
reaction = Reaction('ALCD21_L')
reaction.name = 'alcohol dehydrogenase (L-1,2-propanediol)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd414: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd415: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 or ADHFE1 or ZADH2 )'
model.add_reactions([reaction])
reaction = Reaction('ALCD22_D')
reaction.name = 'alcohol dehydrogenase (D-lactaldehyde)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd413: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd153: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 or ADHFE1 or ZADH2 )'
model.add_reactions([reaction])
reaction = Reaction('ALCD22_L')
reaction.name = 'alcohol dehydrogenase (L-lactaldehyde)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd415: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd153: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 or ADHFE1 or ZADH2 )'
model.add_reactions([reaction])
reaction = Reaction('ALCD2if')
reaction.name = 'alcohol dehydrogenase, forward rxn (ethanol -> acetaldehyde)'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd416: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd417: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 or ADHFE1 or ZADH2 )'
model.add_reactions([reaction])
reaction = Reaction('ALCD2yf')
reaction.name = 'alcohol dehydrogenase (ethanol, NADP), forward reaction'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd416: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd417: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 )'
model.add_reactions([reaction])
reaction = Reaction('ALDD20x')
reaction.name = 'aldehyde dehydrogenase (indole-3-acetaldehyde, NAD)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd418: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd419: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('ALDD20xm')
reaction.name = 'aldehyde dehydrogenase (indole-3-acetaldehyde, NAD), mitochondrial'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd420: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd421: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('ALDD21')
reaction.name = 'aldehyde dehydrogenase (pristanal, NAD)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd422: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd423: 1.0})
reaction.gene_reaction_rule = '( ALDH3A2 )'
model.add_reactions([reaction])
reaction = Reaction('ALDD2x')
reaction.name = 'aldehyde dehydrogenase (acetaldehyde, NAD)'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd417: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd221: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH1A3 or ALDH1B1 or ALDH3A1 or ALDH3A2 or ALDH3B1 or ALDH3B3 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('ALDD2xm')
reaction.name = 'aldehyde dehydrogenase (acetylaldehyde, NAD), mitochondrial'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd424: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd267: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 )'
model.add_reactions([reaction])
reaction = Reaction('ALDD2y')
reaction.name = 'aldehyde dehydrogenase (acetaldehyde, NADP)'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd25: -1.0, Cpd417: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd221: 1.0})
reaction.gene_reaction_rule = '( ALDH1A2 or ALDH1B1 or ALDH3A1 or ALDH3A2 or ALDH3B1 or ALDH3B3 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('ALKP')
reaction.name = 'alkaline phosphatase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd425: -1.0, Cpd108: 1.0, Cpd426: 1.0})
reaction.gene_reaction_rule = '( ALPL or ALPPL2 )'
model.add_reactions([reaction])
reaction = Reaction('ALOX12')
reaction.name = 'Arachidonate 12-lipoxygenase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd427: -1.0, Cpd428: 1.0})
reaction.gene_reaction_rule = '( ALOX12 )'
model.add_reactions([reaction])
reaction = Reaction('ALOX12R')
reaction.name = 'Arachidonate 12-lipoxygenase R'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd427: -1.0, Cpd429: 1.0})
reaction.gene_reaction_rule = '( ALOX12B )'
model.add_reactions([reaction])
reaction = Reaction('ALOX15')
reaction.name = 'Arachidonate 15-lipoxygenase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd427: -1.0, Cpd430: 1.0})
reaction.gene_reaction_rule = '( ALOX15 or ALOX8 )'
model.add_reactions([reaction])
reaction = Reaction('ALOX5')
reaction.name = 'Arachidonate 5-lipoxygenase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd427: -1.0, Cpd431: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('ALOX52')
reaction.name = 'Arachidonate 5-lipoxygenase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd431: -1.0, Cpd2: 1.0, Cpd432: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('ALR2')
reaction.name = 'aldose reductase (methylglyoxal)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd153: -1.0, Cpd25: 1.0, Cpd271: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 or AKR7A5 )'
model.add_reactions([reaction])
reaction = Reaction('ALR3')
reaction.name = 'aldose reductase (acetol)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd271: -1.0, Cpd25: 1.0, Cpd414: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 or AKR7A5 )'
model.add_reactions([reaction])
reaction = Reaction('AMACR2p')
reaction.name = 'alpha-methylacyl-CoA racemase (reductase)'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -0.5, Cpd433: -1.0, Cpd256: 1.0, Cpd261: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('AMACR2r')
reaction.name = 'alpha-methylacyl-CoA racemase (reductase)'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd434: -1.0, Cpd435: -0.5, Cpd436: 1.0, Cpd437: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('AMACRp')
reaction.name = 'alpha-methylacyl-CoA racemase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd438: -1.0, Cpd260: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('AMACRr')
reaction.name = 'alpha-methylacyl-CoA racemase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd439: -1.0, Cpd440: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('AMANK')
reaction.name = 'N-acetyl-D-mannosamine kinase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd216: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd230: 1.0})
reaction.gene_reaction_rule = '( GNE or NAGK )'
model.add_reactions([reaction])
reaction = Reaction('AMETt2m')
reaction.name = 'S-Adenosyl-L-methionine reversible transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd441: -1.0, Cpd53: 1.0, Cpd442: 1.0})
reaction.gene_reaction_rule = '( SLC25A26 )'
model.add_reactions([reaction])
reaction = Reaction('AMPDA')
reaction.name = 'Adenosine monophosphate deaminase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd150: -1.0, Cpd6: 1.0, Cpd316: 1.0})
reaction.gene_reaction_rule = '( AMPD1 or AMPD2 or AMPD3 )'
model.add_reactions([reaction])
reaction = Reaction('AMPTASECG')
reaction.name = 'alanyl aminopeptidase (cys-gly)'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd443: -1.0, Cpd396: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( ANPEP )'
model.add_reactions([reaction])
reaction = Reaction('AMPTASECGe')
reaction.name = 'alanyl aminopeptidase (cys-gly) (e)'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd444: -1.0, Cpd397: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( ANPEP )'
model.add_reactions([reaction])
reaction = Reaction('AMY1e')
reaction.name = 'alpha-amylase, extracellular (strch1 -> strch2)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -8.0, Cpd445: -1.0, Cpd446: 8.0, Cpd447: 1.0})
reaction.gene_reaction_rule = '( AMY2A5 or SLC3A1 or SLC3A2 or SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('AMY2e')
reaction.name = 'alpha-amylase, extracellular (glygn2 -> glygn4)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -8.0, Cpd448: -1.0, Cpd446: 8.0, Cpd449: 1.0})
reaction.gene_reaction_rule = '( AMY2A5 or SLC3A1 or SLC3A2 or SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('ANDRSTRNGLCte')
reaction.name = 'glucuronidated compound transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd450: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd451: 1.0})
reaction.gene_reaction_rule = '( ABCC1 )'
model.add_reactions([reaction])
reaction = Reaction('AP4AH1')
reaction.name = 'Ap4A hydrolase, asymmetrically'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd452: -1.0, Cpd105: 1.0, Cpd33: 2.0, Cpd150: 1.0})
reaction.gene_reaction_rule = '( NUDT2 )'
model.add_reactions([reaction])
reaction = Reaction('APAT2rm')
reaction.name = '3-Aminopropanoate:2-oxoglutarate aminotransferase (m)'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd453: -1.0, Cpd63: 1.0, Cpd454: 1.0})
reaction.gene_reaction_rule = '( ABAT )'
model.add_reactions([reaction])
reaction = Reaction('APRTO2')
reaction.name = 'N-acetylputrescine: oxygen oxireductase (deaminating)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd455: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd456: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('ARABR')
reaction.name = 'arabinose reductase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd457: -1.0, Cpd25: 1.0, Cpd458: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 or AKR7A5 )'
model.add_reactions([reaction])
reaction = Reaction('ARACHCPT1')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd459: -1.0, Cpd78: 1.0, Cpd460: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('ARACHCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd461: -1.0, Cpd307: 1.0, Cpd462: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('ARACHCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd460: -1.0, Cpd461: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('ARACHDt2')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd463: -1.0, Cpd427: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('ARGDCm')
reaction.name = 'arginine decarboxylase (m)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd464: -1.0, Cpd45: 1.0, Cpd348: 1.0})
reaction.gene_reaction_rule = '( AZIN2 )'
model.add_reactions([reaction])
reaction = Reaction('ARGLYSex')
reaction.name = 'Arginine/Lysine exchanger (Arg in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd465: -1.0, Cpd466: -1.0, Cpd467: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('ARGN')
reaction.name = 'arginase'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd467: -1.0, Cpd252: 1.0, Cpd469: 1.0})
reaction.gene_reaction_rule = '( ARG1 or ARG2 )'
model.add_reactions([reaction])
reaction = Reaction('ARGNm')
reaction.name = 'arginase (m)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd464: -1.0, Cpd350: 1.0, Cpd470: 1.0})
reaction.gene_reaction_rule = '( ARG2 )'
model.add_reactions([reaction])
reaction = Reaction('ARGSS')
reaction.name = 'argininosuccinate synthase'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd90: -1.0, Cpd471: -1.0, Cpd33: 1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd472: 1.0})
reaction.gene_reaction_rule = '( ASS1 )'
model.add_reactions([reaction])
reaction = Reaction('ARGt4')
reaction.name = 'L-arginine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd466: -1.0, Cpd41: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('ARGtiDF')
reaction.name = 'L-arginine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC38A4 or SLC7A1 or SLC7A2 or SLC7A3 )'
model.add_reactions([reaction])
reaction = Reaction('ARGtm')
reaction.name = 'arginine mitochondrial transport via ornithine carrier'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd467: -1.0, Cpd33: 1.0, Cpd464: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('ARSA')
reaction.name = 'Arylsulfatase A'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd473: -1.0, Cpd176: 1.0, Cpd474: 1.0, Cpd475: 1.0})
reaction.gene_reaction_rule = '( ARSA )'
model.add_reactions([reaction])
reaction = Reaction('ASAH1')
reaction.name = 'N-acylsphingosine amidohydrolase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd476: -1.0, Cpd477: 1.0, Cpd478: 1.0})
reaction.gene_reaction_rule = '( ASAH1 )'
model.add_reactions([reaction])
reaction = Reaction('ASCBt4')
reaction.name = 'L-ascorbate transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd479: -1.0, Cpd41: 1.0, Cpd480: 1.0})
reaction.gene_reaction_rule = '( SLC23A1 or SLC23A2 )'
model.add_reactions([reaction])
reaction = Reaction('ASNALANaEx')
reaction.name = 'L-alanine/L-asparagine Na-dependent exchange (Asn-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd395: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd393: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ASNCYSNaEx')
reaction.name = 'L-cysteine/L-asparagine Na-dependent exchange (Asn-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd396: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd397: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ASNGLNNaEx')
reaction.name = 'L-glutamine/L-asparagine Na-dependent exchange (Asn-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd399: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd401: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ASNNm')
reaction.name = 'L-asparaginase (mitochondrial)'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd481: -1.0, Cpd92: 1.0, Cpd482: 1.0})
reaction.gene_reaction_rule = '( ASRGL1 )'
model.add_reactions([reaction])
reaction = Reaction('ASNS1')
reaction.name = 'asparagine synthase (glutamine-hydrolysing)'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd90: -1.0, Cpd399: -1.0, Cpd33: 1.0, Cpd74: 1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd393: 1.0})
reaction.gene_reaction_rule = '( ASNS )'
model.add_reactions([reaction])
reaction = Reaction('ASNSERNaEx')
reaction.name = 'L-serine/L-asparagine Na-dependent exchange (Asn-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd404: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd405: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ASNt4')
reaction.name = 'L-asparagine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd41: 1.0, Cpd393: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC38A4 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('ASNTHRNaEx')
reaction.name = 'L-threonine/L-asparagine Na-dependent exchange (Asn-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd408: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('ASNtN1')
reaction.name = 'Asparagine transport (Na, H coupled)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -2.0, Cpd395: -1.0, Cpd37: 1.0, Cpd41: 2.0, Cpd393: 1.0})
reaction.gene_reaction_rule = '( SLC38A3 or SLC38A5 )'
model.add_reactions([reaction])
reaction = Reaction('ASP1DC')
reaction.name = 'aspartate 1-decarboxylase'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd90: -1.0, Cpd60: 1.0, Cpd483: 1.0})
reaction.gene_reaction_rule = '( GAD1 )'
model.add_reactions([reaction])
reaction = Reaction('ASPCTr')
reaction.name = 'aspartate carbamoyltransferase (reversible)'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd90: -1.0, Cpd484: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd485: 1.0})
reaction.gene_reaction_rule = '( CAD )'
model.add_reactions([reaction])
reaction = Reaction('ASPDt6')
reaction.name = 'D-aspartate transport via Na, H symport and K antiport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd39: -3.0, Cpd486: -1.0, Cpd487: -1.0, Cpd33: 1.0, Cpd41: 3.0, Cpd488: 1.0, Cpd489: 1.0})
reaction.gene_reaction_rule = '( SLC1A1 or SLC1A2 or SLC1A3 or SLC1A6 or SLC1A7 )'
model.add_reactions([reaction])
reaction = Reaction('ASPGLUm')
reaction.name = 'aspartate-glutamate mitochondrial shuttle'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd74: -1.0, Cpd92: -1.0, Cpd7: 1.0, Cpd63: 1.0, Cpd90: 1.0})
reaction.gene_reaction_rule = '( SLC25A12 or SLC25A13 )'
model.add_reactions([reaction])
reaction = Reaction('ASPt6')
reaction.name = 'L-aspartate transport via Na, H symport and K antiport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd39: -3.0, Cpd487: -1.0, Cpd490: -1.0, Cpd33: 1.0, Cpd41: 3.0, Cpd90: 1.0, Cpd489: 1.0})
reaction.gene_reaction_rule = '( SLC1A1 or SLC1A2 or SLC1A3 or SLC1A6 or SLC1A7 )'
model.add_reactions([reaction])
reaction = Reaction('ASPTA')
reaction.name = 'aspartate transaminase'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd90: -1.0, Cpd74: 1.0, Cpd229: 1.0})
reaction.gene_reaction_rule = '( GOT1 )'
model.add_reactions([reaction])
reaction = Reaction('ASPTAm')
reaction.name = 'aspartate transaminase'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd92: -1.0, Cpd63: 1.0, Cpd491: 1.0})
reaction.gene_reaction_rule = '( GOT2 )'
model.add_reactions([reaction])
reaction = Reaction('ATPasel')
reaction.name = 'V-type ATPase, H+ transporting, lysosomal'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd107: 1.0, Cpd108: 1.0, Cpd176: 1.0})
reaction.gene_reaction_rule = '( ATP6V0A1 or ATP6V0A2 or ATP6V0A4 or ATP6V0B or ATP6V0C or ATP6V0D1 or ATP6V0D2 or ATP6V0E or ATP6V1A or ATP6V1B1 or ATP6V1B2 or ATP6V1C1 or ATP6V1C2 or ATP6V1D or ATP6V1E1 or ATP6V1E2 or ATP6V1F or ATP6V1G1 or ATP6V1G2 or ATP6V1G3 or ATP6V1H or TCIRG1 )'
model.add_reactions([reaction])
reaction = Reaction('ATPH1e')
reaction.name = 'ATP diphosphohydrolase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -2.0, Cpd492: -1.0, Cpd37: 2.0, Cpd493: 1.0, Cpd494: 2.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD2 or ENTPD3 or ENTPD5 or ENTPD6 )'
model.add_reactions([reaction])
reaction = Reaction('ATPH2e')
reaction.name = 'ATP diphosphohydrolase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd495: -1.0, Cpd37: 1.0, Cpd493: 1.0, Cpd494: 1.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD2 or ENTPD3 or ENTPD5 or ENTPD6 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('ATPS4m')
reaction.name = 'ATP synthase (four protons for one ATP)'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -4.0, Cpd202: -1.0, Cpd204: -1.0, Cpd7: 3.0, Cpd12: 1.0, Cpd200: 1.0})
reaction.gene_reaction_rule = '( ATP5A1 or ATP5B or ATP5C1 or ATP5D or ATP5E or ATP5F1 or ATP5G1 or ATP5G2 or ATP5G3 or ATP5H or ATP5J or ATP5J2 or ATP5K or ATP5O or ATP5S or GM5426 or NSF )'
model.add_reactions([reaction])
reaction = Reaction('ATPtm')
reaction.name = 'ADP/ATP transporter, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd107: -1.0, Cpd200: -1.0, Cpd105: 1.0, Cpd202: 1.0})
reaction.gene_reaction_rule = '( SLC25A4 or SLC25A5 )'
model.add_reactions([reaction])
reaction = Reaction('AVITE1t')
reaction.name = 'alpha-Tocopherol (Vit. E) transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd496: -1.0, Cpd497: 1.0})
reaction.gene_reaction_rule = '( NPC1L1 or SCARB1 )'
model.add_reactions([reaction])
reaction = Reaction('B_MANNASEly')
reaction.name = 'beta-mannosidase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd133: -1.0, Cpd132: 1.0, Cpd373: 1.0})
reaction.gene_reaction_rule = '( MANBA )'
model.add_reactions([reaction])
reaction = Reaction('B3GALT3g')
reaction.name = 'Beta-1,3-galactosyltransferase 3'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd141: -1.0, Cpd155: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd498: 1.0})
reaction.gene_reaction_rule = '( B3GALNT1 )'
model.add_reactions([reaction])
reaction = Reaction('B3GALT41g')
reaction.name = 'Beta-1,3-galactosyltransferase 4'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd499: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd500: 1.0})
reaction.gene_reaction_rule = '( B3GALT4 )'
model.add_reactions([reaction])
reaction = Reaction('B3GALT42g')
reaction.name = 'Beta-1,3-galactosyltransferase 4'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd501: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd502: 1.0})
reaction.gene_reaction_rule = '( B3GALT4 )'
model.add_reactions([reaction])
reaction = Reaction('B3GALT43g')
reaction.name = 'Beta-1,3-galactosyltransferase 4'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd503: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd504: 1.0})
reaction.gene_reaction_rule = '( B3GALT4 )'
model.add_reactions([reaction])
reaction = Reaction('B3GALT44g')
reaction.name = 'Beta-1,3-galactosyltransferase 4'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd505: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd506: 1.0})
reaction.gene_reaction_rule = '( B3GALT4 )'
model.add_reactions([reaction])
reaction = Reaction('B3GALT5g')
reaction.name = 'Beta-1,3-galactosyltransferase 5'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd498: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd507: 1.0})
reaction.gene_reaction_rule = '( B3GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT11g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd508: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd509: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT12g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd510: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd511: 1.0})
reaction.gene_reaction_rule = '( B3GNT2 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT310g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd512: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd508: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT311g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd160: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd513: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT312g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd509: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd510: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT313g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd164: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd514: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT314g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd515: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd516: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT315g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd511: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd517: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT31g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd141: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd518: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT32g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd518: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd519: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT33g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd519: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd520: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT34g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd507: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd521: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT35g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd521: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd522: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT36g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd523: -1.0, Cpd498: -1.0, Cpd524: 1.0, Cpd525: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT37g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd526: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd527: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT39g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd528: -1.0, Cpd138: 1.0, Cpd140: 1.0, Cpd142: 1.0})
reaction.gene_reaction_rule = '( B3GNT3 )'
model.add_reactions([reaction])
reaction = Reaction('B3GNT51g')
reaction.name = 'UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 5'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd138: -1.0, Cpd144: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd512: 1.0})
reaction.gene_reaction_rule = '( B3GNT5 )'
model.add_reactions([reaction])
reaction = Reaction('BAAT1x')
reaction.name = 'bile acid Coenzyme A: amino acid N-acyltransferase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd367: -1.0, Cpd529: -1.0, Cpd184: 1.0, Cpd530: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('BAAT2x')
reaction.name = 'bile acid Coenzyme A: amino acid N-acyltransferase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd529: -1.0, Cpd531: -1.0, Cpd184: 1.0, Cpd532: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('BAAT3x')
reaction.name = 'bile acid Coenzyme A: amino acid N-acyltransferase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd367: -1.0, Cpd533: -1.0, Cpd184: 1.0, Cpd534: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('BAAT4x')
reaction.name = 'bile acid Coenzyme A: amino acid N-acyltransferase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd367: -1.0, Cpd535: -1.0, Cpd184: 1.0, Cpd534: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('BACCL')
reaction.name = 'biotin-[acetyl-CoA-carboxylase] ligase'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd536: -1.0, Cpd151: 1.0, Cpd537: 1.0})
reaction.gene_reaction_rule = '( HLCS )'
model.add_reactions([reaction])
reaction = Reaction('BACCLm')
reaction.name = 'biotin-[acetyl-CoA-carboxylase] ligase, mitochondrial'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd200: -1.0, Cpd538: -1.0, Cpd210: 1.0, Cpd539: 1.0})
reaction.gene_reaction_rule = '( HLCS )'
model.add_reactions([reaction])
reaction = Reaction('BALAVECSEC')
reaction.name = 'B-alanine secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd483: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd540: 1.0})
reaction.gene_reaction_rule = '( SLC32A1 )'
model.add_reactions([reaction])
reaction = Reaction('BAMPPALDOX')
reaction.name = 'beta-Aminopropion aldehyde:NAD+ oxidoreductase'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd4: -1.0, Cpd32: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd483: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('BAMPPALDOXm')
reaction.name = 'beta-Aminopropion aldehyde:NAD+ oxidoreductase (m)'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd541: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd453: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('BBHOX')
reaction.name = '4-Trimethylammoniobutanoate,2-oxoglutarate:oxygen oxidoreductase (3-hydroxylating)'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd30: -1.0, Cpd542: -1.0, Cpd60: 1.0, Cpd303: 1.0, Cpd543: 1.0})
reaction.gene_reaction_rule = '( BBOX1 )'
model.add_reactions([reaction])
reaction = Reaction('BCDO')
reaction.name = 'beta-Carotene dioxygenase'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd544: -1.0, Cpd545: 2.0})
reaction.gene_reaction_rule = '( BCO1 )'
model.add_reactions([reaction])
reaction = Reaction('BDHm')
reaction.name = '(R)-3-Hydroxybutanoate:NAD+ oxidoreductase'
reaction.subsystem = 'Butanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd546: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd186: 1.0})
reaction.gene_reaction_rule = '( BDH1 )'
model.add_reactions([reaction])
reaction = Reaction('BDMT_L')
reaction.name = 'GDPmannose:chitobiosyldiphosphodolichol beta-D-mannosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd547: -0.1, Cpd548: -1.0, Cpd33: 1.0, Cpd283: 1.0, Cpd549: 0.1})
reaction.gene_reaction_rule = '( ALG1 )'
model.add_reactions([reaction])
reaction = Reaction('BDMT_U')
reaction.name = 'GDPmannose:chitobiosyldiphosphodolichol beta-D-mannosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd548: -1.0, Cpd550: -0.1, Cpd33: 1.0, Cpd283: 1.0, Cpd551: 0.1})
reaction.gene_reaction_rule = '( ALG1 )'
model.add_reactions([reaction])
reaction = Reaction('BHMT')
reaction.name = 'betaine-homocysteine S-methyltransferase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd371: -1.0, Cpd552: -1.0, Cpd553: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( BHMT or BHMT2 )'
model.add_reactions([reaction])
reaction = Reaction('BILDGLCURt')
reaction.name = 'bilirubin beta-diglucuronide transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd555: -1.0, Cpd38: 1.0, Cpd556: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('BILDGLCURte')
reaction.name = 'bilirubin beta-diglucuronide transport MDR'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd556: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd555: 1.0})
reaction.gene_reaction_rule = '( ABCC1 or ABCC2 or ABCC3 )'
model.add_reactions([reaction])
reaction = Reaction('BILGLCURt')
reaction.name = 'bilirubin monoglucuronide transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd557: -1.0, Cpd38: 1.0, Cpd558: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('BILGLCURte')
reaction.name = 'bilirubin monoglucuronide transport MDR'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd558: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd557: 1.0})
reaction.gene_reaction_rule = '( ABCC1 )'
model.add_reactions([reaction])
reaction = Reaction('BILIRED')
reaction.name = 'Nad(p)h biliverdin reductase'
reaction.subsystem = 'Heme degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd559: -1.0, Cpd25: 1.0, Cpd560: 1.0})
reaction.gene_reaction_rule = '( BLVRA or BLVRB )'
model.add_reactions([reaction])
reaction = Reaction('BILIRUBt2')
reaction.name = 'bilirubin transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd561: -1.0, Cpd38: 1.0, Cpd560: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('BMTer_L')
reaction.name = 'B mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd563: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd566: 1.0})
reaction.gene_reaction_rule = '( PIGB )'
model.add_reactions([reaction])
reaction = Reaction('BMTer_U')
reaction.name = 'B mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd563: -1.0, Cpd567: -0.1, Cpd565: 1.0, Cpd566: 1.0, Cpd564: 0.1})
reaction.gene_reaction_rule = '( PIGB )'
model.add_reactions([reaction])
reaction = Reaction('BPNT')
reaction.name = '3,5-bisphosphate nucleotidase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd26: -1.0, Cpd108: 1.0, Cpd150: 1.0})
reaction.gene_reaction_rule = '( BPNT1 )'
model.add_reactions([reaction])
reaction = Reaction('BPNT2')
reaction.name = '3,5-bisphosphate nucleotidase (paps)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd23: -1.0, Cpd108: 1.0, Cpd311: 1.0})
reaction.gene_reaction_rule = '( BPNT1 )'
model.add_reactions([reaction])
reaction = Reaction('BTND1')
reaction.name = 'biotinidase (biotin)'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd568: -1.0, Cpd465: 1.0, Cpd536: 1.0})
reaction.gene_reaction_rule = '( BTD )'
model.add_reactions([reaction])
reaction = Reaction('BTND1n')
reaction.name = 'biotinidase (biotin), nuclear'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd569: -1.0, Cpd570: -1.0, Cpd571: 1.0, Cpd572: 1.0})
reaction.gene_reaction_rule = '( BTD )'
model.add_reactions([reaction])
reaction = Reaction('BTNDe')
reaction.name = 'biotinidase (biotin), extracellular'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd573: -1.0, Cpd468: 1.0, Cpd574: 1.0})
reaction.gene_reaction_rule = '( BTD )'
model.add_reactions([reaction])
reaction = Reaction('BTNDm')
reaction.name = 'biotinidase (biotin), mitochondrial'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd575: -1.0, Cpd576: 1.0, Cpd538: 1.0})
reaction.gene_reaction_rule = '( BTD )'
model.add_reactions([reaction])
reaction = Reaction('BTNPL')
reaction.name = 'holocarboxylase synthestase (biotin protein ligase)'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd577: -1.0, Cpd537: -1.0, Cpd33: 1.0, Cpd150: 1.0, Cpd578: 1.0})
reaction.gene_reaction_rule = '( HLCS )'
model.add_reactions([reaction])
reaction = Reaction('BTNPLm')
reaction.name = 'holocarboxylase synthestase (biotin protein ligase), mitochondrial'
reaction.subsystem = 'Biotin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd579: -1.0, Cpd539: -1.0, Cpd7: 1.0, Cpd209: 1.0, Cpd580: 1.0})
reaction.gene_reaction_rule = '( HLCS )'
model.add_reactions([reaction])
reaction = Reaction('BTNt2')
reaction.name = 'Biotin reversible transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd574: -1.0, Cpd33: 1.0, Cpd536: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('BTNt2m')
reaction.name = 'Biotin reversible transport via proton symport, mitochondria'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd536: -1.0, Cpd7: 1.0, Cpd538: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('BTNt3i')
reaction.name = 'Biotin transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -1.0, Cpd574: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 1.0, Cpd536: 1.0})
reaction.gene_reaction_rule = '( SLC5A6 )'
model.add_reactions([reaction])
reaction = Reaction('BTNt4i')
reaction.name = 'Biotin uptake (antiport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd574: -1.0, Cpd37: 1.0, Cpd536: 1.0})
reaction.gene_reaction_rule = '( SLC19A3 )'
model.add_reactions([reaction])
reaction = Reaction('BUP2')
reaction.name = 'beta-ureidopropionase (D-3-amino-isobutanoate forming)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -2.0, Cpd581: -1.0, Cpd6: 1.0, Cpd60: 1.0, Cpd582: 1.0})
reaction.gene_reaction_rule = '( UPB1 )'
model.add_reactions([reaction])
reaction = Reaction('BUTt2m')
reaction.name = 'Butyrate mitochondrial transport via proton symport, reversible'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd583: -1.0, Cpd7: 1.0, Cpd584: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('BUTt2r')
reaction.name = 'Butyrate transport via proton symport, reversible'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd585: -1.0, Cpd33: 1.0, Cpd583: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('C14STRr')
reaction.name = 'C-14 sterol reductase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd586: -1.0, Cpd587: -1.0, Cpd588: 1.0, Cpd589: 1.0})
reaction.gene_reaction_rule = '( TM7SF2 )'
model.add_reactions([reaction])
reaction = Reaction('C160CPT1')
reaction.name = 'production of Palmitoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd590: -1.0, Cpd78: 1.0, Cpd591: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C160CPT2')
reaction.name = 'C160 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd592: -1.0, Cpd307: 1.0, Cpd593: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C160CRNt')
reaction.name = 'C160 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd591: -1.0, Cpd592: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C161CPT1')
reaction.name = 'production of palmitoleoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd594: -1.0, Cpd78: 1.0, Cpd595: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C161CPT12')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd596: -1.0, Cpd78: 1.0, Cpd597: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C161CPT2')
reaction.name = 'transport of Hexadecenoyl-CoA into mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd598: -1.0, Cpd307: 1.0, Cpd599: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C161CPT22')
reaction.name = 'C161 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd600: -1.0, Cpd307: 1.0, Cpd601: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C161CRN2t')
reaction.name = 'C161 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd597: -1.0, Cpd600: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C161CRNt')
reaction.name = 'C161 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd595: -1.0, Cpd598: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C180CPT1')
reaction.name = 'production of stearoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd602: -1.0, Cpd78: 1.0, Cpd603: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C180CPT2')
reaction.name = 'carnitine O-stearoyl transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd604: -1.0, Cpd307: 1.0, Cpd605: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C180CRNt')
reaction.name = 'C180 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd603: -1.0, Cpd604: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C181CPT1')
reaction.name = 'transport of Octadecenoyl-CoA into mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd606: -1.0, Cpd78: 1.0, Cpd607: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C181CPT2')
reaction.name = 'transport of Octadecenoyl-CoA into mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd608: -1.0, Cpd307: 1.0, Cpd609: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C181CRNt')
reaction.name = 'C181 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd607: -1.0, Cpd608: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C204CPT1')
reaction.name = 'carnitine C20:4 transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd610: -1.0, Cpd78: 1.0, Cpd611: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C204CPT2')
reaction.name = 'arachidonic acid transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd612: -1.0, Cpd307: 1.0, Cpd613: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C204CRNt')
reaction.name = 'arachidonic acid transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd611: -1.0, Cpd612: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C226CPT1')
reaction.name = 'carnitine C22:6 transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd614: -1.0, Cpd78: 1.0, Cpd615: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C226CPT2')
reaction.name = 'C226 transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd616: -1.0, Cpd307: 1.0, Cpd617: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C2M26DCOAHLm')
reaction.name = 'cis-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA hydro-lyase (m)'
reaction.subsystem = 'Limonene and pinene degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd618: -1.0, Cpd619: 1.0})
reaction.gene_reaction_rule = '( AUH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('C2M26DCOAHLx')
reaction.name = 'cis-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA hydro-lyase (x)'
reaction.subsystem = 'Limonene and pinene degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd620: -1.0, Cpd621: 1.0})
reaction.gene_reaction_rule = '( ECH1 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('C3STDH1Pr')
reaction.name = 'C-3 sterol dehydrogenase (4-methylzymosterol)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd622: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd623: 1.0, Cpd624: 1.0})
reaction.gene_reaction_rule = '( NSDHL )'
model.add_reactions([reaction])
reaction = Reaction('C3STDH1r')
reaction.name = 'C-3 sterol dehydrogenase (4-methylzymosterol)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd622: -1.0, Cpd625: -1.0, Cpd565: 1.0, Cpd623: 1.0, Cpd624: 1.0, Cpd626: 1.0})
reaction.gene_reaction_rule = '( NSDHL )'
model.add_reactions([reaction])
reaction = Reaction('C3STKR2r')
reaction.name = 'C-3 sterol keto reductase (zymosterol)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd627: -1.0, Cpd589: 1.0, Cpd628: 1.0})
reaction.gene_reaction_rule = '( HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('C4STMO1r')
reaction.name = 'C-4 sterol methyl oxidase (4,4-dimethylzymosterol)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -3.0, Cpd565: -3.0, Cpd587: -3.0, Cpd588: -1.0, Cpd437: 4.0, Cpd589: 3.0, Cpd622: 1.0})
reaction.gene_reaction_rule = '( MSMO1 )'
model.add_reactions([reaction])
reaction = Reaction('C4STMO2Pr')
reaction.name = 'C-4 methyl sterol oxidase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd589: -1.0, Cpd623: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd624: 1.0, Cpd627: 1.0})
reaction.gene_reaction_rule = '( NSDHL )'
model.add_reactions([reaction])
reaction = Reaction('C4STMO2r')
reaction.name = 'C-4 methyl sterol oxidase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd623: -1.0, Cpd625: -1.0, Cpd565: 1.0, Cpd624: 1.0, Cpd626: 1.0, Cpd627: 1.0})
reaction.gene_reaction_rule = '( MSMO1 or NSDHL )'
model.add_reactions([reaction])
reaction = Reaction('CAATPS')
reaction.name = 'Ca ATPase'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd629: -1.0, Cpd107: 1.0, Cpd108: 1.0, Cpd37: 1.0, Cpd630: 1.0})
reaction.gene_reaction_rule = '( ATP2B1 or ATP2B2 or ATP2B3 or ATP2B4 )'
model.add_reactions([reaction])
reaction = Reaction('CAMPt')
reaction.name = 'cAMP transport (ATP-dependent)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd290: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd631: 1.0})
reaction.gene_reaction_rule = '( ABCC4 or ABCC5 )'
model.add_reactions([reaction])
reaction = Reaction('CAT2p')
reaction.name = 'catalase A, peroxisomal (ethanol)'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd247: -1.0, Cpd632: -1.0, Cpd633: 1.0, Cpd261: 2.0})
reaction.gene_reaction_rule = '( CAT )'
model.add_reactions([reaction])
reaction = Reaction('CAt7r')
reaction.name = 'calcium / sodium antiporter (1:3), reversible'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -3.0, Cpd629: -1.0, Cpd41: 3.0, Cpd630: 1.0})
reaction.gene_reaction_rule = '( SLC8A1 or SLC8A2 or SLC8A3 )'
model.add_reactions([reaction])
reaction = Reaction('CATm')
reaction.name = 'catalase'
reaction.subsystem = 'ROS detoxification'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd634: -2.0, Cpd9: 1.0, Cpd12: 2.0})
reaction.gene_reaction_rule = '( CAT )'
model.add_reactions([reaction])
reaction = Reaction('CATp')
reaction.name = 'catalase A, peroxisomal'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd247: -2.0, Cpd246: 1.0, Cpd261: 2.0})
reaction.gene_reaction_rule = '( CAT )'
model.add_reactions([reaction])
reaction = Reaction('CBL2tm')
reaction.name = 'Cob(II)alamine transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd635: -1.0, Cpd636: 1.0})
reaction.gene_reaction_rule = '( MMAA )'
model.add_reactions([reaction])
reaction = Reaction('CBLATm')
reaction.name = 'cob(I)alamin adenosyltransferase, mitochondrial'
reaction.subsystem = 'Vitamin B12 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd200: -1.0, Cpd637: -1.0, Cpd638: 1.0, Cpd639: 1.0})
reaction.gene_reaction_rule = '( MMAA )'
model.add_reactions([reaction])
reaction = Reaction('CBPPer')
reaction.name = 'Carbamoyl phosphate phosphotransferase, endoplasmic reticulum'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd640: -1.0, Cpd641: -1.0, Cpd624: 1.0, Cpd642: 1.0, Cpd643: 1.0})
reaction.gene_reaction_rule = '( G6PC or G6PC2 or G6PC3 )'
model.add_reactions([reaction])
reaction = Reaction('CBPS')
reaction.name = 'carbamoyl-phosphate synthase (glutamine-hydrolysing)'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -2.0, Cpd40: -1.0, Cpd399: -1.0, Cpd107: 2.0, Cpd33: 2.0, Cpd108: 1.0, Cpd74: 1.0, Cpd484: 1.0})
reaction.gene_reaction_rule = '( CAD )'
model.add_reactions([reaction])
reaction = Reaction('CBPSam')
reaction.name = 'carbamoyl-phosphate synthase (ammonia) (mitochondria'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -2.0, Cpd201: -1.0, Cpd482: -1.0, Cpd7: 2.0, Cpd202: 2.0, Cpd204: 1.0, Cpd644: 1.0})
reaction.gene_reaction_rule = '( CPS1 )'
model.add_reactions([reaction])
reaction = Reaction('CBR1')
reaction.name = 'Carbonyl reductase [NADPH] 1'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd645: -1.0, Cpd25: 1.0, Cpd646: 1.0})
reaction.gene_reaction_rule = '( CBR1 )'
model.add_reactions([reaction])
reaction = Reaction('CBR2')
reaction.name = 'Carbonyl reductase [NADPH] 1'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd647: -1.0, Cpd25: 1.0, Cpd648: 1.0})
reaction.gene_reaction_rule = '( CBR1 )'
model.add_reactions([reaction])
reaction = Reaction('CDIPTr')
reaction.name = 'phosphatidylinositol synthase (Homo sapiens)'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd649: -1.0, Cpd650: -1.0, Cpd33: 1.0, Cpd651: 1.0, Cpd652: 1.0})
reaction.gene_reaction_rule = '( CDIPT )'
model.add_reactions([reaction])
reaction = Reaction('CDS')
reaction.name = 'phosphatidate cytidylyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd353: -1.0, Cpd653: -1.0, Cpd151: 1.0, Cpd649: 1.0})
reaction.gene_reaction_rule = '( CDS1 or CDS2 )'
model.add_reactions([reaction])
reaction = Reaction('CDSm')
reaction.name = 'phosphatidate cytidylyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd654: -1.0, Cpd655: -1.0, Cpd210: 1.0, Cpd656: 1.0})
reaction.gene_reaction_rule = '( CDS1 or CDS2 )'
model.add_reactions([reaction])
reaction = Reaction('CEPTC')
reaction.name = 'choline phosphotransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd657: -1.0, Cpd658: -1.0, Cpd33: 1.0, Cpd651: 1.0, Cpd659: 1.0})
reaction.gene_reaction_rule = '( CEPT1 or CHPT1 )'
model.add_reactions([reaction])
reaction = Reaction('CEPTE')
reaction.name = 'ethanolamine phosphotransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd660: -1.0, Cpd658: -1.0, Cpd33: 1.0, Cpd651: 1.0, Cpd661: 1.0})
reaction.gene_reaction_rule = '( CEPT1 )'
model.add_reactions([reaction])
reaction = Reaction('CERK')
reaction.name = 'Ceramide kinase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd662: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd663: 1.0})
reaction.gene_reaction_rule = '( CERK )'
model.add_reactions([reaction])
reaction = Reaction('CERT1gt')
reaction.name = 'ceramide transport protein'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd662: -1.0, Cpd664: 1.0})
reaction.gene_reaction_rule = '( COL4A3BP )'
model.add_reactions([reaction])
reaction = Reaction('CERT1rt')
reaction.name = 'ceramide transport protein'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd662: -1.0, Cpd665: 1.0})
reaction.gene_reaction_rule = '( COL4A3BP )'
model.add_reactions([reaction])
reaction = Reaction('CERT2gt')
reaction.name = 'ceramide transport protein'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd666: -1.0, Cpd528: 1.0})
reaction.gene_reaction_rule = '( COL4A3BP )'
model.add_reactions([reaction])
reaction = Reaction('CERT2rt')
reaction.name = 'ceramide transport protein'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd666: -1.0, Cpd667: 1.0})
reaction.gene_reaction_rule = '( COL4A3BP )'
model.add_reactions([reaction])
reaction = Reaction('CGLYt3_2_')
reaction.name = 'Cys-Gly transport in via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -2.0, Cpd444: -1.0, Cpd33: 2.0, Cpd443: 1.0})
reaction.gene_reaction_rule = '( SLC15A2 )'
model.add_reactions([reaction])
reaction = Reaction('CGMPt')
reaction.name = 'cGMP transport (ATP-dependent)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd668: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd669: 1.0})
reaction.gene_reaction_rule = '( ABCC4 or ABCC5 )'
model.add_reactions([reaction])
reaction = Reaction('CH25H')
reaction.name = 'cholesterol 25-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd670: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd671: 1.0})
reaction.gene_reaction_rule = '( CH25H )'
model.add_reactions([reaction])
reaction = Reaction('CHAT')
reaction.name = 'Choline O-acetyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd672: -1.0, Cpd78: 1.0, Cpd227: 1.0})
reaction.gene_reaction_rule = '( CHAT )'
model.add_reactions([reaction])
reaction = Reaction('CHATn')
reaction.name = 'Choline O-acetyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd673: -1.0, Cpd674: -1.0, Cpd675: 1.0, Cpd676: 1.0})
reaction.gene_reaction_rule = '( CHAT )'
model.add_reactions([reaction])
reaction = Reaction('CHLP')
reaction.name = 'choline phosphate phosphatase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd677: -1.0, Cpd108: 1.0, Cpd672: 1.0})
reaction.gene_reaction_rule = '( PHOSPHO1 )'
model.add_reactions([reaction])
reaction = Reaction('CHLPCTD')
reaction.name = 'choline phosphate cytididyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd653: -1.0, Cpd677: -1.0, Cpd151: 1.0, Cpd657: 1.0})
reaction.gene_reaction_rule = '( PCYT1A or PCYT1B )'
model.add_reactions([reaction])
reaction = Reaction('CHOLATEt')
reaction.name = 'cholate transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd678: -1.0, Cpd38: 1.0, Cpd679: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('CHOLATEt2')
reaction.name = 'cholate transport via sodium cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd678: -1.0, Cpd41: 2.0, Cpd679: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 or SLC10A2 )'
model.add_reactions([reaction])
reaction = Reaction('CHOLATEt3')
reaction.name = 'ABC bile acid transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd679: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd678: 1.0})
reaction.gene_reaction_rule = '( ABCB11 or ABCC3 )'
model.add_reactions([reaction])
reaction = Reaction('CHOLD2m')
reaction.name = 'choline dehydrogenase (FAD acceptor), mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd680: -1.0, Cpd240: 1.0, Cpd681: 1.0})
reaction.gene_reaction_rule = '( CHDH )'
model.add_reactions([reaction])
reaction = Reaction('CHOLK')
reaction.name = 'Choline kinase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd672: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd677: 1.0})
reaction.gene_reaction_rule = '( CHKA or CHKB )'
model.add_reactions([reaction])
reaction = Reaction('CHOLt4')
reaction.name = 'choline, sodium cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd226: -1.0, Cpd41: 1.0, Cpd672: 1.0})
reaction.gene_reaction_rule = '( SLC5A7 )'
model.add_reactions([reaction])
reaction = Reaction('CHOLtu')
reaction.name = 'Choline uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd226: -1.0, Cpd672: 1.0})
reaction.gene_reaction_rule = '( SLC22A2 or SLC22A5 )'
model.add_reactions([reaction])
reaction = Reaction('CHSTEROLSULT')
reaction.name = 'steroid sulfotransferase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd682: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd683: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 or SULT2B1 )'
model.add_reactions([reaction])
reaction = Reaction('CHSTEROLt')
reaction.name = 'cholesterol efflux (ATP depedent)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd682: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd684: 1.0})
reaction.gene_reaction_rule = '( ABCA1 or ABCG5 or ABCG8 )'
model.add_reactions([reaction])
reaction = Reaction('CHSTEROLt1')
reaction.name = 'cholesterol intracellular transport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd685: -1.0, Cpd686: 1.0})
reaction.gene_reaction_rule = '( STARD3 )'
model.add_reactions([reaction])
reaction = Reaction('CHSTEROLtg')
reaction.name = 'cholesterol efflux (ATP depedent), golgi'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd687: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd682: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('CHTNASE')
reaction.name = 'chitinase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2.0, Cpd688: -1.0, Cpd215: 3.0})
reaction.gene_reaction_rule = '( CHIL5 or CHIT1 or MANSC1 )'
model.add_reactions([reaction])
reaction = Reaction('CHTNASEe')
reaction.name = 'chitinase, extracellular'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -2.0, Cpd689: -1.0, Cpd690: 3.0})
reaction.gene_reaction_rule = '( CHIL5 )'
model.add_reactions([reaction])
reaction = Reaction('CITL')
reaction.name = 'Citrate lyase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd228: -1.0, Cpd221: 1.0, Cpd229: 1.0})
reaction.gene_reaction_rule = '( CLYBL )'
model.add_reactions([reaction])
reaction = Reaction('CITRtm')
reaction.name = 'citrulline mitochondrial transport via proton antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd691: -1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('CITt4_2')
reaction.name = 'citrate transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd692: -1.0, Cpd41: 2.0, Cpd228: 1.0})
reaction.gene_reaction_rule = '( SLC13A2 )'
model.add_reactions([reaction])
reaction = Reaction('CITtam')
reaction.name = 'citrate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd228: -1.0, Cpd382: -1.0, Cpd254: 1.0, Cpd381: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('CITtbm')
reaction.name = 'citrate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd228: -1.0, Cpd693: -1.0, Cpd231: 1.0, Cpd254: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('CK')
reaction.name = 'ATP Creatine kinase'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd694: -1.0, Cpd202: 1.0, Cpd695: 1.0})
reaction.gene_reaction_rule = '( CKMT1 or CKMT2 )'
model.add_reactions([reaction])
reaction = Reaction('CKc')
reaction.name = 'ATP Creatine kinase (c)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd696: -1.0, Cpd107: 1.0, Cpd697: 1.0})
reaction.gene_reaction_rule = '( CKB or CKM )'
model.add_reactions([reaction])
reaction = Reaction('CLFORtex')
reaction.name = 'chloride transport via formate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd698: -1.0, Cpd699: -2.0, Cpd700: 1.0, Cpd701: 2.0})
reaction.gene_reaction_rule = '( SLC26A4 )'
model.add_reactions([reaction])
reaction = Reaction('CLHCO3tex2')
reaction.name = 'chloride transport via bicarbonate countertransport (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd698: -2.0, Cpd38: 1.0, Cpd700: 2.0})
reaction.gene_reaction_rule = '( SLC26A6 )'
model.add_reactions([reaction])
reaction = Reaction('CLI2tex')
reaction.name = 'chloride transport via iodide countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd698: -1.0, Cpd702: -2.0, Cpd700: 1.0, Cpd703: 2.0})
reaction.gene_reaction_rule = '( SLC26A4 )'
model.add_reactions([reaction])
reaction = Reaction('CLOHtex2')
reaction.name = 'chloride transport via hydroxide countertransport (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd698: -2.0, Cpd114: 1.0, Cpd700: 2.0})
reaction.gene_reaction_rule = '( SLC26A6 )'
model.add_reactions([reaction])
reaction = Reaction('CLOXAtex2')
reaction.name = 'chloride transport via oxalate countertransport (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd698: -2.0, Cpd704: -1.0, Cpd700: 2.0, Cpd705: 1.0})
reaction.gene_reaction_rule = '( SLC26A6 )'
model.add_reactions([reaction])
reaction = Reaction('CLPNDCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd706: -1.0, Cpd78: 1.0, Cpd707: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('CLPNDCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd708: -1.0, Cpd307: 1.0, Cpd709: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('CLPNDCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd707: -1.0, Cpd708: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('CLS_hs')
reaction.name = 'cardiolipin synthase (homo sapiens)'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd649: -1.0, Cpd710: -1.0, Cpd33: 1.0, Cpd651: 1.0, Cpd711: 1.0})
reaction.gene_reaction_rule = '( CRLS1 )'
model.add_reactions([reaction])
reaction = Reaction('CMPACNAtg')
reaction.name = 'CMP-Sia Golgi transport via CMP antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd712: -1.0, Cpd713: -1.0, Cpd651: 1.0, Cpd714: 1.0})
reaction.gene_reaction_rule = '( SLC35A1 )'
model.add_reactions([reaction])
reaction = Reaction('CMPSAS')
reaction.name = 'CMP sialic acid synthase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd235: -1.0, Cpd653: -1.0, Cpd151: 1.0, Cpd713: 1.0})
reaction.gene_reaction_rule = '( CMAS )'
model.add_reactions([reaction])
reaction = Reaction('CMPSASn')
reaction.name = 'CMP sialic acid synthase, nuclear'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd715: -1.0, Cpd716: -1.0, Cpd717: 1.0, Cpd718: 1.0})
reaction.gene_reaction_rule = '( CMAS )'
model.add_reactions([reaction])
reaction = Reaction('COAtm')
reaction.name = 'CoA transporter'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd78: -1.0, Cpd43: 1.0})
reaction.gene_reaction_rule = '( SLC25A16 )'
model.add_reactions([reaction])
reaction = Reaction('COKECBESr')
reaction.name = 'Carboxylesterase (cocaine) (er)'
reaction.subsystem = 'Alkaloid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd719: -1.0, Cpd565: 1.0, Cpd720: 1.0, Cpd721: 1.0})
reaction.gene_reaction_rule = '( CES2A )'
model.add_reactions([reaction])
reaction = Reaction('COQ3m')
reaction.name = 'methyltransferase COQ3'
reaction.subsystem = 'Ubiquinone synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd442: -1.0, Cpd722: -1.0, Cpd7: 1.0, Cpd441: 1.0, Cpd723: 1.0})
reaction.gene_reaction_rule = '( COQ3 )'
model.add_reactions([reaction])
reaction = Reaction('COQ6m')
reaction.name = 'Ubiquinone biosynthesis monooxgenase COQ6'
reaction.subsystem = 'Ubiquinone synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd724: -1.0, Cpd12: 1.0, Cpd725: 1.0})
reaction.gene_reaction_rule = '( COQ6 )'
model.add_reactions([reaction])
reaction = Reaction('COQ7m')
reaction.name = 'Ubiquinone biosynthesis COQ7'
reaction.subsystem = 'Ubiquinone synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd726: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd722: 1.0})
reaction.gene_reaction_rule = '( COQ7 )'
model.add_reactions([reaction])
reaction = Reaction('CORE2GTg')
reaction.name = 'Core 2 acetylglucosaminyltransferase, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd143: -1.0, Cpd144: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd146: 1.0})
reaction.gene_reaction_rule = '( GCNT1 or GCNT3 or GCNT4 )'
model.add_reactions([reaction])
reaction = Reaction('CORE3GTg')
reaction.name = 'Core 3 beta-GlcNAc-transferase, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd727: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd728: 1.0})
reaction.gene_reaction_rule = '( B3GNT6 )'
model.add_reactions([reaction])
reaction = Reaction('CORE4GTg')
reaction.name = 'Core 4 beta6-GalNAc-transferase, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd728: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd729: 1.0})
reaction.gene_reaction_rule = '( GCNT3 )'
model.add_reactions([reaction])
reaction = Reaction('CORE6GTg')
reaction.name = 'Core 6 beta-GlcNAc-transferase A, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd727: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd730: 1.0})
reaction.gene_reaction_rule = '( GCNT1 )'
model.add_reactions([reaction])
reaction = Reaction('CPCTDTX')
reaction.name = 'choline-phosphate cytidylyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd653: -1.0, Cpd731: -1.0, Cpd151: 1.0, Cpd732: 1.0})
reaction.gene_reaction_rule = '( PCYT1A or PCYT1B )'
model.add_reactions([reaction])
reaction = Reaction('CPPPGO')
reaction.name = 'coproporphyrinogen oxidase (O2 required)'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -2.0, Cpd733: -1.0, Cpd2: 2.0, Cpd60: 2.0, Cpd734: 1.0})
reaction.gene_reaction_rule = '( CPOX )'
model.add_reactions([reaction])
reaction = Reaction('CREATt4_2_r')
reaction.name = 'Creatine transport (sodium symport) (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd735: -1.0, Cpd41: 2.0, Cpd696: 1.0})
reaction.gene_reaction_rule = '( SLC6A8 )'
model.add_reactions([reaction])
reaction = Reaction('CRNCARtp')
reaction.name = 'carnitine-acetylcarnitine carrier, peroxisomal'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd736: -1.0, Cpd265: 1.0, Cpd737: 1.0})
reaction.gene_reaction_rule = '( CRAT )'
model.add_reactions([reaction])
reaction = Reaction('CRNt')
reaction.name = 'L-carnitine reversible transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd738: -1.0, Cpd303: 1.0})
reaction.gene_reaction_rule = '( SLC22A5 )'
model.add_reactions([reaction])
reaction = Reaction('CRNtHa')
reaction.name = 'L-carnitine outward transport (H+ antiport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd303: -1.0, Cpd33: 1.0, Cpd738: 1.0})
reaction.gene_reaction_rule = '( SLC22A4 )'
model.add_reactions([reaction])
reaction = Reaction('CRNtim')
reaction.name = 'L-carnitine transport out of mitochondria via diffusion'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd307: -1.0, Cpd303: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('CRNtuNa')
reaction.name = 'L-carnitine inward transport by Na+ symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd738: -1.0, Cpd41: 1.0, Cpd303: 1.0})
reaction.gene_reaction_rule = '( SLC22A5 )'
model.add_reactions([reaction])
reaction = Reaction('CSm')
reaction.name = 'citrate synthase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd178: -1.0, Cpd491: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd254: 1.0})
reaction.gene_reaction_rule = '( CSL )'
model.add_reactions([reaction])
reaction = Reaction('CSNAT2m')
reaction.name = 'carnitine O-aceyltransferase, mitochondrial'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd739: -1.0, Cpd179: 1.0, Cpd307: 1.0})
reaction.gene_reaction_rule = '( CRAT )'
model.add_reactions([reaction])
reaction = Reaction('CSNAT2x')
reaction.name = 'carnitine dimethyl nonanoyl transferase, revsible, peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd740: -1.0, Cpd184: 1.0, Cpd741: 1.0})
reaction.gene_reaction_rule = '( CRAT or CROT )'
model.add_reactions([reaction])
reaction = Reaction('CSNAT3x')
reaction.name = 'carnitine O-aceyltransferase, peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd742: -1.0, Cpd184: 1.0, Cpd743: 1.0})
reaction.gene_reaction_rule = '( CRAT )'
model.add_reactions([reaction])
reaction = Reaction('CSNATer')
reaction.name = 'carnitine O-acetyltransferase, endolasmic reticulum'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd211: -1.0, Cpd744: -1.0, Cpd745: 1.0, Cpd746: 1.0})
reaction.gene_reaction_rule = '( CRAT )'
model.add_reactions([reaction])
reaction = Reaction('CSNATm')
reaction.name = 'carnitine O-aceyltransferase, mitochondrial'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd266: -1.0, Cpd178: 1.0, Cpd307: 1.0})
reaction.gene_reaction_rule = '( CRAT )'
model.add_reactions([reaction])
reaction = Reaction('CSNATp')
reaction.name = 'carnitine O-acetyltransferase, reverse direction, peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd182: -1.0, Cpd737: -1.0, Cpd184: 1.0, Cpd736: 1.0})
reaction.gene_reaction_rule = '( CRAT )'
model.add_reactions([reaction])
reaction = Reaction('CSNATr')
reaction.name = 'carnitine O-acetyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd303: -1.0, Cpd78: 1.0, Cpd265: 1.0})
reaction.gene_reaction_rule = '( CRAT )'
model.add_reactions([reaction])
reaction = Reaction('CSNt')
reaction.name = 'cytosine transport via facilated diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd747: -1.0, Cpd748: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('CTPS1')
reaction.name = 'CTP synthase (NH3)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd6: -1.0, Cpd105: -1.0, Cpd749: -1.0, Cpd107: 1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd653: 1.0})
reaction.gene_reaction_rule = '( CTPS or CTPS2 )'
model.add_reactions([reaction])
reaction = Reaction('CTPS2')
reaction.name = 'CTP synthase (glutamine)'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd399: -1.0, Cpd749: -1.0, Cpd107: 1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd74: 1.0, Cpd653: 1.0})
reaction.gene_reaction_rule = '( CTPS or CTPS2 )'
model.add_reactions([reaction])
reaction = Reaction('CYOOm2')
reaction.name = 'cytochrome c oxidase, mitochondrial Complex IV'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd750: -4.0, Cpd7: 4.0, Cpd12: 2.0, Cpd751: 4.0})
reaction.gene_reaction_rule = '( COX4I1 or COX5A or COX5B or COX6A1 or COX6A2 or COX6B1 or COX6C or COX7A2 or COX7B2 or COX8A or COX8C or CYC1 or MT-CYTB or UQCR10 or UQCR11 or UQCRB or UQCRC1 or UQCRC2 or UQCRFS1 or UQCRH or UQCRQ )'
model.add_reactions([reaction])
reaction = Reaction('CYOR_u10m')
reaction.name = 'ubiquinol-6 cytochrome c reductase, Complex III'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -2.0, Cpd751: -2.0, Cpd752: -1.0, Cpd33: 4.0, Cpd723: 1.0, Cpd750: 2.0})
reaction.gene_reaction_rule = '( CYC1 or MT-CYTB or UQCR10 or UQCR11 or UQCRB or UQCRC1 or UQCRC2 or UQCRFS1 or UQCRH or UQCRQ )'
model.add_reactions([reaction])
reaction = Reaction('CYSALANaEx')
reaction.name = 'L-alanine/L-cysteine Na-dependent exchange (Cys-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd397: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('CYSASNNaEx')
reaction.name = 'L-cysteine/L-asparagine Na-dependent exchange (Cys-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd397: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('CYSGLNNaEx')
reaction.name = 'L-cysteine/L-glutamine Na-dependent exchange (Cys-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd399: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd401: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('CYSGLUexR')
reaction.name = 'L-cysteine/L-glutamine reversible exchanger'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd399: -1.0, Cpd396: 1.0, Cpd401: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 or SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('CYSGLYexR')
reaction.name = 'L-cysteine/glycine reversible exchanger'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd403: -1.0, Cpd397: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('CYSO')
reaction.name = 'cysteine oxidase'
reaction.subsystem = 'Taurine and hypotaurine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd396: -1.0, Cpd33: 2.0, Cpd91: 1.0})
reaction.gene_reaction_rule = '( CDO1 )'
model.add_reactions([reaction])
reaction = Reaction('CYSSERNaEx')
reaction.name = 'L-serine/L-cysteine Na-dependent exchange (Cys-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd404: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd405: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('CYSt4')
reaction.name = 'L-cysteine reversible transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd41: 1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A2 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('CYSTA')
reaction.name = 'cysteine transaminase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd396: -1.0, Cpd74: 1.0, Cpd753: 1.0})
reaction.gene_reaction_rule = '( GOT1 )'
model.add_reactions([reaction])
reaction = Reaction('CYSTAm')
reaction.name = 'cysteine transaminase (mitochondrial)'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd754: -1.0, Cpd63: 1.0, Cpd755: 1.0})
reaction.gene_reaction_rule = '( GOT2 )'
model.add_reactions([reaction])
reaction = Reaction('CYStec')
reaction.name = 'L-cysteine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( SLC43A2 )'
model.add_reactions([reaction])
reaction = Reaction('CYSTGL')
reaction.name = 'cystathionine g-lyase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd756: -1.0, Cpd6: 1.0, Cpd34: 1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( CTH )'
model.add_reactions([reaction])
reaction = Reaction('CYSTGLUex')
reaction.name = 'L-cystine/L-glutamate exchanger'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd74: -1.0, Cpd757: -1.0, Cpd758: 1.0, Cpd759: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A11 )'
model.add_reactions([reaction])
reaction = Reaction('CYSTHRNaEx')
reaction.name = 'L-cysteine/L-threonine Na-dependent exchange (Cys-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd408: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('CYSTS')
reaction.name = 'cystathionine beta-synthase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd371: -1.0, Cpd404: -1.0, Cpd2: 1.0, Cpd756: 1.0})
reaction.gene_reaction_rule = '( CBS )'
model.add_reactions([reaction])
reaction = Reaction('CYSTSERex')
reaction.name = 'L-cystine/glycine exchanger (cystine in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd757: -1.0, Cpd405: 1.0, Cpd758: 1.0})
reaction.gene_reaction_rule = '( SLC3A1 or SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('CYTD')
reaction.name = 'cytidine deaminase'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd760: -1.0, Cpd6: 1.0, Cpd761: 1.0})
reaction.gene_reaction_rule = '( AICDA or CDA )'
model.add_reactions([reaction])
reaction = Reaction('CYTDK1')
reaction.name = 'ATPcytidine 5-phosphotransferase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd760: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd651: 1.0})
reaction.gene_reaction_rule = '( UCK1 or UCK2 )'
model.add_reactions([reaction])
reaction = Reaction('CYTDn')
reaction.name = 'cytidine deaminase, nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd762: -1.0, Cpd763: -1.0, Cpd764: 1.0, Cpd765: 1.0})
reaction.gene_reaction_rule = '( AICDA )'
model.add_reactions([reaction])
reaction = Reaction('CYTDt')
reaction.name = 'cytidine facilated transport in cytosol'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd766: -1.0, Cpd760: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 or SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('CYTDt4')
reaction.name = 'cytidine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd766: -1.0, Cpd41: 1.0, Cpd760: 1.0})
reaction.gene_reaction_rule = '( SLC28A1 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('CYTDtl')
reaction.name = 'Cytidine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd767: -1.0, Cpd760: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('CYTDtm')
reaction.name = 'cytidine facilated transport in mitochondria'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd760: -1.0, Cpd768: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK1')
reaction.name = 'cytidylate kinase (CMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd651: -1.0, Cpd107: 1.0, Cpd769: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK10')
reaction.name = 'cytidylate kinase (CMP,dGTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd651: -1.0, Cpd770: -1.0, Cpd769: 1.0, Cpd771: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK10n')
reaction.name = 'cytidylate kinase (CMP,dGTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd772: -1.0, Cpd773: -1.0, Cpd774: 1.0, Cpd775: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK11')
reaction.name = 'cytidylate kinase (dCMP,dGTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd770: -1.0, Cpd776: -1.0, Cpd771: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK11n')
reaction.name = 'cytidylate kinase (dCMP,dGTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd773: -1.0, Cpd778: -1.0, Cpd775: 1.0, Cpd779: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK12')
reaction.name = 'cytidylate kinase (dCMP,dCTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd776: -1.0, Cpd780: -1.0, Cpd777: 2.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK12n')
reaction.name = 'cytidylate kinase (dCMP,dCTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd778: -1.0, Cpd781: -1.0, Cpd779: 2.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK13')
reaction.name = 'cytidylate kinase (dCMP,dATP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd776: -1.0, Cpd288: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK13n')
reaction.name = 'cytidylate kinase (dCMP,dATP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd778: -1.0, Cpd782: -1.0, Cpd779: 1.0, Cpd783: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK14')
reaction.name = 'cytidylate kinase (dCMP,UTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd749: -1.0, Cpd776: -1.0, Cpd137: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK14n')
reaction.name = 'cytidylate kinase (dCMP,CTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd778: -1.0, Cpd784: -1.0, Cpd779: 1.0, Cpd785: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK1m')
reaction.name = 'cytidylate kinase (CMP),mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd786: -1.0, Cpd202: 1.0, Cpd787: 1.0})
reaction.gene_reaction_rule = '( CMPK2 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK1n')
reaction.name = 'cytidylate kinase (CMP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd772: -1.0, Cpd774: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK2')
reaction.name = 'cytidylate kinase (dCMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd776: -1.0, Cpd107: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK2n')
reaction.name = 'cytidylate kinase (dCMP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd778: -1.0, Cpd779: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK3')
reaction.name = 'cytidylate kinase (CMP)(GTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd282: -1.0, Cpd651: -1.0, Cpd283: 1.0, Cpd769: 1.0})
reaction.gene_reaction_rule = '( AK5 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK3n')
reaction.name = 'cytidylate kinase (dCMP,CTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd716: -1.0, Cpd778: -1.0, Cpd774: 1.0, Cpd779: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK4')
reaction.name = 'cytidylate kinase (dCMP)(GTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd282: -1.0, Cpd776: -1.0, Cpd283: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( AK5 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK4n')
reaction.name = 'cytidylate kinase (dCMP,GTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd778: -1.0, Cpd790: -1.0, Cpd779: 1.0, Cpd791: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK5')
reaction.name = 'cytidylate kinase (dCMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd653: -1.0, Cpd776: -1.0, Cpd769: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK5n')
reaction.name = 'cytidylate kinase (CMP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd772: -1.0, Cpd790: -1.0, Cpd774: 1.0, Cpd791: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK6')
reaction.name = 'cytidylate kinase (CMP,CTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd651: -1.0, Cpd653: -1.0, Cpd769: 2.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK6n')
reaction.name = 'cytidylate kinase (CMP,CTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd716: -1.0, Cpd772: -1.0, Cpd774: 2.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK7')
reaction.name = 'cytidylate kinase (CMP,UTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd651: -1.0, Cpd749: -1.0, Cpd137: 1.0, Cpd769: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK7n')
reaction.name = 'cytidylate kinase (CMP,UTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd772: -1.0, Cpd784: -1.0, Cpd774: 1.0, Cpd785: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK8')
reaction.name = 'cytidylate kinase (CMP,dATP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd651: -1.0, Cpd288: 1.0, Cpd769: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK8n')
reaction.name = 'cytidylate kinase (CMP,dATP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd772: -1.0, Cpd782: -1.0, Cpd774: 1.0, Cpd783: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK9')
reaction.name = 'cytidylate kinase (CMP,dCTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd651: -1.0, Cpd780: -1.0, Cpd769: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('CYTK9n')
reaction.name = 'cytidylate kinase (CMP,dCTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd772: -1.0, Cpd781: -1.0, Cpd774: 1.0, Cpd779: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('D3AIBTm')
reaction.name = 'D-3-Amino-isobutanoate:pyruvate aminotransferase, mitochondrial'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd363: -1.0, Cpd792: -1.0, Cpd62: 1.0, Cpd361: 1.0})
reaction.gene_reaction_rule = '( AGXT2 )'
model.add_reactions([reaction])
reaction = Reaction('DADA')
reaction.name = 'Deoxyadenosine deaminase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd793: -1.0, Cpd6: 1.0, Cpd794: 1.0})
reaction.gene_reaction_rule = '( ADA )'
model.add_reactions([reaction])
reaction = Reaction('DADAe')
reaction.name = 'Deoxyadenosine deaminase, extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd114: -1.0, Cpd795: -1.0, Cpd278: 1.0, Cpd796: 1.0})
reaction.gene_reaction_rule = '( ADA )'
model.add_reactions([reaction])
reaction = Reaction('DADNt4')
reaction.name = 'deoxyadenosine transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd795: -1.0, Cpd793: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('DAGK_hs')
reaction.name = 'Diacylglycerol phosphate kinase (homo sapiens)'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd658: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd353: 1.0})
reaction.gene_reaction_rule = '( DGKA or DGKB or DGKD or DGKE or DGKG or DGKH or DGKI or DGKQ or DGKZ )'
model.add_reactions([reaction])
reaction = Reaction('DAGKn_hs')
reaction.name = 'Diacylglycerol phosphate kinase (homo sapiens)'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd797: -1.0, Cpd763: 1.0, Cpd789: 1.0, Cpd798: 1.0})
reaction.gene_reaction_rule = '( DGKI or DGKZ )'
model.add_reactions([reaction])
reaction = Reaction('DALAOXx')
reaction.name = 'D-Alanine Oxidase (x)'
reaction.subsystem = 'D-alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd799: -1.0, Cpd247: 1.0, Cpd368: 1.0, Cpd800: 1.0})
reaction.gene_reaction_rule = '( DAO )'
model.add_reactions([reaction])
reaction = Reaction('DALAt2r')
reaction.name = 'D-alanine transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd398: -1.0, Cpd33: 1.0, Cpd400: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('DALAt2rL')
reaction.name = 'D-alanine transport via proton symport (lysosome)'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd801: -1.0, Cpd33: 1.0, Cpd400: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('DARGOp')
reaction.name = 'D-arginine oxidase, perixosomal'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd802: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd803: 1.0})
reaction.gene_reaction_rule = '( DAO )'
model.add_reactions([reaction])
reaction = Reaction('DASCBR')
reaction.name = 'dehydroascorbate reductase'
reaction.subsystem = 'Vitamin C metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd22: -1.0, Cpd804: -1.0, Cpd25: 1.0, Cpd480: 1.0})
reaction.gene_reaction_rule = '( GLRX or GLRX2 )'
model.add_reactions([reaction])
reaction = Reaction('DASPO1p')
reaction.name = 'D-aspartate oxidase, peroxisomal'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd805: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd806: 1.0})
reaction.gene_reaction_rule = '( DDO )'
model.add_reactions([reaction])
reaction = Reaction('DCIm')
reaction.name = 'dodecenoyl-Coenzyme A delta isomerase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd807: -1.0, Cpd808: 1.0})
reaction.gene_reaction_rule = '( ECI1 or ECI3 )'
model.add_reactions([reaction])
reaction = Reaction('DCK1n')
reaction.name = 'Deoxycytidine kinase, nuclear (ATP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd809: -1.0, Cpd763: 1.0, Cpd778: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( DCK )'
model.add_reactions([reaction])
reaction = Reaction('DCK2n')
reaction.name = 'Deoxycytidine kinase, nuclear (UTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd784: -1.0, Cpd809: -1.0, Cpd763: 1.0, Cpd778: 1.0, Cpd785: 1.0})
reaction.gene_reaction_rule = '( DCK )'
model.add_reactions([reaction])
reaction = Reaction('DCMPDA')
reaction.name = 'dCMP deaminase'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd776: -1.0, Cpd6: 1.0, Cpd810: 1.0})
reaction.gene_reaction_rule = '( DCTD )'
model.add_reactions([reaction])
reaction = Reaction('DCSPTN1CPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd811: -1.0, Cpd78: 1.0, Cpd812: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('DCSPTN1CPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd813: -1.0, Cpd307: 1.0, Cpd814: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('DCSPTN1CRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd812: -1.0, Cpd813: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('DCT')
reaction.name = 'Dopachrome tautomerase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd815: -1.0, Cpd816: 1.0})
reaction.gene_reaction_rule = '( DCT )'
model.add_reactions([reaction])
reaction = Reaction('DCYTD')
reaction.name = 'deoxycytidine deaminase'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd817: -1.0, Cpd6: 1.0, Cpd818: 1.0})
reaction.gene_reaction_rule = '( AICDA or CDA )'
model.add_reactions([reaction])
reaction = Reaction('DCYTDn')
reaction.name = 'deoxycytidine deaminase, nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd763: -1.0, Cpd809: -1.0, Cpd764: 1.0, Cpd819: 1.0})
reaction.gene_reaction_rule = '( AICDA )'
model.add_reactions([reaction])
reaction = Reaction('DCYTt')
reaction.name = 'deoxycytidine transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd820: -1.0, Cpd817: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('DEBRISOQUINEt')
reaction.name = 'debrisoquine uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd821: -1.0, Cpd822: 1.0})
reaction.gene_reaction_rule = '( SLC22A2 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT16_2')
reaction.name = 'palmitoyl-CoA desaturase (n-C16:0CoA -> n-C16:1CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd590: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd594: 1.0})
reaction.gene_reaction_rule = '( SCD4 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_10')
reaction.name = 'fatty acyl-CoA desaturase (n-C18:2CoA -> n-C18:3CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd823: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd824: 1.0})
reaction.gene_reaction_rule = '( FADS2 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_3')
reaction.name = 'stearoyl-CoA desaturase (n-C18:0CoA -> n-C18:1CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd602: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd606: 1.0})
reaction.gene_reaction_rule = '( SCD4 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_4')
reaction.name = 'stearoyl-CoA desaturase (n-C18:0CoA -> n-C18:1CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd602: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd825: 1.0})
reaction.gene_reaction_rule = '( SCD4 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_5')
reaction.name = 'stearoyl-CoA desaturase (n-C18:0CoA -> n-C18:1CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd602: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd826: 1.0})
reaction.gene_reaction_rule = '( SCD4 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_6')
reaction.name = 'fatty acyl-CoA desaturase (n-C18:1CoA -> n-C18:2CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd606: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd827: 1.0})
reaction.gene_reaction_rule = '( FADS2 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_7')
reaction.name = 'fatty acyl-CoA desaturase (n-C18:1CoA -> n-C18:2CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd825: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd827: 1.0})
reaction.gene_reaction_rule = '( FADS2 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_8')
reaction.name = 'fatty acyl-CoA desaturase (n-C18:1CoA -> n-C18:2CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd826: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd827: 1.0})
reaction.gene_reaction_rule = '( FADS2 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT18_9')
reaction.name = 'fatty acyl-CoA desaturase (n-C18:2CoA -> n-C18:3CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd828: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd829: 1.0})
reaction.gene_reaction_rule = '( FADS1 or FADS2 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT20_1')
reaction.name = 'fatty acyl-CoA desaturase (n-C20:3CoA -> n-C20:4CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd830: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd610: 1.0})
reaction.gene_reaction_rule = '( FADS1 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT20_2')
reaction.name = 'fatty acyl-CoA desaturase (n-C20:4CoA -> n-C20:5CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd831: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd832: 1.0})
reaction.gene_reaction_rule = '( FADS1 )'
model.add_reactions([reaction])
reaction = Reaction('DESAT24_1')
reaction.name = 'fatty acyl-CoA desaturase (n-C24:5CoA -> n-C24:6CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd35: -1.0, Cpd833: -1.0, Cpd2: 2.0, Cpd32: 1.0, Cpd834: 1.0})
reaction.gene_reaction_rule = '( FADS2 )'
model.add_reactions([reaction])
reaction = Reaction('DGAT')
reaction.name = 'diacylglycerol acyltransferase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd835: -1.0, Cpd658: -1.0, Cpd78: 1.0, Cpd836: 1.0})
reaction.gene_reaction_rule = '( DGAT1 or DGAT2 )'
model.add_reactions([reaction])
reaction = Reaction('DGK1')
reaction.name = 'deoxyguanylate kinase (dGMP:ATP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd837: -1.0, Cpd107: 1.0, Cpd771: 1.0})
reaction.gene_reaction_rule = '( GUK1 )'
model.add_reactions([reaction])
reaction = Reaction('DGNSKm')
reaction.name = 'deoxyguanosine kinase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd838: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd839: 1.0})
reaction.gene_reaction_rule = '( DGUOK )'
model.add_reactions([reaction])
reaction = Reaction('DGSNt')
reaction.name = 'deoyguanosine transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd840: -1.0, Cpd841: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('DHAAt1r')
reaction.name = 'dehydroascorbate transport (uniport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd842: -1.0, Cpd804: 1.0})
reaction.gene_reaction_rule = '( SLC2A1 or SLC2A3 or SLC2A4 )'
model.add_reactions([reaction])
reaction = Reaction('DHAPA')
reaction.name = 'dihydroxyacetone phosphate acyltransferase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd425: -1.0, Cpd843: -1.0, Cpd78: 1.0, Cpd844: 1.0})
reaction.gene_reaction_rule = '( GNPAT )'
model.add_reactions([reaction])
reaction = Reaction('DHAPAx')
reaction.name = 'dihydroxyacetone phosphate acyltransferase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd845: -1.0, Cpd846: -1.0, Cpd184: 1.0, Cpd356: 1.0})
reaction.gene_reaction_rule = '( GNPAT )'
model.add_reactions([reaction])
reaction = Reaction('DHCR241r')
reaction.name = '24-dehydrocholesterol reductase [Precursor]'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd628: -1.0, Cpd847: -1.0, Cpd848: 1.0, Cpd849: 1.0})
reaction.gene_reaction_rule = '( DHCR24 )'
model.add_reactions([reaction])
reaction = Reaction('DHCR242r')
reaction.name = '24-dehydrocholesterol reductase [Precursor]'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd847: -1.0, Cpd850: -1.0, Cpd848: 1.0, Cpd851: 1.0})
reaction.gene_reaction_rule = '( CYP7A1 or DHCR24 )'
model.add_reactions([reaction])
reaction = Reaction('DHCR243r')
reaction.name = '24-dehydrocholesterol reductase [Precursor]'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd847: -1.0, Cpd852: -1.0, Cpd670: 1.0, Cpd848: 1.0})
reaction.gene_reaction_rule = '( DHCR24 )'
model.add_reactions([reaction])
reaction = Reaction('DHCR71r')
reaction.name = '7-dehydrocholesterol reductase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd853: -1.0, Cpd589: 1.0, Cpd852: 1.0})
reaction.gene_reaction_rule = '( DHCR7 )'
model.add_reactions([reaction])
reaction = Reaction('DHCR72r')
reaction.name = '7-dehydrocholesterol reductase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd854: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd589: 1.0, Cpd670: 1.0})
reaction.gene_reaction_rule = '( DHCR7 )'
model.add_reactions([reaction])
reaction = Reaction('DHCRD1')
reaction.name = 'dihydroceramide desaturase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd855: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd662: 1.0})
reaction.gene_reaction_rule = '( DEGS1 or DEGS2 )'
model.add_reactions([reaction])
reaction = Reaction('DHCRD2')
reaction.name = 'dihydroceramide desaturase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd855: -1.0, Cpd856: -1.0, Cpd662: 1.0, Cpd857: 1.0})
reaction.gene_reaction_rule = '( DEGS1 or DEGS2 )'
model.add_reactions([reaction])
reaction = Reaction('DHDPBMTm')
reaction.name = 'dihydroxydecaprenylbenzoate methyltransferase'
reaction.subsystem = 'Ubiquinone synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd858: -1.0, Cpd442: -1.0, Cpd7: 1.0, Cpd441: 1.0, Cpd859: 1.0})
reaction.gene_reaction_rule = '( COQ3 )'
model.add_reactions([reaction])
reaction = Reaction('DHEASt')
reaction.name = 'dehydroepiandrosterone sulfate transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd860: -1.0, Cpd38: 1.0, Cpd861: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 or SLCO2B1 )'
model.add_reactions([reaction])
reaction = Reaction('DHEASULT')
reaction.name = 'Dehydroepiandrosterone sulfotransferase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd862: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd861: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 or SULT1E1 or SULT2A1 )'
model.add_reactions([reaction])
reaction = Reaction('DHFR')
reaction.name = 'dihydrofolate reductase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd863: -1.0, Cpd25: 1.0, Cpd377: 1.0})
reaction.gene_reaction_rule = '( DHFR )'
model.add_reactions([reaction])
reaction = Reaction('DHORD9')
reaction.name = 'dihydoorotic acid dehydrogenase (quinone10)'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd864: -1.0, Cpd752: 1.0, Cpd865: 1.0})
reaction.gene_reaction_rule = '( DHODH )'
model.add_reactions([reaction])
reaction = Reaction('DHORTS')
reaction.name = 'dihydroorotase'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd864: -1.0, Cpd33: 1.0, Cpd485: 1.0})
reaction.gene_reaction_rule = '( CAD )'
model.add_reactions([reaction])
reaction = Reaction('DHPM1')
reaction.name = 'dihydropyrimidinase (5,6-dihydrouracil)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd866: -1.0, Cpd33: 1.0, Cpd867: 1.0})
reaction.gene_reaction_rule = '( DPYS or DPYSL2 or DPYSL3 )'
model.add_reactions([reaction])
reaction = Reaction('DHPM2')
reaction.name = 'dihydropyrimidinase (dihydrothymine)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd868: -1.0, Cpd33: 1.0, Cpd581: 1.0})
reaction.gene_reaction_rule = '( DPYS )'
model.add_reactions([reaction])
reaction = Reaction('DHPR')
reaction.name = '6,7-dihydropteridine reductase'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd869: -1.0, Cpd32: 1.0, Cpd870: 1.0})
reaction.gene_reaction_rule = '( QDPR )'
model.add_reactions([reaction])
reaction = Reaction('DINt')
reaction.name = 'deoxyinosine transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd796: -1.0, Cpd794: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('D_LACt2')
reaction.name = 'D-lactate transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd871: -1.0, Cpd33: 1.0, Cpd872: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A3 or SLC16A7 or SLC16A8 )'
model.add_reactions([reaction])
reaction = Reaction('D_LACtm')
reaction.name = 'D-lactate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd872: -1.0, Cpd7: 1.0, Cpd873: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('DLNLCGCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd830: -1.0, Cpd78: 1.0, Cpd874: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('DLNLCGCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd875: -1.0, Cpd307: 1.0, Cpd876: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('DLNLCGCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd874: -1.0, Cpd875: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('DLNLCGt')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd877: -1.0, Cpd878: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('DMATT')
reaction.name = 'dimethylallyltranstransferase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd879: -1.0, Cpd880: -1.0, Cpd151: 1.0, Cpd881: 1.0})
reaction.gene_reaction_rule = '( GGPS1 )'
model.add_reactions([reaction])
reaction = Reaction('DMATTx')
reaction.name = 'dimethylallyltranstransferase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd882: -1.0, Cpd883: -1.0, Cpd884: 1.0, Cpd885: 1.0})
reaction.gene_reaction_rule = '( FDPS )'
model.add_reactions([reaction])
reaction = Reaction('DMGDHm')
reaction.name = 'dimethylglycine dehydrogenase, mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd886: -1.0, Cpd240: 1.0, Cpd887: 1.0, Cpd888: 1.0})
reaction.gene_reaction_rule = '( DMGDH )'
model.add_reactions([reaction])
reaction = Reaction('DNAMTn')
reaction.name = 'DNA (cytosine-5-)-methyltransferase, nucleus'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd889: -1.0, Cpd890: -1.0, Cpd891: 1.0, Cpd763: 1.0, Cpd892: 1.0})
reaction.gene_reaction_rule = '( DNMT1 or DNMT3A or DNMT3B or TRDMT1 )'
model.add_reactions([reaction])
reaction = Reaction('DNAMTSEn')
reaction.name = 'DNA (cytosine-5-)-methyltransferase, nucleus'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd890: -1.0, Cpd893: -1.0, Cpd763: 1.0, Cpd892: 1.0, Cpd894: 1.0})
reaction.gene_reaction_rule = '( DNMT1 or DNMT3A or DNMT3B or TRDMT1 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt10m')
reaction.name = 'dADP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd288: -1.0, Cpd895: -1.0, Cpd777: 1.0, Cpd896: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt11m')
reaction.name = 'dADP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd288: -1.0, Cpd897: -1.0, Cpd771: 1.0, Cpd896: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt12m')
reaction.name = 'dADP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd288: -1.0, Cpd107: 1.0, Cpd896: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt13m')
reaction.name = 'dADP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd288: -1.0, Cpd105: 1.0, Cpd896: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt14m')
reaction.name = 'dUDP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd898: -1.0, Cpd899: -1.0, Cpd900: 1.0, Cpd901: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt15m')
reaction.name = 'dUDP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd897: -1.0, Cpd899: -1.0, Cpd771: 1.0, Cpd901: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt16m')
reaction.name = 'dUDP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd896: -1.0, Cpd899: -1.0, Cpd288: 1.0, Cpd901: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt17m')
reaction.name = 'dUDP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd895: -1.0, Cpd899: -1.0, Cpd777: 1.0, Cpd901: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt18m')
reaction.name = 'dUDP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd899: -1.0, Cpd107: 1.0, Cpd901: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt19m')
reaction.name = 'dUDP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd899: -1.0, Cpd105: 1.0, Cpd901: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt1m')
reaction.name = 'dATP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd287: -1.0, Cpd107: 1.0, Cpd902: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt20m')
reaction.name = 'dTDP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd900: -1.0, Cpd105: 1.0, Cpd898: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt21m')
reaction.name = 'dTDP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd900: -1.0, Cpd107: 1.0, Cpd898: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt22m')
reaction.name = 'dTDP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd900: -1.0, Cpd901: -1.0, Cpd898: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt23m')
reaction.name = 'dTDP transport via dGTP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd897: -1.0, Cpd900: -1.0, Cpd771: 1.0, Cpd898: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt24m')
reaction.name = 'dTDP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd896: -1.0, Cpd900: -1.0, Cpd288: 1.0, Cpd898: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt25m')
reaction.name = 'dTDP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd895: -1.0, Cpd900: -1.0, Cpd777: 1.0, Cpd898: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt26m')
reaction.name = 'dCDP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd777: -1.0, Cpd901: -1.0, Cpd895: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt27m')
reaction.name = 'dCDP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd777: -1.0, Cpd898: -1.0, Cpd895: 1.0, Cpd900: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt28m')
reaction.name = 'dCDP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd777: -1.0, Cpd897: -1.0, Cpd771: 1.0, Cpd895: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt29m')
reaction.name = 'dCDP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd777: -1.0, Cpd896: -1.0, Cpd288: 1.0, Cpd895: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt2m')
reaction.name = 'dATP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd287: -1.0, Cpd105: 1.0, Cpd902: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt30m')
reaction.name = 'dCDP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd777: -1.0, Cpd107: 1.0, Cpd895: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt31m')
reaction.name = 'dCDP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd777: -1.0, Cpd105: 1.0, Cpd895: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt32m')
reaction.name = 'dGDP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd771: -1.0, Cpd105: 1.0, Cpd897: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt33m')
reaction.name = 'dGDP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd771: -1.0, Cpd901: -1.0, Cpd897: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt34m')
reaction.name = 'dGDP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd771: -1.0, Cpd898: -1.0, Cpd897: 1.0, Cpd900: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt35m')
reaction.name = 'dGDP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd771: -1.0, Cpd896: -1.0, Cpd288: 1.0, Cpd897: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt36m')
reaction.name = 'dGDP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd771: -1.0, Cpd895: -1.0, Cpd777: 1.0, Cpd897: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt37m')
reaction.name = 'dUTP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd898: -1.0, Cpd903: -1.0, Cpd900: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt38m')
reaction.name = 'dUTP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd901: -1.0, Cpd903: -1.0, Cpd899: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt39m')
reaction.name = 'dUTP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd897: -1.0, Cpd903: -1.0, Cpd771: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt3m')
reaction.name = 'dATP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd895: -1.0, Cpd777: 1.0, Cpd902: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt40m')
reaction.name = 'dUTP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd896: -1.0, Cpd903: -1.0, Cpd288: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt41m')
reaction.name = 'dUTP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd895: -1.0, Cpd903: -1.0, Cpd777: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt42m')
reaction.name = 'dUTP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd903: -1.0, Cpd107: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt43m')
reaction.name = 'dUTP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd903: -1.0, Cpd105: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt44m')
reaction.name = 'dTTP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd905: -1.0, Cpd105: 1.0, Cpd906: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt45m')
reaction.name = 'dTTP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd905: -1.0, Cpd107: 1.0, Cpd906: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt46m')
reaction.name = 'dTTP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd901: -1.0, Cpd905: -1.0, Cpd906: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt47m')
reaction.name = 'dTTP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd898: -1.0, Cpd905: -1.0, Cpd906: 1.0, Cpd900: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt48m')
reaction.name = 'dTTP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd897: -1.0, Cpd905: -1.0, Cpd771: 1.0, Cpd906: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt49m')
reaction.name = 'dTTP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd896: -1.0, Cpd905: -1.0, Cpd288: 1.0, Cpd906: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt4m')
reaction.name = 'dATP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd901: -1.0, Cpd902: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt50m')
reaction.name = 'dTTP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd895: -1.0, Cpd905: -1.0, Cpd777: 1.0, Cpd906: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt51m')
reaction.name = 'dCTP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd780: -1.0, Cpd895: -1.0, Cpd777: 1.0, Cpd907: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt52m')
reaction.name = 'dCTP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd780: -1.0, Cpd901: -1.0, Cpd907: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt53m')
reaction.name = 'dCTP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd780: -1.0, Cpd897: -1.0, Cpd771: 1.0, Cpd907: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt54m')
reaction.name = 'dCTP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd780: -1.0, Cpd896: -1.0, Cpd288: 1.0, Cpd907: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt55m')
reaction.name = 'dCTP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd780: -1.0, Cpd107: 1.0, Cpd907: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt56m')
reaction.name = 'dCTP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd780: -1.0, Cpd105: 1.0, Cpd907: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt57m')
reaction.name = 'dGTP transport via ATP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd770: -1.0, Cpd105: 1.0, Cpd908: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt58m')
reaction.name = 'dGTP transport via ADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd770: -1.0, Cpd107: 1.0, Cpd908: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt59m')
reaction.name = 'dGTP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd770: -1.0, Cpd896: -1.0, Cpd288: 1.0, Cpd908: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt5m')
reaction.name = 'dATP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd898: -1.0, Cpd902: 1.0, Cpd900: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt60m')
reaction.name = 'dGTP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd770: -1.0, Cpd901: -1.0, Cpd908: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt61m')
reaction.name = 'dGTP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd770: -1.0, Cpd898: -1.0, Cpd908: 1.0, Cpd900: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt62m')
reaction.name = 'dGTP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd770: -1.0, Cpd897: -1.0, Cpd771: 1.0, Cpd908: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt63m')
reaction.name = 'dGTP transport via dCDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd770: -1.0, Cpd895: -1.0, Cpd777: 1.0, Cpd908: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt6m')
reaction.name = 'dATP transport via dGDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd897: -1.0, Cpd771: 1.0, Cpd902: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt7m')
reaction.name = 'dATP transport via dADP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd896: -1.0, Cpd288: 1.0, Cpd902: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt8m')
reaction.name = 'dADP transport via dUDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd288: -1.0, Cpd901: -1.0, Cpd896: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DNDPt9m')
reaction.name = 'dADP transport via dTDP antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd288: -1.0, Cpd898: -1.0, Cpd896: 1.0, Cpd900: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('DOLASNT_Ler')
reaction.name = 'Dolichyl-diphosphooligosaccharide:protein-L-asparagine oligopolysaccharidotransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd909: -1.0, Cpd910: -0.1, Cpd565: 1.0, Cpd911: 0.1, Cpd912: 1.0})
reaction.gene_reaction_rule = '( DAD1 or DDOST or RPN1 or RPN2 )'
model.add_reactions([reaction])
reaction = Reaction('DOLASNT_Uer')
reaction.name = 'Dolichyl-diphosphooligosaccharide:protein-L-asparagine oligopolysaccharidotransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd909: -1.0, Cpd913: -0.1, Cpd565: 1.0, Cpd912: 1.0, Cpd914: 0.1})
reaction.gene_reaction_rule = '( DAD1 or DDOST or RPN1 or RPN2 )'
model.add_reactions([reaction])
reaction = Reaction('DOLDPP_Ler')
reaction.name = 'Dolichyl-diphosphate phosphohydrolase, human (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd911: -0.1, Cpd564: 0.1, Cpd565: 1.0, Cpd915: 1.0})
reaction.gene_reaction_rule = '( DOLPP1 )'
model.add_reactions([reaction])
reaction = Reaction('DOLDPP_Uer')
reaction.name = 'Dolichyl-diphosphate phosphohydrolase, human (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd914: -0.1, Cpd565: 1.0, Cpd564: 0.1, Cpd915: 1.0})
reaction.gene_reaction_rule = '( DOLPP1 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPGT1_Ler')
reaction.name = 'dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd916: -0.1, Cpd917: -0.1, Cpd564: 0.1, Cpd565: 1.0, Cpd918: 0.1})
reaction.gene_reaction_rule = '( ALG6 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPGT1_Uer')
reaction.name = 'dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd919: -0.1, Cpd920: -0.1, Cpd565: 1.0, Cpd564: 0.1, Cpd921: 0.1})
reaction.gene_reaction_rule = '( ALG6 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPGT2_Ler')
reaction.name = 'dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd916: -0.1, Cpd918: -0.1, Cpd564: 0.1, Cpd565: 1.0, Cpd910: 0.1})
reaction.gene_reaction_rule = '( ALG8 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPGT2_Uer')
reaction.name = 'dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd919: -0.1, Cpd921: -0.1, Cpd565: 1.0, Cpd564: 0.1, Cpd913: 0.1})
reaction.gene_reaction_rule = '( ALG8 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPGT3_Ler')
reaction.name = 'dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd564: -0.1, Cpd915: 1.0, Cpd922: 0.1})
reaction.gene_reaction_rule = '( ALG10B )'
model.add_reactions([reaction])
reaction = Reaction('DOLPGT3_Uer')
reaction.name = 'dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd564: -0.1, Cpd915: 1.0, Cpd923: 0.1})
reaction.gene_reaction_rule = '( ALG10B )'
model.add_reactions([reaction])
reaction = Reaction('DOLPMT_L')
reaction.name = 'Dolichyl-phosphate D-mannosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd924: -0.1, Cpd564: 0.1, Cpd565: 1.0, Cpd925: 0.1})
reaction.gene_reaction_rule = '( DPM2 or DPM3 or GM20716 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPMT_U')
reaction.name = 'Dolichyl-phosphate D-mannosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd567: -0.1, Cpd926: -0.1, Cpd565: 1.0, Cpd564: 0.1, Cpd927: 0.1})
reaction.gene_reaction_rule = '( DPM2 or DPM3 or GM20716 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPMT1_Ler')
reaction.name = 'dolichyl-phosphate-mannose-glycolipid alpha-mannosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd925: -0.1, Cpd564: 0.1, Cpd565: 1.0, Cpd928: 0.1})
reaction.gene_reaction_rule = '( ALG3 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPMT1_Uer')
reaction.name = 'dolichyl-phosphate-mannose-glycolipid alpha-mannosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd567: -0.1, Cpd927: -0.1, Cpd565: 1.0, Cpd564: 0.1, Cpd929: 0.1})
reaction.gene_reaction_rule = '( ALG3 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPMT3_Ler')
reaction.name = 'dolichyl-phosphate-mannose-glycolipid alpha-mannosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd548: -1.0, Cpd930: -0.1, Cpd283: 1.0, Cpd931: 0.1})
reaction.gene_reaction_rule = '( ALG12 )'
model.add_reactions([reaction])
reaction = Reaction('DOLPMT3_Uer')
reaction.name = 'dolichyl-phosphate-mannose-glycolipid alpha-mannosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd548: -1.0, Cpd930: -0.1, Cpd283: 1.0, Cpd932: 0.1})
reaction.gene_reaction_rule = '( ALG12 )'
model.add_reactions([reaction])
reaction = Reaction('DOPABMO')
reaction.name = 'dopamine beta-monooxygenase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd75: -1.0, Cpd480: -1.0, Cpd2: 1.0, Cpd98: 1.0, Cpd804: 1.0})
reaction.gene_reaction_rule = '( DBH or MOXD1 )'
model.add_reactions([reaction])
reaction = Reaction('DOPACHRMISO')
reaction.name = 'L-dopachrome isomerase 1'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd933: -2.0, Cpd2: 2.0, Cpd815: 2.0})
reaction.gene_reaction_rule = '( TYR )'
model.add_reactions([reaction])
reaction = Reaction('DOPAMT')
reaction.name = 'Dopamine:amet O-methyltransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd75: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd84: 1.0})
reaction.gene_reaction_rule = '( COMT )'
model.add_reactions([reaction])
reaction = Reaction('DOPASULT')
reaction.name = 'Dopamine Sulfotransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd75: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd934: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('DOPAt4_2_r')
reaction.name = 'Dopamine reversible transport in via sodium symport (1:2)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd935: -1.0, Cpd41: 2.0, Cpd75: 1.0})
reaction.gene_reaction_rule = '( SLC6A2 or SLC6A3 )'
model.add_reactions([reaction])
reaction = Reaction('DOPAtu')
reaction.name = 'Dopamine uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd935: -1.0, Cpd75: 1.0})
reaction.gene_reaction_rule = '( SLC22A2 or SLC22A3 or SLC22A5 )'
model.add_reactions([reaction])
reaction = Reaction('DOPAVESSEC')
reaction.name = 'Dopamine secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd75: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd935: 1.0})
reaction.gene_reaction_rule = '( SLC18A1 or SLC18A2 )'
model.add_reactions([reaction])
reaction = Reaction('DORNOp')
reaction.name = 'D-ornithine oxidase, perixosomal'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd936: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd937: 1.0})
reaction.gene_reaction_rule = '( DAO )'
model.add_reactions([reaction])
reaction = Reaction('DPCOAK')
reaction.name = 'dephospho-CoA kinase'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd938: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd78: 1.0})
reaction.gene_reaction_rule = '( COASY )'
model.add_reactions([reaction])
reaction = Reaction('DPGase')
reaction.name = 'Diphosphoglycerate phosphatase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd939: -1.0, Cpd108: 1.0, Cpd273: 1.0})
reaction.gene_reaction_rule = '( BPGM or PGAM1 or PGAM2 )'
model.add_reactions([reaction])
reaction = Reaction('DPGM')
reaction.name = 'Diphosphoglyceromutase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd272: -1.0, Cpd33: 1.0, Cpd939: 1.0})
reaction.gene_reaction_rule = '( BPGM or PGAM1 or PGAM2 )'
model.add_reactions([reaction])
reaction = Reaction('DPMVDx')
reaction.name = 'diphosphomevalonate decarboxylase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd940: -1.0, Cpd941: -1.0, Cpd942: 1.0, Cpd943: 1.0, Cpd883: 1.0, Cpd944: 1.0})
reaction.gene_reaction_rule = '( MVD )'
model.add_reactions([reaction])
reaction = Reaction('DPROOp')
reaction.name = 'D-proline oxidase, perixosomal'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd945: -1.0, Cpd358: 1.0, Cpd247: 1.0, Cpd946: 1.0})
reaction.gene_reaction_rule = '( DAO )'
model.add_reactions([reaction])
reaction = Reaction('DRBK')
reaction.name = 'Deoxyribokinase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd947: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd948: 1.0})
reaction.gene_reaction_rule = '( RBKS )'
model.add_reactions([reaction])
reaction = Reaction('DRPA')
reaction.name = 'deoxyribose-phosphate aldolase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd948: -1.0, Cpd417: 1.0, Cpd949: 1.0})
reaction.gene_reaction_rule = '( DERA )'
model.add_reactions([reaction])
reaction = Reaction('DTMPK')
reaction.name = 'dTMP kinase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd950: -1.0, Cpd107: 1.0, Cpd900: 1.0})
reaction.gene_reaction_rule = '( DTYMK )'
model.add_reactions([reaction])
reaction = Reaction('DURAD')
reaction.name = 'dihydrouracil dehydrogenase (NADP)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd866: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd951: 1.0})
reaction.gene_reaction_rule = '( DPYD )'
model.add_reactions([reaction])
reaction = Reaction('DURAD2')
reaction.name = 'dihydrothymin dehydrogenase (NADP)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd868: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd952: 1.0})
reaction.gene_reaction_rule = '( DPYD )'
model.add_reactions([reaction])
reaction = Reaction('DURIK1')
reaction.name = 'deoxyuridine kinase (ATP:Deoxyuridine)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd818: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd810: 1.0})
reaction.gene_reaction_rule = '( TK1 or TK2 )'
model.add_reactions([reaction])
reaction = Reaction('DURIK1m')
reaction.name = 'deoxyuridine kinase (ATP:Deoxyuridine), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd953: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd954: 1.0})
reaction.gene_reaction_rule = '( TK2 )'
model.add_reactions([reaction])
reaction = Reaction('DURIPP')
reaction.name = 'deoxyuridine phosphorylase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd818: -1.0, Cpd955: 1.0, Cpd951: 1.0})
reaction.gene_reaction_rule = '( PNP2 or UPP2 )'
model.add_reactions([reaction])
reaction = Reaction('DURIt')
reaction.name = 'deoxyuridine transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd956: -1.0, Cpd818: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('DUTPDPm')
reaction.name = 'dUTP diphosphatase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd904: -1.0, Cpd7: 1.0, Cpd210: 1.0, Cpd954: 1.0})
reaction.gene_reaction_rule = '( DUT or ITPA )'
model.add_reactions([reaction])
reaction = Reaction('DUTPDPn')
reaction.name = 'dUTP diphosphatase, nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd957: -1.0, Cpd718: 1.0, Cpd763: 1.0, Cpd958: 1.0})
reaction.gene_reaction_rule = '( DUT )'
model.add_reactions([reaction])
reaction = Reaction('EBP1r')
reaction.name = '3-beta-hydroxysteroid-delta(8),delta(7)-isomerase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd628: -1.0, Cpd850: 1.0})
reaction.gene_reaction_rule = '( EBP )'
model.add_reactions([reaction])
reaction = Reaction('EBP2r')
reaction.name = '3-beta-hydroxysteroid-delta(8),delta(7)-isomerase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd849: -1.0, Cpd851: 1.0})
reaction.gene_reaction_rule = '( EBP )'
model.add_reactions([reaction])
reaction = Reaction('ECOAH12m')
reaction.name = '3-hydroxyacyl-CoA dehydratase (3-hydroxyisobutyryl-CoA) (mitochondria)'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd245: -1.0, Cpd959: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('ECOAH1m')
reaction.name = '3-hydroxyacyl-CoA dehydratase (3-hydroxybutanoyl-CoA) (mitochondria)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd960: -1.0, Cpd12: 1.0, Cpd70: 1.0})
reaction.gene_reaction_rule = '( AUH or ECHS1 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('ECOAH1x')
reaction.name = '3-hydroxyacyl-CoA dehydratase (3-hydroxybutanoyl-CoA) (perioxisome)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd961: -1.0, Cpd261: 1.0, Cpd962: 1.0})
reaction.gene_reaction_rule = '( ECH1 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('ECOAH9m')
reaction.name = '2-Methylprop-2-enoyl-CoA (2-Methylbut-2-enoyl-CoA), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd239: -1.0, Cpd963: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('EGMESTr')
reaction.name = 'ecgonine methyl esterase (ER)'
reaction.subsystem = 'Alkaloid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd721: -1.0, Cpd565: 1.0, Cpd964: 1.0, Cpd965: 1.0})
reaction.gene_reaction_rule = '( AADAC )'
model.add_reactions([reaction])
reaction = Reaction('EHGLAT')
reaction.name = 'L-erythro-4-Hydroxyglutamate:2-oxoglutarate aminotransferase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd966: -1.0, Cpd74: 1.0, Cpd967: 1.0})
reaction.gene_reaction_rule = '( GOT1 )'
model.add_reactions([reaction])
reaction = Reaction('EHGLAT2m')
reaction.name = 'L-erythro-4-Hydroxyglutamate:2-oxoglutarate aminotransferase 2, mitochondrial'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd100: -1.0, Cpd491: -1.0, Cpd92: 1.0, Cpd968: 1.0})
reaction.gene_reaction_rule = '( GOT2 )'
model.add_reactions([reaction])
reaction = Reaction('EHGLATm')
reaction.name = 'L-erythro-4-Hydroxyglutamate:2-oxoglutarate aminotransferase, mitochondrial'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd100: -1.0, Cpd63: 1.0, Cpd968: 1.0})
reaction.gene_reaction_rule = '( GOT2 )'
model.add_reactions([reaction])
reaction = Reaction('EICOSTETCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd831: -1.0, Cpd78: 1.0, Cpd969: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('EICOSTETCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd970: -1.0, Cpd307: 1.0, Cpd971: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('EICOSTETCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd969: -1.0, Cpd970: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('ELAIDCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd826: -1.0, Cpd78: 1.0, Cpd972: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('ELAIDCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd973: -1.0, Cpd307: 1.0, Cpd974: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('ELAIDCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd972: -1.0, Cpd973: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('ELAIDt')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd975: -1.0, Cpd976: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('ENGASE')
reaction.name = 'endo-beta-N-acetylglucosaminidase'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd977: -1.0, Cpd215: 1.0, Cpd978: 1.0})
reaction.gene_reaction_rule = '( ENGASE )'
model.add_reactions([reaction])
reaction = Reaction('ENGASE2')
reaction.name = 'endo-beta-N-acetylglucosaminidase'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd979: -1.0, Cpd215: 1.0, Cpd980: 1.0})
reaction.gene_reaction_rule = '( ENGASE )'
model.add_reactions([reaction])
reaction = Reaction('ENO')
reaction.name = 'enolase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd981: -1.0, Cpd2: 1.0, Cpd231: 1.0})
reaction.gene_reaction_rule = '( ENO1 or ENO2 or ENO3 )'
model.add_reactions([reaction])
reaction = Reaction('EPCTX')
reaction.name = 'ethanolamine-phosphate cytidylyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd653: -1.0, Cpd982: -1.0, Cpd151: 1.0, Cpd983: 1.0})
reaction.gene_reaction_rule = '( PCYT2 )'
model.add_reactions([reaction])
reaction = Reaction('ESTRADIOLGLCt')
reaction.name = 'estradiol glucuronide transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd984: -1.0, Cpd38: 1.0, Cpd985: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 or SLCO1C1 or SLCO4A1 )'
model.add_reactions([reaction])
reaction = Reaction('ESTRADIOLGLCt2')
reaction.name = '17-beta-D-glucuronide transport (ATP-dependent)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd985: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd984: 1.0})
reaction.gene_reaction_rule = '( ABCC4 )'
model.add_reactions([reaction])
reaction = Reaction('ESTRIOLGLCte')
reaction.name = 'glucuronidated compound transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd986: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd987: 1.0})
reaction.gene_reaction_rule = '( ABCC1 )'
model.add_reactions([reaction])
reaction = Reaction('ESTRONEGLCt')
reaction.name = 'ABC transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd988: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd989: 1.0})
reaction.gene_reaction_rule = '( ABCC3 )'
model.add_reactions([reaction])
reaction = Reaction('ESTRONESt')
reaction.name = 'estrone 3-sulfate transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd990: -1.0, Cpd38: 1.0, Cpd991: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 or SLCO1C1 or SLCO2B1 or SLCO3A1 or SLCO4A1 )'
model.add_reactions([reaction])
reaction = Reaction('ESTRONESt2')
reaction.name = 'estrone-3-sulfate transport via sodium cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd990: -1.0, Cpd41: 2.0, Cpd991: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('ESTSULT')
reaction.name = 'Estrogen sulfotransferase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd992: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd991: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 or SULT1E1 )'
model.add_reactions([reaction])
reaction = Reaction('ETF')
reaction.name = 'electron transfer flavoprotein'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd240: -1.0, Cpd993: -1.0, Cpd238: 1.0, Cpd994: 1.0})
reaction.gene_reaction_rule = '( ETFA or ETFB )'
model.add_reactions([reaction])
reaction = Reaction('ETFQO')
reaction.name = 'Electron transfer flavoprotein-ubiquinone oxidoreductase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd994: -1.0, Cpd752: 1.0, Cpd993: 1.0})
reaction.gene_reaction_rule = '( ETFDH )'
model.add_reactions([reaction])
reaction = Reaction('ETHAK')
reaction.name = 'Ethanolamine kinase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd995: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd996: 1.0})
reaction.gene_reaction_rule = '( CHKA or CHKB or ETNK1 or ETNK2 )'
model.add_reactions([reaction])
reaction = Reaction('ETHP')
reaction.name = 'ethanolamine phosphate phosphatase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd996: -1.0, Cpd108: 1.0, Cpd995: 1.0})
reaction.gene_reaction_rule = '( PHOSPHO1 )'
model.add_reactions([reaction])
reaction = Reaction('ETOHMO')
reaction.name = 'ethanol monooxygenase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd416: -1.0, Cpd2: 2.0, Cpd25: 1.0, Cpd417: 1.0})
reaction.gene_reaction_rule = '( CYP2E1 )'
model.add_reactions([reaction])
reaction = Reaction('F1PGT')
reaction.name = 'fucose-1-phosphate guanylyltransferase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd282: -1.0, Cpd997: -1.0, Cpd151: 1.0, Cpd998: 1.0})
reaction.gene_reaction_rule = '( FPGT or TNNI3K )'
model.add_reactions([reaction])
reaction = Reaction('F6Tg')
reaction.name = 'glycoprotein 6-alpha-L-fucosyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd999: -1.0, Cpd1000: -1.0, Cpd140: 1.0, Cpd1001: 1.0, Cpd1002: 1.0})
reaction.gene_reaction_rule = '( FUT8 )'
model.add_reactions([reaction])
reaction = Reaction('FA120ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1003: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1004: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FA140ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1005: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1006: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FA141ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1007: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1008: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FA160ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1009: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1010: 1.0})
reaction.gene_reaction_rule = '( FASN or OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FA161ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1011: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1012: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FA180ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1013: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1014: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FA181ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1015: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1016: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FA182ACPH')
reaction.name = 'fatty-acyl-ACP hydrolase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1017: -1.0, Cpd33: 1.0, Cpd249: 1.0, Cpd1018: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL140i')
reaction.name = 'fatty-acid--CoA ligase (tetradecanoate)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1006: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1019: 1.0})
reaction.gene_reaction_rule = '( ACSL3 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL150')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1020: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1021: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL160i')
reaction.name = 'C160 fatty acid activation'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1010: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd590: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL161')
reaction.name = 'fatty-acid--CoA ligase (hexadecenoate)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1012: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd594: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL170')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1022: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1023: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL180i')
reaction.name = 'C180 fatty acid activation'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1014: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd602: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL5 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL1812')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1024: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd825: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL1813')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd976: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd826: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL181i')
reaction.name = 'C181 fatty acid activation'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1016: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd606: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL1821')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1025: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd828: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL1822')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1026: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd827: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL1831')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1027: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd829: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL1832')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1028: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd823: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL184')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1029: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd824: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL191')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd423: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1030: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL200')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1031: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd459: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL203')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd878: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd830: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL204')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd427: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd610: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL2042')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1032: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd831: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL205')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1033: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd832: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL206')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1034: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1035: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL224')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1036: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd302: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL2251')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1037: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd811: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL2252')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1038: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd706: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL226')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1039: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd614: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL240')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1040: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1041: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL241')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1042: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1043: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL244_1')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1044: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1045: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL245_1')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1046: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1047: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL245_2')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1048: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd833: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL246_1')
reaction.name = 'fatty-acid--CoA ligase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1049: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd834: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL260')
reaction.name = 'fatty-acid--CoA ligase (n-C26:0)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1050: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1051: 1.0})
reaction.gene_reaction_rule = '( ACSL1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL40im')
reaction.name = 'fatty-acid--CoA ligase (butanoate), mitochondrial'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd584: -1.0, Cpd209: 1.0, Cpd210: 1.0, Cpd241: 1.0})
reaction.gene_reaction_rule = '( ACSM1 )'
model.add_reactions([reaction])
reaction = Reaction('FACOAL80i')
reaction.name = 'fatty-acid--CoA ligase (octanoate)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1052: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1053: 1.0})
reaction.gene_reaction_rule = '( ACSL3 )'
model.add_reactions([reaction])
reaction = Reaction('FADDP')
reaction.name = 'FAD diphosphatase'
reaction.subsystem = 'Vitamin B2 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd856: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd263: 1.0})
reaction.gene_reaction_rule = '( ENPP1 or ENPP3 )'
model.add_reactions([reaction])
reaction = Reaction('FAEL183')
reaction.name = 'fatty-acyl-CoA elongation (n-C18:3CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -5.0, Cpd22: -4.0, Cpd199: -1.0, Cpd829: -1.0, Cpd2: 3.0, Cpd25: 4.0, Cpd60: 1.0, Cpd78: 1.0, Cpd830: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('FAEL184')
reaction.name = 'fatty-acyl-CoA elongation (n-C20:4CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -5.0, Cpd22: -4.0, Cpd199: -1.0, Cpd824: -1.0, Cpd2: 3.0, Cpd25: 4.0, Cpd60: 1.0, Cpd78: 1.0, Cpd831: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('FAEL204')
reaction.name = 'fatty-acyl-CoA elongation (n-C20:4CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -5.0, Cpd22: -4.0, Cpd199: -1.0, Cpd610: -1.0, Cpd2: 3.0, Cpd25: 4.0, Cpd60: 1.0, Cpd78: 1.0, Cpd302: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('FAEL205')
reaction.name = 'fatty-acyl-CoA elongation (n-C20:5CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -5.0, Cpd22: -4.0, Cpd199: -1.0, Cpd832: -1.0, Cpd2: 3.0, Cpd25: 4.0, Cpd60: 1.0, Cpd78: 1.0, Cpd706: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('FAH1')
reaction.name = 'Fatty acid omega-hydroxylase'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1004: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1054: 1.0})
reaction.gene_reaction_rule = '( CYP4A29 )'
model.add_reactions([reaction])
reaction = Reaction('FAH2')
reaction.name = 'Fatty acid omega-hydroxylase'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1006: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1055: 1.0})
reaction.gene_reaction_rule = '( CYP4A29 )'
model.add_reactions([reaction])
reaction = Reaction('FAH3')
reaction.name = 'Fatty acid omega-hydroxylase'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1010: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1056: 1.0})
reaction.gene_reaction_rule = '( CYP4A29 )'
model.add_reactions([reaction])
reaction = Reaction('FALDH')
reaction.name = 'formaldehyde dehydrogenase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd411: -1.0, Cpd1057: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1058: 1.0})
reaction.gene_reaction_rule = '( ADH5 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC11')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd1059: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd1060: 1.0})
reaction.gene_reaction_rule = '( ACADM )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC140')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -6.0, Cpd43: -6.0, Cpd44: -6.0, Cpd238: -6.0, Cpd1061: -1.0, Cpd7: 6.0, Cpd47: 6.0, Cpd178: 7.0, Cpd240: 6.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC150m')
reaction.name = 'Beta oxidation of long chain fatty acid (odd chain)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -6.0, Cpd43: -6.0, Cpd44: -6.0, Cpd238: -6.0, Cpd1062: -1.0, Cpd7: 6.0, Cpd47: 6.0, Cpd178: 6.0, Cpd179: 1.0, Cpd240: 6.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC160')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -7.0, Cpd43: -7.0, Cpd44: -7.0, Cpd238: -7.0, Cpd593: -1.0, Cpd7: 7.0, Cpd47: 7.0, Cpd178: 8.0, Cpd240: 7.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16080m')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -4.0, Cpd43: -4.0, Cpd44: -4.0, Cpd238: -4.0, Cpd593: -1.0, Cpd7: 4.0, Cpd47: 4.0, Cpd178: 4.0, Cpd240: 4.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16080x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -4.0, Cpd184: -4.0, Cpd196: -1.0, Cpd246: -4.0, Cpd261: -4.0, Cpd358: 4.0, Cpd1065: 4.0, Cpd182: 4.0, Cpd188: 1.0, Cpd247: 4.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC161802m')
reaction.name = 'Beta oxidation  fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -4.0, Cpd43: -4.0, Cpd44: -4.0, Cpd238: -3.0, Cpd601: -1.0, Cpd7: 4.0, Cpd47: 4.0, Cpd178: 4.0, Cpd240: 3.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16180m')
reaction.name = 'Beta oxidation  fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -4.0, Cpd43: -4.0, Cpd44: -4.0, Cpd238: -3.0, Cpd599: -1.0, Cpd7: 4.0, Cpd47: 4.0, Cpd178: 4.0, Cpd240: 3.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC170m')
reaction.name = 'Beta oxidation of long chain fatty acid (odd chain)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -7.0, Cpd43: -7.0, Cpd44: -7.0, Cpd238: -7.0, Cpd1066: -1.0, Cpd7: 7.0, Cpd47: 7.0, Cpd178: 7.0, Cpd179: 1.0, Cpd240: 7.0})
reaction.gene_reaction_rule = '( ACADM )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC180')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd605: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd593: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC180x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1067: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd196: 1.0, Cpd247: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC1811601m')
reaction.name = 'Beta oxidation  fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd238: -4.0, Cpd974: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd240: 4.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC1811602m')
reaction.name = 'Beta oxidation  fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd238: -4.0, Cpd1068: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd240: 4.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC1811603m')
reaction.name = 'Beta oxidation  fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd238: -4.0, Cpd609: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd240: 4.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC182806m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd238: -3.0, Cpd1069: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd240: 3.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC18280m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd238: -3.0, Cpd1070: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd240: 3.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC183803m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd238: -2.0, Cpd1071: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd240: 2.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC183806m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -2.0, Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd1072: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd634: 2.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC183806x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -5.0, Cpd184: -5.0, Cpd246: -2.0, Cpd261: -5.0, Cpd1073: -1.0, Cpd358: 5.0, Cpd1065: 5.0, Cpd182: 5.0, Cpd188: 1.0, Cpd247: 2.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC18480m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -5.0, Cpd43: -5.0, Cpd44: -5.0, Cpd238: -1.0, Cpd1074: -1.0, Cpd7: 5.0, Cpd47: 5.0, Cpd178: 5.0, Cpd240: 1.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC18480x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -5.0, Cpd184: -5.0, Cpd246: -1.0, Cpd261: -5.0, Cpd1075: -1.0, Cpd358: 5.0, Cpd1065: 5.0, Cpd182: 5.0, Cpd188: 1.0, Cpd247: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC200180m')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd462: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd605: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC200180x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1076: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1067: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2031836m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd876: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd1072: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC204')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -9.0, Cpd43: -9.0, Cpd44: -9.0, Cpd238: -5.0, Cpd613: -1.0, Cpd7: 9.0, Cpd47: 9.0, Cpd178: 10.0, Cpd240: 5.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC204184m')
reaction.name = 'Beta oxidation  fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd971: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd1074: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2051843m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd1077: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd1074: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2051843x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd1078: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd1075: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2242046m')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd306: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd613: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2242046x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1079: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1080: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2251836m')
reaction.name = 'Beta oxidation of fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -2.0, Cpd43: -2.0, Cpd44: -2.0, Cpd814: -1.0, Cpd7: 2.0, Cpd47: 2.0, Cpd178: 2.0, Cpd1072: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2251836x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -2.0, Cpd184: -2.0, Cpd261: -2.0, Cpd1081: -1.0, Cpd358: 2.0, Cpd1065: 2.0, Cpd182: 2.0, Cpd1073: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2252053m')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd814: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd634: 1.0, Cpd1077: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2252053x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1081: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1078: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -10.0, Cpd43: -10.0, Cpd44: -10.0, Cpd238: -4.0, Cpd617: -1.0, Cpd7: 10.0, Cpd47: 10.0, Cpd178: 11.0, Cpd240: 4.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226205m')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd617: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd1077: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226205x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd1082: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd1078: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC240200x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -2.0, Cpd184: -2.0, Cpd246: -2.0, Cpd261: -2.0, Cpd1083: -1.0, Cpd358: 2.0, Cpd1065: 2.0, Cpd182: 2.0, Cpd247: 2.0, Cpd1076: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC241181x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -3.0, Cpd184: -3.0, Cpd246: -3.0, Cpd261: -3.0, Cpd1084: -1.0, Cpd358: 3.0, Cpd1065: 3.0, Cpd182: 3.0, Cpd247: 3.0, Cpd1085: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2442246x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1086: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1079: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2452253x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1087: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1088: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC2452256x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1089: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1081: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC246226x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1090: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1082: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC260240x')
reaction.name = 'Beta oxidation of long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1091: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1083: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC80')
reaction.name = 'Beta oxidation of med/long chain fatty acid'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -3.0, Cpd43: -3.0, Cpd44: -3.0, Cpd238: -3.0, Cpd1063: -1.0, Cpd7: 3.0, Cpd47: 3.0, Cpd178: 4.0, Cpd240: 3.0})
reaction.gene_reaction_rule = '( ACADM or ACADS )'
model.add_reactions([reaction])
reaction = Reaction('FAS100COA')
reaction.name = 'fatty acyl-CoA synthase (n-C10:0CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -3.0, Cpd22: -2.0, Cpd199: -1.0, Cpd1053: -1.0, Cpd2: 1.0, Cpd25: 2.0, Cpd60: 1.0, Cpd78: 1.0, Cpd1092: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL5 or ELOVL6 or FASN )'
model.add_reactions([reaction])
reaction = Reaction('FAS120COA')
reaction.name = 'fatty-acyl-CoA synthase (n-C12:0CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -3.0, Cpd22: -2.0, Cpd199: -1.0, Cpd1092: -1.0, Cpd2: 1.0, Cpd25: 2.0, Cpd60: 1.0, Cpd78: 1.0, Cpd1093: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL5 or ELOVL6 or FASN )'
model.add_reactions([reaction])
reaction = Reaction('FAS140COA')
reaction.name = 'fatty-acyl-CoA synthase (n-C14:0CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -3.0, Cpd22: -2.0, Cpd199: -1.0, Cpd1093: -1.0, Cpd2: 1.0, Cpd25: 2.0, Cpd60: 1.0, Cpd78: 1.0, Cpd1019: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL5 or ELOVL6 or FASN )'
model.add_reactions([reaction])
reaction = Reaction('FAS160COA')
reaction.name = 'fatty-acyl-CoA synthase (n-C16:0CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -3.0, Cpd22: -2.0, Cpd199: -1.0, Cpd1019: -1.0, Cpd2: 1.0, Cpd25: 2.0, Cpd60: 1.0, Cpd78: 1.0, Cpd590: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL5 or ELOVL6 or FASN )'
model.add_reactions([reaction])
reaction = Reaction('FAS180COA')
reaction.name = 'fatty-acyl-CoA synthase (n-C18:0CoA)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -3.0, Cpd22: -2.0, Cpd199: -1.0, Cpd590: -1.0, Cpd2: 1.0, Cpd25: 2.0, Cpd60: 1.0, Cpd78: 1.0, Cpd602: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL5 or ELOVL6 or FASN )'
model.add_reactions([reaction])
reaction = Reaction('FAS80COA_L')
reaction.name = 'fatty acyl-CoA synthase (n-C8:0CoA), lumped reaction'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -9.0, Cpd22: -6.0, Cpd180: -1.0, Cpd199: -3.0, Cpd2: 3.0, Cpd25: 6.0, Cpd60: 3.0, Cpd78: 3.0, Cpd1053: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('FATP1t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1010: -1.0, Cpd39: 1.0, Cpd1094: 1.0})
reaction.gene_reaction_rule = '( SLC27A1 or SLC27A3 or SLC27A4 )'
model.add_reactions([reaction])
reaction = Reaction('FATP2t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1016: -1.0, Cpd39: 1.0, Cpd1095: 1.0})
reaction.gene_reaction_rule = '( SLC27A1 )'
model.add_reactions([reaction])
reaction = Reaction('FATP3t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1014: -1.0, Cpd39: 1.0, Cpd1096: 1.0})
reaction.gene_reaction_rule = '( SLC27A1 )'
model.add_reactions([reaction])
reaction = Reaction('FATP4t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1031: -1.0, Cpd39: 1.0, Cpd1097: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('FATP5t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd427: -1.0, Cpd39: 1.0, Cpd463: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('FATP6t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1036: -1.0, Cpd39: 1.0, Cpd1098: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 or SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('FATP7t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1039: -1.0, Cpd39: 1.0, Cpd1099: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 or SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('FATP8t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1040: -1.0, Cpd39: 1.0, Cpd1100: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 or SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('FATP9t')
reaction.name = 'fatty acid electroneutral transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1042: -1.0, Cpd39: 1.0, Cpd1101: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 or SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('FBA')
reaction.name = 'fructose-bisphosphate aldolase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1102: -1.0, Cpd425: 1.0, Cpd949: 1.0})
reaction.gene_reaction_rule = '( ALDOART2 or ALDOB or ALDOC )'
model.add_reactions([reaction])
reaction = Reaction('FBA2')
reaction.name = 'D-Fructose 1-phosphate D-glyceraldehyde-3-phosphate-lyase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1103: -1.0, Cpd425: 1.0, Cpd1104: 1.0})
reaction.gene_reaction_rule = '( ALDOART2 or ALDOB or ALDOC )'
model.add_reactions([reaction])
reaction = Reaction('FBA4')
reaction.name = 'D-Fructose 1-phosphate D-glyceraldehyde-3-phosphate-lyase'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1105: -1.0, Cpd425: 1.0, Cpd1106: 1.0})
reaction.gene_reaction_rule = '( ALDOART2 or ALDOB or ALDOC )'
model.add_reactions([reaction])
reaction = Reaction('FBA5')
reaction.name = 'D-Tagatose 1-phosphate D-glyceraldehyde-3-phosphate-lyase'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1107: -1.0, Cpd425: 1.0, Cpd1104: 1.0})
reaction.gene_reaction_rule = '( ALDOB )'
model.add_reactions([reaction])
reaction = Reaction('FBP')
reaction.name = 'fructose-bisphosphatase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1102: -1.0, Cpd108: 1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( FBP1 or FBP2 )'
model.add_reactions([reaction])
reaction = Reaction('FBP26')
reaction.name = 'Fructose-2,6-bisphosphate 2-phosphatase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1109: -1.0, Cpd108: 1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( FBP1 or FBP2 or PFKFB1 or PFKFB2 or PFKFB3 or PFKFB4 )'
model.add_reactions([reaction])
reaction = Reaction('FCLTm')
reaction.name = 'Ferrochelatase, mitochondrial'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1110: -1.0, Cpd1111: -1.0, Cpd7: 2.0, Cpd1112: 1.0})
reaction.gene_reaction_rule = '( FECH )'
model.add_reactions([reaction])
reaction = Reaction('FDH')
reaction.name = 'formate dehydrogenase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd699: -1.0, Cpd35: 1.0, Cpd60: 1.0})
reaction.gene_reaction_rule = '( ALDH1L1 )'
model.add_reactions([reaction])
reaction = Reaction('FE3R2e')
reaction.name = 'Fe(III) reduction (ascorbate)'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd479: -1.0, Cpd1113: -2.0, Cpd37: 1.0, Cpd842: 1.0, Cpd1114: 2.0})
reaction.gene_reaction_rule = '( CYBRD1 )'
model.add_reactions([reaction])
reaction = Reaction('FK')
reaction.name = 'Fucokinase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1115: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd997: 1.0})
reaction.gene_reaction_rule = '( FUK )'
model.add_reactions([reaction])
reaction = Reaction('FKYNH')
reaction.name = 'N-Formyl-L-kynurenine amidohydrolase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1116: -1.0, Cpd33: 1.0, Cpd699: 1.0, Cpd1117: 1.0})
reaction.gene_reaction_rule = '( AFMID )'
model.add_reactions([reaction])
reaction = Reaction('FMNAT')
reaction.name = 'FMN adenylyltransferase'
reaction.subsystem = 'Vitamin B2 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd263: -1.0, Cpd151: 1.0, Cpd856: 1.0})
reaction.gene_reaction_rule = '( FLAD1 )'
model.add_reactions([reaction])
reaction = Reaction('FOLR2')
reaction.name = 'folate reductase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd22: -1.0, Cpd1118: -1.0, Cpd25: 1.0, Cpd863: 1.0})
reaction.gene_reaction_rule = '( DHFR )'
model.add_reactions([reaction])
reaction = Reaction('FOLt2')
reaction.name = 'folate transport via anion exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1119: -1.0, Cpd114: 1.0, Cpd1118: 1.0})
reaction.gene_reaction_rule = '( SLC19A1 )'
model.add_reactions([reaction])
reaction = Reaction('FPGS')
reaction.name = 'folylpolyglutamate synthetase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd74: -4.0, Cpd377: -1.0, Cpd2: 3.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1120: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS2')
reaction.name = 'folylpolyglutamate synthetase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd74: -1.0, Cpd1120: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1121: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS2m')
reaction.name = 'folylpolyglutamate synthetase, mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -1.0, Cpd1122: -1.0, Cpd200: -1.0, Cpd7: 1.0, Cpd1123: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS3')
reaction.name = 'folylpolyglutamate synthetase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd74: -1.0, Cpd1121: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1124: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS3m')
reaction.name = 'folylpolyglutamate synthetase, mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -1.0, Cpd1123: -1.0, Cpd200: -1.0, Cpd7: 1.0, Cpd1125: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS4')
reaction.name = 'folylpolyglutamate synthetase (DHF)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd74: -4.0, Cpd863: -1.0, Cpd2: 3.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1126: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS4m')
reaction.name = 'folylpolyglutamate synthetase (DHF), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -4.0, Cpd200: -1.0, Cpd1127: -1.0, Cpd7: 1.0, Cpd12: 3.0, Cpd202: 1.0, Cpd204: 1.0, Cpd1128: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS5')
reaction.name = 'folylpolyglutamate  synthetase (DHF)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd74: -1.0, Cpd1126: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1129: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS5m')
reaction.name = 'folylpolyglutamate  synthetase (DHF), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -1.0, Cpd200: -1.0, Cpd1128: -1.0, Cpd7: 1.0, Cpd1130: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS6')
reaction.name = 'folylpolyglutamate synthetase (DHF)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd74: -1.0, Cpd1129: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1131: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS6m')
reaction.name = 'folylpolyglutamate synthetase (DHF), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -1.0, Cpd1130: -1.0, Cpd200: -1.0, Cpd7: 1.0, Cpd1132: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS7')
reaction.name = 'folylpolyglutamate synthetase (10fthf)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd375: -1.0, Cpd105: -1.0, Cpd74: -4.0, Cpd1133: 1.0, Cpd2: 3.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS7m')
reaction.name = 'folylpolyglutamate synthetase (10fthf), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1134: -1.0, Cpd63: -4.0, Cpd200: -1.0, Cpd1135: 1.0, Cpd7: 1.0, Cpd12: 3.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS8')
reaction.name = 'folylpolyglutamate synthetase (10fthf)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1133: -1.0, Cpd105: -1.0, Cpd74: -1.0, Cpd1136: 1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS8m')
reaction.name = 'folylpolyglutamate synthetase (10fthf), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1135: -1.0, Cpd63: -1.0, Cpd200: -1.0, Cpd1137: 1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS9')
reaction.name = 'folylpolyglutamate synthetase (10fthf)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1136: -1.0, Cpd105: -1.0, Cpd74: -1.0, Cpd1138: 1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGS9m')
reaction.name = 'folylpolyglutamate synthetase (10fthf), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1137: -1.0, Cpd63: -1.0, Cpd200: -1.0, Cpd1139: 1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FPGSm')
reaction.name = 'folylpolyglutamate synthetase, mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -4.0, Cpd200: -1.0, Cpd1140: -1.0, Cpd7: 1.0, Cpd12: 3.0, Cpd1122: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('FRTT')
reaction.name = 'farnesyltranstransferase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd880: -1.0, Cpd1141: -1.0, Cpd151: 1.0, Cpd1142: 1.0})
reaction.gene_reaction_rule = '( GGPS1 )'
model.add_reactions([reaction])
reaction = Reaction('FRUt1r')
reaction.name = 'D-fructose transport in via uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1143: -1.0, Cpd1144: 1.0})
reaction.gene_reaction_rule = '( SLC2A2 or SLC2A5 or SLC2A7 or SLC2A8 )'
model.add_reactions([reaction])
reaction = Reaction('FRUt4')
reaction.name = 'D-fructose transport  via sodium cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1143: -1.0, Cpd41: 1.0, Cpd1144: 1.0})
reaction.gene_reaction_rule = '( SLC5A10 or SLC5A9 )'
model.add_reactions([reaction])
reaction = Reaction('FTCD')
reaction.name = 'formimidoyltransferase cyclodeaminase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd1145: -1.0, Cpd6: 1.0, Cpd1146: 1.0})
reaction.gene_reaction_rule = '( FTCD )'
model.add_reactions([reaction])
reaction = Reaction('FTHFCL')
reaction.name = '5-formethyltetrahydrofolate cyclo-ligase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd113: -1.0, Cpd107: 1.0, Cpd108: 1.0, Cpd1146: 1.0})
reaction.gene_reaction_rule = '( MTHFSL )'
model.add_reactions([reaction])
reaction = Reaction('FTHFDH')
reaction.name = 'formyltetrahydrofolate dehydrogenase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd375: -1.0, Cpd2: -1.0, Cpd25: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd60: 1.0, Cpd377: 1.0})
reaction.gene_reaction_rule = '( ALDH1L1 or ALDH1L2 )'
model.add_reactions([reaction])
reaction = Reaction('FTHFL')
reaction.name = 'Formate-tetrahydrofolate ligase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd377: -1.0, Cpd699: -1.0, Cpd375: 1.0, Cpd107: 1.0, Cpd108: 1.0})
reaction.gene_reaction_rule = '( MTHFD1 )'
model.add_reactions([reaction])
reaction = Reaction('FTHFLm')
reaction.name = 'formate-tetrahydrofolate ligase, mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1147: -1.0, Cpd1140: -1.0, Cpd1134: 1.0, Cpd202: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( MTHFD1 or MTHFD1L )'
model.add_reactions([reaction])
reaction = Reaction('FUCASE2e')
reaction.name = 'alpha-fucosidase, extracellular'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1148: -1.0, Cpd1149: 1.0, Cpd1150: 1.0})
reaction.gene_reaction_rule = '( FUCA2 )'
model.add_reactions([reaction])
reaction = Reaction('FUCASE2ly')
reaction.name = 'alpha-fucosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1151: -1.0, Cpd1152: 1.0, Cpd1153: 1.0})
reaction.gene_reaction_rule = '( FUCA1 )'
model.add_reactions([reaction])
reaction = Reaction('FUCASEe')
reaction.name = 'alpha-fucosidase, extracellular'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1154: -1.0, Cpd1149: 1.0, Cpd1155: 1.0})
reaction.gene_reaction_rule = '( FUCA2 )'
model.add_reactions([reaction])
reaction = Reaction('FUCASEly')
reaction.name = 'alpha-fucosidase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1156: -1.0, Cpd1152: 1.0, Cpd1157: 1.0})
reaction.gene_reaction_rule = '( FUCA1 )'
model.add_reactions([reaction])
reaction = Reaction('FUM')
reaction.name = 'fumarase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd313: -1.0, Cpd381: 1.0})
reaction.gene_reaction_rule = '( FH1 )'
model.add_reactions([reaction])
reaction = Reaction('FUMAC')
reaction.name = 'fumarylacetoacetase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1158: -1.0, Cpd33: 1.0, Cpd148: 1.0, Cpd313: 1.0})
reaction.gene_reaction_rule = '( FAH )'
model.add_reactions([reaction])
reaction = Reaction('FUMm')
reaction.name = 'fumarase, mitochondrial'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1159: -1.0, Cpd382: 1.0})
reaction.gene_reaction_rule = '( FH1 )'
model.add_reactions([reaction])
reaction = Reaction('FUMSO3tm')
reaction.name = 'Fumarate:sulfite antiport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1160: -1.0, Cpd313: -1.0, Cpd1161: 1.0, Cpd1159: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('FUMSO4tm')
reaction.name = 'Fumarate:sulfate antiport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd313: -1.0, Cpd1162: -1.0, Cpd1159: 1.0, Cpd1163: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('FUMtm')
reaction.name = 'fumarate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd204: -1.0, Cpd313: -1.0, Cpd108: 1.0, Cpd1159: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('FUMTSULtm')
reaction.name = 'Fumarate:thiosulfate antiport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd313: -1.0, Cpd1164: -1.0, Cpd1159: 1.0, Cpd1165: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('FUT11g')
reaction.name = 'Galactoside 2-alpha-L-fucosyltransferase 1'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd507: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1166: 1.0})
reaction.gene_reaction_rule = '( FUT1 )'
model.add_reactions([reaction])
reaction = Reaction('FUT12g')
reaction.name = 'Galactoside 2-alpha-L-fucosyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1167: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd154: 1.0, Cpd1002: 1.0})
reaction.gene_reaction_rule = '( FUT1 )'
model.add_reactions([reaction])
reaction = Reaction('FUT14g')
reaction.name = 'Galactoside 2-alpha-L-fucosyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd508: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd158: 1.0, Cpd1002: 1.0})
reaction.gene_reaction_rule = '( FUT1 )'
model.add_reactions([reaction])
reaction = Reaction('FUT15g')
reaction.name = 'Galactoside 2-alpha-L-fucosyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd513: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd161: 1.0, Cpd1002: 1.0})
reaction.gene_reaction_rule = '( FUT1 )'
model.add_reactions([reaction])
reaction = Reaction('FUT16g')
reaction.name = 'Galactoside 2-alpha-L-fucosyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd510: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd163: 1.0, Cpd1002: 1.0})
reaction.gene_reaction_rule = '( FUT1 )'
model.add_reactions([reaction])
reaction = Reaction('FUT17g')
reaction.name = 'Galactoside 2-alpha-L-fucosyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd514: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd165: 1.0, Cpd1002: 1.0})
reaction.gene_reaction_rule = '( FUT1 )'
model.add_reactions([reaction])
reaction = Reaction('FUT18g')
reaction.name = 'Galactoside 2-alpha-L-fucosyltransferase 1'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd516: -1.0, Cpd999: -2.0, Cpd140: 2.0, Cpd168: 1.0, Cpd1002: 2.0})
reaction.gene_reaction_rule = '( FUT1 )'
model.add_reactions([reaction])
reaction = Reaction('FUT910g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd999: -1.0, Cpd1168: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd168: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT911g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd999: -1.0, Cpd168: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1169: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT91g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd508: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1170: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT92g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd999: -1.0, Cpd1171: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1172: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT93g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd158: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1173: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT94g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd163: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1174: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT95g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd999: -1.0, Cpd1174: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1175: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT96g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd510: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1176: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT97g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd999: -1.0, Cpd1176: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1177: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT98g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd510: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1178: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('FUT99g')
reaction.name = 'Alpha-(1,3)-fucosyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd517: -1.0, Cpd999: -1.0, Cpd140: 1.0, Cpd1002: 1.0, Cpd1168: 1.0})
reaction.gene_reaction_rule = '( FUT9 )'
model.add_reactions([reaction])
reaction = Reaction('G13MT_L')
reaction.name = 'Glycolipid 1,3-alpha-D-mannosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd548: -1.0, Cpd549: -0.1, Cpd33: 1.0, Cpd283: 1.0, Cpd1179: 0.1})
reaction.gene_reaction_rule = '( ALG2 )'
model.add_reactions([reaction])
reaction = Reaction('G13MT_U')
reaction.name = 'Glycolipid 1,3-alpha-D-mannosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd548: -1.0, Cpd551: -0.1, Cpd33: 1.0, Cpd283: 1.0, Cpd1180: 0.1})
reaction.gene_reaction_rule = '( ALG2 )'
model.add_reactions([reaction])
reaction = Reaction('G14T10g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1181: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd346: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T11g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1182: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd317: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T12g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1183: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd319: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T13g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1184: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd321: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T14g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1185: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd323: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T15g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1186: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd325: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T16g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1187: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd327: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T17g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1188: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1189: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T18g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -2.0, Cpd729: -1.0, Cpd140: 2.0, Cpd142: 2.0, Cpd1190: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T19g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd330: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd331: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T20g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1191: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd333: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T21g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1192: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1193: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T2g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1190: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd335: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T3g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd330: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd331: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T4g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1191: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd333: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T5g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1192: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1193: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T6g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd337: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd338: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T7g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1194: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd340: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T8g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1195: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd342: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14T9g')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase, Golgi'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1196: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd344: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G14Tg')
reaction.name = 'beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -2.0, Cpd1001: -1.0, Cpd140: 2.0, Cpd142: 2.0, Cpd1197: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or B4GALT3 or B4GALT5 )'
model.add_reactions([reaction])
reaction = Reaction('G3PD1')
reaction.name = 'glycerol-3-phosphate dehydrogenase (NAD)'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1198: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd425: 1.0})
reaction.gene_reaction_rule = '( GPD1 )'
model.add_reactions([reaction])
reaction = Reaction('G3PD2m')
reaction.name = 'glycerol-3-phosphate dehydrogenase (FAD), mitochondrial'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd1198: -1.0, Cpd240: 1.0, Cpd425: 1.0})
reaction.gene_reaction_rule = '( GPD2 )'
model.add_reactions([reaction])
reaction = Reaction('G5SDym')
reaction.name = 'glutamate-5-semialdehyde dehydrogenase (m)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd1199: -1.0, Cpd13: 1.0, Cpd204: 1.0, Cpd1200: 1.0})
reaction.gene_reaction_rule = '( ALDH18A1 )'
model.add_reactions([reaction])
reaction = Reaction('G6PDA')
reaction.name = 'glucosamine-6-phosphate deaminase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd217: -1.0, Cpd6: 1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( GNPDA1 or GNPDA2 )'
model.add_reactions([reaction])
reaction = Reaction('G6PDH1rer')
reaction.name = 'glucose-6-phosphate dehydrogenase, endoplasmic reticulum'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd625: -1.0, Cpd642: -1.0, Cpd565: 1.0, Cpd626: 1.0, Cpd1201: 1.0})
reaction.gene_reaction_rule = '( H6PD )'
model.add_reactions([reaction])
reaction = Reaction('G6PDH2r')
reaction.name = 'glucose 6-phosphate dehydrogenase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1202: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd1203: 1.0})
reaction.gene_reaction_rule = '( G6PD2 )'
model.add_reactions([reaction])
reaction = Reaction('G6PDH2rer')
reaction.name = 'glucose 6-phosphate dehydrogenase, endoplasmic reticulum'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd642: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd1201: 1.0})
reaction.gene_reaction_rule = '( G6PD2 or H6PD )'
model.add_reactions([reaction])
reaction = Reaction('G6PPer')
reaction.name = 'glucose-6-phosphate phosphatase, edoplasmic reticular'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd642: -1.0, Cpd641: 1.0, Cpd915: 1.0})
reaction.gene_reaction_rule = '( G6PC or G6PC2 or G6PC3 )'
model.add_reactions([reaction])
reaction = Reaction('G6Pter')
reaction.name = 'glucose 6-phosphate endoplasmic reticular transport via diffusion'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1202: -1.0, Cpd642: 1.0})
reaction.gene_reaction_rule = '( SLC37A4 )'
model.add_reactions([reaction])
reaction = Reaction('GABAVESSEC')
reaction.name = 'GABA secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd173: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd174: 1.0})
reaction.gene_reaction_rule = '( SLC32A1 )'
model.add_reactions([reaction])
reaction = Reaction('GACMTRc')
reaction.name = 'guanidinoacetate N-methyltransferase (c)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd1204: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd696: 1.0})
reaction.gene_reaction_rule = '( GAMT )'
model.add_reactions([reaction])
reaction = Reaction('GAL3ST11')
reaction.name = 'Galactosylceramide sulfotransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1205: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd1208: 1.0})
reaction.gene_reaction_rule = '( GAL3ST1 )'
model.add_reactions([reaction])
reaction = Reaction('GAL3ST12')
reaction.name = 'Galactosylceramide sulfotransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1209: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1210: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( GAL3ST1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE10ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1211: -1.0, Cpd1212: 1.0, Cpd1213: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE11ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1214: -1.0, Cpd1212: 1.0, Cpd1215: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE12ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1216: -1.0, Cpd1212: 1.0, Cpd1217: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE13ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1218: -1.0, Cpd1212: 1.0, Cpd1219: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE14ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1220: -1.0, Cpd1212: 1.0, Cpd1221: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE15ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1222: -1.0, Cpd1212: 1.0, Cpd1223: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE16ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1224: -1.0, Cpd1212: 2.0, Cpd1225: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE17ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1226: -1.0, Cpd1212: 1.0, Cpd1227: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE18ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1228: -1.0, Cpd1212: 1.0, Cpd1229: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE19ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1230: -1.0, Cpd1212: 1.0, Cpd1231: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE1ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1232: -1.0, Cpd374: 1.0, Cpd1212: 2.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE20ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1233: -1.0, Cpd1212: 2.0, Cpd1234: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE3ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1235: -1.0, Cpd1212: 2.0, Cpd1236: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE4ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1237: -1.0, Cpd1212: 1.0, Cpd1238: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE5ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1239: -1.0, Cpd1212: 1.0, Cpd1240: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE6ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1241: -1.0, Cpd1212: 1.0, Cpd1242: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE7ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1243: -1.0, Cpd1212: 1.0, Cpd1244: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE8ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1245: -1.0, Cpd1212: 1.0, Cpd1246: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALASE9ly')
reaction.name = 'beta-galactosidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1247: -1.0, Cpd1212: 1.0, Cpd1248: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GALC')
reaction.name = 'Galactocerebrosidase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd474: -1.0, Cpd476: 1.0, Cpd1212: 1.0})
reaction.gene_reaction_rule = '( GALC )'
model.add_reactions([reaction])
reaction = Reaction('GALGT1')
reaction.name = 'Beta-1,4 N-acetylgalactosaminyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd138: -1.0, Cpd155: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd499: 1.0})
reaction.gene_reaction_rule = '( B4GALNT1 )'
model.add_reactions([reaction])
reaction = Reaction('GALGT2')
reaction.name = 'Beta-1,4 N-acetylgalactosaminyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd1249: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd501: 1.0})
reaction.gene_reaction_rule = '( B4GALNT1 )'
model.add_reactions([reaction])
reaction = Reaction('GALGT3')
reaction.name = 'Beta-1,4 N-acetylgalactosaminyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd1250: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd503: 1.0})
reaction.gene_reaction_rule = '( B4GALNT1 )'
model.add_reactions([reaction])
reaction = Reaction('GALGT4')
reaction.name = 'Beta-1,4 N-acetylgalactosaminyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd1251: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd505: 1.0})
reaction.gene_reaction_rule = '( B4GALNT1 )'
model.add_reactions([reaction])
reaction = Reaction('GALK')
reaction.name = 'galactokinase'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1252: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1253: 1.0})
reaction.gene_reaction_rule = '( GALK1 or GALK2 )'
model.add_reactions([reaction])
reaction = Reaction('GALNACT1g')
reaction.name = 'UDPGal-chondroitin acetylgalactosaminyltransferase I, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd1254: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1255: 1.0})
reaction.gene_reaction_rule = '( CSGALNACT1 or CSGALNACT2 )'
model.add_reactions([reaction])
reaction = Reaction('GALNACT2g')
reaction.name = 'uridine diphosphoacetylgalactosamine-chondroitin acetylgalactosaminyltransferase II, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1256: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1257: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHSY1 or CHSY3 or CSGALNACT1 or CSGALNACT2 )'
model.add_reactions([reaction])
reaction = Reaction('GALNACT3g')
reaction.name = 'uridine diphosphoacetylgalactosamine-chondroitin acetylgalactosaminyltransferase II, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1258: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1259: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHSY1 or CHSY3 or CSGALNACT1 or CSGALNACT2 )'
model.add_reactions([reaction])
reaction = Reaction('GALNACT4g')
reaction.name = 'uridine diphosphoacetylgalactosamine-chondroitin acetylgalactosaminyltransferase II, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1260: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1261: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHSY1 or CHSY3 or CSGALNACT1 or CSGALNACT2 )'
model.add_reactions([reaction])
reaction = Reaction('GALNACT5g')
reaction.name = 'uridine diphosphoacetylgalactosamine-chondroitin acetylgalactosaminyltransferase II, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1262: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1263: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHSY1 or CHSY3 or CSGALNACT1 or CSGALNACT2 )'
model.add_reactions([reaction])
reaction = Reaction('GALNTg')
reaction.name = 'GalNAc transferase, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd1264: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd727: 1.0})
reaction.gene_reaction_rule = '( GALNT1 or GALNT10 or GALNT11 or GALNT12 or GALNT13 or GALNT14 or GALNT15 or GALNT2 or GALNT3 or GALNT4 or GALNT5 or GALNT6 or GALNT7 or GALNT9 )'
model.add_reactions([reaction])
reaction = Reaction('GALOR')
reaction.name = 'D-Galactose:NADP+ 1-oxidoreductase'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd1252: -1.0, Cpd25: 1.0, Cpd1265: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 )'
model.add_reactions([reaction])
reaction = Reaction('GALT')
reaction.name = 'galactose-1-phosphate uridylyltransferase'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd749: -1.0, Cpd1253: -1.0, Cpd135: 1.0, Cpd151: 1.0})
reaction.gene_reaction_rule = '( GALT )'
model.add_reactions([reaction])
reaction = Reaction('GALt1r')
reaction.name = 'galactose transport (uniport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1266: -1.0, Cpd1252: 1.0})
reaction.gene_reaction_rule = '( SLC2A1 or SLC2A10 or SLC2A2 or SLC2A3 or SLC2A8 )'
model.add_reactions([reaction])
reaction = Reaction('GALt2_2')
reaction.name = 'D-galactose transport  via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -2.0, Cpd1266: -1.0, Cpd33: 2.0, Cpd1252: 1.0})
reaction.gene_reaction_rule = '( SLC5A1 )'
model.add_reactions([reaction])
reaction = Reaction('GALT2g')
reaction.name = 'UDP-D-galactose:galactosylxylose galactosyltransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1267: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1268: 1.0})
reaction.gene_reaction_rule = '( B3GALT6 )'
model.add_reactions([reaction])
reaction = Reaction('GALt4')
reaction.name = 'galactose transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1266: -1.0, Cpd41: 1.0, Cpd1252: 1.0})
reaction.gene_reaction_rule = '( SLC5A10 or SLC5A2 or SLC5A9 )'
model.add_reactions([reaction])
reaction = Reaction('GALTg')
reaction.name = 'UDP-D-galactose:D-xylose galactosyltransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1269: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1267: 1.0})
reaction.gene_reaction_rule = '( B4GALT7 )'
model.add_reactions([reaction])
reaction = Reaction('GALU')
reaction.name = 'UTP-glucose-1-phosphate uridylyltransferase'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd293: -1.0, Cpd749: -1.0, Cpd151: 1.0, Cpd1270: 1.0})
reaction.gene_reaction_rule = '( UGP2 )'
model.add_reactions([reaction])
reaction = Reaction('GAMt1r')
reaction.name = 'glucosamine transport (uniport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1271: -1.0, Cpd1272: 1.0})
reaction.gene_reaction_rule = '( SLC2A1 or SLC2A2 or SLC2A4 )'
model.add_reactions([reaction])
reaction = Reaction('GAMYe')
reaction.name = 'glucoamylase, extracellular (glygn5 -> malt)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1273: -1.0, Cpd1274: 1.0, Cpd1275: 1.0})
reaction.gene_reaction_rule = '( MGAM )'
model.add_reactions([reaction])
reaction = Reaction('GAO1')
reaction.name = 'Ganglioside O-acetylation'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd1276: -1.0, Cpd78: 1.0, Cpd1277: 1.0})
reaction.gene_reaction_rule = '( SLC33A1 )'
model.add_reactions([reaction])
reaction = Reaction('GAO1g')
reaction.name = 'Ganglioside O-acetylation'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd205: -1.0, Cpd1250: -1.0, Cpd1278: 1.0, Cpd1279: 1.0})
reaction.gene_reaction_rule = '( SLC33A1 )'
model.add_reactions([reaction])
reaction = Reaction('GAO2')
reaction.name = 'Ganglioside O-acetylation'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd1280: -1.0, Cpd78: 1.0, Cpd1281: 1.0})
reaction.gene_reaction_rule = '( SLC33A1 )'
model.add_reactions([reaction])
reaction = Reaction('GAO2g')
reaction.name = 'Ganglioside O-acetylation'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd205: -1.0, Cpd1251: -1.0, Cpd1278: 1.0, Cpd1282: 1.0})
reaction.gene_reaction_rule = '( SLC33A1 )'
model.add_reactions([reaction])
reaction = Reaction('GAPD')
reaction.name = 'glyceraldehyde-3-phosphate dehydrogenase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd32: -1.0, Cpd949: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd272: 1.0})
reaction.gene_reaction_rule = '( GAPDHS or GM15294 )'
model.add_reactions([reaction])
reaction = Reaction('GARFT')
reaction.name = 'phosphoribosylglycinamide formyltransferase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd375: -1.0, Cpd1283: -1.0, Cpd33: 1.0, Cpd377: 1.0, Cpd1284: 1.0})
reaction.gene_reaction_rule = '( GART )'
model.add_reactions([reaction])
reaction = Reaction('GASNASE2ly')
reaction.name = 'glycosylasparaginase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1285: -1.0, Cpd1286: 1.0, Cpd1287: 1.0})
reaction.gene_reaction_rule = '( AGA )'
model.add_reactions([reaction])
reaction = Reaction('GASNASE3ly')
reaction.name = 'glycosylasparaginase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1153: -1.0, Cpd1286: 1.0, Cpd1288: 1.0})
reaction.gene_reaction_rule = '( AGA )'
model.add_reactions([reaction])
reaction = Reaction('GASNASEly')
reaction.name = 'glycosylasparaginase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1157: -1.0, Cpd1286: 1.0, Cpd1289: 1.0})
reaction.gene_reaction_rule = '( AGA )'
model.add_reactions([reaction])
reaction = Reaction('GBA')
reaction.name = 'Glucosylceramidase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd666: -1.0, Cpd1290: 1.0, Cpd662: 1.0})
reaction.gene_reaction_rule = '( GBA )'
model.add_reactions([reaction])
reaction = Reaction('GBAl')
reaction.name = 'Glucosylceramidase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1291: -1.0, Cpd476: 1.0, Cpd1292: 1.0})
reaction.gene_reaction_rule = '( GBA )'
model.add_reactions([reaction])
reaction = Reaction('GBGT1')
reaction.name = 'globoside alpha-1,3-N-acetylgalactosaminyltransferase 1'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd155: -1.0, Cpd498: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1293: 1.0})
reaction.gene_reaction_rule = '( GBGT1 )'
model.add_reactions([reaction])
reaction = Reaction('GCALDD')
reaction.name = 'Glycolaldehyde dehydrogenase'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1106: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd1294: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH1A3 or ALDH3A1 or ALDH3A2 or ALDH3B1 or ALDH3B3 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('GCALDDm')
reaction.name = 'Glycolaldehyde dehydrogenase, mitochondrial'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd1295: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd1296: 1.0})
reaction.gene_reaction_rule = '( ALDH18A1 or ALDH1B1 or ALDH2 or ALDH4A1 or ALDH5A1 or ALDH6A1 )'
model.add_reactions([reaction])
reaction = Reaction('GCC2am')
reaction.name = 'glycine-cleavage complex (lipoamide), mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd364: -1.0, Cpd1297: -1.0, Cpd45: 1.0, Cpd1298: 1.0})
reaction.gene_reaction_rule = '( AMT or DLD or GCSH or GLDC )'
model.add_reactions([reaction])
reaction = Reaction('GCC2bim')
reaction.name = 'glycine-cleavage system (lipoamide) irreversible, mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1140: -1.0, Cpd1298: -1.0, Cpd482: 1.0, Cpd1299: 1.0, Cpd1300: 1.0})
reaction.gene_reaction_rule = '( AMT or DLD or GCSH or GLDC )'
model.add_reactions([reaction])
reaction = Reaction('GCC2cm')
reaction.name = 'glycine-cleavage complex (lipoamide), mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1299: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd1297: 1.0})
reaction.gene_reaction_rule = '( AMT or DLD or GCSH or GLDC )'
model.add_reactions([reaction])
reaction = Reaction('GCCam')
reaction.name = 'glycine-cleavage complex (lipoylprotein), mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd364: -1.0, Cpd1301: -1.0, Cpd45: 1.0, Cpd1302: 1.0})
reaction.gene_reaction_rule = '( AMT or DLD or GCSH or GLDC )'
model.add_reactions([reaction])
reaction = Reaction('GCCbim')
reaction.name = 'glycine-cleavage complex (lipoylprotein) irreversible, mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1140: -1.0, Cpd1302: -1.0, Cpd482: 1.0, Cpd1300: 1.0, Cpd1303: 1.0})
reaction.gene_reaction_rule = '( AMT or DLD or GCSH or GLDC )'
model.add_reactions([reaction])
reaction = Reaction('GCCcm')
reaction.name = 'glycine-cleavage complex (lipoylprotein), mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1303: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd1301: 1.0})
reaction.gene_reaction_rule = '( AMT or DLD or GCSH or GLDC )'
model.add_reactions([reaction])
reaction = Reaction('GCHOLAt')
reaction.name = 'glycocholate transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1304: -1.0, Cpd38: 1.0, Cpd1305: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('GCHOLAt2')
reaction.name = 'glycocholate transport via sodium cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd1304: -1.0, Cpd41: 2.0, Cpd1305: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 or SLC10A2 )'
model.add_reactions([reaction])
reaction = Reaction('GCHOLAt3')
reaction.name = 'ABC bile acid transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1305: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1304: 1.0})
reaction.gene_reaction_rule = '( ABCB11 or ABCC3 )'
model.add_reactions([reaction])
reaction = Reaction('GCNTg')
reaction.name = 'N-acetyllactosaminide beta-1,6-N-acetylglucosaminyl-transferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd510: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd515: 1.0})
reaction.gene_reaction_rule = '( GCNT2 )'
model.add_reactions([reaction])
reaction = Reaction('GDPFUCtg')
reaction.name = 'GDPFuc Golgi transport via CMP antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd998: -1.0, Cpd1306: -1.0, Cpd999: 1.0, Cpd1307: 1.0})
reaction.gene_reaction_rule = '( SLC35C1 )'
model.add_reactions([reaction])
reaction = Reaction('GF6PTA')
reaction.name = 'glutamine-fructose-6-phosphate transaminase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd1108: -1.0, Cpd74: 1.0, Cpd217: 1.0})
reaction.gene_reaction_rule = '( GFPT1 or GFPT2 )'
model.add_reactions([reaction])
reaction = Reaction('GFUCS')
reaction.name = 'GDP-L-fucose synthase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd1308: -1.0, Cpd25: 1.0, Cpd998: 1.0})
reaction.gene_reaction_rule = '( TSTA3 )'
model.add_reactions([reaction])
reaction = Reaction('GGH_10FTHF5GLUe')
reaction.name = 'Gamma-glutamyl hydrolase (10FTHF5GLU), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -4.0, Cpd1309: -1.0, Cpd759: 4.0, Cpd1310: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_10FTHF5GLUl')
reaction.name = 'Gamma-glutamyl hydrolase (10FTHF5GLU), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1311: -1.0, Cpd86: -4.0, Cpd1312: 1.0, Cpd1313: 4.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_10FTHF6GLUe')
reaction.name = 'Gamma-glutamyl hydrolase (10FTHF6GLU), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1314: -1.0, Cpd759: 1.0, Cpd1309: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_10FTHF6GLUl')
reaction.name = 'Gamma-glutamyl hydrolase (10FTHF6GLU), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1315: -1.0, Cpd86: -1.0, Cpd1311: 1.0, Cpd1313: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_10FTHF7GLUe')
reaction.name = 'Gamma-glutamyl hydrolase (10FTHF7GLU), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1316: -1.0, Cpd759: 1.0, Cpd1314: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_10FTHF7GLUl')
reaction.name = 'Gamma-glutamyl hydrolase (10FTHF7GLU), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1317: -1.0, Cpd86: -1.0, Cpd1315: 1.0, Cpd1313: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_5DHFe')
reaction.name = 'Gamma-glutamyl hydrolase (5DHF), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -4.0, Cpd1318: -1.0, Cpd759: 4.0, Cpd1319: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_5DHFl')
reaction.name = 'Gamma-glutamyl hydrolase (5DHF), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -4.0, Cpd1320: -1.0, Cpd1321: 1.0, Cpd1313: 4.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_5THFe')
reaction.name = 'Gamma-glutamyl hydrolase (5THF), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -4.0, Cpd1322: -1.0, Cpd759: 4.0, Cpd1323: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_5THFl')
reaction.name = 'Gamma-glutamyl hydrolase (5THF), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -4.0, Cpd1324: -1.0, Cpd1313: 4.0, Cpd1325: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_6DHFe')
reaction.name = 'Gamma-glutamyl hydrolase (6DHF), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1326: -1.0, Cpd759: 1.0, Cpd1318: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_6DHFl')
reaction.name = 'Gamma-glutamyl hydrolase (6DHF), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1327: -1.0, Cpd1320: 1.0, Cpd1313: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_6THFe')
reaction.name = 'Gamma-glutamyl hydrolase (6THF), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1328: -1.0, Cpd759: 1.0, Cpd1322: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_6THFl')
reaction.name = 'Gamma-glutamyl hydrolase (6THF), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1329: -1.0, Cpd1324: 1.0, Cpd1313: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_7DHFe')
reaction.name = 'Gamma-glutamyl hydrolase (7DHF), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1330: -1.0, Cpd759: 1.0, Cpd1326: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_7DHFl')
reaction.name = 'Gamma-glutamyl hydrolase (7DHF), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1331: -1.0, Cpd1327: 1.0, Cpd1313: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_7THFe')
reaction.name = 'Gamma-glutamyl hydrolase (7THF), extracellular'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1332: -1.0, Cpd759: 1.0, Cpd1328: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGH_7THFl')
reaction.name = 'Gamma-glutamyl hydrolase (7THF), lysosomal'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1333: -1.0, Cpd1329: 1.0, Cpd1313: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('GGLUCT')
reaction.name = 'gamma-glutamylcyclotransferase'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1334: -1.0, Cpd394: 1.0, Cpd1335: 1.0})
reaction.gene_reaction_rule = '( GGCT )'
model.add_reactions([reaction])
reaction = Reaction('GGNG')
reaction.name = 'glycogenin self-glucosylation'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1270: -8.0, Cpd1336: -1.0, Cpd33: 8.0, Cpd137: 8.0, Cpd1337: 1.0})
reaction.gene_reaction_rule = '( GYG or GYS1 or GYS2 )'
model.add_reactions([reaction])
reaction = Reaction('GGT5r')
reaction.name = 'Gamma-glutamyltransferase 5'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1338: -1.0, Cpd1339: 1.0, Cpd1340: 1.0})
reaction.gene_reaction_rule = '( GGT5 )'
model.add_reactions([reaction])
reaction = Reaction('GHMT2r')
reaction.name = 'glycine hydroxymethyltransferase, reversible'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd377: -1.0, Cpd404: -1.0, Cpd2: 1.0, Cpd402: 1.0, Cpd1341: 1.0})
reaction.gene_reaction_rule = '( SHMT1 )'
model.add_reactions([reaction])
reaction = Reaction('GHMT2rm')
reaction.name = 'glycine hydroxymethyltransferase, reversible, mitochondrial'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1140: -1.0, Cpd1342: -1.0, Cpd12: 1.0, Cpd364: 1.0, Cpd1300: 1.0})
reaction.gene_reaction_rule = '( SHMT1 or SHMT2 )'
model.add_reactions([reaction])
reaction = Reaction('GHMT3')
reaction.name = 'glycine hydroxymethyltransferase'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1343: -1.0, Cpd402: 1.0, Cpd1344: 1.0})
reaction.gene_reaction_rule = '( SHMT1 )'
model.add_reactions([reaction])
reaction = Reaction('GHMT3m')
reaction.name = 'glycine hydroxymethyltransferase, mitochondrial'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd1345: -1.0, Cpd364: 1.0, Cpd1346: 1.0})
reaction.gene_reaction_rule = '( SHMT2 )'
model.add_reactions([reaction])
reaction = Reaction('GK1')
reaction.name = 'guanylate kinase (GMP:ATP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1307: -1.0, Cpd107: 1.0, Cpd283: 1.0})
reaction.gene_reaction_rule = '( GUK1 )'
model.add_reactions([reaction])
reaction = Reaction('GK1m')
reaction.name = 'guanylate kinase (GMP:ATP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1347: -1.0, Cpd202: 1.0, Cpd285: 1.0})
reaction.gene_reaction_rule = '( GUK1 )'
model.add_reactions([reaction])
reaction = Reaction('GLACO')
reaction.name = 'D-Glucuronolactone:NAD+ oxidoreductase'
reaction.subsystem = 'Vitamin C metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2.0, Cpd32: -1.0, Cpd1348: -1.0, Cpd33: 3.0, Cpd35: 1.0, Cpd1349: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH1A3 or ALDH3A1 or ALDH3A2 or ALDH3B1 or ALDH3B3 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLACOm')
reaction.name = 'D-Glucuronolactone:NAD+ oxidoreductase, mitochondrial'
reaction.subsystem = 'Vitamin C metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -2.0, Cpd44: -1.0, Cpd1350: -1.0, Cpd7: 3.0, Cpd47: 1.0, Cpd1351: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLAl')
reaction.name = 'galactosidase, alpha'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1352: -1.0, Cpd474: 1.0, Cpd1212: 1.0})
reaction.gene_reaction_rule = '( GLA )'
model.add_reactions([reaction])
reaction = Reaction('GLB1')
reaction.name = 'Beta-galactosidase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1353: -1.0, Cpd1212: 1.0, Cpd1291: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('GLBRAN')
reaction.name = '1,4-alpha-glucan branching enzyme (glygn1 -> glygn2)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1354: -1.0, Cpd1355: 1.0})
reaction.gene_reaction_rule = '( GBE1 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAASE1ly')
reaction.name = 'beta-glucuronidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1356: -1.0, Cpd1357: 1.0, Cpd1358: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('GLCAASE4ly')
reaction.name = 'beta-glucuronidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1359: -1.0, Cpd1357: 1.0, Cpd1360: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('GLCAASE5ly')
reaction.name = 'beta-glucuronidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1361: -1.0, Cpd1357: 1.0, Cpd1362: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('GLCAASE6ly')
reaction.name = 'beta-glucuronidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1363: -1.0, Cpd1357: 1.0, Cpd1364: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('GLCAASE7ly')
reaction.name = 'beta-glucuronidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1365: -1.0, Cpd1357: 1.0, Cpd1366: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('GLCAASE8ly')
reaction.name = 'beta-glucuronidase, lysosomal'
reaction.subsystem = 'Hyaluronan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1367: -1.0, Cpd1357: 1.0, Cpd1368: 1.0})
reaction.gene_reaction_rule = '( GUSB or HYAL3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAASE9ly')
reaction.name = 'beta-glucuronidase, lysosomal'
reaction.subsystem = 'Hyaluronan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1369: -1.0, Cpd373: 1.0, Cpd1357: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('GLCAE2g')
reaction.name = 'heparosan-N-sulfate-glucuronate 5-epimerase'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1370: -1.0, Cpd1371: 1.0})
reaction.gene_reaction_rule = '( GLCE )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT2g')
reaction.name = 'chondroitin glucuronyltransferase II, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1372: -1.0, Cpd1373: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1256: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHPF2 or CHSY1 or CHSY3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT3g')
reaction.name = 'chondroitin glucuronyltransferase II, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1373: -1.0, Cpd1374: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1258: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHPF2 or CHSY1 or CHSY3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT4g')
reaction.name = 'chondroitin glucuronyltransferase II, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1373: -1.0, Cpd1375: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1260: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHPF2 or CHSY1 or CHSY3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT5g')
reaction.name = 'chondroitin glucuronyltransferase II, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1373: -1.0, Cpd1376: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1262: 1.0})
reaction.gene_reaction_rule = '( CHPF or CHPF2 or CHSY1 or CHSY3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT6g')
reaction.name = 'heparan glucuronyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1373: -1.0, Cpd1377: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1378: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT7g')
reaction.name = 'heparan glucuronyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1373: -1.0, Cpd1379: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1380: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT8g')
reaction.name = 'heparan glucuronyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1373: -1.0, Cpd1381: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1382: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 )'
model.add_reactions([reaction])
reaction = Reaction('GLCAT9g')
reaction.name = 'heparan glucuronyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1373: -1.0, Cpd1383: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1384: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 )'
model.add_reactions([reaction])
reaction = Reaction('GLCATg')
reaction.name = 'UDP-D-glucuronate:galactosylgalactosylxylose glucuronosyltransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1268: -1.0, Cpd1373: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1254: 1.0})
reaction.gene_reaction_rule = '( B3GAT1 or B3GAT2 or B3GAT3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACASE1ly')
reaction.name = 'alpha-N-acetylglucosaminidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1385: -1.0, Cpd373: 1.0, Cpd1386: 1.0})
reaction.gene_reaction_rule = '( NAGLU )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACASE2ly')
reaction.name = 'alpha-N-acetylglucosaminidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1387: -1.0, Cpd373: 1.0, Cpd1356: 1.0})
reaction.gene_reaction_rule = '( NAGLU )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACASE3ly')
reaction.name = 'alpha-N-acetylglucosaminidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1388: -1.0, Cpd373: 1.0, Cpd1389: 1.0})
reaction.gene_reaction_rule = '( NAGLU )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACASE4ly')
reaction.name = 'alpha-N-acetylglucosaminidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1390: -1.0, Cpd373: 1.0, Cpd1391: 1.0})
reaction.gene_reaction_rule = '( NAGLU )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACASE5ly')
reaction.name = 'alpha-N-acetylglucosaminidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1392: -1.0, Cpd373: 1.0, Cpd1393: 1.0})
reaction.gene_reaction_rule = '( NAGLU )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACDASg')
reaction.name = 'GlcNAc N-deacetylase / N-sulfotransferase'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -4.0, Cpd1206: -4.0, Cpd1394: -1.0, Cpd140: 4.0, Cpd1395: 4.0, Cpd1207: 4.0, Cpd1370: 1.0})
reaction.gene_reaction_rule = '( NDST1 or NDST2 or NDST3 or NDST4 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACPT_L')
reaction.name = 'UDP-GlcNAc:dolichol-phosphate GlcNAc phosphotransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd930: -0.1, Cpd1396: -1.0, Cpd1397: 0.1, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( DPAGT1 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACPT_U')
reaction.name = 'UDP-GlcNAc:dolichol-phosphate GlcNAc phosphotransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd930: -0.1, Cpd1396: -1.0, Cpd1398: 1.0, Cpd1399: 0.1})
reaction.gene_reaction_rule = '( DPAGT1 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACT1g')
reaction.name = 'alpha-N-acetylglucosaminyltransferase I, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1254: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1377: 1.0})
reaction.gene_reaction_rule = '( EXTL2 or EXTL3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACT2g')
reaction.name = 'alpha-N-acetylglucosaminyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1378: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1379: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 or EXTL1 or EXTL3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACT3g')
reaction.name = 'alpha-N-acetylglucosaminyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1380: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1381: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 or EXTL1 or EXTL3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACT4g')
reaction.name = 'alpha-N-acetylglucosaminyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1382: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1383: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 or EXTL1 or EXTL3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCNACT5g')
reaction.name = 'alpha-N-acetylglucosaminyltransferase II'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1384: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1394: 1.0})
reaction.gene_reaction_rule = '( EXT1 or EXT2 or EXTL1 or EXTL3 )'
model.add_reactions([reaction])
reaction = Reaction('GLCt1r')
reaction.name = 'glucose transport (uniport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd446: -1.0, Cpd1290: 1.0})
reaction.gene_reaction_rule = '( SLC2A1 or SLC2A10 or SLC2A12 or SLC2A2 or SLC2A3 or SLC2A4 or SLC2A6 or SLC2A7 or SLC2A8 or SLC2A9 )'
model.add_reactions([reaction])
reaction = Reaction('GLCt2_2')
reaction.name = 'D-glucose transport in via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -2.0, Cpd446: -1.0, Cpd33: 2.0, Cpd1290: 1.0})
reaction.gene_reaction_rule = '( SLC5A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLCt4')
reaction.name = 'glucose transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd446: -1.0, Cpd41: 1.0, Cpd1290: 1.0})
reaction.gene_reaction_rule = '( SLC5A10 or SLC5A11 or SLC5A12 or SLC5A2 or SLC5A3 or SLC5A9 )'
model.add_reactions([reaction])
reaction = Reaction('GLCtg')
reaction.name = 'glucose transport, Golgi apparatus'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1290: -1.0, Cpd1400: 1.0})
reaction.gene_reaction_rule = '( SLC2A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLCURtly')
reaction.name = 'glucuronate transport into lysososme'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1401: -1.0, Cpd176: 1.0, Cpd1357: 1.0})
reaction.gene_reaction_rule = '( SLC17A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLDBRAN')
reaction.name = 'glycogen debranching enzyme'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1402: -1.0, Cpd1290: 1.0, Cpd1403: 1.0})
reaction.gene_reaction_rule = '( AGL )'
model.add_reactions([reaction])
reaction = Reaction('GLGNS1')
reaction.name = 'glycogen synthase (ggn -> glygn1)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1270: -3.0, Cpd1337: -1.0, Cpd33: 3.0, Cpd137: 3.0, Cpd1354: 1.0})
reaction.gene_reaction_rule = '( GYG or GYS1 or GYS2 )'
model.add_reactions([reaction])
reaction = Reaction('GLNALANaEx')
reaction.name = 'L-alanine/L-glutamine Na-dependent exchange (Gln-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd401: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLNASNNaEx')
reaction.name = 'L-glutamine/L-asparagine Na-dependent exchange (Gln-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd401: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLNCYSNaEx')
reaction.name = 'L-cysteine/L-glutamine Na-dependent exchange (Gln-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd401: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLNS')
reaction.name = 'glutamine synthetase'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd6: -1.0, Cpd105: -1.0, Cpd74: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( GLUL or LGSN )'
model.add_reactions([reaction])
reaction = Reaction('GLNSERNaEx')
reaction.name = 'L-serine/L-glutamine Na-dependent exchange (Gln-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd404: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd405: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLNt4')
reaction.name = 'L-glutamine reversible transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd41: 1.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC38A4 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('GLNTHRNaEx')
reaction.name = 'L-threonine/L-glutamine Na-dependent exchange (Gln-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd408: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLNtN1')
reaction.name = 'Glutamine transport (Na, H coupled)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -2.0, Cpd401: -1.0, Cpd37: 1.0, Cpd41: 2.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( SLC38A3 or SLC38A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLPASE1')
reaction.name = 'glycogen phosphorylase (glygn2 -> dxtrn)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -3.0, Cpd1355: -1.0, Cpd293: 3.0, Cpd1402: 1.0})
reaction.gene_reaction_rule = '( PYGB or PYGL or PYGM )'
model.add_reactions([reaction])
reaction = Reaction('GLPASE2')
reaction.name = 'glycogen phosphorylase (amyls -> glc-D)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -7.0, Cpd1403: -1.0, Cpd1290: 7.0, Cpd1336: 1.0})
reaction.gene_reaction_rule = '( PYGB or PYGL or PYGM )'
model.add_reactions([reaction])
reaction = Reaction('GLU5Km')
reaction.name = 'glutamate 5-kinase (m)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd63: -1.0, Cpd200: -1.0, Cpd202: 1.0, Cpd1199: 1.0})
reaction.gene_reaction_rule = '( ALDH18A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLUCYS')
reaction.name = 'Gamma-glutamylcysteine synthetase'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd74: -1.0, Cpd396: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1404: 1.0})
reaction.gene_reaction_rule = '( GCLC or GCLM )'
model.add_reactions([reaction])
reaction = Reaction('GLUDC')
reaction.name = 'Glutamate Decarboxylase'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd74: -1.0, Cpd60: 1.0, Cpd173: 1.0})
reaction.gene_reaction_rule = '( GAD1 or GAD2 )'
model.add_reactions([reaction])
reaction = Reaction('GLUDxm')
reaction.name = 'glutamate dehydrogenase (NAD) (mitochondrial)'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd63: -1.0, Cpd7: 1.0, Cpd28: 1.0, Cpd47: 1.0, Cpd482: 1.0})
reaction.gene_reaction_rule = '( GLUD1 )'
model.add_reactions([reaction])
reaction = Reaction('GLUDym')
reaction.name = 'glutamate dehydrogenase (NADP), mitochondrial'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd63: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd28: 1.0, Cpd482: 1.0})
reaction.gene_reaction_rule = '( GLUD1 )'
model.add_reactions([reaction])
reaction = Reaction('GluForTx')
reaction.name = 'Glutamate formimidoyltransferase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd377: -1.0, Cpd1405: -1.0, Cpd74: 1.0, Cpd1145: 1.0})
reaction.gene_reaction_rule = '( FTCD )'
model.add_reactions([reaction])
reaction = Reaction('GLUNm')
reaction.name = 'glutaminase (mitochondrial)'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1406: -1.0, Cpd63: 1.0, Cpd482: 1.0})
reaction.gene_reaction_rule = '( GLS or GLS2 )'
model.add_reactions([reaction])
reaction = Reaction('GLUPRT')
reaction.name = 'glutamine phosphoribosyldiphosphate amidotransferase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd301: -1.0, Cpd399: -1.0, Cpd74: 1.0, Cpd151: 1.0, Cpd1407: 1.0})
reaction.gene_reaction_rule = '( PPAT )'
model.add_reactions([reaction])
reaction = Reaction('GLUt2m')
reaction.name = 'L-glutamate reversible transport via proton symport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd74: -1.0, Cpd7: 1.0, Cpd63: 1.0})
reaction.gene_reaction_rule = '( SLC25A12 or SLC25A13 or SLC25A18 or SLC25A22 )'
model.add_reactions([reaction])
reaction = Reaction('GLUt6')
reaction.name = 'Glutamate transport via Na, H symport and K antiport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd39: -3.0, Cpd487: -1.0, Cpd759: -1.0, Cpd33: 1.0, Cpd41: 3.0, Cpd74: 1.0, Cpd489: 1.0})
reaction.gene_reaction_rule = '( SLC1A1 or SLC1A2 or SLC1A3 or SLC1A6 or SLC1A7 )'
model.add_reactions([reaction])
reaction = Reaction('GLUTCOADHm')
reaction.name = 'glutaryl-CoA dehydrogenase (mitochondria)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd46: -1.0, Cpd238: -1.0, Cpd45: 1.0, Cpd70: 1.0, Cpd240: 1.0})
reaction.gene_reaction_rule = '( GCDH )'
model.add_reactions([reaction])
reaction = Reaction('GLUVESSEC')
reaction.name = 'L-glutamate secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd74: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd759: 1.0})
reaction.gene_reaction_rule = '( SLC17A6 or SLC17A7 or SLC17A8 )'
model.add_reactions([reaction])
reaction = Reaction('GLXO1')
reaction.name = 'glyoxylate oxidase'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1408: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd704: 1.0})
reaction.gene_reaction_rule = '( LDHA or LDHAL6B or LDHB or LDHC or LDHD or UEVLD )'
model.add_reactions([reaction])
reaction = Reaction('GLXO2p')
reaction.name = 'glyoxylate oxidase, peroxisomal'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd366: -1.0, Cpd358: 1.0, Cpd247: 1.0, Cpd1409: 1.0})
reaction.gene_reaction_rule = '( HAO1 or HAO2 )'
model.add_reactions([reaction])
reaction = Reaction('GLYAMDTRc')
reaction.name = 'glycine amidinotransferase (c)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd467: -1.0, Cpd252: 1.0, Cpd1204: 1.0})
reaction.gene_reaction_rule = '( GATM )'
model.add_reactions([reaction])
reaction = Reaction('GLYATm')
reaction.name = 'glycine C-acetyltransferase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd364: -1.0, Cpd43: 1.0, Cpd1410: 1.0})
reaction.gene_reaction_rule = '( GCAT )'
model.add_reactions([reaction])
reaction = Reaction('GLYBt4_2_r')
reaction.name = 'Betaine transport (sodium symport) (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd1411: -1.0, Cpd41: 2.0, Cpd552: 1.0})
reaction.gene_reaction_rule = '( SLC6A12 )'
model.add_reactions([reaction])
reaction = Reaction('GLYCLTDy')
reaction.name = 'Glycolate dehydrogenase (NADP)'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd1408: -1.0, Cpd25: 1.0, Cpd1294: 1.0})
reaction.gene_reaction_rule = '( GRHPR )'
model.add_reactions([reaction])
reaction = Reaction('GLYCTO1p')
reaction.name = 'Glycolate oxidase, peroxisome'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd1412: -1.0, Cpd247: 1.0, Cpd366: 1.0})
reaction.gene_reaction_rule = '( HAO1 or HAO2 )'
model.add_reactions([reaction])
reaction = Reaction('GLYK')
reaction.name = 'glycerol kinase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1413: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1198: 1.0})
reaction.gene_reaction_rule = '( GK or GK2 )'
model.add_reactions([reaction])
reaction = Reaction('GLYKm')
reaction.name = 'glycerol kinase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1414: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd1415: 1.0})
reaction.gene_reaction_rule = '( GK or GK2 )'
model.add_reactions([reaction])
reaction = Reaction('GLYOX')
reaction.name = 'hydroxyacylglutathione hydrolase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1416: -1.0, Cpd33: 1.0, Cpd1057: 1.0, Cpd872: 1.0})
reaction.gene_reaction_rule = '( HAGH or HAGHL )'
model.add_reactions([reaction])
reaction = Reaction('GLYOXm')
reaction.name = 'hydroxyacylglutathione hydrolase, mitochondrial'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1417: -1.0, Cpd7: 1.0, Cpd873: 1.0, Cpd1418: 1.0})
reaction.gene_reaction_rule = '( HAGH )'
model.add_reactions([reaction])
reaction = Reaction('GLYt2r')
reaction.name = 'glycine reversible transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd403: -1.0, Cpd33: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 or SLC36A2 )'
model.add_reactions([reaction])
reaction = Reaction('GLYt2rL')
reaction.name = 'glycine reversible transport via proton symport (lysosome)'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd1419: -1.0, Cpd33: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYt4')
reaction.name = 'glycine transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd41: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC38A4 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('GLYt7_211_r')
reaction.name = 'glycine reversible transport via sodium and chloride symport (2:1:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd403: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd402: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A9 )'
model.add_reactions([reaction])
reaction = Reaction('GLYVESSEC')
reaction.name = 'Glycine secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd402: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC32A1 )'
model.add_reactions([reaction])
reaction = Reaction('GMAND')
reaction.name = 'GDP-D-mannose dehydratase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd548: -1.0, Cpd2: 1.0, Cpd1308: 1.0})
reaction.gene_reaction_rule = '( GMDS )'
model.add_reactions([reaction])
reaction = Reaction('GMPR')
reaction.name = 'GMP reductase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd22: -1.0, Cpd1307: -1.0, Cpd6: 1.0, Cpd25: 1.0, Cpd316: 1.0})
reaction.gene_reaction_rule = '( GMPR or GMPR2 )'
model.add_reactions([reaction])
reaction = Reaction('GMPS2')
reaction.name = 'GMP synthase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd399: -1.0, Cpd1420: -1.0, Cpd33: 2.0, Cpd74: 1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1307: 1.0})
reaction.gene_reaction_rule = '( GMPS )'
model.add_reactions([reaction])
reaction = Reaction('GND')
reaction.name = 'phosphogluconate dehydrogenase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1421: -1.0, Cpd22: 1.0, Cpd60: 1.0, Cpd1422: 1.0})
reaction.gene_reaction_rule = '( PGD )'
model.add_reactions([reaction])
reaction = Reaction('GNDer')
reaction.name = 'phosphogluconate dehydrogenase, endoplasmic reticulum'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd1423: -1.0, Cpd587: 1.0, Cpd624: 1.0, Cpd1424: 1.0})
reaction.gene_reaction_rule = '( PGD )'
model.add_reactions([reaction])
reaction = Reaction('GNMT')
reaction.name = 'glycine N-methyltransferase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd402: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd1425: 1.0})
reaction.gene_reaction_rule = '( GNMT )'
model.add_reactions([reaction])
reaction = Reaction('GPAM_hs')
reaction.name = 'glycerol-3-phosphate acyltransferase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd843: -1.0, Cpd1198: -1.0, Cpd78: 1.0, Cpd352: 1.0})
reaction.gene_reaction_rule = '( GPAM or MBOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('GPAMm_hs')
reaction.name = 'glycerol-3-phosphate acyltransferase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1426: -1.0, Cpd1415: -1.0, Cpd43: 1.0, Cpd1427: 1.0})
reaction.gene_reaction_rule = '( GPAM or GPAT2 or GPAT3 or GPAT4 )'
model.add_reactions([reaction])
reaction = Reaction('GPIAT')
reaction.name = 'glucosaminylphosphatidyl inositol acetyltransferase'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd222: -1.0, Cpd590: -1.0, Cpd78: 1.0, Cpd1428: 1.0})
reaction.gene_reaction_rule = '( PIGW )'
model.add_reactions([reaction])
reaction = Reaction('GPIDA2er')
reaction.name = 'glycophosphatidylinositol (GPI) deacylase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1429: -1.0, Cpd565: 1.0, Cpd1430: 1.0, Cpd1431: 1.0})
reaction.gene_reaction_rule = '( PGAP1 )'
model.add_reactions([reaction])
reaction = Reaction('GPIDAer')
reaction.name = 'glycophosphatidylinositol (GPI) deacylase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1432: -1.0, Cpd565: 1.0, Cpd1433: 1.0, Cpd1431: 1.0})
reaction.gene_reaction_rule = '( PGAP1 )'
model.add_reactions([reaction])
reaction = Reaction('GPIMTer_L')
reaction.name = 'GlcN-acylPI mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd1434: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd1435: 1.0})
reaction.gene_reaction_rule = '( PIGM or PIGX )'
model.add_reactions([reaction])
reaction = Reaction('GPIMTer_U')
reaction.name = 'GlcN-acylPI mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd567: -0.1, Cpd1434: -1.0, Cpd565: 1.0, Cpd564: 0.1, Cpd1435: 1.0})
reaction.gene_reaction_rule = '( PIGM or PIGX )'
model.add_reactions([reaction])
reaction = Reaction('GRTT')
reaction.name = 'geranyltranstransferase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd880: -1.0, Cpd881: -1.0, Cpd151: 1.0, Cpd1141: 1.0})
reaction.gene_reaction_rule = '( GGPS1 )'
model.add_reactions([reaction])
reaction = Reaction('GRTTx')
reaction.name = 'geranyltranstransferase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd883: -1.0, Cpd884: -1.0, Cpd885: 1.0, Cpd1436: 1.0})
reaction.gene_reaction_rule = '( FDPS )'
model.add_reactions([reaction])
reaction = Reaction('GSNt')
reaction.name = 'guanosine facilated transport in cytosol'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1437: -1.0, Cpd1438: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 or SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('GSNt4')
reaction.name = 'guanosine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1437: -1.0, Cpd41: 1.0, Cpd1438: 1.0})
reaction.gene_reaction_rule = '( SLC28A2 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('GSNtl')
reaction.name = 'guanosine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1439: -1.0, Cpd1438: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('GSNtm')
reaction.name = 'guanosine facilated transport in mitochondria'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1438: -1.0, Cpd1440: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 )'
model.add_reactions([reaction])
reaction = Reaction('GTHO')
reaction.name = 'glutathione oxidoreductase'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd1441: -1.0, Cpd25: 1.0, Cpd1057: 2.0})
reaction.gene_reaction_rule = '( GSR )'
model.add_reactions([reaction])
reaction = Reaction('GTHOm')
reaction.name = 'glutathione oxidoreductase'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd1442: -1.0, Cpd13: 1.0, Cpd1418: 2.0})
reaction.gene_reaction_rule = '( GSR )'
model.add_reactions([reaction])
reaction = Reaction('GTHP')
reaction.name = 'glutathione peroxidase'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd1057: -2.0, Cpd2: 2.0, Cpd1441: 1.0})
reaction.gene_reaction_rule = '( GPX1 or GPX2 or GPX4 or PRDX1 or PRDX2 )'
model.add_reactions([reaction])
reaction = Reaction('GTHPe')
reaction.name = 'glutathione peroxidase (e)'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1443: -2.0, Cpd1444: -1.0, Cpd114: 2.0, Cpd1445: 1.0})
reaction.gene_reaction_rule = '( GPX1 or GPX2 or GPX3 or GPX6 or GPX7 )'
model.add_reactions([reaction])
reaction = Reaction('GTHPm')
reaction.name = 'glutathione peroxidase, mitochondria'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd634: -1.0, Cpd1418: -2.0, Cpd12: 2.0, Cpd1442: 1.0})
reaction.gene_reaction_rule = '( GPX1 or GPX4 or PRDX3 )'
model.add_reactions([reaction])
reaction = Reaction('GTHS')
reaction.name = 'glutathione synthetase'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd402: -1.0, Cpd1404: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1057: 1.0})
reaction.gene_reaction_rule = '( GSS )'
model.add_reactions([reaction])
reaction = Reaction('GTMLTe')
reaction.name = 'g-glutamyltransferase (e)'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd1057: -1.0, Cpd444: 1.0, Cpd1334: 1.0})
reaction.gene_reaction_rule = '( GGT1 )'
model.add_reactions([reaction])
reaction = Reaction('GTPCI')
reaction.name = 'GTP cyclohydrolase I'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd282: -1.0, Cpd33: 1.0, Cpd699: 1.0, Cpd1446: 1.0})
reaction.gene_reaction_rule = '( GCH1 )'
model.add_reactions([reaction])
reaction = Reaction('GTPCIn')
reaction.name = 'GTP cyclohydrolase I, nuclear'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd790: -1.0, Cpd763: 1.0, Cpd1447: 1.0, Cpd1448: 1.0})
reaction.gene_reaction_rule = '( GCH1 )'
model.add_reactions([reaction])
reaction = Reaction('GUACYC')
reaction.name = 'guanylate cyclase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd282: -1.0, Cpd668: 1.0, Cpd151: 1.0})
reaction.gene_reaction_rule = '( GUCY1A2 or GUCY1A3 or GUCY1B3 or GUCY2C or GUCY2E or GUCY2F or NPR1 or NPR2 )'
model.add_reactions([reaction])
reaction = Reaction('GUAD')
reaction.name = 'guanine deaminase'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd1449: -1.0, Cpd6: 1.0, Cpd1450: 1.0})
reaction.gene_reaction_rule = '( GDA )'
model.add_reactions([reaction])
reaction = Reaction('GUAPRT')
reaction.name = 'guanine phosphoribosyltransferase'
reaction.subsystem = 'Nucleotide salvage pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd301: -1.0, Cpd1449: -1.0, Cpd151: 1.0, Cpd1307: 1.0})
reaction.gene_reaction_rule = '( HPRT )'
model.add_reactions([reaction])
reaction = Reaction('GUAt')
reaction.name = 'Guanine transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1451: -1.0, Cpd1449: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('H2CO3D')
reaction.name = 'carboxylic acid dissociation'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd60: -1.0, Cpd33: 1.0, Cpd40: 1.0})
reaction.gene_reaction_rule = '( CAR1 or CAR12 or CAR13 or CAR14 or CAR2 or CAR3 or CAR4 or CAR6 or CAR7 or CAR9 )'
model.add_reactions([reaction])
reaction = Reaction('H2CO3Dm')
reaction.name = 'carboxylic acid dissociation'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd45: -1.0, Cpd7: 1.0, Cpd201: 1.0})
reaction.gene_reaction_rule = '( CAR5A or CAR5B )'
model.add_reactions([reaction])
reaction = Reaction('H2ETer')
reaction.name = 'H2 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1435: -1.0, Cpd1452: -1.0, Cpd1453: 1.0, Cpd1454: 1.0})
reaction.gene_reaction_rule = '( PIGN )'
model.add_reactions([reaction])
reaction = Reaction('H2MTer_L')
reaction.name = 'H2 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd1435: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd1455: 1.0})
reaction.gene_reaction_rule = '( PIGV )'
model.add_reactions([reaction])
reaction = Reaction('H2MTer_U')
reaction.name = 'H2 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd567: -0.1, Cpd1435: -1.0, Cpd565: 1.0, Cpd564: 0.1, Cpd1455: 1.0})
reaction.gene_reaction_rule = '( PIGV )'
model.add_reactions([reaction])
reaction = Reaction('H2O2syn')
reaction.name = 'Hydrogen peroxide synthesis (NADPH dependent)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd5: 1.0, Cpd25: 1.0})
reaction.gene_reaction_rule = '( DUOX1 or DUOX2 )'
model.add_reactions([reaction])
reaction = Reaction('H2Ot')
reaction.name = 'H2O transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2: 1.0})
reaction.gene_reaction_rule = '( AQP1 or AQP2 or AQP4 or AQP5 or AQP8 or MIP )'
model.add_reactions([reaction])
reaction = Reaction('H2Otm')
reaction.name = 'H2O transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd12: 1.0})
reaction.gene_reaction_rule = '( AQP8 )'
model.add_reactions([reaction])
reaction = Reaction('H3ETer')
reaction.name = 'H3 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1455: -1.0, Cpd563: 1.0, Cpd1453: 1.0})
reaction.gene_reaction_rule = '( PIGN )'
model.add_reactions([reaction])
reaction = Reaction('H3MTer_L')
reaction.name = 'H3 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd1455: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd1456: 1.0})
reaction.gene_reaction_rule = '( PIGB )'
model.add_reactions([reaction])
reaction = Reaction('H3MTer_U')
reaction.name = 'H3 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd567: -0.1, Cpd1455: -1.0, Cpd565: 1.0, Cpd564: 0.1, Cpd1456: 1.0})
reaction.gene_reaction_rule = '( PIGB )'
model.add_reactions([reaction])
reaction = Reaction('H4ET3er')
reaction.name = 'H4 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1456: -1.0, Cpd1453: 1.0, Cpd1457: 1.0})
reaction.gene_reaction_rule = '( PIGF or PIGO )'
model.add_reactions([reaction])
reaction = Reaction('H4ETer')
reaction.name = 'H4 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1456: -1.0, Cpd566: 1.0, Cpd1453: 1.0})
reaction.gene_reaction_rule = '( PIGN )'
model.add_reactions([reaction])
reaction = Reaction('H5MTer_L')
reaction.name = 'H5 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd1454: -1.0, Cpd563: 1.0, Cpd564: 0.1, Cpd565: 1.0})
reaction.gene_reaction_rule = '( PIGV )'
model.add_reactions([reaction])
reaction = Reaction('H5MTer_U')
reaction.name = 'H5 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd567: -0.1, Cpd1454: -1.0, Cpd563: 1.0, Cpd565: 1.0, Cpd564: 0.1})
reaction.gene_reaction_rule = '( PIGV )'
model.add_reactions([reaction])
reaction = Reaction('H6_ET2er')
reaction.name = 'H6 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1457: -1.0, Cpd1453: 1.0, Cpd1458: 1.0})
reaction.gene_reaction_rule = '( PIGF or PIGG )'
model.add_reactions([reaction])
reaction = Reaction('H6ET3er')
reaction.name = 'H6 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd566: -1.0, Cpd1452: -1.0, Cpd1453: 1.0, Cpd1459: 1.0})
reaction.gene_reaction_rule = '( PIGF or PIGO )'
model.add_reactions([reaction])
reaction = Reaction('H6_ETer')
reaction.name = 'H6 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1457: -1.0, Cpd1453: 1.0, Cpd1459: 1.0})
reaction.gene_reaction_rule = '( PIGN )'
model.add_reactions([reaction])
reaction = Reaction('H6MTer_L')
reaction.name = 'H6 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd566: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd1460: 1.0})
reaction.gene_reaction_rule = '( PIGZ )'
model.add_reactions([reaction])
reaction = Reaction('H6MTer_U')
reaction.name = 'H6 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd566: -1.0, Cpd567: -0.1, Cpd565: 1.0, Cpd564: 0.1, Cpd1460: 1.0})
reaction.gene_reaction_rule = '( PIGZ )'
model.add_reactions([reaction])
reaction = Reaction('H7ET2er')
reaction.name = 'H7 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1459: -1.0, Cpd1453: 1.0, Cpd1461: 1.0})
reaction.gene_reaction_rule = '( PIGF or PIGG )'
model.add_reactions([reaction])
reaction = Reaction('H7_ETer')
reaction.name = 'H7 phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1458: -1.0, Cpd1453: 1.0, Cpd1461: 1.0})
reaction.gene_reaction_rule = '( PIGN )'
model.add_reactions([reaction])
reaction = Reaction('H7MTer_L')
reaction.name = 'H7 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd1459: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd1462: 1.0})
reaction.gene_reaction_rule = '( PIGZ )'
model.add_reactions([reaction])
reaction = Reaction('H7MTer_U')
reaction.name = 'H7 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd567: -0.1, Cpd1459: -1.0, Cpd565: 1.0, Cpd564: 0.1, Cpd1462: 1.0})
reaction.gene_reaction_rule = '( PIGZ )'
model.add_reactions([reaction])
reaction = Reaction('H7_TAer')
reaction.name = 'H7 transamidase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1459: -1.0, Cpd1463: -1.0, Cpd1464: 1.0, Cpd1429: 1.0})
reaction.gene_reaction_rule = '( GPAA1 or PIGK or PIGS or PIGT or PIGU )'
model.add_reactions([reaction])
reaction = Reaction('H8MTer_L')
reaction.name = 'H8 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd1461: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd1465: 1.0})
reaction.gene_reaction_rule = '( PIGZ )'
model.add_reactions([reaction])
reaction = Reaction('H8MTer_U')
reaction.name = 'H8 mannosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd562: -0.1, Cpd1461: -1.0, Cpd564: 0.1, Cpd565: 1.0, Cpd1466: 1.0})
reaction.gene_reaction_rule = '( PIGZ )'
model.add_reactions([reaction])
reaction = Reaction('H8TAer')
reaction.name = 'H8 transamidase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1461: -1.0, Cpd1463: -1.0, Cpd1464: 1.0, Cpd1432: 1.0})
reaction.gene_reaction_rule = '( GPAA1 or PIGK or PIGS or PIGT or PIGU )'
model.add_reactions([reaction])
reaction = Reaction('HACD1m')
reaction.name = '3-hydroxyacyl-CoA dehydrogenase (acetoacetyl-CoA) (mitochondria)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd47: -1.0, Cpd181: -1.0, Cpd44: 1.0, Cpd960: 1.0})
reaction.gene_reaction_rule = '( HADH or HADHA or HADHB or HSD17B10 )'
model.add_reactions([reaction])
reaction = Reaction('HACD1x')
reaction.name = '3-hydroxyacyl-CoA dehydrogenase (acetoacetyl-CoA) (peroxisome)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1065: -1.0, Cpd183: -1.0, Cpd1064: 1.0, Cpd961: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('HACD9m')
reaction.name = '3-hydroxyacyl-CoA dehydrogenase (2-Methylacetoacetyl-CoA), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd963: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd177: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 )'
model.add_reactions([reaction])
reaction = Reaction('HAS1')
reaction.name = 'hyaluronan synthase'
reaction.subsystem = 'Hyaluronan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1396: -1.0, Cpd1467: -1.0, Cpd33: 2.0, Cpd137: 2.0, Cpd1468: 1.0})
reaction.gene_reaction_rule = '( HAS1 or HAS2 or HAS3 )'
model.add_reactions([reaction])
reaction = Reaction('HAS2')
reaction.name = 'hyaluronan synthase'
reaction.subsystem = 'Hyaluronan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1468: -1.0, Cpd1396: -1.0, Cpd1467: -1.0, Cpd33: 2.0, Cpd137: 2.0, Cpd1469: 1.0})
reaction.gene_reaction_rule = '( HAS1 or HAS2 or HAS3 )'
model.add_reactions([reaction])
reaction = Reaction('HBZOPT10m')
reaction.name = 'Hydroxybenzoate Decaprenyltransferase'
reaction.subsystem = 'Ubiquinone synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1470: -1.0, Cpd1471: -1.0, Cpd1472: 1.0, Cpd210: 1.0})
reaction.gene_reaction_rule = '( COQ2 )'
model.add_reactions([reaction])
reaction = Reaction('HCO3_CLt')
reaction.name = 'bicarbonate transport (Cl-/HCO3- exchange)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd700: -1.0, Cpd40: 1.0, Cpd698: 1.0})
reaction.gene_reaction_rule = '( SLC4A1 or SLC4A2 or SLC4A3 or SLC4A9 )'
model.add_reactions([reaction])
reaction = Reaction('HDCAtr')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1094: -1.0, Cpd1010: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('HDCEAtr')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1473: -1.0, Cpd1012: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('HEX1')
reaction.name = 'hexokinase (D-glucose:ATP)'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1290: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1202: 1.0})
reaction.gene_reaction_rule = '( GCK or HK1 or HK2 or HK3 or HKDC1 )'
model.add_reactions([reaction])
reaction = Reaction('HEX10')
reaction.name = 'hexokinase (D-glucosamine:ATP)'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1272: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd217: 1.0})
reaction.gene_reaction_rule = '( GCK or HK1 or HK2 or HK3 or HKDC1 )'
model.add_reactions([reaction])
reaction = Reaction('HEX4')
reaction.name = 'hexokinase (D-mannose:ATP)'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd129: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd233: 1.0})
reaction.gene_reaction_rule = '( GCK or HK1 or HK2 or HK3 or HKDC1 )'
model.add_reactions([reaction])
reaction = Reaction('HEX7')
reaction.name = 'hexokinase (D-fructose:ATP)'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1144: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( GCK or HK1 or HK2 or HK3 or HKDC1 )'
model.add_reactions([reaction])
reaction = Reaction('HEXCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1051: -1.0, Cpd78: 1.0, Cpd1474: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('HEXCCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1475: -1.0, Cpd307: 1.0, Cpd1476: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('HEXCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1474: -1.0, Cpd1475: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('HGNTOR')
reaction.name = 'Homogentisate:oxygen 1,2-oxidoreductase (decyclizing)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd61: -1.0, Cpd33: 1.0, Cpd1477: 1.0})
reaction.gene_reaction_rule = '( HGD )'
model.add_reactions([reaction])
reaction = Reaction('HIBDm')
reaction.name = '3-hydroxyisobutyrate dehydrogenase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1478: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd62: 1.0})
reaction.gene_reaction_rule = '( HIBADH )'
model.add_reactions([reaction])
reaction = Reaction('HISD')
reaction.name = 'histidase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1479: -1.0, Cpd6: 1.0, Cpd1480: 1.0})
reaction.gene_reaction_rule = '( HAL )'
model.add_reactions([reaction])
reaction = Reaction('HISDC')
reaction.name = 'histidine decarboxylase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1479: -1.0, Cpd60: 1.0, Cpd1481: 1.0})
reaction.gene_reaction_rule = '( DDC or HDC )'
model.add_reactions([reaction])
reaction = Reaction('HISt4')
reaction.name = 'L-histidine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1482: -1.0, Cpd41: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC38A2 or SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('HISTASE')
reaction.name = 'Histaminase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1481: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd1483: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 )'
model.add_reactions([reaction])
reaction = Reaction('HISTAtu')
reaction.name = 'Histamine uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1484: -1.0, Cpd1481: 1.0})
reaction.gene_reaction_rule = '( SLC22A2 or SLC22A3 or SLC22A5 )'
model.add_reactions([reaction])
reaction = Reaction('HISTAVESSEC')
reaction.name = 'Histamine secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1481: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1484: 1.0})
reaction.gene_reaction_rule = '( SLC18A1 or SLC18A2 )'
model.add_reactions([reaction])
reaction = Reaction('HIStiDF')
reaction.name = 'L-histidine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC38A4 )'
model.add_reactions([reaction])
reaction = Reaction('HIStN1')
reaction.name = 'Histidine transport (Na, H coupled)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -2.0, Cpd1482: -1.0, Cpd37: 1.0, Cpd41: 2.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC38A3 )'
model.add_reactions([reaction])
reaction = Reaction('HKt')
reaction.name = 'H+/K+ gastric/non-gastric P-ATPase and ABC ATPase'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd489: -1.0, Cpd107: 1.0, Cpd108: 1.0, Cpd37: 1.0, Cpd487: 1.0})
reaction.gene_reaction_rule = '( ATP4A or ATP4B )'
model.add_reactions([reaction])
reaction = Reaction('HKYNH')
reaction.name = '3-Hydroxy-L-kynurenine hydrolase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd72: -1.0, Cpd68: 1.0, Cpd394: 1.0})
reaction.gene_reaction_rule = '( KYNU )'
model.add_reactions([reaction])
reaction = Reaction('HMBS')
reaction.name = 'hydroxymethylbilane synthase'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1485: -4.0, Cpd6: 4.0, Cpd1486: 1.0})
reaction.gene_reaction_rule = '( HMBS )'
model.add_reactions([reaction])
reaction = Reaction('HMGCOARr')
reaction.name = 'Hydroxymethylglutaryl CoA reductase (ir)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -2.0, Cpd587: -2.0, Cpd1487: -1.0, Cpd589: 2.0, Cpd745: 1.0, Cpd1488: 1.0})
reaction.gene_reaction_rule = '( HMGCR )'
model.add_reactions([reaction])
reaction = Reaction('HMGCOASi')
reaction.name = 'Hydroxymethylglutaryl CoA synthase (ir)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd149: -1.0, Cpd180: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd1489: 1.0})
reaction.gene_reaction_rule = '( HMGCS1 )'
model.add_reactions([reaction])
reaction = Reaction('HMGCOASim')
reaction.name = 'Hydroxymethylglutaryl CoA synthase (ir)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd178: -1.0, Cpd181: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd1490: 1.0})
reaction.gene_reaction_rule = '( HMGCS2 )'
model.add_reactions([reaction])
reaction = Reaction('HMGLm')
reaction.name = 'hydroxymethylglutaryl-CoA lyase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1490: -1.0, Cpd178: 1.0, Cpd186: 1.0})
reaction.gene_reaction_rule = '( HMGCL or HMGCLL1 )'
model.add_reactions([reaction])
reaction = Reaction('HMGLx')
reaction.name = 'hydroxymethylglutaryl-CoA lyase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1491: -1.0, Cpd182: 1.0, Cpd1492: 1.0})
reaction.gene_reaction_rule = '( HMGCL )'
model.add_reactions([reaction])
reaction = Reaction('HOMt4')
reaction.name = 'L-homoserine via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A1 or SLC7A2 or SLC7A3 )'
model.add_reactions([reaction])
reaction = Reaction('HOXG')
reaction.name = 'Heme oxygenase 1'
reaction.subsystem = 'Heme degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -3.0, Cpd33: -5.0, Cpd22: -3.0, Cpd1495: -1.0, Cpd2: 3.0, Cpd25: 3.0, Cpd559: 1.0, Cpd1496: 1.0, Cpd1497: 1.0})
reaction.gene_reaction_rule = '( HMOX1 or HMOX2 )'
model.add_reactions([reaction])
reaction = Reaction('HPDCACRNCPT1')
reaction.name = 'carnitine fatty-acyl transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1023: -1.0, Cpd78: 1.0, Cpd1498: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('HPDCACRNCPT2')
reaction.name = 'heptadecanoate transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1499: -1.0, Cpd307: 1.0, Cpd1066: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('HPDCACRNt')
reaction.name = 'heptadecanoate transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1498: -1.0, Cpd1499: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('HPYRR2x')
reaction.name = 'hydroxypyruvate reductase (NADH)'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -3.0, Cpd35: -1.0, Cpd1500: -2.0, Cpd32: 1.0, Cpd1501: 2.0})
reaction.gene_reaction_rule = '( LDHA or LDHAL6B or LDHB or LDHC or LDHD )'
model.add_reactions([reaction])
reaction = Reaction('HPYRRy')
reaction.name = 'Hydroxypyruvate reductase (NADPH)'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd1500: -1.0, Cpd25: 1.0, Cpd1502: 1.0})
reaction.gene_reaction_rule = '( GRHPR )'
model.add_reactions([reaction])
reaction = Reaction('HS1ly')
reaction.name = 'heparan-N-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1503: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1504: 1.0})
reaction.gene_reaction_rule = '( SGSH )'
model.add_reactions([reaction])
reaction = Reaction('HS2ly')
reaction.name = 'heparan-N-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1505: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1506: 1.0})
reaction.gene_reaction_rule = '( SGSH )'
model.add_reactions([reaction])
reaction = Reaction('HS3ly')
reaction.name = 'heparan-N-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1507: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1508: 1.0})
reaction.gene_reaction_rule = '( SGSH )'
model.add_reactions([reaction])
reaction = Reaction('HS4ly')
reaction.name = 'heparan-N-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1509: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1510: 1.0})
reaction.gene_reaction_rule = '( SGSH )'
model.add_reactions([reaction])
reaction = Reaction('HSD11B1r')
reaction.name = '11-beta-hydroxysteroid dehydrogenase type 1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1511: -1.0, Cpd589: 1.0, Cpd1512: 1.0})
reaction.gene_reaction_rule = '( HSD11B1 )'
model.add_reactions([reaction])
reaction = Reaction('HSD11B2r')
reaction.name = '11-beta-hydroxysteroid dehydrogenase type 2'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd626: -1.0, Cpd1511: -1.0, Cpd625: 1.0, Cpd1512: 1.0})
reaction.gene_reaction_rule = '( HSD11B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B1')
reaction.name = 'testicular 17-beta-hydroxysteroid dehydrogenase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd992: -1.0, Cpd25: 1.0, Cpd1513: 1.0})
reaction.gene_reaction_rule = '( H2-KE6 or HSD17B1 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B2r')
reaction.name = '17-beta-hydroxysteroid dehydrogenase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd626: -1.0, Cpd1514: -1.0, Cpd625: 1.0, Cpd1515: 1.0})
reaction.gene_reaction_rule = '( HSD17B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B42x')
reaction.name = 'hydroxysteroid (17-beta) dehydrogenase 4'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -2.0, Cpd246: -1.0, Cpd259: -1.0, Cpd1516: -2.0, Cpd256: 1.0, Cpd261: 3.0, Cpd1517: 2.0})
reaction.gene_reaction_rule = '( HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B4x')
reaction.name = 'hydroxysteroid (17-beta) dehydrogenase 4'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd246: -1.0, Cpd262: -1.0, Cpd1516: -1.0, Cpd261: 1.0, Cpd1517: 1.0, Cpd1518: 1.0})
reaction.gene_reaction_rule = '( HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B7r')
reaction.name = 'testicular 17-beta-hydroxysteroid dehydrogenase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1519: -1.0, Cpd589: 1.0, Cpd1520: 1.0})
reaction.gene_reaction_rule = '( HSD17B7 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B8r')
reaction.name = '17-beta-hydroxysteroid dehydrogenase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd626: -1.0, Cpd1519: -1.0, Cpd625: 1.0, Cpd1520: 1.0})
reaction.gene_reaction_rule = '( HSD17B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B9r')
reaction.name = '17-beta-hydroxysteroid dehydrogenase (type 7)'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.5, Cpd1521: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd437: 2.0, Cpd589: 1.0, Cpd1522: 1.0})
reaction.gene_reaction_rule = '( HSD17B7 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3A1r')
reaction.name = '3 alpha-hydroxysteroid dehydrogenase (type 3)'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd626: -1.0, Cpd1523: -1.0, Cpd1521: 1.0, Cpd625: 1.0})
reaction.gene_reaction_rule = '( AKR1C6 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3A2r')
reaction.name = '3 alpha-hydroxysteroid dehydrogenase (type 3)'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1523: -1.0, Cpd1521: 1.0, Cpd589: 1.0})
reaction.gene_reaction_rule = '( AKR1C6 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B11')
reaction.name = '3 beta-hydroxysteroid dehydrogenase/delta 5-->4-isomerase type I'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1524: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd384: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B11r')
reaction.name = '3 beta-hydroxysteroid dehydrogenase/delta 5-->4-isomerase type I'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd625: -1.0, Cpd1525: -1.0, Cpd565: 1.0, Cpd626: 1.0, Cpd1526: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B12r')
reaction.name = '3 beta-hydroxysteroid dehydrogenase/delta 5-->4-isomerase type I'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd625: -1.0, Cpd1527: -1.0, Cpd565: 1.0, Cpd626: 1.0, Cpd1514: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B13')
reaction.name = '3 beta-hydroxysteroid dehydrogenase/delta 5-->4-isomerase type I'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1528: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1529: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B13r')
reaction.name = '3 beta-hydroxysteroid dehydrogenase/delta 5-->4-isomerase type I'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd625: -1.0, Cpd1530: -1.0, Cpd565: 1.0, Cpd626: 1.0, Cpd1531: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B2r')
reaction.name = '3 beta-hydroxysteroid dehydrogenase type 1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd626: -1.0, Cpd1522: -1.0, Cpd437: 1.0, Cpd625: 1.0, Cpd1523: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B3r')
reaction.name = '3 beta-hydroxysteroid dehydrogenase type 1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1522: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1523: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('HSD3B7P')
reaction.name = 'hydroxy-delta-5-steroid dehydrogenase, 3 beta- and steroid delta-isomerase 7'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd1532: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd1533: 1.0})
reaction.gene_reaction_rule = '( HSD3B7 )'
model.add_reactions([reaction])
reaction = Reaction('Htm')
reaction.name = 'Uncoupling protein'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd7: 1.0})
reaction.gene_reaction_rule = '( SLC25A14 or SLC25A27 or UCP1 or UCP2 or UCP3 )'
model.add_reactions([reaction])
reaction = Reaction('HXANtl')
reaction.name = 'hypoxanthine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1534: -1.0, Cpd1535: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('HXPRT')
reaction.name = 'hypoxanthine phosphoribosyltransferase (Hypoxanthine)'
reaction.subsystem = 'Nucleotide salvage pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd301: -1.0, Cpd1535: -1.0, Cpd151: 1.0, Cpd316: 1.0})
reaction.gene_reaction_rule = '( HPRT )'
model.add_reactions([reaction])
reaction = Reaction('HYXNt')
reaction.name = 'Hypoxanthine transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1536: -1.0, Cpd1535: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('ICDHxm')
reaction.name = 'Isocitrate dehydrogenase (NAD+)'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd255: -1.0, Cpd28: 1.0, Cpd45: 1.0, Cpd47: 1.0})
reaction.gene_reaction_rule = '( IDH3A or IDH3B or IDH3G )'
model.add_reactions([reaction])
reaction = Reaction('ICDHy')
reaction.name = 'isocitrate dehydrogenase (NADP)'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd253: -1.0, Cpd22: 1.0, Cpd30: 1.0, Cpd60: 1.0})
reaction.gene_reaction_rule = '( IDH1 )'
model.add_reactions([reaction])
reaction = Reaction('ICDHyp')
reaction.name = 'Isocitrate dehydrogenase (NADP+)'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1517: -1.0, Cpd1537: -1.0, Cpd1538: 1.0, Cpd943: 1.0, Cpd1516: 1.0})
reaction.gene_reaction_rule = '( IDH1 )'
model.add_reactions([reaction])
reaction = Reaction('ICDHyrm')
reaction.name = 'Isocitrate dehydrogenase (NADP+)'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd255: -1.0, Cpd8: 1.0, Cpd28: 1.0, Cpd45: 1.0})
reaction.gene_reaction_rule = '( IDH2 )'
model.add_reactions([reaction])
reaction = Reaction('IDHPOXOX2b')
reaction.name = 'Iodide:hydrogen-peroxide oxidoreductase 2'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1539: -1.0, Cpd1540: -2.0, Cpd33: 1.0, Cpd702: 1.0, Cpd1541: 1.0})
reaction.gene_reaction_rule = '( TPO )'
model.add_reactions([reaction])
reaction = Reaction('IDHPOXOX3')
reaction.name = 'Iodide:hydrogen-peroxide oxidoreductase 3'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd1539: -1.0, Cpd1541: -1.0, Cpd2: 2.0, Cpd20: 1.0, Cpd1542: 1.0})
reaction.gene_reaction_rule = '( TPO )'
model.add_reactions([reaction])
reaction = Reaction('IDHPOXOX4')
reaction.name = 'Iodide:hydrogen-peroxide oxidoreductase 4'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd1541: -2.0, Cpd2: 2.0, Cpd20: 1.0, Cpd1543: 1.0})
reaction.gene_reaction_rule = '( TPO )'
model.add_reactions([reaction])
reaction = Reaction('IDHPOXOXb')
reaction.name = 'Iodide:hydrogen-peroxide oxidoreductase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1540: -2.0, Cpd1544: -1.0, Cpd33: 1.0, Cpd702: 1.0, Cpd1539: 1.0})
reaction.gene_reaction_rule = '( TPO )'
model.add_reactions([reaction])
reaction = Reaction('IDOAASE1ly')
reaction.name = 'alpha-L-iduronidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1386: -1.0, Cpd1545: 1.0, Cpd1546: 1.0})
reaction.gene_reaction_rule = '( IDUA )'
model.add_reactions([reaction])
reaction = Reaction('IDOAASE2ly')
reaction.name = 'alpha-L-iduronidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1547: -1.0, Cpd1546: 1.0, Cpd1548: 1.0})
reaction.gene_reaction_rule = '( IDUA )'
model.add_reactions([reaction])
reaction = Reaction('IDOAASE3ly')
reaction.name = 'alpha-L-iduronidase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1549: -1.0, Cpd1546: 1.0, Cpd1550: 1.0})
reaction.gene_reaction_rule = '( IDUA )'
model.add_reactions([reaction])
reaction = Reaction('IDOAASE4ly')
reaction.name = 'alpha-L-iduronidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1551: -1.0, Cpd1360: 1.0, Cpd1546: 1.0})
reaction.gene_reaction_rule = '( IDUA )'
model.add_reactions([reaction])
reaction = Reaction('IDOURtly')
reaction.name = 'iduronate transport into lysososme'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1552: -1.0, Cpd176: 1.0, Cpd1546: 1.0})
reaction.gene_reaction_rule = '( SLC17A5 )'
model.add_reactions([reaction])
reaction = Reaction('ILEt4')
reaction.name = 'L-isoleucine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('ILETA')
reaction.name = 'isoleucine transaminase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd1554: -1.0, Cpd74: 1.0, Cpd1555: 1.0})
reaction.gene_reaction_rule = '( BCAT1 )'
model.add_reactions([reaction])
reaction = Reaction('ILETAm')
reaction.name = 'isoleucine transaminase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd1556: -1.0, Cpd63: 1.0, Cpd1557: 1.0})
reaction.gene_reaction_rule = '( BCAT2 )'
model.add_reactions([reaction])
reaction = Reaction('ILEtec')
reaction.name = 'L-isoleucine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1553: -1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 or SLC43A2 )'
model.add_reactions([reaction])
reaction = Reaction('IMACTD')
reaction.name = 'Imidazole acetaldeyde dehydrogenase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1483: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd1558: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('IMACTD_m')
reaction.name = 'Imidazole acetaldeyde dehydrogenase (mito)'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd1559: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd1560: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('IMPC')
reaction.name = 'IMP cyclohydrolase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd316: -1.0, Cpd376: 1.0})
reaction.gene_reaction_rule = '( ATIC )'
model.add_reactions([reaction])
reaction = Reaction('IMPD')
reaction.name = 'IMP dehydrogenase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd316: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1420: 1.0})
reaction.gene_reaction_rule = '( IMPDH1 or IMPDH2 )'
model.add_reactions([reaction])
reaction = Reaction('INOSTO')
reaction.name = 'inositol oxygenase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd650: -1.0, Cpd2: 1.0, Cpd33: 1.0, Cpd1401: 1.0})
reaction.gene_reaction_rule = '( MIOX )'
model.add_reactions([reaction])
reaction = Reaction('INSt')
reaction.name = 'Inosine transport (diffusion)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd277: -1.0, Cpd275: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 or SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('INSt4')
reaction.name = 'inosine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd277: -1.0, Cpd41: 1.0, Cpd275: 1.0})
reaction.gene_reaction_rule = '( SLC28A2 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('INStl')
reaction.name = 'inosine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1561: -1.0, Cpd275: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('INStm')
reaction.name = 'inosine facilated transport in mitochondria'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd275: -1.0, Cpd1562: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 )'
model.add_reactions([reaction])
reaction = Reaction('INSTt2r')
reaction.name = 'inositol transport in via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd1563: -1.0, Cpd33: 1.0, Cpd650: 1.0})
reaction.gene_reaction_rule = '( SLC2A13 )'
model.add_reactions([reaction])
reaction = Reaction('INSTt4')
reaction.name = 'inositol transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1563: -1.0, Cpd41: 1.0, Cpd650: 1.0})
reaction.gene_reaction_rule = '( SLC5A3 )'
model.add_reactions([reaction])
reaction = Reaction('IPDDIx')
reaction.name = 'isopentenyl-diphosphate D-isomerase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd883: -1.0, Cpd882: 1.0})
reaction.gene_reaction_rule = '( GM9745 or IDI1 )'
model.add_reactions([reaction])
reaction = Reaction('It')
reaction.name = 'Iodide uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd703: -1.0, Cpd702: 1.0})
reaction.gene_reaction_rule = '( SLC5A8 )'
model.add_reactions([reaction])
reaction = Reaction('ITCOAL1m')
reaction.name = 'Itaconate--CoA ligase (GDP-forming), mitochondrial'
reaction.subsystem = 'C5-branched dibasic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd284: -1.0, Cpd1564: -1.0, Cpd204: 1.0, Cpd285: 1.0, Cpd1565: 1.0})
reaction.gene_reaction_rule = '( SUCLG1 or SUCLG2 )'
model.add_reactions([reaction])
reaction = Reaction('ITCOALm')
reaction.name = 'Itaconate--CoA ligase (ADP-forming), mitochondrial'
reaction.subsystem = 'C5-branched dibasic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd1564: -1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd1565: 1.0})
reaction.gene_reaction_rule = '( SUCLA2 or SUCLG1 )'
model.add_reactions([reaction])
reaction = Reaction('IZPN')
reaction.name = 'Imidazolonepropionase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1566: -1.0, Cpd33: 1.0, Cpd1405: 1.0})
reaction.gene_reaction_rule = '( AMDHD1 )'
model.add_reactions([reaction])
reaction = Reaction('KAS8')
reaction.name = 'b-ketoacyl synthetase (palmitate, n-C16:0)'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -20.0, Cpd22: -14.0, Cpd180: -1.0, Cpd199: -7.0, Cpd2: 6.0, Cpd25: 14.0, Cpd60: 7.0, Cpd78: 8.0, Cpd1010: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL5 or ELOVL6 or FASN )'
model.add_reactions([reaction])
reaction = Reaction('KCC2t')
reaction.name = 'K+-Cl- cotransporter (NH4+)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd278: -1.0, Cpd698: -1.0, Cpd6: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC12A4 or SLC12A6 or SLC12A7 )'
model.add_reactions([reaction])
reaction = Reaction('KCCt')
reaction.name = 'K+-Cl- cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd489: -1.0, Cpd698: -1.0, Cpd487: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC12A4 or SLC12A5 or SLC12A6 or SLC12A7 )'
model.add_reactions([reaction])
reaction = Reaction('KHK')
reaction.name = 'ketohexokinase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1144: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1103: 1.0})
reaction.gene_reaction_rule = '( KHK )'
model.add_reactions([reaction])
reaction = Reaction('KHK2')
reaction.name = 'ketohexokinase (D-xylulose)'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1567: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1105: 1.0})
reaction.gene_reaction_rule = '( KHK )'
model.add_reactions([reaction])
reaction = Reaction('KHK3')
reaction.name = 'ketohexokinase (D-tagatose)'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1568: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1107: 1.0})
reaction.gene_reaction_rule = '( KHK )'
model.add_reactions([reaction])
reaction = Reaction('Kt3g')
reaction.name = 'potassium transport  via proton antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd140: -1.0, Cpd487: -1.0, Cpd33: 1.0, Cpd1569: 1.0})
reaction.gene_reaction_rule = '( SLC9A7 or SLC9A8 )'
model.add_reactions([reaction])
reaction = Reaction('KYN')
reaction.name = 'kynureninase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1117: -1.0, Cpd33: 1.0, Cpd394: 1.0, Cpd1570: 1.0})
reaction.gene_reaction_rule = '( KYNU )'
model.add_reactions([reaction])
reaction = Reaction('KYN3OX')
reaction.name = 'kynurenine 3-monooxygenase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1117: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd72: 1.0})
reaction.gene_reaction_rule = '( KMO )'
model.add_reactions([reaction])
reaction = Reaction('KYNAKGAT')
reaction.name = 'L-Kynurenine:2-oxoglutarate aminotransferase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd1117: -1.0, Cpd74: 1.0, Cpd1571: 1.0})
reaction.gene_reaction_rule = '( AADAT or KYAT1 )'
model.add_reactions([reaction])
reaction = Reaction('LACZe')
reaction.name = 'b-galactosidase, extracellular'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1572: -1.0, Cpd446: 1.0, Cpd1266: 1.0})
reaction.gene_reaction_rule = '( GLB1 or LCT )'
model.add_reactions([reaction])
reaction = Reaction('LACZly')
reaction.name = 'b-galactosidase,lysosomal'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1573: -1.0, Cpd1212: 1.0, Cpd1292: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('LALDO')
reaction.name = 'D-Lactaldehyde:NAD+ oxidoreductase (glutathione-formylating)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd413: -1.0, Cpd1057: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1416: 1.0})
reaction.gene_reaction_rule = '( ADH5 )'
model.add_reactions([reaction])
reaction = Reaction('LALDO2x')
reaction.name = 'D-Lactaldehyde:NAD+ 1-oxidoreductase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd153: -1.0, Cpd32: 1.0, Cpd413: 1.0})
reaction.gene_reaction_rule = '( AKR7A5 )'
model.add_reactions([reaction])
reaction = Reaction('LAPCOAl')
reaction.name = 'lysosomal acid phosphorylase (CoA)'
reaction.subsystem = 'CoA catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1574: -1.0, Cpd88: 1.0, Cpd1575: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('LCADi')
reaction.name = 'lactaldehyde dehydrogenase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd415: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH1A3 or ALDH3A1 or ALDH3A2 or ALDH3B1 or ALDH3B3 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('LCADi_D')
reaction.name = 'lactaldehyde dehydrogenase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd413: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd872: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH1A3 or ALDH3A1 or ALDH3A2 or ALDH3B1 or ALDH3B3 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('LCADi_Dm')
reaction.name = 'lactaldehyde dehydrogenase, mitochondrial'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd1577: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd873: 1.0})
reaction.gene_reaction_rule = '( ALDH18A1 or ALDH1B1 or ALDH2 or ALDH4A1 or ALDH5A1 or ALDH6A1 )'
model.add_reactions([reaction])
reaction = Reaction('LCADim')
reaction.name = 'lactaldehyde dehydrogenase, mitochondrial'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd1578: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd1579: 1.0})
reaction.gene_reaction_rule = '( ALDH18A1 or ALDH1B1 or ALDH2 or ALDH4A1 or ALDH5A1 or ALDH6A1 )'
model.add_reactions([reaction])
reaction = Reaction('LCAT1e')
reaction.name = 'Lecithin-cholesterol acyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd684: -1.0, Cpd1580: -1.0, Cpd1581: 1.0, Cpd1582: 1.0})
reaction.gene_reaction_rule = '( LCAT )'
model.add_reactions([reaction])
reaction = Reaction('LCYSTAT')
reaction.name = 'L-Cysteate:2-oxoglutarate aminotransferase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd24: -1.0, Cpd30: -1.0, Cpd74: 1.0, Cpd1583: 1.0})
reaction.gene_reaction_rule = '( GOT1 )'
model.add_reactions([reaction])
reaction = Reaction('LCYSTATm')
reaction.name = 'L-Cysteate:2-oxoglutarate aminotransferase, mitochondrial'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd1584: -1.0, Cpd63: 1.0, Cpd1585: 1.0})
reaction.gene_reaction_rule = '( GOT2 )'
model.add_reactions([reaction])
reaction = Reaction('LCYSTCBOXL')
reaction.name = '3-Sulfoalanine carboxy-lyase'
reaction.subsystem = 'Taurine and hypotaurine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd24: -1.0, Cpd60: 1.0, Cpd1586: 1.0})
reaction.gene_reaction_rule = '( CSAD or GAD1 or GAD2 )'
model.add_reactions([reaction])
reaction = Reaction('LDH_D')
reaction.name = 'D-lactate dehydrogenase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd872: -1.0, Cpd33: 1.0, Cpd21: 1.0, Cpd35: 1.0})
reaction.gene_reaction_rule = '( LDHD )'
model.add_reactions([reaction])
reaction = Reaction('LDH_L')
reaction.name = 'L-lactate dehydrogenase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1576: -1.0, Cpd33: 1.0, Cpd21: 1.0, Cpd35: 1.0})
reaction.gene_reaction_rule = '( LDHA or LDHAL6B or LDHB or LDHC or UEVLD )'
model.add_reactions([reaction])
reaction = Reaction('LDH_Lm')
reaction.name = 'L-lactate dehydrogenase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1579: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd363: 1.0})
reaction.gene_reaction_rule = '( LDHA or LDHB or LDHD )'
model.add_reactions([reaction])
reaction = Reaction('LEUKTRC4t')
reaction.name = 'leukotriene C4 transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1587: -1.0, Cpd38: 1.0, Cpd1588: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('LEUt4')
reaction.name = 'L-leucine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A4 or SLC3A2 or SLC6A14 or SLC6A19 or SLC7A1 or SLC7A2 or SLC7A3 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('LEUTA')
reaction.name = 'leucine transaminase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd1590: -1.0, Cpd74: 1.0, Cpd1591: 1.0})
reaction.gene_reaction_rule = '( BCAT1 )'
model.add_reactions([reaction])
reaction = Reaction('LEUTAm')
reaction.name = 'leucine transaminase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd1592: -1.0, Cpd63: 1.0, Cpd1593: 1.0})
reaction.gene_reaction_rule = '( BCAT2 )'
model.add_reactions([reaction])
reaction = Reaction('LEUtec')
reaction.name = 'L-leucine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1589: -1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 or SLC43A2 )'
model.add_reactions([reaction])
reaction = Reaction('LFORKYNHYD')
reaction.name = 'L-Formylkynurenine hydrolase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1116: -1.0, Cpd33: 1.0, Cpd394: 1.0, Cpd1594: 1.0})
reaction.gene_reaction_rule = '( KYNU )'
model.add_reactions([reaction])
reaction = Reaction('LGNCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1041: -1.0, Cpd78: 1.0, Cpd1595: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('LGNCCPT2')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1596: -1.0, Cpd307: 1.0, Cpd1597: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('LGNCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1595: -1.0, Cpd1596: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('LGNCt')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1100: -1.0, Cpd1040: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('LGTHL')
reaction.name = 'lactoylglutathione lyase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd153: -1.0, Cpd1057: -1.0, Cpd1416: 1.0})
reaction.gene_reaction_rule = '( GLO1 )'
model.add_reactions([reaction])
reaction = Reaction('LIPOti')
reaction.name = 'Lipoate transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -1.0, Cpd1598: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 1.0, Cpd1599: 1.0})
reaction.gene_reaction_rule = '( SLC5A6 )'
model.add_reactions([reaction])
reaction = Reaction('L_LACDcm')
reaction.name = 'L-Lactate dehydrogenase, cytosolic/mitochondrial'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd751: -2.0, Cpd1576: -1.0, Cpd33: 2.0, Cpd21: 1.0, Cpd750: 2.0})
reaction.gene_reaction_rule = '( CYB5D1 )'
model.add_reactions([reaction])
reaction = Reaction('L_LACt2r')
reaction.name = 'L-lactate reversible transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd1600: -1.0, Cpd33: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( BSG or SLC16A1 or SLC16A3 or SLC16A7 or SLC16A8 )'
model.add_reactions([reaction])
reaction = Reaction('L_LACt4r')
reaction.name = 'L-lactate reversible transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1600: -1.0, Cpd41: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( SLC5A8 )'
model.add_reactions([reaction])
reaction = Reaction('L_LACtcm')
reaction.name = 'L-lactate transport via diffusion (cytosol to mitochondria)'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1576: -1.0, Cpd1579: 1.0})
reaction.gene_reaction_rule = '( AQP9 )'
model.add_reactions([reaction])
reaction = Reaction('L_LACtm')
reaction.name = 'L-lactate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1576: -1.0, Cpd7: 1.0, Cpd1579: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('LNELDCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd827: -1.0, Cpd78: 1.0, Cpd1601: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('LNELDCCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1602: -1.0, Cpd307: 1.0, Cpd1070: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('LNELDCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1601: -1.0, Cpd1602: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('LNLCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd828: -1.0, Cpd78: 1.0, Cpd1603: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('LNLCCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1604: -1.0, Cpd307: 1.0, Cpd1069: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('LNLCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1603: -1.0, Cpd1604: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('LNLCt')
reaction.name = 'Linoleic acid (n-C18:2) transport in via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1605: -1.0, Cpd1025: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCACPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd823: -1.0, Cpd78: 1.0, Cpd1606: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCACPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1607: -1.0, Cpd307: 1.0, Cpd1071: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCACRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1606: -1.0, Cpd1607: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCAt')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1608: -1.0, Cpd1028: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCGCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd829: -1.0, Cpd78: 1.0, Cpd1609: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCGCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1610: -1.0, Cpd307: 1.0, Cpd1072: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCGCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1609: -1.0, Cpd1610: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('LNLNCGt')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1611: -1.0, Cpd1027: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('LNS14DM')
reaction.name = 'cytochrome P450 lanosterol 14-alpha-demethylase (NADP)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -3.0, Cpd33: -2.0, Cpd22: -3.0, Cpd1612: -1.0, Cpd2: 4.0, Cpd25: 3.0, Cpd699: 1.0, Cpd1613: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('LNSTLSr')
reaction.name = 'lanosterol synthase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1614: -1.0, Cpd1615: 1.0})
reaction.gene_reaction_rule = '( LSS )'
model.add_reactions([reaction])
reaction = Reaction('LPASE')
reaction.name = 'lysophospholipase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1616: -1.0, Cpd33: 1.0, Cpd1617: 1.0, Cpd1618: 1.0})
reaction.gene_reaction_rule = '( PLA2G4A )'
model.add_reactions([reaction])
reaction = Reaction('LPCOXp')
reaction.name = 'L-pipecolate oxidase, peroxisomal'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1619: -1.0, Cpd246: -1.0, Cpd358: 1.0, Cpd247: 1.0, Cpd1620: 1.0})
reaction.gene_reaction_rule = '( PIPOX )'
model.add_reactions([reaction])
reaction = Reaction('LPS')
reaction.name = 'lipase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd836: -1.0, Cpd33: 1.0, Cpd1621: 1.0, Cpd658: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPC or LPL or PNPLA3 )'
model.add_reactions([reaction])
reaction = Reaction('LPS2')
reaction.name = 'lipase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd658: -1.0, Cpd33: 1.0, Cpd1617: 1.0, Cpd1622: 1.0})
reaction.gene_reaction_rule = '( CEL or LPL )'
model.add_reactions([reaction])
reaction = Reaction('LPS2e')
reaction.name = 'lipase, extracellular'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1623: -1.0, Cpd37: 1.0, Cpd1624: 1.0, Cpd1625: 1.0})
reaction.gene_reaction_rule = '( LIPC )'
model.add_reactions([reaction])
reaction = Reaction('LPS3')
reaction.name = 'lipase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1622: -1.0, Cpd33: 1.0, Cpd1626: 1.0, Cpd1413: 1.0})
reaction.gene_reaction_rule = '( MGLL )'
model.add_reactions([reaction])
reaction = Reaction('LPS3e')
reaction.name = 'lipase, extracellular'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1624: -1.0, Cpd37: 1.0, Cpd1627: 1.0, Cpd1628: 1.0})
reaction.gene_reaction_rule = '( LIPC )'
model.add_reactions([reaction])
reaction = Reaction('LPS4e')
reaction.name = 'phospholipase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1629: -1.0, Cpd37: 1.0, Cpd1630: 1.0, Cpd1628: 1.0})
reaction.gene_reaction_rule = '( PLA2G3 )'
model.add_reactions([reaction])
reaction = Reaction('LPSe')
reaction.name = 'lipase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1631: -1.0, Cpd37: 1.0, Cpd1623: 1.0, Cpd1632: 1.0})
reaction.gene_reaction_rule = '( LIPC or LIPF or LIPG or LPL or PNLIP or PNLIPRP1 or PNLIPRP2 )'
model.add_reactions([reaction])
reaction = Reaction('LRAT')
reaction.name = 'Lecithin retinol acyltransferase'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd659: -1.0, Cpd1633: -1.0, Cpd1616: 1.0, Cpd1634: 1.0})
reaction.gene_reaction_rule = '( LRAT )'
model.add_reactions([reaction])
reaction = Reaction('LSTO1r')
reaction.name = 'Lathosterol oxidase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd850: -1.0, Cpd437: 2.0, Cpd589: 1.0, Cpd853: 1.0})
reaction.gene_reaction_rule = '( SC5D )'
model.add_reactions([reaction])
reaction = Reaction('LSTO2r')
reaction.name = 'Lathosterol oxidase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd851: -1.0, Cpd854: 1.0, Cpd437: 2.0, Cpd589: 1.0})
reaction.gene_reaction_rule = '( CYP7A1 or SC5D )'
model.add_reactions([reaction])
reaction = Reaction('LTA4H')
reaction.name = 'Leukotriene A-4 hydrolase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd432: -1.0, Cpd1635: 1.0})
reaction.gene_reaction_rule = '( LTA4H )'
model.add_reactions([reaction])
reaction = Reaction('LTC4Sr')
reaction.name = 'Leukotriene C4 synthase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd1637: -1.0, Cpd1338: 1.0})
reaction.gene_reaction_rule = '( LTC4S or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('LTDCL')
reaction.name = 'L-Tryptophan decarboxy-lyase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1638: -1.0, Cpd60: 1.0, Cpd1639: 1.0})
reaction.gene_reaction_rule = '( DDC )'
model.add_reactions([reaction])
reaction = Reaction('LYSMTF1n')
reaction.name = 'histone-lysine N-methyltransferase, nuclear'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd889: -1.0, Cpd1640: -1.0, Cpd891: 1.0, Cpd1641: 1.0})
reaction.gene_reaction_rule = '( DOT1L or EHMT1 or EHMT2 or KMT2C or KMT5A or SETD7 or SETDB1 or SETDB2 or SUV39H1 or SUV39H2 )'
model.add_reactions([reaction])
reaction = Reaction('LYSMTF2n')
reaction.name = 'histone-lysine N-methyltransferase, nuclear'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd889: -1.0, Cpd1641: -1.0, Cpd891: 1.0, Cpd1642: 1.0})
reaction.gene_reaction_rule = '( DOT1L or EHMT1 or EHMT2 or KMT2C or KMT5A or SETD7 or SETDB1 or SETDB2 or SUV39H1 or SUV39H2 )'
model.add_reactions([reaction])
reaction = Reaction('LYSMTF3n')
reaction.name = 'histone-lysine N-methyltransferase, nuclear'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd889: -1.0, Cpd1642: -1.0, Cpd891: 1.0, Cpd1643: 1.0})
reaction.gene_reaction_rule = '( DOT1L or EHMT1 or EHMT2 or KMT2C or KMT5A or SETD7 or SETDB1 or SETDB2 or SUV39H1 or SUV39H2 )'
model.add_reactions([reaction])
reaction = Reaction('LYSt4')
reaction.name = 'L-lysine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd468: -1.0, Cpd41: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC22A5 or SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('LYStiDF')
reaction.name = 'L-lysine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC38A4 or SLC7A1 or SLC7A2 or SLC7A3 )'
model.add_reactions([reaction])
reaction = Reaction('LYStm')
reaction.name = 'Lysine mitochondrial transport via ornithine carrier'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd465: -1.0, Cpd33: 1.0, Cpd576: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('M1316Mg')
reaction.name = 'mannosyl-oligosaccharide 1,3-1,6-alpha-mannosidase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -2.0, Cpd1644: -1.0, Cpd1645: 2.0, Cpd1646: 1.0})
reaction.gene_reaction_rule = '( MAN2A1 or MAN2A2 )'
model.add_reactions([reaction])
reaction = Reaction('M13N2Tg')
reaction.name = 'alpha-1,3-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1647: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1644: 1.0})
reaction.gene_reaction_rule = '( MGAT1 )'
model.add_reactions([reaction])
reaction = Reaction('M13N4Tg')
reaction.name = 'alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1000: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1648: 1.0})
reaction.gene_reaction_rule = '( MGAT4A or MGAT4B )'
model.add_reactions([reaction])
reaction = Reaction('M14NTg')
reaction.name = 'beta-1,4-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1000: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1649: 1.0})
reaction.gene_reaction_rule = '( MGAT3 )'
model.add_reactions([reaction])
reaction = Reaction('M16N6Tg')
reaction.name = 'alpha-1,6-mannosyl-glycoprotein 6-beta-N-acetylglucosaminyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1648: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1650: 1.0})
reaction.gene_reaction_rule = '( MGAT5 )'
model.add_reactions([reaction])
reaction = Reaction('M16NTg')
reaction.name = 'alpha-1,6-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd144: -1.0, Cpd1646: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1000: 1.0})
reaction.gene_reaction_rule = '( MGAT2 )'
model.add_reactions([reaction])
reaction = Reaction('M4ATAer')
reaction.name = 'M4A transamidase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1463: -1.0, Cpd1651: -1.0, Cpd1464: 1.0, Cpd1652: 1.0})
reaction.gene_reaction_rule = '( GPAA1 or PIGK or PIGS or PIGU )'
model.add_reactions([reaction])
reaction = Reaction('M4BET2er')
reaction.name = 'M4B phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1462: -1.0, Cpd1453: 1.0, Cpd1651: 1.0})
reaction.gene_reaction_rule = '( PIGF or PIGG )'
model.add_reactions([reaction])
reaction = Reaction('M4BTAer')
reaction.name = 'M4B transamidase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1462: -1.0, Cpd1463: -1.0, Cpd1464: 1.0, Cpd1653: 1.0})
reaction.gene_reaction_rule = '( GPAA1 or PIGK or PIGS or PIGT or PIGU )'
model.add_reactions([reaction])
reaction = Reaction('M4CET3er')
reaction.name = 'M4C phosphoethanolaminyl transferase, endoplasmic reticulum'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1452: -1.0, Cpd1460: -1.0, Cpd1453: 1.0, Cpd1462: 1.0})
reaction.gene_reaction_rule = '( PIGF or PIGO )'
model.add_reactions([reaction])
reaction = Reaction('MACACI')
reaction.name = 'maleylacetoacetate isomerase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1477: -1.0, Cpd1158: 1.0})
reaction.gene_reaction_rule = '( GSTZ1 )'
model.add_reactions([reaction])
reaction = Reaction('MACOXO')
reaction.name = '3-Methylimidazole acetaldehyde:NAD+ oxidoreductase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1654: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd1655: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('MALSO3tm')
reaction.name = 'Malate:sulfite antiport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1160: -1.0, Cpd381: -1.0, Cpd1161: 1.0, Cpd382: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('MALSO4tm')
reaction.name = 'Malate:sulfate antiport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd381: -1.0, Cpd1162: -1.0, Cpd382: 1.0, Cpd1163: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('MALT')
reaction.name = 'alpha-glucosidase'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1656: -1.0, Cpd1290: 2.0})
reaction.gene_reaction_rule = '( GAA or GANC or MGAM )'
model.add_reactions([reaction])
reaction = Reaction('MALTe')
reaction.name = 'alpha-glucosidase, extracellular'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1274: -1.0, Cpd446: 2.0})
reaction.gene_reaction_rule = '( MGAM )'
model.add_reactions([reaction])
reaction = Reaction('MALTly')
reaction.name = 'alpha-glucosidase, lysosomal'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1657: -1.0, Cpd1292: 2.0})
reaction.gene_reaction_rule = '( GAA )'
model.add_reactions([reaction])
reaction = Reaction('MALtm')
reaction.name = 'malate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd204: -1.0, Cpd381: -1.0, Cpd108: 1.0, Cpd382: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('MALTSULtm')
reaction.name = 'Malate:thiosulfate antiport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd381: -1.0, Cpd1164: -1.0, Cpd382: 1.0, Cpd1165: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('MALTt1r')
reaction.name = 'maltose transport (uniport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1274: -1.0, Cpd1656: 1.0})
reaction.gene_reaction_rule = '( SLC2A3 )'
model.add_reactions([reaction])
reaction = Reaction('MAN1_6B1er')
reaction.name = 'mannosidase I, endoplasmic reticulum (g1m6masnB1-producing)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1658: -1.0, Cpd1659: 1.0, Cpd1660: 1.0})
reaction.gene_reaction_rule = '( MAN1B1 )'
model.add_reactions([reaction])
reaction = Reaction('MAN1_7Ber')
reaction.name = 'mannosidase I, endoplasmic reticulum (g1m7masnB-producing)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1661: -1.0, Cpd1662: 1.0, Cpd1660: 1.0})
reaction.gene_reaction_rule = '( MAN1B1 )'
model.add_reactions([reaction])
reaction = Reaction('MAN1PT2')
reaction.name = 'mannose-1-phosphate guanylyltransferase (GDP)'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd283: -1.0, Cpd295: -1.0, Cpd108: 1.0, Cpd548: 1.0})
reaction.gene_reaction_rule = '( GMPPA or GMPPB )'
model.add_reactions([reaction])
reaction = Reaction('MAN6PI')
reaction.name = 'mannose-6-phosphate isomerase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd233: -1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( MPI )'
model.add_reactions([reaction])
reaction = Reaction('MANt1r')
reaction.name = 'mannose transport (uniport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1663: -1.0, Cpd129: 1.0})
reaction.gene_reaction_rule = '( SLC2A1 or SLC2A2 or SLC2A3 )'
model.add_reactions([reaction])
reaction = Reaction('MANt4')
reaction.name = 'D-mannose transport via sodium cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1663: -1.0, Cpd41: 1.0, Cpd129: 1.0})
reaction.gene_reaction_rule = '( SLC5A10 or SLC5A9 )'
model.add_reactions([reaction])
reaction = Reaction('MAOLNOR')
reaction.name = 'monoamine oxidase (L-Normetanephrine)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1664: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd82: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('MAOX')
reaction.name = 'Methylamine oxidase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd97: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd411: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('MCCCrm')
reaction.name = 'methylcrotonoyl-CoA carboxylase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd201: -1.0, Cpd243: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd1665: 1.0})
reaction.gene_reaction_rule = '( MCCC1 or MCCC2 )'
model.add_reactions([reaction])
reaction = Reaction('MCD')
reaction.name = 'Malonyl-CoA Decarboxylase cytoplasmic'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd60: 1.0, Cpd180: 1.0})
reaction.gene_reaction_rule = '( MLYCD )'
model.add_reactions([reaction])
reaction = Reaction('MCDm')
reaction.name = 'Malonyl-CoA Decarboxylase, mitochondrial'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd203: -1.0, Cpd45: 1.0, Cpd178: 1.0})
reaction.gene_reaction_rule = '( MLYCD )'
model.add_reactions([reaction])
reaction = Reaction('MCDp')
reaction.name = 'Malonyl-CoA Decarboxylase peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1666: -1.0, Cpd182: 1.0, Cpd943: 1.0})
reaction.gene_reaction_rule = '( MLYCD )'
model.add_reactions([reaction])
reaction = Reaction('MCLOR')
reaction.name = '3-Mercaptolactate:NAD+ oxidoreductase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1667: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd753: 1.0})
reaction.gene_reaction_rule = '( LDHA or LDHAL6B or LDHB or LDHC or UEVLD )'
model.add_reactions([reaction])
reaction = Reaction('MCOATA')
reaction.name = 'Malonyl-CoA-ACP transacylase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd199: -1.0, Cpd249: -1.0, Cpd78: 1.0, Cpd1668: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('MCOATAm')
reaction.name = 'Malonyl-CoA-ACP transacylase, mitochondrial'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd203: -1.0, Cpd1669: -1.0, Cpd43: 1.0, Cpd1670: 1.0})
reaction.gene_reaction_rule = '( MCAT )'
model.add_reactions([reaction])
reaction = Reaction('MCPST')
reaction.name = '3-mercaptopyruvate sulfurtransferase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1671: -1.0, Cpd753: -1.0, Cpd33: 1.0, Cpd21: 1.0, Cpd1672: 1.0})
reaction.gene_reaction_rule = '( MPST )'
model.add_reactions([reaction])
reaction = Reaction('MDH')
reaction.name = 'malate dehydrogenase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd381: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd229: 1.0})
reaction.gene_reaction_rule = '( MDH1 or MDH1B )'
model.add_reactions([reaction])
reaction = Reaction('MDHm')
reaction.name = 'malate dehydrogenase, mitochondrial'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd382: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd491: 1.0})
reaction.gene_reaction_rule = '( MDH1 or MDH2 )'
model.add_reactions([reaction])
reaction = Reaction('ME1m')
reaction.name = 'malic enzyme (NAD), mitochondrial'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd382: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd363: 1.0})
reaction.gene_reaction_rule = '( ME2 )'
model.add_reactions([reaction])
reaction = Reaction('ME2')
reaction.name = 'malic enzyme (NADP)'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd381: -1.0, Cpd21: 1.0, Cpd22: 1.0, Cpd60: 1.0})
reaction.gene_reaction_rule = '( ME1 )'
model.add_reactions([reaction])
reaction = Reaction('ME2m')
reaction.name = 'malic enzyme (NADP), mitochondrial'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd382: -1.0, Cpd8: 1.0, Cpd45: 1.0, Cpd363: 1.0})
reaction.gene_reaction_rule = '( ME2 or ME3 )'
model.add_reactions([reaction])
reaction = Reaction('MECOALm')
reaction.name = 'mesaconate--CoA ligase (ADP-forming), mitochondrial'
reaction.subsystem = 'C5-branched dibasic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd1673: -1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd1674: 1.0})
reaction.gene_reaction_rule = '( SUCLA2 or SUCLG1 )'
model.add_reactions([reaction])
reaction = Reaction('MECOAS1m')
reaction.name = 'mesaconate--CoA ligase (GDP-forming)'
reaction.subsystem = 'C5-branched dibasic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd284: -1.0, Cpd1673: -1.0, Cpd204: 1.0, Cpd285: 1.0, Cpd1674: 1.0})
reaction.gene_reaction_rule = '( SUCLG1 or SUCLG2 )'
model.add_reactions([reaction])
reaction = Reaction('MELATN23DOX')
reaction.name = 'Melatonin:oxygen 2,3-dioxygenase (indole-decyclizing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd270: -1.0, Cpd1675: 1.0})
reaction.gene_reaction_rule = '( IDO1 )'
model.add_reactions([reaction])
reaction = Reaction('MELATNOX')
reaction.name = 'Melatonin:NADP oxidoreductase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd270: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1676: 1.0})
reaction.gene_reaction_rule = '( CYP1A1 or CYP1A2 or CYP1B1 )'
model.add_reactions([reaction])
reaction = Reaction('METAT')
reaction.name = 'methionine adenosyltransferase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd554: -1.0, Cpd108: 1.0, Cpd52: 1.0, Cpd151: 1.0})
reaction.gene_reaction_rule = '( MAT1A or MAT2B )'
model.add_reactions([reaction])
reaction = Reaction('METLEUex')
reaction.name = 'Methionine/Leucine exchange (Met in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd1590: -1.0, Cpd554: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('METS')
reaction.name = 'methionine synthase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd127: -1.0, Cpd371: -1.0, Cpd33: 1.0, Cpd377: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( MTR )'
model.add_reactions([reaction])
reaction = Reaction('METt4')
reaction.name = 'L-methionine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC22A5 or SLC38A1 or SLC38A2 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('METtec')
reaction.name = 'L-methionine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 or SLC43A2 )'
model.add_reactions([reaction])
reaction = Reaction('MEVK1x')
reaction.name = 'mevalonate kinase (atp)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd940: -1.0, Cpd1678: -1.0, Cpd358: 1.0, Cpd942: 1.0, Cpd1679: 1.0})
reaction.gene_reaction_rule = '( MVK )'
model.add_reactions([reaction])
reaction = Reaction('MG1er')
reaction.name = 'mannosyl-oligosaccharide glucosidase, endoplasmic reticulum'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd912: -1.0, Cpd641: 1.0, Cpd1680: 1.0})
reaction.gene_reaction_rule = '( MOGS )'
model.add_reactions([reaction])
reaction = Reaction('MGCHrm')
reaction.name = 'methylglutaconyl-CoA hydratase (reversible), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1665: -1.0, Cpd1490: 1.0})
reaction.gene_reaction_rule = '( AUH )'
model.add_reactions([reaction])
reaction = Reaction('MHISOR')
reaction.name = 'N-Methylhistamine:oxygen oxidoreductase (deaminating)'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1681: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd1654: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('MI1345PKn')
reaction.name = 'inositol-1,3,4,5-triphosphate 6-kinase, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1682: -1.0, Cpd763: 1.0, Cpd789: 1.0, Cpd1683: 1.0})
reaction.gene_reaction_rule = '( IPMK )'
model.add_reactions([reaction])
reaction = Reaction('MI1345PP')
reaction.name = 'inositol-1,3,4,5-trisphosphate 5-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1684: -1.0, Cpd108: 1.0, Cpd1685: 1.0})
reaction.gene_reaction_rule = '( INPP5A or INPP5B or INPP5D or INPP5E or INPP5J or INPPL1 or SYNJ1 )'
model.add_reactions([reaction])
reaction = Reaction('MI1346PKn')
reaction.name = 'inositol-1,3,4,6-tetrakisphosphate 5-kinase, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1686: -1.0, Cpd763: 1.0, Cpd789: 1.0, Cpd1683: 1.0})
reaction.gene_reaction_rule = '( IPMK )'
model.add_reactions([reaction])
reaction = Reaction('MI134PP')
reaction.name = 'inositol-1,3,4-trisphosphate 1-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1685: -1.0, Cpd108: 1.0, Cpd1687: 1.0})
reaction.gene_reaction_rule = '( INPP1 )'
model.add_reactions([reaction])
reaction = Reaction('MI1456PKn')
reaction.name = 'inositol-1,4,5,6- tetrakisphosphate 3-kinase, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1688: -1.0, Cpd763: 1.0, Cpd789: 1.0, Cpd1683: 1.0})
reaction.gene_reaction_rule = '( IPMK )'
model.add_reactions([reaction])
reaction = Reaction('MI145P6Kn')
reaction.name = 'inositol-1,4,5-triphosphate 6-kinase, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1689: -1.0, Cpd763: 1.0, Cpd789: 1.0, Cpd1688: 1.0})
reaction.gene_reaction_rule = '( IPMK )'
model.add_reactions([reaction])
reaction = Reaction('MI145PK')
reaction.name = 'inositol-1,4,5-trisphosphate 3-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1690: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1684: 1.0})
reaction.gene_reaction_rule = '( ITPKA or ITPKB or ITPKC )'
model.add_reactions([reaction])
reaction = Reaction('MI145PKn')
reaction.name = 'inositol-1,4,5-trisphosphate 3-kinase, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1689: -1.0, Cpd763: 1.0, Cpd789: 1.0, Cpd1682: 1.0})
reaction.gene_reaction_rule = '( IPMK )'
model.add_reactions([reaction])
reaction = Reaction('MI145PP')
reaction.name = 'inositol-1,4,5-trisphosphate 5-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1690: -1.0, Cpd108: 1.0, Cpd1691: 1.0})
reaction.gene_reaction_rule = '( INPP5A or INPP5B or INPP5D or INPP5E or INPP5J or INPPL1 or SYNJ1 )'
model.add_reactions([reaction])
reaction = Reaction('MI14PP')
reaction.name = 'inositol-1,4-bisphosphate 1-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1691: -1.0, Cpd108: 1.0, Cpd1692: 1.0})
reaction.gene_reaction_rule = '( INPP1 )'
model.add_reactions([reaction])
reaction = Reaction('MI1PP')
reaction.name = 'myo-inositol 1-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1693: -1.0, Cpd108: 1.0, Cpd650: 1.0})
reaction.gene_reaction_rule = '( IMPA1 or IMPA2 or MTMR1 or MTMR2 )'
model.add_reactions([reaction])
reaction = Reaction('MI1PS')
reaction.name = 'myo-Inositol-1-phosphate synthase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1202: -1.0, Cpd1693: 1.0})
reaction.gene_reaction_rule = '( ISYNA1 )'
model.add_reactions([reaction])
reaction = Reaction('MI34PP')
reaction.name = 'inositol-3,4-bisphosphate 4-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1687: -1.0, Cpd108: 1.0, Cpd1694: 1.0})
reaction.gene_reaction_rule = '( INPP4A or INPP4B )'
model.add_reactions([reaction])
reaction = Reaction('MI3PP')
reaction.name = 'myo-inositol 3-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1694: -1.0, Cpd108: 1.0, Cpd650: 1.0})
reaction.gene_reaction_rule = '( IMPA1 or IMPA2 )'
model.add_reactions([reaction])
reaction = Reaction('MI4PP')
reaction.name = 'myo-inositol 4-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1692: -1.0, Cpd108: 1.0, Cpd650: 1.0})
reaction.gene_reaction_rule = '( IMPA1 or IMPA2 )'
model.add_reactions([reaction])
reaction = Reaction('MLTG1')
reaction.name = 'Maltodextrin glucosidase (maltotriose)'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1695: -1.0, Cpd1290: 1.0, Cpd1656: 1.0})
reaction.gene_reaction_rule = '( GANC )'
model.add_reactions([reaction])
reaction = Reaction('MLTG1e')
reaction.name = 'Maltodextrin glucosidase (maltotriose), extracellular'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1696: -1.0, Cpd446: 1.0, Cpd1274: 1.0})
reaction.gene_reaction_rule = '( MGAM )'
model.add_reactions([reaction])
reaction = Reaction('MLTG1ly')
reaction.name = 'Maltodextrin glucosidase (maltotriose), lysosome'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1697: -1.0, Cpd1292: 1.0, Cpd1657: 1.0})
reaction.gene_reaction_rule = '( GAA )'
model.add_reactions([reaction])
reaction = Reaction('MM5ag')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1698: -1.0, Cpd1645: 1.0, Cpd1647: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM5bg')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1698: -1.0, Cpd1645: 1.0, Cpd1647: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM5cg')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1698: -1.0, Cpd1645: 1.0, Cpd1647: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM6ag')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1699: -1.0, Cpd1645: 1.0, Cpd1698: 1.0})
reaction.gene_reaction_rule = '( MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM6B1ag')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1699: -1.0, Cpd1698: 1.0, Cpd1645: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM6B1bg')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1699: -1.0, Cpd1698: 1.0, Cpd1645: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM6B2g')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1699: -1.0, Cpd1645: 1.0, Cpd1698: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM6bg')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1699: -1.0, Cpd1645: 1.0, Cpd1698: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM7Ag')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1700: -1.0, Cpd1645: 1.0, Cpd1699: 1.0})
reaction.gene_reaction_rule = '( MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM7B1g')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1701: -1.0, Cpd1645: 1.0, Cpd1699: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM7B2g')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1701: -1.0, Cpd1699: 1.0, Cpd1645: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM7Cag')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1700: -1.0, Cpd1699: 1.0, Cpd1645: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM7Cbg')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1702: -1.0, Cpd1699: 1.0, Cpd1645: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM8Ag')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1703: -1.0, Cpd1700: 1.0, Cpd1645: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MM8Ber')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, endoplasmic reticulum'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1704: -1.0, Cpd1705: 1.0, Cpd1660: 1.0})
reaction.gene_reaction_rule = '( MAN1B1 )'
model.add_reactions([reaction])
reaction = Reaction('MM8Cg')
reaction.name = 'mannosyl-oligosaccharide 1,2-alpha-mannosidase, Golgi apparatus'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1703: -1.0, Cpd1645: 1.0, Cpd1702: 1.0})
reaction.gene_reaction_rule = '( MAN1A or MAN1A2 or MAN1C1 )'
model.add_reactions([reaction])
reaction = Reaction('MMCD')
reaction.name = 'Methylmalonyl-CoA decarboxylase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1706: -1.0, Cpd60: 1.0, Cpd207: 1.0})
reaction.gene_reaction_rule = '( MLYCD )'
model.add_reactions([reaction])
reaction = Reaction('MMCDm')
reaction.name = 'Methylmalonyl-CoA decarboxylase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd1707: -1.0, Cpd45: 1.0, Cpd179: 1.0})
reaction.gene_reaction_rule = '( MLYCD )'
model.add_reactions([reaction])
reaction = Reaction('MMCDp')
reaction.name = 'Methylmalonyl-CoA decarboxylase, peroxisomal'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1708: -1.0, Cpd943: 1.0, Cpd742: 1.0})
reaction.gene_reaction_rule = '( MLYCD )'
model.add_reactions([reaction])
reaction = Reaction('MMEm')
reaction.name = 'methylmalonyl-CoA epimerase/racemase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1709: -1.0, Cpd1707: 1.0})
reaction.gene_reaction_rule = '( MCEE )'
model.add_reactions([reaction])
reaction = Reaction('MMMm')
reaction.name = 'methylmalonyl-CoA mutase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1709: -1.0, Cpd380: 1.0})
reaction.gene_reaction_rule = '( MUT )'
model.add_reactions([reaction])
reaction = Reaction('MMSAD1m')
reaction.name = 'methylmalonate-semialdehyde dehydrogenase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd44: -1.0, Cpd62: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd179: 1.0})
reaction.gene_reaction_rule = '( ALDH6A1 )'
model.add_reactions([reaction])
reaction = Reaction('MMSAD3m')
reaction.name = 'methylmalonate-semialdehyde dehydrogenase (malonic semialdehyde), mitochondrial'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd44: -1.0, Cpd454: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd178: 1.0})
reaction.gene_reaction_rule = '( ALDH6A1 )'
model.add_reactions([reaction])
reaction = Reaction('MOGAT')
reaction.name = 'monoacylglycerol acyltransferase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd843: -1.0, Cpd1622: -1.0, Cpd78: 1.0, Cpd658: 1.0})
reaction.gene_reaction_rule = '( MOGAT1 or MOGAT2 )'
model.add_reactions([reaction])
reaction = Reaction('MTAP')
reaction.name = '5-methylthioadenosine phosphorylase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd1710: -1.0, Cpd280: 1.0, Cpd1711: 1.0})
reaction.gene_reaction_rule = '( MTAP )'
model.add_reactions([reaction])
reaction = Reaction('MTHFC')
reaction.name = 'methenyltetrahydrofolate cyclohydrolase'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1146: -1.0, Cpd375: 1.0, Cpd33: 1.0})
reaction.gene_reaction_rule = '( MTHFD1 or MTHFD2L )'
model.add_reactions([reaction])
reaction = Reaction('MTHFCm')
reaction.name = 'methenyltetrahydrifikate cyclohydrolase, mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1712: -1.0, Cpd1134: 1.0, Cpd7: 1.0})
reaction.gene_reaction_rule = '( MTHFD1L or MTHFD2 )'
model.add_reactions([reaction])
reaction = Reaction('MTHFD')
reaction.name = 'methylenetetrahydrofolate dehydrogenase (NADP)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1341: -1.0, Cpd22: 1.0, Cpd1146: 1.0})
reaction.gene_reaction_rule = '( MTHFD1 or MTHFR )'
model.add_reactions([reaction])
reaction = Reaction('MTHFD2')
reaction.name = 'methylenetetrahydrofolate dehydrogenase (NAD)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1341: -1.0, Cpd35: 1.0, Cpd1146: 1.0})
reaction.gene_reaction_rule = '( MTHFD2 or MTHFD2L )'
model.add_reactions([reaction])
reaction = Reaction('MTHFD2m')
reaction.name = 'methylenetetrahydrofolate dehydrogenase (NAD), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1300: -1.0, Cpd47: 1.0, Cpd1712: 1.0})
reaction.gene_reaction_rule = '( MTHFD2 )'
model.add_reactions([reaction])
reaction = Reaction('MTHFDm')
reaction.name = 'methylenetetrahydrofolate dehydrogenase (NADP), mitochondrial'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd1300: -1.0, Cpd8: 1.0, Cpd1712: 1.0})
reaction.gene_reaction_rule = '( MTHFD1 or MTHFD1L or MTHFD2 )'
model.add_reactions([reaction])
reaction = Reaction('N3Tg')
reaction.name = 'N-acetylgalactosamine 3-beta-galactosyltransferase, Golgi apparatus'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd727: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd143: 1.0})
reaction.gene_reaction_rule = '( C1GALT1 )'
model.add_reactions([reaction])
reaction = Reaction('NABTNO')
reaction.name = 'N4-Acetylaminobutanal:NAD+ oxidoreductase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd456: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd1713: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('NABTNOm')
reaction.name = 'N4-Acetylaminobutanal:NAD+ oxidoreductase (m)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd1714: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd1715: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('NACASPAH')
reaction.name = 'N-Acetyl-L-aspartate amidohydrolase'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1716: -1.0, Cpd90: 1.0, Cpd221: 1.0})
reaction.gene_reaction_rule = '( ACY3 or ASPA )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX10ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1717: -1.0, Cpd373: 2.0, Cpd1237: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX11ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1718: -1.0, Cpd373: 1.0, Cpd1239: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX12ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1719: -1.0, Cpd373: 1.0, Cpd1241: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX13ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1720: -1.0, Cpd373: 1.0, Cpd1243: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX14ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1721: -1.0, Cpd373: 1.0, Cpd1245: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX15ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1722: -1.0, Cpd373: 1.0, Cpd1247: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX16ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1723: -1.0, Cpd373: 1.0, Cpd1211: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX17ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1724: -1.0, Cpd373: 1.0, Cpd1214: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX18ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1725: -1.0, Cpd373: 1.0, Cpd1216: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX19ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1726: -1.0, Cpd373: 1.0, Cpd1218: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX1ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1727: -1.0, Cpd1728: 1.0, Cpd1359: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX20ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1729: -1.0, Cpd373: 1.0, Cpd1220: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX21ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1221: -1.0, Cpd373: 1.0, Cpd1222: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX22ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1223: -1.0, Cpd131: 1.0, Cpd373: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX23ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1730: -1.0, Cpd373: 1.0, Cpd1226: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX24ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1731: -1.0, Cpd373: 1.0, Cpd1228: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX25ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1229: -1.0, Cpd373: 1.0, Cpd1230: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX26ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1732: -1.0, Cpd373: 2.0, Cpd1226: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX27ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Hyaluronan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1368: -1.0, Cpd373: 1.0, Cpd1369: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX2ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1733: -1.0, Cpd1728: 1.0, Cpd1734: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX3ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1735: -1.0, Cpd1728: 1.0, Cpd1736: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX4ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1737: -1.0, Cpd1728: 1.0, Cpd1361: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX5ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1738: -1.0, Cpd1728: 1.0, Cpd1739: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX6ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1740: -1.0, Cpd1728: 1.0, Cpd1741: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX7ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1742: -1.0, Cpd1728: 1.0, Cpd1743: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX8ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1744: -1.0, Cpd1728: 1.0, Cpd1365: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEX9ly')
reaction.name = 'beta-N-acetylhexosaminidase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1745: -1.0, Cpd1728: 1.0, Cpd1746: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA10ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1238: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1239: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA11ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1240: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1241: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA12ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1242: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1243: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA13ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1244: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1245: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA14ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1246: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1247: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA15ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1248: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1211: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA16ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1213: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1214: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA17ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1215: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1216: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA18ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1217: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1218: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA19ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1219: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1220: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA1ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1747: -1.0, Cpd176: 1.0, Cpd1728: 1.0, Cpd475: 1.0, Cpd1359: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA20ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1225: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1226: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA21ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1227: -1.0, Cpd176: 1.0, Cpd373: 1.0, Cpd475: 1.0, Cpd1228: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA22ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -3.0, Cpd1234: -1.0, Cpd176: 1.0, Cpd373: 2.0, Cpd475: 1.0, Cpd1226: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA2ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1748: -1.0, Cpd176: 1.0, Cpd1728: 1.0, Cpd475: 1.0, Cpd1736: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA3ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1749: -1.0, Cpd176: 1.0, Cpd1728: 1.0, Cpd475: 1.0, Cpd1361: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA4ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1362: -1.0, Cpd176: 1.0, Cpd1728: 1.0, Cpd475: 1.0, Cpd1739: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA5ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1750: -1.0, Cpd176: 1.0, Cpd1728: 1.0, Cpd475: 1.0, Cpd1741: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA6ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd1364: -1.0, Cpd176: 1.0, Cpd1728: 1.0, Cpd475: 1.0, Cpd1743: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA7ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -3.0, Cpd1751: -1.0, Cpd176: 2.0, Cpd1728: 1.0, Cpd475: 2.0, Cpd1365: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA8ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -3.0, Cpd1366: -1.0, Cpd176: 2.0, Cpd1728: 1.0, Cpd475: 2.0, Cpd1746: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NACHEXA9ly')
reaction.name = 'beta-N-acetylhexosaminidase A, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -3.0, Cpd1236: -1.0, Cpd176: 1.0, Cpd373: 2.0, Cpd475: 1.0, Cpd1237: 1.0})
reaction.gene_reaction_rule = '( HEXA or HEXB )'
model.add_reactions([reaction])
reaction = Reaction('NADH2_u10m')
reaction.name = 'NADH dehydrogenase, mitochondrial'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -5.0, Cpd47: -1.0, Cpd723: -1.0, Cpd33: 4.0, Cpd44: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( 1700029P11RIK or MT-ND1 or MT-ND2 or MT-ND3 or MT-ND4 or MT-ND4L or MT-ND5 or MT-ND6 or NDUFA1 or NDUFA10 or NDUFA11 or NDUFA12 or NDUFA13 or NDUFA2 or NDUFA3 or NDUFA4 or NDUFA5 or NDUFA6 or NDUFA7 or NDUFA8 or NDUFA9 or NDUFAB1 or NDUFB10 or NDUFB2 or NDUFB3 or NDUFB4 or NDUFB5 or NDUFB6 or NDUFB7 or NDUFB8 or NDUFB9 or NDUFC1 or NDUFC2 or NDUFS1 or NDUFS2 or NDUFS3 or NDUFS4 or NDUFS5 or NDUFS6 or NDUFS7 or NDUFS8 or NDUFV1 or NDUFV2 or NDUFV3 or TUSC3 )'
model.add_reactions([reaction])
reaction = Reaction('NADK')
reaction.name = 'NAD kinase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd32: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd25: 1.0})
reaction.gene_reaction_rule = '( NADK )'
model.add_reactions([reaction])
reaction = Reaction('NADNe')
reaction.name = 'NAD nucleosidase,extracellular'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1752: -1.0, Cpd37: 1.0, Cpd300: 1.0, Cpd1753: 1.0})
reaction.gene_reaction_rule = '( CD38 )'
model.add_reactions([reaction])
reaction = Reaction('NADPNe')
reaction.name = 'NADP nucleosidase,extracellular'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1754: -1.0, Cpd37: 1.0, Cpd1755: 1.0, Cpd1753: 1.0})
reaction.gene_reaction_rule = '( CD38 )'
model.add_reactions([reaction])
reaction = Reaction('NADS2')
reaction.name = 'NAD synthase (glutamine-hydrolysing)'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd399: -1.0, Cpd1756: -1.0, Cpd33: 1.0, Cpd32: 1.0, Cpd74: 1.0, Cpd150: 1.0, Cpd151: 1.0})
reaction.gene_reaction_rule = '( NADSYN1 )'
model.add_reactions([reaction])
reaction = Reaction('NAGA2ly')
reaction.name = 'N-acetylgalactosaminidase, alpha-'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd1757: -1.0, Cpd1758: -1.0, Cpd1759: 1.0, Cpd1760: 1.0})
reaction.gene_reaction_rule = '( NAGA )'
model.add_reactions([reaction])
reaction = Reaction('NAGAly')
reaction.name = 'N-acetylgalactosaminidase, alpha-'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd1761: -1.0, Cpd1758: -1.0, Cpd1762: 1.0, Cpd1760: 1.0})
reaction.gene_reaction_rule = '( NAGA )'
model.add_reactions([reaction])
reaction = Reaction('NAHCO3_HCLt')
reaction.name = 'bicarbonate transport (HCl/NaHCO3 exchange)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd38: -1.0, Cpd39: -1.0, Cpd700: -1.0, Cpd37: 1.0, Cpd40: 1.0, Cpd41: 1.0, Cpd698: 1.0})
reaction.gene_reaction_rule = '( SLC4A10 or SLC4A8 )'
model.add_reactions([reaction])
reaction = Reaction('NAIt')
reaction.name = 'Na+ / iodide cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd703: -1.0, Cpd41: 2.0, Cpd702: 1.0})
reaction.gene_reaction_rule = '( SLC5A5 )'
model.add_reactions([reaction])
reaction = Reaction('NaKt')
reaction.name = 'Na+/K+ exchanging ATPase'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd41: -1.0, Cpd489: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd39: 1.0, Cpd487: 1.0})
reaction.gene_reaction_rule = '( ATP1A1 or ATP1A2 or ATP1A3 or ATP1A4 or ATP1B1 or ATP1B2 or ATP1B3 or ATP1B4 )'
model.add_reactions([reaction])
reaction = Reaction('NAt')
reaction.name = 'sodium transport (uniport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd41: 1.0})
reaction.gene_reaction_rule = '( SLC5A1 or SLC5A3 or SLC5A5 )'
model.add_reactions([reaction])
reaction = Reaction('NAt3_1')
reaction.name = 'sodium proton antiporter (H:NA is 1:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd41: -1.0, Cpd33: 1.0, Cpd39: 1.0})
reaction.gene_reaction_rule = '( SLC9A1 or SLC9A2 or SLC9A3 or SLC9A4 or SLC9A5 )'
model.add_reactions([reaction])
reaction = Reaction('NAt3_1g')
reaction.name = 'sodium proton antiporter (H:NA is 1:1)'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd140: -1.0, Cpd33: 1.0, Cpd1763: 1.0})
reaction.gene_reaction_rule = '( SLC9A7 or SLC9A8 )'
model.add_reactions([reaction])
reaction = Reaction('NAt5')
reaction.name = 'sodium/ammonium proton antiporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd278: -1.0, Cpd6: 1.0, Cpd39: 1.0})
reaction.gene_reaction_rule = '( SLC9A1 or SLC9A2 or SLC9A3 )'
model.add_reactions([reaction])
reaction = Reaction('NBAHH_ir')
reaction.name = 'Nalpha-(beta-alanyl)-L-histidine hydrolase IR'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1764: -1.0, Cpd483: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( CNDP1 or CNDP2 )'
model.add_reactions([reaction])
reaction = Reaction('NCCt')
reaction.name = 'Na+-Cl- cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd698: -1.0, Cpd41: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC12A3 )'
model.add_reactions([reaction])
reaction = Reaction('NCKt')
reaction.name = 'Na+/Ca2+-K+ exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -4.0, Cpd489: -1.0, Cpd630: -1.0, Cpd39: 4.0, Cpd487: 1.0, Cpd629: 1.0})
reaction.gene_reaction_rule = '( SLC24A1 or SLC24A2 or SLC24A3 or SLC24A4 )'
model.add_reactions([reaction])
reaction = Reaction('NCNt')
reaction.name = 'Na+/Ca2+-NH4+ exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -4.0, Cpd278: -1.0, Cpd630: -1.0, Cpd6: 1.0, Cpd39: 4.0, Cpd629: 1.0})
reaction.gene_reaction_rule = '( SLC24A1 or SLC24A2 )'
model.add_reactions([reaction])
reaction = Reaction('NDP10ex')
reaction.name = 'nucleoside-diphosphatase (IDP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1765: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd1766: 1.0})
reaction.gene_reaction_rule = '( CANT1 or ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('NDP3ex')
reaction.name = 'nucleoside-diphosphatase (GDP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1767: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd1768: 1.0})
reaction.gene_reaction_rule = '( CANT1 or ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('NDP6')
reaction.name = 'nucleoside-diphosphatase (dCDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd777: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd776: 1.0})
reaction.gene_reaction_rule = '( DCTPP1 )'
model.add_reactions([reaction])
reaction = Reaction('NDP7er')
reaction.name = 'nucleoside-diphosphatase (UDP), endoplasmic reticulum'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1769: -1.0, Cpd565: 1.0, Cpd915: 1.0, Cpd1770: 1.0})
reaction.gene_reaction_rule = '( CANT1 or ENTPD4 or ENTPD5 )'
model.add_reactions([reaction])
reaction = Reaction('NDP7ex')
reaction.name = 'nucleoside-diphosphatase (UDP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1771: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd1772: 1.0})
reaction.gene_reaction_rule = '( CANT1 or ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('NDP7g')
reaction.name = 'nucleoside-diphosphatase (UDP), Golgi apparatus'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd142: -1.0, Cpd525: -1.0, Cpd140: 1.0, Cpd1773: 1.0, Cpd1774: 1.0})
reaction.gene_reaction_rule = '( ENTPD4 )'
model.add_reactions([reaction])
reaction = Reaction('NDP8')
reaction.name = 'nucleoside-diphosphatase (dUDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd899: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd810: 1.0})
reaction.gene_reaction_rule = '( DTYMK )'
model.add_reactions([reaction])
reaction = Reaction('NDP8ex')
reaction.name = 'nucleoside-diphosphatase (UTP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1775: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd1771: 1.0})
reaction.gene_reaction_rule = '( CANT1 or ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK1')
reaction.name = 'nucleoside-diphosphate kinase (ATP:GDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd283: -1.0, Cpd107: 1.0, Cpd282: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK10')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dIDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1776: -1.0, Cpd107: 1.0, Cpd1777: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK10m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dIDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1778: -1.0, Cpd202: 1.0, Cpd1779: 1.0})
reaction.gene_reaction_rule = '( NME4 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK10n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dIDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1780: -1.0, Cpd789: 1.0, Cpd1781: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK1m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:GDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd285: -1.0, Cpd202: 1.0, Cpd284: 1.0})
reaction.gene_reaction_rule = '( NME4 or NME6 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK1n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:GDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd791: -1.0, Cpd789: 1.0, Cpd790: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK2')
reaction.name = 'nucleoside-diphosphate kinase (ATP:UDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd137: -1.0, Cpd107: 1.0, Cpd749: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK2m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:UDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1782: -1.0, Cpd202: 1.0, Cpd1783: 1.0})
reaction.gene_reaction_rule = '( NME4 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK2n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:UDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd785: -1.0, Cpd784: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK3')
reaction.name = 'nucleoside-diphosphate kinase (ATP:CDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd769: -1.0, Cpd107: 1.0, Cpd653: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK3m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:CDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd787: -1.0, Cpd202: 1.0, Cpd654: 1.0})
reaction.gene_reaction_rule = '( NME4 or NME6 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK3n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:CDP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd774: -1.0, Cpd716: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK4')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dTDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd900: -1.0, Cpd107: 1.0, Cpd905: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK4m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dTDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd898: -1.0, Cpd202: 1.0, Cpd906: 1.0})
reaction.gene_reaction_rule = '( NME4 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK4n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dTDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1784: -1.0, Cpd789: 1.0, Cpd1785: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK5')
reaction.name = 'Nucleoside-diphosphate kinase (ATP:dGDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd771: -1.0, Cpd107: 1.0, Cpd770: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK5m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dGDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd897: -1.0, Cpd202: 1.0, Cpd908: 1.0})
reaction.gene_reaction_rule = '( NME4 or NME6 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK5n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dGDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd775: -1.0, Cpd773: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK6')
reaction.name = 'Nucleoside-diphosphate kinase (ATP:dUDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd899: -1.0, Cpd107: 1.0, Cpd903: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK6m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dUDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd901: -1.0, Cpd202: 1.0, Cpd904: 1.0})
reaction.gene_reaction_rule = '( NME4 or NME6 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK6n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dUDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1786: -1.0, Cpd789: 1.0, Cpd957: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK7')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dCDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd777: -1.0, Cpd107: 1.0, Cpd780: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK7m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dCDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd895: -1.0, Cpd202: 1.0, Cpd907: 1.0})
reaction.gene_reaction_rule = '( NME4 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK7n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dCDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd779: -1.0, Cpd781: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK8')
reaction.name = 'Nucleoside-diphosphate kinase (ATP:dADP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd288: -1.0, Cpd107: 1.0, Cpd287: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK8m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dADP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd896: -1.0, Cpd202: 1.0, Cpd902: 1.0})
reaction.gene_reaction_rule = '( NME4 or NME6 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK8n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:dADP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd783: -1.0, Cpd782: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK9')
reaction.name = 'nucleoside-diphosphate kinase (ATP:IDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1787: -1.0, Cpd107: 1.0, Cpd1788: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 or NME3 or NME6 or NME7 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK9m')
reaction.name = 'nucleoside-diphosphate kinase (ATP:IDP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1789: -1.0, Cpd202: 1.0, Cpd1790: 1.0})
reaction.gene_reaction_rule = '( NME4 )'
model.add_reactions([reaction])
reaction = Reaction('NDPK9n')
reaction.name = 'nucleoside-diphosphate kinase (ATP:IDP), nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd1791: -1.0, Cpd789: 1.0, Cpd1792: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('NH4t3r')
reaction.name = 'ammonia transport via proton antiport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd6: -1.0, Cpd37: -1.0, Cpd33: 1.0, Cpd278: 1.0})
reaction.gene_reaction_rule = '( RHAG or RHBG )'
model.add_reactions([reaction])
reaction = Reaction('NKCC2t')
reaction.name = 'Na+-K+-Cl- cotransport (NH4+)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd278: -1.0, Cpd698: -2.0, Cpd6: 1.0, Cpd41: 1.0, Cpd700: 2.0})
reaction.gene_reaction_rule = '( SLC12A1 or SLC12A2 )'
model.add_reactions([reaction])
reaction = Reaction('NKCCt')
reaction.name = 'Na+-K+-Cl- cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd489: -1.0, Cpd698: -2.0, Cpd41: 1.0, Cpd487: 1.0, Cpd700: 2.0})
reaction.gene_reaction_rule = '( SLC12A1 or SLC12A2 )'
model.add_reactions([reaction])
reaction = Reaction('NMNATm')
reaction.name = 'nicotinamide-nucleotide adenylyltransferase, mitochondrial'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd200: -1.0, Cpd1793: -1.0, Cpd44: 1.0, Cpd210: 1.0})
reaction.gene_reaction_rule = '( NMNAT3 )'
model.add_reactions([reaction])
reaction = Reaction('NMNATn')
reaction.name = 'nicotinamide-nucleotide adenylyltransferase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd763: -1.0, Cpd1794: -1.0, Cpd718: 1.0, Cpd1795: 1.0})
reaction.gene_reaction_rule = '( NMNAT1 )'
model.add_reactions([reaction])
reaction = Reaction('NMNATr')
reaction.name = 'nicotinamide-nucleotide adenylyltransferase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd1796: -1.0, Cpd32: 1.0, Cpd151: 1.0})
reaction.gene_reaction_rule = '( NMNAT2 or NMNAT3 )'
model.add_reactions([reaction])
reaction = Reaction('NMNS')
reaction.name = 'NMN synthetase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd301: -1.0, Cpd1797: -1.0, Cpd151: 1.0, Cpd1796: 1.0})
reaction.gene_reaction_rule = '( NAMPT )'
model.add_reactions([reaction])
reaction = Reaction('NMPTRCOX')
reaction.name = 'N-Methylputrescine:oxygen oxidoreductase (deaminating)'
reaction.subsystem = 'Alkaloid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd1798: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd1799: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('NNATm')
reaction.name = 'nicotinate-nucleotide adenylyltransferase, mitochondrial'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd200: -1.0, Cpd1800: -1.0, Cpd210: 1.0, Cpd1801: 1.0})
reaction.gene_reaction_rule = '( NMNAT3 )'
model.add_reactions([reaction])
reaction = Reaction('NNATn')
reaction.name = 'nicotinate-nucleotide adenylyltransferase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd763: -1.0, Cpd1802: -1.0, Cpd718: 1.0, Cpd1803: 1.0})
reaction.gene_reaction_rule = '( NMNAT1 )'
model.add_reactions([reaction])
reaction = Reaction('NNATr')
reaction.name = 'nicotinate-nucleotide adenylyltransferase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd1804: -1.0, Cpd151: 1.0, Cpd1756: 1.0})
reaction.gene_reaction_rule = '( NMNAT1 or NMNAT2 or NMNAT3 )'
model.add_reactions([reaction])
reaction = Reaction('NNMT')
reaction.name = 'Nicotinamide N-methyltransferase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd1797: -1.0, Cpd1805: 1.0, Cpd53: 1.0})
reaction.gene_reaction_rule = '( NNMT )'
model.add_reactions([reaction])
reaction = Reaction('NORANMT')
reaction.name = 'noradrenaline N-methyltransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd98: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd96: 1.0})
reaction.gene_reaction_rule = '( PNMT )'
model.add_reactions([reaction])
reaction = Reaction('NOS1')
reaction.name = 'Nitric Oxide Synthase (intermediate forming)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd467: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1806: 1.0})
reaction.gene_reaction_rule = '( NOS1 or NOS2 or NOS3 )'
model.add_reactions([reaction])
reaction = Reaction('NOS2')
reaction.name = 'Nitric Oxide Synthase (NO forming)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd22: -1.0, Cpd1806: -1.0, Cpd2: 1.0, Cpd33: 1.0, Cpd25: 1.0, Cpd471: 1.0, Cpd1807: 1.0})
reaction.gene_reaction_rule = '( NOS1 or NOS2 or NOS3 )'
model.add_reactions([reaction])
reaction = Reaction('NP1')
reaction.name = 'nucleotide phosphatase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1808: -1.0, Cpd1809: -1.0, Cpd108: 1.0, Cpd1810: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('NRPPHRSULT')
reaction.name = 'Norepinephrine Sulfotransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd98: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd1811: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('NRPPHRt4_2_r')
reaction.name = 'Norepinephrine reversible transport in via sodium symport (1:2)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd1812: -1.0, Cpd41: 2.0, Cpd98: 1.0})
reaction.gene_reaction_rule = '( SLC6A2 or SLC6A3 )'
model.add_reactions([reaction])
reaction = Reaction('NRPPHRtu')
reaction.name = 'Norepinephrine uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1812: -1.0, Cpd98: 1.0})
reaction.gene_reaction_rule = '( SLC22A1 or SLC22A2 or SLC22A3 )'
model.add_reactions([reaction])
reaction = Reaction('NRPPHRVESSEC')
reaction.name = 'Noradrenaline secretion via secretory vesicle (ATP driven)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd98: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1812: 1.0})
reaction.gene_reaction_rule = '( SLC18A1 or SLC18A2 )'
model.add_reactions([reaction])
reaction = Reaction('NRVNCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1043: -1.0, Cpd78: 1.0, Cpd1813: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('NRVNCCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd1814: -1.0, Cpd307: 1.0, Cpd1815: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('NRVNCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1813: -1.0, Cpd1814: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('NS26T2g')
reaction.name = 'alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd1816: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1817: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC1 )'
model.add_reactions([reaction])
reaction = Reaction('NS26Tg')
reaction.name = 'alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd727: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1818: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC1 )'
model.add_reactions([reaction])
reaction = Reaction('NT5C')
reaction.name = 'Nicotinate D-ribonucleotide phosphohydrolase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1804: -1.0, Cpd108: 1.0, Cpd1810: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD1')
reaction.name = '5-nucleotidase (dUMP)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd810: -1.0, Cpd108: 1.0, Cpd818: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C3 )'
model.add_reactions([reaction])
reaction = Reaction('NTD10')
reaction.name = '5-nucleotidase (XMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1420: -1.0, Cpd108: 1.0, Cpd1819: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD11')
reaction.name = ' 5-nucleotidase (IMP)'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd316: -1.0, Cpd108: 1.0, Cpd275: 1.0})
reaction.gene_reaction_rule = '( NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD12')
reaction.name = '5-nucleotidase (dIMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1820: -1.0, Cpd108: 1.0, Cpd794: 1.0})
reaction.gene_reaction_rule = '( NT5C or PHEX )'
model.add_reactions([reaction])
reaction = Reaction('NTD1m')
reaction.name = '5-nucleotidase (dUMP),mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd954: -1.0, Cpd204: 1.0, Cpd953: 1.0})
reaction.gene_reaction_rule = '( NT5M )'
model.add_reactions([reaction])
reaction = Reaction('NTD2')
reaction.name = '5-nucleotidase (UMP)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1398: -1.0, Cpd108: 1.0, Cpd761: 1.0})
reaction.gene_reaction_rule = '( NT5C3 )'
model.add_reactions([reaction])
reaction = Reaction('NTD2e')
reaction.name = '5-nucleotidase (UMP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1772: -1.0, Cpd494: 1.0, Cpd1821: 1.0})
reaction.gene_reaction_rule = '( NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD2l')
reaction.name = '5-nucleotidase (UMP), lysosome'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1822: -1.0, Cpd88: 1.0, Cpd1823: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTD2m')
reaction.name = '5-nucleotidase (UMP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1824: -1.0, Cpd204: 1.0, Cpd1825: 1.0})
reaction.gene_reaction_rule = '( NT5M )'
model.add_reactions([reaction])
reaction = Reaction('NTD3')
reaction.name = ' 5-nucleotidase (dCMP)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd776: -1.0, Cpd108: 1.0, Cpd817: 1.0})
reaction.gene_reaction_rule = '( GUCA1A or NT5C or NT5C1A or NT5C1B or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD3l')
reaction.name = '5-nucleotidase (dCMP), lysosomal'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1826: -1.0, Cpd88: 1.0, Cpd1827: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTD4')
reaction.name = ' 5-nucleotidase (CMP)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd651: -1.0, Cpd108: 1.0, Cpd760: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD4e')
reaction.name = '5-nucleotidase (CMP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1828: -1.0, Cpd494: 1.0, Cpd766: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD4l')
reaction.name = '5-nucleotidase (CMP), lysosome'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1829: -1.0, Cpd88: 1.0, Cpd767: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTD5')
reaction.name = ' 5-nucleotidase (dTMP)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd950: -1.0, Cpd108: 1.0, Cpd1830: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD5l')
reaction.name = '5-nucleotidase (dTMP), lysosomal'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1831: -1.0, Cpd88: 1.0, Cpd1832: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTD5m')
reaction.name = '5-nucleotidase (dTMP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1833: -1.0, Cpd204: 1.0, Cpd1834: 1.0})
reaction.gene_reaction_rule = '( NT5M )'
model.add_reactions([reaction])
reaction = Reaction('NTD6')
reaction.name = '5-nucleotidase (dAMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd286: -1.0, Cpd108: 1.0, Cpd793: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD6l')
reaction.name = '5-nucleotidase (dAMP), lysosome'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1835: -1.0, Cpd88: 1.0, Cpd1836: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTD7')
reaction.name = ' 5-nucleotidase (AMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd150: -1.0, Cpd108: 1.0, Cpd274: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 )'
model.add_reactions([reaction])
reaction = Reaction('NTD7e')
reaction.name = '5-nucleotidase (AMP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd493: -1.0, Cpd276: 1.0, Cpd494: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD7l')
reaction.name = '5-nucleotidase (AMP), lysosome'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1837: -1.0, Cpd87: 1.0, Cpd88: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTD8')
reaction.name = '5-nucleotidase (dGMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd837: -1.0, Cpd108: 1.0, Cpd841: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD8l')
reaction.name = '5-nucleotidase (dGMP), lysosomal'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1838: -1.0, Cpd88: 1.0, Cpd1839: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTD9')
reaction.name = ' 5-nucleotidase (GMP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1307: -1.0, Cpd108: 1.0, Cpd1438: 1.0})
reaction.gene_reaction_rule = '( NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD9e')
reaction.name = '5-nucleotidase (GMP), extracellular'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1768: -1.0, Cpd494: 1.0, Cpd1437: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5E )'
model.add_reactions([reaction])
reaction = Reaction('NTD9l')
reaction.name = '5-nucleotidase (GMP), lysosome'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1840: -1.0, Cpd88: 1.0, Cpd1439: 1.0})
reaction.gene_reaction_rule = '( ACP2 or ACP5 or ACP6 or ACPP )'
model.add_reactions([reaction])
reaction = Reaction('NTP3e')
reaction.name = 'nucleoside-triphosphatase (GTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1841: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd1767: 1.0})
reaction.gene_reaction_rule = '( CANT1 or ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('NTPP10')
reaction.name = 'Nucleoside triphosphate pyrophosphorylase (ditp)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1777: -1.0, Cpd33: 1.0, Cpd151: 1.0, Cpd1820: 1.0})
reaction.gene_reaction_rule = '( ITPA )'
model.add_reactions([reaction])
reaction = Reaction('NTPP11')
reaction.name = 'Nucleoside triphosphate pyrophosphorylase (xtp)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1842: -1.0, Cpd33: 1.0, Cpd151: 1.0, Cpd1420: 1.0})
reaction.gene_reaction_rule = '( ITPA )'
model.add_reactions([reaction])
reaction = Reaction('NTPP9')
reaction.name = 'Nucleoside triphosphate pyrophosphorylase (itp)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1788: -1.0, Cpd33: 1.0, Cpd151: 1.0, Cpd316: 1.0})
reaction.gene_reaction_rule = '( ITPA )'
model.add_reactions([reaction])
reaction = Reaction('O16G1e')
reaction.name = 'oligo-1,6-glucosidase (glygn4 -> glygn5), extracellular'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd449: -1.0, Cpd446: 1.0, Cpd1273: 1.0})
reaction.gene_reaction_rule = '( SIS )'
model.add_reactions([reaction])
reaction = Reaction('O16G2e')
reaction.name = 'oligo-1,6-glucosidase (strch2 ->strch3), extracellular'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd447: -1.0, Cpd446: 1.0, Cpd1274: 1.0})
reaction.gene_reaction_rule = '( SIS )'
model.add_reactions([reaction])
reaction = Reaction('OCBTm')
reaction.name = 'ornithine carbamoyltransferase, irreversible'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd470: -1.0, Cpd644: -1.0, Cpd7: 1.0, Cpd204: 1.0, Cpd691: 1.0})
reaction.gene_reaction_rule = '( OTC )'
model.add_reactions([reaction])
reaction = Reaction('OCDCAtr')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1096: -1.0, Cpd1014: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('OCDCEAtr')
reaction.name = 'fatty acid transport via diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1095: -1.0, Cpd1016: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('OCOAT1m')
reaction.name = '3-oxoacid CoA-transferase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd186: -1.0, Cpd380: -1.0, Cpd181: 1.0, Cpd1843: 1.0})
reaction.gene_reaction_rule = '( OXCT1 or OXCT2B )'
model.add_reactions([reaction])
reaction = Reaction('OIVD1m')
reaction.name = '2-oxoisovalerate dehydrogenase (acylating; 4-methyl-2-oxopentaoate), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd44: -1.0, Cpd1593: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd242: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or DBT or DLD )'
model.add_reactions([reaction])
reaction = Reaction('OIVD2m')
reaction.name = '2-oxoisovalerate dehydrogenase (acylating; 3-methyl-2-oxobutanoate), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd44: -1.0, Cpd1844: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd244: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or DBT or DLD )'
model.add_reactions([reaction])
reaction = Reaction('OIVD3m')
reaction.name = '2-oxoisovalerate dehydrogenase (acylating; 3-methyl-2-oxopentanoate), mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd44: -1.0, Cpd1557: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd237: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or DBT or DLD )'
model.add_reactions([reaction])
reaction = Reaction('OMPDC')
reaction.name = 'orotidine-5-phosphate decarboxylase'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1845: -1.0, Cpd60: 1.0, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( UMPS )'
model.add_reactions([reaction])
reaction = Reaction('OPAHir')
reaction.name = '5-Oxoproline amidohydrolase (ATP-hydrolysing) (ir)'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2.0, Cpd105: -1.0, Cpd1335: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd74: 1.0})
reaction.gene_reaction_rule = '( OPLAH )'
model.add_reactions([reaction])
reaction = Reaction('ORNDC')
reaction.name = 'Ornithine Decarboxylase'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd252: -1.0, Cpd60: 1.0, Cpd1846: 1.0})
reaction.gene_reaction_rule = '( ODC1 )'
model.add_reactions([reaction])
reaction = Reaction('ORNt3m')
reaction.name = 'ornithine mitochondrial transport via proton antiport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd470: -1.0, Cpd7: 1.0, Cpd252: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('ORNt4m')
reaction.name = 'ornithine mitochondrial transport exchange with citruline'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd470: -1.0, Cpd471: -1.0, Cpd7: 1.0, Cpd252: 1.0, Cpd691: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('ORNTArm')
reaction.name = 'ornithine transaminase reversible (m)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd470: -1.0, Cpd63: 1.0, Cpd1200: 1.0})
reaction.gene_reaction_rule = '( OAT )'
model.add_reactions([reaction])
reaction = Reaction('ORNtiDF')
reaction.name = 'ornithine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd252: 1.0})
reaction.gene_reaction_rule = '( SLC7A1 or SLC7A2 or SLC7A3 )'
model.add_reactions([reaction])
reaction = Reaction('ORPT')
reaction.name = 'orotate phosphoribosyltransferase'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd151: -1.0, Cpd1845: -1.0, Cpd301: 1.0, Cpd865: 1.0})
reaction.gene_reaction_rule = '( UMPS )'
model.add_reactions([reaction])
reaction = Reaction('OXAHCOtex')
reaction.name = 'oxalate transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -2.0, Cpd705: -1.0, Cpd38: 2.0, Cpd704: 1.0})
reaction.gene_reaction_rule = '( SLC26A1 or SLC26A2 or SLC26A3 )'
model.add_reactions([reaction])
reaction = Reaction('P45011A1m')
reaction.name = 'Cytochrome P450 11A1, mitochondrial [Precursor]'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -2.0, Cpd686: -1.0, Cpd12: 2.0, Cpd13: 1.0, Cpd1848: 1.0, Cpd1849: 1.0})
reaction.gene_reaction_rule = '( CYP11A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45011B11m')
reaction.name = 'Steroid 11-beta-hydroxylase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1850: -1.0, Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd1851: 1.0})
reaction.gene_reaction_rule = '( CYP11B1 )'
model.add_reactions([reaction])
reaction = Reaction('P45011B12m')
reaction.name = 'Steroid 11-beta-hydroxylase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1852: -1.0, Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd1853: 1.0})
reaction.gene_reaction_rule = '( CYP11B1 )'
model.add_reactions([reaction])
reaction = Reaction('P45011B21m')
reaction.name = 'Steroid 11-beta-hydroxylase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1850: -1.0, Cpd7: -1.0, Cpd8: -1.0, Cpd9: -2.0, Cpd12: 2.0, Cpd13: 1.0, Cpd1854: 1.0})
reaction.gene_reaction_rule = '( CYP11B1 )'
model.add_reactions([reaction])
reaction = Reaction('P45017A1r')
reaction.name = 'Cytochrome P450 17A1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1525: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1530: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45017A2r')
reaction.name = 'Cytochrome P450 17A1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -0.5, Cpd565: -1.0, Cpd587: -1.0, Cpd1530: -1.0, Cpd1855: 1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1527: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45017A3r')
reaction.name = 'Cytochrome P450 17A1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1526: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1531: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45017A4r')
reaction.name = 'Cytochrome P450 17A1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -0.5, Cpd565: -1.0, Cpd587: -1.0, Cpd1531: -1.0, Cpd1855: 1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1514: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45019A1r')
reaction.name = 'aromatase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd587: -1.0, Cpd1514: -1.0, Cpd437: 2.0, Cpd589: 1.0, Cpd1519: 1.0, Cpd1856: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45019A2r')
reaction.name = 'aromatase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd587: -1.0, Cpd1515: -1.0, Cpd437: 2.0, Cpd589: 1.0, Cpd1520: 1.0, Cpd1856: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 )'
model.add_reactions([reaction])
reaction = Reaction('P4501B1r')
reaction.name = 'cytochrome P450 1B1'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1520: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1857: 1.0})
reaction.gene_reaction_rule = '( CYP1B1 or CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('P45021A1r')
reaction.name = 'Steroid 21-hydroxylase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1526: -1.0, Cpd1858: 1.0, Cpd437: 1.0, Cpd589: 1.0})
reaction.gene_reaction_rule = '( CYP21A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45021A2r')
reaction.name = 'Steroid 21-hydroxylase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1531: -1.0, Cpd1859: 1.0, Cpd437: 1.0, Cpd589: 1.0})
reaction.gene_reaction_rule = '( CYP21A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45027A11m')
reaction.name = '5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol 27-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd1860: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd1861: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45027A12m')
reaction.name = '5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol 27-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd1861: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd1862: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45027A13m')
reaction.name = '5-beta-cholestane-3-alpha,7-alpha,12-alpha-triol 27-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd8: -1.0, Cpd9: -1.0, Cpd1862: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd1863: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45027A14m')
reaction.name = '5-beta-cytochrome P450, family 27, subfamily A, polypeptide 1'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd1864: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd1865: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45027A15m')
reaction.name = '5-beta-cytochrome P450, family 27, subfamily A, polypeptide 1'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd1865: -1.0, Cpd12: 2.0, Cpd13: 1.0, Cpd1866: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45027A16m')
reaction.name = 'Cytochrome P450 27'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd8: -1.0, Cpd9: -1.0, Cpd1866: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd1867: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('P45027A1m')
reaction.name = 'Cytochrome P450 27'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd686: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd1868: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('P4502A6')
reaction.name = 'cytochrome P450 2A6'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1869: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1870: 1.0})
reaction.gene_reaction_rule = '( CYP2A4 )'
model.add_reactions([reaction])
reaction = Reaction('P4502C18')
reaction.name = 'cytochrome P450 2C18'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -0.5, Cpd33: -1.0, Cpd22: -1.0, Cpd1871: -1.0, Cpd25: 1.0, Cpd410: 1.0, Cpd1872: 1.0})
reaction.gene_reaction_rule = '( CYP2C55 )'
model.add_reactions([reaction])
reaction = Reaction('P4502D6')
reaction.name = 'cytochrome P450 2D6'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd822: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1873: 1.0})
reaction.gene_reaction_rule = '( CYP2D26 )'
model.add_reactions([reaction])
reaction = Reaction('P4502E1')
reaction.name = 'cytochrome P450 2E1'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd103: -1.0, Cpd2: 1.0, Cpd1874: 1.0, Cpd25: 1.0})
reaction.gene_reaction_rule = '( CYP2E1 )'
model.add_reactions([reaction])
reaction = Reaction('P4502F1')
reaction.name = 'cytochrome P450 2F1'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1875: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1876: 1.0})
reaction.gene_reaction_rule = '( CYP2F2 )'
model.add_reactions([reaction])
reaction = Reaction('P45039A1r')
reaction.name = 'oxysterol 7-alpha-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1877: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1878: 1.0})
reaction.gene_reaction_rule = '( CYP39A1 )'
model.add_reactions([reaction])
reaction = Reaction('P4503A4')
reaction.name = 'cytochrome P450 3A4'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1879: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1880: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('P4503A43r')
reaction.name = 'cytochrome p450 P450 3A43'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1515: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1881: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('P4503A5')
reaction.name = 'cytochrome P450 3A5'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd1882: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1883: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('P4503A7r')
reaction.name = 'cytochrome p450 P450 3A7'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1515: -1.0, Cpd1884: 1.0, Cpd437: 1.0, Cpd589: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('P45046A1r')
reaction.name = 'cytochrome P450, family 46, subfamily A, polypeptide 1'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd670: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1877: 1.0})
reaction.gene_reaction_rule = '( CYP46A1 )'
model.add_reactions([reaction])
reaction = Reaction('P4504B1r')
reaction.name = 'cytochrome P450 4B1'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd1885: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1886: 1.0})
reaction.gene_reaction_rule = '( CYP2F2 )'
model.add_reactions([reaction])
reaction = Reaction('P4504F121r')
reaction.name = 'cytochrome p450 4F12/4F2'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd1885: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1887: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('P4504F122r')
reaction.name = 'cytochrome p450 4F12/4F2'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1888: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1889: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('P4504F123r')
reaction.name = 'cytochrome p450 4F12/4F2'
reaction.subsystem = 'Cytochrome metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1890: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1891: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('P4504F81r')
reaction.name = 'cytochrome p450 4F8'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd1885: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1892: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('P4507A1r')
reaction.name = 'cytochrome P450, family 7, subfamily A, polypeptide 1'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd670: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1532: 1.0})
reaction.gene_reaction_rule = '( CYP7A1 )'
model.add_reactions([reaction])
reaction = Reaction('P4507B11r')
reaction.name = 'oxysterol 7alpha-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd671: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1893: 1.0})
reaction.gene_reaction_rule = '( CYP7B1 )'
model.add_reactions([reaction])
reaction = Reaction('P4507B12r')
reaction.name = 'oxysterol 7alpha-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1894: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1895: 1.0})
reaction.gene_reaction_rule = '( CYP7B1 )'
model.add_reactions([reaction])
reaction = Reaction('P4508B11r')
reaction.name = 'sterol 12-alpha-hydroxylase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1533: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1896: 1.0})
reaction.gene_reaction_rule = '( CYP8B1 )'
model.add_reactions([reaction])
reaction = Reaction('P4508B13r')
reaction.name = 'sterol 12-alpha-hydroxylase (nadh)'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd626: -1.0, Cpd1533: -1.0, Cpd437: 1.0, Cpd625: 1.0, Cpd1896: 1.0})
reaction.gene_reaction_rule = '( CYP8B1 )'
model.add_reactions([reaction])
reaction = Reaction('P5CDm')
reaction.name = '1-pyrroline-5-carboxylate dehydrogenase, mitochondrial'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -2.0, Cpd44: -1.0, Cpd1897: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd63: 1.0})
reaction.gene_reaction_rule = '( ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('P5CRm')
reaction.name = 'pyrroline-5-carboxylate reductase (m)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -2.0, Cpd8: -1.0, Cpd1897: -1.0, Cpd13: 1.0, Cpd1898: 1.0})
reaction.gene_reaction_rule = '( PYCR1 )'
model.add_reactions([reaction])
reaction = Reaction('P5CRxm')
reaction.name = 'pyrroline-5-carboxylate reductase (m)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -2.0, Cpd47: -1.0, Cpd1897: -1.0, Cpd44: 1.0, Cpd1898: 1.0})
reaction.gene_reaction_rule = '( PYCR1 )'
model.add_reactions([reaction])
reaction = Reaction('PAFH')
reaction.name = 'Platelet-activating factor acetylhydrolase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1899: -1.0, Cpd33: 1.0, Cpd221: 1.0, Cpd1900: 1.0})
reaction.gene_reaction_rule = '( PAFAH1B1 or PAFAH1B2 or PAFAH1B3 or PAFAH2 )'
model.add_reactions([reaction])
reaction = Reaction('PAFHe')
reaction.name = 'Platelet-activating factor acetylhydrolase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1901: -1.0, Cpd37: 1.0, Cpd225: 1.0, Cpd1902: 1.0})
reaction.gene_reaction_rule = '( PLA2G7 )'
model.add_reactions([reaction])
reaction = Reaction('PAPStg')
reaction.name = '3-Phosphoadenylyl sufate Golgi transport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd1206: 1.0})
reaction.gene_reaction_rule = '( SLC35B2 )'
model.add_reactions([reaction])
reaction = Reaction('PCFLOPm')
reaction.name = 'phosphatidylcholine flippase'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1903: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd659: 1.0})
reaction.gene_reaction_rule = '( ATP10A )'
model.add_reactions([reaction])
reaction = Reaction('PCHOLP_hs')
reaction.name = 'choline phosphatase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd659: -1.0, Cpd33: 1.0, Cpd353: 1.0, Cpd672: 1.0})
reaction.gene_reaction_rule = '( PLD1 or PLD2 )'
model.add_reactions([reaction])
reaction = Reaction('PCHOLPg_hs')
reaction.name = 'choline phosphatase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1904: -1.0, Cpd140: 1.0, Cpd1905: 1.0, Cpd1906: 1.0})
reaction.gene_reaction_rule = '( PLD1 )'
model.add_reactions([reaction])
reaction = Reaction('PCHOLPm_hs')
reaction.name = 'choline phosphatase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1903: -1.0, Cpd7: 1.0, Cpd655: 1.0, Cpd680: 1.0})
reaction.gene_reaction_rule = '( PLD1 )'
model.add_reactions([reaction])
reaction = Reaction('PCHOLPr_hs')
reaction.name = 'choline phosphatase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1907: -1.0, Cpd565: 1.0, Cpd1908: 1.0, Cpd1909: 1.0})
reaction.gene_reaction_rule = '( PLD1 )'
model.add_reactions([reaction])
reaction = Reaction('PCLAD')
reaction.name = 'picolinic acid decarboxylase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd69: -1.0, Cpd60: 1.0, Cpd1910: 1.0})
reaction.gene_reaction_rule = '( ACMSD )'
model.add_reactions([reaction])
reaction = Reaction('PCLYSOX')
reaction.name = 'Procollagen-lysine 1, 2-oxoglutarate 5-dioxygenase'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd30: -1.0, Cpd1911: -1.0, Cpd60: 1.0, Cpd543: 1.0, Cpd1912: 1.0})
reaction.gene_reaction_rule = '( PLOD1 or PLOD2 or PLOD3 )'
model.add_reactions([reaction])
reaction = Reaction('PCm')
reaction.name = 'pyruvate carboxylase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd363: -1.0, Cpd200: -1.0, Cpd201: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd491: 1.0})
reaction.gene_reaction_rule = '( PCX )'
model.add_reactions([reaction])
reaction = Reaction('PCRNtm')
reaction.name = 'transport into the mitochondria from cytosol (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1913: -1.0, Cpd739: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('PCt')
reaction.name = 'phosphatidylcholine transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd659: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1580: 1.0})
reaction.gene_reaction_rule = '( ABCA1 or ABCB4 )'
model.add_reactions([reaction])
reaction = Reaction('PDE1')
reaction.name = '3,5-cyclic-nucleotide phosphodiesterase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd290: -1.0, Cpd33: 1.0, Cpd150: 1.0})
reaction.gene_reaction_rule = '( PDE10A or PDE11A or PDE1A or PDE1B or PDE1C or PDE2A or PDE3A or PDE3B or PDE4A or PDE4B or PDE4C or PDE4D or PDE7A or PDE7B or PDE8A or PDE8B )'
model.add_reactions([reaction])
reaction = Reaction('PDE1g')
reaction.name = '3,5-cyclic-nucleotide phosphodiesterase,Golgi'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1914: -1.0, Cpd140: 1.0, Cpd1915: 1.0})
reaction.gene_reaction_rule = '( PDE10A )'
model.add_reactions([reaction])
reaction = Reaction('PDE4')
reaction.name = '3,5-cyclic-nucleotide phosphodiesterase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd668: -1.0, Cpd33: 1.0, Cpd1307: 1.0})
reaction.gene_reaction_rule = '( PDE10A or PDE11A or PDE1A or PDE1B or PDE1C or PDE2A or PDE3A or PDE3B or PDE5A or PDE6A or PDE6B or PDE6C or PDE6D or PDE6G or PDE6H or PDE9A )'
model.add_reactions([reaction])
reaction = Reaction('PDE4g')
reaction.name = '3,5-cyclic-nucleotide phosphodiesterase, Golgi'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd1916: -1.0, Cpd140: 1.0, Cpd1306: 1.0})
reaction.gene_reaction_rule = '( PDE10A )'
model.add_reactions([reaction])
reaction = Reaction('PDE4n')
reaction.name = '3,5-cyclic-nucleotide phosphodiesterase, Nucleus'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1917: -1.0, Cpd570: -1.0, Cpd763: 1.0, Cpd1918: 1.0})
reaction.gene_reaction_rule = '( PDE9A )'
model.add_reactions([reaction])
reaction = Reaction('PDHm')
reaction.name = 'pyruvate dehydrogenase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd44: -1.0, Cpd363: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd178: 1.0})
reaction.gene_reaction_rule = '( DLAT or DLD or PDHA1 or PDHA2 or PDHB or PDHX )'
model.add_reactions([reaction])
reaction = Reaction('PDX5PO')
reaction.name = 'pyridoxine 5-phosphate oxidase'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd1919: -1.0, Cpd5: 1.0, Cpd1920: 1.0})
reaction.gene_reaction_rule = '( PNPO )'
model.add_reactions([reaction])
reaction = Reaction('PEAMNO')
reaction.name = 'Phenethylamine oxidase'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1921: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd1922: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('PEFLIP')
reaction.name = 'phosphatidylethanolamine flippase'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1923: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd661: 1.0})
reaction.gene_reaction_rule = '( ABCB4 or ATP10A or ATP8A1 )'
model.add_reactions([reaction])
reaction = Reaction('PEFLIPm')
reaction.name = 'phosphatidylethanolamine flippase'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd661: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1924: 1.0})
reaction.gene_reaction_rule = '( ATP10A or ATP8A1 )'
model.add_reactions([reaction])
reaction = Reaction('PEPCK')
reaction.name = 'Phosphoenolpyruvate carboxykinase (GTP)'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd229: -1.0, Cpd282: -1.0, Cpd60: 1.0, Cpd231: 1.0, Cpd283: 1.0})
reaction.gene_reaction_rule = '( PCK1 )'
model.add_reactions([reaction])
reaction = Reaction('PEPCKm')
reaction.name = 'Phosphoenolpyruvate carboxykinase (GTP)'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd284: -1.0, Cpd491: -1.0, Cpd45: 1.0, Cpd285: 1.0, Cpd693: 1.0})
reaction.gene_reaction_rule = '( PCK1 or PCK2 )'
model.add_reactions([reaction])
reaction = Reaction('PEROXx')
reaction.name = 'peroxisomal lumped long chain fatty acid oxidation'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -3.0, Cpd184: -3.0, Cpd246: -2.0, Cpd261: -3.0, Cpd1925: -1.0, Cpd358: 3.0, Cpd1065: 3.0, Cpd182: 1.0, Cpd247: 2.0, Cpd740: 1.0, Cpd742: 2.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX3 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('PETHCT')
reaction.name = 'phosphoethanolamine cytidyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd653: -1.0, Cpd996: -1.0, Cpd151: 1.0, Cpd660: 1.0})
reaction.gene_reaction_rule = '( PCYT2 )'
model.add_reactions([reaction])
reaction = Reaction('PETOHMm_hs')
reaction.name = 'phosphatidylethanolamine N-methyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd442: -3.0, Cpd1924: -1.0, Cpd7: 3.0, Cpd441: 3.0, Cpd1903: 1.0})
reaction.gene_reaction_rule = '( PEMT )'
model.add_reactions([reaction])
reaction = Reaction('PETOHMr_hs')
reaction.name = 'phosphatidylethanolamine N-methyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1926: -3.0, Cpd1452: -1.0, Cpd1927: 3.0, Cpd565: 3.0, Cpd1907: 1.0})
reaction.gene_reaction_rule = '( PEMT )'
model.add_reactions([reaction])
reaction = Reaction('PFK')
reaction.name = 'phosphofructokinase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1108: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1102: 1.0})
reaction.gene_reaction_rule = '( PFKL or PFKM or PFKP )'
model.add_reactions([reaction])
reaction = Reaction('PFK26')
reaction.name = '6-phosphofructo-2-kinase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1108: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1109: 1.0})
reaction.gene_reaction_rule = '( PFKFB1 or PFKFB2 or PFKFB3 or PFKFB4 )'
model.add_reactions([reaction])
reaction = Reaction('PGCD')
reaction.name = 'phosphoglycerate dehydrogenase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd273: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1928: 1.0})
reaction.gene_reaction_rule = '( PHGDH )'
model.add_reactions([reaction])
reaction = Reaction('PGDI')
reaction.name = 'Prostaglandin-H2 D-isomerase [Precursor]'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1929: -1.0, Cpd1930: 1.0})
reaction.gene_reaction_rule = '( HPGDS or PTGDS )'
model.add_reactions([reaction])
reaction = Reaction('PGDIr')
reaction.name = 'Prostaglandin-H2 D-isomerase [Precursor]'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1931: -1.0, Cpd1932: 1.0})
reaction.gene_reaction_rule = '( HPGDS or PTGDS )'
model.add_reactions([reaction])
reaction = Reaction('PGESr')
reaction.name = 'Prostaglandin E synthase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1931: -1.0, Cpd1933: 1.0})
reaction.gene_reaction_rule = '( PTGES or PTGES2 )'
model.add_reactions([reaction])
reaction = Reaction('PGI')
reaction.name = 'glucose-6-phosphate isomerase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1202: -1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( GPI1 )'
model.add_reactions([reaction])
reaction = Reaction('PGISr')
reaction.name = 'Prostaglandin I2 synthase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1931: -1.0, Cpd1934: 1.0})
reaction.gene_reaction_rule = '( PTGIS )'
model.add_reactions([reaction])
reaction = Reaction('PGK')
reaction.name = 'phosphoglycerate kinase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd273: -1.0, Cpd107: 1.0, Cpd272: 1.0})
reaction.gene_reaction_rule = '( MIA3 or PGK1 or PGK2 )'
model.add_reactions([reaction])
reaction = Reaction('PGL')
reaction.name = '6-phosphogluconolactonase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1203: -1.0, Cpd33: 1.0, Cpd1421: 1.0})
reaction.gene_reaction_rule = '( PGLS )'
model.add_reactions([reaction])
reaction = Reaction('PGLer')
reaction.name = '6-phosphogluconolactonase, endoplasmic reticulum'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1201: -1.0, Cpd565: 1.0, Cpd1423: 1.0})
reaction.gene_reaction_rule = '( H6PD )'
model.add_reactions([reaction])
reaction = Reaction('PGLYCP')
reaction.name = 'Phosphoglycolate phosphatase'
reaction.subsystem = 'Glyoxylate and dicarboxylate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1935: -1.0, Cpd108: 1.0, Cpd1294: 1.0})
reaction.gene_reaction_rule = '( PGP )'
model.add_reactions([reaction])
reaction = Reaction('PGM')
reaction.name = 'phosphoglycerate mutase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd981: -1.0, Cpd273: 1.0})
reaction.gene_reaction_rule = '( BPGM or PGAM1 or PGAM2 )'
model.add_reactions([reaction])
reaction = Reaction('PGMT')
reaction.name = 'phosphoglucomutase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd293: -1.0, Cpd1202: 1.0})
reaction.gene_reaction_rule = '( PGM1 or PGM2 )'
model.add_reactions([reaction])
reaction = Reaction('PGPPT')
reaction.name = 'phosphatidyl-CMP: glycerophosphate phosphatidyltransferase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd649: -1.0, Cpd1198: -1.0, Cpd33: 1.0, Cpd651: 1.0, Cpd1936: 1.0})
reaction.gene_reaction_rule = '( PGS1 )'
model.add_reactions([reaction])
reaction = Reaction('PGS')
reaction.name = 'Prostaglandin G/H synthase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -2.0, Cpd33: -1.0, Cpd22: -1.0, Cpd427: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd1929: 1.0})
reaction.gene_reaction_rule = '( PTGS1 or PTGS2 )'
model.add_reactions([reaction])
reaction = Reaction('PGSr')
reaction.name = 'Prostaglandin G/H synthase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd1885: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1931: 1.0})
reaction.gene_reaction_rule = '( PTGS1 or PTGS2 )'
model.add_reactions([reaction])
reaction = Reaction('PHCDm')
reaction.name = 'L-1-pyrroline-3-hydroxy-5-carboxylate dehydrogenase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -2.0, Cpd44: -1.0, Cpd1937: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd100: 1.0})
reaction.gene_reaction_rule = '( ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('PHEMEt')
reaction.name = 'heme transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1938: -1.0, Cpd1495: 1.0})
reaction.gene_reaction_rule = '( LRP1 or SLC46A1 )'
model.add_reactions([reaction])
reaction = Reaction('PHEt4')
reaction.name = 'L-phenylalanine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC38A4 or SLC6A14 or SLC6A19 or SLC7A1 or SLC7A2 or SLC7A3 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('PHETA1')
reaction.name = 'phenylalanine transaminase'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd1940: -1.0, Cpd74: 1.0, Cpd1941: 1.0})
reaction.gene_reaction_rule = '( GOT1 )'
model.add_reactions([reaction])
reaction = Reaction('PHETA1m')
reaction.name = 'phenylalanine transaminase (m)'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd1942: -1.0, Cpd63: 1.0, Cpd1943: 1.0})
reaction.gene_reaction_rule = '( GOT2 or TAT )'
model.add_reactions([reaction])
reaction = Reaction('PHEtec')
reaction.name = 'L-phenylalanine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1939: -1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC16A10 or SLC43A1 or SLC43A2 )'
model.add_reactions([reaction])
reaction = Reaction('PHETHPTOX2')
reaction.name = 'L-Phenylalanine,tetrahydrobiopterin:oxygen oxidoreductase (4-hydroxylating)'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd870: -1.0, Cpd1940: -1.0, Cpd1544: 1.0, Cpd1944: 1.0})
reaction.gene_reaction_rule = '( PAH )'
model.add_reactions([reaction])
reaction = Reaction('PHYCBOXL')
reaction.name = 'L-Phenylalanine carboxy-lyase'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1940: -1.0, Cpd60: 1.0, Cpd1921: 1.0})
reaction.gene_reaction_rule = '( DDC )'
model.add_reactions([reaction])
reaction = Reaction('PHYHx')
reaction.name = 'Phytanoyl-CoA dioxygenase, peroxisomal'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd1538: -1.0, Cpd1945: -1.0, Cpd943: 1.0, Cpd1946: 1.0, Cpd1947: 1.0})
reaction.gene_reaction_rule = '( PHYH )'
model.add_reactions([reaction])
reaction = Reaction('PI345P3P')
reaction.name = 'phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1948: -1.0, Cpd108: 1.0, Cpd1949: 1.0})
reaction.gene_reaction_rule = '( PTEN )'
model.add_reactions([reaction])
reaction = Reaction('PI345P3Pn')
reaction.name = 'phosphatidylinositol-3,4,5-trisphosphate 3-phosphatase, nuclear'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd1950: -1.0, Cpd1951: 1.0, Cpd1952: 1.0})
reaction.gene_reaction_rule = '( PTEN )'
model.add_reactions([reaction])
reaction = Reaction('PI345P5P')
reaction.name = 'phosphatidylinositol-3,4,5-trisphosphate 5-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1948: -1.0, Cpd108: 1.0, Cpd1953: 1.0})
reaction.gene_reaction_rule = '( INPP5A or INPP5B or INPP5D or INPP5E or INPP5J or INPPL1 or SYNJ1 or SYNJ2 )'
model.add_reactions([reaction])
reaction = Reaction('PI34P5K')
reaction.name = 'phosphatidylinositol 3,4-bisphosphate 5-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1953: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1948: 1.0})
reaction.gene_reaction_rule = '( PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('PI3P4K')
reaction.name = 'phosphatidylinositol 3-phosphate 4-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1954: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1953: 1.0})
reaction.gene_reaction_rule = '( PI4K2A or PI4KA or PI4KB or PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('PI3P5K')
reaction.name = 'phosphatidylinositol 3-phosphate 5-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1954: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1955: 1.0})
reaction.gene_reaction_rule = '( PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('PI45P3K')
reaction.name = 'phosphatidylinositol 4,5-bisphosphate 3-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1949: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1948: 1.0})
reaction.gene_reaction_rule = '( HCST or PIK3CA or PIK3CB or PIK3CD or PIK3CG or PIK3R1 or PIK3R2 or PIK3R3 or PIK3R5 )'
model.add_reactions([reaction])
reaction = Reaction('PI45P5P')
reaction.name = 'phosphatidylinositol-4,5-bisphosphate 5-phosphatase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1949: -1.0, Cpd108: 1.0, Cpd1956: 1.0})
reaction.gene_reaction_rule = '( INPP5B or INPP5E or OCRL or SYNJ1 or SYNJ2 )'
model.add_reactions([reaction])
reaction = Reaction('PI45PLC')
reaction.name = 'phosphatidylinositol 4,5-bisphosphate phospholipase C'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1949: -1.0, Cpd33: 1.0, Cpd658: 1.0, Cpd1690: 1.0})
reaction.gene_reaction_rule = '( PLCB1 or PLCB2 or PLCB3 or PLCB4 or PLCD1 or PLCD3 or PLCD4 or PLCE1 or PLCG1 or PLCG2 or PLCH1 or PLCH2 or PLCL1 or PLCXD2 or PLCZ1 )'
model.add_reactions([reaction])
reaction = Reaction('PI45PLCn')
reaction.name = 'phosphatidylinositol 4,5-bisphosphate phospholipase C, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd1951: -1.0, Cpd763: 1.0, Cpd797: 1.0, Cpd1689: 1.0})
reaction.gene_reaction_rule = '( PLCB1 )'
model.add_reactions([reaction])
reaction = Reaction('PI4P3K')
reaction.name = 'phosphatidylinositol 4-phosphate 3-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1956: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1953: 1.0})
reaction.gene_reaction_rule = '( HCST or PIK3C2A or PIK3C2B or PIK3C2G or PIK3CA or PIK3CB or PIK3CD or PIK3CG or PIK3R1 or PIK3R2 or PIK3R3 or PIK3R5 )'
model.add_reactions([reaction])
reaction = Reaction('PI4P3Ker')
reaction.name = 'phosphatidylinositol 4-phosphate 3-kinase, endoplasmic reticulum'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1958: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd1960: 1.0})
reaction.gene_reaction_rule = '( PIK3C2B )'
model.add_reactions([reaction])
reaction = Reaction('PI4P5K')
reaction.name = 'phosphatidylinositol 4-phosphate 5-kinase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1956: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1949: 1.0})
reaction.gene_reaction_rule = '( PIKFYVE or PIP4K2A or PIP4K2B or PIP4K2C or PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('PI4PLC')
reaction.name = 'phosphatidylinositol 4-phosphate phospholipase C'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1956: -1.0, Cpd33: 1.0, Cpd658: 1.0, Cpd1691: 1.0})
reaction.gene_reaction_rule = '( PLCB1 or PLCB2 or PLCB3 or PLCB4 or PLCD1 or PLCD3 or PLCD4 or PLCE1 or PLCG1 or PLCG2 or PLCH1 or PLCH2 or PLCL1 or PLCXD2 or PLCZ1 )'
model.add_reactions([reaction])
reaction = Reaction('PI4PLCn')
reaction.name = 'phosphatidylinositol 4-phosphate phospholipase C, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd1961: -1.0, Cpd763: 1.0, Cpd797: 1.0, Cpd1962: 1.0})
reaction.gene_reaction_rule = '( PLCB1 )'
model.add_reactions([reaction])
reaction = Reaction('PI5P3K')
reaction.name = 'phosphatidylinositol-5-phosphate 3-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1963: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1955: 1.0})
reaction.gene_reaction_rule = '( HCST or PIK3C2A or PIK3C2B or PIK3C2G or PIK3C3 or PIK3CA or PIK3CB or PIK3CD or PIK3CG or PIK3R1 or PIK3R2 or PIK3R3 or PIK3R5 )'
model.add_reactions([reaction])
reaction = Reaction('PI5P3Ker')
reaction.name = 'phosphatidylinositol-5-phosphate 3-kinase, endoplasmic reticulum'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1964: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd1965: 1.0})
reaction.gene_reaction_rule = '( PIK3C2B )'
model.add_reactions([reaction])
reaction = Reaction('PI5P4K')
reaction.name = 'phosphatidylinositol-5-phosphate 4-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1963: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1949: 1.0})
reaction.gene_reaction_rule = '( PI4K2A or PI4KA or PI4KB )'
model.add_reactions([reaction])
reaction = Reaction('PIACGT')
reaction.name = 'phosphatidylinositol N-acetylglucosaminyltransferase'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd652: -1.0, Cpd1396: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd220: 1.0})
reaction.gene_reaction_rule = '( DPM2 or PIGA or PIGC or PIGH or PIGP or PIGQ )'
model.add_reactions([reaction])
reaction = Reaction('PIK3')
reaction.name = 'phosphatidylinositol 3-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd652: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1954: 1.0})
reaction.gene_reaction_rule = '( HCST or PIK3C2A or PIK3C2B or PIK3C2G or PIK3C3 or PIK3CA or PIK3CB or PIK3CD or PIK3CG or PIK3R1 or PIK3R2 or PIK3R3 or PIK3R5 )'
model.add_reactions([reaction])
reaction = Reaction('PIK3er')
reaction.name = 'phosphatidylinositol 3-kinase, endoplasmic reticulum'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1966: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd1967: 1.0})
reaction.gene_reaction_rule = '( PIK3C2B )'
model.add_reactions([reaction])
reaction = Reaction('PIK4')
reaction.name = 'phosphatidylinositol 4-kinase'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd652: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1956: 1.0})
reaction.gene_reaction_rule = '( PI4K2A or PI4KA or PI4KB )'
model.add_reactions([reaction])
reaction = Reaction('PIK5')
reaction.name = 'phosphatidylinositol 5-kinase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd652: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1963: 1.0})
reaction.gene_reaction_rule = '( PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('PIPLC')
reaction.name = 'phosphatidylinositol phospholipase C'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd652: -1.0, Cpd33: 1.0, Cpd658: 1.0, Cpd1693: 1.0})
reaction.gene_reaction_rule = '( PLCB1 or PLCB2 or PLCB3 or PLCB4 or PLCD1 or PLCD3 or PLCD4 or PLCE1 or PLCG1 or PLCG2 or PLCH1 or PLCH2 or PLCL1 or PLCXD2 or PLCZ1 )'
model.add_reactions([reaction])
reaction = Reaction('PIPLCn')
reaction.name = 'phosphatidylinositol phospholipase C, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd1968: -1.0, Cpd763: 1.0, Cpd797: 1.0, Cpd1969: 1.0})
reaction.gene_reaction_rule = '( PLCB1 )'
model.add_reactions([reaction])
reaction = Reaction('PIt2m')
reaction.name = 'phosphate transporter, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd108: -1.0, Cpd7: 1.0, Cpd204: 1.0})
reaction.gene_reaction_rule = '( SLC25A3 )'
model.add_reactions([reaction])
reaction = Reaction('PIt7')
reaction.name = 'phosphate transport in/out via three Na+ symporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -3.0, Cpd494: -1.0, Cpd108: 1.0, Cpd41: 3.0})
reaction.gene_reaction_rule = '( SLC17A1 or SLC17A2 or SLC17A3 or SLC17A4 or SLC34A1 or SLC34A2 )'
model.add_reactions([reaction])
reaction = Reaction('PIter')
reaction.name = 'phosphate transport, endoplasmic reticulum'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd915: -1.0, Cpd108: 1.0})
reaction.gene_reaction_rule = '( SLC37A4 )'
model.add_reactions([reaction])
reaction = Reaction('PLA2_2')
reaction.name = 'phospholipase A2'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd659: -1.0, Cpd33: 1.0, Cpd1626: 1.0, Cpd1616: 1.0})
reaction.gene_reaction_rule = '( PLA2G12A or PLA2G2A or PLA2G2F or PLA2G3 or PLA2G4A or PLA2G4B or PLA2G4E or PLA2G5 or PLA2G6 )'
model.add_reactions([reaction])
reaction = Reaction('PLA2_2e')
reaction.name = 'phospholipase A2'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1580: -1.0, Cpd37: 1.0, Cpd1581: 1.0, Cpd1628: 1.0})
reaction.gene_reaction_rule = '( PLA2G10 or PLA2G12A or PLA2G1B or PLA2G2A or PLA2G2D or PLA2G2E or PLA2G2F or PLA2G5 )'
model.add_reactions([reaction])
reaction = Reaction('PLYSPSer')
reaction.name = 'protein lysine peptidase (endoplasmic reticulum)'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1970: -1.0, Cpd1971: 1.0, Cpd1972: 1.0})
reaction.gene_reaction_rule = '( SEC11A or SEC11C or SPCS1 or SPCS2 or SPCS3 )'
model.add_reactions([reaction])
reaction = Reaction('PMANM')
reaction.name = 'phosphomannomutase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd295: -1.0, Cpd233: 1.0})
reaction.gene_reaction_rule = '( PGM1 or PMM1 or PMM2 )'
model.add_reactions([reaction])
reaction = Reaction('PMEVKx')
reaction.name = 'phosphomevalonate kinase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd940: -1.0, Cpd1679: -1.0, Cpd942: 1.0, Cpd941: 1.0})
reaction.gene_reaction_rule = '( PMVK )'
model.add_reactions([reaction])
reaction = Reaction('PMI12346PH')
reaction.name = '5-diphosphoinositol-1,2,3,4,6-pentakisphosphate diphosphohydrolase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1973: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1974: 1.0})
reaction.gene_reaction_rule = '( IP6K1 or IP6K3 )'
model.add_reactions([reaction])
reaction = Reaction('PMI12346PHn')
reaction.name = '5-diphosphoinositol-1,2,3,4,6-pentakisphosphate diphosphohydrolase, nuclear'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd1975: -1.0, Cpd763: 1.0, Cpd1976: 1.0, Cpd1952: 1.0})
reaction.gene_reaction_rule = '( IP6K1 or IP6K2 or IP6K3 )'
model.add_reactions([reaction])
reaction = Reaction('PMI1346PH')
reaction.name = 'diphosphoinositol-1,3,4,6-tetrakisphosphate diphosphohydrolase'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1977: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1978: 1.0})
reaction.gene_reaction_rule = '( IP6K1 or IP6K3 )'
model.add_reactions([reaction])
reaction = Reaction('PMI1346PHn')
reaction.name = 'diphosphoinositol-1,3,4,6-tetrakisphosphate diphosphohydrolase, nucleus'
reaction.subsystem = 'Inositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd1979: -1.0, Cpd763: 1.0, Cpd1683: 1.0, Cpd1952: 1.0})
reaction.gene_reaction_rule = '( IP6K1 or IP6K2 or IP6K3 )'
model.add_reactions([reaction])
reaction = Reaction('PNP')
reaction.name = 'purine-nucleoside phosphorylase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd1980: -1.0, Cpd33: 1.0, Cpd1797: 1.0, Cpd1809: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PNTEH')
reaction.name = 'Hydrolase Class ( RXN R02973)'
reaction.subsystem = 'CoA catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1981: -1.0, Cpd1982: 1.0, Cpd1983: 1.0})
reaction.gene_reaction_rule = '( VNN1 or VNN3 )'
model.add_reactions([reaction])
reaction = Reaction('PNTK')
reaction.name = 'pantothenate kinase'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1983: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1984: 1.0})
reaction.gene_reaction_rule = '( PANK1 or PANK2 or PANK3 or PANK4 )'
model.add_reactions([reaction])
reaction = Reaction('PNTKm')
reaction.name = 'pantothenate kinase (mitochondrial)'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1985: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd1986: 1.0})
reaction.gene_reaction_rule = '( PANK2 )'
model.add_reactions([reaction])
reaction = Reaction('PNTOt5')
reaction.name = 'Pantothenate sodium symporter  II'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -1.0, Cpd1987: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 1.0, Cpd1983: 1.0})
reaction.gene_reaction_rule = '( SLC5A6 )'
model.add_reactions([reaction])
reaction = Reaction('PPA')
reaction.name = 'inorganic diphosphatase'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd151: -1.0, Cpd33: 1.0, Cpd108: 2.0})
reaction.gene_reaction_rule = '( LHPP or PPA1 )'
model.add_reactions([reaction])
reaction = Reaction('PPAer')
reaction.name = 'inorganic diphosphatase, endoplasmic reticulum'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1988: -1.0, Cpd565: 1.0, Cpd915: 2.0})
reaction.gene_reaction_rule = '( G6PC or G6PC2 or G6PC3 )'
model.add_reactions([reaction])
reaction = Reaction('PPAm')
reaction.name = 'inorganic diphosphatase'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd210: -1.0, Cpd7: 1.0, Cpd204: 2.0})
reaction.gene_reaction_rule = '( PPA2 )'
model.add_reactions([reaction])
reaction = Reaction('PPAP')
reaction.name = 'phosphatidic acid phosphatase'
reaction.subsystem = 'Triacylglycerol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd353: -1.0, Cpd108: 1.0, Cpd658: 1.0})
reaction.gene_reaction_rule = '( PLPP1 or PLPP2 or PLPP3 )'
model.add_reactions([reaction])
reaction = Reaction('PPBNGS')
reaction.name = 'porphobilinogen synthase'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1989: -2.0, Cpd2: 2.0, Cpd33: 1.0, Cpd1485: 1.0})
reaction.gene_reaction_rule = '( ALAD )'
model.add_reactions([reaction])
reaction = Reaction('PPCDC')
reaction.name = 'phosphopantothenoylcysteine decarboxylase'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1990: -1.0, Cpd60: 1.0, Cpd1991: 1.0})
reaction.gene_reaction_rule = '( PPCDC )'
model.add_reactions([reaction])
reaction = Reaction('PPCOACm')
reaction.name = 'Propionyl-CoA carboxylase, mitochondrial'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd179: -1.0, Cpd200: -1.0, Cpd201: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd1707: 1.0})
reaction.gene_reaction_rule = '( PCCA or PCCB )'
model.add_reactions([reaction])
reaction = Reaction('PPCOAOm')
reaction.name = 'Propanoyl-CoA:FAD 2,3-oxidoreductase, mitochondrial'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd179: -1.0, Cpd238: -1.0, Cpd240: 1.0, Cpd1992: 1.0})
reaction.gene_reaction_rule = '( ACAD11 or ACAD12 or ACAD8 or ACAD9 or ACADM or ACADS or ACADSB )'
model.add_reactions([reaction])
reaction = Reaction('PPDOx')
reaction.name = 'Propane-1,2-diol:NAD+ 1-oxidoreductase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd413: -1.0, Cpd32: 1.0, Cpd412: 1.0})
reaction.gene_reaction_rule = '( AKR7A5 )'
model.add_reactions([reaction])
reaction = Reaction('PPDOy')
reaction.name = 'Propane-1,2-diol:NADP+ 1-oxidoreductase'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd413: -1.0, Cpd25: 1.0, Cpd412: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 or AKR7A5 )'
model.add_reactions([reaction])
reaction = Reaction('PPM')
reaction.name = 'phosphopentomutase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1809: -1.0, Cpd297: 1.0})
reaction.gene_reaction_rule = '( PGM1 or PGM2 )'
model.add_reactions([reaction])
reaction = Reaction('PPNCL3')
reaction.name = 'phosphopantothenate-cysteine ligase'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd396: -1.0, Cpd1984: -1.0, Cpd33: 1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1990: 1.0})
reaction.gene_reaction_rule = '( PPCS )'
model.add_reactions([reaction])
reaction = Reaction('PPOR')
reaction.name = 'Phenylpyruvate:oxygen oxidoreductase (hydroxylating,decarboxylating)'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd1941: -1.0, Cpd60: 1.0, Cpd1993: 1.0})
reaction.gene_reaction_rule = '( HPD )'
model.add_reactions([reaction])
reaction = Reaction('PPPGOm')
reaction.name = 'protoporphyrinogen oxidase, mitochondrial'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -3.0, Cpd1994: -2.0, Cpd12: 6.0, Cpd1111: 2.0})
reaction.gene_reaction_rule = '( PPOX )'
model.add_reactions([reaction])
reaction = Reaction('PRAGSr')
reaction.name = 'phosphoribosylglycinamide synthase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd402: -1.0, Cpd1407: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1283: 1.0})
reaction.gene_reaction_rule = '( GART )'
model.add_reactions([reaction])
reaction = Reaction('PRASCS')
reaction.name = 'phosphoribosylaminoimidazolesuccinocarboxamide synthase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd90: -1.0, Cpd379: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd314: 1.0})
reaction.gene_reaction_rule = '( PAICS )'
model.add_reactions([reaction])
reaction = Reaction('PRDX')
reaction.name = 'Peroxidase (multiple substrates)'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd410: -1.0, Cpd2: 2.0, Cpd411: 1.0})
reaction.gene_reaction_rule = '( EPX or PRDX6 )'
model.add_reactions([reaction])
reaction = Reaction('PRDXl')
reaction.name = 'Peroxidase (multiple substrates)'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1995: -1.0, Cpd1996: -1.0, Cpd86: 2.0, Cpd1997: 1.0})
reaction.gene_reaction_rule = '( MPO )'
model.add_reactions([reaction])
reaction = Reaction('PRFGS')
reaction.name = 'phosphoribosylformylglycinamidine synthase'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd399: -1.0, Cpd1284: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd74: 1.0, Cpd1998: 1.0})
reaction.gene_reaction_rule = '( PFAS )'
model.add_reactions([reaction])
reaction = Reaction('PRGNLONESULT')
reaction.name = 'steroid sulfotransferase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd1524: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd1999: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 or SULT2A1 or SULT2B1 )'
model.add_reactions([reaction])
reaction = Reaction('PRO1xm')
reaction.name = 'proline oxidase (NAD), mitochondrial'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1898: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd1897: 1.0})
reaction.gene_reaction_rule = '( PRODH )'
model.add_reactions([reaction])
reaction = Reaction('PROAKGOX1r')
reaction.name = 'L-Proline,2-oxoglutarate:oxygen oxidoreductase (4-hydroxylating) (ER)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd2000: -1.0, Cpd2001: -1.0, Cpd624: 1.0, Cpd2002: 1.0, Cpd2003: 1.0})
reaction.gene_reaction_rule = '( P4HA1 or P4HA2 or P4HB )'
model.add_reactions([reaction])
reaction = Reaction('PROD2')
reaction.name = 'Proline dehydrogenase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd856: -1.0, Cpd2004: -1.0, Cpd33: 1.0, Cpd857: 1.0, Cpd2005: 1.0})
reaction.gene_reaction_rule = '( PRODH2 )'
model.add_reactions([reaction])
reaction = Reaction('PROD2m')
reaction.name = 'Proline dehydrogenase (m)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd1898: -1.0, Cpd7: 1.0, Cpd240: 1.0, Cpd1897: 1.0})
reaction.gene_reaction_rule = '( PRODH )'
model.add_reactions([reaction])
reaction = Reaction('PRODt2r')
reaction.name = 'D-proline reversible transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd2006: -1.0, Cpd33: 1.0, Cpd2007: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('PRODt2rL')
reaction.name = 'D-proline reversible transport via proton symport (lysosome)'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd2008: -1.0, Cpd33: 1.0, Cpd2007: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('PROSTGD2t')
reaction.name = 'prostaglandin transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2009: -1.0, Cpd38: 1.0, Cpd1930: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('PROSTGE1t')
reaction.name = 'prostaglandin transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2010: -1.0, Cpd38: 1.0, Cpd646: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('PROSTGE1t3')
reaction.name = 'prostaglandine E1 transport (ATP-dependent)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd646: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2010: 1.0})
reaction.gene_reaction_rule = '( ABCC4 )'
model.add_reactions([reaction])
reaction = Reaction('PROSTGE2t')
reaction.name = 'prostaglandin transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2011: -1.0, Cpd38: 1.0, Cpd647: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO2A1 or SLCO3A1 or SLCO4A1 )'
model.add_reactions([reaction])
reaction = Reaction('PROSTGE2t2')
reaction.name = 'prostaglandin uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2011: -1.0, Cpd647: 1.0})
reaction.gene_reaction_rule = '( SLC22A1 or SLC22A2 )'
model.add_reactions([reaction])
reaction = Reaction('PROSTGE2t3')
reaction.name = 'prostaglandine E2 transport (ATP-dependent)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd647: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2011: 1.0})
reaction.gene_reaction_rule = '( ABCC4 )'
model.add_reactions([reaction])
reaction = Reaction('PROSTGF2t')
reaction.name = 'prostaglandin uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2012: -1.0, Cpd648: 1.0})
reaction.gene_reaction_rule = '( SLC22A1 or SLC22A2 )'
model.add_reactions([reaction])
reaction = Reaction('PROt2r')
reaction.name = 'L-proline reversible transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd2013: -1.0, Cpd33: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 or SLC36A2 )'
model.add_reactions([reaction])
reaction = Reaction('PROt2rL')
reaction.name = 'L-proline reversible transport via proton symport (lysosome)'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd2014: -1.0, Cpd33: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('PROt4')
reaction.name = 'Na+/Proline-L symporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC38A4 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('PRPNCOAHYDm')
reaction.name = 'Propenoyl-CoA hydrolase (m)'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1992: -1.0, Cpd2015: 1.0})
reaction.gene_reaction_rule = '( AUH or ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('PRPNCOAHYDx')
reaction.name = 'Propenoyl-CoA hydrolase (x)'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2016: -1.0, Cpd2017: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('PRPPS')
reaction.name = 'phosphoribosylpyrophosphate synthetase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd297: -1.0, Cpd33: 1.0, Cpd150: 1.0, Cpd301: 1.0})
reaction.gene_reaction_rule = '( PRPS1 or PRPS1L1 or PRPS2 )'
model.add_reactions([reaction])
reaction = Reaction('PSERT')
reaction.name = 'phosphoserine transaminase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd74: -1.0, Cpd1928: -1.0, Cpd33: 1.0, Cpd30: 1.0, Cpd2018: 1.0})
reaction.gene_reaction_rule = '( PSAT1 )'
model.add_reactions([reaction])
reaction = Reaction('PSFLIP')
reaction.name = 'phosphatidylserine flippase'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2019: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2020: 1.0})
reaction.gene_reaction_rule = '( ATP10A or ATP8A1 )'
model.add_reactions([reaction])
reaction = Reaction('PSFLIPm')
reaction.name = 'phosphatidylserine flippase'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2020: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2021: 1.0})
reaction.gene_reaction_rule = '( ATP10A or ATP8A1 )'
model.add_reactions([reaction])
reaction = Reaction('PSP_L')
reaction.name = 'phosphoserine phosphatase (L-serine)'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -1.0, Cpd2018: -1.0, Cpd108: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( PSPH )'
model.add_reactions([reaction])
reaction = Reaction('PSSA1_hs')
reaction.name = 'Phosphatidylserine synthase homo sapiens'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd659: -1.0, Cpd672: 1.0, Cpd2020: 1.0})
reaction.gene_reaction_rule = '( PTDSS1 )'
model.add_reactions([reaction])
reaction = Reaction('PSSA2_hs')
reaction.name = 'Phosphatidylserine synthase homo sapiens'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd661: -1.0, Cpd995: 1.0, Cpd2020: 1.0})
reaction.gene_reaction_rule = '( PTDSS2 )'
model.add_reactions([reaction])
reaction = Reaction('PTDCACRNCPT1')
reaction.name = 'carnitine fatty-acyl transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1021: -1.0, Cpd78: 1.0, Cpd2022: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('PTDCACRNCPT2')
reaction.name = 'pentadecanoate transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2023: -1.0, Cpd307: 1.0, Cpd1062: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('PTDCACRNt')
reaction.name = 'pentadecanoate transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2022: -1.0, Cpd2023: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('PTE2x')
reaction.name = 'peroxisomal acyl-CoA thioesterase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd196: -1.0, Cpd261: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2024: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT4 or ACOT8 )'
model.add_reactions([reaction])
reaction = Reaction('PTE3x')
reaction.name = 'peroxisomal acyl-CoA thioesterase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd1925: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2025: 1.0})
reaction.gene_reaction_rule = '( ACOT2 )'
model.add_reactions([reaction])
reaction = Reaction('PTE4x')
reaction.name = 'peroxisomal acyl-CoA thioesterase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd1945: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2026: 1.0})
reaction.gene_reaction_rule = '( ACOT2 )'
model.add_reactions([reaction])
reaction = Reaction('PTE5x')
reaction.name = 'peroxisomal acyl-CoA thioesterase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd1079: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2027: 1.0})
reaction.gene_reaction_rule = '( ACOT2 )'
model.add_reactions([reaction])
reaction = Reaction('PTHPS')
reaction.name = '6-pyruvoyltetrahydropterin synthase'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1446: -1.0, Cpd2028: 1.0, Cpd2029: 1.0})
reaction.gene_reaction_rule = '( PTS )'
model.add_reactions([reaction])
reaction = Reaction('PTHPSn')
reaction.name = '6-pyruvoyltetrahydropterin synthase, nuclear'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1448: -1.0, Cpd2030: 1.0, Cpd2031: 1.0})
reaction.gene_reaction_rule = '( PTS )'
model.add_reactions([reaction])
reaction = Reaction('PTPAT')
reaction.name = 'Phosphopantetheine adenylyltransferase (EC 2.7.7.3)'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd1991: -1.0, Cpd151: 1.0, Cpd938: 1.0})
reaction.gene_reaction_rule = '( COASY )'
model.add_reactions([reaction])
reaction = Reaction('PTRCAT1')
reaction.name = 'Putrescine acetyltransferase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd180: -1.0, Cpd1846: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd455: 1.0})
reaction.gene_reaction_rule = '( SAT1 or SAT2 )'
model.add_reactions([reaction])
reaction = Reaction('PTRCOX1')
reaction.name = 'Putrescine:oxygen oxidoreductase (deaminating)'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1846: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd172: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('PUNP1')
reaction.name = 'purine-nucleoside phosphorylase (Adenosine)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd274: -1.0, Cpd280: 1.0, Cpd1809: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PUNP2')
reaction.name = 'purine-nucleoside phosphorylase (Deoxyadenosine)'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd793: -1.0, Cpd955: 1.0, Cpd280: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PUNP3')
reaction.name = 'purine-nucleoside phosphorylase (Guanosine)'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd1438: -1.0, Cpd1449: 1.0, Cpd1809: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PUNP4')
reaction.name = 'purine-nucleoside phosphorylase (Deoxyguanosine)'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd841: -1.0, Cpd955: 1.0, Cpd1449: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PUNP5')
reaction.name = 'purine-nucleoside phosphorylase (Inosine)'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd275: -1.0, Cpd1535: 1.0, Cpd1809: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PUNP6')
reaction.name = 'purine-nucleoside phosphorylase (Deoxyinosine)'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd794: -1.0, Cpd955: 1.0, Cpd1535: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PUNP7')
reaction.name = 'purine-nucleoside phosphorylase (Xanthosine)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd1819: -1.0, Cpd1450: 1.0, Cpd1809: 1.0})
reaction.gene_reaction_rule = '( PNP2 )'
model.add_reactions([reaction])
reaction = Reaction('PYAM5POr')
reaction.name = 'pyridoxamine 5-phosphate oxidase'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2032: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd1920: 1.0})
reaction.gene_reaction_rule = '( PNPO )'
model.add_reactions([reaction])
reaction = Reaction('PYDAMK')
reaction.name = 'pyridoxamine kinase'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2033: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd2032: 1.0})
reaction.gene_reaction_rule = '( PDXK )'
model.add_reactions([reaction])
reaction = Reaction('PYDXDH')
reaction.name = 'pyridoxal dehydrogenase'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2034: -1.0, Cpd5: 1.0, Cpd2035: 1.0})
reaction.gene_reaction_rule = '( AOX1 )'
model.add_reactions([reaction])
reaction = Reaction('PYDXK')
reaction.name = 'pyridoxal kinase'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2034: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1920: 1.0})
reaction.gene_reaction_rule = '( PDXK )'
model.add_reactions([reaction])
reaction = Reaction('PYDXNK')
reaction.name = 'pyridoxine kinase'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2036: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1919: 1.0})
reaction.gene_reaction_rule = '( PDXK )'
model.add_reactions([reaction])
reaction = Reaction('PYK')
reaction.name = 'pyruvate kinase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd107: -1.0, Cpd33: -1.0, Cpd231: -1.0, Cpd105: 1.0, Cpd21: 1.0})
reaction.gene_reaction_rule = '( PKLR or PKM )'
model.add_reactions([reaction])
reaction = Reaction('PYLALDOX')
reaction.name = 'Perillyl aldehyde:NAD+ oxidoreductase'
reaction.subsystem = 'Limonene and pinene degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd2037: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd2038: 1.0})
reaction.gene_reaction_rule = '( ALDH1A1 or ALDH1A2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('PYLALDOXm')
reaction.name = 'Perillyl aldehyde:NAD+ oxidoreductase (m)'
reaction.subsystem = 'Limonene and pinene degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2039: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd2040: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('PYNP2r')
reaction.name = 'pyrimidine-nucleoside phosphorylase (uracil)'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd761: -1.0, Cpd951: 1.0, Cpd1809: 1.0})
reaction.gene_reaction_rule = '( UPP1 or UPP2 )'
model.add_reactions([reaction])
reaction = Reaction('PYRt2m')
reaction.name = 'pyruvate mitochondrial transport via proton symport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd21: -1.0, Cpd7: 1.0, Cpd363: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('PYRt2p')
reaction.name = 'pyruvate peroxisomal transport via proton symport'
reaction.subsystem = 'D-alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd21: -1.0, Cpd358: 1.0, Cpd368: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A7 )'
model.add_reactions([reaction])
reaction = Reaction('PYRt2r')
reaction.name = 'pyruvate reversible transport via proton symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd2041: -1.0, Cpd33: 1.0, Cpd21: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A3 or SLC16A7 )'
model.add_reactions([reaction])
reaction = Reaction('RBFK')
reaction.name = 'riboflavin kinase'
reaction.subsystem = 'Vitamin B2 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd264: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd263: 1.0})
reaction.gene_reaction_rule = '( RFK )'
model.add_reactions([reaction])
reaction = Reaction('RBK')
reaction.name = 'ribokinase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2042: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd297: 1.0})
reaction.gene_reaction_rule = '( RBKS )'
model.add_reactions([reaction])
reaction = Reaction('RBK_D')
reaction.name = 'D-ribulokinase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2043: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1422: 1.0})
reaction.gene_reaction_rule = '( RBKS )'
model.add_reactions([reaction])
reaction = Reaction('RDH1')
reaction.name = 'retinol dehydrogenase (all-trans)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1633: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd545: 1.0})
reaction.gene_reaction_rule = '( RDH16F1 or RDH5 )'
model.add_reactions([reaction])
reaction = Reaction('RDH1a')
reaction.name = 'retinol dehydrogenase (all-trans,NADPH)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1633: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd545: 1.0})
reaction.gene_reaction_rule = '( RDH10 or RDH11 or RDH12 or RDH13 or RDH14 or RDH8 or SDR16C5 )'
model.add_reactions([reaction])
reaction = Reaction('RDH2')
reaction.name = 'retinol dehydrogenase (9-cis,NADH)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2044: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd2045: 1.0})
reaction.gene_reaction_rule = '( RDH5 )'
model.add_reactions([reaction])
reaction = Reaction('RDH2a')
reaction.name = 'retinol dehydrogenase (9-cis,NADPH)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2044: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2045: 1.0})
reaction.gene_reaction_rule = '( RDH11 or RDH12 or RDH13 or RDH14 or RDH8 or SDR16C5 )'
model.add_reactions([reaction])
reaction = Reaction('RDH3')
reaction.name = 'retinol dehydrogenase (11-cis,NADH)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2046: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd2047: 1.0})
reaction.gene_reaction_rule = '( RDH5 )'
model.add_reactions([reaction])
reaction = Reaction('RDH3a')
reaction.name = 'retinol dehydrogenase (11-cis,NADPH)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2046: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2047: 1.0})
reaction.gene_reaction_rule = '( RDH11 or RDH12 or RDH13 or RDH14 or SDR16C5 )'
model.add_reactions([reaction])
reaction = Reaction('RDH4')
reaction.name = 'retinol dehydrogenase (13-cis,NADH)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2048: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd2049: 1.0})
reaction.gene_reaction_rule = '( RDH5 )'
model.add_reactions([reaction])
reaction = Reaction('RETt')
reaction.name = 'Retinol transport via faciliated diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2050: -1.0, Cpd1633: 1.0})
reaction.gene_reaction_rule = '( RBP1 or RBP2 or RBP4 or STRA6 )'
model.add_reactions([reaction])
reaction = Reaction('RIBFLVt3')
reaction.name = 'riboflavin transport (ATP dependent)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2051: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd264: 1.0})
reaction.gene_reaction_rule = '( SLC52A3 )'
model.add_reactions([reaction])
reaction = Reaction('RNDR1')
reaction.name = 'ribonucleoside-diphosphate reductase (ADP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd107: -1.0, Cpd2052: -1.0, Cpd2: 1.0, Cpd288: 1.0, Cpd2053: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('RNDR2')
reaction.name = 'ribonucleoside-diphosphate reductase (GDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd283: -1.0, Cpd2052: -1.0, Cpd2: 1.0, Cpd771: 1.0, Cpd2053: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('RNDR3')
reaction.name = 'ribonucleoside-diphosphate reductase (CDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd769: -1.0, Cpd2052: -1.0, Cpd2: 1.0, Cpd777: 1.0, Cpd2053: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('RNDR4')
reaction.name = 'ribonucleoside-diphosphate reductase (UDP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd137: -1.0, Cpd2052: -1.0, Cpd2: 1.0, Cpd899: 1.0, Cpd2053: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('RNMK')
reaction.name = 'ribosylnicotinamide kinase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1980: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1796: 1.0})
reaction.gene_reaction_rule = '( NMRK1 )'
model.add_reactions([reaction])
reaction = Reaction('RPE')
reaction.name = 'ribulose 5-phosphate 3-epimerase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1422: -1.0, Cpd2054: 1.0})
reaction.gene_reaction_rule = '( RPE )'
model.add_reactions([reaction])
reaction = Reaction('RPI')
reaction.name = 'ribose-5-phosphate isomerase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd297: -1.0, Cpd1422: 1.0})
reaction.gene_reaction_rule = '( RPIA )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL2CRNCPT1')
reaction.name = 'carnitine fatty-acyl transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd351: -1.0, Cpd78: 1.0, Cpd2055: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL2CRNCPT2')
reaction.name = 'R group transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2056: -1.0, Cpd307: 1.0, Cpd2057: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL2CRNt')
reaction.name = 'R group transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2055: -1.0, Cpd2056: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL3CRNCPT1')
reaction.name = 'carnitine fatty-acyl transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd835: -1.0, Cpd78: 1.0, Cpd2058: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL3CRNCPT2')
reaction.name = 'R group transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2059: -1.0, Cpd307: 1.0, Cpd2060: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL3CRNt')
reaction.name = 'R group transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2058: -1.0, Cpd2059: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('RTOTALCRNCPT1')
reaction.name = 'carnitine fatty-acyl transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd843: -1.0, Cpd78: 1.0, Cpd2061: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('RTOTALCRNCPT2')
reaction.name = 'R group transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2062: -1.0, Cpd307: 1.0, Cpd1426: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('RTOTALCRNt')
reaction.name = 'R group transport into the mitochondria'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2061: -1.0, Cpd2062: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('S23T2g')
reaction.name = 'beta-galactoside alpha-2,3-sialyltransferase (core 2)'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd146: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1190: 1.0})
reaction.gene_reaction_rule = '( ST3GAL1 or ST3GAL2 or ST3GAL4 )'
model.add_reactions([reaction])
reaction = Reaction('S23T3g')
reaction.name = 'beta-galactoside alpha-2,3-sialyltransferase (complex N-glycan)'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd1197: -1.0, Cpd140: 1.0, Cpd336: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST3GAL3 )'
model.add_reactions([reaction])
reaction = Reaction('S23T4g')
reaction.name = 'beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd1190: -1.0, Cpd140: 1.0, Cpd329: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST3GAL1 or ST3GAL2 or ST3GAL4 )'
model.add_reactions([reaction])
reaction = Reaction('S23Tg')
reaction.name = 'beta-galactoside alpha-2,3-sialyltransferase (T antigen)'
reaction.subsystem = 'O-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd143: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1816: 1.0})
reaction.gene_reaction_rule = '( ST3GAL1 )'
model.add_reactions([reaction])
reaction = Reaction('S26Tg')
reaction.name = 'beta-galactoside alpha-2,6-sialyltransferase'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -2.0, Cpd1197: -1.0, Cpd140: 2.0, Cpd712: 2.0, Cpd2063: 1.0})
reaction.gene_reaction_rule = '( ST6GAL1 )'
model.add_reactions([reaction])
reaction = Reaction('S2T1g')
reaction.name = 'chondroitin 2-sulfotransferase, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd1374: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd1375: 1.0})
reaction.gene_reaction_rule = '( UST )'
model.add_reactions([reaction])
reaction = Reaction('S2T2g')
reaction.name = 'chondroitin 2-sulfotransferase, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd2064: -1.0, Cpd140: 1.0, Cpd2065: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( UST )'
model.add_reactions([reaction])
reaction = Reaction('S2T3g')
reaction.name = 'uronyl 2-sulfotransferase, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd2066: -1.0, Cpd140: 1.0, Cpd2067: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( UST )'
model.add_reactions([reaction])
reaction = Reaction('S2T4g')
reaction.name = 'uronyl 2-sulfotransferase, Golgi'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -2.0, Cpd1371: -1.0, Cpd140: 2.0, Cpd1207: 2.0, Cpd2068: 1.0})
reaction.gene_reaction_rule = '( HS2ST1 )'
model.add_reactions([reaction])
reaction = Reaction('S2TASE1ly')
reaction.name = 'iduronate-2-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1389: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1547: 1.0})
reaction.gene_reaction_rule = '( IDS )'
model.add_reactions([reaction])
reaction = Reaction('S2TASE2ly')
reaction.name = 'iduronate-2-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1391: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1549: 1.0})
reaction.gene_reaction_rule = '( IDS )'
model.add_reactions([reaction])
reaction = Reaction('S2TASE3ly')
reaction.name = 'iduronate-2-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1736: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1551: 1.0})
reaction.gene_reaction_rule = '( IDS )'
model.add_reactions([reaction])
reaction = Reaction('S3T1g')
reaction.name = 'heparin-glucosamine 3-O-sulfotransferase'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd2069: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2070: 1.0})
reaction.gene_reaction_rule = '( HS3ST1 or HS3ST4 or HS3ST5 )'
model.add_reactions([reaction])
reaction = Reaction('S3T2g')
reaction.name = 'heparin-glucosamine 3-O-sulfotransferase'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd2070: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2071: 1.0})
reaction.gene_reaction_rule = '( HS3ST2 or HS3ST4 or HS3ST5 )'
model.add_reactions([reaction])
reaction = Reaction('S3T3g')
reaction.name = 'heparin-glucosamine 3-O-sulfotransferase'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd2071: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2072: 1.0})
reaction.gene_reaction_rule = '( HS3ST3B1 or HS3ST4 or HS3ST6 )'
model.add_reactions([reaction])
reaction = Reaction('S4T1g')
reaction.name = 'chondroitin 4-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -2.0, Cpd1255: -1.0, Cpd140: 2.0, Cpd1207: 2.0, Cpd1372: 1.0})
reaction.gene_reaction_rule = '( CHST11 or CHST12 or CHST13 )'
model.add_reactions([reaction])
reaction = Reaction('S4T2g')
reaction.name = 'chondroitin 4-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd1257: -1.0, Cpd140: 1.0, Cpd2073: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST11 or CHST12 or CHST13 )'
model.add_reactions([reaction])
reaction = Reaction('S4T3g')
reaction.name = 'chondroitin 4-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd1263: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2074: 1.0})
reaction.gene_reaction_rule = '( CHST11 or CHST12 or CHST13 )'
model.add_reactions([reaction])
reaction = Reaction('S4T6g')
reaction.name = 'chondroitin 4-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd2075: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2066: 1.0})
reaction.gene_reaction_rule = '( CHST11 or CHST12 or CHST13 )'
model.add_reactions([reaction])
reaction = Reaction('S4TASE1ly')
reaction.name = 'N-acetylgalactosamine-4-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1747: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1727: 1.0})
reaction.gene_reaction_rule = '( ARSB )'
model.add_reactions([reaction])
reaction = Reaction('S4TASE2ly')
reaction.name = 'N-acetylgalactosamine-4-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1360: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1733: 1.0})
reaction.gene_reaction_rule = '( ARSB )'
model.add_reactions([reaction])
reaction = Reaction('S4TASE3ly')
reaction.name = 'N-acetylgalactosamine-4-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1748: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1735: 1.0})
reaction.gene_reaction_rule = '( ARSB )'
model.add_reactions([reaction])
reaction = Reaction('S4TASE4ly')
reaction.name = 'N-acetylgalactosamine-4-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1751: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd2076: 1.0})
reaction.gene_reaction_rule = '( ARSB )'
model.add_reactions([reaction])
reaction = Reaction('S4TASE5ly')
reaction.name = 'N-acetylgalactosamine-4-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1366: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd2077: 1.0})
reaction.gene_reaction_rule = '( ARSB )'
model.add_reactions([reaction])
reaction = Reaction('S6T10g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd320: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1184: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T11g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd322: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1185: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T12g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd324: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1186: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T13g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd326: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1187: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T14g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd328: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1188: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T15g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1189: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2078: 1.0})
reaction.gene_reaction_rule = '( CHST1 or CHST3 )'
model.add_reactions([reaction])
reaction = Reaction('S6T16g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd332: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1191: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T17g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd334: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1192: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T18g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1193: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2079: 1.0})
reaction.gene_reaction_rule = '( CHST1 or CHST3 )'
model.add_reactions([reaction])
reaction = Reaction('S6T19g')
reaction.name = 'chondroitin 6-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -2.0, Cpd1255: -1.0, Cpd140: 2.0, Cpd1207: 2.0, Cpd1374: 1.0})
reaction.gene_reaction_rule = '( CHST3 or CHST7 )'
model.add_reactions([reaction])
reaction = Reaction('S6T1g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd332: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1191: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T20g')
reaction.name = 'chondroitin 6-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd1259: -1.0, Cpd140: 1.0, Cpd2080: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST3 or CHST7 )'
model.add_reactions([reaction])
reaction = Reaction('S6T21g')
reaction.name = 'chondroitin 6-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd1261: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2064: 1.0})
reaction.gene_reaction_rule = '( CHST3 or CHST7 )'
model.add_reactions([reaction])
reaction = Reaction('S6T22g')
reaction.name = 'chondroitin 6-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -2.0, Cpd1372: -1.0, Cpd140: 2.0, Cpd1207: 2.0, Cpd1376: 1.0})
reaction.gene_reaction_rule = '( CHST7 )'
model.add_reactions([reaction])
reaction = Reaction('S6T23g')
reaction.name = 'chondroitin 6-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd2074: -1.0, Cpd140: 1.0, Cpd2081: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST7 )'
model.add_reactions([reaction])
reaction = Reaction('S6T24g')
reaction.name = 'chondroitin 6-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd1263: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2082: 1.0})
reaction.gene_reaction_rule = '( CHST3 or CHST7 )'
model.add_reactions([reaction])
reaction = Reaction('S6T25g')
reaction.name = 'heparan sulfate 6-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -3.0, Cpd2068: -1.0, Cpd140: 3.0, Cpd1207: 3.0, Cpd2069: 1.0})
reaction.gene_reaction_rule = '( HS6ST1 or HS6ST2 or HS6ST3 )'
model.add_reactions([reaction])
reaction = Reaction('S6T2g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd334: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1192: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T3g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1193: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd2083: 1.0})
reaction.gene_reaction_rule = '( CHST1 or CHST3 )'
model.add_reactions([reaction])
reaction = Reaction('S6T4g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd339: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1194: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T5g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd341: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1195: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T6g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd343: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1196: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T7g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd345: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1181: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T8g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd347: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1182: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6T9g')
reaction.name = 'galactose/N-acetylglucosamine 6-O-sulfotransferase, Golgi apparatus'
reaction.subsystem = 'Keratan sulfate synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd318: -1.0, Cpd1206: -1.0, Cpd140: 1.0, Cpd1183: 1.0, Cpd1207: 1.0})
reaction.gene_reaction_rule = '( CHST2 or CHST4 or CHST5 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE10ly')
reaction.name = 'galactose-6-sulfate sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2084: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1235: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE11ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1236: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1717: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE12ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1238: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1718: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE13ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1240: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1719: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE14ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1242: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1720: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE15ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1244: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1721: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE16ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1246: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1722: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE17ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1248: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1723: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE18ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1213: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1724: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE19ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1215: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1725: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE1ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2085: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1503: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE20ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1217: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1726: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE21ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1219: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1729: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE22ly')
reaction.name = 'galactose-6-sulfate sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2086: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1224: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE23ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1225: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1730: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE24ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1227: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1731: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE25ly')
reaction.name = 'galactose-6-sulfate sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2087: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1233: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE26ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1234: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1732: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE2ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1545: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1505: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE3ly')
reaction.name = 'N-acetylglucosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Heparan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2088: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1507: 1.0})
reaction.gene_reaction_rule = '( GNS )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE4ly')
reaction.name = 'N-acetylgalactosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1749: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1737: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE5ly')
reaction.name = 'N-acetylgalactosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1362: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1738: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE6ly')
reaction.name = 'N-acetylgalactosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1750: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1740: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE7ly')
reaction.name = 'N-acetylgalactosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd1364: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1742: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE8ly')
reaction.name = 'N-acetylgalactosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2076: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1744: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('S6TASE9ly')
reaction.name = 'N-acetylgalactosamine-6-sulfatase, lysosomal'
reaction.subsystem = 'Chondroitin sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2077: -1.0, Cpd176: 1.0, Cpd475: 1.0, Cpd1745: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('SACCD3m')
reaction.name = 'saccharopine dehydrogenase (NADP, L-lysine forming), mitochondrial'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd28: -1.0, Cpd576: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd2089: 1.0})
reaction.gene_reaction_rule = '( AASS )'
model.add_reactions([reaction])
reaction = Reaction('SACCD4m')
reaction.name = 'saccharopine dehydrogenase (NAD, L-glutamate forming), mitochondrial'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd2089: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd63: 1.0, Cpd2090: 1.0})
reaction.gene_reaction_rule = '( AASS )'
model.add_reactions([reaction])
reaction = Reaction('SADT')
reaction.name = 'sulfate adenylyltransferase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd1163: -1.0, Cpd151: 1.0, Cpd311: 1.0})
reaction.gene_reaction_rule = '( PAPSS1 or PAPSS2 )'
model.add_reactions([reaction])
reaction = Reaction('SALMCOM')
reaction.name = 'S-Adenosyl-L-methionine:catechol O-methyltransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd98: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd1664: 1.0})
reaction.gene_reaction_rule = '( COMT or COMTD1 )'
model.add_reactions([reaction])
reaction = Reaction('SALMCOM2')
reaction.name = 'S-Adenosyl-L-methionine:catechol O-methyltransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd96: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd2091: 1.0})
reaction.gene_reaction_rule = '( COMTD1 )'
model.add_reactions([reaction])
reaction = Reaction('SAMHISTA')
reaction.name = 'S-Adenosyl-L-methionine:histamine N-tele-methyltransferase'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd1481: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd1681: 1.0})
reaction.gene_reaction_rule = '( HNMT )'
model.add_reactions([reaction])
reaction = Reaction('SARCOXp')
reaction.name = 'sarcosine oxidase, peroxisomal'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2092: -1.0, Cpd247: 1.0, Cpd367: 1.0, Cpd2093: 1.0})
reaction.gene_reaction_rule = '( PIPOX )'
model.add_reactions([reaction])
reaction = Reaction('SARDHm')
reaction.name = 'Sarcosine dehydrogenase (m)'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd888: -1.0, Cpd1140: -1.0, Cpd240: 1.0, Cpd364: 1.0, Cpd1300: 1.0})
reaction.gene_reaction_rule = '( SARDH )'
model.add_reactions([reaction])
reaction = Reaction('SBPP1er')
reaction.name = 'sphingoid base-phosphate phosphatase (sphinganine 1-phosphatase), endoplasmic reticulum'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2094: -1.0, Cpd915: 1.0, Cpd2095: 1.0})
reaction.gene_reaction_rule = '( SGPP1 )'
model.add_reactions([reaction])
reaction = Reaction('SBPP3er')
reaction.name = 'sphingoid base-phosphate phosphatase (sphinganine 1-phosphatase), endoplasmic reticulum'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2096: -1.0, Cpd915: 1.0, Cpd2097: 1.0})
reaction.gene_reaction_rule = '( SGPP1 )'
model.add_reactions([reaction])
reaction = Reaction('SBTD_D2')
reaction.name = 'D-sorbitol dehydrogenase (D-fructose producing)'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2098: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1144: 1.0})
reaction.gene_reaction_rule = '( SORD )'
model.add_reactions([reaction])
reaction = Reaction('SBTR')
reaction.name = 'D-sorbitol reductase'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd1290: -1.0, Cpd25: 1.0, Cpd2098: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 or AKR7A5 )'
model.add_reactions([reaction])
reaction = Reaction('SCP21x')
reaction.name = 'Sterol carrier protein 2'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1035: -1.0, Cpd1945: 1.0})
reaction.gene_reaction_rule = '( SCP2 )'
model.add_reactions([reaction])
reaction = Reaction('SCP22x')
reaction.name = 'Sterol carrier protein 2'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2099: -1.0, Cpd740: 1.0})
reaction.gene_reaction_rule = '( SCP2 )'
model.add_reactions([reaction])
reaction = Reaction('SCP2x')
reaction.name = 'peroxisomal thiolase 2'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -8.0, Cpd256: -7.0, Cpd261: -2.0, Cpd943: -3.0, Cpd533: 7.0, Cpd742: 8.0})
reaction.gene_reaction_rule = '( SCP2 )'
model.add_reactions([reaction])
reaction = Reaction('SCPx')
reaction.name = 'peroxisomal thiolase 2'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd1518: -1.0, Cpd529: 1.0, Cpd742: 1.0})
reaction.gene_reaction_rule = '( SCP2 )'
model.add_reactions([reaction])
reaction = Reaction('SEAHCYSHYD')
reaction.name = 'Se-Adenosylselenohomocysteine hydrolase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2100: -1.0, Cpd274: 1.0, Cpd2101: 1.0})
reaction.gene_reaction_rule = '( AHCYL1 or AHCYL2 or GM4737 )'
model.add_reactions([reaction])
reaction = Reaction('SELADT')
reaction.name = 'selenate adenylyltransferase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd2102: -1.0, Cpd151: 1.0, Cpd309: 1.0})
reaction.gene_reaction_rule = '( PAPSS1 or PAPSS2 )'
model.add_reactions([reaction])
reaction = Reaction('SELCYSLY')
reaction.name = 'selenocysteine lyase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2103: -1.0, Cpd2104: -1.0, Cpd33: 1.0, Cpd394: 1.0, Cpd2105: 1.0, Cpd2106: 1.0})
reaction.gene_reaction_rule = '( SCLY )'
model.add_reactions([reaction])
reaction = Reaction('SELCYSLY2')
reaction.name = 'selenocysteine lyase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1919: -1.0, Cpd2104: -1.0, Cpd33: 1.0, Cpd394: 1.0, Cpd1920: 1.0, Cpd2106: 1.0})
reaction.gene_reaction_rule = '( SCLY )'
model.add_reactions([reaction])
reaction = Reaction('SELCYSTGL')
reaction.name = 'selanocystathionine g-lyase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2107: -1.0, Cpd6: 1.0, Cpd34: 1.0, Cpd2104: 1.0})
reaction.gene_reaction_rule = '( CTH )'
model.add_reactions([reaction])
reaction = Reaction('SELCYSTS')
reaction.name = 'selanocystathionine beta-synthase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd2101: -1.0, Cpd2: 1.0, Cpd2107: 1.0})
reaction.gene_reaction_rule = '( CBS )'
model.add_reactions([reaction])
reaction = Reaction('SELMETAT')
reaction.name = 'selenomethionine adenosyltransferase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2108: -1.0, Cpd108: 1.0, Cpd151: 1.0, Cpd2109: 1.0})
reaction.gene_reaction_rule = '( MAT1A or MAT2B )'
model.add_reactions([reaction])
reaction = Reaction('SELNPS')
reaction.name = 'Selenophosphate synthase'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2106: -1.0, Cpd108: 1.0, Cpd150: 1.0, Cpd2110: 1.0})
reaction.gene_reaction_rule = '( SEPHS1 or SEPHS2 )'
model.add_reactions([reaction])
reaction = Reaction('SELt4_3')
reaction.name = 'selenate transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -3.0, Cpd2111: -1.0, Cpd41: 3.0, Cpd2102: 1.0})
reaction.gene_reaction_rule = '( SLC13A1 )'
model.add_reactions([reaction])
reaction = Reaction('SERALANaEx')
reaction.name = 'L-alanine/L-serine Na-dependent exchange (Ser-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd405: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('SERASNNaEx')
reaction.name = 'L-serine/L-asparagine Na-dependent exchange (Ser-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd405: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('SERCYSNaEx')
reaction.name = 'L-serine/L-cysteine Na-dependent exchange (Ser-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd405: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('SERDGLNexR')
reaction.name = 'D-Serine/Glutamine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd2112: -1.0, Cpd401: 1.0, Cpd2113: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('SERDGLYexR')
reaction.name = 'D-Serine/Glycine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd2112: -1.0, Cpd403: 1.0, Cpd2113: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('SERGLNexR')
reaction.name = 'L-Serine/Glutamine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd405: -1.0, Cpd401: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('SERGLNNaEx')
reaction.name = 'L-serine/L-glutamine Na-dependent exchange (Ser-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd405: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('SERGLYexR')
reaction.name = 'L-Serine/Glycine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd405: -1.0, Cpd403: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('SERHL')
reaction.name = 'L-Serine hydro-lyase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd2: 1.0, Cpd20: 1.0})
reaction.gene_reaction_rule = '( SDS )'
model.add_reactions([reaction])
reaction = Reaction('SERLYSNaex')
reaction.name = 'Serine/Lysine Na-dependent exchange (Ser in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd465: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC3A1 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('SERPT')
reaction.name = 'serine C-palmitoyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd404: -1.0, Cpd590: -1.0, Cpd60: 1.0, Cpd66: 1.0, Cpd78: 1.0})
reaction.gene_reaction_rule = '( SPTLC1 or SPTLC2 or SPTLC3 )'
model.add_reactions([reaction])
reaction = Reaction('SERt4')
reaction.name = 'L-serine via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd41: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC38A4 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('SERTHRNaEx')
reaction.name = 'L-serine/L-threonine Na-dependent exchange (Ser-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd408: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('SERtN1')
reaction.name = 'Serine transport (Na, H coupled)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -2.0, Cpd405: -1.0, Cpd37: 1.0, Cpd41: 2.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC38A5 )'
model.add_reactions([reaction])
reaction = Reaction('SFGTH')
reaction.name = 'S-Formylglutathione hydralase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1058: -1.0, Cpd33: 1.0, Cpd699: 1.0, Cpd1057: 1.0})
reaction.gene_reaction_rule = '( ESD )'
model.add_reactions([reaction])
reaction = Reaction('SGPL11r')
reaction.name = 'Sphingosine-1-phosphate lyase 1'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2094: -1.0, Cpd2114: 1.0, Cpd2115: 1.0})
reaction.gene_reaction_rule = '( SGPL1 )'
model.add_reactions([reaction])
reaction = Reaction('SGPL12r')
reaction.name = 'Sphingosine-1-phosphate lyase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2096: -1.0, Cpd565: 1.0, Cpd2114: 1.0, Cpd1431: 1.0})
reaction.gene_reaction_rule = '( SGPL1 )'
model.add_reactions([reaction])
reaction = Reaction('SIAASE')
reaction.name = 'sialidase'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2.0, Cpd978: -1.0, Cpd235: 2.0, Cpd2116: 1.0})
reaction.gene_reaction_rule = '( NEU2 or NEU4 )'
model.add_reactions([reaction])
reaction = Reaction('SIAASE2ly')
reaction.name = 'sialidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2117: -1.0, Cpd236: 1.0, Cpd2084: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('SIAASE3ly')
reaction.name = 'sialidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2118: -1.0, Cpd236: 1.0, Cpd2086: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('SIAASE4ly')
reaction.name = 'sialidase, lysosomal'
reaction.subsystem = 'Keratan sulfate degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2119: -1.0, Cpd236: 1.0, Cpd2087: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('SIAASEly')
reaction.name = 'sialidase, lysosomal'
reaction.subsystem = 'N-glycan degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -2.0, Cpd2120: -1.0, Cpd236: 2.0, Cpd1232: 1.0})
reaction.gene_reaction_rule = '( CTSA or GALNS or GLB1 or NEU1 )'
model.add_reactions([reaction])
reaction = Reaction('SIAT4Bg')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd507: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2121: 1.0})
reaction.gene_reaction_rule = '( ST3GAL2 )'
model.add_reactions([reaction])
reaction = Reaction('SIAT9g')
reaction.name = 'Lactosylceramide alpha-2,3-sialyltransferase, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd138: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1249: 1.0})
reaction.gene_reaction_rule = '( ST3GAL5 )'
model.add_reactions([reaction])
reaction = Reaction('SLCBK1')
reaction.name = 'sphingolipid long chain base kinase (sphinganine)'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd67: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd2122: 1.0})
reaction.gene_reaction_rule = '( SPHK1 or SPHK2 )'
model.add_reactions([reaction])
reaction = Reaction('SLDx')
reaction.name = 'L-sulfolactate dehydrogenase (NAD+)'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2123: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1583: 1.0})
reaction.gene_reaction_rule = '( MDH1 or MDH1B )'
model.add_reactions([reaction])
reaction = Reaction('SLDxm')
reaction.name = 'L-sulfolactate dehydrogenase (NAD+), mitochondrial'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2124: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd1585: 1.0})
reaction.gene_reaction_rule = '( MDH2 )'
model.add_reactions([reaction])
reaction = Reaction('SMPD3g')
reaction.name = 'sphingomyelin phosphodiesterase 3, neutral membrane (neutral sphingomyelinase II)'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd2125: -1.0, Cpd140: 1.0, Cpd664: 1.0, Cpd2126: 1.0})
reaction.gene_reaction_rule = '( ENPP7 or SMPD3 or SMPD4 )'
model.add_reactions([reaction])
reaction = Reaction('SMPD3l')
reaction.name = 'sphingomyelin phosphodiesterase 3, neutral membrane (neutral sphingomyelinase II)'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2127: -1.0, Cpd176: 1.0, Cpd476: 1.0, Cpd2128: 1.0})
reaction.gene_reaction_rule = '( SMPD1 or SMPD3 )'
model.add_reactions([reaction])
reaction = Reaction('SMPD4')
reaction.name = 'sphingomyelinase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2129: -1.0, Cpd33: 1.0, Cpd677: 1.0, Cpd2130: 1.0})
reaction.gene_reaction_rule = '( SMPD2 )'
model.add_reactions([reaction])
reaction = Reaction('SMS')
reaction.name = 'Sphingomyelin synthase (homo sapiens)'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd659: -1.0, Cpd662: -1.0, Cpd658: 1.0, Cpd2131: 1.0})
reaction.gene_reaction_rule = '( SGMS1 )'
model.add_reactions([reaction])
reaction = Reaction('SO4CLtex2')
reaction.name = 'sulfate transport via chloride countertransport (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd700: -1.0, Cpd2132: -2.0, Cpd698: 1.0, Cpd1163: 2.0})
reaction.gene_reaction_rule = '( SLC26A6 )'
model.add_reactions([reaction])
reaction = Reaction('SO4HCOtex')
reaction.name = 'sulfate transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -2.0, Cpd2132: -1.0, Cpd38: 2.0, Cpd1163: 1.0})
reaction.gene_reaction_rule = '( SLC26A1 or SLC26A11 or SLC26A2 or SLC26A3 or SLC26A7 or SLC26A8 or SLC26A9 )'
model.add_reactions([reaction])
reaction = Reaction('SO4OXAtex2')
reaction.name = 'sulfate transport via oxalate countertransport (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd704: -1.0, Cpd2132: -2.0, Cpd705: 1.0, Cpd1163: 2.0})
reaction.gene_reaction_rule = '( SLC26A6 )'
model.add_reactions([reaction])
reaction = Reaction('SO4t4_2')
reaction.name = 'sulfate transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2132: -1.0, Cpd41: 2.0, Cpd1163: 1.0})
reaction.gene_reaction_rule = '( SLC13A4 )'
model.add_reactions([reaction])
reaction = Reaction('SOAT11')
reaction.name = 'sterol O-acyltransferase (acyl-Coenzyme A: cholesterol acyltransferase) 1'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2133: -1.0, Cpd682: -1.0, Cpd78: 1.0, Cpd2134: 1.0})
reaction.gene_reaction_rule = '( SOAT1 )'
model.add_reactions([reaction])
reaction = Reaction('SOAT11r')
reaction.name = 'sterol O-acyltransferase (acyl-Coenzyme A: cholesterol acyltransferase) 1'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2135: -1.0, Cpd745: 1.0, Cpd2136: 1.0})
reaction.gene_reaction_rule = '( SOAT1 )'
model.add_reactions([reaction])
reaction = Reaction('SOAT12')
reaction.name = 'sterol O-acyltransferase (acyl-Coenzyme A: cholesterol acyltransferase) 1'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2137: -1.0, Cpd682: -1.0, Cpd78: 1.0, Cpd2134: 1.0})
reaction.gene_reaction_rule = '( SOAT1 )'
model.add_reactions([reaction])
reaction = Reaction('SOAT12r')
reaction.name = 'sterol O-acyltransferase (acyl-Coenzyme A: cholesterol acyltransferase) 1'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2138: -1.0, Cpd745: 1.0, Cpd2136: 1.0})
reaction.gene_reaction_rule = '( SOAT1 )'
model.add_reactions([reaction])
reaction = Reaction('SPHK21c')
reaction.name = 'sphingosine kinase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2130: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd2139: 1.0})
reaction.gene_reaction_rule = '( SPHK1 or SPHK2 )'
model.add_reactions([reaction])
reaction = Reaction('SPMS')
reaction.name = 'spermidine synthase'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd289: -1.0, Cpd1846: -1.0, Cpd33: 1.0, Cpd1710: 1.0, Cpd2140: 1.0})
reaction.gene_reaction_rule = '( SRM )'
model.add_reactions([reaction])
reaction = Reaction('SPODM')
reaction.name = 'superoxide dismutase'
reaction.subsystem = 'ROS detoxification'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd2141: -2.0, Cpd3: 1.0, Cpd5: 1.0})
reaction.gene_reaction_rule = '( SOD1 )'
model.add_reactions([reaction])
reaction = Reaction('SPODMe')
reaction.name = 'superoxide dismutase, extracellular'
reaction.subsystem = 'ROS detoxification'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -2.0, Cpd2142: -2.0, Cpd1444: 1.0, Cpd2143: 1.0})
reaction.gene_reaction_rule = '( SOD3 )'
model.add_reactions([reaction])
reaction = Reaction('SPODMm')
reaction.name = 'superoxide dismutase'
reaction.subsystem = 'ROS detoxification'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -2.0, Cpd2144: -2.0, Cpd9: 1.0, Cpd634: 1.0})
reaction.gene_reaction_rule = '( SOD2 )'
model.add_reactions([reaction])
reaction = Reaction('SPODMn')
reaction.name = 'superoxide dismutase, nuclear'
reaction.subsystem = 'ROS detoxification'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -2.0, Cpd2145: -2.0, Cpd2146: 1.0, Cpd2147: 1.0})
reaction.gene_reaction_rule = '( SOD1 )'
model.add_reactions([reaction])
reaction = Reaction('SPODMx')
reaction.name = 'superoxide dismutase, peroxisome'
reaction.subsystem = 'ROS detoxification'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -2.0, Cpd2148: -2.0, Cpd246: 1.0, Cpd247: 1.0})
reaction.gene_reaction_rule = '( SOD1 )'
model.add_reactions([reaction])
reaction = Reaction('SPR')
reaction.name = 'sepiapterin reductase'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd22: -2.0, Cpd2029: -1.0, Cpd25: 2.0, Cpd870: 1.0})
reaction.gene_reaction_rule = '( SPR )'
model.add_reactions([reaction])
reaction = Reaction('SPRMS')
reaction.name = 'spermine synthase'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd289: -1.0, Cpd2140: -1.0, Cpd33: 1.0, Cpd2149: 1.0, Cpd1710: 1.0})
reaction.gene_reaction_rule = '( SMS or SRM )'
model.add_reactions([reaction])
reaction = Reaction('SPRn')
reaction.name = 'sepiapterin reductase, nuclear'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -2.0, Cpd2031: -1.0, Cpd2150: -2.0, Cpd2151: 2.0, Cpd2152: 1.0})
reaction.gene_reaction_rule = '( SPR )'
model.add_reactions([reaction])
reaction = Reaction('SPTix')
reaction.name = 'serine-pyruvate aminotransferase (irreversible), peroxisomal'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd368: -1.0, Cpd2153: -1.0, Cpd365: 1.0, Cpd2154: 1.0})
reaction.gene_reaction_rule = '( AGXT )'
model.add_reactions([reaction])
reaction = Reaction('SQLEr')
reaction.name = 'Squalene epoxidase, endoplasmic reticular (NADP)'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2155: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1614: 1.0})
reaction.gene_reaction_rule = '( SQLE )'
model.add_reactions([reaction])
reaction = Reaction('SQLSr')
reaction.name = 'Squalene synthase'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd2156: -2.0, Cpd589: 1.0, Cpd1988: 2.0, Cpd2155: 1.0})
reaction.gene_reaction_rule = '( FDFT1 )'
model.add_reactions([reaction])
reaction = Reaction('SR5AR2r')
reaction.name = '3-oxo-5-alpha-steroid 4-dehydrogenase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1514: -1.0, Cpd589: 1.0, Cpd1523: 1.0})
reaction.gene_reaction_rule = '( SRD5A1 or SRD5A2 )'
model.add_reactions([reaction])
reaction = Reaction('SR5ARr')
reaction.name = '3-oxo-5-alpha-steroid 4-dehydrogenase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1515: -1.0, Cpd2157: 1.0, Cpd589: 1.0})
reaction.gene_reaction_rule = '( SRD5A1 or SRD5A2 )'
model.add_reactions([reaction])
reaction = Reaction('SRTN23OX')
reaction.name = '5-Hydroxytryptamine:oxygen 2,3-dioxygenase (indole-decyclizing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd116: -1.0, Cpd2158: 1.0})
reaction.gene_reaction_rule = '( IDO1 )'
model.add_reactions([reaction])
reaction = Reaction('SRTNACT')
reaction.name = 'Acetyl-CoA:aralkylamine N-acetyltransferase'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd116: -1.0, Cpd180: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd269: 1.0})
reaction.gene_reaction_rule = '( AANAT )'
model.add_reactions([reaction])
reaction = Reaction('SRTNMTX')
reaction.name = 'S-Adenosyl-L-methionine:amine N-methyltransferase (srtn)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd52: -1.0, Cpd116: -1.0, Cpd33: 1.0, Cpd53: 1.0, Cpd2159: 1.0})
reaction.gene_reaction_rule = '( INMT )'
model.add_reactions([reaction])
reaction = Reaction('SRTNt6_2_r')
reaction.name = 'Seratonin reversible transport in via sodium symport/potassium antiport (1:2)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd487: -2.0, Cpd2160: -1.0, Cpd41: 2.0, Cpd116: 1.0, Cpd489: 2.0})
reaction.gene_reaction_rule = '( SLC6A4 )'
model.add_reactions([reaction])
reaction = Reaction('SRTNtu')
reaction.name = 'Serotonin uniport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2160: -1.0, Cpd116: 1.0})
reaction.gene_reaction_rule = '( SLC22A1 or SLC22A2 )'
model.add_reactions([reaction])
reaction = Reaction('ST3GAL21g')
reaction.name = 'CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd500: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2161: 1.0})
reaction.gene_reaction_rule = '( ST3GAL2 )'
model.add_reactions([reaction])
reaction = Reaction('ST3GAL22g')
reaction.name = 'CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd502: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2162: 1.0})
reaction.gene_reaction_rule = '( ST3GAL2 )'
model.add_reactions([reaction])
reaction = Reaction('ST3GAL23g')
reaction.name = 'CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase, Golgi apparatus'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd504: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2163: 1.0})
reaction.gene_reaction_rule = '( ST3GAL2 )'
model.add_reactions([reaction])
reaction = Reaction('ST3GAL31g')
reaction.name = 'ST3 beta-galactoside alpha-2,3-sialyltransferase 3'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1167: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2164: 1.0})
reaction.gene_reaction_rule = '( ST3GAL3 )'
model.add_reactions([reaction])
reaction = Reaction('ST3GAL61g')
reaction.name = 'Type 2 lactosamine alpha-2,3-sialyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd508: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1171: 1.0})
reaction.gene_reaction_rule = '( ST3GAL6 )'
model.add_reactions([reaction])
reaction = Reaction('ST3GAL62g')
reaction.name = 'Type 2 lactosamine alpha-2,3-sialyltransferase'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd510: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd2165: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST3GAL6 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC21')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2121: -1.0, Cpd140: 1.0, Cpd2166: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC22')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2121: -1.0, Cpd140: 1.0, Cpd2167: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC23')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd499: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd526: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC24')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd527: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2168: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC25')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2162: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2169: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC26')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2169: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2170: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC27')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd506: -1.0, Cpd714: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2171: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC28')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 2'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2171: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2172: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC2 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC31')
reaction.name = 'Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 3'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2161: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2162: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC3 or ST6GALNAC5 or ST6GALNAC6 )'
model.add_reactions([reaction])
reaction = Reaction('ST6GALNAC62')
reaction.name = 'sialytransferase 7 ((alpha-N-acetylneuraminyl 2,3-betagalactosyl-1,3)-N-acetyl galactosaminide alpha-2,6-sialytransferase) F'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2163: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2170: 1.0})
reaction.gene_reaction_rule = '( ST6GALNAC6 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA11')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2121: -1.0, Cpd140: 1.0, Cpd2173: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST8SIA1 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA12')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd1249: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1250: 1.0})
reaction.gene_reaction_rule = '( ST8SIA1 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA51g')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd1250: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd1251: 1.0})
reaction.gene_reaction_rule = '( ST8SIA5 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA52g')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2163: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2174: 1.0})
reaction.gene_reaction_rule = '( ST8SIA5 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA53g')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2162: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2175: 1.0})
reaction.gene_reaction_rule = '( ST8SIA5 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA54g')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2161: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2176: 1.0})
reaction.gene_reaction_rule = '( ST8SIA5 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA55g')
reaction.name = 'Beta-galactoside alpha-2,3-sialyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd2171: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd2177: 1.0})
reaction.gene_reaction_rule = '( ST8SIA5 )'
model.add_reactions([reaction])
reaction = Reaction('ST8SIA56g')
reaction.name = 'Alpha-2,8-sialyltransferase 8E'
reaction.subsystem = 'Blood group synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd714: -1.0, Cpd1171: -1.0, Cpd140: 1.0, Cpd2178: 1.0, Cpd712: 1.0})
reaction.gene_reaction_rule = '( ST8SIA5 )'
model.add_reactions([reaction])
reaction = Reaction('STRDNCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd824: -1.0, Cpd78: 1.0, Cpd2179: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('STRDNCCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2180: -1.0, Cpd307: 1.0, Cpd1074: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('STRDNCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2179: -1.0, Cpd2180: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('SUCCt2m')
reaction.name = 'succinate transport, mitochondrial'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd204: -1.0, Cpd543: -1.0, Cpd108: 1.0, Cpd1843: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('SUCCt4_2')
reaction.name = 'succinate transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2181: -1.0, Cpd41: 2.0, Cpd543: 1.0})
reaction.gene_reaction_rule = '( SLC13A2 )'
model.add_reactions([reaction])
reaction = Reaction('SUCD1m')
reaction.name = 'succinate dehydrogenase'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd1843: -1.0, Cpd240: 1.0, Cpd1159: 1.0})
reaction.gene_reaction_rule = '( SDHA or SDHB or SDHC or SDHD )'
model.add_reactions([reaction])
reaction = Reaction('SUCOAS1m')
reaction.name = 'Succinate--CoA ligase (GDP-forming)'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd284: -1.0, Cpd1843: -1.0, Cpd204: 1.0, Cpd285: 1.0, Cpd380: 1.0})
reaction.gene_reaction_rule = '( SUCLG1 or SUCLG2 )'
model.add_reactions([reaction])
reaction = Reaction('SUCOASm')
reaction.name = 'Succinate--CoA ligase (ADP-forming)'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd1843: -1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd380: 1.0})
reaction.gene_reaction_rule = '( SUCLA2 or SUCLG1 )'
model.add_reactions([reaction])
reaction = Reaction('SUCRe')
reaction.name = 'sucrose hydrolyzing enxyme, extracellular'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2182: -1.0, Cpd446: 1.0, Cpd1143: 1.0})
reaction.gene_reaction_rule = '( MGAM or SIS )'
model.add_reactions([reaction])
reaction = Reaction('SULFOX')
reaction.name = 'sulfite oxidase'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1161: -1.0, Cpd751: -2.0, Cpd33: 2.0, Cpd750: 2.0, Cpd1163: 1.0})
reaction.gene_reaction_rule = '( SUOX )'
model.add_reactions([reaction])
reaction = Reaction('T2M26DCOAHLm')
reaction.name = 'trans-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA hydro-lyase (m)'
reaction.subsystem = 'Limonene and pinene degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2183: -1.0, Cpd619: 1.0})
reaction.gene_reaction_rule = '( AUH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('T2M26DCOAHLx')
reaction.name = 'trans-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA hydro-lyase (x)'
reaction.subsystem = 'Limonene and pinene degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2184: -1.0, Cpd621: 1.0})
reaction.gene_reaction_rule = '( ECH1 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('T4HCINNOX')
reaction.name = '4-Coumarate:oxygen oxidoreductase'
reaction.subsystem = 'Stilbene, coumarine and lignin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd2185: -2.0, Cpd2186: 2.0})
reaction.gene_reaction_rule = '( TYR )'
model.add_reactions([reaction])
reaction = Reaction('TALA')
reaction.name = 'transaldolase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd949: -1.0, Cpd2187: -1.0, Cpd1108: 1.0, Cpd2188: 1.0})
reaction.gene_reaction_rule = '( TALDO1 )'
model.add_reactions([reaction])
reaction = Reaction('TAURt4_2_r')
reaction.name = 'taurine transport (sodium symport) (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2189: -1.0, Cpd41: 2.0, Cpd1586: 1.0})
reaction.gene_reaction_rule = '( SLC6A6 )'
model.add_reactions([reaction])
reaction = Reaction('TAURtcx')
reaction.name = 'taurine transport (sodium symport) (cytosol to peroxisome)'
reaction.subsystem = 'Taurine and hypotaurine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd1586: -1.0, Cpd531: 1.0, Cpd2190: 1.0})
reaction.gene_reaction_rule = '( SLC6A6 )'
model.add_reactions([reaction])
reaction = Reaction('TCHOLAt')
reaction.name = 'taurocholate transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2191: -1.0, Cpd38: 1.0, Cpd2192: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 or SLCO4A1 )'
model.add_reactions([reaction])
reaction = Reaction('TCHOLAt2')
reaction.name = 'taurocholate transport via sodium cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2191: -1.0, Cpd41: 2.0, Cpd2192: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 or SLC10A2 )'
model.add_reactions([reaction])
reaction = Reaction('TCHOLAt3')
reaction.name = 'ABC bile acid transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2192: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2191: 1.0})
reaction.gene_reaction_rule = '( ABCB11 or ABCC3 )'
model.add_reactions([reaction])
reaction = Reaction('TDPGDH')
reaction.name = 'dTDPglucose 4,6-dehydratase'
reaction.subsystem = 'Nucleotide sugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2193: -1.0, Cpd2: 1.0, Cpd2194: 1.0})
reaction.gene_reaction_rule = '( TGDS )'
model.add_reactions([reaction])
reaction = Reaction('TETPENT3CPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd833: -1.0, Cpd78: 1.0, Cpd2195: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('TETPENT3CPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2196: -1.0, Cpd307: 1.0, Cpd2197: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('TETPENT3CRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2195: -1.0, Cpd2196: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('TETPENT6CPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1047: -1.0, Cpd78: 1.0, Cpd2195: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('TETPENT6CPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2196: -1.0, Cpd307: 1.0, Cpd2198: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('TETPENT6CRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2195: -1.0, Cpd2196: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('TETTET6CPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1045: -1.0, Cpd78: 1.0, Cpd2199: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('TETTET6CPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2200: -1.0, Cpd307: 1.0, Cpd2201: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('TETTET6CRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2199: -1.0, Cpd2200: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('THBPT4ACAMDASE')
reaction.name = 'Tetrahydrobiopterin-4a-carbinolamine dehydratase'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1944: -1.0, Cpd2: 1.0, Cpd869: 1.0})
reaction.gene_reaction_rule = '( PCBD1 )'
model.add_reactions([reaction])
reaction = Reaction('THD1m')
reaction.name = 'NAD(P) transhydrogenase'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd33: -1.0, Cpd47: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd44: 1.0})
reaction.gene_reaction_rule = '( NNT )'
model.add_reactions([reaction])
reaction = Reaction('THFt2')
reaction.name = 'tetrahydrofolate transport via anion exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1323: -1.0, Cpd114: 1.0, Cpd377: 1.0})
reaction.gene_reaction_rule = '( SLC19A1 )'
model.add_reactions([reaction])
reaction = Reaction('THMDt4')
reaction.name = 'thymidine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2202: -1.0, Cpd41: 1.0, Cpd1830: 1.0})
reaction.gene_reaction_rule = '( SLC28A1 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('THMMPt4')
reaction.name = 'Thiamine monophosphate transport in via anion antiport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2203: -1.0, Cpd114: 1.0, Cpd2204: 1.0})
reaction.gene_reaction_rule = '( SLC19A1 )'
model.add_reactions([reaction])
reaction = Reaction('THMt3')
reaction.name = 'Thiamine transport in via proton  antiport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd2205: -1.0, Cpd37: 1.0, Cpd2206: 1.0})
reaction.gene_reaction_rule = '( SLC19A2 or SLC19A3 )'
model.add_reactions([reaction])
reaction = Reaction('THMTP')
reaction.name = 'thiamine-triphosphatase'
reaction.subsystem = 'Thiamine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2207: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2208: 1.0})
reaction.gene_reaction_rule = '( THTPA )'
model.add_reactions([reaction])
reaction = Reaction('THMTPt')
reaction.name = 'Thiamine triphosphate transport in via anion antiport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -3.0, Cpd2207: -1.0, Cpd2: 3.0, Cpd2209: 1.0})
reaction.gene_reaction_rule = '( SLC19A1 )'
model.add_reactions([reaction])
reaction = Reaction('THRALANaEx')
reaction.name = 'L-alanine/L-threonine Na-dependent exchange (Thr-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('THRASNNaEx')
reaction.name = 'L-threonine/L-asparagine Na-dependent exchange (Thr-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('THRCYSNaEx')
reaction.name = 'L-cysteine/L-threonine Na-dependent exchange (Thr-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('THRD_L')
reaction.name = 'L-threonine deaminase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd6: 1.0, Cpd34: 1.0})
reaction.gene_reaction_rule = '( SDS or SDSL )'
model.add_reactions([reaction])
reaction = Reaction('THRGLNexR')
reaction.name = 'L-threonine/glycine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd409: -1.0, Cpd401: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('THRGLNNaEx')
reaction.name = 'L-threonine/L-glutamine Na-dependent exchange (Thr-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('THRGLYexR')
reaction.name = 'L-threonine/glycine reversible exchange'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd409: -1.0, Cpd403: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A10 )'
model.add_reactions([reaction])
reaction = Reaction('THRS')
reaction.name = 'threonine synthase'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2210: -1.0, Cpd108: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( THNSL1 )'
model.add_reactions([reaction])
reaction = Reaction('THRSERNaEx')
reaction.name = 'L-serine/L-threonine Na-dependent exchange (Thr-L in)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 or SLC1A5 )'
model.add_reactions([reaction])
reaction = Reaction('THRt4')
reaction.name = 'L-threonine  via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC38A1 or SLC38A2 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('THYMDt1')
reaction.name = 'thymd transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2202: -1.0, Cpd1830: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 or SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('THYMDtl')
reaction.name = 'thymidine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1832: -1.0, Cpd1830: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('THYMDtm')
reaction.name = 'thymd transport'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1830: -1.0, Cpd1834: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 )'
model.add_reactions([reaction])
reaction = Reaction('THYMt')
reaction.name = 'thymine reversible transport via facilated diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2211: -1.0, Cpd952: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('THYOXt')
reaction.name = 'T4 transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2212: -1.0, Cpd38: 1.0, Cpd1543: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 or SLCO1C1 or SLCO4A1 )'
model.add_reactions([reaction])
reaction = Reaction('THYOXt2')
reaction.name = 'T4 transport via facilitated diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2212: -1.0, Cpd1543: 1.0})
reaction.gene_reaction_rule = '( SLC16A2 )'
model.add_reactions([reaction])
reaction = Reaction('THYPX')
reaction.name = 'thyroid peroxidase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd33: -2.0, Cpd702: -2.0, Cpd2: 2.0, Cpd1540: 2.0})
reaction.gene_reaction_rule = '( TPO )'
model.add_reactions([reaction])
reaction = Reaction('TKT1')
reaction.name = 'transketolase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd297: -1.0, Cpd2054: -1.0, Cpd949: 1.0, Cpd2187: 1.0})
reaction.gene_reaction_rule = '( TKT or TKTL1 or TKTL2 )'
model.add_reactions([reaction])
reaction = Reaction('TKT2')
reaction.name = 'transketolase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2054: -1.0, Cpd2188: -1.0, Cpd949: 1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( TKT or TKTL1 or TKTL2 )'
model.add_reactions([reaction])
reaction = Reaction('TMABADH')
reaction.name = '4-trimethylaminobutyraldehyde dehydrogenase'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1344: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd542: 1.0})
reaction.gene_reaction_rule = '( ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('TMDK1')
reaction.name = 'thymidine kinase (ATP:thymidine)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1830: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd950: 1.0})
reaction.gene_reaction_rule = '( TK1 or TK2 )'
model.add_reactions([reaction])
reaction = Reaction('TMDK1m')
reaction.name = 'thymidine kinase (ATP:thymidine)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1834: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd1833: 1.0})
reaction.gene_reaction_rule = '( TK2 )'
model.add_reactions([reaction])
reaction = Reaction('TMDPK')
reaction.name = 'Thiamin pyrophosphokinase (EC 2.7.6.2)'
reaction.subsystem = 'Thiamine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2206: -1.0, Cpd33: 1.0, Cpd150: 1.0, Cpd2208: 1.0})
reaction.gene_reaction_rule = '( TPK1 )'
model.add_reactions([reaction])
reaction = Reaction('TMDPP')
reaction.name = 'thymidine phosphorylase'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -1.0, Cpd1830: -1.0, Cpd955: 1.0, Cpd952: 1.0})
reaction.gene_reaction_rule = '( TYMP )'
model.add_reactions([reaction])
reaction = Reaction('TMDS')
reaction.name = 'thymidylate synthase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd810: -1.0, Cpd1341: -1.0, Cpd863: 1.0, Cpd950: 1.0})
reaction.gene_reaction_rule = '( TYMS )'
model.add_reactions([reaction])
reaction = Reaction('TMLYSOX')
reaction.name = 'trimethyllysine dioxygenase'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd30: -1.0, Cpd2213: -1.0, Cpd60: 1.0, Cpd543: 1.0, Cpd1343: 1.0})
reaction.gene_reaction_rule = '( TMLHE )'
model.add_reactions([reaction])
reaction = Reaction('TMNDNCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd832: -1.0, Cpd78: 1.0, Cpd2214: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('TMNDNCCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2215: -1.0, Cpd307: 1.0, Cpd1077: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('TMNDNCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2214: -1.0, Cpd2215: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('TPI')
reaction.name = 'triose-phosphate isomerase'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd425: -1.0, Cpd949: 1.0})
reaction.gene_reaction_rule = '( TPI1 )'
model.add_reactions([reaction])
reaction = Reaction('TRDR')
reaction.name = 'thioredoxin reductase (NADPH)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2053: -1.0, Cpd25: 1.0, Cpd2052: 1.0})
reaction.gene_reaction_rule = '( TXNRD1 )'
model.add_reactions([reaction])
reaction = Reaction('TRDR2')
reaction.name = 'Thioredoxin (ubiquinone 10) reductase (NADPH)'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2216: -1.0, Cpd25: 1.0, Cpd2217: 1.0})
reaction.gene_reaction_rule = '( TXNRD1 )'
model.add_reactions([reaction])
reaction = Reaction('TRDR3')
reaction.name = 'Thioredoxin (ubiquinone 10) reductase (NADH)'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd2216: -1.0, Cpd32: 1.0, Cpd2217: 1.0})
reaction.gene_reaction_rule = '( TXNRD1 )'
model.add_reactions([reaction])
reaction = Reaction('TRDRm')
reaction.name = 'thioredoxin reductase (NADPH)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd2218: -1.0, Cpd13: 1.0, Cpd2219: 1.0})
reaction.gene_reaction_rule = '( TXNRD2 or TXNRD3 )'
model.add_reactions([reaction])
reaction = Reaction('TREH')
reaction.name = 'alpha,alpha-trehalase'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2220: -1.0, Cpd1290: 2.0})
reaction.gene_reaction_rule = '( PGGHG or TREH )'
model.add_reactions([reaction])
reaction = Reaction('TREHe')
reaction.name = 'alpha,alpha-trehalase'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2221: -1.0, Cpd446: 2.0})
reaction.gene_reaction_rule = '( TREH )'
model.add_reactions([reaction])
reaction = Reaction('TRIODTHYSULT')
reaction.name = 'Triiodothyronine Sulfotransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd1542: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd2222: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('TRIODTHYt')
reaction.name = 'T3 transport via bicarbonate countertransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2223: -1.0, Cpd38: 1.0, Cpd1542: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 or SLCO1B2 or SLCO1C1 or SLCO4A1 )'
model.add_reactions([reaction])
reaction = Reaction('TRIODTHYt2')
reaction.name = 'T3 transport via facilitated diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2223: -1.0, Cpd1542: 1.0})
reaction.gene_reaction_rule = '( SLC16A2 )'
model.add_reactions([reaction])
reaction = Reaction('TRPHYDRO2')
reaction.name = 'L-Tryptophan,tetrahydrobiopterin:oxygen oxidoreductase (5-hydroxylating)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd870: -1.0, Cpd1638: -1.0, Cpd115: 1.0, Cpd1944: 1.0})
reaction.gene_reaction_rule = '( TPH1 or TPH2 )'
model.add_reactions([reaction])
reaction = Reaction('TRPO2')
reaction.name = 'L-Tryptophan:oxygen 2,3-oxidoreductase (decyclizing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd1638: -1.0, Cpd1116: 1.0})
reaction.gene_reaction_rule = '( IDO1 or IDO2 or TDO2 )'
model.add_reactions([reaction])
reaction = Reaction('TRPt')
reaction.name = 'L-tryptophan transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC16A10 or SLC36A4 )'
model.add_reactions([reaction])
reaction = Reaction('TRPt4')
reaction.name = 'L-tryptophan transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('TRYPTAOX')
reaction.name = 'Tryptamine:oxygen oxidoreductase(deaminating)(flavin-containing)'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1639: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd418: 1.0})
reaction.gene_reaction_rule = '( AOC1 or AOC2 or MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('TSTSTERONEGLCte')
reaction.name = 'glucuronidated compound transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2225: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2226: 1.0})
reaction.gene_reaction_rule = '( ABCC1 )'
model.add_reactions([reaction])
reaction = Reaction('TSTSTERONESULT')
reaction.name = 'testosterone sulfotransferase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd2227: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd2228: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 or SULT2A1 )'
model.add_reactions([reaction])
reaction = Reaction('TSULt4_3')
reaction.name = 'thiosulfate transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -3.0, Cpd2229: -1.0, Cpd41: 3.0, Cpd1165: 1.0})
reaction.gene_reaction_rule = '( SLC13A1 )'
model.add_reactions([reaction])
reaction = Reaction('TTDCAtr')
reaction.name = 'tetradecanoate (transport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2230: -1.0, Cpd1006: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('TTDCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1019: -1.0, Cpd78: 1.0, Cpd2231: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('TTDCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2232: -1.0, Cpd307: 1.0, Cpd1061: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('TTDCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2231: -1.0, Cpd2232: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('TXASr')
reaction.name = 'Thromboxane-A synthase'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1931: -1.0, Cpd2233: 1.0})
reaction.gene_reaction_rule = '( TBXAS1 )'
model.add_reactions([reaction])
reaction = Reaction('TYMSULT')
reaction.name = 'Tyramine Sulfotransferase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd2234: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd2235: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('TYR3MO2')
reaction.name = 'tyrosine 3-monooxygenase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd870: -1.0, Cpd1544: -1.0, Cpd56: 1.0, Cpd1944: 1.0})
reaction.gene_reaction_rule = '( TH )'
model.add_reactions([reaction])
reaction = Reaction('TYRASE')
reaction.name = 'Tyrosinase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd816: -1.0, Cpd2236: 1.0})
reaction.gene_reaction_rule = '( TYRP1 )'
model.add_reactions([reaction])
reaction = Reaction('TYRCBOX')
reaction.name = 'L-Tyrosine carboxy-lyase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1544: -1.0, Cpd60: 1.0, Cpd2234: 1.0})
reaction.gene_reaction_rule = '( DDC )'
model.add_reactions([reaction])
reaction = Reaction('TYRDOPO')
reaction.name = 'Tyrosine:dopa oxidase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd1544: -2.0, Cpd56: 2.0})
reaction.gene_reaction_rule = '( TYR )'
model.add_reactions([reaction])
reaction = Reaction('TYRDOPO3')
reaction.name = 'Tyrosine:dopa oxidase (dopaquinone producing 2)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd56: -2.0, Cpd2: 2.0, Cpd2237: 2.0})
reaction.gene_reaction_rule = '( TYR )'
model.add_reactions([reaction])
reaction = Reaction('TYROXDAc')
reaction.name = 'Tyramine:oxygen oxidoreductase(deaminating)(flavin-containing) (cytosol)'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2234: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd101: 1.0})
reaction.gene_reaction_rule = '( MAOA or MAOB )'
model.add_reactions([reaction])
reaction = Reaction('TYRt')
reaction.name = 'L-tyrosine transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2238: -1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC16A10 )'
model.add_reactions([reaction])
reaction = Reaction('TYRt4')
reaction.name = 'L-tyrosine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('TYRTA')
reaction.name = 'tyrosine transaminase'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd1544: -1.0, Cpd59: 1.0, Cpd74: 1.0})
reaction.gene_reaction_rule = '( GOT1 or TAT )'
model.add_reactions([reaction])
reaction = Reaction('TYRTAm')
reaction.name = 'tyrosine transaminase, mitochondrial'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd2239: -1.0, Cpd2240: 1.0, Cpd63: 1.0})
reaction.gene_reaction_rule = '( GOT2 or TAT )'
model.add_reactions([reaction])
reaction = Reaction('UAG2EMAi')
reaction.name = 'UDP-N-acetyl-D-glucosamine 2-epimerase (Hydrolysis)'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1396: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd216: 1.0})
reaction.gene_reaction_rule = '( GNE )'
model.add_reactions([reaction])
reaction = Reaction('UAG4E')
reaction.name = 'UDP-N-acetylglucosamine 4-epimerase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1396: -1.0, Cpd2241: 1.0})
reaction.gene_reaction_rule = '( GALE )'
model.add_reactions([reaction])
reaction = Reaction('UAGDP')
reaction.name = 'UDP-N-acetylglucosamine diphosphorylase'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd219: -1.0, Cpd749: -1.0, Cpd151: 1.0, Cpd1396: 1.0})
reaction.gene_reaction_rule = '( UAP1 or UAP1L1 )'
model.add_reactions([reaction])
reaction = Reaction('UDPDOLPT_L')
reaction.name = 'UDPglucose:dolichyl-phosphate beta-D-glucosyltransferase (liver)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd930: -0.1, Cpd1270: -1.0, Cpd137: 1.0, Cpd2242: 0.1})
reaction.gene_reaction_rule = '( ALG5 )'
model.add_reactions([reaction])
reaction = Reaction('UDPDOLPT_U')
reaction.name = 'UDPglucose:dolichyl-phosphate beta-D-glucosyltransferase (uterus)'
reaction.subsystem = 'N-glycan synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd930: -0.1, Cpd1270: -1.0, Cpd137: 1.0, Cpd2243: 0.1})
reaction.gene_reaction_rule = '( ALG5 )'
model.add_reactions([reaction])
reaction = Reaction('UDPG4E')
reaction.name = 'UDPglucose 4-epimerase'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1270: -1.0, Cpd135: 1.0})
reaction.gene_reaction_rule = '( GALE )'
model.add_reactions([reaction])
reaction = Reaction('UDPGALtg')
reaction.name = 'UDP-Gal Golgi transport via CMP antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd135: -1.0, Cpd1774: -1.0, Cpd139: 1.0, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( SLC35A2 )'
model.add_reactions([reaction])
reaction = Reaction('UDPGD')
reaction.name = 'UDPglucose 6-dehydrogenase'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -2.0, Cpd1270: -1.0, Cpd33: 3.0, Cpd35: 2.0, Cpd1467: 1.0})
reaction.gene_reaction_rule = '( UGDH )'
model.add_reactions([reaction])
reaction = Reaction('UDPGLCAter')
reaction.name = 'UDPGlcA endoplasmic reticulum transport via UMP antiport'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1398: -1.0, Cpd2244: -1.0, Cpd1467: 1.0, Cpd1770: 1.0})
reaction.gene_reaction_rule = '( SLC35D1 )'
model.add_reactions([reaction])
reaction = Reaction('UDPGLCtg')
reaction.name = 'UDP-Glc Golgi transport via CMP antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1270: -1.0, Cpd1774: -1.0, Cpd1398: 1.0, Cpd2245: 1.0})
reaction.gene_reaction_rule = '( SLC35A3 or SLC35B4 or SLC35D2 )'
model.add_reactions([reaction])
reaction = Reaction('UDPGLDCg')
reaction.name = 'UDP-D-glucuronate decarboxylase, Golgi apparatus'
reaction.subsystem = 'Nucleotide sugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd140: -1.0, Cpd1373: -1.0, Cpd2246: 1.0, Cpd2247: 1.0})
reaction.gene_reaction_rule = '( UXS1 )'
model.add_reactions([reaction])
reaction = Reaction('UDPGP')
reaction.name = 'UDPglucose pyrophosphohydrolase'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1270: -1.0, Cpd33: 2.0, Cpd293: 1.0, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( ENPP1 or ENPP2 or ENPP3 )'
model.add_reactions([reaction])
reaction = Reaction('UDPXYLtg')
reaction.name = 'UDP-Xyl Golgi transport via CMP antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1774: -1.0, Cpd2248: -1.0, Cpd1398: 1.0, Cpd2247: 1.0})
reaction.gene_reaction_rule = '( SLC35B4 )'
model.add_reactions([reaction])
reaction = Reaction('UGALGTg')
reaction.name = 'UDPgalactose:D-glucose 4-beta-D-galactosyltransferase, Golgi apparatus'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1400: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd2249: 1.0})
reaction.gene_reaction_rule = '( B4GALT1 or B4GALT2 or LALBA )'
model.add_reactions([reaction])
reaction = Reaction('UGALNACter')
reaction.name = 'UDP-GalNAc endoplasmic reticulum transport via CMP antiport'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1770: -1.0, Cpd2241: -1.0, Cpd1398: 1.0, Cpd2250: 1.0})
reaction.gene_reaction_rule = '( SLC35D1 )'
model.add_reactions([reaction])
reaction = Reaction('UGALNACtg')
reaction.name = 'UDP-GalNAc Golgi transport via CMP antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1774: -1.0, Cpd2241: -1.0, Cpd155: 1.0, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( SLC35A2 )'
model.add_reactions([reaction])
reaction = Reaction('UGCG')
reaction.name = 'Ceramide glucosyltransferase'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd665: -1.0, Cpd2251: -1.0, Cpd565: 1.0, Cpd667: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGCG )'
model.add_reactions([reaction])
reaction = Reaction('UGLCNACtg')
reaction.name = 'UDP-GlcNAc Golgi transport via CMP antiport'
reaction.subsystem = 'Transport, golgi apparatus'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1396: -1.0, Cpd1774: -1.0, Cpd144: 1.0, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( SLC35D2 )'
model.add_reactions([reaction])
reaction = Reaction('UGLT')
reaction.name = 'UDPglucose--hexose-1-phosphate uridylyltransferase'
reaction.subsystem = 'Galactose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1253: -1.0, Cpd1270: -1.0, Cpd135: 1.0, Cpd293: 1.0})
reaction.gene_reaction_rule = '( GALT )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A10r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2252: -1.0, Cpd565: -2.0, Cpd2244: -1.0, Cpd2253: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A1r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1519: -1.0, Cpd2244: -1.0, Cpd2254: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A2r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2252: -1.0, Cpd2244: -2.0, Cpd2255: 1.0, Cpd1769: 2.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A3r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1521: -1.0, Cpd2244: -1.0, Cpd2256: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A4r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1515: -1.0, Cpd2244: -1.0, Cpd1769: 1.0, Cpd2257: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A5r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2258: -1.0, Cpd2244: -1.0, Cpd1769: 1.0, Cpd2259: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A5r2')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal (13-cis)'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2260: -1.0, Cpd2244: -1.0, Cpd1769: 1.0, Cpd2261: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A6r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1520: -1.0, Cpd2244: -1.0, Cpd2262: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A7r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2263: -1.0, Cpd2264: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A8r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2265: -1.0, Cpd2244: -1.0, Cpd2266: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('UGT1A9r')
reaction.name = 'UDP-glucuronosyltransferase 1-10 precursor, microsomal'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2157: -1.0, Cpd2244: -1.0, Cpd2267: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK')
reaction.name = 'UMP kinase'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1398: -1.0, Cpd107: 1.0, Cpd137: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK2')
reaction.name = 'UMP kinase (CTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd653: -1.0, Cpd1398: -1.0, Cpd137: 1.0, Cpd769: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK2n')
reaction.name = 'UMP kinase (CTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd716: -1.0, Cpd2268: -1.0, Cpd774: 1.0, Cpd785: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK3')
reaction.name = 'UMP kinase (UTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd749: -1.0, Cpd1398: -1.0, Cpd137: 2.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK3n')
reaction.name = 'UMP kinase (UTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd784: -1.0, Cpd2268: -1.0, Cpd785: 2.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK4')
reaction.name = 'UMP kinase (GTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd282: -1.0, Cpd1398: -1.0, Cpd137: 1.0, Cpd283: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK4n')
reaction.name = 'UMP kinase (GTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd790: -1.0, Cpd2268: -1.0, Cpd785: 1.0, Cpd791: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK5')
reaction.name = 'UMP kinase (dATP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd1398: -1.0, Cpd137: 1.0, Cpd288: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK5n')
reaction.name = 'UMP kinase (dATP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd782: -1.0, Cpd2268: -1.0, Cpd783: 1.0, Cpd785: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK6')
reaction.name = 'UMP kinase (dCTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd780: -1.0, Cpd1398: -1.0, Cpd137: 1.0, Cpd777: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK6n')
reaction.name = 'UMP kinase (dCTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd781: -1.0, Cpd2268: -1.0, Cpd779: 1.0, Cpd785: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK7')
reaction.name = 'UMP kinase (dGTP)'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd770: -1.0, Cpd1398: -1.0, Cpd137: 1.0, Cpd771: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPK7n')
reaction.name = 'UMP kinase (dGTP),nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd773: -1.0, Cpd2268: -1.0, Cpd775: 1.0, Cpd785: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UMPKn')
reaction.name = 'UMP kinase, nuclear'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd2268: -1.0, Cpd785: 1.0, Cpd789: 1.0})
reaction.gene_reaction_rule = '( CMPK1 )'
model.add_reactions([reaction])
reaction = Reaction('UPP3S')
reaction.name = 'uroporphyrinogen-III synthase'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1486: -1.0, Cpd2: 1.0, Cpd2269: 1.0})
reaction.gene_reaction_rule = '( UROS )'
model.add_reactions([reaction])
reaction = Reaction('UPPDC1')
reaction.name = 'uroporphyrinogen decarboxylase (uroporphyrinogen III)'
reaction.subsystem = 'Heme synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -4.0, Cpd2269: -1.0, Cpd60: 4.0, Cpd733: 1.0})
reaction.gene_reaction_rule = '( UROD )'
model.add_reactions([reaction])
reaction = Reaction('UPPN')
reaction.name = 'b-ureidopropionase'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -2.0, Cpd867: -1.0, Cpd6: 1.0, Cpd60: 1.0, Cpd483: 1.0})
reaction.gene_reaction_rule = '( UPB1 )'
model.add_reactions([reaction])
reaction = Reaction('URAt')
reaction.name = 'uracil transport via facilated diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2270: -1.0, Cpd951: 1.0})
reaction.gene_reaction_rule = '( SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('URCN')
reaction.name = 'URCN'
reaction.subsystem = 'Histidine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1480: -1.0, Cpd1566: 1.0})
reaction.gene_reaction_rule = '( UROC1 )'
model.add_reactions([reaction])
reaction = Reaction('UREAt')
reaction.name = 'Urea transport via facilitate diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2271: -1.0, Cpd469: 1.0})
reaction.gene_reaction_rule = '( SLC14A1 or SLC14A2 or SLC5A1 or SLC5A5 )'
model.add_reactions([reaction])
reaction = Reaction('UREAt5')
reaction.name = 'urea, water cotransport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2271: -1.0, Cpd2: 1.0, Cpd469: 1.0})
reaction.gene_reaction_rule = '( SLC5A1 )'
model.add_reactions([reaction])
reaction = Reaction('UREAtm')
reaction.name = 'Urea transport via diffusion'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd469: -1.0, Cpd350: 1.0})
reaction.gene_reaction_rule = '( AQP9 )'
model.add_reactions([reaction])
reaction = Reaction('URIDK2m')
reaction.name = 'uridylate kinase (dUMP), mitochondrial'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd954: -1.0, Cpd202: 1.0, Cpd901: 1.0})
reaction.gene_reaction_rule = '( DTYMK )'
model.add_reactions([reaction])
reaction = Reaction('URIK1')
reaction.name = 'uridine kinase (ATP:Uridine)'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd761: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( UCK1 or UCK2 or UCKL1 )'
model.add_reactions([reaction])
reaction = Reaction('URIt')
reaction.name = 'uridine facilated transport in cytosol'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1821: -1.0, Cpd761: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 or SLC29A2 )'
model.add_reactions([reaction])
reaction = Reaction('URIt4')
reaction.name = 'uridine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1821: -1.0, Cpd41: 1.0, Cpd761: 1.0})
reaction.gene_reaction_rule = '( SLC28A1 or SLC28A2 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('Uritl')
reaction.name = 'uridine facilated transport from lysosome'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1823: -1.0, Cpd761: 1.0})
reaction.gene_reaction_rule = '( SLC29A3 )'
model.add_reactions([reaction])
reaction = Reaction('Uritm')
reaction.name = 'uridine facilated transport in mitochondria'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd761: -1.0, Cpd1825: 1.0})
reaction.gene_reaction_rule = '( SLC29A1 )'
model.add_reactions([reaction])
reaction = Reaction('VACCCPT1')
reaction.name = 'carnitine O-palmitoyltransferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd825: -1.0, Cpd78: 1.0, Cpd2272: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('VACCCPT2')
reaction.name = 'carnitine transferase'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2273: -1.0, Cpd307: 1.0, Cpd1068: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('VACCCRNt')
reaction.name = 'transport into the mitochondria (carnitine)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2272: -1.0, Cpd2273: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('VALt4')
reaction.name = 'L-valine transport in via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( ACE2 or SLC6A14 or SLC6A19 or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('VALTA')
reaction.name = 'valine transaminase'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd2275: -1.0, Cpd74: 1.0, Cpd2276: 1.0})
reaction.gene_reaction_rule = '( BCAT1 )'
model.add_reactions([reaction])
reaction = Reaction('VALTAm')
reaction.name = 'valine transaminase, mitochondiral'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd2277: -1.0, Cpd63: 1.0, Cpd1844: 1.0})
reaction.gene_reaction_rule = '( BCAT2 )'
model.add_reactions([reaction])
reaction = Reaction('VALtec')
reaction.name = 'L-valine transport via diffusion (extracellular to cytosol)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2274: -1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 or SLC43A2 )'
model.add_reactions([reaction])
reaction = Reaction('VITD2Hm')
reaction.name = 'Vitamin D-25-hydroxylase (D2)'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd2278: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd10: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('VITD3Hm')
reaction.name = 'Vitamin D-25-hydroxylase (D3)'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd2279: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd14: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('VITD3t2')
reaction.name = 'Vitamin D3 uptake'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2280: -1.0, Cpd2281: 1.0})
reaction.gene_reaction_rule = '( CD36 or GOT2 )'
model.add_reactions([reaction])
reaction = Reaction('VLCS2p')
reaction.name = 'Very-long-chain-fatty-acid-CoA ligase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd2282: -1.0, Cpd433: 1.0, Cpd2283: 1.0, Cpd885: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 or SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('VLCS2r')
reaction.name = 'Very-long-chain-fatty-acid-CoA ligase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2284: -1.0, Cpd434: 1.0, Cpd2285: 1.0, Cpd1988: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 or SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('VLCSp')
reaction.name = 'Very-long-chain-fatty-acid-CoA ligase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd2286: -1.0, Cpd438: 1.0, Cpd2283: 1.0, Cpd885: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('VLCSr')
reaction.name = 'Very-long-chain-fatty-acid-CoA ligase'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2287: -1.0, Cpd439: 1.0, Cpd2285: 1.0, Cpd1988: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('XANDp')
reaction.name = 'xanthine dehydrogenase, peroxisomal'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd261: -1.0, Cpd2288: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd2289: 1.0})
reaction.gene_reaction_rule = '( XDH )'
model.add_reactions([reaction])
reaction = Reaction('XAO2x')
reaction.name = 'xanthine oxidase'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2290: -1.0, Cpd247: 1.0, Cpd2288: 1.0})
reaction.gene_reaction_rule = '( XDH )'
model.add_reactions([reaction])
reaction = Reaction('XAOx')
reaction.name = 'xanthine  oxidase,peroxisomal'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2288: -1.0, Cpd247: 1.0, Cpd2289: 1.0})
reaction.gene_reaction_rule = '( XDH )'
model.add_reactions([reaction])
reaction = Reaction('XYLK')
reaction.name = 'xylulokinase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1567: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd2054: 1.0})
reaction.gene_reaction_rule = '( XYLB )'
model.add_reactions([reaction])
reaction = Reaction('XYLt')
reaction.name = 'D-xylose reversible transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2291: -1.0, Cpd2292: 1.0})
reaction.gene_reaction_rule = '( SLC2A3 )'
model.add_reactions([reaction])
reaction = Reaction('XYLTer')
reaction.name = 'UDP-xylose:protein xylosyltransferase, endoplasmic reticulum'
reaction.subsystem = 'Chondroitin synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2293: -1.0, Cpd2294: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2295: 1.0})
reaction.gene_reaction_rule = '( XYLT1 or XYLT2 )'
model.add_reactions([reaction])
reaction = Reaction('XYLUR')
reaction.name = 'xylulose reductase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2296: -1.0, Cpd25: 1.0, Cpd2297: 1.0})
reaction.gene_reaction_rule = '( DCXR )'
model.add_reactions([reaction])
reaction = Reaction('r0009')
reaction.name = 'Pyrophosphate phosphohydrolase EC:3.6.1.1'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd885: -1.0, Cpd358: 1.0, Cpd944: 2.0})
reaction.gene_reaction_rule = '( LHPP or PPA1 or PPA2 )'
model.add_reactions([reaction])
reaction = Reaction('r0010')
reaction.name = 'hydrogen-peroxide:hydrogen-peroxide oxidoreductase EC:1.11.1.6'
reaction.subsystem = 'ROS detoxification'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -2.0, Cpd2: 2.0, Cpd3: 1.0})
reaction.gene_reaction_rule = '( CAT )'
model.add_reactions([reaction])
reaction = Reaction('r0013')
reaction.name = 'beta-N-acetylhexosaminidase Aminosugars metabolism EC:3.2.1.52'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2298: -1.0, Cpd373: 2.0})
reaction.gene_reaction_rule = '( HEXA or HEXB or MGEA5 )'
model.add_reactions([reaction])
reaction = Reaction('r0016')
reaction.name = 'Fe(II):oxygen oxidoreductase Porphyrin and chlorophyll metabolism EC:1.16.3.1'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -4.0, Cpd1114: -4.0, Cpd2143: -1.0, Cpd114: 2.0, Cpd1113: 4.0})
reaction.gene_reaction_rule = '( CP or FTH1 or FTL1 or FTMT )'
model.add_reactions([reaction])
reaction = Reaction('r0021')
reaction.name = 'glutathione:NAD+ oxidoreductase Glutamate metabolism EC:1.8.1.7'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1057: -2.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1441: 1.0})
reaction.gene_reaction_rule = '( GSR )'
model.add_reactions([reaction])
reaction = Reaction('r0022')
reaction.name = 'glutathione:NAD+ oxidoreductase Glutamate metabolism EC:1.8.1.7'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1418: -2.0, Cpd7: 1.0, Cpd47: 1.0, Cpd1442: 1.0})
reaction.gene_reaction_rule = '( GSR )'
model.add_reactions([reaction])
reaction = Reaction('r0023')
reaction.name = 'NADH:ferricytochrome-b5 oxidoreductase Aminosugars metabolism EC:1.6.2.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd35: -1.0, Cpd2299: -2.0, Cpd32: 1.0, Cpd2300: 2.0})
reaction.gene_reaction_rule = '( CYB5R1 or CYB5R2 or CYB5R3 )'
model.add_reactions([reaction])
reaction = Reaction('r0024')
reaction.name = 'N-(omega)-Hydroxyarginine,NADPH:oxygen oxidoreductase (nitric-oxide-forming) Arginine and proline metabolism EC:1.14.13.39'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd1806: -2.0, Cpd471: 2.0, Cpd1807: 2.0})
reaction.gene_reaction_rule = '( NOS1 or NOS2 or NOS3 )'
model.add_reactions([reaction])
reaction = Reaction('r0027')
reaction.name = 'NADPH:CoA-glutathione oxidoreductase EC:1.8.1.9'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd396: -2.0, Cpd33: 1.0, Cpd22: 1.0, Cpd758: 1.0})
reaction.gene_reaction_rule = '( TXNRD1 or TXNRD2 )'
model.add_reactions([reaction])
reaction = Reaction('r0028')
reaction.name = 'ATP pyrophosphohydrolase Purine metabolism EC:3.6.1.5'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd107: -1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd150: 1.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('r0033')
reaction.name = 'ATP:dephospho-CoA 3-phosphotransferase Pantothenate and CoA biosynthesis EC:2.7.1.24'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd2301: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd202: 1.0})
reaction.gene_reaction_rule = '( COASY )'
model.add_reactions([reaction])
reaction = Reaction('r0034')
reaction.name = 'Carbon-dioxide:ammonia ligase (ADP-forming,carbamate-phosphorylating) Urea cycle and metabolism of amino groups / Nitrogen metabolism EC:6.3.4.16'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd45: -1.0, Cpd200: -2.0, Cpd482: -1.0, Cpd7: 3.0, Cpd202: 2.0, Cpd204: 1.0, Cpd644: 1.0})
reaction.gene_reaction_rule = '( CPS1 )'
model.add_reactions([reaction])
reaction = Reaction('r0047')
reaction.name = 'Adenosine 5-monophosphate phosphohydrolase Purine metabolism EC:3.1.3.5'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd209: -1.0, Cpd204: 1.0, Cpd291: 1.0})
reaction.gene_reaction_rule = '( NT5C or NT5C1A or NT5C1B or NT5C2 or NT5C3 or NT5E or NT5M )'
model.add_reactions([reaction])
reaction = Reaction('r0051')
reaction.name = 'AMP:pyrophosphate phosphoribosyltransferase Purine metabolism EC:2.4.2.7'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd493: -1.0, Cpd2302: -1.0, Cpd279: 1.0, Cpd2303: 1.0})
reaction.gene_reaction_rule = '( APRT or HPRT )'
model.add_reactions([reaction])
reaction = Reaction('r0060')
reaction.name = 'L-serine ammonia-lyase Glycine, serine and threonine metabolism EC:4.3.1.17'
reaction.subsystem = 'Cysteine Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd6: 1.0, Cpd21: 1.0})
reaction.gene_reaction_rule = '( SDS )'
model.add_reactions([reaction])
reaction = Reaction('r0062')
reaction.name = 'Acetyl-CoA hydrolase Pyruvate metabolism EC:3.1.2.1'
reaction.subsystem = 'Pyruvate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd178: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd267: 1.0})
reaction.gene_reaction_rule = '( ACOT12 )'
model.add_reactions([reaction])
reaction = Reaction('r0068')
reaction.name = 'Acetyl adenylate:CoA ligase (AMP-forming) Pyruvate metabolism EC:6.2.1.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd78: -1.0, Cpd2304: -1.0, Cpd33: 1.0, Cpd150: 1.0, Cpd180: 1.0})
reaction.gene_reaction_rule = '( ACSS1 or ACSS2 )'
model.add_reactions([reaction])
reaction = Reaction('r0074')
reaction.name = 'L-Glutamate 5-semialdehyde:NAD+ oxidoreductase Arginine and proline metabolism EC:1.5.1.12'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd1200: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd63: 1.0})
reaction.gene_reaction_rule = '( ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0081')
reaction.name = 'L-Alanine:2-oxoglutarate aminotransferase Glutamate metabolism / Alanine and aspartate metabolism EC:2.6.1.2'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd361: -1.0, Cpd63: 1.0, Cpd363: 1.0})
reaction.gene_reaction_rule = '( GPT or GPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0082')
reaction.name = 'Oxalosuccinate:NADP+ oxidoreductase (decarboxylating) Citrate cycle (TCA cycle) EC:1.1.1.42'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd2305: -1.0, Cpd30: 1.0, Cpd60: 1.0})
reaction.gene_reaction_rule = '( IDH1 )'
model.add_reactions([reaction])
reaction = Reaction('r0083')
reaction.name = 'Oxalosuccinate:NADP+ oxidoreductase (decarboxylating) Citrate cycle (TCA cycle) EC:1.1.1.42'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd2306: -1.0, Cpd28: 1.0, Cpd45: 1.0})
reaction.gene_reaction_rule = '( IDH1 or IDH2 or IDH3A or IDH3B or IDH3G )'
model.add_reactions([reaction])
reaction = Reaction('r0084')
reaction.name = 'Oxalosuccinate:NADP+ oxidoreductase (decarboxylating) Citrate cycle (TCA cycle) EC:1.1.1.42'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd2307: -1.0, Cpd1538: 1.0, Cpd943: 1.0})
reaction.gene_reaction_rule = '( IDH1 or IDH2 )'
model.add_reactions([reaction])
reaction = Reaction('r0085')
reaction.name = '2-Oxoglutaramate amidohydrolase Glutamate metabolism EC:3.5.1.3'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2308: -1.0, Cpd6: 1.0, Cpd30: 1.0})
reaction.gene_reaction_rule = '( NIT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0086')
reaction.name = '2-Oxoglutaramate amidohydrolase Glutamate metabolism EC:3.5.1.3'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2309: -1.0, Cpd28: 1.0, Cpd482: 1.0})
reaction.gene_reaction_rule = '( NIT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0093')
reaction.name = 'UDPglucose pyrophosphohydrolase Pyrimidine metabolism / Starch and sucrose metabolism EC:3.6.1.45'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2310: -1.0, Cpd37: 2.0, Cpd1772: 1.0, Cpd2311: 1.0})
reaction.gene_reaction_rule = '( ENPP1 or ENPP3 )'
model.add_reactions([reaction])
reaction = Reaction('r0097')
reaction.name = 'Acetate:CoA ligase (AMP-forming) Pyruvate metabolism EC:6.2.1.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd221: -1.0, Cpd151: 1.0, Cpd2304: 1.0})
reaction.gene_reaction_rule = '( ACSS1 or ACSS2 )'
model.add_reactions([reaction])
reaction = Reaction('r0113')
reaction.name = 'UDP-N-acetyl-D-glucosamine 2-epimerase Aminosugars metabolism EC:5.1.3.14'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2312: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2313: 1.0})
reaction.gene_reaction_rule = '( GNE )'
model.add_reactions([reaction])
reaction = Reaction('r0119')
reaction.name = 'Guanosine 5-triphosphate pyrophosphohydrolase Purine metabolism EC:3.6.1.19'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd282: -1.0, Cpd33: 1.0, Cpd151: 1.0, Cpd1307: 1.0})
reaction.gene_reaction_rule = '( ITPA )'
model.add_reactions([reaction])
reaction = Reaction('r0120')
reaction.name = 'GTP 7,8-8,9-dihydrolase Folate biosynthesis EC:3.5.4.16'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -4.0, Cpd282: -1.0, Cpd2314: 1.0})
reaction.gene_reaction_rule = '( GCH1 )'
model.add_reactions([reaction])
reaction = Reaction('r0121')
reaction.name = 'r0121'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd763: -4.0, Cpd790: -1.0, Cpd2315: 1.0})
reaction.gene_reaction_rule = '( GCH1 )'
model.add_reactions([reaction])
reaction = Reaction('r0122')
reaction.name = 'GTP:pyruvate O2-phosphotransferase Purine metabolism EC:2.7.1.40'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd231: -1.0, Cpd283: -1.0, Cpd21: 1.0, Cpd282: 1.0})
reaction.gene_reaction_rule = '( PKLR )'
model.add_reactions([reaction])
reaction = Reaction('r0127')
reaction.name = 'L-Asparagine amidohydrolase Alanine and aspartate metabolism / Cyanoamino acid metabolism / Nitrogen metabolism EC:3.5.1.1 EC:3.5.1.38'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd393: -1.0, Cpd6: 1.0, Cpd90: 1.0})
reaction.gene_reaction_rule = '( ASPG )'
model.add_reactions([reaction])
reaction = Reaction('r0129')
reaction.name = 'gamma-glutamyltranspeptidase Glutathione metabolism EC:3.4.11.4'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1057: -1.0, Cpd74: 1.0, Cpd443: 1.0})
reaction.gene_reaction_rule = '( GGT1 or GGT5 or GGT6 or GGT7 )'
model.add_reactions([reaction])
reaction = Reaction('r0130')
reaction.name = 'gamma-glutamyltranspeptidase Glutathione metabolism EC:3.4.11.4'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1443: -1.0, Cpd444: 1.0, Cpd759: 1.0})
reaction.gene_reaction_rule = '( GGT1 or GGT5 or GGT6 or GGT7 )'
model.add_reactions([reaction])
reaction = Reaction('r0139')
reaction.name = 'CDP diphosphohydrolase Pyrimidine metabolism EC:3.6.1.5'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2316: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd1828: 1.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('r0142')
reaction.name = 'Sulfite:oxygen oxidoreductase Sulfur metabolism EC:1.8.3.1'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd12: -1.0, Cpd1160: -1.0, Cpd634: 1.0, Cpd1162: 1.0})
reaction.gene_reaction_rule = '( SUOX )'
model.add_reactions([reaction])
reaction = Reaction('r0145')
reaction.name = 'L-Arginine,NADPH:oxygen oxidoreductase (nitric-oxide-forming) Arginine and proline metabolism EC:1.14.13.39'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -2.0, Cpd22: -2.0, Cpd467: -1.0, Cpd2: 2.0, Cpd25: 2.0, Cpd471: 1.0, Cpd1807: 1.0})
reaction.gene_reaction_rule = '( NOS1 or NOS2 or NOS3 )'
model.add_reactions([reaction])
reaction = Reaction('r0149')
reaction.name = 'CTP diphosphohydrolase Pyrimidine metabolism EC:3.6.1.5'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2317: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd2316: 1.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('r0153')
reaction.name = 'CTP:pyruvate O2-phosphotransferase EC:2.7.1.40'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd231: -1.0, Cpd769: -1.0, Cpd21: 1.0, Cpd653: 1.0})
reaction.gene_reaction_rule = '( PKLR )'
model.add_reactions([reaction])
reaction = Reaction('r0160')
reaction.name = 'L-Serine:pyruvate aminotransferase Glycine, serine and threonine metabolism EC:2.6.1.51'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd21: -1.0, Cpd404: -1.0, Cpd394: 1.0, Cpd1500: 1.0})
reaction.gene_reaction_rule = '( AGXT )'
model.add_reactions([reaction])
reaction = Reaction('r0163')
reaction.name = '2-oxoglutarate dehydrogenase E1 component Citrate cycle (TCA cycle) EC:1.2.4.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd28: -1.0, Cpd2318: -1.0, Cpd45: 1.0, Cpd2319: 1.0})
reaction.gene_reaction_rule = '( OGDH )'
model.add_reactions([reaction])
reaction = Reaction('r0165')
reaction.name = 'UTP:pyruvate O2-phosphotransferase EC:2.7.1.40'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd137: -1.0, Cpd231: -1.0, Cpd21: 1.0, Cpd749: 1.0})
reaction.gene_reaction_rule = '( PKLR )'
model.add_reactions([reaction])
reaction = Reaction('r0166')
reaction.name = 'Uridine triphosphate pyrophosphohydrolase Pyrimidine metabolism EC:3.6.1.19'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1775: -1.0, Cpd37: 1.0, Cpd1772: 1.0, Cpd2302: 1.0})
reaction.gene_reaction_rule = '( ITPA )'
model.add_reactions([reaction])
reaction = Reaction('r0170')
reaction.name = 'Farnesyl-diphosphate:farnesyl-diphosphate farnesyltransferase Biosynthesis of steroids EC:2.5.1.21'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2156: -2.0, Cpd1988: 1.0, Cpd2320: 1.0})
reaction.gene_reaction_rule = '( FDFT1 )'
model.add_reactions([reaction])
reaction = Reaction('r0173')
reaction.name = '(S)-Lactate:NAD+ oxidoreductase Glycolysis / Gluconeogenesis / Pyruvate metabolism EC:1.1.1.27'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2321: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd368: 1.0})
reaction.gene_reaction_rule = '( LDHA or LDHB or LDHC )'
model.add_reactions([reaction])
reaction = Reaction('r0178')
reaction.name = 'Succinate-semialdehyde:NAD+ oxidoreductase Glutamate metabolism / Tyrosine metabolism / Butanoate metabolism EC:1.2.1.24 EC:1.2.1.16'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd171: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd1843: 1.0})
reaction.gene_reaction_rule = '( ALDH5A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0181')
reaction.name = 'ITP diphosphohydrolase Purine metabolism EC:3.6.1.5'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2322: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd1765: 1.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('r0186')
reaction.name = 'Ethanolamine-phosphate phospho-lyase (deaminating) EC:4.2.3.2'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd996: -1.0, Cpd6: 1.0, Cpd108: 1.0, Cpd417: 1.0})
reaction.gene_reaction_rule = '( ETNPPL )'
model.add_reactions([reaction])
reaction = Reaction('r0191')
reaction.name = 'UTP:D-fructose-6-phosphate 1-phosphotransferase EC:2.7.1.11'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd749: -1.0, Cpd1108: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd1102: 1.0})
reaction.gene_reaction_rule = '( PFKL )'
model.add_reactions([reaction])
reaction = Reaction('r0193')
reaction.name = 'L-Cysteine L-homocysteine-lyase (deaminating) Cysteine metabolism EC:4.4.1.1'
reaction.subsystem = 'Cysteine Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd396: -1.0, Cpd6: 1.0, Cpd33: 1.0, Cpd21: 1.0, Cpd2323: 1.0})
reaction.gene_reaction_rule = '( CTH )'
model.add_reactions([reaction])
reaction = Reaction('r0196')
reaction.name = 'Succinyl-CoA:glycine C-succinyl-transferase(decarboxylating) EC:2.3.1.37'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd364: -1.0, Cpd380: -1.0, Cpd43: 1.0, Cpd2324: 1.0})
reaction.gene_reaction_rule = '( ALAS1 or ALAS2 )'
model.add_reactions([reaction])
reaction = Reaction('r0202')
reaction.name = 'sn-Glycerol-3-phosphate:NAD+ 2-oxidoreductase Glycerophospholipid metabolism EC:1.1.1.8'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2325: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd846: 1.0})
reaction.gene_reaction_rule = '( GPD1 )'
model.add_reactions([reaction])
reaction = Reaction('r0205')
reaction.name = 'sn-Glycerol-3-phosphate:(acceptor) 2-oxidoreductase Glycerophospholipid metabolism EC:1.1.5.3'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd1198: -1.0, Cpd425: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( GPD2 )'
model.add_reactions([reaction])
reaction = Reaction('r0208')
reaction.name = 'GTP:alpha-D-mannose-1-phosphate guanylyltransferase Fructose and mannose metabolism EC:2.7.7.13'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd282: -1.0, Cpd295: -1.0, Cpd151: 1.0, Cpd548: 1.0})
reaction.gene_reaction_rule = '( GMPPA or GMPPB )'
model.add_reactions([reaction])
reaction = Reaction('r0210')
reaction.name = 'L-Serine hydro-lyase (adding homocysteine) Glycine, serine and threonine metabolism EC:4.2.1.22'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd404: -1.0, Cpd2323: -1.0, Cpd2: 1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( CBS )'
model.add_reactions([reaction])
reaction = Reaction('r0220')
reaction.name = 'Propinol adenylate:CoA ligase (AMP-forming) Propanoate metabolism EC:6.2.1.17 EC:6.2.1.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd78: -1.0, Cpd2326: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd207: 1.0})
reaction.gene_reaction_rule = '( ACSS1 or ACSS2 or ACSS3 )'
model.add_reactions([reaction])
reaction = Reaction('r0221')
reaction.name = 'Propinol adenylate:CoA ligase (AMP-forming) Propanoate metabolism EC:6.2.1.17'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2327: -1.0, Cpd7: 2.0, Cpd179: 1.0, Cpd209: 1.0})
reaction.gene_reaction_rule = '( ACSS1 or ACSS2 or ACSS3 )'
model.add_reactions([reaction])
reaction = Reaction('r0224')
reaction.name = '5,6,7,8-Tetrahydrofolate:NAD+ oxidoreductase One carbon pool by folate EC:1.5.1.3'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd863: -1.0, Cpd32: 1.0, Cpd377: 1.0})
reaction.gene_reaction_rule = '( DHFR )'
model.add_reactions([reaction])
reaction = Reaction('r0226')
reaction.name = '5,6,7,8-Tetrahydrofolate:NADP+ oxidoreductase One carbon pool by folate / Folate biosynthesis EC:1.5.1.3'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd1127: -1.0, Cpd13: 1.0, Cpd1140: 1.0})
reaction.gene_reaction_rule = '( DHFR )'
model.add_reactions([reaction])
reaction = Reaction('r0233')
reaction.name = 'UDPglucose:alpha-D-galactose-1-phosphate uridylyltransferase Galactose metabolism / Nucleotide sugars metabolism EC:2.7.7.12'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1270: -1.0, Cpd2328: -1.0, Cpd135: 1.0, Cpd293: 1.0})
reaction.gene_reaction_rule = '( CBS )'
model.add_reactions([reaction])
reaction = Reaction('r0239')
reaction.name = 'N-Formylanthranilate amidohydrolase Tryptophan metabolism EC:3.5.1.9'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1594: -1.0, Cpd33: 1.0, Cpd699: 1.0, Cpd1570: 1.0})
reaction.gene_reaction_rule = '( AFMID )'
model.add_reactions([reaction])
reaction = Reaction('r0242')
reaction.name = 'Glycerone phosphate phosphohydrolase (alkaline optimum) EC:3.1.3.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2329: -1.0, Cpd494: 1.0, Cpd2330: 1.0})
reaction.gene_reaction_rule = '( ALPL )'
model.add_reactions([reaction])
reaction = Reaction('r0245')
reaction.name = 'Glycerol:NADP+ oxidoreductase Glycerolipid metabolism EC:1.1.1.2 EC:1.1.1.72'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1413: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd1104: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0246')
reaction.name = 'Glycerol:NADP+ oxidoreductase Glycerolipid metabolism EC:1.1.1.72 EC:1.1.1.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd1414: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2331: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 or AKR1B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0249')
reaction.name = 'D-Ribose-5-phosphate ketol-isomerase Pentose phosphate pathway EC:5.3.1.6'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2332: -1.0, Cpd1424: 1.0})
reaction.gene_reaction_rule = '( RPIA )'
model.add_reactions([reaction])
reaction = Reaction('r0276')
reaction.name = 'NADH:guanosine-5-phosphate oxidoreductase(deaminating) Purine metabolism EC:1.7.1.7'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd6: -1.0, Cpd25: -1.0, Cpd316: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd1307: 1.0})
reaction.gene_reaction_rule = '( GMPR or GMPR2 )'
model.add_reactions([reaction])
reaction = Reaction('r0280')
reaction.name = 'dATP:pyruvate O2-phosphotransferase Purine metabolism EC:2.7.1.40'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd231: -1.0, Cpd288: -1.0, Cpd21: 1.0, Cpd287: 1.0})
reaction.gene_reaction_rule = '( PKLR )'
model.add_reactions([reaction])
reaction = Reaction('r0281')
reaction.name = 'Putrescine:oxygen oxidoreductase (deaminating) Urea cycle and metabolism of amino groups EC:1.4.3.6'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2143: -1.0, Cpd2333: -1.0, Cpd278: 1.0, Cpd1444: 1.0, Cpd2334: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('r0283')
reaction.name = 'L-Histidine:beta-alanine ligase (AMP-forming) Alanine and aspartate metabolism / Histidine metabolism / beta-Alanine metabolism EC:6.3.2.11'
reaction.subsystem = 'beta-Alanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd483: -1.0, Cpd1479: -1.0, Cpd33: 1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1764: 1.0})
reaction.gene_reaction_rule = '( CARNS1 )'
model.add_reactions([reaction])
reaction = Reaction('r0287')
reaction.name = 'Acetyl-CoA:acetyl-CoA C-acetyltransferase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:2.3.1.16'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2335: -1.0, Cpd178: 1.0, Cpd241: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or ACAT1 or ACAT3 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r0295')
reaction.name = 'glycine synthase Nitrogen metabolism EC:2.1.2.10'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd45: -1.0, Cpd47: -1.0, Cpd482: -1.0, Cpd1300: -1.0, Cpd44: 1.0, Cpd364: 1.0, Cpd1140: 1.0})
reaction.gene_reaction_rule = '( AMT )'
model.add_reactions([reaction])
reaction = Reaction('r0301')
reaction.name = 'Xanthosine-5-phosphate:ammonia ligase (AMP-forming) Purine metabolism EC:6.3.4.1'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd6: -1.0, Cpd105: -1.0, Cpd1420: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd151: 1.0, Cpd1307: 1.0})
reaction.gene_reaction_rule = '( GMPS )'
model.add_reactions([reaction])
reaction = Reaction('r0308')
reaction.name = 'Palmitoyl-CoA hydrolase Fatty acid elongation in mitochondria EC:3.1.2.22 EC:3.1.2.2'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd184: -1.0, Cpd2336: -1.0, Cpd261: 1.0, Cpd1080: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT4 or ACOT7 or PPT1 or PPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0309')
reaction.name = 'trans-Hexadec-2-enoyl-CoA reductase Fatty acid elongation in mitochondria EC:1.3.1.38'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd593: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd601: 1.0})
reaction.gene_reaction_rule = '( MECR or PECR )'
model.add_reactions([reaction])
reaction = Reaction('r0310')
reaction.name = 'Palmitoyl-CoA:oxygen 2-oxidoreductase Fatty acid metabolism EC:1.3.3.6 EC:1.3.99.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd593: -1.0, Cpd723: -1.0, Cpd601: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACADM or ACADVL or ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r0311')
reaction.name = 'Palmitate:CoA ligase (AMP-forming) Fatty acid metabolism EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd1431: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2337: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r0317')
reaction.name = 'citrate hydro-lyase Citrate cycle (TCA cycle) / Glyoxylate and dicarboxylate metabolism EC:4.2.1.4'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd254: -1.0, Cpd12: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( ACO1 or ACO2 )'
model.add_reactions([reaction])
reaction = Reaction('r0318')
reaction.name = 'Propanoate:CoA ligase (AMP-forming) Propanoate metabolism EC:6.2.1.1 EC:6.2.1.17'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -1.0, Cpd206: -1.0, Cpd151: 1.0, Cpd2326: 1.0})
reaction.gene_reaction_rule = '( ACSS1 or ACSS2 or ACSS3 )'
model.add_reactions([reaction])
reaction = Reaction('r0319')
reaction.name = 'Propanoate:CoA ligase (AMP-forming) Propanoate metabolism EC:6.2.1.1 EC:6.2.1.17'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd200: -1.0, Cpd208: -1.0, Cpd210: 1.0, Cpd2327: 1.0})
reaction.gene_reaction_rule = '( ACSS1 or ACSS2 or ACSS3 )'
model.add_reactions([reaction])
reaction = Reaction('r0321')
reaction.name = 'Acetoacetate:CoA ligase (AMP-forming) Butanoate metabolism EC:6.2.1.16'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd186: -1.0, Cpd200: -1.0, Cpd181: 1.0, Cpd209: 1.0, Cpd210: 1.0})
reaction.gene_reaction_rule = '( AACS )'
model.add_reactions([reaction])
reaction = Reaction('r0339')
reaction.name = 'ATP:(R)-glycerate 3-phosphotransferase Glycine, serine and threonine metabolism EC:2.7.1.31'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1502: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd273: 1.0})
reaction.gene_reaction_rule = '( GLYCTK )'
model.add_reactions([reaction])
reaction = Reaction('r0340')
reaction.name = 'ATP:(R)-glycerate 3-phosphotransferase Glycine, serine and threonine metabolism EC:2.7.1.31'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd2339: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd2340: 1.0})
reaction.gene_reaction_rule = '( GLYCTK )'
model.add_reactions([reaction])
reaction = Reaction('r0345')
reaction.name = 'ATP:AMP phosphotransferase Purine metabolism EC:2.7.4.11'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd286: -1.0, Cpd107: 1.0, Cpd288: 1.0})
reaction.gene_reaction_rule = '( AK1 or AK2 or AK4 or AK5 or AK7 )'
model.add_reactions([reaction])
reaction = Reaction('r0354')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1788: -1.0, Cpd1290: -1.0, Cpd33: 1.0, Cpd1787: 1.0, Cpd1202: 1.0})
reaction.gene_reaction_rule = '( GCK or HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0355')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd1290: -1.0, Cpd33: 1.0, Cpd288: 1.0, Cpd1202: 1.0})
reaction.gene_reaction_rule = '( GCK or HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0357')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1788: -1.0, Cpd1144: -1.0, Cpd33: 1.0, Cpd1787: 1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0358')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd1144: -1.0, Cpd33: 1.0, Cpd288: 1.0, Cpd1108: 1.0})
reaction.gene_reaction_rule = '( HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0360')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd129: -1.0, Cpd1788: -1.0, Cpd33: 1.0, Cpd1787: 1.0, Cpd233: 1.0})
reaction.gene_reaction_rule = '( HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0361')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Fructose and mannose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd129: -1.0, Cpd287: -1.0, Cpd33: 1.0, Cpd233: 1.0, Cpd288: 1.0})
reaction.gene_reaction_rule = '( HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0363')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1788: -1.0, Cpd1272: -1.0, Cpd33: 1.0, Cpd1787: 1.0, Cpd217: 1.0})
reaction.gene_reaction_rule = '( HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0364')
reaction.name = 'Glycolysis / Glyconeogenesis EC:2.7.1.1'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd287: -1.0, Cpd1272: -1.0, Cpd33: 1.0, Cpd217: 1.0, Cpd288: 1.0})
reaction.gene_reaction_rule = '( HK1 or HK2 or HK3 )'
model.add_reactions([reaction])
reaction = Reaction('r0368')
reaction.name = 'CoA:apo-[acyl-carrier-protein] pantetheinephosphotransferase Pantothenate and CoA biosynthesis EC:2.7.8.7'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd78: -1.0, Cpd2341: -1.0, Cpd26: 1.0, Cpd249: 1.0})
reaction.gene_reaction_rule = '( AASDHPPT )'
model.add_reactions([reaction])
reaction = Reaction('r0377')
reaction.name = 'ATP:deoxycitidine 5-phosphotransferase Pyrimidine metabolism EC:2.7.1.74'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd817: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd776: 1.0})
reaction.gene_reaction_rule = '( DCK )'
model.add_reactions([reaction])
reaction = Reaction('r0380')
reaction.name = 'Lactose 6-phosphate galactohydrolase Galactose metabolism EC:3.2.1.23'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2342: -1.0, Cpd37: 2.0, Cpd494: 1.0, Cpd1572: 1.0})
reaction.gene_reaction_rule = '( GLB1 )'
model.add_reactions([reaction])
reaction = Reaction('r0383')
reaction.name = 'pyruvate:[dihydrolipoyllysine-residue acetyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-acetylating) EC:1.2.4.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd363: -1.0, Cpd1297: -1.0, Cpd45: 1.0, Cpd2343: 1.0})
reaction.gene_reaction_rule = '( PDHA1 or PDHA2 or PDHB )'
model.add_reactions([reaction])
reaction = Reaction('r0384')
reaction.name = '2-oxoglutarate:[dihydrolipoyllysine-residue succinyltransferase]-lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-succinylating) EC:1.2.4.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd28: -1.0, Cpd1297: -1.0, Cpd45: 1.0, Cpd2344: 1.0})
reaction.gene_reaction_rule = '( OGDH )'
model.add_reactions([reaction])
reaction = Reaction('r0385')
reaction.name = '3-methyl-2-oxobutanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating) EC:1.2.4.4'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd1844: -1.0, Cpd1297: -1.0, Cpd45: 1.0, Cpd2345: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or TMEM91 )'
model.add_reactions([reaction])
reaction = Reaction('r0386')
reaction.name = '4-methyl-2-oxopentanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating) EC:1.2.4.4'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd1593: -1.0, Cpd1297: -1.0, Cpd45: 1.0, Cpd2346: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or TMEM91 )'
model.add_reactions([reaction])
reaction = Reaction('r0388')
reaction.name = 'Pyridoxamine:oxygen oxidoreductase (deaminating) Vitamin B6 metabolism EC:1.4.3.5'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2033: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd2034: 1.0})
reaction.gene_reaction_rule = '( PNPO )'
model.add_reactions([reaction])
reaction = Reaction('r0389')
reaction.name = 'Pyridoxine:oxygen oxidoreductase (deaminating) Vitamin B6 metabolism EC:1.1.3.12'
reaction.subsystem = 'Vitamin B6 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd2036: -1.0, Cpd5: 1.0, Cpd2034: 1.0})
reaction.gene_reaction_rule = '( PNPO )'
model.add_reactions([reaction])
reaction = Reaction('r0390')
reaction.name = 'Isomaltose 6-alpha-D-glucanohydrolase Starch and sucrose metabolism EC:3.2.1.10'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2347: -1.0, Cpd446: 2.0})
reaction.gene_reaction_rule = '( SIS )'
model.add_reactions([reaction])
reaction = Reaction('r0391')
reaction.name = 'Nicotinate D-ribonucleotide:pyrophosphate phosphoribosyltransferase Nicotinate and nicotinamide metabolism EC:2.4.2.11'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd151: -1.0, Cpd1804: -1.0, Cpd33: 1.0, Cpd301: 1.0, Cpd1808: 1.0})
reaction.gene_reaction_rule = '( NAPRT or QPRT )'
model.add_reactions([reaction])
reaction = Reaction('r0392')
reaction.name = 'D-Glyceraldehyde:NAD+ oxidoreductase Glycerolipid metabolism EC:1.2.1.3'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1104: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd1502: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0393')
reaction.name = 'D-Glyceraldehyde:NAD+ oxidoreductase Glycerolipid metabolism EC:1.2.1.3'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2331: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd2339: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0394')
reaction.name = 'Xanthine:NAD+ oxidoreductase Purine metabolism EC:1.17.1.4'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1535: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1450: 1.0})
reaction.gene_reaction_rule = '( XDH )'
model.add_reactions([reaction])
reaction = Reaction('r0395')
reaction.name = 'Hypoxanthine:oxygen oxidoreductase Purine metabolism EC:1.17.3.2'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1535: -1.0, Cpd5: 1.0, Cpd1450: 1.0})
reaction.gene_reaction_rule = '( XDH )'
model.add_reactions([reaction])
reaction = Reaction('r0398')
reaction.name = 'NADPH:6,7-dihydropteridine oxidoreductase Folate biosynthesis EC:1.5.1.34'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd869: -1.0, Cpd25: 1.0, Cpd870: 1.0})
reaction.gene_reaction_rule = '( QDPR )'
model.add_reactions([reaction])
reaction = Reaction('r0399')
reaction.name = 'L-Phenylalanine,tetrahydrobiopterin:oxygen oxidoreductase (4-hydroxylating) Phenylalanine, tyrosine and tryptophan biosynthesis EC:1.14.16.1'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd870: -1.0, Cpd1940: -1.0, Cpd2: 1.0, Cpd869: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( PAH )'
model.add_reactions([reaction])
reaction = Reaction('r0402')
reaction.name = 'Tetrahydrobiopterin:NADP+ oxidoreductase Folate biosynthesis EC:1.1.1.153'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2348: -1.0, Cpd25: 1.0, Cpd870: 1.0})
reaction.gene_reaction_rule = '( SPR )'
model.add_reactions([reaction])
reaction = Reaction('r0403')
reaction.name = 'L-Tryptophan,tetrahydrobiopterin:oxygen oxidoreductase (5-hydroxylating) Tryptophan metabolism EC:1.14.16.4'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd870: -1.0, Cpd1638: -1.0, Cpd2: 1.0, Cpd115: 1.0, Cpd869: 1.0})
reaction.gene_reaction_rule = '( TPH1 or TPH2 )'
model.add_reactions([reaction])
reaction = Reaction('r0407')
reaction.name = 'Sedoheptulose 1,7-bisphosphate D-glyceraldehyde-3-phosphate-lyase Carbon fixation EC:4.1.2.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2349: -1.0, Cpd425: 1.0, Cpd2188: 1.0})
reaction.gene_reaction_rule = '( ALDOART2 or ALDOB or ALDOC )'
model.add_reactions([reaction])
reaction = Reaction('r0408')
reaction.name = 'ATP:Sedoheptulose 7-phosphate 1-phosphotransferase EC:2.7.1.11'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2187: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd2349: 1.0})
reaction.gene_reaction_rule = '( PFKL )'
model.add_reactions([reaction])
reaction = Reaction('r0409')
reaction.name = 'UTP:Sedoheptulose 7-phosphate 1-phosphotransferase EC:2.7.1.11'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd749: -1.0, Cpd2187: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2349: 1.0})
reaction.gene_reaction_rule = '( PFKL )'
model.add_reactions([reaction])
reaction = Reaction('r0410')
reaction.name = '2-Deoxyguanosine 5-triphosphate pyrophosphohydrolase Purine metabolism EC:3.6.1.19'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2350: -1.0, Cpd37: 1.0, Cpd2351: 1.0, Cpd2302: 1.0})
reaction.gene_reaction_rule = '( ITPA )'
model.add_reactions([reaction])
reaction = Reaction('r0413')
reaction.name = 'dATP:pyruvate O2-phosphotransferase Purine metabolism EC:2.7.1.40'
reaction.subsystem = 'Glycolysis/gluconeogenesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd231: -1.0, Cpd771: -1.0, Cpd21: 1.0, Cpd770: 1.0})
reaction.gene_reaction_rule = '( PKLR )'
model.add_reactions([reaction])
reaction = Reaction('r0422')
reaction.name = 'Isocitrate:NADP+ oxidoreductase (decarboxylating) Citrate cycle (TCA cycle) EC:1.1.1.42'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd253: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2305: 1.0})
reaction.gene_reaction_rule = '( IDH1 or IDH2 )'
model.add_reactions([reaction])
reaction = Reaction('r0423')
reaction.name = 'Isocitrate:NADP+ oxidoreductase (decarboxylating) Citrate cycle (TCA cycle) EC:1.1.1.42'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd255: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2306: 1.0})
reaction.gene_reaction_rule = '( IDH1 or IDH2 )'
model.add_reactions([reaction])
reaction = Reaction('r0424')
reaction.name = 'Isocitrate:NADP+ oxidoreductase (decarboxylating) Citrate cycle (TCA cycle) EC:1.1.1.42'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1517: -1.0, Cpd1537: -1.0, Cpd358: 1.0, Cpd1516: 1.0, Cpd2307: 1.0})
reaction.gene_reaction_rule = '( IDH1 or IDH2 )'
model.add_reactions([reaction])
reaction = Reaction('r0425')
reaction.name = 'EC:1.1.1.42'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd255: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2306: 1.0})
reaction.gene_reaction_rule = '( IDH1 or IDH2 or IDH3A or IDH3B or IDH3G )'
model.add_reactions([reaction])
reaction = Reaction('r0426')
reaction.name = 'isocitrate hydro-lyase Citrate cycle (TCA cycle) / Glyoxylate and dicarboxylate metabolism EC:4.2.1.3'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd255: -1.0, Cpd12: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( ACO1 or ACO2 )'
model.add_reactions([reaction])
reaction = Reaction('r0430')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd744: -1.0, Cpd2352: -1.0, Cpd745: 1.0, Cpd2353: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0431')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd199: -1.0, Cpd303: -1.0, Cpd78: 1.0, Cpd2354: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0432')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd203: -1.0, Cpd307: -1.0, Cpd43: 1.0, Cpd2355: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0433')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd2356: -1.0, Cpd737: 1.0, Cpd1666: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0434')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd744: -1.0, Cpd2337: -1.0, Cpd745: 1.0, Cpd2357: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0437')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd196: -1.0, Cpd737: -1.0, Cpd184: 1.0, Cpd2358: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0438')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd744: -1.0, Cpd2359: -1.0, Cpd745: 1.0, Cpd2360: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0440')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd2361: -1.0, Cpd184: 1.0, Cpd2362: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0441')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd744: -1.0, Cpd2363: -1.0, Cpd745: 1.0, Cpd2364: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0443')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1080: -1.0, Cpd737: -1.0, Cpd184: 1.0, Cpd2365: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0444')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd744: -1.0, Cpd2366: -1.0, Cpd745: 1.0, Cpd2367: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0445')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd2368: -1.0, Cpd78: 1.0, Cpd2369: 1.0})
reaction.gene_reaction_rule = '( IDH1 or IDH2 )'
model.add_reactions([reaction])
reaction = Reaction('r0446')
reaction.name = 'Palmitoyl-CoA:L-carnitine O-palmitoyltransferase Fatty acid metabolism EC:2.3.1.21'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd307: -1.0, Cpd2370: -1.0, Cpd43: 1.0, Cpd2371: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('r0450')
reaction.name = 'L-2-Aminoadipate:2-oxoglutarate aminotransferase Lysine biosynthesis / Lysine degradation EC:2.6.1.39'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd29: -1.0, Cpd42: 1.0, Cpd63: 1.0})
reaction.gene_reaction_rule = '( AADAT )'
model.add_reactions([reaction])
reaction = Reaction('r0451')
reaction.name = '2-Oxoadipate:lipoamde 2-oxidoreductase(decarboxylating and acceptor-succinylating) Lysine degradation EC:1.2.4.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd42: -1.0, Cpd1297: -1.0, Cpd45: 1.0, Cpd2372: 1.0})
reaction.gene_reaction_rule = '( OGDH )'
model.add_reactions([reaction])
reaction = Reaction('r0456')
reaction.name = 'ATP:deoxyguanosine 5-phosphotransferase Purine metabolism EC:2.7.1.113'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd841: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd837: 1.0})
reaction.gene_reaction_rule = '( DGUOK )'
model.add_reactions([reaction])
reaction = Reaction('r0463')
reaction.name = '(S)-3-Hydroxy-3-methylglutaryl-CoA acetoacetyl-CoA-lyase (CoA-acetylating) Synthesis and degradation of ketone bodies / Valine, leucine and isoleucine degradation / Butanoate metabolism EC:2.3.3.10'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd184: -1.0, Cpd1491: -1.0, Cpd182: 1.0, Cpd183: 1.0, Cpd261: 1.0})
reaction.gene_reaction_rule = '( HMGCS1 )'
model.add_reactions([reaction])
reaction = Reaction('r0464')
reaction.name = '4-Aminobutyraldehyde:NAD+ oxidoreductase Urea cycle and metabolism of amino groups EC:1.2.1.3'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd2373: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd170: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0465')
reaction.name = 'L-Histidine:4-aminobutanoate ligase (AMP-forming) Urea cycle and metabolism of amino groups EC:6.3.2.11'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd173: -1.0, Cpd1479: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd151: 1.0, Cpd2374: 1.0})
reaction.gene_reaction_rule = '( CARNS1 )'
model.add_reactions([reaction])
reaction = Reaction('r0470')
reaction.name = '2-Deoxyadenosine 5-diphosphate:oxidized-thioredoxin 2-oxidoreductase Purine metabolism EC:1.17.4.1'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd287: -1.0, Cpd2053: -1.0, Cpd105: 1.0, Cpd2052: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('r0472')
reaction.name = '2-Deoxyadenosine 5-diphosphate:oxidized-thioredoxin 2-oxidoreductase Purine metabolism EC:1.17.4.1'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd770: -1.0, Cpd2053: -1.0, Cpd282: 1.0, Cpd2052: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('r0474')
reaction.name = '2-Deoxyadenosine 5-diphosphate:oxidized-thioredoxin 2-oxidoreductase Purine metabolism EC:1.17.4.1'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd780: -1.0, Cpd2053: -1.0, Cpd653: 1.0, Cpd2052: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('r0475')
reaction.name = '2-Deoxyadenosine 5-diphosphate:oxidized-thioredoxin 2-oxidoreductase Purine metabolism EC:1.17.4.1'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd903: -1.0, Cpd2053: -1.0, Cpd749: 1.0, Cpd2052: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('r0483')
reaction.name = '(R)-3-Amino-2-methylpropanoate:2-oxoglutarate aminotransferase EC:2.6.1.22'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd792: -1.0, Cpd62: 1.0, Cpd63: 1.0})
reaction.gene_reaction_rule = '( ABAT )'
model.add_reactions([reaction])
reaction = Reaction('r0488')
reaction.name = '(R)-Mevalonate:NADP+ oxidoreductase (CoA acylating) Biosynthesis of steroids EC:1.1.1.34'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -2.0, Cpd78: -1.0, Cpd2375: -1.0, Cpd33: 2.0, Cpd22: 2.0, Cpd1489: 1.0})
reaction.gene_reaction_rule = '( HMGCR )'
model.add_reactions([reaction])
reaction = Reaction('r0494')
reaction.name = 'dTDP diphosphohydrolase Pyrimidine metabolism EC:3.6.1.5'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2376: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd2377: 1.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('r0497')
reaction.name = 'dTTP nucleotidohydrolase Pyrimidine metabolism EC:3.6.1.39'
reaction.subsystem = 'Pyrimidine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2378: -1.0, Cpd37: 1.0, Cpd494: 1.0, Cpd2376: 1.0})
reaction.gene_reaction_rule = '( ENTPD1 or ENTPD3 or ENTPD8 )'
model.add_reactions([reaction])
reaction = Reaction('r0502')
reaction.name = 'xanthine:NAD+ oxidoreductase Purine metabolism EC:1.17.1.4'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1450: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd2379: 1.0})
reaction.gene_reaction_rule = '( XDH )'
model.add_reactions([reaction])
reaction = Reaction('r0504')
reaction.name = 'Xanthine:oxygen oxidoreductase Purine metabolism EC:1.17.3.2'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd1450: -1.0, Cpd5: 1.0, Cpd2379: 1.0})
reaction.gene_reaction_rule = '( XDH )'
model.add_reactions([reaction])
reaction = Reaction('r0509')
reaction.name = 'Succinate:ubiquinone oxidoreductase Citrate cycle (TCA cycle) EC:1.3.5.1'
reaction.subsystem = 'Citric acid cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd1843: -1.0, Cpd752: 1.0, Cpd1159: 1.0})
reaction.gene_reaction_rule = '( SDHA or SDHB or SDHC or SDHD )'
model.add_reactions([reaction])
reaction = Reaction('r0510')
reaction.name = 'steroyl-CoA,hydrogen-donor:oxygen oxidoreductase Polyunsaturated fatty acid biosynthesis EC:1.14.19.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd590: -1.0, Cpd2300: -2.0, Cpd2: 2.0, Cpd2299: 2.0, Cpd2368: 1.0})
reaction.gene_reaction_rule = '( SCD4 )'
model.add_reactions([reaction])
reaction = Reaction('r0511')
reaction.name = 'steroyl-CoA,hydrogen-donor:oxygen oxidoreductase Polyunsaturated fatty acid biosynthesis EC:1.14.19.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd602: -1.0, Cpd2300: -2.0, Cpd2: 2.0, Cpd606: 1.0, Cpd2299: 2.0})
reaction.gene_reaction_rule = '( SCD4 )'
model.add_reactions([reaction])
reaction = Reaction('r0512')
reaction.name = 'Dihydrofolate:NAD+ oxidoreductase Folate biosynthesis EC:1.5.1.3'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd863: -1.0, Cpd35: 1.0, Cpd1118: 1.0})
reaction.gene_reaction_rule = '( DHFR )'
model.add_reactions([reaction])
reaction = Reaction('r0514')
reaction.name = 'Dihydrofolate:NADP+ oxidoreductase One carbon pool by folate / Folate biosynthesis EC:1.5.1.3'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd8: -1.0, Cpd2380: -1.0, Cpd13: 1.0, Cpd1127: 1.0})
reaction.gene_reaction_rule = '( DHFR )'
model.add_reactions([reaction])
reaction = Reaction('r0517')
reaction.name = 'Succinyl-CoA:glycine C-succinyl-transferase(decarboxylating) EC:2.3.1.37'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd2324: -1.0, Cpd45: 1.0, Cpd406: 1.0})
reaction.gene_reaction_rule = '( ALAS1 or ALAS2 )'
model.add_reactions([reaction])
reaction = Reaction('r0522')
reaction.name = 'S-Aminomethyldihydrolipoylprotein:(6S)-tetrahydrofolate aminomethyltransferase (ammonia-forming) One carbon pool by folate EC:2.1.2.10'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd1712: -1.0, Cpd7: 1.0, Cpd2381: 1.0})
reaction.gene_reaction_rule = '( AMT )'
model.add_reactions([reaction])
reaction = Reaction('r0523')
reaction.name = '5-Formyltetrahydrofolate cyclo-ligase (ADP-forming) One carbon pool by folate EC:6.3.3.2'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd2381: -1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd1712: 1.0})
reaction.gene_reaction_rule = '( MTHFSL )'
model.add_reactions([reaction])
reaction = Reaction('r0525')
reaction.name = 'N6-(L-1,3-Dicarboxypropyl)-L-lysine:NAD+ oxidoreductase; N6-(L-1,3-Dicarboxypropyl)-L-lysine:NAD+ oxidoreductase (L-glutamate-forming) Lysine degradation EC:1.5.1.9'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2089: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd63: 1.0, Cpd2090: 1.0})
reaction.gene_reaction_rule = '( AASS )'
model.add_reactions([reaction])
reaction = Reaction('r0531')
reaction.name = 'dUTP:cytidine 5-phosphotransferase Pyrimidine metabolism EC:2.7.1.48'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd760: -1.0, Cpd903: -1.0, Cpd33: 1.0, Cpd651: 1.0, Cpd899: 1.0})
reaction.gene_reaction_rule = '( UCK1 or UCK2 )'
model.add_reactions([reaction])
reaction = Reaction('r0537')
reaction.name = 'Sphinganine-1-phosphate pamlmitaldehyde-lyase Sphingolipid metabolism EC:4.1.2.27'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd996: -1.0, Cpd2382: -1.0, Cpd2122: 1.0})
reaction.gene_reaction_rule = '( SGPL1 )'
model.add_reactions([reaction])
reaction = Reaction('r0539')
reaction.name = 'Cysteamine:oxygen oxidoreductase Taurine and hypotaurine metabolism EC:1.13.11.19'
reaction.subsystem = 'Taurine and hypotaurine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd1982: -1.0, Cpd33: 1.0, Cpd93: 1.0})
reaction.gene_reaction_rule = '( ADO )'
model.add_reactions([reaction])
reaction = Reaction('r0541')
reaction.name = 'Glutaryl-CoA:(acceptor) 2,3-oxidoreductase (decarboxylating) Fatty acid metabolism EC:1.3.99.7'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd46: -1.0, Cpd723: -1.0, Cpd45: 1.0, Cpd70: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( GCDH )'
model.add_reactions([reaction])
reaction = Reaction('r0545')
reaction.name = 'Phenylacetaldehyde:NAD+ oxidoreductase Phenylalanine metabolism / Styrene degradation EC:1.2.1.5 EC:1.2.1.39'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd1922: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd2383: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0546')
reaction.name = 'Phenylacetaldehyde:NAD+ oxidoreductase Phenylalanine metabolism / Styrene degradation EC:1.2.1.5 EC:1.2.1.39'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2384: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd2385: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0547')
reaction.name = 'Aldehyde:NADP+ oxidoreductase Phenylalanine metabolism EC:1.2.1.5'
reaction.subsystem = 'Phenylalanine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd25: -1.0, Cpd1922: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd2383: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0548')
reaction.name = 'Aldehyde:NADP+ oxidoreductase Phenylalanine metabolism EC:1.2.1.5'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd2384: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd2385: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0549')
reaction.name = '4-aminobutanal:NAD+ 1-oxidoreductase; 4-aminobutyraldehyde:NAD+ oxidoreductase Urea cycle and metabolism of amino groups / beta-Alanine metabolism EC:1.2.1.3 EC:1.2.1.19'
reaction.subsystem = 'Glutamate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2373: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd170: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0552')
reaction.name = 'p-cumic alcohol:NAD+ oxidoreductase Glycine, serine and threonine metabolism EC:1.2.1.8'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd2386: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd552: 1.0})
reaction.gene_reaction_rule = '( ALDH7A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0553')
reaction.name = 'p-Cumic alcohol:NADP+ oxidoreductase Glycine, serine and threonine metabolism EC:1.2.1.8'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd25: -1.0, Cpd2386: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd552: 1.0})
reaction.gene_reaction_rule = '( ALDH7A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0555')
reaction.name = 'acetyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-acetyltransferase Glycolysis / Gluconeogenesis / Alanine and aspartate metabolism / Pyruvate metabolism EC:2.3.1.12'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2343: -1.0, Cpd178: 1.0, Cpd1299: 1.0})
reaction.gene_reaction_rule = '( DLAT )'
model.add_reactions([reaction])
reaction = Reaction('r0556')
reaction.name = 'succinyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-succinyltransferase Citrate cycle (TCA cycle) EC:2.3.1.61'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd380: -1.0, Cpd1299: -1.0, Cpd43: 1.0, Cpd2344: 1.0})
reaction.gene_reaction_rule = '( DLST )'
model.add_reactions([reaction])
reaction = Reaction('r0557')
reaction.name = 'Glutaryl-CoA:dihydrolipoamide S-succinyltransferase Lysine degradation EC:2.3.1.61'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2372: -1.0, Cpd46: 1.0, Cpd1299: 1.0})
reaction.gene_reaction_rule = '( DLST )'
model.add_reactions([reaction])
reaction = Reaction('r0559')
reaction.name = 'aldehyde oxidase Tyrosine metabolism EC:1.2.3.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd33: -1.0, Cpd2387: -1.0, Cpd2: 1.0, Cpd3: 1.0, Cpd2388: 1.0})
reaction.gene_reaction_rule = '( AOX1 )'
model.add_reactions([reaction])
reaction = Reaction('r0560')
reaction.name = '2-Methylpropanoyl-CoA:oxygen 2,3-oxidoreductase Valine, leucine and isoleucine degradation EC:1.3.99.2'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd244: -1.0, Cpd723: -1.0, Cpd245: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( ACADS )'
model.add_reactions([reaction])
reaction = Reaction('r0561')
reaction.name = '2-methylpropanoyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-(2-methylpropanoyl)transferase Valine, leucine and isoleucine degradation EC:2.3.1.168'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2345: -1.0, Cpd244: 1.0, Cpd1299: 1.0})
reaction.gene_reaction_rule = '( DBT )'
model.add_reactions([reaction])
reaction = Reaction('r0568')
reaction.name = '(5-L-Glutamyl)-L-amino-acid 5-glutamyltransferase (cyclizing) Glutathione metabolism EC:2.3.2.4'
reaction.subsystem = 'Glutathione metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1404: -1.0, Cpd396: 1.0, Cpd1335: 1.0})
reaction.gene_reaction_rule = '( GGCT )'
model.add_reactions([reaction])
reaction = Reaction('r0570')
reaction.name = '2-Deoxy-D-ribose 1-phosphate 1,5-phosphomutase Pentose phosphate pathway EC:5.4.2.7'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd955: -1.0, Cpd948: 1.0})
reaction.gene_reaction_rule = '( PGM1 )'
model.add_reactions([reaction])
reaction = Reaction('r0573')
reaction.name = 'Cholate:CoA ligase (AMP-forming) Bile acid biosynthesis EC:6.2.1.7'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2389: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2390: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('r0575')
reaction.name = 'Presqualene diphosphate:farnesyl-diphosphate farnesyltransferase Biosynthesis of steroids EC:2.5.1.21'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd2320: -1.0, Cpd589: 1.0, Cpd1988: 1.0, Cpd2155: 1.0})
reaction.gene_reaction_rule = '( FDFT1 )'
model.add_reactions([reaction])
reaction = Reaction('r0578')
reaction.name = 'ATP:pantothenate 4-phosphotransferase Pantothenate and CoA biosynthesis EC:2.7.1.33'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1981: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1991: 1.0})
reaction.gene_reaction_rule = '( PANK1 or PANK2 or PANK3 or PANK4 )'
model.add_reactions([reaction])
reaction = Reaction('r0579')
reaction.name = 'ATP:pantothenate 4-phosphotransferase Pantothenate and CoA biosynthesis EC:2.7.1.33'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd2391: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd2392: 1.0})
reaction.gene_reaction_rule = '( PANK1 or PANK2 or PANK3 or PANK4 )'
model.add_reactions([reaction])
reaction = Reaction('r0584')
reaction.name = 'Deamino-NAD+ nucleotidohydrolase Nicotinate and nicotinamide metabolism EC:3.6.1.9'
reaction.subsystem = 'NAD metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1756: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd1804: 1.0})
reaction.gene_reaction_rule = '( ENPP1 or ENPP3 or NUDT12 )'
model.add_reactions([reaction])
reaction = Reaction('r0587')
reaction.name = '4-Nitrophenyl phosphate phosphohydrolase gamma-Hexachlorocyclohexane degradation EC:3.1.3.2 EC:3.1.3.41 EC:3.1.3.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2393: -1.0, Cpd2394: 1.0, Cpd494: 1.0})
reaction.gene_reaction_rule = '( ALPL )'
model.add_reactions([reaction])
reaction = Reaction('r0590')
reaction.name = 'ATP:pantetheine-4-phosphate adenylyltransferase Pantothenate and CoA biosynthesis EC:2.7.7.3'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd200: -1.0, Cpd2392: -1.0, Cpd210: 1.0, Cpd2301: 1.0})
reaction.gene_reaction_rule = '( COASY )'
model.add_reactions([reaction])
reaction = Reaction('r0591')
reaction.name = 'Dephospho-CoA nucleotidohydrolase Pantothenate and CoA biosynthesis EC:3.6.1.9'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd938: -1.0, Cpd33: 2.0, Cpd150: 1.0, Cpd1991: 1.0})
reaction.gene_reaction_rule = '( ENPP1 or ENPP3 )'
model.add_reactions([reaction])
reaction = Reaction('r0594')
reaction.name = 'L-2-Aminoadipate-6-semialdehyde:NAD+ 6-oxidoreductase Lysine degradation EC:1.2.1.31'
reaction.subsystem = 'Lysine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd2395: -1.0, Cpd33: 2.0, Cpd27: 1.0, Cpd35: 1.0})
reaction.gene_reaction_rule = '( ALDH7A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0595')
reaction.name = '3-Mercaptopyruvate:cyanide sulfurtransferase Cysteine metabolism EC:2.8.1.2'
reaction.subsystem = 'Cysteine Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1161: -1.0, Cpd753: -1.0, Cpd21: 1.0, Cpd2396: 1.0})
reaction.gene_reaction_rule = '( MPST or TST )'
model.add_reactions([reaction])
reaction = Reaction('r0596')
reaction.name = '3-hydroxyisobutyryl-CoA hydrolase beta-Alanine metabolism / Propanoate metabolism EC:3.1.2.4'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2015: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2397: 1.0})
reaction.gene_reaction_rule = '( HIBCH )'
model.add_reactions([reaction])
reaction = Reaction('r0603')
reaction.name = '(S)-2-methylbutanoyl-CoA:acceptor 2,3-oxidoreductase Valine, leucine and isoleucine degradation EC:1.3.99.12'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd237: -1.0, Cpd723: -1.0, Cpd239: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACADS or ACADSB )'
model.add_reactions([reaction])
reaction = Reaction('r0604')
reaction.name = '(S)-2-methylbutanoyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-(2-methylbutanoyl)transferase Valine, leucine and isoleucine degradation EC:2.3.1.168'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2398: -1.0, Cpd237: 1.0, Cpd1299: 1.0})
reaction.gene_reaction_rule = '( DBT )'
model.add_reactions([reaction])
reaction = Reaction('r0610')
reaction.name = 'CTP:D-Tagatose 6-phosphate 1-phosphotransferase Galactose metabolism EC:2.7.1.11'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd653: -1.0, Cpd2399: -1.0, Cpd33: 1.0, Cpd769: 1.0, Cpd2400: 1.0})
reaction.gene_reaction_rule = '( PFKL )'
model.add_reactions([reaction])
reaction = Reaction('r0611')
reaction.name = 'ITP:D-Tagatose 6-phosphate 1-phosphotransferase Galactose metabolism EC:2.7.1.11'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1788: -1.0, Cpd2399: -1.0, Cpd33: 1.0, Cpd1787: 1.0, Cpd2400: 1.0})
reaction.gene_reaction_rule = '( PFKL )'
model.add_reactions([reaction])
reaction = Reaction('r0614')
reaction.name = 'N-[(R)-4-Phosphopantothenoyl]-L-cysteine carboxy-lyase Pantothenate and CoA biosynthesis EC:4.1.1.36'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd2401: -1.0, Cpd45: 1.0, Cpd2392: 1.0})
reaction.gene_reaction_rule = '( PPCDC )'
model.add_reactions([reaction])
reaction = Reaction('r0615')
reaction.name = 'trans-4-Hydroxy-L-proline:NAD+ 5-oxidoreductase Arginine and proline metabolism EC:1.5.1.2'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2402: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd2403: 1.0})
reaction.gene_reaction_rule = '( PYCR1 or PYCR2 )'
model.add_reactions([reaction])
reaction = Reaction('r0616')
reaction.name = 'trans-4-Hydroxy-L-proline:NAD+ 5-oxidoreductase Arginine and proline metabolism EC:1.5.1.2'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2404: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd1937: 1.0})
reaction.gene_reaction_rule = '( PYCR1 or PYCR2 )'
model.add_reactions([reaction])
reaction = Reaction('r0617')
reaction.name = 'trans-4-Hydroxy-L-proline:NADP+ 5-oxidoreductase Arginine and proline metabolism EC:1.5.1.2'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2402: -1.0, Cpd33: 2.0, Cpd22: 1.0, Cpd2403: 1.0})
reaction.gene_reaction_rule = '( PYCR1 or PYCR2 )'
model.add_reactions([reaction])
reaction = Reaction('r0618')
reaction.name = 'trans-4-Hydroxy-L-proline:NADP+ 5-oxidoreductase Arginine and proline metabolism EC:1.5.1.2'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd2404: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd1937: 1.0})
reaction.gene_reaction_rule = '( PYCR1 or PYCR2 )'
model.add_reactions([reaction])
reaction = Reaction('r0620')
reaction.name = '2-oxoglutarate dehydrogenase E1 component Citrate cycle (TCA cycle) EC:1.2.4.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1297: -1.0, Cpd2319: -1.0, Cpd2318: 1.0, Cpd2344: 1.0})
reaction.gene_reaction_rule = '( OGDH )'
model.add_reactions([reaction])
reaction = Reaction('r0627')
reaction.name = 'Galactosylglycerol galactohydrolase Galactose metabolism EC:3.2.1.22 EC:3.2.1.23'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2405: -1.0, Cpd1266: 1.0, Cpd1627: 1.0})
reaction.gene_reaction_rule = '( GLA )'
model.add_reactions([reaction])
reaction = Reaction('r0629')
reaction.name = 'Choloyl-CoA:glycine N-choloyltransferase Bile acid biosynthesis / Taurine and hypotaurine metabolism EC:2.3.1.65'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd2406: -1.0, Cpd78: 1.0, Cpd1305: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('r0630')
reaction.name = 'Choloyl-CoA:glycine N-choloyltransferase Bile acid biosynthesis / Taurine and hypotaurine metabolism EC:2.3.1.65'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1586: -1.0, Cpd2406: -1.0, Cpd78: 1.0, Cpd2192: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('r0633')
reaction.name = 'trans-Oct-2-enoyl-CoA reductase Fatty acid elongation in mitochondria EC:1.3.1.38'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd1063: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2407: 1.0})
reaction.gene_reaction_rule = '( MECR or PECR )'
model.add_reactions([reaction])
reaction = Reaction('r0634')
reaction.name = 'Octanoyl-CoA:acetyl-CoA C-acyltransferase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:2.3.1.16'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd1063: -1.0, Cpd43: 1.0, Cpd2408: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r0636')
reaction.name = 'Octanoyl-CoA:L-carnitine O-octanoyltransferase EC:2.3.1.137'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2409: -1.0, Cpd307: 1.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('r0637')
reaction.name = 'Octanoyl-CoA:L-carnitine O-octanoyltransferase EC:2.3.1.137'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd188: -1.0, Cpd737: -1.0, Cpd184: 1.0, Cpd2410: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('r0638')
reaction.name = 'trans-Dodec-2-enoyl-CoA reductase Fatty acid elongation in mitochondria EC:1.3.1.38'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd2411: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd807: 1.0})
reaction.gene_reaction_rule = '( MECR or PECR )'
model.add_reactions([reaction])
reaction = Reaction('r0639')
reaction.name = 'Lauroyl-CoA:acetyl-CoA C-acyltransferase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:2.3.1.16'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2412: -1.0, Cpd178: 1.0, Cpd2411: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r0641')
reaction.name = '(5-Glutamyl)-peptide:amino-acid 5-glutamyltransferase Arachidonic acid metabolism EC:2.3.2.2'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1588: -1.0, Cpd74: 1.0, Cpd2413: 1.0})
reaction.gene_reaction_rule = '( GGT1 or GGT5 or GGT6 or GGT7 )'
model.add_reactions([reaction])
reaction = Reaction('r0642')
reaction.name = '(S)-Methylmalonate semialdehyde:NAD+ oxidoreductase Valine, leucine and isoleucine degradation EC:1.2.1.3'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd2414: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd2415: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0643')
reaction.name = '(S)-Methylmalonate semialdehyde:NAD+ oxidoreductase Valine, leucine and isoleucine degradation EC:1.2.1.3'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd62: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd2416: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0644')
reaction.name = '(S)-Methylmalonate semialdehyde:oxygen oxidoreductase Valine, leucine and isoleucine degradation EC:1.2.3.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd33: -1.0, Cpd2415: -1.0, Cpd2: 1.0, Cpd3: 1.0, Cpd2414: 1.0})
reaction.gene_reaction_rule = '( AOX1 )'
model.add_reactions([reaction])
reaction = Reaction('r0648')
reaction.name = '(5-Glutamyl)-peptide:amino-acid 5-glutamyltransferase Cyanoamino acid metabolism EC:2.3.2.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd759: -1.0, Cpd2417: -1.0, Cpd114: 1.0, Cpd2418: 1.0})
reaction.gene_reaction_rule = '( GGT1 or GGT5 or GGT6 or GGT7 )'
model.add_reactions([reaction])
reaction = Reaction('r0649')
reaction.name = '(5-Glutamyl)-peptide:amino-acid 5-glutamyltransferase Cyanoamino acid metabolism EC:2.3.2.2'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd759: -1.0, Cpd2417: -1.0, Cpd114: 1.0, Cpd2419: 1.0, Cpd2420: 1.0})
reaction.gene_reaction_rule = '( GGT1 or GGT5 or GGT6 or GGT7 )'
model.add_reactions([reaction])
reaction = Reaction('r0650')
reaction.name = 'Chenodeoxycholate:CoA ligase (AMP-forming) Bile acid biosynthesis EC:6.2.1.7'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2421: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2422: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('r0651')
reaction.name = 'Chenodeoxycholate:CoA ligase (AMP-forming) Bile acid biosynthesis EC:6.2.1.7'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd2423: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd2424: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('r0652')
reaction.name = 'trans-Tetradec-2-enoyl-CoA reductase Fatty acid elongation in mitochondria EC:1.3.1.38'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd1061: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2425: 1.0})
reaction.gene_reaction_rule = '( MECR or PECR )'
model.add_reactions([reaction])
reaction = Reaction('r0653')
reaction.name = 'myristoyl-CoA:acetylCoA C-myristoyltransferase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:2.3.1.16'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2426: -1.0, Cpd178: 1.0, Cpd1061: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r0655')
reaction.name = '3-Methylbutanoyl-CoA:(acceptor) 2,3-oxidoreductase Valine, leucine and isoleucine degradation EC:1.3.99.10'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd242: -1.0, Cpd723: -1.0, Cpd243: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( ACADM or IVD )'
model.add_reactions([reaction])
reaction = Reaction('r0656')
reaction.name = '3-methylbutanoyl-CoA:enzyme N6-(dihydrolipoyl)lysine S-(3-methylbutanoyl)transferase Valine, leucine and isoleucine degradation EC:2.3.1.168'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2346: -1.0, Cpd242: 1.0, Cpd1299: 1.0})
reaction.gene_reaction_rule = '( DBT )'
model.add_reactions([reaction])
reaction = Reaction('r0660')
reaction.name = '(S)-3-Hydroxydodecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2427: -1.0, Cpd12: 1.0, Cpd807: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0661')
reaction.name = '(S)-3-Hydroxydodecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2428: -1.0, Cpd2429: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0666')
reaction.name = '2-(Formamido)-N1-(5-phosphoribosyl)acetamidine cyclo-ligase (ADP-forming) Purine metabolism EC:6.3.3.1'
reaction.subsystem = 'Purine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1998: -1.0, Cpd107: 1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd378: 1.0})
reaction.gene_reaction_rule = '( GART )'
model.add_reactions([reaction])
reaction = Reaction('r0668')
reaction.name = 'CTP:N-acylneuraminate cytidylyltransferase Aminosugars metabolism EC:2.7.7.43'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd653: -1.0, Cpd2430: -1.0, Cpd151: 1.0, Cpd2431: 1.0})
reaction.gene_reaction_rule = '( CMAS )'
model.add_reactions([reaction])
reaction = Reaction('r0669')
reaction.name = '(S)-3-Hydroxyisobutyryl-CoA hydro-lyase Valine, leucine and isoleucine degradation EC:4.2.1.17'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd245: -1.0, Cpd2432: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0670')
reaction.name = '(S)-3-Methyl-2-oxopentanoate:[dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] lipoyllysine 2-oxidoreductase (decarboxylating, acceptor-2-methylpropanoylating) EC:1.2.4.4'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd1557: -1.0, Cpd1297: -1.0, Cpd45: 1.0, Cpd2398: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or TMEM91 )'
model.add_reactions([reaction])
reaction = Reaction('r0671')
reaction.name = '(R)-4-Phosphopantothenate:L-cysteine ligase EC:6.3.2.5'
reaction.subsystem = 'CoA synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd653: -1.0, Cpd1984: -1.0, Cpd33: 1.0, Cpd151: 1.0, Cpd651: 1.0, Cpd1990: 1.0})
reaction.gene_reaction_rule = '( PPCS )'
model.add_reactions([reaction])
reaction = Reaction('r0672')
reaction.name = 'Cholest-5-ene-3beta,7alpha-diol:NAD+ 3-oxidoreductase Bile acid biosynthesis EC:1.1.1.181'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd671: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd1533: 1.0})
reaction.gene_reaction_rule = '( HSD3B7 )'
model.add_reactions([reaction])
reaction = Reaction('r0673')
reaction.name = '6-Lactoyl-5,6,7,8-tetrahydropterin:NADP+ 2-oxidoreductase Folate biosynthesis EC:1.1.1.220'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2029: -1.0, Cpd25: 1.0, Cpd2348: 1.0})
reaction.gene_reaction_rule = '( SPR )'
model.add_reactions([reaction])
reaction = Reaction('r0678')
reaction.name = 'Acyl-[acyl-carrier-protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd250: -1.0, Cpd1668: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2433: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0679')
reaction.name = 'ATP:pantothenate 4-phosphotransferase Pantothenate and CoA biosynthesis EC:2.7.1.33'
reaction.subsystem = 'CoA catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2434: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1990: 1.0})
reaction.gene_reaction_rule = '( PANK1 or PANK2 or PANK3 or PANK4 )'
model.add_reactions([reaction])
reaction = Reaction('r0680')
reaction.name = 'ATP:pantothenate 4-phosphotransferase Pantothenate and CoA biosynthesis EC:2.7.1.33'
reaction.subsystem = 'CoA catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd2435: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd2401: 1.0})
reaction.gene_reaction_rule = '( PANK1 or PANK2 or PANK3 or PANK4 )'
model.add_reactions([reaction])
reaction = Reaction('r0681')
reaction.name = '(3R)-3-Hydroxybutanoyl-[acyl-carrier-protein] hydro-lyase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2436: -1.0, Cpd2: 1.0, Cpd2437: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0682')
reaction.name = 'Butyryl-[acyl-carrier protein]:malonyl-CoA C-acyltransferase(decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing) Fatty acid biosynthesis EC:2.1.3.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2438: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2437: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0683')
reaction.name = 'Propanoyl-CoA:(acceptor) 2,3-oxidoreductase beta-Alanine metabolism EC:1.3.3.6 EC:1.3.99.3'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd179: -1.0, Cpd723: -1.0, Cpd752: 1.0, Cpd1992: 1.0})
reaction.gene_reaction_rule = '( ACADM )'
model.add_reactions([reaction])
reaction = Reaction('r0686')
reaction.name = 'L-1-Pyrroline-3-hydroxy-5-carboxylate:NADP+ oxidoreductase Arginine and proline metabolism EC:1.5.1.12'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -2.0, Cpd13: -1.0, Cpd1937: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd100: 1.0})
reaction.gene_reaction_rule = '( ALDH4A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0688')
reaction.name = '3alpha,7alpha-Dihydroxy-5beta-cholestan-26-al:NAD+ oxidoreductase Bile acid biosynthesis EC:1.2.1.3'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd2439: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd2440: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0691')
reaction.name = '(3R)-3-Hydroxybutanoyl-[acyl-carrier protein]:NADP+ oxidoreductase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2436: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2433: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0692')
reaction.name = '(3R)-3-Hydroxydecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2441: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2442: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0693')
reaction.name = '(3R)-3-Hydroxybutanoyl-[acyl-carrier-protein] hydro-lyase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2441: -1.0, Cpd2: 1.0, Cpd2443: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0694')
reaction.name = '(3R)-3-Hydroxyoctanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2444: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2445: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0695')
reaction.name = '(3R)-3-Hydroxybutanoyl-[acyl-carrier-protein] hydro-lyase; (3R)-3-Hydroxyoctanoyl-[acyl-carrier-protein] hydro-lyase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2444: -1.0, Cpd2: 1.0, Cpd2446: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0696')
reaction.name = '(3R)-3-Hydroxypalmitoyl-[acyl-carrier-protein]:NADP+ oxidoreductase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2447: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2448: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0697')
reaction.name = '(3R)-3-Hydroxypalmitoyl-[acyl-carrier-protein] hydro-lyase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2447: -1.0, Cpd2: 1.0, Cpd2449: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0698')
reaction.name = 'Propanoyl-CoA:acetyl-CoA C-acyltransferase Bile acid biosynthesis EC:2.3.1.16'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -4.0, Cpd535: -1.0, Cpd742: -1.0, Cpd184: 1.0, Cpd261: 1.0, Cpd433: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r0701')
reaction.name = '(3R)-3-Hydroxytetradecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2450: -1.0, Cpd22: 1.0, Cpd2451: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0702')
reaction.name = '(3R)-3-Hydroxypalmitoyl-[acyl-carrier-protein] hydro-lyase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2450: -1.0, Cpd2: 1.0, Cpd2452: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0706')
reaction.name = 'acyl-Coenzyme A dehydrogenase family, member 9  Bile acid biosynthesis EC:1.3.3.6'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd438: -1.0, Cpd247: 1.0, Cpd262: 1.0})
reaction.gene_reaction_rule = '( ACAD9 )'
model.add_reactions([reaction])
reaction = Reaction('r0707')
reaction.name = '2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl) dihydropteridine triphosphate phosphohydrolase (alkaline optimum) Folate biosynthesis EC:3.1.3.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -3.0, Cpd2453: -1.0, Cpd37: 2.0, Cpd494: 3.0, Cpd2454: 1.0})
reaction.gene_reaction_rule = '( ALPL )'
model.add_reactions([reaction])
reaction = Reaction('r0708')
reaction.name = '2-Amino-4-hydroxy-6-(erythro-1,2,3-trihydroxypropyl) dihydropteridine triphosphate 7,8-8,9-dihydrolase Folate biosynthesis EC:3.5.4.16'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd33: -4.0, Cpd1446: -1.0, Cpd2455: 1.0})
reaction.gene_reaction_rule = '( GCH1 )'
model.add_reactions([reaction])
reaction = Reaction('r0709')
reaction.name = 'r0709'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd763: -4.0, Cpd1448: -1.0, Cpd2456: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0712')
reaction.name = 'Dodecanoyl-[acyl-carrier protein]: malonyl-CoA C-acyltransferase(decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1003: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2457: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0713')
reaction.name = 'dodecanoyl-[acyl-carrier-protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1003: -1.0, Cpd1668: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2451: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0714')
reaction.name = '(S)-3-Hydroxyhexadecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.211 EC:1.1.1.35'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2458: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2426: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0715')
reaction.name = '(S)-3-Hydroxyhexadecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.35 EC:1.1.1.211'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2459: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd193: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0716')
reaction.name = '(S)-3-Hydroxyhexadecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd601: -1.0, Cpd2458: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0717')
reaction.name = '(S)-3-Hydroxyhexadecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd248: -1.0, Cpd261: -1.0, Cpd2459: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0718')
reaction.name = '(S)-3-Hydroxytetradecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.35 EC:1.1.1.211'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2460: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2412: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0719')
reaction.name = '(S)-3-Hydroxytetradecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.35 EC:1.1.1.211'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2461: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd191: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0720')
reaction.name = '(S)-3-Hydroxytetradecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2425: -1.0, Cpd2460: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0721')
reaction.name = '(S)-3-Hydroxytetradecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2462: -1.0, Cpd2461: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0722')
reaction.name = '(S)-3-Hydroxydodecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.35 EC:1.1.1.211'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2427: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2463: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0723')
reaction.name = '(S)-3-Hydroxydodecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.211 EC:1.1.1.35'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2429: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd189: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0724')
reaction.name = 'Decanoyl-CoA:acetyl-CoA C-acyltransferase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:2.3.1.16'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd2464: -1.0, Cpd43: 1.0, Cpd2463: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r0726')
reaction.name = '(S)-Hydroxydecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.35 EC:1.1.1.211'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2465: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2408: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0727')
reaction.name = '(S)-Hydroxydecanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.211 EC:1.1.1.35'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2466: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd187: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0728')
reaction.name = '(S)-Hydroxydecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2465: -1.0, Cpd12: 1.0, Cpd2467: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0729')
reaction.name = '(S)-Hydroxydecanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2468: -1.0, Cpd2466: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0730')
reaction.name = '(S)-Hydroxyoctanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.211'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2469: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2470: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0731')
reaction.name = '(S)-Hydroxyoctanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2407: -1.0, Cpd2469: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0732')
reaction.name = 'Hexanoyl-CoA:acetyl-CoA C-acyltransferase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:2.3.1.16'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2470: -1.0, Cpd178: 1.0, Cpd2471: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r0733')
reaction.name = '(S)-Hydroxyhexanoyl-CoA:NAD+ oxidoreductase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:1.1.1.211 EC:1.1.1.35'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2472: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2335: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0734')
reaction.name = '(S)-Hydroxyhexanoyl-CoA hydro-lyase Fatty acid elongation in mitochondria / Fatty acid metabolism EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2473: -1.0, Cpd2472: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0735')
reaction.name = 'trans-Dec-2-enoyl-CoA reductase Fatty acid elongation in mitochondria EC:1.3.1.38'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd2464: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2467: 1.0})
reaction.gene_reaction_rule = '( MECR or PECR )'
model.add_reactions([reaction])
reaction = Reaction('r0737')
reaction.name = '3-Ketolactose galactohydrolase Galactose metabolism EC:3.2.1.23'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2474: -1.0, Cpd446: 1.0, Cpd2475: 1.0})
reaction.gene_reaction_rule = '( GLB1 )'
model.add_reactions([reaction])
reaction = Reaction('r0739')
reaction.name = 'alcohol dehydrogenase Bile acid biosynthesis EC:1.1.1.1'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2476: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd2439: 1.0})
reaction.gene_reaction_rule = '( ADH1 or ADH4 or ADH5 or ADH6A or ADH7 )'
model.add_reactions([reaction])
reaction = Reaction('r0741')
reaction.name = '5beta-Cholestane-3alpha,7alpha,12alpha-triol,NADPH:oxygen oxidoreductase (26-hydroxylating) Bile acid biosynthesis EC:1.14.13.15'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd389: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd2477: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0743')
reaction.name = 'hydroxysteroid (17-beta) dehydrogenase 4  Bile acid biosynthesis EC:1.1.1.35'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2478: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd1518: 1.0})
reaction.gene_reaction_rule = '( HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('r0744')
reaction.name = '(24R,25R)-3alpha,7alpha,12alpha,24-tetrahydroxy-5beta-cholestanoyl- CoA hydro-lyase Bile acid biosynthesis EC:4.2.1.107'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd262: -1.0, Cpd2478: 1.0})
reaction.gene_reaction_rule = '( HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('r0747')
reaction.name = '3alpha,7alpha-Dihydroxy-5beta-cholestane:NADP+ oxidoreductase (B-specific); 3alpha,7alpha-Dihydroxy-5beta-cholestane:NADP+ oxidoreductase Bile acid biosynthesis EC:1.1.1.50'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd387: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd386: 1.0})
reaction.gene_reaction_rule = '( AKR1C6 )'
model.add_reactions([reaction])
reaction = Reaction('r0750')
reaction.name = '3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane:NADP+ oxidoreductase (B-specific); 3alpha,7alpha,12alpha-Trihydroxy-5beta-cholestane:NADP+ oxidoreductase Bile acid biosynthesis EC:1.1.1.50'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd389: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd388: 1.0})
reaction.gene_reaction_rule = '( AKR1C6 )'
model.add_reactions([reaction])
reaction = Reaction('r0752')
reaction.name = '3,4-Dihydroxymandelaldehyde:NAD+ oxidoreductase Tyrosine metabolism EC:1.2.1.5'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2479: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd2480: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0753')
reaction.name = '3,4-Dihydroxymandelaldehyde:NADP+ oxidoreductase Tyrosine metabolism EC:1.2.1.5'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd2479: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd2480: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0754')
reaction.name = '3-Methoxy-4-hydroxyphenylacetaldehyde:NAD+ oxidoreductase Tyrosine metabolism EC:1.2.1.5'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2481: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd2482: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0755')
reaction.name = '3-Methoxy-4-hydroxyphenylacetaldehyde:NADP+ oxidoreductase Tyrosine metabolism EC:1.2.1.5'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd2481: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd2482: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0756')
reaction.name = '3-Methoxy-4-hydroxyphenylglycolaldehyde:NAD+ oxidoreductase Tyrosine metabolism EC:1.2.1.5'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd2483: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd2484: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0757')
reaction.name = '3-Methoxy-4-hydroxyphenylglycolaldehyde:NADP+ oxidoreductase Tyrosine metabolism EC:1.2.1.5'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd2483: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd2484: 1.0})
reaction.gene_reaction_rule = '( ALDH1A3 or ALDH3A1 or ALDH3B1 or ALDH3B3 )'
model.add_reactions([reaction])
reaction = Reaction('r0758')
reaction.name = '5-Hydroxyindoleacetaldehyde:NAD+ oxidoreductase Tryptophan metabolism EC:1.2.1.3'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd117: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd118: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH3A2 or ALDH7A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0760')
reaction.name = 'butyryl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1668: -1.0, Cpd2438: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2485: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0761')
reaction.name = '(3R)-3-Hydroxyhexanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2486: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2485: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0762')
reaction.name = '(3R)-3-Hydroxybutanoyl-[acyl-carrier-protein] hydro-lyase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2486: -1.0, Cpd2: 1.0, Cpd2487: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0763')
reaction.name = 'Hexanoyl-[acyl-carrier protein]:oxoacyl- and enoyl-reducing and thioester-hydrolysing) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2488: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2487: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0764')
reaction.name = 'hexanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1668: -1.0, Cpd2488: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2445: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0765')
reaction.name = 'Octanoyl-[acyl-carrier protein]:malonyl-CoA C-acyltransferase(decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2489: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2446: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0766')
reaction.name = 'Octanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1668: -1.0, Cpd2489: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2442: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0767')
reaction.name = 'Decanoyl-[acyl-carrier protein]:malonyl-CoA C-acyltransferase(decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2490: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2443: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0768')
reaction.name = 'Decanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1668: -1.0, Cpd2490: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2491: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0769')
reaction.name = '(3R)-3-Hydroxydodecanoyl-[acyl-carrier-protein]:NADP+ oxidoreductase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2492: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2491: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0770')
reaction.name = '(3R)-3-Hydroxybutanoyl-[acyl-carrier-protein] hydro-lyase Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2492: -1.0, Cpd2: 1.0, Cpd2457: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0771')
reaction.name = 'Tetradecanoyl-[acyl-carrier protein]:malonyl-CoA C-acyltransferase(decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1005: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2452: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0772')
reaction.name = 'Tetradecanoyl-[acyl-carrier protein]:malonyl-[acyl-carrier-protein] C-acyltransferase (decarboxylating) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1005: -1.0, Cpd1668: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2448: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0773')
reaction.name = 'Hexadecanoyl-[acyl-carrier protein:malonyl-CoA C-acyltransferase(decarboxylating, oxoacyl- and enoyl-reducing and thioester-hydrolysing) Fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1009: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd2449: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r0774')
reaction.name = 'Uroporphyrinogen I carboxy-lyase Porphyrin and chlorophyll metabolism EC:4.1.1.37'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -4.0, Cpd2493: -1.0, Cpd2419: 4.0, Cpd2494: 1.0})
reaction.gene_reaction_rule = '( UROD )'
model.add_reactions([reaction])
reaction = Reaction('r0775')
reaction.name = 'Formamidopyrimidine nucleoside triphosphate 7,8-8,9-dihydrolase Folate biosynthesis EC:3.5.4.16'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2314: -1.0, Cpd33: 5.0, Cpd699: 1.0, Cpd2495: 1.0})
reaction.gene_reaction_rule = '( GCH1 )'
model.add_reactions([reaction])
reaction = Reaction('r0776')
reaction.name = 'r0776'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd570: -1.0, Cpd2315: -1.0, Cpd763: 5.0, Cpd1447: 1.0, Cpd2496: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r0777')
reaction.name = 'GTP 7,8-8,9-dihydrolase Folate biosynthesis EC:3.5.4.16'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -4.0, Cpd2495: -1.0, Cpd2455: 1.0})
reaction.gene_reaction_rule = '( GCH1 )'
model.add_reactions([reaction])
reaction = Reaction('r0778')
reaction.name = 'r0778'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -4.0, Cpd2496: -1.0, Cpd2456: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('r0779')
reaction.name = '3-Hydroxy-2-methylpropanoyl-CoA hydrolase EC:3.1.2.4'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2432: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd1478: 1.0})
reaction.gene_reaction_rule = '( HIBCH )'
model.add_reactions([reaction])
reaction = Reaction('r0780')
reaction.name = '4,4-dimethyl-5a-cholesta-8,24-dien-3b-ol:NADP+ D14-oxidoreductase Biosynthesis of steroids EC:1.3.1.70'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd588: -1.0, Cpd589: -1.0, Cpd565: 1.0, Cpd586: 1.0, Cpd587: 1.0})
reaction.gene_reaction_rule = '( TM7SF2 )'
model.add_reactions([reaction])
reaction = Reaction('r0781')
reaction.name = 'Lanosterol,NADPH:oxygen oxidoreductase (14-methyl cleaving) Biosynthesis of steroids EC:1.14.13.70'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -3.0, Cpd565: -2.0, Cpd587: -3.0, Cpd1615: -1.0, Cpd437: 4.0, Cpd586: 1.0, Cpd589: 3.0, Cpd1856: 1.0})
reaction.gene_reaction_rule = '( CYP51 )'
model.add_reactions([reaction])
reaction = Reaction('r0782')
reaction.name = 'GDP-L-fucose:NADP+ 4-oxidoreductase (3,5-epimerizing) Fructose and mannose metabolism EC:1.1.1.271'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd1308: -1.0, Cpd32: 1.0, Cpd998: 1.0})
reaction.gene_reaction_rule = '( TSTA3 )'
model.add_reactions([reaction])
reaction = Reaction('r0783')
reaction.name = 'lanosterol D24-reductase Biosynthesis of steroids EC:1.3.1.72'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd851: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd850: 1.0})
reaction.gene_reaction_rule = '( DHCR24 )'
model.add_reactions([reaction])
reaction = Reaction('r0786')
reaction.name = 'sphinganine-1-phosphate palmitaldehyde-lyase Sphingolipid metabolism EC:4.1.2.27'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2139: -1.0, Cpd996: 1.0, Cpd2497: 1.0})
reaction.gene_reaction_rule = '( SGPL1 )'
model.add_reactions([reaction])
reaction = Reaction('r0787')
reaction.name = '3-sn-phosphatidate phosphohydrolase Sphingolipid metabolism EC:3.1.3.4'
reaction.subsystem = 'Sphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2122: -1.0, Cpd108: 1.0, Cpd67: 1.0})
reaction.gene_reaction_rule = '( PLPP1 or PLPP2 or PLPP3 or SGPP1 or SGPP2 )'
model.add_reactions([reaction])
reaction = Reaction('r0791')
reaction.name = 'trans-Hex-2-enoyl-CoA reductase Fatty acid elongation in mitochondria EC:1.3.1.38'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd2473: -1.0, Cpd13: 1.0, Cpd2471: 1.0})
reaction.gene_reaction_rule = '( MECR or PECR )'
model.add_reactions([reaction])
reaction = Reaction('r0792')
reaction.name = '5-methyltetrahydrofolate:NAD+ oxidoreductase One carbon pool by folate / Methane metabolism EC:1.5.1.20'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd127: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd1341: 1.0})
reaction.gene_reaction_rule = '( MTHFR )'
model.add_reactions([reaction])
reaction = Reaction('r0797')
reaction.name = 'glycine N-choloyltransferase EC:2.3.1.65'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd2424: -1.0, Cpd78: 1.0, Cpd2498: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('r0812')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.20.1'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2283: -1.0, Cpd150: 1.0, Cpd940: 1.0})
reaction.gene_reaction_rule = '( SLC25A17 )'
model.add_reactions([reaction])
reaction = Reaction('r0813')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.9'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1957: -1.0, Cpd2255: -1.0, Cpd1959: 1.0, Cpd555: 1.0, Cpd565: 1.0, Cpd915: 1.0})
reaction.gene_reaction_rule = '( ABCC3 )'
model.add_reactions([reaction])
reaction = Reaction('r0817')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.3.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd471: -1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A2 )'
model.add_reactions([reaction])
reaction = Reaction('r0819')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd1843: -1.0, Cpd28: 1.0, Cpd543: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0821')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd313: -1.0, Cpd1843: -1.0, Cpd543: 1.0, Cpd1159: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0822')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd313: -1.0, Cpd382: -1.0, Cpd381: 1.0, Cpd1159: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0829')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd543: -1.0, Cpd1162: -1.0, Cpd1163: 1.0, Cpd1843: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0830')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1160: -1.0, Cpd543: -1.0, Cpd1161: 1.0, Cpd1843: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0834')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd204: -1.0, Cpd2500: -1.0, Cpd108: 1.0, Cpd2501: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0835')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1162: -1.0, Cpd2500: -1.0, Cpd1163: 1.0, Cpd2501: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0836')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1160: -1.0, Cpd2500: -1.0, Cpd1161: 1.0, Cpd2501: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0845')
reaction.name = 'Drug/Metabolite Transporter (DMT) TCDB:2.A.7.12.7'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1396: -1.0, Cpd1770: -1.0, Cpd1398: 1.0, Cpd2312: 1.0})
reaction.gene_reaction_rule = '( SLC35A3 )'
model.add_reactions([reaction])
reaction = Reaction('r0879')
reaction.name = 'TCDB:2.A.29.2.4 TCDB:2.A.29.16.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd383: -1.0, Cpd30: 1.0, Cpd41: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 or SLC25A21 )'
model.add_reactions([reaction])
reaction = Reaction('r0885')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.7'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd204: -1.0, Cpd1057: -1.0, Cpd108: 1.0, Cpd1418: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r0913')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd255: -1.0, Cpd381: -1.0, Cpd253: 1.0, Cpd382: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0915')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd228: -1.0, Cpd1843: -1.0, Cpd254: 1.0, Cpd543: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0917')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd228: -1.0, Cpd255: -1.0, Cpd253: 1.0, Cpd254: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0924')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.204.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1957: -1.0, Cpd670: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd915: 1.0, Cpd684: 1.0})
reaction.gene_reaction_rule = '( ABCG5 or ABCG8 )'
model.add_reactions([reaction])
reaction = Reaction('r0934')
reaction.name = 'Neurotransmitter:Sodium Symporter (NSS) TCDB:2.A.22.3.3'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1586: -1.0, Cpd531: 1.0})
reaction.gene_reaction_rule = '( SLC6A6 )'
model.add_reactions([reaction])
reaction = Reaction('r0942')
reaction.name = 'Neurotransmitter:Sodium Symporter (NSS) TCDB:2.A.22.3.4'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd696: -1.0, Cpd735: 1.0})
reaction.gene_reaction_rule = '( SLC6A8 )'
model.add_reactions([reaction])
reaction = Reaction('r0946')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd471: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r0947')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd470: -1.0, Cpd471: -1.0, Cpd252: 1.0, Cpd691: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r0954')
reaction.name = 'MCT 1 Transport reaction'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2375: -1.0, Cpd1678: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0960')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.65.1.1'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd41: -1.0, Cpd560: -1.0, Cpd2252: 1.0, Cpd2502: 1.0})
reaction.gene_reaction_rule = '( CP )'
model.add_reactions([reaction])
reaction = Reaction('r0961')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.65.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd561: -1.0, Cpd41: 1.0, Cpd560: 1.0})
reaction.gene_reaction_rule = '( CP )'
model.add_reactions([reaction])
reaction = Reaction('r0963')
reaction.name = 'Facilitated diffusion'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1119: -1.0, Cpd1118: 1.0})
reaction.gene_reaction_rule = '( FOLR1 or FOLR2 )'
model.add_reactions([reaction])
reaction = Reaction('r0983')
reaction.name = 'solute carrier family 27 (fatty acid transporter), member 5  TCDB:4.C.1.1.5 TCDB:4.C.1.1.8'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2503: -1.0, Cpd2504: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('r0984')
reaction.name = 'solute carrier family 27 (fatty acid transporter), member 5  TCDB:4.C.1.1.5 TCDB:4.C.1.1.8'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2503: -1.0, Cpd1029: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('r0992')
reaction.name = 'Na(+)/bile acid cotransporter Active transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1304: -1.0, Cpd2505: -1.0, Cpd1305: 1.0, Cpd2506: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r0993')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2423: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2507: 1.0})
reaction.gene_reaction_rule = '( ABCC2 )'
model.add_reactions([reaction])
reaction = Reaction('r0995')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.8.3'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd746: -1.0, Cpd265: 1.0, Cpd744: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('r0997')
reaction.name = 'COT Facilitated diffusion'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd2410: -1.0, Cpd737: 1.0, Cpd2508: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('r1015')
reaction.name = 'Na(+)/bile acid cotransporter Active transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2191: -1.0, Cpd41: 1.0, Cpd2192: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1026')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.201.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2498: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2509: 1.0})
reaction.gene_reaction_rule = '( ABCB11 )'
model.add_reactions([reaction])
reaction = Reaction('r1028')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.201.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2510: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2511: 1.0})
reaction.gene_reaction_rule = '( ABCB11 )'
model.add_reactions([reaction])
reaction = Reaction('r1043')
reaction.name = 'Concentrative Nucleoside Transporter (CNT) TCDB:2.A.41.2.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd795: -1.0, Cpd41: 1.0, Cpd793: 1.0})
reaction.gene_reaction_rule = '( SLC28A1 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('r1044')
reaction.name = 'Concentrative Nucleoside Transporter (CNT) TCDB:2.A.41.2.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd820: -1.0, Cpd41: 1.0, Cpd817: 1.0})
reaction.gene_reaction_rule = '( SLC28A1 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('r1045')
reaction.name = 'Concentrative Nucleoside Transporter (CNT) TCDB:2.A.41.2.4'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd840: -1.0, Cpd41: 1.0, Cpd841: 1.0})
reaction.gene_reaction_rule = '( SLC28A2 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('r1059')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.18.6.7'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2512: -1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC38A4 )'
model.add_reactions([reaction])
reaction = Reaction('r1090')
reaction.name = 'TCDB:2.A.47.1.9 TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd692: -1.0, Cpd41: 1.0, Cpd228: 1.0})
reaction.gene_reaction_rule = '( SLC13A5 or SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1092')
reaction.name = 'albumin  Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2436.0, Cpd105: -2436.0, Cpd74: -62.0, Cpd90: -36.0, Cpd393: -17.0, Cpd394: -63.0, Cpd396: -35.0, Cpd399: -20.0, Cpd402: -13.0, Cpd404: -28.0, Cpd408: -29.0, Cpd465: -60.0, Cpd467: -27.0, Cpd554: -7.0, Cpd1479: -16.0, Cpd1544: -19.0, Cpd1554: -9.0, Cpd1590: -64.0, Cpd1638: -2.0, Cpd1940: -35.0, Cpd2004: -24.0, Cpd2275: -43.0, Cpd107: 1827.0, Cpd108: 1827.0, Cpd150: 609.0, Cpd151: 609.0, Cpd2513: 1.0})
reaction.gene_reaction_rule = '( ALB )'
model.add_reactions([reaction])
reaction = Reaction('r1093')
reaction.name = 'serpin peptidase inhibitor, clade A (alpha-1 antiproteinase, antitrypsin), member 3  Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1692.0, Cpd105: -1692.0, Cpd74: -29.0, Cpd90: -25.0, Cpd393: -19.0, Cpd394: -34.0, Cpd396: -3.0, Cpd399: -17.0, Cpd402: -16.0, Cpd404: -30.0, Cpd408: -29.0, Cpd465: -26.0, Cpd467: -16.0, Cpd554: -14.0, Cpd1479: -9.0, Cpd1544: -9.0, Cpd1554: -20.0, Cpd1590: -59.0, Cpd1638: -3.0, Cpd1940: -25.0, Cpd2004: -16.0, Cpd2275: -24.0, Cpd107: 1269.0, Cpd108: 1269.0, Cpd150: 423.0, Cpd151: 423.0, Cpd2514: 1.0})
reaction.gene_reaction_rule = '( SERPINA3A )'
model.add_reactions([reaction])
reaction = Reaction('r1094')
reaction.name = 'serpin peptidase inhibitor, clade A (alpha-1 antiproteinase, antitrypsin), member 1  Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1672.0, Cpd105: -1672.0, Cpd74: -32.0, Cpd90: -24.0, Cpd393: -19.0, Cpd394: -26.0, Cpd396: -3.0, Cpd399: -18.0, Cpd402: -24.0, Cpd404: -25.0, Cpd408: -30.0, Cpd465: -34.0, Cpd467: -7.0, Cpd554: -10.0, Cpd1479: -13.0, Cpd1544: -6.0, Cpd1554: -20.0, Cpd1590: -51.0, Cpd1638: -3.0, Cpd1940: -27.0, Cpd2004: -19.0, Cpd2275: -27.0, Cpd107: 1254.0, Cpd108: 1254.0, Cpd150: 418.0, Cpd151: 418.0, Cpd2515: 1.0})
reaction.gene_reaction_rule = '( SERPINA1B )'
model.add_reactions([reaction])
reaction = Reaction('r1095')
reaction.name = 'apolipoprotein B  Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -18244.0, Cpd105: -18244.0, Cpd74: -298.0, Cpd90: -233.0, Cpd393: -247.0, Cpd394: -275.0, Cpd396: -25.0, Cpd399: -230.0, Cpd402: -207.0, Cpd404: -392.0, Cpd408: -300.0, Cpd465: -357.0, Cpd467: -150.0, Cpd554: -79.0, Cpd1479: -115.0, Cpd1544: -151.0, Cpd1554: -285.0, Cpd1590: -533.0, Cpd1638: -37.0, Cpd1940: -224.0, Cpd2004: -171.0, Cpd2275: -252.0, Cpd107: 13683.0, Cpd108: 13683.0, Cpd150: 4561.0, Cpd151: 4561.0, Cpd2516: 1.0})
reaction.gene_reaction_rule = '( APOB )'
model.add_reactions([reaction])
reaction = Reaction('r1096')
reaction.name = 'NADH dehydrogenase (ubiquinone) 1, alpha/beta subcomplex, 1  Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -496.0, Cpd105: -496.0, Cpd74: -5.0, Cpd90: -8.0, Cpd393: -1.0, Cpd394: -11.0, Cpd396: -2.0, Cpd399: -7.0, Cpd402: -6.0, Cpd404: -8.0, Cpd408: -6.0, Cpd465: -4.0, Cpd467: -9.0, Cpd554: -5.0, Cpd1479: -1.0, Cpd1544: -4.0, Cpd1554: -4.0, Cpd1590: -20.0, Cpd1940: -3.0, Cpd2004: -9.0, Cpd2275: -11.0, Cpd107: 372.0, Cpd108: 372.0, Cpd150: 124.0, Cpd151: 124.0, Cpd2341: 1.0})
reaction.gene_reaction_rule = '( NDUFAB1 )'
model.add_reactions([reaction])
reaction = Reaction('r1097')
reaction.name = 'ACP Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -348.0, Cpd63: -9.0, Cpd92: -13.0, Cpd200: -348.0, Cpd361: -3.0, Cpd364: -3.0, Cpd576: -7.0, Cpd464: -1.0, Cpd481: -1.0, Cpd754: -1.0, Cpd1342: -3.0, Cpd1406: -3.0, Cpd1556: -8.0, Cpd1592: -10.0, Cpd1898: -5.0, Cpd1942: -3.0, Cpd2239: -4.0, Cpd2277: -6.0, Cpd2517: -5.0, Cpd2518: -1.0, Cpd2519: -1.0, Cpd202: 261.0, Cpd204: 261.0, Cpd209: 87.0, Cpd210: 87.0, Cpd2520: 1.0})
reaction.gene_reaction_rule = '( NDUFAB1 )'
model.add_reactions([reaction])
reaction = Reaction('r1098')
reaction.name = 'Apo-CIB Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -332.0, Cpd105: -332.0, Cpd74: -8.0, Cpd90: -4.0, Cpd393: -1.0, Cpd394: -5.0, Cpd399: -3.0, Cpd402: -3.0, Cpd404: -9.0, Cpd408: -3.0, Cpd465: -9.0, Cpd467: -4.0, Cpd554: -2.0, Cpd1554: -4.0, Cpd1590: -12.0, Cpd1638: -1.0, Cpd1940: -4.0, Cpd2004: -4.0, Cpd2275: -7.0, Cpd107: 249.0, Cpd108: 249.0, Cpd150: 83.0, Cpd151: 83.0, Cpd2521: 1.0})
reaction.gene_reaction_rule = '( APOC1 )'
model.add_reactions([reaction])
reaction = Reaction('r1099')
reaction.name = 'Apo-CII Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -404.0, Cpd105: -404.0, Cpd74: -8.0, Cpd90: -4.0, Cpd393: -1.0, Cpd394: -7.0, Cpd399: -8.0, Cpd402: -5.0, Cpd404: -9.0, Cpd408: -10.0, Cpd465: -6.0, Cpd467: -2.0, Cpd554: -3.0, Cpd1544: -5.0, Cpd1554: -1.0, Cpd1590: -15.0, Cpd1638: -1.0, Cpd1940: -4.0, Cpd2004: -5.0, Cpd2275: -7.0, Cpd107: 303.0, Cpd108: 303.0, Cpd150: 101.0, Cpd151: 101.0, Cpd2522: 1.0})
reaction.gene_reaction_rule = '( APOC2 )'
model.add_reactions([reaction])
reaction = Reaction('r1100')
reaction.name = 'Apo-CIII Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -396.0, Cpd105: -396.0, Cpd74: -5.0, Cpd90: -7.0, Cpd394: -15.0, Cpd399: -6.0, Cpd402: -3.0, Cpd404: -12.0, Cpd408: -5.0, Cpd465: -6.0, Cpd467: -4.0, Cpd554: -3.0, Cpd1479: -1.0, Cpd1544: -2.0, Cpd1590: -11.0, Cpd1638: -3.0, Cpd1940: -4.0, Cpd2004: -3.0, Cpd2275: -9.0, Cpd107: 297.0, Cpd108: 297.0, Cpd150: 99.0, Cpd151: 99.0, Cpd2523: 1.0})
reaction.gene_reaction_rule = '( APOC3 )'
model.add_reactions([reaction])
reaction = Reaction('r1101')
reaction.name = 'Fibrinogen alpha chain Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -10696.0, Cpd105: -10696.0, Cpd74: -186.0, Cpd90: -176.0, Cpd393: -138.0, Cpd394: -134.0, Cpd396: -48.0, Cpd399: -110.0, Cpd402: -262.0, Cpd404: -278.0, Cpd408: -182.0, Cpd465: -154.0, Cpd467: -134.0, Cpd554: -48.0, Cpd1479: -62.0, Cpd1544: -98.0, Cpd1554: -104.0, Cpd1590: -166.0, Cpd1638: -60.0, Cpd1940: -98.0, Cpd2004: -114.0, Cpd2275: -122.0, Cpd107: 8022.0, Cpd108: 8022.0, Cpd150: 2674.0, Cpd151: 2674.0, Cpd2524: 1.0})
reaction.gene_reaction_rule = '( FGA )'
model.add_reactions([reaction])
reaction = Reaction('r1103')
reaction.name = 'Plasminogen Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -3240.0, Cpd105: -3240.0, Cpd74: -56.0, Cpd90: -36.0, Cpd393: -40.0, Cpd394: -37.0, Cpd396: -48.0, Cpd399: -31.0, Cpd402: -62.0, Cpd404: -56.0, Cpd408: -61.0, Cpd465: -49.0, Cpd467: -42.0, Cpd554: -11.0, Cpd1479: -24.0, Cpd1544: -30.0, Cpd1554: -22.0, Cpd1590: -48.0, Cpd1638: -19.0, Cpd1940: -21.0, Cpd2004: -69.0, Cpd2275: -48.0, Cpd107: 2430.0, Cpd108: 2430.0, Cpd150: 810.0, Cpd151: 810.0, Cpd2525: 1.0})
reaction.gene_reaction_rule = '( PLG )'
model.add_reactions([reaction])
reaction = Reaction('r1104')
reaction.name = 'Prothrombin Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2488.0, Cpd105: -2488.0, Cpd74: -51.0, Cpd90: -35.0, Cpd393: -25.0, Cpd394: -42.0, Cpd396: -26.0, Cpd399: -26.0, Cpd402: -49.0, Cpd404: -38.0, Cpd408: -36.0, Cpd465: -29.0, Cpd467: -44.0, Cpd554: -9.0, Cpd1479: -13.0, Cpd1544: -21.0, Cpd1554: -22.0, Cpd1590: -51.0, Cpd1638: -14.0, Cpd1940: -21.0, Cpd2004: -33.0, Cpd2275: -37.0, Cpd107: 1866.0, Cpd108: 1866.0, Cpd150: 622.0, Cpd151: 622.0, Cpd2526: 1.0})
reaction.gene_reaction_rule = '( F2 )'
model.add_reactions([reaction])
reaction = Reaction('r1105')
reaction.name = 'T9 Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -2792.0, Cpd105: -2792.0, Cpd74: -42.0, Cpd90: -45.0, Cpd393: -34.0, Cpd394: -61.0, Cpd396: -40.0, Cpd399: -17.0, Cpd402: -52.0, Cpd404: -41.0, Cpd408: -30.0, Cpd465: -58.0, Cpd467: -27.0, Cpd554: -10.0, Cpd1479: -19.0, Cpd1544: -26.0, Cpd1554: -15.0, Cpd1590: -65.0, Cpd1638: -8.0, Cpd1940: -28.0, Cpd2004: -32.0, Cpd2275: -48.0, Cpd107: 2094.0, Cpd108: 2094.0, Cpd150: 698.0, Cpd151: 698.0, Cpd2527: 1.0})
reaction.gene_reaction_rule = '( TFRC )'
model.add_reactions([reaction])
reaction = Reaction('r1112')
reaction.name = 'Apo-E Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1268.0, Cpd105: -1268.0, Cpd74: -40.0, Cpd90: -11.0, Cpd393: -1.0, Cpd394: -39.0, Cpd396: -2.0, Cpd399: -32.0, Cpd402: -18.0, Cpd404: -14.0, Cpd408: -12.0, Cpd465: -13.0, Cpd467: -34.0, Cpd554: -8.0, Cpd1479: -2.0, Cpd1544: -4.0, Cpd1554: -2.0, Cpd1590: -41.0, Cpd1638: -8.0, Cpd1940: -4.0, Cpd2004: -8.0, Cpd2275: -24.0, Cpd107: 951.0, Cpd108: 951.0, Cpd150: 317.0, Cpd151: 317.0, Cpd2528: 1.0})
reaction.gene_reaction_rule = '( APOE )'
model.add_reactions([reaction])
reaction = Reaction('r1113')
reaction.name = 'Apo-AI Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1068.0, Cpd105: -1068.0, Cpd74: -30.0, Cpd90: -16.0, Cpd393: -5.0, Cpd394: -23.0, Cpd399: -19.0, Cpd402: -11.0, Cpd404: -16.0, Cpd408: -12.0, Cpd465: -22.0, Cpd467: -17.0, Cpd554: -4.0, Cpd1479: -6.0, Cpd1544: -7.0, Cpd1590: -41.0, Cpd1638: -5.0, Cpd1940: -8.0, Cpd2004: -10.0, Cpd2275: -15.0, Cpd107: 801.0, Cpd108: 801.0, Cpd150: 267.0, Cpd151: 267.0, Cpd2529: 1.0})
reaction.gene_reaction_rule = '( APOA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1135')
reaction.name = 'hydroxysteroid (17-beta) dehydrogenase 7  Biosynthesis of steroids EC:1.1.1.270'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -3.0, Cpd589: -1.0, Cpd2530: -1.0, Cpd587: 1.0, Cpd2531: 1.0})
reaction.gene_reaction_rule = '( HSD17B7 )'
model.add_reactions([reaction])
reaction = Reaction('r1143')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.18.6.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd490: -1.0, Cpd41: 1.0, Cpd90: 1.0})
reaction.gene_reaction_rule = '( SLC38A3 )'
model.add_reactions([reaction])
reaction = Reaction('r1144')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.18.6.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd759: -1.0, Cpd41: 1.0, Cpd74: 1.0})
reaction.gene_reaction_rule = '( SLC38A3 )'
model.add_reactions([reaction])
reaction = Reaction('r1147')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd255: -1.0, Cpd28: 1.0, Cpd253: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1154')
reaction.name = 'EC:1.2.7.2'
reaction.subsystem = 'Propanoate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd44: -1.0, Cpd2532: -1.0, Cpd45: 1.0, Cpd47: 1.0, Cpd179: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or DBT or DLD )'
model.add_reactions([reaction])
reaction = Reaction('r1155')
reaction.name = 'TCDB:2.A.1.13.5 TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd34: -1.0, Cpd7: 1.0, Cpd2532: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A7 )'
model.add_reactions([reaction])
reaction = Reaction('r1156')
reaction.name = 'dUTP:uridine 5-phosphotransferase Pyrimidine metabolism EC:2.7.1.48'
reaction.subsystem = 'Nucleotide interconversion'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd761: -1.0, Cpd903: -1.0, Cpd33: 1.0, Cpd899: 1.0, Cpd1398: 1.0})
reaction.gene_reaction_rule = '( UCK1 or UCK2 or UCKL1 )'
model.add_reactions([reaction])
reaction = Reaction('r1163')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1304: -1.0, Cpd41: 1.0, Cpd1305: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1164')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2337: -1.0, Cpd745: 1.0, Cpd2533: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1165')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2534: -1.0, Cpd745: 1.0, Cpd2535: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1166')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2536: -1.0, Cpd745: 1.0, Cpd2537: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1167')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2538: -1.0, Cpd745: 1.0, Cpd2539: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1168')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2359: -1.0, Cpd745: 1.0, Cpd2540: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1169')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2541: -1.0, Cpd745: 1.0, Cpd2542: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1170')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2541: -1.0, Cpd745: 1.0, Cpd2543: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1171')
reaction.name = 'EC:2.3.1.26'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd670: -1.0, Cpd2363: -1.0, Cpd745: 1.0, Cpd2544: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('r1172')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2533: -1.0, Cpd565: 1.0, Cpd670: 1.0, Cpd1431: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1173')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2545: -1.0, Cpd176: 1.0, Cpd685: 1.0, Cpd2546: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1174')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2535: -1.0, Cpd565: 1.0, Cpd670: 1.0, Cpd2547: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1175')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2537: -1.0, Cpd670: 1.0, Cpd2504: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1176')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2548: -1.0, Cpd685: 1.0, Cpd2503: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1177')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2539: -1.0, Cpd565: 1.0, Cpd670: 1.0, Cpd2549: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1178')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2550: -1.0, Cpd176: 1.0, Cpd685: 1.0, Cpd2551: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1179')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2540: -1.0, Cpd565: 1.0, Cpd670: 1.0, Cpd2552: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1180')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2553: -1.0, Cpd176: 1.0, Cpd685: 1.0, Cpd2554: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1181')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2542: -1.0, Cpd565: 1.0, Cpd670: 1.0, Cpd2555: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1182')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2543: -1.0, Cpd565: 1.0, Cpd670: 1.0, Cpd2556: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1183')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2544: -1.0, Cpd1885: 1.0, Cpd565: 1.0, Cpd670: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1184')
reaction.name = 'EC:3.1.1.13'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2557: -1.0, Cpd176: 1.0, Cpd685: 1.0, Cpd2558: 1.0})
reaction.gene_reaction_rule = '( CEL or LIPA )'
model.add_reactions([reaction])
reaction = Reaction('r1185')
reaction.name = 'EC:2.3.1.15'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd590: -1.0, Cpd1198: -1.0, Cpd78: 1.0, Cpd2559: 1.0})
reaction.gene_reaction_rule = '( GPAT3 or GPAT4 )'
model.add_reactions([reaction])
reaction = Reaction('r1186')
reaction.name = 'EC:2.3.1.15'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd596: -1.0, Cpd1198: -1.0, Cpd78: 1.0, Cpd2560: 1.0})
reaction.gene_reaction_rule = '( GPAT3 or GPAT4 )'
model.add_reactions([reaction])
reaction = Reaction('r1187')
reaction.name = 'EC:2.3.1.15'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd602: -1.0, Cpd1198: -1.0, Cpd78: 1.0, Cpd2561: 1.0})
reaction.gene_reaction_rule = '( GPAT3 or GPAT4 )'
model.add_reactions([reaction])
reaction = Reaction('r1188')
reaction.name = 'EC:2.3.1.15'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd606: -1.0, Cpd1198: -1.0, Cpd78: 1.0, Cpd2562: 1.0})
reaction.gene_reaction_rule = '( GPAT3 or GPAT4 )'
model.add_reactions([reaction])
reaction = Reaction('r1189')
reaction.name = 'EC:2.3.1.15'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd828: -1.0, Cpd1198: -1.0, Cpd78: 1.0, Cpd2563: 1.0})
reaction.gene_reaction_rule = '( GPAT3 or GPAT4 )'
model.add_reactions([reaction])
reaction = Reaction('r1190')
reaction.name = 'EC:2.3.1.15'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd610: -1.0, Cpd1198: -1.0, Cpd78: 1.0, Cpd2564: 1.0})
reaction.gene_reaction_rule = '( GPAT3 or GPAT4 )'
model.add_reactions([reaction])
reaction = Reaction('r1251')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2547: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2534: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1252')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1012: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd596: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1253')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd565: -8.0, Cpd745: -1.0, Cpd2504: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2536: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1254')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd33: -8.0, Cpd78: -1.0, Cpd1029: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd602: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1255')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2549: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2538: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1257')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2552: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2359: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1259')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2555: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2541: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1260')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2556: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2541: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1262')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1885: -1.0, Cpd1957: -1.0, Cpd745: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2363: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1293')
reaction.name = 'ADF Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -420.0, Cpd105: -420.0, Cpd74: -10.0, Cpd90: -7.0, Cpd393: -3.0, Cpd394: -8.0, Cpd396: -5.0, Cpd399: -5.0, Cpd402: -5.0, Cpd404: -7.0, Cpd408: -4.0, Cpd465: -12.0, Cpd554: -3.0, Cpd1479: -1.0, Cpd1544: -1.0, Cpd1554: -4.0, Cpd1590: -6.0, Cpd1638: -1.0, Cpd1940: -9.0, Cpd2004: -3.0, Cpd2275: -11.0, Cpd107: 315.0, Cpd108: 315.0, Cpd150: 105.0, Cpd151: 105.0, Cpd2052: 1.0})
reaction.gene_reaction_rule = '( TXN1 )'
model.add_reactions([reaction])
reaction = Reaction('r1294')
reaction.name = 'MTRX Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -664.0, Cpd105: -664.0, Cpd74: -6.0, Cpd90: -12.0, Cpd393: -4.0, Cpd394: -11.0, Cpd396: -3.0, Cpd399: -10.0, Cpd402: -10.0, Cpd404: -8.0, Cpd408: -12.0, Cpd465: -11.0, Cpd467: -9.0, Cpd554: -4.0, Cpd1479: -3.0, Cpd1544: -2.0, Cpd1554: -9.0, Cpd1590: -15.0, Cpd1638: -2.0, Cpd1940: -6.0, Cpd2004: -12.0, Cpd2275: -17.0, Cpd107: 498.0, Cpd108: 498.0, Cpd150: 166.0, Cpd151: 166.0, Cpd2052: 1.0})
reaction.gene_reaction_rule = '( TXN2 )'
model.add_reactions([reaction])
reaction = Reaction('r1313')
reaction.name = '3-oxoacyl-ACP synthase, mitochondrial  Polyunsaturated fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1009: -1.0, Cpd1668: -1.0, Cpd60: 1.0, Cpd249: 1.0, Cpd2565: 1.0})
reaction.gene_reaction_rule = '( OXSM )'
model.add_reactions([reaction])
reaction = Reaction('r1314')
reaction.name = 'fatty acid synthase  Polyunsaturated fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2565: -1.0, Cpd25: 1.0, Cpd2566: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r1315')
reaction.name = 'fatty acid synthase  Polyunsaturated fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2566: -1.0, Cpd2: 1.0, Cpd2567: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r1316')
reaction.name = 'fatty acid synthase  Polyunsaturated fatty acid biosynthesis EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2567: -1.0, Cpd25: 1.0, Cpd2568: 1.0})
reaction.gene_reaction_rule = '( FASN )'
model.add_reactions([reaction])
reaction = Reaction('r1317')
reaction.name = 'oleoyl-ACP hydrolase  EC:2.3.1.85'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2568: -1.0, Cpd249: 1.0, Cpd1029: 1.0})
reaction.gene_reaction_rule = '( OLAH )'
model.add_reactions([reaction])
reaction = Reaction('r1374')
reaction.name = 'EC:3.2.1.14'
reaction.subsystem = 'Aminosugar metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd689: -2.0, Cpd2569: 3.0})
reaction.gene_reaction_rule = '( CHIL5 or CHIT1 )'
model.add_reactions([reaction])
reaction = Reaction('r1378')
reaction.name = 'S-(hydroxymethyl)glutathione dehydrogenase Methane metabolism EC:1.1.1.284'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2570: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd1058: 1.0})
reaction.gene_reaction_rule = '( ADH5 )'
model.add_reactions([reaction])
reaction = Reaction('r1380')
reaction.name = 'delta24-sterol reductase Biosynthesis of steroids EC:1.3.1.72'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd628: -1.0, Cpd589: 1.0, Cpd849: 1.0})
reaction.gene_reaction_rule = '( DHCR24 )'
model.add_reactions([reaction])
reaction = Reaction('r1381')
reaction.name = '5alpha-Cholest-7-en-3beta-ol delta7-delta8-isomerase Biosynthesis of steroids EC:5.3.3.5'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd851: -1.0, Cpd849: 1.0})
reaction.gene_reaction_rule = '( EBP )'
model.add_reactions([reaction])
reaction = Reaction('r1382')
reaction.name = 'folylpolyglutamyl synthetase  EC:6.3.2.17'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -6.0, Cpd105: -6.0, Cpd74: -6.0, Cpd377: -1.0, Cpd107: 6.0, Cpd108: 6.0, Cpd2571: 1.0})
reaction.gene_reaction_rule = '( FPGS )'
model.add_reactions([reaction])
reaction = Reaction('r1383')
reaction.name = 'gamma-glutamyl hydrolase  EC:3.4.19.9'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2571: -1.0, Cpd74: 6.0, Cpd377: 1.0})
reaction.gene_reaction_rule = '( GGH )'
model.add_reactions([reaction])
reaction = Reaction('r1391')
reaction.name = 'amylo-1,6-glucosidase, 4-alpha-glucanotransferase  EC:2.4.1.25'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1402: -1.0, Cpd2572: 1.0})
reaction.gene_reaction_rule = '( AGL )'
model.add_reactions([reaction])
reaction = Reaction('r1392')
reaction.name = 'amylo-1,6-glucosidase, 4-alpha-glucanotransferase  EC:3.2.1.33'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2572: -1.0, Cpd1290: 1.0, Cpd1403: 1.0})
reaction.gene_reaction_rule = '( AGL )'
model.add_reactions([reaction])
reaction = Reaction('r1393')
reaction.name = 'EC:2.4.1.1'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd108: -7.0, Cpd1403: -1.0, Cpd293: 7.0, Cpd1337: 1.0})
reaction.gene_reaction_rule = '( PYGL )'
model.add_reactions([reaction])
reaction = Reaction('r1402')
reaction.name = 'glycogenin  Protein assembly'
reaction.subsystem = 'Exchange/demand reaction'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1400.0, Cpd105: -1400.0, Cpd74: -15.0, Cpd90: -26.0, Cpd393: -12.0, Cpd394: -22.0, Cpd396: -6.0, Cpd399: -14.0, Cpd402: -18.0, Cpd404: -31.0, Cpd408: -25.0, Cpd465: -18.0, Cpd467: -12.0, Cpd554: -8.0, Cpd1479: -9.0, Cpd1544: -13.0, Cpd1554: -12.0, Cpd1590: -39.0, Cpd1638: -7.0, Cpd1940: -20.0, Cpd2004: -15.0, Cpd2275: -28.0, Cpd107: 1050.0, Cpd108: 1050.0, Cpd150: 350.0, Cpd151: 350.0, Cpd1337: 1.0})
reaction.gene_reaction_rule = '( GYG )'
model.add_reactions([reaction])
reaction = Reaction('r1411')
reaction.name = 'D-Galactosyl-N-acetyl-D-galactosaminyl-(N-acetylneuraminyl)-D- galactosyl-D-glucosylceramide galactohydrolase EC:3.2.1.23'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2573: -1.0, Cpd1266: 1.0, Cpd2574: 1.0})
reaction.gene_reaction_rule = '( GLB1 )'
model.add_reactions([reaction])
reaction = Reaction('r1418')
reaction.name = 'Carbonic acid hydro-lyase Nitrogen metabolism EC:4.2.1.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd38: -1.0, Cpd114: 1.0, Cpd2419: 1.0})
reaction.gene_reaction_rule = '( CAR1 or CAR12 or CAR13 or CAR14 or CAR2 or CAR4 or CAR6 or CAR7 or CAR8 or CAR9 )'
model.add_reactions([reaction])
reaction = Reaction('r1427')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1479: -1.0, Cpd7: 1.0, Cpd2518: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1431')
reaction.name = '2-Deoxyuridine 5-diphosphate:oxidized-thioredoxin 2-oxidoreductase Pyrimidine metabolism EC:1.17.4.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd896: -1.0, Cpd2218: -1.0, Cpd202: 1.0, Cpd2219: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('r1432')
reaction.name = '2-Deoxyuridine 5-diphosphate:oxidized-thioredoxin 2-oxidoreductase Pyrimidine metabolism EC:1.17.4.1'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd897: -1.0, Cpd2218: -1.0, Cpd285: 1.0, Cpd2219: 1.0})
reaction.gene_reaction_rule = '( RRM1 or RRM2 or RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('r1433')
reaction.name = 'NADPH:oxidized-thioredoxin oxidoreductase Pyrimidine metabolism EC:1.8.1.9'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd2219: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2218: 1.0})
reaction.gene_reaction_rule = '( TXNRD1 or TXNRD2 )'
model.add_reactions([reaction])
reaction = Reaction('r1443')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd192: -1.0, Cpd246: -1.0, Cpd247: 1.0, Cpd2428: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1444')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd194: -1.0, Cpd246: -1.0, Cpd247: 1.0, Cpd2462: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1445')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd190: -1.0, Cpd246: -1.0, Cpd247: 1.0, Cpd2468: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1446')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd241: -1.0, Cpd723: -1.0, Cpd70: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1447')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd2411: -1.0, Cpd752: 1.0, Cpd807: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1448')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd1063: -1.0, Cpd752: 1.0, Cpd2407: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1449')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd1061: -1.0, Cpd752: 1.0, Cpd2425: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1450')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd2471: -1.0, Cpd752: 1.0, Cpd2473: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1451')
reaction.name = 'EC:1.3.3.6'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd723: -1.0, Cpd2464: -1.0, Cpd752: 1.0, Cpd2467: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('r1453')
reaction.name = 'L-proline:(acceptor) oxidoreductase Arginine and proline metabolism EC:1.5.99.8'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd752: -1.0, Cpd1898: -1.0, Cpd7: 5.0, Cpd723: 1.0, Cpd1897: 1.0})
reaction.gene_reaction_rule = '( PRODH )'
model.add_reactions([reaction])
reaction = Reaction('r1454')
reaction.name = 'TCDB:2.A.1.13.5 TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd2532: -1.0, Cpd7: 1.0, Cpd34: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 or SLC16A7 )'
model.add_reactions([reaction])
reaction = Reaction('r1472')
reaction.name = 'long-chain-acyl-CoA dehydrogenase EC:1.3.99.13'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd609: -1.0, Cpd240: 1.0, Cpd2575: 1.0})
reaction.gene_reaction_rule = '( ACADL )'
model.add_reactions([reaction])
reaction = Reaction('r1474')
reaction.name = 'EC:4.2.1.17'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2575: -1.0, Cpd2576: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('r1477')
reaction.name = 'EC:1.1.1.35'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2576: -1.0, Cpd47: 1.0, Cpd2577: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('r1479')
reaction.name = 'EC:2.3.1.16'
reaction.subsystem = 'Unassigned'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2578: -1.0, Cpd178: 1.0, Cpd808: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r1481')
reaction.name = 'EC:2.3.1.16'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2577: -1.0, Cpd178: 1.0, Cpd2370: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACAA2 or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('r1487')
reaction.name = 'EC:6.2.1.3'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2547: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd2366: 1.0})
reaction.gene_reaction_rule = '( ACSBG2 or ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('r1492')
reaction.name = 'Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd487: -1.0, Cpd489: 1.0})
reaction.gene_reaction_rule = '( ABCC8 or ABCC9 or KCNJ11 or KCNJ8 )'
model.add_reactions([reaction])
reaction = Reaction('r1495')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2507: -1.0, Cpd41: 2.0, Cpd2423: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1497')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2511: -1.0, Cpd41: 2.0, Cpd2510: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1498')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2509: -1.0, Cpd41: 2.0, Cpd2498: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1499')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2579: -1.0, Cpd41: 2.0, Cpd2580: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1500')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2581: -1.0, Cpd41: 2.0, Cpd2582: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1501')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2583: -1.0, Cpd41: 2.0, Cpd2584: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1502')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2585: -1.0, Cpd41: 2.0, Cpd2586: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1503')
reaction.name = 'Glycoside-Pentoside-Hexuronide (GPH):Cation Symporter TCDB:2.A.28.1.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2587: -1.0, Cpd41: 2.0, Cpd2588: 1.0})
reaction.gene_reaction_rule = '( SLC10A1 )'
model.add_reactions([reaction])
reaction = Reaction('r1514')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd427: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd463: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1515')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1010: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1094: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1516')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1016: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1095: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1517')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1029: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2589: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1518')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1025: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1605: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1519')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd976: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd975: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1520')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1027: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1611: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1521')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1028: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1608: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1522')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1040: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1100: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1523')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1012: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1473: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1525')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd878: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd877: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1526')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2590: -1.0, Cpd107: 1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd2591: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1527')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2592: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2593: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1529')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.211.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1006: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2230: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('r1530')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1057: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( ABCC5 )'
model.add_reactions([reaction])
reaction = Reaction('r1531')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2594: -1.0, Cpd107: 1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( ABCC5 )'
model.add_reactions([reaction])
reaction = Reaction('r1532')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2596: -1.0, Cpd107: 1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( ABCC5 )'
model.add_reactions([reaction])
reaction = Reaction('r1533')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2598: -1.0, Cpd107: 1.0, Cpd33: 2.0, Cpd108: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( ABCC5 )'
model.add_reactions([reaction])
reaction = Reaction('r1536')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.7'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2379: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2600: 1.0})
reaction.gene_reaction_rule = '( ABCC4 )'
model.add_reactions([reaction])
reaction = Reaction('r1538')
reaction.name = 'Anion Exchanger (AE) TCDB:2.A.31.2.8'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd39: -2.0, Cpd40: 1.0, Cpd41: 2.0})
reaction.gene_reaction_rule = '( SLC4A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1540')
reaction.name = 'Neurotransmitter:Sodium Symporter (NSS) TCDB:2.A.22.3.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd698: -1.0, Cpd2189: -1.0, Cpd41: 1.0, Cpd700: 1.0, Cpd1586: 1.0})
reaction.gene_reaction_rule = '( SLC6A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1544')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd403: -1.0, Cpd401: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1546')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd554: -1.0, Cpd402: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1547')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd1638: -1.0, Cpd402: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1548')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd1940: -1.0, Cpd402: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1549')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd1544: -1.0, Cpd402: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1551')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd1590: -1.0, Cpd402: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1552')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd2004: -1.0, Cpd402: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1553')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd403: -1.0, Cpd395: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1554')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd2275: -1.0, Cpd402: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1556')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd1494: -1.0, Cpd402: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1557')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd403: -1.0, Cpd1554: -1.0, Cpd402: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1559')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd404: -1.0, Cpd394: 1.0, Cpd405: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1560')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd554: -1.0, Cpd394: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1561')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd1638: -1.0, Cpd394: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1562')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd1940: -1.0, Cpd394: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1563')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd1544: -1.0, Cpd394: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1564')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd396: -1.0, Cpd394: 1.0, Cpd397: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1565')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd1590: -1.0, Cpd394: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1566')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd2004: -1.0, Cpd394: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1567')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd393: -1.0, Cpd394: 1.0, Cpd395: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1568')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd2275: -1.0, Cpd394: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1569')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd408: -1.0, Cpd394: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1570')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd1494: -1.0, Cpd394: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1571')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd392: -1.0, Cpd1554: -1.0, Cpd394: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1573')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd554: -1.0, Cpd399: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1574')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd1638: -1.0, Cpd399: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1575')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd1940: -1.0, Cpd399: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1576')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd1544: -1.0, Cpd399: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1578')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd1590: -1.0, Cpd399: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1579')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd2004: -1.0, Cpd399: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1580')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd401: -1.0, Cpd395: 1.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1581')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd2275: -1.0, Cpd399: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1583')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd1494: -1.0, Cpd399: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1584')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd1554: -1.0, Cpd399: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1585')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd554: -1.0, Cpd404: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1586')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd1638: -1.0, Cpd404: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1587')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd1940: -1.0, Cpd404: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1588')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd1544: -1.0, Cpd404: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1589')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd405: -1.0, Cpd397: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1590')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd1590: -1.0, Cpd404: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1591')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd2004: -1.0, Cpd404: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1592')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd405: -1.0, Cpd395: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1593')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd2275: -1.0, Cpd404: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1594')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd408: -1.0, Cpd404: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1595')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd1494: -1.0, Cpd404: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1596')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd1554: -1.0, Cpd404: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1597')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd1638: -1.0, Cpd554: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1598')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd1940: -1.0, Cpd554: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1599')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd1544: -1.0, Cpd554: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1600')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd1677: -1.0, Cpd397: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1602')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd2004: -1.0, Cpd554: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1603')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd1677: -1.0, Cpd395: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1604')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd2275: -1.0, Cpd554: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1605')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd1677: -1.0, Cpd409: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1606')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd1494: -1.0, Cpd554: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1607')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1677: -1.0, Cpd1554: -1.0, Cpd554: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1608')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd1940: -1.0, Cpd1939: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1609')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd1544: -1.0, Cpd2238: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1610')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd2224: -1.0, Cpd397: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1611')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd1590: -1.0, Cpd1589: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1612')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd2004: -1.0, Cpd2013: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1613')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd2224: -1.0, Cpd395: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1614')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd2275: -1.0, Cpd2274: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1615')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd2224: -1.0, Cpd409: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1616')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd1494: -1.0, Cpd1493: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1617')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2224: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1618')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1939: -1.0, Cpd1544: -1.0, Cpd2238: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1619')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd1939: -1.0, Cpd397: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1620')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1939: -1.0, Cpd1590: -1.0, Cpd1589: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1621')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1939: -1.0, Cpd2004: -1.0, Cpd2013: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1622')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd1939: -1.0, Cpd395: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1623')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1939: -1.0, Cpd2275: -1.0, Cpd2274: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1624')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd1939: -1.0, Cpd409: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1625')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1939: -1.0, Cpd1494: -1.0, Cpd1493: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1626')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1939: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1627')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd2238: -1.0, Cpd397: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1628')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2238: -1.0, Cpd1590: -1.0, Cpd1589: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1629')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2238: -1.0, Cpd2004: -1.0, Cpd2013: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1630')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd2238: -1.0, Cpd395: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1631')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2238: -1.0, Cpd2275: -1.0, Cpd2274: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1632')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd2238: -1.0, Cpd409: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1633')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2238: -1.0, Cpd1494: -1.0, Cpd1493: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1634')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2238: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1635')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd1590: -1.0, Cpd396: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1636')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd2004: -1.0, Cpd396: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1637')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd397: -1.0, Cpd395: 1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1638')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd2275: -1.0, Cpd396: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1639')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd408: -1.0, Cpd396: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1640')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd1494: -1.0, Cpd396: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1641')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd1554: -1.0, Cpd396: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1642')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1589: -1.0, Cpd2004: -1.0, Cpd2013: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1643')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd1589: -1.0, Cpd395: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1644')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1589: -1.0, Cpd2275: -1.0, Cpd2274: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1645')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd1589: -1.0, Cpd409: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1646')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1589: -1.0, Cpd1494: -1.0, Cpd1493: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1647')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1589: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1648')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd2013: -1.0, Cpd395: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1649')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2013: -1.0, Cpd2275: -1.0, Cpd2274: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1650')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd2013: -1.0, Cpd409: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1651')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2013: -1.0, Cpd1494: -1.0, Cpd1493: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1652')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2013: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1653')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd395: -1.0, Cpd2275: -1.0, Cpd393: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1654')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd395: -1.0, Cpd408: -1.0, Cpd393: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1655')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd395: -1.0, Cpd1494: -1.0, Cpd393: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1656')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd395: -1.0, Cpd1554: -1.0, Cpd393: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1657')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd2274: -1.0, Cpd409: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1658')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2274: -1.0, Cpd1494: -1.0, Cpd1493: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1659')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2274: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1660')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd409: -1.0, Cpd1494: -1.0, Cpd408: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1661')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd409: -1.0, Cpd1554: -1.0, Cpd408: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1662')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1493: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('r1664')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd252: -1.0, Cpd468: -1.0, Cpd465: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1665')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd1479: -1.0, Cpd465: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1666')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd471: -1.0, Cpd465: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1667')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd252: -1.0, Cpd466: -1.0, Cpd467: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1668')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd1479: -1.0, Cpd467: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1669')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd471: -1.0, Cpd467: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1670')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd1479: -1.0, Cpd252: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1671')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd471: -1.0, Cpd1482: -1.0, Cpd1479: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1672')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd403: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1673')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd403: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1674')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd404: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd405: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1675')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd554: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1676')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1677')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1678')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1679')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd403: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1680')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1681')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1682')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd403: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1683')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1684')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd408: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1685')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1686')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1687')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd402: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1690')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd554: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1691')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1692')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1693')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1695')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1696')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1698')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1700')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1701')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1702')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd402: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1705')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd554: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1706')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1707')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1708')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1710')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1711')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1713')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1715')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1716')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1717')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd405: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1720')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd554: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1721')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1722')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1723')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1725')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1726')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1728')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1730')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1731')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1732')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1733')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1734')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1735')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1736')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1737')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1738')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1739')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1740')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1741')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1742')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1743')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1744')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1745')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1746')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1747')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1748')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1749')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1750')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1751')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1752')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd1939: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1753')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd2238: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1754')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1755')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd1589: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1756')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd2013: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1757')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1758')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd2274: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1759')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1760')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd1493: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1761')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd1553: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1762')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1763')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1764')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1765')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1766')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1767')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd2224: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1768')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd2238: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1769')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1770')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd1589: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1771')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd2013: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1772')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1773')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd2274: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1774')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1775')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd1493: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1776')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd1553: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1777')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1778')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1779')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1780')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1781')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1782')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd2224: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1783')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd1939: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1784')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1785')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd1589: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1786')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd2013: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1787')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1788')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd2274: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1789')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1790')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd1493: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1791')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd1553: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1792')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd402: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1796')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd554: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1797')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1798')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1799')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1800')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1801')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1803')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1805')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1806')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1807')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1808')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1809')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1810')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1811')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1812')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd2224: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1813')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd1939: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1814')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd2238: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1815')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1816')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd2013: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1817')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1818')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd2274: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1819')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1820')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd1493: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1821')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd1553: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1822')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1823')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1824')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1825')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1826')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1827')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd2224: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1828')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd1939: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1829')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd2238: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1830')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1831')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd1589: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1832')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1833')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd2274: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1834')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1835')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd1493: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1836')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd1553: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1837')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd402: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1841')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd554: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1842')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1843')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1844')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1846')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1847')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1848')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1850')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1851')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1852')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1853')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1854')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1855')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1856')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1857')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd2224: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1858')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd1939: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1859')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd2238: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1860')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1861')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd1589: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1862')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd2013: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1863')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1864')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1865')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd1493: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1866')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd1553: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1867')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1871')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd554: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1872')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1873')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1874')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1876')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1877')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1879')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1880')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1881')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1882')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1883')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1884')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1885')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1886')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1887')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd2224: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1888')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd1939: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1889')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd2238: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1890')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1891')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd1589: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1892')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd2013: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1893')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1894')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd2274: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1895')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1896')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd1554: -1.0, Cpd41: 1.0, Cpd1553: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1897')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd402: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd403: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1898')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd394: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd392: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1899')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd399: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd401: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1900')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd404: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd405: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1901')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd554: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd1677: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1902')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd1638: -1.0, Cpd41: 1.0, Cpd2224: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1903')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd1940: -1.0, Cpd41: 1.0, Cpd1939: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1904')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd1544: -1.0, Cpd41: 1.0, Cpd2238: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1905')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd396: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd397: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1906')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd1590: -1.0, Cpd41: 1.0, Cpd1589: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1907')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd2013: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1908')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd393: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd395: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1909')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd2275: -1.0, Cpd41: 1.0, Cpd2274: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1910')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd408: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd409: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1911')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd1494: -1.0, Cpd41: 1.0, Cpd1493: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1912')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd465: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1913')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd467: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1914')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd403: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1915')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1916')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd403: -1.0, Cpd471: -1.0, Cpd41: 1.0, Cpd402: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1917')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd465: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1918')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd467: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1919')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd392: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1920')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1921')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd471: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1922')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd465: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1923')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd467: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1924')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd401: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1925')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1926')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd401: -1.0, Cpd471: -1.0, Cpd41: 1.0, Cpd399: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1928')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd467: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1929')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd405: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1930')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1931')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd405: -1.0, Cpd471: -1.0, Cpd41: 1.0, Cpd404: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1932')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1933')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1934')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1935')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1677: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1936')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd1677: -1.0, Cpd41: 1.0, Cpd554: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1937')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1938')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1939')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1940')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2224: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1941')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd2224: -1.0, Cpd41: 1.0, Cpd1638: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1942')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1943')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1944')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1945')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1939: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1946')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd1939: -1.0, Cpd41: 1.0, Cpd1940: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1947')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1948')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1949')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1950')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2238: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1951')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd2238: -1.0, Cpd41: 1.0, Cpd1544: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1952')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd465: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1953')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd467: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1954')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd397: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1955')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1956')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd397: -1.0, Cpd471: -1.0, Cpd41: 1.0, Cpd396: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1957')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1958')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1959')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1960')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1589: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1961')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd1589: -1.0, Cpd41: 1.0, Cpd1590: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1962')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1963')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1964')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1965')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2013: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1966')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd2013: -1.0, Cpd41: 1.0, Cpd2004: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1967')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd465: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1968')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd467: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1969')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd395: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1970')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1971')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd395: -1.0, Cpd471: -1.0, Cpd41: 1.0, Cpd393: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1972')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1973')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1974')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1975')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd2274: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1976')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd2274: -1.0, Cpd41: 1.0, Cpd2499: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1977')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd465: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd468: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1978')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd467: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1979')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd409: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd1847: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1980')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1981')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd409: -1.0, Cpd471: -1.0, Cpd41: 1.0, Cpd408: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1982')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1983')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1984')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1985')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1493: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1986')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd1493: -1.0, Cpd41: 1.0, Cpd1494: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1987')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd465: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd468: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1988')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1989')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd252: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd1847: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1990')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd1553: -1.0, Cpd1479: -1.0, Cpd41: 1.0, Cpd1482: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1991')
reaction.name = 'Y+LAT2 Utilized transport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd471: -1.0, Cpd1553: -1.0, Cpd41: 1.0, Cpd1554: 1.0, Cpd2499: 1.0})
reaction.gene_reaction_rule = '( SLC7A6 )'
model.add_reactions([reaction])
reaction = Reaction('r1992')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd468: -1.0, Cpd403: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r1993')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd468: -1.0, Cpd392: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r1994')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd468: -1.0, Cpd401: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r1995')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd468: -1.0, Cpd405: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r1996')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd554: -1.0, Cpd465: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r1997')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd1638: -1.0, Cpd465: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r1998')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd1940: -1.0, Cpd465: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r1999')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd1544: -1.0, Cpd465: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2000')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd468: -1.0, Cpd397: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2001')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd1590: -1.0, Cpd465: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2002')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd2004: -1.0, Cpd465: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2003')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd468: -1.0, Cpd395: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2004')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd2275: -1.0, Cpd465: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2005')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd468: -1.0, Cpd409: 1.0, Cpd465: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2006')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd1494: -1.0, Cpd465: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2007')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd468: -1.0, Cpd1554: -1.0, Cpd465: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2008')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd466: -1.0, Cpd403: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2009')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd466: -1.0, Cpd392: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2010')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd466: -1.0, Cpd401: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2011')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd466: -1.0, Cpd405: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2012')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd554: -1.0, Cpd467: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2013')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd1638: -1.0, Cpd467: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2014')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd1940: -1.0, Cpd467: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2015')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd1544: -1.0, Cpd467: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2016')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd466: -1.0, Cpd397: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2017')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd1590: -1.0, Cpd467: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2018')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd2004: -1.0, Cpd467: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2019')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd466: -1.0, Cpd395: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2020')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd2275: -1.0, Cpd467: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2021')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd466: -1.0, Cpd409: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2022')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd1494: -1.0, Cpd467: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2023')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd466: -1.0, Cpd1554: -1.0, Cpd467: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2024')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd403: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2026')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd401: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2027')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd405: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2028')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd554: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2029')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd1638: -1.0, Cpd252: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2030')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd1940: -1.0, Cpd252: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2031')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd1544: -1.0, Cpd252: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2032')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd397: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2034')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd2004: -1.0, Cpd252: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2035')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd395: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2036')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd2275: -1.0, Cpd252: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2037')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd409: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2038')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd1494: -1.0, Cpd252: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2039')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd1554: -1.0, Cpd252: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2040')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd1482: -1.0, Cpd403: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2041')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd1482: -1.0, Cpd392: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2042')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd1482: -1.0, Cpd401: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2043')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd1482: -1.0, Cpd405: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2044')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd554: -1.0, Cpd1482: -1.0, Cpd1677: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2045')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd1638: -1.0, Cpd2224: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2046')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd1940: -1.0, Cpd1939: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2047')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd1544: -1.0, Cpd2238: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2048')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd1482: -1.0, Cpd397: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2049')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd1590: -1.0, Cpd1589: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2050')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd2004: -1.0, Cpd2013: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2051')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd1482: -1.0, Cpd395: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2052')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd2275: -1.0, Cpd2274: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2053')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd1482: -1.0, Cpd409: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2054')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd1494: -1.0, Cpd1493: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2055')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1482: -1.0, Cpd1554: -1.0, Cpd1553: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2056')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd402: -1.0, Cpd2499: -1.0, Cpd403: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2057')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd2499: -1.0, Cpd392: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2058')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd399: -1.0, Cpd2499: -1.0, Cpd401: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2059')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd404: -1.0, Cpd2499: -1.0, Cpd405: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2060')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd554: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd1677: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2061')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1638: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd2224: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2062')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1940: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2063')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1544: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd2238: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2064')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd396: -1.0, Cpd2499: -1.0, Cpd397: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2065')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1590: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2066')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2004: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2067')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd393: -1.0, Cpd2499: -1.0, Cpd395: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2068')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2499: -1.0, Cpd2275: -1.0, Cpd471: 1.0, Cpd2274: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2069')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd408: -1.0, Cpd2499: -1.0, Cpd409: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2070')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1494: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd1493: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2071')
reaction.name = 'Amino Acid-Polyamine-Organocation (APC) TCDB:2.A.3.8.15'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1554: -1.0, Cpd2499: -1.0, Cpd471: 1.0, Cpd1553: 1.0})
reaction.gene_reaction_rule = '( SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('r2073')
reaction.name = 'Zinc (Zn2+)-Iron (Fe2+) Permease (ZIP), TCDB:2.A.55.2.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd2601: -1.0, Cpd33: 1.0, Cpd2602: 1.0})
reaction.gene_reaction_rule = '( SLC11A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2079')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd148: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd185: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2080')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd148: -1.0, Cpd225: -1.0, Cpd185: 1.0, Cpd221: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2081')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd148: -1.0, Cpd701: -1.0, Cpd185: 1.0, Cpd699: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2082')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd148: -1.0, Cpd2603: -1.0, Cpd185: 1.0, Cpd206: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2083')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd148: -1.0, Cpd1600: -1.0, Cpd185: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2084')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd148: -1.0, Cpd2604: -1.0, Cpd185: 1.0, Cpd2605: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2085')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd36: -1.0, Cpd148: -1.0, Cpd31: 1.0, Cpd185: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2086')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd36: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2087')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd225: -1.0, Cpd36: 1.0, Cpd221: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2088')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd701: -1.0, Cpd36: 1.0, Cpd699: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2089')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd2603: -1.0, Cpd36: 1.0, Cpd206: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2090')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd1600: -1.0, Cpd36: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2091')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd585: -1.0, Cpd36: 1.0, Cpd583: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2092')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd2604: -1.0, Cpd36: 1.0, Cpd2605: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2093')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd383: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2094')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd221: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd225: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2095')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd699: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd701: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2096')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd206: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd2603: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2097')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2041: -1.0, Cpd1576: -1.0, Cpd21: 1.0, Cpd1600: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2098')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd583: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd585: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2099')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2041: -1.0, Cpd2605: -1.0, Cpd21: 1.0, Cpd2604: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2100')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2606: -1.0, Cpd2041: -1.0, Cpd21: 1.0, Cpd2607: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2101')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd221: -1.0, Cpd383: -1.0, Cpd30: 1.0, Cpd225: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2102')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd383: -1.0, Cpd699: -1.0, Cpd30: 1.0, Cpd701: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2103')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd206: -1.0, Cpd383: -1.0, Cpd30: 1.0, Cpd2603: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2104')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd148: -1.0, Cpd383: -1.0, Cpd30: 1.0, Cpd185: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2105')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd383: -1.0, Cpd1576: -1.0, Cpd30: 1.0, Cpd1600: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2106')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd383: -1.0, Cpd583: -1.0, Cpd30: 1.0, Cpd585: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2107')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd383: -1.0, Cpd2605: -1.0, Cpd30: 1.0, Cpd2604: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2108')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd383: -1.0, Cpd2606: -1.0, Cpd30: 1.0, Cpd2607: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2109')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd383: -1.0, Cpd30: 1.0, Cpd36: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2110')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd225: -1.0, Cpd699: -1.0, Cpd221: 1.0, Cpd701: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2111')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd206: -1.0, Cpd225: -1.0, Cpd221: 1.0, Cpd2603: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2112')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd225: -1.0, Cpd1576: -1.0, Cpd221: 1.0, Cpd1600: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2113')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd225: -1.0, Cpd583: -1.0, Cpd221: 1.0, Cpd585: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2114')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd225: -1.0, Cpd2605: -1.0, Cpd221: 1.0, Cpd2604: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2115')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd225: -1.0, Cpd2606: -1.0, Cpd221: 1.0, Cpd2607: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2116')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd206: -1.0, Cpd701: -1.0, Cpd699: 1.0, Cpd2603: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2117')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd701: -1.0, Cpd1576: -1.0, Cpd699: 1.0, Cpd1600: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2118')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd583: -1.0, Cpd701: -1.0, Cpd585: 1.0, Cpd699: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2119')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd701: -1.0, Cpd2605: -1.0, Cpd699: 1.0, Cpd2604: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2120')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2606: -1.0, Cpd701: -1.0, Cpd2607: 1.0, Cpd699: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2121')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2603: -1.0, Cpd1576: -1.0, Cpd206: 1.0, Cpd1600: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2122')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd583: -1.0, Cpd2603: -1.0, Cpd206: 1.0, Cpd585: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2123')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2603: -1.0, Cpd2605: -1.0, Cpd206: 1.0, Cpd2604: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2124')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2606: -1.0, Cpd2603: -1.0, Cpd206: 1.0, Cpd2607: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2125')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd185: -1.0, Cpd583: -1.0, Cpd148: 1.0, Cpd585: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2126')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd185: -1.0, Cpd2606: -1.0, Cpd148: 1.0, Cpd2607: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2127')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd583: -1.0, Cpd1600: -1.0, Cpd585: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2128')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1600: -1.0, Cpd2605: -1.0, Cpd2604: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2129')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2606: -1.0, Cpd1600: -1.0, Cpd2607: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2130')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd585: -1.0, Cpd2605: -1.0, Cpd583: 1.0, Cpd2604: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2131')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2606: -1.0, Cpd585: -1.0, Cpd2607: 1.0, Cpd583: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2132')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2606: -1.0, Cpd2604: -1.0, Cpd2607: 1.0, Cpd2605: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2133')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd31: -1.0, Cpd2607: -1.0, Cpd36: 1.0, Cpd2606: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2136')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.14.6'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd494: -1.0, Cpd108: 1.0, Cpd41: 1.0})
reaction.gene_reaction_rule = '( SLC17A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2137')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.14.6'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd698: -1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC17A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2139')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd678: -1.0, Cpd1057: -1.0, Cpd38: 1.0, Cpd679: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2140')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd1304: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1305: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2141')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2191: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2192: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2142')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2511: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2510: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2143')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2608: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2609: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2144')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2579: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2580: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2145')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2581: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2582: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2146')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2610: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2611: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2147')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2583: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2584: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2148')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2585: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2586: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2149')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2612: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2613: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2150')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2614: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2615: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2151')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd678: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd679: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2152')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd678: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd679: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2153')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd678: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd679: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2154')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1304: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd1305: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2155')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1304: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd1305: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2156')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1304: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd1305: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2157')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2507: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2423: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2158')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2507: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2423: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2159')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2507: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2423: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2160')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2191: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2192: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2161')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2191: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2192: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2162')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2191: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2192: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2163')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2511: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2510: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2164')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2511: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2510: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2165')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2511: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2510: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2166')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2608: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2609: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2167')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2608: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2609: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2168')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2608: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2609: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2169')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2579: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2580: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2170')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2579: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2580: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2171')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2579: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2580: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2172')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2581: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2582: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2173')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2581: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2582: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2174')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2581: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2582: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2175')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2610: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2611: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2176')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2610: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2611: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2177')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2610: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2611: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2178')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2583: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2584: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2179')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2583: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2584: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2180')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2583: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2584: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2181')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2585: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2586: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2182')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2585: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2586: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2183')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2585: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2586: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2184')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2612: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2613: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2185')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2612: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2613: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2186')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2612: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2613: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2187')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2614: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2615: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2188')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2614: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2615: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2189')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2614: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2615: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2190')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2587: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2588: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2191')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2587: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2588: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2192')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2587: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2588: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2193')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd561: -1.0, Cpd1057: -1.0, Cpd38: 1.0, Cpd560: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2194')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2212: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1543: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2195')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2223: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1542: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2196')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2616: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2617: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2197')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2011: -1.0, Cpd38: 1.0, Cpd647: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2198')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd1587: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1588: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2199')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd555: -1.0, Cpd1057: -1.0, Cpd38: 1.0, Cpd556: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2200')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2618: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2619: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2201')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2620: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2621: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2202')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd557: -1.0, Cpd1057: -1.0, Cpd38: 1.0, Cpd558: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2203')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd561: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd560: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2204')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd561: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd560: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2205')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd561: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd560: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2206')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2212: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd1543: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2207')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2212: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd1543: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2208')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2212: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd1543: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2209')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2223: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd1542: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2210')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2223: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd1542: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2211')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2223: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd1542: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2212')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2616: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2617: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2213')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2616: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2617: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2214')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2616: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2617: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2215')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2011: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd647: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2216')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2011: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd647: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2217')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2011: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd647: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2218')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd463: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd427: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2219')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd463: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd427: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2220')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd463: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd427: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2221')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1587: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd1588: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2222')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1587: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd1588: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2223')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1587: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd1588: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2224')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd990: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd991: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2225')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd990: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd991: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2226')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd990: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd991: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2227')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd860: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd861: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2228')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd860: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd861: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2229')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd860: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd861: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2230')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd555: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd556: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2231')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd555: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd556: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2232')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd555: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd556: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2233')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2622: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2413: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2234')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2622: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2413: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2235')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2622: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2413: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2236')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2623: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2624: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2237')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2623: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2624: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2238')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2623: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2624: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2239')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2618: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2619: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2240')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2618: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2619: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2241')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2618: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2619: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2242')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2625: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd432: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2243')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2625: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd432: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2244')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2625: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd432: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2245')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2626: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd1635: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2246')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2626: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd1635: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2247')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2626: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd1635: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2248')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2620: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2621: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2249')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2620: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2621: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2250')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2620: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2621: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2251')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2627: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2628: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2252')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2627: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2628: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2253')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2627: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2628: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2254')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd557: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd558: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2255')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd557: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd558: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2256')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd557: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd558: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2257')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2629: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2630: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2258')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2629: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2630: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2259')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2629: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2630: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2260')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2631: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2632: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2261')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2631: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2632: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2262')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2631: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2632: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2263')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2633: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2634: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2264')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2633: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2634: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2265')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2633: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2634: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2266')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2635: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2636: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2267')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2635: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2636: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2268')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2635: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2636: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2269')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2637: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2638: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2270')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2637: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2638: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2271')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2637: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2638: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2272')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2639: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2640: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2273')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2639: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2640: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2274')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2639: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2640: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2275')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2641: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2642: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2276')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2641: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2642: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2277')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2641: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2642: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2278')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2009: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd1930: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2279')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2009: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd1930: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2280')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2009: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd1930: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2281')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2643: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2644: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2282')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2643: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2644: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2283')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2643: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2644: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2284')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2010: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd646: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2285')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2010: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd646: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2286')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2010: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd646: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2287')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2645: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2646: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2288')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2645: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2646: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2289')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2645: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2646: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2290')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2647: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2648: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2291')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2647: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2648: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2292')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2647: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2648: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2293')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2012: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd648: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2294')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2012: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd648: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2295')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2012: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd648: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2296')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2649: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2650: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2297')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2649: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2650: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2298')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2649: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2650: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2299')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2594: -1.0, Cpd2651: -1.0, Cpd38: 1.0, Cpd2595: 1.0, Cpd2652: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2300')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2596: -1.0, Cpd2651: -1.0, Cpd38: 1.0, Cpd2597: 1.0, Cpd2652: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2301')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2598: -1.0, Cpd2651: -1.0, Cpd38: 1.0, Cpd2599: 1.0, Cpd2652: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2302')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2653: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd1929: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2303')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2653: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd1929: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2304')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2653: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd1929: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2305')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2654: -1.0, Cpd2594: -1.0, Cpd38: 1.0, Cpd2655: 1.0, Cpd2595: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2306')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2654: -1.0, Cpd2596: -1.0, Cpd38: 1.0, Cpd2655: 1.0, Cpd2597: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2307')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.5'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2654: -1.0, Cpd2598: -1.0, Cpd38: 1.0, Cpd2655: 1.0, Cpd2599: 1.0})
reaction.gene_reaction_rule = '( SLCO1B2 )'
model.add_reactions([reaction])
reaction = Reaction('r2308')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd1484: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1481: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2309')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd1812: -1.0, Cpd38: 1.0, Cpd98: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2310')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2160: -1.0, Cpd38: 1.0, Cpd116: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2311')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd224: -1.0, Cpd1057: -1.0, Cpd38: 1.0, Cpd227: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2312')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd935: -1.0, Cpd38: 1.0, Cpd75: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2313')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd463: -1.0, Cpd1057: -1.0, Cpd38: 1.0, Cpd427: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2314')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2622: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2413: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2315')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2623: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2624: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2316')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2625: -1.0, Cpd38: 1.0, Cpd432: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2317')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2626: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1635: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2318')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2627: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2628: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2319')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2629: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2630: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2320')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2631: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2632: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2321')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2633: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2634: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2322')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2635: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2636: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2323')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2637: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2638: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2324')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2639: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2640: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2325')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2641: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2642: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2326')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2009: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1930: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2327')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2643: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2644: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2328')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2010: -1.0, Cpd38: 1.0, Cpd646: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2329')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2645: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2646: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2330')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2647: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2648: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2331')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2012: -1.0, Cpd38: 1.0, Cpd648: 1.0, Cpd1443: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2332')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2649: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2650: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2333')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2651: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2652: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2334')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2653: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd1929: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2335')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd1057: -1.0, Cpd2654: -1.0, Cpd38: 1.0, Cpd1443: 1.0, Cpd2655: 1.0})
reaction.gene_reaction_rule = '( SLCO2A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2338')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.19.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd224: -1.0, Cpd227: 1.0})
reaction.gene_reaction_rule = '( SLC22A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2370')
reaction.name = 'Nucleobase:Cation Symporter-2 (NCS2) TCDB:2.A.40.6.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd479: -1.0, Cpd41: 2.0, Cpd480: 1.0})
reaction.gene_reaction_rule = '( SLC23A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2371')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd254: -1.0, Cpd2656: -1.0, Cpd228: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2372')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd255: -1.0, Cpd2656: -1.0, Cpd253: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2373')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd254: -1.0, Cpd28: 1.0, Cpd228: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2374')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd254: -1.0, Cpd704: -1.0, Cpd228: 1.0, Cpd2657: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2375')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd255: -1.0, Cpd543: -1.0, Cpd253: 1.0, Cpd1843: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2376')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd255: -1.0, Cpd704: -1.0, Cpd253: 1.0, Cpd2657: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2377')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd2338: -1.0, Cpd28: 1.0, Cpd2656: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2378')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd543: -1.0, Cpd2338: -1.0, Cpd1843: 1.0, Cpd2656: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2379')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd381: -1.0, Cpd2338: -1.0, Cpd382: 1.0, Cpd2656: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2380')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd704: -1.0, Cpd2338: -1.0, Cpd2656: 1.0, Cpd2657: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2381')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd28: -1.0, Cpd228: -1.0, Cpd7: 1.0, Cpd30: 1.0, Cpd254: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2382')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd228: -1.0, Cpd1843: -1.0, Cpd7: 1.0, Cpd254: 1.0, Cpd543: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2384')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd228: -1.0, Cpd2657: -1.0, Cpd7: 1.0, Cpd254: 1.0, Cpd704: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2385')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd28: -1.0, Cpd253: -1.0, Cpd7: 1.0, Cpd30: 1.0, Cpd255: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2386')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd253: -1.0, Cpd1843: -1.0, Cpd7: 1.0, Cpd255: 1.0, Cpd543: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2387')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd253: -1.0, Cpd382: -1.0, Cpd7: 1.0, Cpd255: 1.0, Cpd381: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2388')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd253: -1.0, Cpd2657: -1.0, Cpd7: 1.0, Cpd255: 1.0, Cpd704: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2389')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd28: -1.0, Cpd2656: -1.0, Cpd7: 1.0, Cpd30: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2390')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd1843: -1.0, Cpd2656: -1.0, Cpd7: 1.0, Cpd543: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2391')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd382: -1.0, Cpd2656: -1.0, Cpd7: 1.0, Cpd381: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2392')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd2656: -1.0, Cpd2657: -1.0, Cpd7: 1.0, Cpd704: 1.0, Cpd2338: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2393')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd231: -1.0, Cpd255: -1.0, Cpd253: 1.0, Cpd693: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2394')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.7.2'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd231: -1.0, Cpd2338: -1.0, Cpd693: 1.0, Cpd2656: 1.0})
reaction.gene_reaction_rule = '( SLC25A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2395')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd576: -1.0, Cpd467: -1.0, Cpd7: 1.0, Cpd465: 1.0, Cpd464: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2396')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd252: -1.0, Cpd576: -1.0, Cpd7: 1.0, Cpd465: 1.0, Cpd470: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2397')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd576: -1.0, Cpd1479: -1.0, Cpd7: 1.0, Cpd465: 1.0, Cpd2518: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2398')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd576: -1.0, Cpd471: -1.0, Cpd7: 1.0, Cpd465: 1.0, Cpd691: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2399')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd465: -1.0, Cpd464: -1.0, Cpd7: 1.0, Cpd576: 1.0, Cpd467: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2400')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd252: -1.0, Cpd464: -1.0, Cpd7: 1.0, Cpd467: 1.0, Cpd470: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2401')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd464: -1.0, Cpd1479: -1.0, Cpd7: 1.0, Cpd467: 1.0, Cpd2518: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2402')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd464: -1.0, Cpd471: -1.0, Cpd7: 1.0, Cpd467: 1.0, Cpd691: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2403')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd465: -1.0, Cpd470: -1.0, Cpd7: 1.0, Cpd252: 1.0, Cpd576: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2404')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd467: -1.0, Cpd470: -1.0, Cpd7: 1.0, Cpd252: 1.0, Cpd464: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2405')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd470: -1.0, Cpd1479: -1.0, Cpd7: 1.0, Cpd252: 1.0, Cpd2518: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2406')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd465: -1.0, Cpd2518: -1.0, Cpd7: 1.0, Cpd576: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2407')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd467: -1.0, Cpd2518: -1.0, Cpd7: 1.0, Cpd464: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2408')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd252: -1.0, Cpd2518: -1.0, Cpd7: 1.0, Cpd470: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2409')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd471: -1.0, Cpd2518: -1.0, Cpd7: 1.0, Cpd691: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2410')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd465: -1.0, Cpd691: -1.0, Cpd7: 1.0, Cpd576: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2411')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd467: -1.0, Cpd691: -1.0, Cpd7: 1.0, Cpd464: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2412')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd252: -1.0, Cpd691: -1.0, Cpd7: 1.0, Cpd470: 1.0, Cpd471: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2413')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd691: -1.0, Cpd1479: -1.0, Cpd7: 1.0, Cpd471: 1.0, Cpd2518: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2416')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.19.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd2518: -1.0, Cpd7: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC25A2 )'
model.add_reactions([reaction])
reaction = Reaction('r2419')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.7'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd204: -1.0, Cpd108: 1.0, Cpd28: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r2420')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.7'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd204: -1.0, Cpd704: -1.0, Cpd108: 1.0, Cpd2657: 1.0})
reaction.gene_reaction_rule = '( SLC25A10 )'
model.add_reactions([reaction])
reaction = Reaction('r2425')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.16.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd202: -1.0, Cpd771: -1.0, Cpd107: 1.0, Cpd897: 1.0})
reaction.gene_reaction_rule = '( SLC25A19 )'
model.add_reactions([reaction])
reaction = Reaction('r2433')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.8.3'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd265: -1.0, Cpd307: -1.0, Cpd266: 1.0, Cpd303: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('r2434')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.8.3'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd307: -1.0, Cpd2508: -1.0, Cpd303: 1.0, Cpd2409: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('r2435')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.8.3'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd307: -1.0, Cpd591: -1.0, Cpd303: 1.0, Cpd592: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('r2436')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.8.3'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd307: -1.0, Cpd2658: -1.0, Cpd303: 1.0, Cpd2659: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('r2437')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.8.3'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd307: -1.0, Cpd2660: -1.0, Cpd303: 1.0, Cpd2661: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('r2438')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.8.3'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd307: -1.0, Cpd2662: -1.0, Cpd303: 1.0, Cpd2663: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('r2439')
reaction.name = 'Zinc (Zn2+)-Iron (Fe2+) Permease (ZIP), TCDB:2.A.53.2.2'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd40: -1.0, Cpd2132: -1.0, Cpd38: 1.0, Cpd1163: 1.0})
reaction.gene_reaction_rule = '( SLC26A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2447')
reaction.name = 'Concentrative Nucleoside Transporter (CNT) TCDB:2.A.41.2.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd956: -1.0, Cpd41: 1.0, Cpd818: 1.0})
reaction.gene_reaction_rule = '( SLC28A1 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('r2449')
reaction.name = 'Concentrative Nucleoside Transporter (CNT) TCDB:2.A.41.2.4'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd796: -1.0, Cpd41: 1.0, Cpd794: 1.0})
reaction.gene_reaction_rule = '( SLC28A2 or SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('r2465')
reaction.name = 'Cation Diffusion Facilitator (CDF) TCDB:2.A.4.2.3'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2602: -1.0, Cpd2601: 1.0})
reaction.gene_reaction_rule = '( SLC30A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2471')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.18.8.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd405: -1.0, Cpd33: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2472')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.4.7'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd204: -1.0, Cpd1198: -1.0, Cpd108: 1.0, Cpd1415: 1.0})
reaction.gene_reaction_rule = '( SLC37A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2473')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.4.5'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd915: -1.0, Cpd1202: -1.0, Cpd108: 1.0, Cpd642: 1.0})
reaction.gene_reaction_rule = '( SLC37A4 )'
model.add_reactions([reaction])
reaction = Reaction('r2482')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd1481: -1.0, Cpd2595: -1.0, Cpd40: 1.0, Cpd1484: 1.0, Cpd2594: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2483')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd1481: -1.0, Cpd2597: -1.0, Cpd40: 1.0, Cpd1484: 1.0, Cpd2596: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2484')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd1481: -1.0, Cpd2599: -1.0, Cpd40: 1.0, Cpd1484: 1.0, Cpd2598: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2485')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd98: -1.0, Cpd2595: -1.0, Cpd40: 1.0, Cpd1812: 1.0, Cpd2594: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2486')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd98: -1.0, Cpd2597: -1.0, Cpd40: 1.0, Cpd1812: 1.0, Cpd2596: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2487')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd98: -1.0, Cpd2599: -1.0, Cpd40: 1.0, Cpd1812: 1.0, Cpd2598: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2488')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd116: -1.0, Cpd2595: -1.0, Cpd40: 1.0, Cpd2160: 1.0, Cpd2594: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2489')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd116: -1.0, Cpd2597: -1.0, Cpd40: 1.0, Cpd2160: 1.0, Cpd2596: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2490')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd116: -1.0, Cpd2599: -1.0, Cpd40: 1.0, Cpd2160: 1.0, Cpd2598: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2491')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd227: -1.0, Cpd2595: -1.0, Cpd40: 1.0, Cpd224: 1.0, Cpd2594: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2492')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd227: -1.0, Cpd2597: -1.0, Cpd40: 1.0, Cpd224: 1.0, Cpd2596: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2493')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd227: -1.0, Cpd2599: -1.0, Cpd40: 1.0, Cpd224: 1.0, Cpd2598: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2494')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd75: -1.0, Cpd2595: -1.0, Cpd40: 1.0, Cpd935: 1.0, Cpd2594: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2495')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd75: -1.0, Cpd2597: -1.0, Cpd40: 1.0, Cpd935: 1.0, Cpd2596: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2496')
reaction.name = 'Resistance-Nodulation-Cell Division (RND) TCDB:2.A.60.1.14'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd75: -1.0, Cpd2599: -1.0, Cpd40: 1.0, Cpd935: 1.0, Cpd2598: 1.0})
reaction.gene_reaction_rule = '( SLCO1A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2497')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.3'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd427: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2336: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('r2498')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.3'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1010: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2024: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('r2499')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.3'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd207: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd742: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('r2501')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.3'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2406: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd529: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('r2502')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.3'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd828: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2361: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('r2503')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.3'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd610: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1080: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('r2505')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.208.8'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2615: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2614: 1.0})
reaction.gene_reaction_rule = '( ABCC1 )'
model.add_reactions([reaction])
reaction = Reaction('r2516')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.13.1'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd2321: -1.0, Cpd33: 1.0, Cpd1576: 1.0})
reaction.gene_reaction_rule = '( SLC16A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2517')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.1'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2664: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2286: 1.0})
reaction.gene_reaction_rule = '( ABCD3 )'
model.add_reactions([reaction])
reaction = Reaction('r2518')
reaction.name = 'ATP-binding Cassette (ABC) TCDB:3.A.1.203.1'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd2440: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2282: 1.0})
reaction.gene_reaction_rule = '( ABCD3 )'
model.add_reactions([reaction])
reaction = Reaction('r2520')
reaction.name = 'Mitochondrial Carrier (MC) TCDB:2.A.29.2.1'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd1057: -1.0, Cpd30: 1.0, Cpd1418: 1.0})
reaction.gene_reaction_rule = '( SLC25A11 )'
model.add_reactions([reaction])
reaction = Reaction('r2525')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.44.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd401: -1.0, Cpd399: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2526')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.44.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd405: -1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2532')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.44.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd395: -1.0, Cpd393: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2534')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.44.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd409: -1.0, Cpd408: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 )'
model.add_reactions([reaction])
reaction = Reaction('r2535')
reaction.name = 'Major Facilitator(MFS) TCDB:2.A.1.44.1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1493: -1.0, Cpd1494: 1.0})
reaction.gene_reaction_rule = '( SLC43A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0066M')
reaction.name = 'RE0066'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd442: -1.0, Cpd2665: -1.0, Cpd7: 2.0, Cpd441: 1.0, Cpd2666: 1.0})
reaction.gene_reaction_rule = '( PEMT )'
model.add_reactions([reaction])
reaction = Reaction('RE0066R')
reaction.name = 'RE0066'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1926: -1.0, Cpd2667: -1.0, Cpd1927: 1.0, Cpd565: 2.0, Cpd2668: 1.0})
reaction.gene_reaction_rule = '( PEMT )'
model.add_reactions([reaction])
reaction = Reaction('RE0344C')
reaction.name = 'RE0344'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd602: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd1014: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0344M')
reaction.name = 'RE0344'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd605: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2669: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE0344X')
reaction.name = 'RE0344'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd1067: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2670: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0383C')
reaction.name = 'RE0383'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2671: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd1401: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0453M')
reaction.name = 'RE0453'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd902: -1.0, Cpd898: -1.0, Cpd896: 1.0, Cpd906: 1.0})
reaction.gene_reaction_rule = '( NME4 )'
model.add_reactions([reaction])
reaction = Reaction('RE0453N')
reaction.name = 'RE0453'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd782: -1.0, Cpd1784: -1.0, Cpd783: 1.0, Cpd1785: 1.0})
reaction.gene_reaction_rule = '( GM20390 or NME2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0456N')
reaction.name = 'RE0456'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd784: -1.0, Cpd2672: -1.0, Cpd570: 1.0, Cpd957: 1.0, Cpd2673: 1.0})
reaction.gene_reaction_rule = '( RRM2B )'
model.add_reactions([reaction])
reaction = Reaction('RE0512M')
reaction.name = 'RE0512'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2674: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2675: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HADHB or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE0512X')
reaction.name = 'RE0512'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2676: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd2677: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE0565C')
reaction.name = 'RE0565'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd602: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd2678: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE0568C')
reaction.name = 'RE0568'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2679: -1.0, Cpd25: 1.0, Cpd459: 1.0})
reaction.gene_reaction_rule = '( RPL14 )'
model.add_reactions([reaction])
reaction = Reaction('RE0569C')
reaction.name = 'RE0569'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd459: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd2680: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE0569E')
reaction.name = 'RE0569'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd2681: -1.0, Cpd2682: -1.0, Cpd2419: 1.0, Cpd2683: 1.0, Cpd2684: 1.0})
reaction.gene_reaction_rule = '( LOXL2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0572N')
reaction.name = 'RE0572'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd2150: -1.0, Cpd2685: -1.0, Cpd2151: 1.0, Cpd2686: 1.0})
reaction.gene_reaction_rule = '( CINP )'
model.add_reactions([reaction])
reaction = Reaction('RE0573N')
reaction.name = 'RE0573'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd2686: -1.0, Cpd2687: -1.0, Cpd676: 1.0, Cpd2688: 1.0, Cpd2689: 1.0})
reaction.gene_reaction_rule = '( HIRIP3 )'
model.add_reactions([reaction])
reaction = Reaction('RE0577C')
reaction.name = 'RE0577'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd459: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd1031: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0577M')
reaction.name = 'RE0577'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd462: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2690: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE0577X')
reaction.name = 'RE0577'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd1076: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2691: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0578C')
reaction.name = 'RE0578'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2692: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd2693: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0578M')
reaction.name = 'RE0578'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2694: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2695: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE0578X')
reaction.name = 'RE0578'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2696: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2697: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0579C')
reaction.name = 'RE0579'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2698: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd1040: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0579M')
reaction.name = 'RE0579'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2699: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2700: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE0579X')
reaction.name = 'RE0579'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd198: -1.0, Cpd261: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2701: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE0580L')
reaction.name = 'RE0580'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd2702: -1.0, Cpd2703: -1.0, Cpd1574: 1.0, Cpd2704: 1.0, Cpd2705: 1.0})
reaction.gene_reaction_rule = '( CTSA )'
model.add_reactions([reaction])
reaction = Reaction('RE0580R')
reaction.name = 'RE0580'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2337: -1.0, Cpd2352: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd2706: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE0581R')
reaction.name = 'RE0581'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd2706: -1.0, Cpd589: 1.0, Cpd2707: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0582N')
reaction.name = 'RE0582'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2708: -1.0, Cpd570: 1.0, Cpd2709: 1.0})
reaction.gene_reaction_rule = '( HIRIP3 )'
model.add_reactions([reaction])
reaction = Reaction('RE0583N')
reaction.name = 'RE0583'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd2150: -1.0, Cpd2709: -1.0, Cpd2151: 1.0, Cpd2710: 1.0})
reaction.gene_reaction_rule = '( ANAPC11 )'
model.add_reactions([reaction])
reaction = Reaction('RE0688C')
reaction.name = 'RE0688'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2149: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd2711: 1.0})
reaction.gene_reaction_rule = '( AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('RE0688E')
reaction.name = 'RE0688'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2712: -1.0, Cpd2143: -1.0, Cpd278: 1.0, Cpd1444: 1.0, Cpd2713: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0688X')
reaction.name = 'RE0688'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2714: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd2715: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0689C')
reaction.name = 'RE0689'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2140: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd2716: 1.0})
reaction.gene_reaction_rule = '( AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('RE0689E')
reaction.name = 'RE0689'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2143: -1.0, Cpd2717: -1.0, Cpd278: 1.0, Cpd1444: 1.0, Cpd2718: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0689X')
reaction.name = 'RE0689'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2719: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd2720: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0690C')
reaction.name = 'RE0690'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2716: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd2721: 1.0})
reaction.gene_reaction_rule = '( AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('RE0690E')
reaction.name = 'RE0690'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2143: -1.0, Cpd2718: -1.0, Cpd278: 1.0, Cpd1444: 1.0, Cpd2722: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0690X')
reaction.name = 'RE0690'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2720: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd2723: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0702C')
reaction.name = 'RE0702'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd33: -1.0, Cpd1672: -1.0, Cpd2: 1.0, Cpd2724: 1.0})
reaction.gene_reaction_rule = '( PRDX6 or TPO )'
model.add_reactions([reaction])
reaction = Reaction('RE0702E')
reaction.name = 'RE0702'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd1444: -1.0, Cpd2725: -1.0, Cpd114: 1.0, Cpd2726: 1.0})
reaction.gene_reaction_rule = '( LPO )'
model.add_reactions([reaction])
reaction = Reaction('RE0702L')
reaction.name = 'RE0702'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd1995: -1.0, Cpd2727: -1.0, Cpd86: 1.0, Cpd2728: 1.0})
reaction.gene_reaction_rule = '( MPO or PRDX6 )'
model.add_reactions([reaction])
reaction = Reaction('RE0702M')
reaction.name = 'RE0702'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd634: -1.0, Cpd2729: -1.0, Cpd12: 1.0, Cpd2730: 1.0})
reaction.gene_reaction_rule = '( BCKDHA or BCKDHB or DLD )'
model.add_reactions([reaction])
reaction = Reaction('RE0702N')
reaction.name = 'RE0702'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd2146: -1.0, Cpd2731: -1.0, Cpd570: 1.0, Cpd2732: 1.0})
reaction.gene_reaction_rule = '( MPO )'
model.add_reactions([reaction])
reaction = Reaction('RE0827C')
reaction.name = 'RE0827'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2711: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd2733: 1.0})
reaction.gene_reaction_rule = '( AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('RE0827E')
reaction.name = 'RE0827'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2143: -1.0, Cpd2713: -1.0, Cpd278: 1.0, Cpd1444: 1.0, Cpd2734: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0827X')
reaction.name = 'RE0827'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2715: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd2735: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0828C')
reaction.name = 'RE0828'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2140: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd2736: 1.0})
reaction.gene_reaction_rule = '( AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('RE0828E')
reaction.name = 'RE0828'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2143: -1.0, Cpd2717: -1.0, Cpd278: 1.0, Cpd1444: 1.0, Cpd2737: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0828X')
reaction.name = 'RE0828'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2719: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd2738: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0830N')
reaction.name = 'RE0830'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2739: -1.0, Cpd570: 1.0, Cpd2740: 1.0})
reaction.gene_reaction_rule = '( PCBD1 or PCBD2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0912C')
reaction.name = 'RE0912'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd2741: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd2742: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 or SULT4A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0915E')
reaction.name = 'RE0915'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2743: -1.0, Cpd446: 1.0, Cpd1696: 1.0})
reaction.gene_reaction_rule = '( AMY2A5 or LYZL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0919C')
reaction.name = 'RE0919'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2744: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2745: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0919R')
reaction.name = 'RE0919'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2746: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2747: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0920C')
reaction.name = 'RE0920'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd1543: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2748: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0920R')
reaction.name = 'RE0920'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2749: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2750: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0921C')
reaction.name = 'RE0921'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2751: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2752: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0921R')
reaction.name = 'RE0921'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2753: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2754: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0922C')
reaction.name = 'RE0922'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd1542: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2755: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0922R')
reaction.name = 'RE0922'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2756: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2757: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0923C')
reaction.name = 'RE0923'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2741: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2758: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0923R')
reaction.name = 'RE0923'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2759: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2760: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0924C')
reaction.name = 'RE0924'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2761: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2762: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0924R')
reaction.name = 'RE0924'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2763: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2764: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0925C')
reaction.name = 'RE0925'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2765: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2766: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0925R')
reaction.name = 'RE0925'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2767: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2768: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0926E')
reaction.name = 'RE0926'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2769: -1.0, Cpd446: 1.0, Cpd2743: 1.0})
reaction.gene_reaction_rule = '( AMY2A5 or LYZL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0927C')
reaction.name = 'RE0927'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2770: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2771: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0927R')
reaction.name = 'RE0927'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2772: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2773: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0928C')
reaction.name = 'RE0928'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2774: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2775: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0928R')
reaction.name = 'RE0928'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2776: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2777: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE0935E')
reaction.name = 'RE0935'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2778: -1.0, Cpd446: 1.0, Cpd2769: 1.0})
reaction.gene_reaction_rule = '( AMY2A5 or LYZL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0936C')
reaction.name = 'RE0936'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2779: -1.0, Cpd1554: 1.0, Cpd2780: 1.0})
reaction.gene_reaction_rule = '( ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0936E')
reaction.name = 'RE0936'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2781: -1.0, Cpd1553: 1.0, Cpd2782: 1.0})
reaction.gene_reaction_rule = '( ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0937C')
reaction.name = 'RE0937'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2783: -1.0, Cpd1554: 1.0, Cpd2784: 1.0})
reaction.gene_reaction_rule = '( ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0937E')
reaction.name = 'RE0937'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2785: -1.0, Cpd1553: 1.0, Cpd2786: 1.0})
reaction.gene_reaction_rule = '( ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0938C')
reaction.name = 'RE0938'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2787: -1.0, Cpd1940: 1.0, Cpd2788: 1.0})
reaction.gene_reaction_rule = '( ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0938E')
reaction.name = 'RE0938'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2789: -1.0, Cpd1939: 1.0, Cpd2790: 1.0})
reaction.gene_reaction_rule = '( ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE0944E')
reaction.name = 'RE0944'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2791: -1.0, Cpd446: 1.0, Cpd2778: 1.0})
reaction.gene_reaction_rule = '( AMY2A5 or LYZL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE0958E')
reaction.name = 'RE0958'
reaction.subsystem = 'Starch and sucrose metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2792: -1.0, Cpd446: 1.0, Cpd2793: 1.0})
reaction.gene_reaction_rule = '( AMY2A5 or LYZL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1050C')
reaction.name = 'RE1050'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -3.0, Cpd2794: -2.0, Cpd2: 2.0, Cpd33: 2.0, Cpd1161: 2.0, Cpd2795: 2.0})
reaction.gene_reaction_rule = '( PRDX6 or TPO )'
model.add_reactions([reaction])
reaction = Reaction('RE1050E')
reaction.name = 'RE1050'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1444: -3.0, Cpd2796: -2.0, Cpd37: 2.0, Cpd114: 2.0, Cpd2797: 2.0, Cpd2798: 2.0})
reaction.gene_reaction_rule = '( LPO )'
model.add_reactions([reaction])
reaction = Reaction('RE1050L')
reaction.name = 'RE1050'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1995: -3.0, Cpd2799: -2.0, Cpd86: 2.0, Cpd176: 2.0, Cpd2800: 2.0, Cpd2801: 2.0})
reaction.gene_reaction_rule = '( MPO or PRDX6 )'
model.add_reactions([reaction])
reaction = Reaction('RE1050N')
reaction.name = 'RE1050'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2146: -3.0, Cpd2802: -2.0, Cpd570: 2.0, Cpd763: 2.0, Cpd2803: 2.0, Cpd2804: 2.0})
reaction.gene_reaction_rule = '( MPO )'
model.add_reactions([reaction])
reaction = Reaction('RE1062M')
reaction.name = 'RE1062'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2805: -1.0, Cpd2806: 1.0, Cpd2807: 1.0})
reaction.gene_reaction_rule = '( NLN )'
model.add_reactions([reaction])
reaction = Reaction('RE1096M')
reaction.name = 'RE1096'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd8: -1.0, Cpd9: -1.0, Cpd2808: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd267: 1.0, Cpd2809: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1096R')
reaction.name = 'RE1096'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd587: -1.0, Cpd1531: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1514: 1.0, Cpd2810: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1134M')
reaction.name = 'RE1134'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd8: -1.0, Cpd9: -1.0, Cpd2811: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd267: 1.0, Cpd2812: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1134R')
reaction.name = 'RE1134'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd587: -1.0, Cpd1530: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd1527: 1.0, Cpd2810: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1233C')
reaction.name = 'RE1233'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd1117: -1.0, Cpd2: 1.0, Cpd74: 1.0, Cpd2813: 1.0})
reaction.gene_reaction_rule = '( KYAT1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1233M')
reaction.name = 'RE1233'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd2814: -1.0, Cpd12: 1.0, Cpd63: 1.0, Cpd2815: 1.0})
reaction.gene_reaction_rule = '( AADAT or KYAT3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1308M')
reaction.name = 'RE1308'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd14: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd2816: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1309M')
reaction.name = 'RE1309'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd9: -1.0, Cpd15: -1.0, Cpd47: -1.0, Cpd12: 1.0, Cpd44: 1.0, Cpd2817: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1310M')
reaction.name = 'RE1310'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd2817: -1.0, Cpd634: 1.0, Cpd2818: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1311M')
reaction.name = 'RE1311'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd2818: -1.0, Cpd634: 1.0, Cpd2819: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1441G')
reaction.name = 'RE1441'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2820: -1.0, Cpd2821: -1.0, Cpd140: 1.0, Cpd2822: 1.0, Cpd2823: 1.0})
reaction.gene_reaction_rule = '( PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('RE1441R')
reaction.name = 'RE1441'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1966: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd1964: 1.0})
reaction.gene_reaction_rule = '( PIP4K2A or PIP4K2B or PIP4K2C )'
model.add_reactions([reaction])
reaction = Reaction('RE1447M')
reaction.name = 'RE1447'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2824: -1.0, Cpd204: 1.0, Cpd2825: 1.0})
reaction.gene_reaction_rule = '( OXSM )'
model.add_reactions([reaction])
reaction = Reaction('RE1448R')
reaction.name = 'RE1448'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1964: -1.0, Cpd915: 1.0, Cpd1966: 1.0})
reaction.gene_reaction_rule = '( TPTE )'
model.add_reactions([reaction])
reaction = Reaction('RE1514M')
reaction.name = 'RE1514'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd2826: -1.0, Cpd209: 1.0, Cpd210: 1.0, Cpd1069: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE1514X')
reaction.name = 'RE1514'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd2827: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd2361: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1516M')
reaction.name = 'RE1516'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd1069: -1.0, Cpd240: 1.0, Cpd2828: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1516X')
reaction.name = 'RE1516'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd258: -1.0, Cpd2361: -1.0, Cpd257: 1.0, Cpd2829: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1517M')
reaction.name = 'RE1517'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd2830: -1.0, Cpd240: 1.0, Cpd2831: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1517X')
reaction.name = 'RE1517'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd258: -1.0, Cpd2832: -1.0, Cpd257: 1.0, Cpd2833: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1518M')
reaction.name = 'RE1518'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd2834: -1.0, Cpd240: 1.0, Cpd2835: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1518X')
reaction.name = 'RE1518'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd258: -1.0, Cpd2836: -1.0, Cpd257: 1.0, Cpd2837: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1519M')
reaction.name = 'RE1519'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd2838: -1.0, Cpd240: 1.0, Cpd2839: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1519X')
reaction.name = 'RE1519'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd258: -1.0, Cpd2840: -1.0, Cpd257: 1.0, Cpd2841: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1520M')
reaction.name = 'RE1520'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2842: -1.0, Cpd2843: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1520X')
reaction.name = 'RE1520'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2844: -1.0, Cpd2845: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE1521M')
reaction.name = 'RE1521'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2835: -1.0, Cpd2846: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1521X')
reaction.name = 'RE1521'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2837: -1.0, Cpd2847: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE1522M')
reaction.name = 'RE1522'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2831: -1.0, Cpd2848: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1522X')
reaction.name = 'RE1522'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2833: -1.0, Cpd2849: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE1523M')
reaction.name = 'RE1523'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2828: -1.0, Cpd2674: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1523X')
reaction.name = 'RE1523'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2829: -1.0, Cpd2676: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE1525M')
reaction.name = 'RE1525'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2848: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2850: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1525X')
reaction.name = 'RE1525'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2849: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd2851: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1526M')
reaction.name = 'RE1526'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2846: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2852: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1526X')
reaction.name = 'RE1526'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2847: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd2853: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1527M')
reaction.name = 'RE1527'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2843: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2854: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1527X')
reaction.name = 'RE1527'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd2845: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd2855: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1530M')
reaction.name = 'RE1530'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd908: -1.0, Cpd953: -1.0, Cpd7: 1.0, Cpd897: 1.0, Cpd954: 1.0})
reaction.gene_reaction_rule = '( TK2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1531M')
reaction.name = 'RE1531'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2675: -1.0, Cpd178: 1.0, Cpd2830: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1531X')
reaction.name = 'RE1531'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd2677: -1.0, Cpd182: 1.0, Cpd2832: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE1532M')
reaction.name = 'RE1532'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2850: -1.0, Cpd178: 1.0, Cpd2834: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1532X')
reaction.name = 'RE1532'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd2851: -1.0, Cpd182: 1.0, Cpd2836: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE1533M')
reaction.name = 'RE1533'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2852: -1.0, Cpd178: 1.0, Cpd2856: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1533X')
reaction.name = 'RE1533'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd2853: -1.0, Cpd182: 1.0, Cpd2857: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE1534M')
reaction.name = 'RE1534'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd2854: -1.0, Cpd178: 1.0, Cpd2838: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE1534X')
reaction.name = 'RE1534'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd2855: -1.0, Cpd182: 1.0, Cpd2840: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE1537X')
reaction.name = 'RE1537'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2858: -1.0, Cpd247: 1.0, Cpd2859: 1.0, Cpd2860: 1.0})
reaction.gene_reaction_rule = '( PAOX )'
model.add_reactions([reaction])
reaction = Reaction('RE1538X')
reaction.name = 'RE1538'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2861: -1.0, Cpd247: 1.0, Cpd2858: 1.0, Cpd2860: 1.0})
reaction.gene_reaction_rule = '( PAOX )'
model.add_reactions([reaction])
reaction = Reaction('RE1539X')
reaction.name = 'RE1539'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2862: -1.0, Cpd247: 1.0, Cpd2719: 1.0, Cpd2860: 1.0})
reaction.gene_reaction_rule = '( PAOX )'
model.add_reactions([reaction])
reaction = Reaction('RE1573M')
reaction.name = 'RE1573'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2856: -1.0, Cpd2842: 1.0})
reaction.gene_reaction_rule = '( ECI1 or ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE1573X')
reaction.name = 'RE1573'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2857: -1.0, Cpd2844: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE1582R')
reaction.name = 'RE1582'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1519: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2863: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE1587R')
reaction.name = 'RE1587'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1519: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2864: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE1632R')
reaction.name = 'RE1632'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1520: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2865: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE1635M')
reaction.name = 'RE1635'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd2866: -1.0, Cpd2867: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1635R')
reaction.name = 'RE1635'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd2868: -1.0, Cpd2869: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1635X')
reaction.name = 'RE1635'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2870: -1.0, Cpd2871: -1.0, Cpd2872: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1691M')
reaction.name = 'RE1691'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2873: -1.0, Cpd1164: -1.0, Cpd7: 1.0, Cpd1160: 1.0, Cpd2729: 1.0})
reaction.gene_reaction_rule = '( TST )'
model.add_reactions([reaction])
reaction = Reaction('RE1709N')
reaction.name = 'RE1709'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2739: -1.0, Cpd570: 1.0, Cpd2874: 1.0})
reaction.gene_reaction_rule = '( PCBD1 or PCBD2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1711M')
reaction.name = 'RE1711'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd2875: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd119: 1.0})
reaction.gene_reaction_rule = '( ADHFE1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1796M')
reaction.name = 'RE1796'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd2876: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2877: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1796R')
reaction.name = 'RE1796'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd625: -1.0, Cpd1532: -1.0, Cpd565: 1.0, Cpd626: 1.0, Cpd1533: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1804M')
reaction.name = 'RE1804'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1865: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd1866: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1806R')
reaction.name = 'RE1806'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2878: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2879: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE1807M')
reaction.name = 'RE1807'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd1866: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd1867: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1809R')
reaction.name = 'RE1809'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2879: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2880: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE1811R')
reaction.name = 'RE1811'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2879: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2881: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE1812R')
reaction.name = 'RE1812'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2879: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2882: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE1815M')
reaction.name = 'RE1815'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd2883: -1.0, Cpd2884: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1815R')
reaction.name = 'RE1815'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd2885: -1.0, Cpd2886: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1815X')
reaction.name = 'RE1815'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd2887: -1.0, Cpd2888: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1816M')
reaction.name = 'RE1816'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd2889: -1.0, Cpd2890: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1816R')
reaction.name = 'RE1816'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd2891: -1.0, Cpd2892: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1816X')
reaction.name = 'RE1816'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd2893: -1.0, Cpd2894: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1817M')
reaction.name = 'RE1817'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd2895: -1.0, Cpd2896: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1817R')
reaction.name = 'RE1817'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd2897: -1.0, Cpd2898: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1817X')
reaction.name = 'RE1817'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd2899: -1.0, Cpd2900: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1818M')
reaction.name = 'RE1818'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd2901: -1.0, Cpd2902: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1818R')
reaction.name = 'RE1818'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd2903: -1.0, Cpd2904: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1818X')
reaction.name = 'RE1818'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd2905: -1.0, Cpd2906: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1819C')
reaction.name = 'RE1819'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd2907: -1.0, Cpd25: 1.0, Cpd2908: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE1819M')
reaction.name = 'RE1819'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd2902: -1.0, Cpd13: 1.0, Cpd2909: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1819X')
reaction.name = 'RE1819'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd2906: -1.0, Cpd1517: 1.0, Cpd2910: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE1826M')
reaction.name = 'RE1826'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd1864: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd2911: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1827M')
reaction.name = 'RE1827'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd1868: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2912: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1828M')
reaction.name = 'RE1828'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd2913: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd2914: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1829M')
reaction.name = 'RE1829'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd2914: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd2915: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1830M')
reaction.name = 'RE1830'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd2912: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd2913: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1834C')
reaction.name = 'RE1834'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2406: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd679: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1834M')
reaction.name = 'RE1834'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2916: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2917: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE1834X')
reaction.name = 'RE1834'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd529: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2918: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1835C')
reaction.name = 'RE1835'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2424: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd2423: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1835M')
reaction.name = 'RE1835'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd2919: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2920: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE1835X')
reaction.name = 'RE1835'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd535: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2921: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1836M')
reaction.name = 'RE1836'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd43: -1.0, Cpd2922: -1.0, Cpd12: 1.0, Cpd179: 1.0, Cpd2919: 1.0})
reaction.gene_reaction_rule = '( SCP2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1836X')
reaction.name = 'RE1836'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd246: -1.0, Cpd433: -1.0, Cpd261: 1.0, Cpd535: 1.0, Cpd742: 1.0})
reaction.gene_reaction_rule = '( SCP2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1845C')
reaction.name = 'RE1845'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1586: -1.0, Cpd2424: -1.0, Cpd78: 1.0, Cpd2510: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1845X')
reaction.name = 'RE1845'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd531: -1.0, Cpd535: -1.0, Cpd184: 1.0, Cpd2923: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1846C')
reaction.name = 'RE1846'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1586: -1.0, Cpd2924: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd2925: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1846X')
reaction.name = 'RE1846'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd531: -1.0, Cpd2926: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd2927: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE1860E')
reaction.name = 'RE1860'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2928: -1.0, Cpd37: 1.0, Cpd2929: 1.0})
reaction.gene_reaction_rule = '( CNP )'
model.add_reactions([reaction])
reaction = Reaction('RE1904R')
reaction.name = 'RE1904'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2930: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2931: 1.0})
reaction.gene_reaction_rule = '( CYP26A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1905R')
reaction.name = 'RE1905'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2932: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2933: 1.0})
reaction.gene_reaction_rule = '( CYP1A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1906R')
reaction.name = 'RE1906'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2934: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2935: 1.0})
reaction.gene_reaction_rule = '( CYP1A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1916X')
reaction.name = 'RE1916'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd2936: -1.0, Cpd2937: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1938R')
reaction.name = 'RE1938'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2938: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2939: 1.0})
reaction.gene_reaction_rule = '( CYP26B1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1941R')
reaction.name = 'RE1941'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd565: -2.0, Cpd587: -2.0, Cpd2940: -1.0, Cpd437: 3.0, Cpd589: 2.0, Cpd2941: 1.0})
reaction.gene_reaction_rule = '( CYP26B1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1942R')
reaction.name = 'RE1942'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd565: -2.0, Cpd587: -2.0, Cpd2260: -1.0, Cpd437: 3.0, Cpd589: 2.0, Cpd2942: 1.0})
reaction.gene_reaction_rule = '( CYP26B1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1943R')
reaction.name = 'RE1943'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2258: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2943: 1.0})
reaction.gene_reaction_rule = '( CYP26B1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1952C')
reaction.name = 'RE1952'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2944: -1.0, Cpd2945: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1952R')
reaction.name = 'RE1952'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2946: -1.0, Cpd2947: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1952X')
reaction.name = 'RE1952'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2948: -1.0, Cpd2949: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1956C')
reaction.name = 'RE1956'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2944: -1.0, Cpd2950: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1956R')
reaction.name = 'RE1956'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2946: -1.0, Cpd2951: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE1956X')
reaction.name = 'RE1956'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd2948: -1.0, Cpd2952: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE1957G')
reaction.name = 'RE1957'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2821: -1.0, Cpd2953: -1.0, Cpd140: 1.0, Cpd2823: 1.0, Cpd2954: 1.0})
reaction.gene_reaction_rule = '( PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('RE1957R')
reaction.name = 'RE1957'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1965: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd2955: 1.0})
reaction.gene_reaction_rule = '( PIP4K2A or PIP4K2B or PIP4K2C )'
model.add_reactions([reaction])
reaction = Reaction('RE2030M')
reaction.name = 'RE2030'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd2517: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2956: 1.0})
reaction.gene_reaction_rule = '( NAGS )'
model.add_reactions([reaction])
reaction = Reaction('RE2031M')
reaction.name = 'RE2031'
reaction.subsystem = 'Alanine and aspartate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd361: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2957: 1.0})
reaction.gene_reaction_rule = '( NAGS )'
model.add_reactions([reaction])
reaction = Reaction('RE2032M')
reaction.name = 'RE2032'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd481: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2958: 1.0})
reaction.gene_reaction_rule = '( NAGS )'
model.add_reactions([reaction])
reaction = Reaction('RE2048N')
reaction.name = 'RE2048'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd2959: -1.0, Cpd570: 1.0, Cpd2960: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE2050R')
reaction.name = 'RE2050'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd2961: -1.0, Cpd2962: 1.0})
reaction.gene_reaction_rule = '( PTGS1 or PTGS2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2051C')
reaction.name = 'RE2051'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2963: -1.0, Cpd108: 1.0, Cpd2964: 1.0})
reaction.gene_reaction_rule = '( PLPP1 or PLPP3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2051G')
reaction.name = 'RE2051'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd2965: -1.0, Cpd1773: 1.0, Cpd2966: 1.0})
reaction.gene_reaction_rule = '( PLPP3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2051R')
reaction.name = 'RE2051'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2967: -1.0, Cpd915: 1.0, Cpd2961: 1.0})
reaction.gene_reaction_rule = '( PLPP3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2111M')
reaction.name = 'RE2111'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd243: -1.0, Cpd364: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2968: 1.0})
reaction.gene_reaction_rule = '( GLYAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2112R')
reaction.name = 'RE2112'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd589: -1.0, Cpd2389: -1.0, Cpd435: 1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd2969: 1.0})
reaction.gene_reaction_rule = '( CYP7A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2117M')
reaction.name = 'RE2117'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd364: -1.0, Cpd613: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2970: 1.0})
reaction.gene_reaction_rule = '( GLYAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2146C')
reaction.name = 'RE2146'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2971: -1.0, Cpd137: 1.0, Cpd2972: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2146R')
reaction.name = 'RE2146'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2973: -1.0, Cpd1769: 1.0, Cpd2974: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2147C')
reaction.name = 'RE2147'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2975: -1.0, Cpd137: 1.0, Cpd2976: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2147R')
reaction.name = 'RE2147'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2260: -1.0, Cpd2244: -1.0, Cpd1769: 1.0, Cpd2977: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2149C')
reaction.name = 'RE2149'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2978: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd2979: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2149R')
reaction.name = 'RE2149'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2980: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd2981: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2150C')
reaction.name = 'RE2150'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2982: -1.0, Cpd137: 1.0, Cpd2983: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2150R')
reaction.name = 'RE2150'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2940: -1.0, Cpd1769: 1.0, Cpd2984: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2151C')
reaction.name = 'RE2151'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd2985: -1.0, Cpd137: 1.0, Cpd2986: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2151R')
reaction.name = 'RE2151'
reaction.subsystem = 'Vitamin A metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd2941: -1.0, Cpd1769: 1.0, Cpd2987: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2155R')
reaction.name = 'RE2155'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd626: -1.0, Cpd2988: -1.0, Cpd437: 1.0, Cpd625: 1.0, Cpd2989: 1.0})
reaction.gene_reaction_rule = '( CYP21A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2156M')
reaction.name = 'RE2156'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd2990: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2991: 1.0})
reaction.gene_reaction_rule = '( NAGS )'
model.add_reactions([reaction])
reaction = Reaction('RE2221M')
reaction.name = 'RE2221'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd9: -1.0, Cpd47: -1.0, Cpd2992: -1.0, Cpd12: 1.0, Cpd44: 1.0, Cpd2993: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2223M')
reaction.name = 'RE2223'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd178: -1.0, Cpd754: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd2994: 1.0})
reaction.gene_reaction_rule = '( NAGS )'
model.add_reactions([reaction])
reaction = Reaction('RE2235R')
reaction.name = 'RE2235'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1519: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2995: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2269E')
reaction.name = 'RE2269'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -2.0, Cpd2996: -1.0, Cpd1589: 1.0, Cpd1939: 1.0, Cpd2997: 1.0})
reaction.gene_reaction_rule = '( CMA1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2270E')
reaction.name = 'RE2270'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2996: -1.0, Cpd1589: 1.0, Cpd2998: 1.0})
reaction.gene_reaction_rule = '( CPA1 or CPA2 or CPA5 or CPA6 )'
model.add_reactions([reaction])
reaction = Reaction('RE2272L')
reaction.name = 'RE2272'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd2999: -1.0, Cpd3000: 1.0, Cpd3001: 1.0})
reaction.gene_reaction_rule = '( TPP1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2273E')
reaction.name = 'RE2273'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd3002: -1.0, Cpd1589: 1.0, Cpd3003: 1.0})
reaction.gene_reaction_rule = '( CPA1 or CPA2 or CPA5 or CPA6 )'
model.add_reactions([reaction])
reaction = Reaction('RE2296X')
reaction.name = 'RE2296'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd3004: -1.0, Cpd3005: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2304E')
reaction.name = 'RE2304'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -2.0, Cpd2996: -1.0, Cpd1589: 1.0, Cpd3006: 1.0, Cpd3007: 1.0})
reaction.gene_reaction_rule = '( TPSAB1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2306C')
reaction.name = 'RE2306'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3008: -1.0, Cpd1335: 1.0, Cpd3009: 1.0})
reaction.gene_reaction_rule = '( TRHDE )'
model.add_reactions([reaction])
reaction = Reaction('RE2318M')
reaction.name = 'RE2318'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd2866: -1.0, Cpd3010: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2318R')
reaction.name = 'RE2318'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd2868: -1.0, Cpd3011: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2318X')
reaction.name = 'RE2318'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2870: -1.0, Cpd2871: -1.0, Cpd3012: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2319M')
reaction.name = 'RE2319'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd2883: -1.0, Cpd3013: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2319R')
reaction.name = 'RE2319'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd2885: -1.0, Cpd3014: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2319X')
reaction.name = 'RE2319'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd2887: -1.0, Cpd3015: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2349C')
reaction.name = 'RE2349'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd30: -1.0, Cpd72: -1.0, Cpd2: 1.0, Cpd74: 1.0, Cpd3016: 1.0})
reaction.gene_reaction_rule = '( KYAT1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2349M')
reaction.name = 'RE2349'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd28: -1.0, Cpd3017: -1.0, Cpd12: 1.0, Cpd63: 1.0, Cpd3018: 1.0})
reaction.gene_reaction_rule = '( AADAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2360N')
reaction.name = 'RE2360'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd673: -1.0, Cpd3019: -1.0, Cpd676: 1.0, Cpd763: 1.0, Cpd3020: 1.0})
reaction.gene_reaction_rule = '( NAA10 )'
model.add_reactions([reaction])
reaction = Reaction('RE2382R')
reaction.name = 'RE2382'
reaction.subsystem = 'Vitamin E metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3021: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3022: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE2383R')
reaction.name = 'RE2383'
reaction.subsystem = 'Vitamin E metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd565: -1.0, Cpd587: -2.0, Cpd3023: -1.0, Cpd437: 3.0, Cpd589: 2.0, Cpd3024: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE2387R')
reaction.name = 'RE2387'
reaction.subsystem = 'Vitamin E metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3025: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3023: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE2398R')
reaction.name = 'RE2398'
reaction.subsystem = 'Vitamin E metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd565: -1.0, Cpd587: -2.0, Cpd3022: -1.0, Cpd437: 3.0, Cpd589: 2.0, Cpd3026: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE2404C')
reaction.name = 'RE2404'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd3027: -1.0, Cpd137: 1.0, Cpd3028: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2404R')
reaction.name = 'RE2404'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd3029: -1.0, Cpd1769: 1.0, Cpd3030: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2405C')
reaction.name = 'RE2405'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd3031: -1.0, Cpd137: 1.0, Cpd3032: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2405R')
reaction.name = 'RE2405'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd3033: -1.0, Cpd1769: 1.0, Cpd3034: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2410N')
reaction.name = 'RE2410'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd2150: -1.0, Cpd3035: -1.0, Cpd2151: 1.0, Cpd3036: 1.0})
reaction.gene_reaction_rule = '( DHCR7 )'
model.add_reactions([reaction])
reaction = Reaction('RE2427M')
reaction.name = 'RE2427'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd242: -1.0, Cpd364: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3037: 1.0})
reaction.gene_reaction_rule = '( GLYAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2428M')
reaction.name = 'RE2428'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd237: -1.0, Cpd364: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3038: 1.0})
reaction.gene_reaction_rule = '( GLYAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2429M')
reaction.name = 'RE2429'
reaction.subsystem = 'Glycine, serine, alanine and threonine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd244: -1.0, Cpd364: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3039: 1.0})
reaction.gene_reaction_rule = '( GLYAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2443M')
reaction.name = 'RE2443'
reaction.subsystem = 'Tryptophan metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd3040: -1.0, Cpd12: 1.0, Cpd3041: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2445C')
reaction.name = 'RE2445'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3042: -1.0, Cpd3043: 1.0, Cpd3044: 1.0})
reaction.gene_reaction_rule = '( ACE or ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2445E')
reaction.name = 'RE2445'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd3045: -1.0, Cpd3046: 1.0, Cpd3047: 1.0})
reaction.gene_reaction_rule = '( ACE or ACE2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2453M')
reaction.name = 'RE2453'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd201: -1.0, Cpd239: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd204: 1.0, Cpd3048: 1.0})
reaction.gene_reaction_rule = '( MCCC1 or MCCC2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2454M')
reaction.name = 'RE2454'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3048: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3049: 1.0})
reaction.gene_reaction_rule = '( MCCC1 or MCCC2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2474R')
reaction.name = 'RE2474'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3050: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3051: 1.0, Cpd3052: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE2513C')
reaction.name = 'RE2513'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd33: -1.0, Cpd700: -1.0, Cpd2: 1.0, Cpd3053: 1.0})
reaction.gene_reaction_rule = '( PRDX6 or TPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2513E')
reaction.name = 'RE2513'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd698: -1.0, Cpd1444: -1.0, Cpd114: 1.0, Cpd3054: 1.0})
reaction.gene_reaction_rule = '( LPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2513L')
reaction.name = 'RE2513'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd1995: -1.0, Cpd3055: -1.0, Cpd86: 1.0, Cpd3056: 1.0})
reaction.gene_reaction_rule = '( MPO or PRDX6 )'
model.add_reactions([reaction])
reaction = Reaction('RE2513N')
reaction.name = 'RE2513'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd2146: -1.0, Cpd3057: -1.0, Cpd570: 1.0, Cpd3058: 1.0})
reaction.gene_reaction_rule = '( MPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2514C')
reaction.name = 'RE2514'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd3059: -1.0, Cpd3053: -1.0, Cpd2: 1.0, Cpd3060: 1.0})
reaction.gene_reaction_rule = '( PRDX6 or TPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2514E')
reaction.name = 'RE2514'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3061: -1.0, Cpd3054: -1.0, Cpd114: 1.0, Cpd3062: 1.0})
reaction.gene_reaction_rule = '( LPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2514L')
reaction.name = 'RE2514'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd3056: -1.0, Cpd3063: -1.0, Cpd86: 1.0, Cpd3064: 1.0})
reaction.gene_reaction_rule = '( MPO or PRDX6 )'
model.add_reactions([reaction])
reaction = Reaction('RE2514N')
reaction.name = 'RE2514'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd3058: -1.0, Cpd3065: -1.0, Cpd570: 1.0, Cpd3066: 1.0})
reaction.gene_reaction_rule = '( MPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2522X')
reaction.name = 'RE2522'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd3067: -1.0, Cpd3068: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2523X')
reaction.name = 'RE2523'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd3069: -1.0, Cpd3070: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2524X')
reaction.name = 'RE2524'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd3071: -1.0, Cpd3072: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2525X')
reaction.name = 'RE2525'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd3073: -1.0, Cpd3074: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2541C')
reaction.name = 'RE2541'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3032: -1.0, Cpd33: 1.0, Cpd1401: 1.0, Cpd3031: 1.0})
reaction.gene_reaction_rule = '( KL )'
model.add_reactions([reaction])
reaction = Reaction('RE2541E')
reaction.name = 'RE2541'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd3075: -1.0, Cpd37: 1.0, Cpd3076: 1.0, Cpd3077: 1.0})
reaction.gene_reaction_rule = '( KL )'
model.add_reactions([reaction])
reaction = Reaction('RE2541L')
reaction.name = 'RE2541'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd3078: -1.0, Cpd176: 1.0, Cpd1357: 1.0, Cpd3079: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('RE2622C')
reaction.name = 'RE2622'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd3080: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd3081: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2622R')
reaction.name = 'RE2622'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd3082: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd3083: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE2624M')
reaction.name = 'RE2624'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3084: -1.0, Cpd3085: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('RE2624X')
reaction.name = 'RE2624'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3086: -1.0, Cpd3087: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('RE2625M')
reaction.name = 'RE2625'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd1861: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3088: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2626M')
reaction.name = 'RE2626'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd13: -1.0, Cpd3088: -1.0, Cpd7: 2.0, Cpd8: 1.0, Cpd1863: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2632M')
reaction.name = 'RE2632'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -2.0, Cpd9: -2.0, Cpd3089: -1.0, Cpd12: 3.0, Cpd13: 2.0, Cpd2914: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2633R')
reaction.name = 'RE2633'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd3090: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd3091: 1.0})
reaction.gene_reaction_rule = '( CYP3A13 )'
model.add_reactions([reaction])
reaction = Reaction('RE2635R')
reaction.name = 'RE2635'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2969: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3092: 1.0})
reaction.gene_reaction_rule = '( SLC27A5 )'
model.add_reactions([reaction])
reaction = Reaction('RE2636R')
reaction.name = 'RE2636'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3093: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd2969: 1.0})
reaction.gene_reaction_rule = '( CYP8B1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2637C')
reaction.name = 'RE2637'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1586: -1.0, Cpd2609: -1.0, Cpd33: 2.0, Cpd78: 1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd2580: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2637X')
reaction.name = 'RE2637'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd940: -1.0, Cpd531: -1.0, Cpd3094: -1.0, Cpd358: 1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3095: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2638C')
reaction.name = 'RE2638'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd402: -1.0, Cpd2609: -1.0, Cpd33: 2.0, Cpd78: 1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd2582: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2638X')
reaction.name = 'RE2638'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd367: -1.0, Cpd940: -1.0, Cpd3094: -1.0, Cpd358: 1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3096: 1.0})
reaction.gene_reaction_rule = '( BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2640C')
reaction.name = 'RE2640'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3097: -1.0, Cpd221: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( ACY1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2644C')
reaction.name = 'RE2644'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3098: -1.0, Cpd221: 1.0, Cpd393: 1.0})
reaction.gene_reaction_rule = '( ACY3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2649C')
reaction.name = 'RE2649'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd207: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd206: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2649M')
reaction.name = 'RE2649'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd179: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd208: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE2649X')
reaction.name = 'RE2649'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd742: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3099: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE2660N')
reaction.name = 'RE2660'
reaction.subsystem = 'Tetrahydrobiopterin metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2874: -1.0, Cpd2740: 1.0})
reaction.gene_reaction_rule = '( PCBD1 or PCBD2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2666G')
reaction.name = 'RE2666'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd1205: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd1209: 1.0})
reaction.gene_reaction_rule = '( B4GALT6 )'
model.add_reactions([reaction])
reaction = Reaction('RE2677R')
reaction.name = 'RE2677'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd665: -1.0, Cpd3100: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd3101: 1.0})
reaction.gene_reaction_rule = '( B4GALT6 or UGT8A )'
model.add_reactions([reaction])
reaction = Reaction('RE2680G')
reaction.name = 'RE2680'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd139: -1.0, Cpd528: -1.0, Cpd140: 1.0, Cpd142: 1.0, Cpd3102: 1.0})
reaction.gene_reaction_rule = '( B4GALT6 )'
model.add_reactions([reaction])
reaction = Reaction('RE2717L')
reaction.name = 'RE2717'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd3103: -1.0, Cpd478: 1.0, Cpd1212: 1.0})
reaction.gene_reaction_rule = '( GALC )'
model.add_reactions([reaction])
reaction = Reaction('RE2718C')
reaction.name = 'RE2718'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd3104: -1.0, Cpd33: 1.0, Cpd26: 1.0, Cpd3105: 1.0})
reaction.gene_reaction_rule = '( GAL3ST1 or SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2718G')
reaction.name = 'RE2718'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1206: -1.0, Cpd3106: -1.0, Cpd140: 1.0, Cpd1207: 1.0, Cpd3107: 1.0})
reaction.gene_reaction_rule = '( GAL3ST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2722C')
reaction.name = 'RE2722'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd3108: -1.0, Cpd26: 1.0, Cpd3109: 1.0})
reaction.gene_reaction_rule = '( GAL3ST1 or SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2722G')
reaction.name = 'RE2722'
reaction.subsystem = 'Glycosphingolipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd138: -1.0, Cpd1206: -1.0, Cpd1207: 1.0, Cpd3110: 1.0})
reaction.gene_reaction_rule = '( GAL3ST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2759X')
reaction.name = 'RE2759'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3111: -1.0, Cpd3112: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('RE2766C')
reaction.name = 'RE2766'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd2227: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd3113: 1.0})
reaction.gene_reaction_rule = '( RDH8 )'
model.add_reactions([reaction])
reaction = Reaction('RE2768M')
reaction.name = 'RE2768'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -2.0, Cpd9: -1.0, Cpd1853: -1.0, Cpd12: 2.0, Cpd13: 2.0, Cpd267: 1.0, Cpd3114: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2768R')
reaction.name = 'RE2768'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1512: -1.0, Cpd1855: 1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3115: 1.0})
reaction.gene_reaction_rule = '( CYP17A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2888C')
reaction.name = 'RE2888'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd2379: -1.0, Cpd469: 1.0, Cpd3116: 1.0})
reaction.gene_reaction_rule = '( PRDX6 or TPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2888E')
reaction.name = 'RE2888'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1444: -1.0, Cpd2600: -1.0, Cpd2271: 1.0, Cpd3117: 1.0})
reaction.gene_reaction_rule = '( LPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2888N')
reaction.name = 'RE2888'
reaction.subsystem = 'Purine catabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2146: -1.0, Cpd3118: -1.0, Cpd3119: 1.0, Cpd3120: 1.0})
reaction.gene_reaction_rule = '( MPO )'
model.add_reactions([reaction])
reaction = Reaction('RE2908M')
reaction.name = 'RE2908'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3121: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3122: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HADHB or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2908X')
reaction.name = 'RE2908'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3123: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3124: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2909M')
reaction.name = 'RE2909'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3125: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3126: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HADHB or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2909X')
reaction.name = 'RE2909'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3127: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3128: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2910M')
reaction.name = 'RE2910'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3129: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2578: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HADHB or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2910X')
reaction.name = 'RE2910'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3130: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3131: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2911C')
reaction.name = 'RE2911'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd3132: -1.0, Cpd25: 1.0, Cpd3133: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2912M')
reaction.name = 'RE2912'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd3134: -1.0, Cpd209: 1.0, Cpd210: 1.0, Cpd2370: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE2912X')
reaction.name = 'RE2912'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3135: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3136: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2913M')
reaction.name = 'RE2913'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd2370: -1.0, Cpd240: 1.0, Cpd3137: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACADM or ACADSB or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2913X')
reaction.name = 'RE2913'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd258: -1.0, Cpd3136: -1.0, Cpd257: 1.0, Cpd3138: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2914M')
reaction.name = 'RE2914'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3139: -1.0, Cpd240: 1.0, Cpd3140: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACADM or ACADSB or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2914X')
reaction.name = 'RE2914'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd258: -1.0, Cpd3141: -1.0, Cpd257: 1.0, Cpd3142: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2915M')
reaction.name = 'RE2915'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd240: -1.0, Cpd3143: -1.0, Cpd238: 1.0, Cpd3144: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACADM or ACADSB or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2915X')
reaction.name = 'RE2915'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd257: -1.0, Cpd3145: -1.0, Cpd258: 1.0, Cpd3146: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2916M')
reaction.name = 'RE2916'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3122: -1.0, Cpd178: 1.0, Cpd3139: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE2916X')
reaction.name = 'RE2916'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3124: -1.0, Cpd182: 1.0, Cpd3141: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE2917M')
reaction.name = 'RE2917'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3126: -1.0, Cpd178: 1.0, Cpd3144: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE2917X')
reaction.name = 'RE2917'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3128: -1.0, Cpd182: 1.0, Cpd3146: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE2919M')
reaction.name = 'RE2919'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3137: -1.0, Cpd3121: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE2919X')
reaction.name = 'RE2919'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3138: -1.0, Cpd3123: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2920M')
reaction.name = 'RE2920'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3140: -1.0, Cpd3125: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE2920X')
reaction.name = 'RE2920'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3142: -1.0, Cpd3127: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2921M')
reaction.name = 'RE2921'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3143: -1.0, Cpd3129: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE2921X')
reaction.name = 'RE2921'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3145: -1.0, Cpd3130: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2954C')
reaction.name = 'RE2954'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd231: -1.0, Cpd900: -1.0, Cpd21: 1.0, Cpd905: 1.0})
reaction.gene_reaction_rule = '( PKM )'
model.add_reactions([reaction])
reaction = Reaction('RE2972G')
reaction.name = 'RE2972'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2821: -1.0, Cpd2822: -1.0, Cpd140: 1.0, Cpd2823: 1.0, Cpd3147: 1.0})
reaction.gene_reaction_rule = '( PI4KA or PI4KB )'
model.add_reactions([reaction])
reaction = Reaction('RE2972M')
reaction.name = 'RE2972'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd2825: -1.0, Cpd7: 1.0, Cpd202: 1.0, Cpd2824: 1.0})
reaction.gene_reaction_rule = '( PI4KA or PI4KB )'
model.add_reactions([reaction])
reaction = Reaction('RE2972R')
reaction.name = 'RE2972'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1964: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd3148: 1.0})
reaction.gene_reaction_rule = '( PI4KA or PI4KB )'
model.add_reactions([reaction])
reaction = Reaction('RE2973C')
reaction.name = 'RE2973'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1963: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1955: 1.0})
reaction.gene_reaction_rule = '( PIK3C2A or PIK3C2B or PIK3C2G or PIK3C3 or PIK3CA or PIK3CB or PIK3CG )'
model.add_reactions([reaction])
reaction = Reaction('RE2973G')
reaction.name = 'RE2973'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2821: -1.0, Cpd2822: -1.0, Cpd140: 1.0, Cpd2823: 1.0, Cpd2953: 1.0})
reaction.gene_reaction_rule = '( PIK3C2A or PIK3C2B or PIK3C2G )'
model.add_reactions([reaction])
reaction = Reaction('RE2973N')
reaction.name = 'RE2973'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd788: -1.0, Cpd3149: -1.0, Cpd763: 1.0, Cpd789: 1.0, Cpd3150: 1.0})
reaction.gene_reaction_rule = '( PIK3C2A or PIK3C2B or PIK3C2G )'
model.add_reactions([reaction])
reaction = Reaction('RE2973R')
reaction.name = 'RE2973'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1964: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd1965: 1.0})
reaction.gene_reaction_rule = '( PIK3C2A or PIK3C2B or PIK3C2G )'
model.add_reactions([reaction])
reaction = Reaction('RE2974C')
reaction.name = 'RE2974'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd1954: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd1955: 1.0})
reaction.gene_reaction_rule = '( PIP4K2A or PIP4K2B or PIP4K2C or PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('RE2974G')
reaction.name = 'RE2974'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2821: -1.0, Cpd3151: -1.0, Cpd140: 1.0, Cpd2823: 1.0, Cpd2953: 1.0})
reaction.gene_reaction_rule = '( PIP5K1A or PIP5K1B or PIP5K1C )'
model.add_reactions([reaction])
reaction = Reaction('RE2974R')
reaction.name = 'RE2974'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd1967: -1.0, Cpd1959: 1.0, Cpd565: 1.0, Cpd1965: 1.0})
reaction.gene_reaction_rule = '( PIP4K2A or PIP4K2B or PIP4K2C )'
model.add_reactions([reaction])
reaction = Reaction('RE2975M')
reaction.name = 'RE2975'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd3152: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd3153: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2977C')
reaction.name = 'RE2977'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd3154: -1.0, Cpd25: 1.0, Cpd3155: 1.0})
reaction.gene_reaction_rule = '( AKR1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2985M')
reaction.name = 'RE2985'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd613: -1.0, Cpd634: 1.0, Cpd3156: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2985X')
reaction.name = 'RE2985'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd1080: -1.0, Cpd247: 1.0, Cpd3157: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2986X')
reaction.name = 'RE2986'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3158: -1.0, Cpd3159: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2987X')
reaction.name = 'RE2987'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3159: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3160: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2988X')
reaction.name = 'RE2988'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3160: -1.0, Cpd182: 1.0, Cpd1073: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE2989X')
reaction.name = 'RE2989'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3161: -1.0, Cpd182: 1.0, Cpd3162: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE2990X')
reaction.name = 'RE2990'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3163: -1.0, Cpd3164: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2991X')
reaction.name = 'RE2991'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3164: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3161: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE2992M')
reaction.name = 'RE2992'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3165: -1.0, Cpd2831: 1.0})
reaction.gene_reaction_rule = '( ECI1 or ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2992X')
reaction.name = 'RE2992'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3166: -1.0, Cpd2833: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2993X')
reaction.name = 'RE2993'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3167: -1.0, Cpd1517: 1.0, Cpd3166: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2994X')
reaction.name = 'RE2994'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3162: -1.0, Cpd247: 1.0, Cpd3167: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2995M')
reaction.name = 'RE2995'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3156: -1.0, Cpd3168: 1.0})
reaction.gene_reaction_rule = '( ECH1 or ECI1 or ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2995X')
reaction.name = 'RE2995'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3157: -1.0, Cpd3169: 1.0})
reaction.gene_reaction_rule = '( ECH1 or ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE2996X')
reaction.name = 'RE2996'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3157: -1.0, Cpd1517: 1.0, Cpd3170: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('RE2997X')
reaction.name = 'RE2997'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd1073: -1.0, Cpd247: 1.0, Cpd3163: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE2998M')
reaction.name = 'RE2998'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd2834: -1.0, Cpd634: 1.0, Cpd2835: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE2999M')
reaction.name = 'RE2999'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2835: -1.0, Cpd3171: 1.0})
reaction.gene_reaction_rule = '( ECI1 or ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3000M')
reaction.name = 'RE3000'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3171: -1.0, Cpd3172: 1.0})
reaction.gene_reaction_rule = '( ECH1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3001M')
reaction.name = 'RE3001'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3172: -1.0, Cpd13: 1.0, Cpd3173: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3002X')
reaction.name = 'RE3002'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3158: -1.0, Cpd3170: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3003M')
reaction.name = 'RE3003'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3174: -1.0, Cpd3173: 1.0})
reaction.gene_reaction_rule = '( ECI1 or ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3004M')
reaction.name = 'RE3004'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3174: -1.0, Cpd3175: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3005M')
reaction.name = 'RE3005'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3175: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3176: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3006M')
reaction.name = 'RE3006'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3176: -1.0, Cpd178: 1.0, Cpd3177: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3009C')
reaction.name = 'RE3009'
reaction.subsystem = 'Methionine and cysteine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd23: -1.0, Cpd2052: -1.0, Cpd33: 2.0, Cpd26: 1.0, Cpd1161: 1.0, Cpd2053: 1.0})
reaction.gene_reaction_rule = '( SULT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3010C')
reaction.name = 'RE3010'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd1635: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd3178: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3010M')
reaction.name = 'RE3010'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3179: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd2901: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3010X')
reaction.name = 'RE3010'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3180: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd2905: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3011M')
reaction.name = 'RE3011'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3181: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3182: 1.0})
reaction.gene_reaction_rule = '( ADHFE1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3012C')
reaction.name = 'RE3012'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -1.0, Cpd3183: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd3184: 1.0})
reaction.gene_reaction_rule = '( ALDH3A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3012M')
reaction.name = 'RE3012'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd44: -1.0, Cpd3182: -1.0, Cpd7: 2.0, Cpd47: 1.0, Cpd3185: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH7A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3012R')
reaction.name = 'RE3012'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd625: -1.0, Cpd3186: -1.0, Cpd565: 2.0, Cpd626: 1.0, Cpd3187: 1.0})
reaction.gene_reaction_rule = '( ALDH3A1 or ALDH3A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3013R')
reaction.name = 'RE3013'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1520: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3188: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3014R')
reaction.name = 'RE3014'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3189: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3190: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3015R')
reaction.name = 'RE3015'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3190: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3191: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3018R')
reaction.name = 'RE3018'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd2903: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3192: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3019R')
reaction.name = 'RE3019'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3192: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3193: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3020R')
reaction.name = 'RE3020'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3193: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3194: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3033N')
reaction.name = 'RE3033'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3195: -1.0, Cpd2146: 1.0, Cpd3196: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3036C')
reaction.name = 'RE3036'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd430: -1.0, Cpd3197: 1.0})
reaction.gene_reaction_rule = '( ALOX12 )'
model.add_reactions([reaction])
reaction = Reaction('RE3038C')
reaction.name = 'RE3038'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3198: -1.0, Cpd3199: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3038R')
reaction.name = 'RE3038'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3200: -1.0, Cpd3201: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3038X')
reaction.name = 'RE3038'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3202: -1.0, Cpd3203: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3040C')
reaction.name = 'RE3040'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3198: -1.0, Cpd3204: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3040R')
reaction.name = 'RE3040'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3200: -1.0, Cpd3205: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3040X')
reaction.name = 'RE3040'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3202: -1.0, Cpd3206: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3041C')
reaction.name = 'RE3041'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd22: -1.0, Cpd432: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd3198: 1.0})
reaction.gene_reaction_rule = '( ALOX12 )'
model.add_reactions([reaction])
reaction = Reaction('RE3041N')
reaction.name = 'RE3041'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd763: -1.0, Cpd2147: -1.0, Cpd2150: -1.0, Cpd3207: -1.0, Cpd570: 1.0, Cpd2151: 1.0, Cpd3196: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3044N')
reaction.name = 'RE3044'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3208: -2.0, Cpd3209: 2.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3066X')
reaction.name = 'RE3066'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd1538: -1.0, Cpd3210: -1.0, Cpd943: 1.0, Cpd1947: 1.0, Cpd3211: 1.0})
reaction.gene_reaction_rule = '( PHYH )'
model.add_reactions([reaction])
reaction = Reaction('RE3072X')
reaction.name = 'RE3072'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3211: -1.0, Cpd3212: 1.0, Cpd3213: 1.0})
reaction.gene_reaction_rule = '( HACL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3074X')
reaction.name = 'RE3074'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3214: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3111: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3075X')
reaction.name = 'RE3075'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3215: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3210: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3079X')
reaction.name = 'RE3079'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3214: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3112: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3081X')
reaction.name = 'RE3081'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3216: -1.0, Cpd3217: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3082X')
reaction.name = 'RE3082'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3217: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3218: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3083X')
reaction.name = 'RE3083'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3219: -1.0, Cpd3220: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('RE3084X')
reaction.name = 'RE3084'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3221: -1.0, Cpd247: 1.0, Cpd3222: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3086X')
reaction.name = 'RE3086'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3111: -1.0, Cpd247: 1.0, Cpd3216: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3087X')
reaction.name = 'RE3087'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3222: -1.0, Cpd3223: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3088X')
reaction.name = 'RE3088'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3223: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3224: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3089X')
reaction.name = 'RE3089'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3219: -1.0, Cpd247: 1.0, Cpd3225: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3090X')
reaction.name = 'RE3090'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3226: -1.0, Cpd742: 1.0, Cpd3227: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3092X')
reaction.name = 'RE3092'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3225: -1.0, Cpd3228: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3093X')
reaction.name = 'RE3093'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3228: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3226: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3095C')
reaction.name = 'RE3095'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd5: -1.0, Cpd75: -1.0, Cpd2: 2.0, Cpd3229: 1.0})
reaction.gene_reaction_rule = '( PRDX6 or TPO )'
model.add_reactions([reaction])
reaction = Reaction('RE3095L')
reaction.name = 'RE3095'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1995: -1.0, Cpd3230: -1.0, Cpd86: 2.0, Cpd3231: 1.0})
reaction.gene_reaction_rule = '( MPO or PRDX6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3097X')
reaction.name = 'RE3097'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3224: -1.0, Cpd182: 1.0, Cpd3220: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3103C')
reaction.name = 'RE3103'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd829: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3232: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3103R')
reaction.name = 'RE3103'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd2541: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3233: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3104R')
reaction.name = 'RE3104'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3234: -1.0, Cpd589: 1.0, Cpd3235: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3106R')
reaction.name = 'RE3106'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3236: -1.0, Cpd589: 1.0, Cpd3237: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3110C')
reaction.name = 'RE3110'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3238: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3239: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3110R')
reaction.name = 'RE3110'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3240: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3241: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3111M')
reaction.name = 'RE3111'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd44: -1.0, Cpd3242: -1.0, Cpd47: 1.0, Cpd3243: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3111R')
reaction.name = 'RE3111'
reaction.subsystem = 'Androgen and estrogen synthesis and metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd625: -1.0, Cpd3244: -1.0, Cpd2157: 1.0, Cpd626: 1.0})
reaction.gene_reaction_rule = '( HSD3B2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3112R')
reaction.name = 'RE3112'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3241: -1.0, Cpd589: 1.0, Cpd3245: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3113R')
reaction.name = 'RE3113'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3245: -1.0, Cpd437: 1.0, Cpd3236: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3114R')
reaction.name = 'RE3114'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd2555: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3246: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3119R')
reaction.name = 'RE3119'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3247: -1.0, Cpd589: 1.0, Cpd3248: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3120C')
reaction.name = 'RE3120'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3249: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3250: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3120R')
reaction.name = 'RE3120'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3251: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3252: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3121R')
reaction.name = 'RE3121'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3252: -1.0, Cpd589: 1.0, Cpd3253: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3122R')
reaction.name = 'RE3122'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3253: -1.0, Cpd437: 1.0, Cpd3247: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3123R')
reaction.name = 'RE3123'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3254: -1.0, Cpd589: 1.0, Cpd3255: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3124C')
reaction.name = 'RE3124'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3256: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3257: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3124R')
reaction.name = 'RE3124'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3248: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3258: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3125R')
reaction.name = 'RE3125'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3258: -1.0, Cpd589: 1.0, Cpd3259: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3126R')
reaction.name = 'RE3126'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3259: -1.0, Cpd437: 1.0, Cpd3254: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3132C')
reaction.name = 'RE3132'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3260: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3249: 1.0})
reaction.gene_reaction_rule = '( ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3132R')
reaction.name = 'RE3132'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3261: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3251: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3134C')
reaction.name = 'RE3134'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd3262: -1.0, Cpd25: 1.0, Cpd3263: 1.0})
reaction.gene_reaction_rule = '( TM7SF2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3134R')
reaction.name = 'RE3134'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3264: -1.0, Cpd589: 1.0, Cpd3265: 1.0})
reaction.gene_reaction_rule = '( TM7SF2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3136C')
reaction.name = 'RE3136'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3266: -1.0, Cpd3267: 1.0})
reaction.gene_reaction_rule = '( EBP )'
model.add_reactions([reaction])
reaction = Reaction('RE3139X')
reaction.name = 'RE3139'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3268: -1.0, Cpd182: 1.0, Cpd3269: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3140X')
reaction.name = 'RE3140'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3270: -1.0, Cpd3271: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3141X')
reaction.name = 'RE3141'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3271: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3268: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3144M')
reaction.name = 'RE3144'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd8: -1.0, Cpd9: -1.0, Cpd3272: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd3273: 1.0})
reaction.gene_reaction_rule = '( CYP24A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3145X')
reaction.name = 'RE3145'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3274: -1.0, Cpd247: 1.0, Cpd3270: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3146C')
reaction.name = 'RE3146'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3275: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3276: 1.0})
reaction.gene_reaction_rule = '( ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3146R')
reaction.name = 'RE3146'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3277: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3278: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3147R')
reaction.name = 'RE3147'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3279: -1.0, Cpd589: 1.0, Cpd3280: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3148C')
reaction.name = 'RE3148'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3281: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3257: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3148R')
reaction.name = 'RE3148'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3280: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3258: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3150R')
reaction.name = 'RE3150'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3282: -1.0, Cpd589: 1.0, Cpd3283: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3151C')
reaction.name = 'RE3151'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd610: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3284: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3151R')
reaction.name = 'RE3151'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd2363: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3285: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3152R')
reaction.name = 'RE3152'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3285: -1.0, Cpd589: 1.0, Cpd3286: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3153R')
reaction.name = 'RE3153'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3286: -1.0, Cpd437: 1.0, Cpd3287: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3154R')
reaction.name = 'RE3154'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3288: -1.0, Cpd589: 1.0, Cpd3289: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3155R')
reaction.name = 'RE3155'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3289: -1.0, Cpd437: 1.0, Cpd3282: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3156X')
reaction.name = 'RE3156'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3290: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3291: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3157X')
reaction.name = 'RE3157'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3292: -1.0, Cpd247: 1.0, Cpd3293: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3158X')
reaction.name = 'RE3158'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3291: -1.0, Cpd182: 1.0, Cpd3294: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3159X')
reaction.name = 'RE3159'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3293: -1.0, Cpd3290: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3160C')
reaction.name = 'RE3160'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3295: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3296: 1.0})
reaction.gene_reaction_rule = '( ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3160R')
reaction.name = 'RE3160'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3297: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3298: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3161C')
reaction.name = 'RE3161'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd828: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3299: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3161R')
reaction.name = 'RE3161'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd2359: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3300: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3162R')
reaction.name = 'RE3162'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3300: -1.0, Cpd589: 1.0, Cpd3301: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3163R')
reaction.name = 'RE3163'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3301: -1.0, Cpd437: 1.0, Cpd3302: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3164R')
reaction.name = 'RE3164'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3302: -1.0, Cpd589: 1.0, Cpd3303: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3165C')
reaction.name = 'RE3165'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd830: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3304: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3165R')
reaction.name = 'RE3165'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3235: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3305: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3166R')
reaction.name = 'RE3166'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3305: -1.0, Cpd589: 1.0, Cpd3306: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3167R')
reaction.name = 'RE3167'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3306: -1.0, Cpd437: 1.0, Cpd3307: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3168R')
reaction.name = 'RE3168'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3307: -1.0, Cpd589: 1.0, Cpd3308: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3169R')
reaction.name = 'RE3169'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3309: -1.0, Cpd589: 1.0, Cpd3310: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3170C')
reaction.name = 'RE3170'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3311: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3312: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3170R')
reaction.name = 'RE3170'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3237: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3313: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3171R')
reaction.name = 'RE3171'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3313: -1.0, Cpd589: 1.0, Cpd3314: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3172R')
reaction.name = 'RE3172'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3314: -1.0, Cpd437: 1.0, Cpd3309: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3173R')
reaction.name = 'RE3173'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3315: -1.0, Cpd589: 1.0, Cpd3316: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3174C')
reaction.name = 'RE3174'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3317: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3318: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3174R')
reaction.name = 'RE3174'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3310: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3319: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3175R')
reaction.name = 'RE3175'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3319: -1.0, Cpd589: 1.0, Cpd3289: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3176R')
reaction.name = 'RE3176'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3289: -1.0, Cpd437: 1.0, Cpd3315: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3177M')
reaction.name = 'RE3177'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3320: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3321: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HADHB or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3178M')
reaction.name = 'RE3178'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3322: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3323: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HADHB or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3179M')
reaction.name = 'RE3179'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3324: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3325: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HADHA or HADHB or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3184M')
reaction.name = 'RE3184'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3326: -1.0, Cpd240: 1.0, Cpd3327: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACADM or ACADSB )'
model.add_reactions([reaction])
reaction = Reaction('RE3185M')
reaction.name = 'RE3185'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3328: -1.0, Cpd240: 1.0, Cpd3329: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACADM or ACADSB )'
model.add_reactions([reaction])
reaction = Reaction('RE3186M')
reaction.name = 'RE3186'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3330: -1.0, Cpd240: 1.0, Cpd3331: 1.0})
reaction.gene_reaction_rule = '( ACADL or ACADM or ACADSB )'
model.add_reactions([reaction])
reaction = Reaction('RE3189M')
reaction.name = 'RE3189'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3321: -1.0, Cpd178: 1.0, Cpd3332: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3190M')
reaction.name = 'RE3190'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3323: -1.0, Cpd179: 1.0, Cpd3330: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3191M')
reaction.name = 'RE3191'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3325: -1.0, Cpd178: 1.0, Cpd244: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3192M')
reaction.name = 'RE3192'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3327: -1.0, Cpd3320: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3193M')
reaction.name = 'RE3193'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3329: -1.0, Cpd3322: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3194M')
reaction.name = 'RE3194'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3331: -1.0, Cpd3324: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3195M')
reaction.name = 'RE3195'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3328: -1.0, Cpd3332: 1.0})
reaction.gene_reaction_rule = '( AMACR )'
model.add_reactions([reaction])
reaction = Reaction('RE3224C')
reaction.name = 'RE3224'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd606: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3333: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3224R')
reaction.name = 'RE3224'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd2538: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3334: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3225C')
reaction.name = 'RE3225'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3335: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3336: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3225R')
reaction.name = 'RE3225'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3337: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3338: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3226C')
reaction.name = 'RE3226'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd3339: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3340: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3226R')
reaction.name = 'RE3226'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd3341: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3342: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3227R')
reaction.name = 'RE3227'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3334: -1.0, Cpd589: 1.0, Cpd3343: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3228R')
reaction.name = 'RE3228'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3343: -1.0, Cpd437: 1.0, Cpd3344: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3229R')
reaction.name = 'RE3229'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3344: -1.0, Cpd589: 1.0, Cpd3337: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3230R')
reaction.name = 'RE3230'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3338: -1.0, Cpd589: 1.0, Cpd3345: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3231R')
reaction.name = 'RE3231'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3345: -1.0, Cpd437: 1.0, Cpd3346: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3232R')
reaction.name = 'RE3232'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3346: -1.0, Cpd589: 1.0, Cpd3341: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3233G')
reaction.name = 'RE3233'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3347: -1.0, Cpd3348: -1.0, Cpd525: 1.0, Cpd3349: 1.0})
reaction.gene_reaction_rule = '( TYR )'
model.add_reactions([reaction])
reaction = Reaction('RE3233L')
reaction.name = 'RE3233'
reaction.subsystem = 'Tyrosine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3350: -1.0, Cpd3351: -1.0, Cpd86: 1.0, Cpd3352: 1.0})
reaction.gene_reaction_rule = '( TYR )'
model.add_reactions([reaction])
reaction = Reaction('RE3234R')
reaction.name = 'RE3234'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3342: -1.0, Cpd589: 1.0, Cpd3353: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3235R')
reaction.name = 'RE3235'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3353: -1.0, Cpd437: 1.0, Cpd3354: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3236R')
reaction.name = 'RE3236'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3354: -1.0, Cpd589: 1.0, Cpd3355: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3241C')
reaction.name = 'RE3241'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd199: -1.0, Cpd2368: -1.0, Cpd60: 1.0, Cpd78: 1.0, Cpd3356: 1.0})
reaction.gene_reaction_rule = '( ELOVL2 or ELOVL4 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3241R')
reaction.name = 'RE3241'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd2352: -1.0, Cpd2366: -1.0, Cpd624: 1.0, Cpd745: 1.0, Cpd3357: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 or ELOVL2 or ELOVL5 or ELOVL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3242R')
reaction.name = 'RE3242'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3357: -1.0, Cpd589: 1.0, Cpd3358: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3243R')
reaction.name = 'RE3243'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3358: -1.0, Cpd437: 1.0, Cpd3359: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3244R')
reaction.name = 'RE3244'
reaction.subsystem = 'Fatty acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3359: -1.0, Cpd589: 1.0, Cpd2538: 1.0})
reaction.gene_reaction_rule = '( ELOVL1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3247M')
reaction.name = 'RE3247'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3084: -1.0, Cpd634: 1.0, Cpd3360: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3247X')
reaction.name = 'RE3247'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3086: -1.0, Cpd247: 1.0, Cpd262: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3248M')
reaction.name = 'RE3248'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3360: -1.0, Cpd3361: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3248X')
reaction.name = 'RE3248'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd262: -1.0, Cpd3362: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3250M')
reaction.name = 'RE3250'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3361: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3363: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3250X')
reaction.name = 'RE3250'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3362: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3364: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3251M')
reaction.name = 'RE3251'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd9: -1.0, Cpd1861: -1.0, Cpd12: 1.0, Cpd13: 1.0, Cpd3365: 1.0})
reaction.gene_reaction_rule = '( CYP27A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3258C')
reaction.name = 'RE3258'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3366: -1.0, Cpd3367: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3258R')
reaction.name = 'RE3258'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3368: -1.0, Cpd3369: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3259C')
reaction.name = 'RE3259'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3370: -1.0, Cpd3371: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3259R')
reaction.name = 'RE3259'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3372: -1.0, Cpd3373: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3260C')
reaction.name = 'RE3260'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd3367: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd3374: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3260R')
reaction.name = 'RE3260'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd3369: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd3375: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE3261C')
reaction.name = 'RE3261'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd3371: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd3376: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3261R')
reaction.name = 'RE3261'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd3373: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd3377: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE3264C')
reaction.name = 'RE3264'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd3367: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd3378: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3264R')
reaction.name = 'RE3264'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd3369: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd3379: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE3265C')
reaction.name = 'RE3265'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1467: -1.0, Cpd3371: -1.0, Cpd33: 1.0, Cpd137: 1.0, Cpd3380: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3265R')
reaction.name = 'RE3265'
reaction.subsystem = 'Linoleate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2244: -1.0, Cpd3373: -1.0, Cpd565: 1.0, Cpd1769: 1.0, Cpd3381: 1.0})
reaction.gene_reaction_rule = '( UGT1A1 or UGT1A2 or UGT1A7C or UGT1A8 or UGT2B34 )'
model.add_reactions([reaction])
reaction = Reaction('RE3267G')
reaction.name = 'RE3267'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3382: -1.0, Cpd3383: -1.0, Cpd140: 1.0, Cpd712: 1.0, Cpd3384: 1.0})
reaction.gene_reaction_rule = '( CHPT1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3267N')
reaction.name = 'RE3267'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3385: -1.0, Cpd3386: -1.0, Cpd763: 1.0, Cpd772: 1.0, Cpd3387: 1.0})
reaction.gene_reaction_rule = '( CEPT1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3267R')
reaction.name = 'RE3267'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3388: -1.0, Cpd3389: -1.0, Cpd565: 1.0, Cpd1452: 1.0, Cpd3390: 1.0})
reaction.gene_reaction_rule = '( CEPT1 or SELENOI )'
model.add_reactions([reaction])
reaction = Reaction('RE3268R')
reaction.name = 'RE3268'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd2955: -1.0, Cpd915: 1.0, Cpd1965: 1.0})
reaction.gene_reaction_rule = '( TPTE )'
model.add_reactions([reaction])
reaction = Reaction('RE3269C')
reaction.name = 'RE3269'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd1955: -1.0, Cpd108: 1.0, Cpd1963: 1.0})
reaction.gene_reaction_rule = '( MTMR2 or MTMR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3273C')
reaction.name = 'RE3273'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd652: -1.0, Cpd33: 1.0, Cpd353: 1.0, Cpd650: 1.0})
reaction.gene_reaction_rule = '( PLD2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3273G')
reaction.name = 'RE3273'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd2820: -1.0, Cpd140: 1.0, Cpd1906: 1.0, Cpd3391: 1.0})
reaction.gene_reaction_rule = '( PLD1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3273R')
reaction.name = 'RE3273'
reaction.subsystem = 'Phosphatidylinositol phosphate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd1966: -1.0, Cpd565: 1.0, Cpd1909: 1.0, Cpd3392: 1.0})
reaction.gene_reaction_rule = '( PLD1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3286R')
reaction.name = 'RE3286'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd1885: -1.0, Cpd847: -1.0, Cpd437: 1.0, Cpd848: 1.0, Cpd3393: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3287R')
reaction.name = 'RE3287'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd1885: -1.0, Cpd847: -1.0, Cpd437: 1.0, Cpd848: 1.0, Cpd3394: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3288R')
reaction.name = 'RE3288'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd1885: -1.0, Cpd847: -1.0, Cpd437: 1.0, Cpd848: 1.0, Cpd3395: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3289R')
reaction.name = 'RE3289'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd1885: -1.0, Cpd847: -1.0, Cpd437: 1.0, Cpd848: 1.0, Cpd3396: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3301C')
reaction.name = 'RE3301'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd2020: -1.0, Cpd33: 1.0, Cpd353: 1.0, Cpd404: 1.0})
reaction.gene_reaction_rule = '( PLD2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3301G')
reaction.name = 'RE3301'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd525: -1.0, Cpd3397: -1.0, Cpd140: 1.0, Cpd1906: 1.0, Cpd3398: 1.0})
reaction.gene_reaction_rule = '( PLD1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3301R')
reaction.name = 'RE3301'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3399: -1.0, Cpd565: 1.0, Cpd1909: 1.0, Cpd3400: 1.0})
reaction.gene_reaction_rule = '( PLD1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3307C')
reaction.name = 'RE3307'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd3401: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd3402: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3307M')
reaction.name = 'RE3307'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd3403: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd3404: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3307X')
reaction.name = 'RE3307'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1517: -1.0, Cpd3405: -1.0, Cpd358: 1.0, Cpd1516: 1.0, Cpd3406: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3308R')
reaction.name = 'RE3308'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd3407: -1.0, Cpd589: 1.0, Cpd3408: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3310R')
reaction.name = 'RE3310'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd589: -1.0, Cpd3409: -1.0, Cpd565: 1.0, Cpd587: 1.0, Cpd3410: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3326M')
reaction.name = 'RE3326'
reaction.subsystem = 'Valine, leucine, and isoleucine metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd244: -1.0, Cpd1299: -1.0, Cpd43: 1.0, Cpd2345: 1.0})
reaction.gene_reaction_rule = '( BCKDHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3334M')
reaction.name = 'RE3334'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3411: -1.0, Cpd634: 1.0, Cpd3412: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3334X')
reaction.name = 'RE3334'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3413: -1.0, Cpd247: 1.0, Cpd3414: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3335C')
reaction.name = 'RE3335'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3415: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3416: 1.0})
reaction.gene_reaction_rule = '( ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3335M')
reaction.name = 'RE3335'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd3417: -1.0, Cpd209: 1.0, Cpd210: 1.0, Cpd3411: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3335R')
reaction.name = 'RE3335'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3408: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3418: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3335X')
reaction.name = 'RE3335'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3419: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3413: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3336M')
reaction.name = 'RE3336'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3420: -1.0, Cpd178: 1.0, Cpd3421: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3336X')
reaction.name = 'RE3336'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3422: -1.0, Cpd182: 1.0, Cpd3423: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3337M')
reaction.name = 'RE3337'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3412: -1.0, Cpd3424: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3337X')
reaction.name = 'RE3337'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3414: -1.0, Cpd3425: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3338C')
reaction.name = 'RE3338'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd3426: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd3427: 1.0})
reaction.gene_reaction_rule = '( HSD17B10 )'
model.add_reactions([reaction])
reaction = Reaction('RE3338M')
reaction.name = 'RE3338'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3424: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3420: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3338X')
reaction.name = 'RE3338'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3425: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3422: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3339C')
reaction.name = 'RE3339'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3428: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd3429: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3339M')
reaction.name = 'RE3339'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3421: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3430: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE3339X')
reaction.name = 'RE3339'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3423: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3431: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3340C')
reaction.name = 'RE3340'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd3432: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd3433: 1.0})
reaction.gene_reaction_rule = '( HSD17B10 )'
model.add_reactions([reaction])
reaction = Reaction('RE3340M')
reaction.name = 'RE3340'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3434: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3435: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3340X')
reaction.name = 'RE3340'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3436: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3437: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3341M')
reaction.name = 'RE3341'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3438: -1.0, Cpd634: 1.0, Cpd3439: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3341X')
reaction.name = 'RE3341'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3440: -1.0, Cpd247: 1.0, Cpd3441: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3342M')
reaction.name = 'RE3342'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3435: -1.0, Cpd178: 1.0, Cpd3442: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3342X')
reaction.name = 'RE3342'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3437: -1.0, Cpd182: 1.0, Cpd3443: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3343C')
reaction.name = 'RE3343'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3444: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd3445: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3343M')
reaction.name = 'RE3343'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3442: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3446: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE3343X')
reaction.name = 'RE3343'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3443: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3447: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3344M')
reaction.name = 'RE3344'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3439: -1.0, Cpd3434: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3344X')
reaction.name = 'RE3344'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3441: -1.0, Cpd3436: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3345C')
reaction.name = 'RE3345'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3448: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3449: 1.0})
reaction.gene_reaction_rule = '( ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3345M')
reaction.name = 'RE3345'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd3417: -1.0, Cpd209: 1.0, Cpd210: 1.0, Cpd3438: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3345R')
reaction.name = 'RE3345'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3408: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3450: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3345X')
reaction.name = 'RE3345'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3419: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3440: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3346C')
reaction.name = 'RE3346'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd35: -1.0, Cpd2664: -1.0, Cpd2: 1.0, Cpd32: 1.0, Cpd3451: 1.0})
reaction.gene_reaction_rule = '( ALDH3A1 or ALDH9A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3346M')
reaction.name = 'RE3346'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -2.0, Cpd47: -1.0, Cpd1863: -1.0, Cpd12: 1.0, Cpd44: 1.0, Cpd3088: 1.0})
reaction.gene_reaction_rule = '( ALDH1B1 or ALDH2 or ALDH7A1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3346R')
reaction.name = 'RE3346'
reaction.subsystem = 'Bile acid synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -2.0, Cpd626: -1.0, Cpd2287: -1.0, Cpd437: 1.0, Cpd625: 1.0, Cpd3452: 1.0})
reaction.gene_reaction_rule = '( ALDH3A1 or ALDH3A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3367C')
reaction.name = 'RE3367'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3: -1.0, Cpd2736: -1.0, Cpd5: 1.0, Cpd6: 1.0, Cpd2721: 1.0})
reaction.gene_reaction_rule = '( AOC3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3367E')
reaction.name = 'RE3367'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd2143: -1.0, Cpd2737: -1.0, Cpd278: 1.0, Cpd1444: 1.0, Cpd2722: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3367X')
reaction.name = 'RE3367'
reaction.subsystem = 'Urea cycle'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd261: -1.0, Cpd2738: -1.0, Cpd247: 1.0, Cpd800: 1.0, Cpd2723: 1.0})
reaction.gene_reaction_rule = '( AOC1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3370R')
reaction.name = 'RE3370'
reaction.subsystem = 'Vitamin E metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd565: -1.0, Cpd587: -2.0, Cpd3453: -1.0, Cpd437: 3.0, Cpd589: 2.0, Cpd3454: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3381C')
reaction.name = 'RE3381'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3028: -1.0, Cpd33: 1.0, Cpd1401: 1.0, Cpd3027: 1.0})
reaction.gene_reaction_rule = '( KL )'
model.add_reactions([reaction])
reaction = Reaction('RE3381E')
reaction.name = 'RE3381'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd3455: -1.0, Cpd37: 1.0, Cpd3076: 1.0, Cpd3456: 1.0})
reaction.gene_reaction_rule = '( KL )'
model.add_reactions([reaction])
reaction = Reaction('RE3381L')
reaction.name = 'RE3381'
reaction.subsystem = 'Miscellaneous'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd86: -1.0, Cpd3457: -1.0, Cpd176: 1.0, Cpd1357: 1.0, Cpd3458: 1.0})
reaction.gene_reaction_rule = '( GUSB )'
model.add_reactions([reaction])
reaction = Reaction('RE3383M')
reaction.name = 'RE3383'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3430: -1.0, Cpd178: 1.0, Cpd3459: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3384M')
reaction.name = 'RE3384'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3460: -1.0, Cpd178: 1.0, Cpd3461: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3385M')
reaction.name = 'RE3385'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3459: -1.0, Cpd634: 1.0, Cpd3462: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3386M')
reaction.name = 'RE3386'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3463: -1.0, Cpd3464: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3387M')
reaction.name = 'RE3387'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3464: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3460: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3388M')
reaction.name = 'RE3388'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3465: -1.0, Cpd178: 1.0, Cpd3466: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3389M')
reaction.name = 'RE3389'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3461: -1.0, Cpd634: 1.0, Cpd3467: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3390M')
reaction.name = 'RE3390'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3468: -1.0, Cpd3469: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3391M')
reaction.name = 'RE3391'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3462: -1.0, Cpd13: 1.0, Cpd3463: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3392M')
reaction.name = 'RE3392'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3467: -1.0, Cpd13: 1.0, Cpd3468: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3393M')
reaction.name = 'RE3393'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3469: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3465: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3394M')
reaction.name = 'RE3394'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3470: -1.0, Cpd178: 1.0, Cpd3471: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3395M')
reaction.name = 'RE3395'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3471: -1.0, Cpd634: 1.0, Cpd3472: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3396M')
reaction.name = 'RE3396'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3446: -1.0, Cpd178: 1.0, Cpd3473: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3397M')
reaction.name = 'RE3397'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3473: -1.0, Cpd634: 1.0, Cpd3474: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3398M')
reaction.name = 'RE3398'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3475: -1.0, Cpd3476: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3399M')
reaction.name = 'RE3399'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3476: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3470: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3400M')
reaction.name = 'RE3400'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3477: -1.0, Cpd178: 1.0, Cpd3478: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3401M')
reaction.name = 'RE3401'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3479: -1.0, Cpd3480: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3402M')
reaction.name = 'RE3402'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3480: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3477: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3403M')
reaction.name = 'RE3403'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3474: -1.0, Cpd13: 1.0, Cpd3475: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3404M')
reaction.name = 'RE3404'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3472: -1.0, Cpd13: 1.0, Cpd3479: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3430C')
reaction.name = 'RE3430'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd35: -1.0, Cpd3178: -1.0, Cpd32: 1.0, Cpd3481: 1.0})
reaction.gene_reaction_rule = '( CBR1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3432C')
reaction.name = 'RE3432'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd3481: -1.0, Cpd25: 1.0, Cpd3482: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3432M')
reaction.name = 'RE3432'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3483: -1.0, Cpd13: 1.0, Cpd3484: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3432X')
reaction.name = 'RE3432'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3485: -1.0, Cpd1517: 1.0, Cpd3486: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3434R')
reaction.name = 'RE3434'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3487: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3488: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3435R')
reaction.name = 'RE3435'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3489: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3490: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3436R')
reaction.name = 'RE3436'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3488: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3489: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3440R')
reaction.name = 'RE3440'
reaction.subsystem = 'Vitamin E metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd565: -1.0, Cpd587: -2.0, Cpd3491: -1.0, Cpd437: 3.0, Cpd589: 2.0, Cpd3492: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3443M')
reaction.name = 'RE3443'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3493: -1.0, Cpd178: 1.0, Cpd3494: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3443X')
reaction.name = 'RE3443'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3495: -1.0, Cpd182: 1.0, Cpd3496: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3444C')
reaction.name = 'RE3444'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3497: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd3498: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3444M')
reaction.name = 'RE3444'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3494: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3499: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE3444X')
reaction.name = 'RE3444'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3496: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3500: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3445M')
reaction.name = 'RE3445'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3501: -1.0, Cpd634: 1.0, Cpd3502: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3445X')
reaction.name = 'RE3445'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3503: -1.0, Cpd247: 1.0, Cpd3504: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3446C')
reaction.name = 'RE3446'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3482: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3505: 1.0})
reaction.gene_reaction_rule = '( ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3446M')
reaction.name = 'RE3446'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd3484: -1.0, Cpd209: 1.0, Cpd210: 1.0, Cpd3501: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3446R')
reaction.name = 'RE3446'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3487: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3506: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3446X')
reaction.name = 'RE3446'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3486: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3503: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3447M')
reaction.name = 'RE3447'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3502: -1.0, Cpd3507: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3447X')
reaction.name = 'RE3447'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3504: -1.0, Cpd3508: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3448C')
reaction.name = 'RE3448'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd3509: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd3510: 1.0})
reaction.gene_reaction_rule = '( HSD17B10 )'
model.add_reactions([reaction])
reaction = Reaction('RE3448M')
reaction.name = 'RE3448'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3507: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3493: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3448X')
reaction.name = 'RE3448'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3508: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3495: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3464R')
reaction.name = 'RE3464'
reaction.subsystem = 'Vitamin E metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3511: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3453: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3470C')
reaction.name = 'RE3470'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd1635: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd3512: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3470M')
reaction.name = 'RE3470'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd3179: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd3513: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3470X')
reaction.name = 'RE3470'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1517: -1.0, Cpd3180: -1.0, Cpd358: 1.0, Cpd1516: 1.0, Cpd3514: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3474R')
reaction.name = 'RE3474'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3515: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3516: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3475N')
reaction.name = 'RE3475'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3517: -2.0, Cpd3518: 2.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3476C')
reaction.name = 'RE3476'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd3519: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd3520: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3476M')
reaction.name = 'RE3476'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd13: -1.0, Cpd3521: -1.0, Cpd7: 1.0, Cpd8: 1.0, Cpd3522: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3476X')
reaction.name = 'RE3476'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1517: -1.0, Cpd3523: -1.0, Cpd358: 1.0, Cpd1516: 1.0, Cpd3524: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3485N')
reaction.name = 'RE3485'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3525: -1.0, Cpd3526: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3488N')
reaction.name = 'RE3488'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3525: -1.0, Cpd3527: 1.0})
reaction.gene_reaction_rule = '( PTGS1 or PTGS2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3488R')
reaction.name = 'RE3488'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd3261: -1.0, Cpd3528: 1.0})
reaction.gene_reaction_rule = '( PTGS1 or PTGS2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3488X')
reaction.name = 'RE3488'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3529: -1.0, Cpd3530: 1.0})
reaction.gene_reaction_rule = '( ACOX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3496N')
reaction.name = 'RE3496'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3531: -1.0, Cpd3532: 1.0})
reaction.gene_reaction_rule = '( PTGES3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3498N')
reaction.name = 'RE3498'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3533: -1.0, Cpd570: 1.0, Cpd3534: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3500C')
reaction.name = 'RE3500'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3535: -1.0, Cpd3536: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3500R')
reaction.name = 'RE3500'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3537: -1.0, Cpd3538: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3500X')
reaction.name = 'RE3500'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3539: -1.0, Cpd3540: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3501C')
reaction.name = 'RE3501'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3535: -1.0, Cpd3541: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3501R')
reaction.name = 'RE3501'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3537: -1.0, Cpd3542: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3501X')
reaction.name = 'RE3501'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3539: -1.0, Cpd3543: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3502C')
reaction.name = 'RE3502'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -1.0, Cpd3544: -1.0, Cpd33: 1.0, Cpd22: 1.0, Cpd3545: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3502X')
reaction.name = 'RE3502'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1517: -1.0, Cpd3546: -1.0, Cpd358: 1.0, Cpd1516: 1.0, Cpd3547: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3503N')
reaction.name = 'RE3503'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3548: -1.0, Cpd570: 1.0, Cpd3549: 1.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3506R')
reaction.name = 'RE3506'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3550: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3551: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3511M')
reaction.name = 'RE3511'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd442: -1.0, Cpd2666: -1.0, Cpd441: 1.0, Cpd1903: 1.0})
reaction.gene_reaction_rule = '( PEMT )'
model.add_reactions([reaction])
reaction = Reaction('RE3511R')
reaction.name = 'RE3511'
reaction.subsystem = 'Glycerophospholipid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1926: -1.0, Cpd2668: -1.0, Cpd1927: 1.0, Cpd1907: 1.0})
reaction.gene_reaction_rule = '( PEMT )'
model.add_reactions([reaction])
reaction = Reaction('RE3513N')
reaction.name = 'RE3513'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2147: -1.0, Cpd3552: -2.0, Cpd3553: 2.0})
reaction.gene_reaction_rule = '( ALOX5 )'
model.add_reactions([reaction])
reaction = Reaction('RE3514R')
reaction.name = 'RE3514'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd1888: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3554: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3518R')
reaction.name = 'RE3518'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3555: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3556: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3519C')
reaction.name = 'RE3519'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3557: -1.0, Cpd3558: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3519R')
reaction.name = 'RE3519'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3555: -1.0, Cpd3559: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3519X')
reaction.name = 'RE3519'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3560: -1.0, Cpd3561: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3520C')
reaction.name = 'RE3520'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3562: -1.0, Cpd3557: 1.0})
reaction.gene_reaction_rule = '( GPX4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3520E')
reaction.name = 'RE3520'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3563: -1.0, Cpd3564: 1.0})
reaction.gene_reaction_rule = '( GPX3 or GPX5 or GPX6 or GPX7 )'
model.add_reactions([reaction])
reaction = Reaction('RE3520M')
reaction.name = 'RE3520'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3565: -1.0, Cpd3566: 1.0})
reaction.gene_reaction_rule = '( GPX1 or GPX4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3520N')
reaction.name = 'RE3520'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3567: -1.0, Cpd3568: 1.0})
reaction.gene_reaction_rule = '( GPX4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3521M')
reaction.name = 'RE3521'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd3566: -1.0, Cpd3569: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3521R')
reaction.name = 'RE3521'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd3555: -1.0, Cpd3570: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3521X')
reaction.name = 'RE3521'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2871: -1.0, Cpd3560: -1.0, Cpd3571: 1.0})
reaction.gene_reaction_rule = '( GSTK1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3522R')
reaction.name = 'RE3522'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3572: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3573: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 or CYP4F39 )'
model.add_reactions([reaction])
reaction = Reaction('RE3525C')
reaction.name = 'RE3525'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd3519: -2.0, Cpd1635: 2.0})
reaction.gene_reaction_rule = '( ALOX12 )'
model.add_reactions([reaction])
reaction = Reaction('RE3526C')
reaction.name = 'RE3526'
reaction.subsystem = 'Arachidonic acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd3520: -2.0, Cpd3512: 2.0})
reaction.gene_reaction_rule = '( ALOX12 )'
model.add_reactions([reaction])
reaction = Reaction('RE3532M')
reaction.name = 'RE3532'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd3574: -1.0, Cpd3575: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3532R')
reaction.name = 'RE3532'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd3576: -1.0, Cpd3577: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3533M')
reaction.name = 'RE3533'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd3578: -1.0, Cpd3579: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3533R')
reaction.name = 'RE3533'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd3580: -1.0, Cpd3581: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3534M')
reaction.name = 'RE3534'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd3582: -1.0, Cpd3583: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3534R')
reaction.name = 'RE3534'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd3584: -1.0, Cpd3585: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3554M')
reaction.name = 'RE3554'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd3586: -1.0, Cpd3587: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3554R')
reaction.name = 'RE3554'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd3588: -1.0, Cpd3589: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3557M')
reaction.name = 'RE3557'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1418: -1.0, Cpd3590: -1.0, Cpd3591: 1.0})
reaction.gene_reaction_rule = '( GSTK1 or GSTZ1 or MGST1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3557R')
reaction.name = 'RE3557'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1636: -1.0, Cpd3592: -1.0, Cpd3593: 1.0})
reaction.gene_reaction_rule = '( MGST1 or MGST2 or MGST3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3559M')
reaction.name = 'RE3559'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3594: -1.0, Cpd178: 1.0, Cpd3595: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3559X')
reaction.name = 'RE3559'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3596: -1.0, Cpd182: 1.0, Cpd3597: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3560C')
reaction.name = 'RE3560'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3598: -1.0, Cpd33: 1.0, Cpd78: 1.0, Cpd3599: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT6 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3560M')
reaction.name = 'RE3560'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3595: -1.0, Cpd7: 1.0, Cpd43: 1.0, Cpd3600: 1.0})
reaction.gene_reaction_rule = '( ACOT2 or ACOT7 )'
model.add_reactions([reaction])
reaction = Reaction('RE3560X')
reaction.name = 'RE3560'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3597: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3601: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3561M')
reaction.name = 'RE3561'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3602: -1.0, Cpd634: 1.0, Cpd3603: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3561X')
reaction.name = 'RE3561'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3604: -1.0, Cpd247: 1.0, Cpd3605: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3562C')
reaction.name = 'RE3562'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3184: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3606: 1.0})
reaction.gene_reaction_rule = '( ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3562M')
reaction.name = 'RE3562'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd200: -1.0, Cpd3185: -1.0, Cpd209: 1.0, Cpd210: 1.0, Cpd3602: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 )'
model.add_reactions([reaction])
reaction = Reaction('RE3562R')
reaction.name = 'RE3562'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3187: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3607: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3562X')
reaction.name = 'RE3562'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3608: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3604: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3563M')
reaction.name = 'RE3563'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd3603: -1.0, Cpd3609: 1.0})
reaction.gene_reaction_rule = '( ECHS1 or EHHADH or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('RE3563X')
reaction.name = 'RE3563'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3605: -1.0, Cpd3610: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3564C')
reaction.name = 'RE3564'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd3611: -1.0, Cpd33: 1.0, Cpd35: 1.0, Cpd3612: 1.0})
reaction.gene_reaction_rule = '( HSD17B10 )'
model.add_reactions([reaction])
reaction = Reaction('RE3564M')
reaction.name = 'RE3564'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd44: -1.0, Cpd3609: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd3594: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3564X')
reaction.name = 'RE3564'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd3610: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd3596: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3571R')
reaction.name = 'RE3571'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -2.0, Cpd3613: -1.0, Cpd3614: 1.0})
reaction.gene_reaction_rule = '( PTGS1 or PTGS2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3572X')
reaction.name = 'RE3572'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1065: -1.0, Cpd3615: -1.0, Cpd1064: 1.0, Cpd3616: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3573X')
reaction.name = 'RE3573'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1065: -1.0, Cpd3617: -1.0, Cpd1064: 1.0, Cpd3618: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3574X')
reaction.name = 'RE3574'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3619: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3620: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3575X')
reaction.name = 'RE3575'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd3615: -1.0, Cpd182: 1.0, Cpd3619: 1.0})
reaction.gene_reaction_rule = '( ACAA1B )'
model.add_reactions([reaction])
reaction = Reaction('RE3576X')
reaction.name = 'RE3576'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3621: -1.0, Cpd247: 1.0, Cpd3622: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3577X')
reaction.name = 'RE3577'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd3623: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd3621: 1.0})
reaction.gene_reaction_rule = '( ACSL1 or ACSL3 or ACSL4 or ACSL5 or ACSL6 or SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3578X')
reaction.name = 'RE3578'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3622: -1.0, Cpd3616: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3580X')
reaction.name = 'RE3580'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3619: -1.0, Cpd247: 1.0, Cpd3624: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3581X')
reaction.name = 'RE3581'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3624: -1.0, Cpd1517: 1.0, Cpd3625: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('RE3582X')
reaction.name = 'RE3582'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3625: -1.0, Cpd3626: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3583X')
reaction.name = 'RE3583'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3626: -1.0, Cpd3618: 1.0})
reaction.gene_reaction_rule = '( EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RE3586X')
reaction.name = 'RE3586'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3627: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3628: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 or BAAT )'
model.add_reactions([reaction])
reaction = Reaction('RE3587N')
reaction.name = 'RE3587'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3629: -1.0, Cpd3630: 1.0})
reaction.gene_reaction_rule = '( PTGES3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3596C')
reaction.name = 'RE3596'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd3631: -1.0, Cpd25: 1.0, Cpd3632: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3596M')
reaction.name = 'RE3596'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3633: -1.0, Cpd13: 1.0, Cpd3634: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3596X')
reaction.name = 'RE3596'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3635: -1.0, Cpd1517: 1.0, Cpd3636: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3597C')
reaction.name = 'RE3597'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd3184: -1.0, Cpd25: 1.0, Cpd3637: 1.0})
reaction.gene_reaction_rule = '( CBR1 or CBR3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3597M')
reaction.name = 'RE3597'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3185: -1.0, Cpd13: 1.0, Cpd3638: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3597X')
reaction.name = 'RE3597'
reaction.subsystem = 'Eicosanoid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3608: -1.0, Cpd1517: 1.0, Cpd3639: 1.0})
reaction.gene_reaction_rule = '( DHRS4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3624M')
reaction.name = 'RE3624'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd3177: -1.0, Cpd634: 1.0, Cpd2842: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3624X')
reaction.name = 'RE3624'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3640: -1.0, Cpd247: 1.0, Cpd2844: 1.0})
reaction.gene_reaction_rule = '( ACOX1 or ACOX2 or ACOX3 )'
model.add_reactions([reaction])
reaction = Reaction('RE3626M')
reaction.name = 'RE3626'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd9: -1.0, Cpd2838: -1.0, Cpd634: 1.0, Cpd2839: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('RE3627C')
reaction.name = 'RE3627'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd22: -1.0, Cpd3641: -1.0, Cpd25: 1.0, Cpd3642: 1.0})
reaction.gene_reaction_rule = '( HSD17B10 )'
model.add_reactions([reaction])
reaction = Reaction('RE3627M')
reaction.name = 'RE3627'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd2839: -1.0, Cpd13: 1.0, Cpd3643: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HADH or HSD17B10 or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3627X')
reaction.name = 'RE3627'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd2841: -1.0, Cpd1517: 1.0, Cpd3644: 1.0})
reaction.gene_reaction_rule = '( EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('RE3628M')
reaction.name = 'RE3628'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2467: -1.0, Cpd3643: 1.0})
reaction.gene_reaction_rule = '( ECI1 or ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('RN0001R')
reaction.name = 'RN0001'
reaction.subsystem = 'Selenoamino acid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3645: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3646: 1.0})
reaction.gene_reaction_rule = '( FMO1 or FMO2 or FMO3 or FMO4 or FMO5 )'
model.add_reactions([reaction])
reaction = Reaction('RN0020R')
reaction.name = 'RN0020'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3647: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3648: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0021C')
reaction.name = 'RN0021'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3649: -1.0, Cpd3650: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0021R')
reaction.name = 'RN0021'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3648: -1.0, Cpd565: 2.0, Cpd3651: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0021X')
reaction.name = 'RN0021'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3652: -1.0, Cpd358: 2.0, Cpd3653: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0022C')
reaction.name = 'RN0022'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3654: -1.0, Cpd3655: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0022R')
reaction.name = 'RN0022'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3656: -1.0, Cpd565: 2.0, Cpd3657: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0022X')
reaction.name = 'RN0022'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3658: -1.0, Cpd358: 2.0, Cpd3659: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0023C')
reaction.name = 'RN0023'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3660: -1.0, Cpd3661: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0023R')
reaction.name = 'RN0023'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3662: -1.0, Cpd565: 2.0, Cpd3663: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0023X')
reaction.name = 'RN0023'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3664: -1.0, Cpd358: 2.0, Cpd3665: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0027R')
reaction.name = 'RN0027'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -3.0, Cpd587: -1.0, Cpd3666: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3667: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0028C')
reaction.name = 'RN0028'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3668: -1.0, Cpd3669: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0028R')
reaction.name = 'RN0028'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3667: -1.0, Cpd3670: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0028X')
reaction.name = 'RN0028'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3671: -1.0, Cpd3672: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0029R')
reaction.name = 'RN0029'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd587: -1.0, Cpd3670: -1.0, Cpd437: 1.0, Cpd565: 1.0, Cpd589: 1.0, Cpd3673: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0030R')
reaction.name = 'RN0030'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -3.0, Cpd587: -1.0, Cpd3674: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3675: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0031C')
reaction.name = 'RN0031'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3676: -1.0, Cpd3677: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0031R')
reaction.name = 'RN0031'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd437: -1.0, Cpd3675: -1.0, Cpd3678: 1.0})
reaction.gene_reaction_rule = '( EPHX1 )'
model.add_reactions([reaction])
reaction = Reaction('RN0031X')
reaction.name = 'RN0031'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3679: -1.0, Cpd3680: 1.0})
reaction.gene_reaction_rule = '( EPHX2 )'
model.add_reactions([reaction])
reaction = Reaction('RN0032R')
reaction.name = 'RN0032'
reaction.subsystem = 'Xenobiotics metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd587: -1.0, Cpd3678: -1.0, Cpd437: 1.0, Cpd565: 1.0, Cpd589: 1.0, Cpd3681: 1.0})
reaction.gene_reaction_rule = '( CYP19A1 or CYP1A1 or CYP1A2 or CYP1B1 or CYP2A4 or CYP2B9 or CYP2C55 or CYP2D26 or CYP2E1 or CYP2F2 or CYP2J8 or CYP2S1 or CYP3A13 or CYP4B1 or CYP4F15 or CYP4X1 )'
model.add_reactions([reaction])
reaction = Reaction('CITt4_4')
reaction.name = 'citrate transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -4.0, Cpd692: -1.0, Cpd41: 4.0, Cpd228: 1.0})
reaction.gene_reaction_rule = '( SLC13A5 )'
model.add_reactions([reaction])
reaction = Reaction('GLYt7_311_r')
reaction.name = 'glycine reversible transport via sodium and chloride symport (3:1:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -3.0, Cpd403: -1.0, Cpd698: -1.0, Cpd41: 3.0, Cpd402: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A5 )'
model.add_reactions([reaction])
reaction = Reaction('HCO3_NAt')
reaction.name = 'bicarbonate transport (Na/HCO3 cotransport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -1.0, Cpd39: -1.0, Cpd40: 1.0, Cpd41: 1.0})
reaction.gene_reaction_rule = '( SLC4A7 )'
model.add_reactions([reaction])
reaction = Reaction('INSTt4_2')
reaction.name = 'inositol transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd1563: -1.0, Cpd41: 2.0, Cpd650: 1.0})
reaction.gene_reaction_rule = '( SLC5A11 )'
model.add_reactions([reaction])
reaction = Reaction('PIt8')
reaction.name = 'phosphate transport in/out via Na+ symporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.5, Cpd494: -1.0, Cpd108: 1.0, Cpd41: 1.5})
reaction.gene_reaction_rule = '( SLC20A1 or SLC20A2 )'
model.add_reactions([reaction])
reaction = Reaction('PIt9')
reaction.name = 'phosphate transport in/out via two Na+ symporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd494: -1.0, Cpd108: 1.0, Cpd41: 2.0})
reaction.gene_reaction_rule = '( SLC34A3 )'
model.add_reactions([reaction])
reaction = Reaction('PROt4_2_r')
reaction.name = 'Proline transport (sodium symport) (2:1)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2013: -1.0, Cpd41: 2.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC6A7 )'
model.add_reactions([reaction])
reaction = Reaction('HSD17B3r')
reaction.name = 'testicular 17-beta-hydroxysteroid dehydrogenase'
reaction.subsystem = 'Steroid metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd1514: -1.0, Cpd589: 1.0, Cpd1515: 1.0})
reaction.gene_reaction_rule = '( HSD17B3 )'
model.add_reactions([reaction])
reaction = Reaction('CYOOm3')
reaction.name = 'cytochrome c oxidase, mitochondrial Complex IV'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -7.92, Cpd9: -1.0, Cpd750: -4.0, Cpd12: 1.96, Cpd33: 4.0, Cpd751: 4.0, Cpd2144: 0.02})
reaction.gene_reaction_rule = '( COX4I1 or COX4I2 or COX5A or COX5B or COX6A1 or COX6A2 or COX6B1 or COX6B2 or COX6C or COX7A1 or COX7A2 or COX7A2L or COX7B2 or COX8A or COX8C or MT-CO1 or MT-CO2 or MT-CO3 )'
model.add_reactions([reaction])
reaction = Reaction('2MB2COAc')
reaction.name = 'transport of 2-methylcrotonoyl-CoA into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd239: -1.0, Cpd3682: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('3HBCOARc')
reaction.name = 'transport of (R)-3-hydroxybutanoyl-CoA into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd71: -1.0, Cpd3683: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('ADPCOACROT')
reaction.name = 'production of adipoyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd3684: -1.0, Cpd184: 1.0, Cpd3685: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('ADPCOAPTE')
reaction.name = 'thioesterification of adipoyl coA for release into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3684: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3686: 1.0})
reaction.gene_reaction_rule = '( ACOT8 )'
model.add_reactions([reaction])
reaction = Reaction('C100CPT1')
reaction.name = 'production of decanoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1092: -1.0, Cpd78: 1.0, Cpd3687: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C101CPT1')
reaction.name = 'production of decenoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3688: -1.0, Cpd78: 1.0, Cpd3689: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C102CPT1')
reaction.name = 'production of decadienoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3690: -1.0, Cpd78: 1.0, Cpd3691: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C10DCCACT')
reaction.name = 'transport of sebacoyl carnitine into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3692: -1.0, Cpd3693: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C10DCc')
reaction.name = 'production of sebacoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3694: -1.0, Cpd78: 1.0, Cpd3693: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C10OHc')
reaction.name = 'production of 3-OHdecanoylcarnitinec'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3695: -1.0, Cpd78: 1.0, Cpd3696: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C120CPT1')
reaction.name = 'production of dodecanoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1093: -1.0, Cpd78: 1.0, Cpd3697: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C121CPT1')
reaction.name = 'production of dodecenoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3698: -1.0, Cpd78: 1.0, Cpd3699: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C12DCACOT')
reaction.name = 'thioesterification of dodecanedioyl coa for release into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3700: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3701: 1.0})
reaction.gene_reaction_rule = '( ACOT8 )'
model.add_reactions([reaction])
reaction = Reaction('C12OHc')
reaction.name = 'production of 3-OHdodecanoylcarnitinec'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3702: -1.0, Cpd78: 1.0, Cpd3703: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C140CPT1')
reaction.name = 'production of myristoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1019: -1.0, Cpd78: 1.0, Cpd3704: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C141ACBP')
reaction.name = 'transport of 3-hydroxytetradecenoyl coa from mitochondria into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3705: -1.0, Cpd3706: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('C141CPT1')
reaction.name = 'production of tetradecenoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3707: -1.0, Cpd78: 1.0, Cpd3708: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C141OHc')
reaction.name = 'production of 3-hydroxytetradecenoyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3706: -1.0, Cpd78: 1.0, Cpd3709: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C142ACBP')
reaction.name = 'transport of 3-hydroxytetradeca dienoyl coa from mitochondria into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3710: -1.0, Cpd3711: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('C142CPT1')
reaction.name = 'production of tetradecadienoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3712: -1.0, Cpd78: 1.0, Cpd3713: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C142OHc')
reaction.name = 'production of 3-hydroxytetradeca dienoyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3711: -1.0, Cpd78: 1.0, Cpd3714: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C14OHc')
reaction.name = 'production of 3-hydroxytetradecanoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3715: -1.0, Cpd78: 1.0, Cpd3716: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C161OHc')
reaction.name = 'production of 3-hydroxyhexadecenoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3717: -1.0, Cpd78: 1.0, Cpd3718: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C162ACBP')
reaction.name = 'transport of 3-hydroxy trans7,10-hexadecadienoylcoa from mitochondria into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3719: -1.0, Cpd3720: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('C162OHc')
reaction.name = 'production of 3-hydroxy hexadecadienoyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3720: -1.0, Cpd78: 1.0, Cpd3721: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C16DCc')
reaction.name = 'production of hexadecanedioylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3722: -1.0, Cpd78: 1.0, Cpd3723: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C16OHc')
reaction.name = 'production of 3-hydroxyhexadecanoylcarnitinec'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3724: -1.0, Cpd78: 1.0, Cpd3725: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C16txc')
reaction.name = 'transport of palmitoylcoa into peroxisomes'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd590: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd196: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('C181OHc')
reaction.name = 'production of 3-hydroxyoctadecenoylcarnitinec'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3726: -1.0, Cpd78: 1.0, Cpd3727: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C182OHc')
reaction.name = 'production of 3-hydroxyoctadecadienoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3728: -1.0, Cpd78: 1.0, Cpd3729: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C18OHc')
reaction.name = 'production of 3-hydroxyoctadecanoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3730: -1.0, Cpd78: 1.0, Cpd3731: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C2tcx')
reaction.name = 'transport of acetylcarnitine from peroxisomes to cytosol'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd736: -1.0, Cpd265: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C30CPT1')
reaction.name = 'production of propionylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd207: -1.0, Cpd303: -1.0, Cpd78: 1.0, Cpd1913: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C40CPT1')
reaction.name = 'production of butyrylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3732: -1.0, Cpd78: 1.0, Cpd3733: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C4CRNCPT2')
reaction.name = 'transport of butyryl carnitine in the mitochondrial matrix for final hydrolysis'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3734: -1.0, Cpd241: 1.0, Cpd307: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('C4DCCACT')
reaction.name = 'transport of succinyl carnitine into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3735: -1.0, Cpd3736: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C4OHc')
reaction.name = 'production of 3-hydroxybytyryl carnitinec'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3683: -1.0, Cpd78: 1.0, Cpd3737: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C4tcx')
reaction.name = 'transport of bytyrylcarnitine from peroxisomes'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3738: -1.0, Cpd3733: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C4tmc')
reaction.name = 'transport of bytyrylcarnitine into mitochondrial matrix'
reaction.subsystem = 'Transport, mitochondrial'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3733: -1.0, Cpd3734: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C4x')
reaction.name = 'esterification of butyrylcoa to butyrylcarnitine for transport'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd3739: -1.0, Cpd184: 1.0, Cpd3738: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('C50CPT1')
reaction.name = 'production of isovalerylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3740: -1.0, Cpd78: 1.0, Cpd3741: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C51CPT1')
reaction.name = 'production of tiglylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3682: -1.0, Cpd78: 1.0, Cpd3742: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C60CPT1')
reaction.name = 'production of hexanoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3743: -1.0, Cpd78: 1.0, Cpd3744: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C6COAt')
reaction.name = 'esterification of of hexanoylcoa for transport  into cytosol'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd3745: -1.0, Cpd184: 1.0, Cpd3746: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('C6CRNtcx')
reaction.name = 'transport of hexanoylcarnitine into cytosol'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3746: -1.0, Cpd3744: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C6DCCACT')
reaction.name = 'transport of adipoyl carnitine into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3685: -1.0, Cpd3747: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('C6DCc')
reaction.name = 'production of adipoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3748: -1.0, Cpd78: 1.0, Cpd3747: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C80CPT1')
reaction.name = 'production of octanoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd1053: -1.0, Cpd78: 1.0, Cpd3749: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C81CPT1')
reaction.name = 'production of ocetenoylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3750: -1.0, Cpd78: 1.0, Cpd3751: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C8DCc')
reaction.name = 'production of suberylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3752: -1.0, Cpd78: 1.0, Cpd3753: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('C9BRxtc')
reaction.name = 'transport of dimethylnonanoylcarnitine from peroxisomes to cytosol'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd741: -1.0, Cpd3754: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('DCATDc')
reaction.name = 'transport of Decanoate (n-C10:0) into the cell by diffusion'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3755: -1.0, Cpd3756: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('DCSPTN1COAtxc')
reaction.name = 'transport of docosapentenoylcoa into peroxisomes.'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd811: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1081: 1.0})
reaction.gene_reaction_rule = '( ABCD1 or ABCD2 )'
model.add_reactions([reaction])
reaction = Reaction('DOCO13ECOAtxc')
reaction.name = 'transport of 13-docosenoylcoa into peroxisomes'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd3757: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd3758: 1.0})
reaction.gene_reaction_rule = '( ABCD1 or ABCD2 )'
model.add_reactions([reaction])
reaction = Reaction('DOCO13EFATP')
reaction.name = 'absorption or uptake of 13-docosenoic acid into cells'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3759: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3757: 1.0})
reaction.gene_reaction_rule = '( SLC27A1 or SLC27A2 or SLC27A3 or SLC27A4 or SLC27A5 or SLC27A6 )'
model.add_reactions([reaction])
reaction = Reaction('DOCOSACT')
reaction.name = 'activation of docosanoic acid for transport'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3760: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3761: 1.0})
reaction.gene_reaction_rule = '( ACSBG1 or ACSBG2 )'
model.add_reactions([reaction])
reaction = Reaction('DOCOSCOAtxc')
reaction.name = 'transport of behenoylcoa into peroxisomes'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd3761: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd3762: 1.0})
reaction.gene_reaction_rule = '( ABCD1 or ABCD2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC101C102m')
reaction.name = 'fatty acid beta oxidation(C10:1-->C10:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3763: -1.0, Cpd240: 1.0, Cpd3764: 1.0})
reaction.gene_reaction_rule = '( ACADM or ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC101C102x')
reaction.name = 'fatty acid beta oxidation(C10:1-->C10:2)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3765: -1.0, Cpd247: 1.0, Cpd3766: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC101C8m')
reaction.name = 'fatty acid beta oxidation(C10:1-->C8)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd2467: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC101C8x')
reaction.name = 'fatty acid beta oxidation(C10:1-->C8:0)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3767: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd188: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC101m')
reaction.name = 'isomerization(C10:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3768: -1.0, Cpd2467: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC101x')
reaction.name = 'isomerization(C10:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3769: -1.0, Cpd3767: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102C101m')
reaction.name = 'reduction(C10:2-->C10:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3764: -1.0, Cpd13: 1.0, Cpd3768: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102C101x')
reaction.name = 'fatty acid beta oxidation(C10:2-->C10:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3766: -1.0, Cpd1517: 1.0, Cpd3769: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102C103m')
reaction.name = 'fatty acid beta oxidation(C10:2-->C10:3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3770: -1.0, Cpd240: 1.0, Cpd3771: 1.0})
reaction.gene_reaction_rule = '( ACADM )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102C103x')
reaction.name = 'fatty acid beta oxidation(C10:2-->C10:3)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3772: -1.0, Cpd247: 1.0, Cpd3773: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102C81m')
reaction.name = 'fatty acid beta oxidation(C10:2-->C8:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3774: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3775: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102C81x')
reaction.name = 'fatty acid beta oxidation(C10:2-->C8:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3776: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3777: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102m')
reaction.name = 'isomerization(C10:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3778: -1.0, Cpd3774: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC102x')
reaction.name = 'isomerization(C10:2)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3779: -1.0, Cpd3776: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC103C102m')
reaction.name = 'fatty acid beta oxidation(C10:3-->C10:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3771: -1.0, Cpd13: 1.0, Cpd3778: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC103C102x')
reaction.name = 'fatty acid beta oxidation(C10:3-->C10:2)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3773: -1.0, Cpd1517: 1.0, Cpd3779: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC10C10OHm')
reaction.name = 'fatty acid beta oxidation(C10-->OHC10)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd2464: -1.0, Cpd240: 1.0, Cpd3780: 1.0})
reaction.gene_reaction_rule = '( ACADM or ECHS1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC10C8m')
reaction.name = 'fatty acid beta oxidation(C10-->C8)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd2464: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd1063: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC10C8x')
reaction.name = 'fatty acid beta oxidation(C10-->C8)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd190: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd188: 1.0, Cpd247: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC10DCC8DCx')
reaction.name = 'fatty acid beta oxidation(C10DC-->C8DC)'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3781: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3782: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC11BRC9BRx')
reaction.name = 'fatty acid beta oxidation(C11br-->C9br)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3783: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd247: 1.0, Cpd740: 1.0, Cpd742: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX3 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC11C9m')
reaction.name = 'fatty acid beta oxidation(C11-->C9)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3784: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3785: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC121C101m')
reaction.name = 'fatty acid beta oxidation(C12:1-->C10:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3786: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3768: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC121C10x')
reaction.name = 'fatty acid beta xoidation(C12:1-->C10)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd2428: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd190: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC121x')
reaction.name = 'isomerization(C12:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3787: -1.0, Cpd2428: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC122C101m')
reaction.name = 'fatty acid beta oxidation(C12:2-->C10:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3788: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3763: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC122m')
reaction.name = 'isomerization(C12:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3789: -1.0, Cpd3788: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC122x')
reaction.name = 'isomerization(C12:2)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3790: -1.0, Cpd3791: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC123C102m')
reaction.name = 'fatty acid beta oxidation(C12:3-->C10:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3792: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3770: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC123C102x')
reaction.name = 'fatty acid beta oxidation(C12:3-->C10:2)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3793: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3772: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC123m')
reaction.name = 'isomerization(C12:3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3794: -1.0, Cpd3792: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC123x')
reaction.name = 'isomerization(C12:3)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3795: -1.0, Cpd3793: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC12C10m')
reaction.name = 'fatty acid beta oxidation(C12-->C10)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd2411: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd2464: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC12C10x')
reaction.name = 'fatty acid beta oxidation(C12-->C10)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd192: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd190: 1.0, Cpd247: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC12C12OHm')
reaction.name = 'fatty acid beta oxidation(C12-->OHC12)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd2411: -1.0, Cpd240: 1.0, Cpd2427: 1.0})
reaction.gene_reaction_rule = '( ACADM or ECHS1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC12DCC10DCx')
reaction.name = 'fatty acid beta oxidation(C12DC-->C10DC)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3700: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3781: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC12DCTc')
reaction.name = 'transport of dodecanedioyl carnitine into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3796: -1.0, Cpd3797: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC12DCc')
reaction.name = 'formation of dodecanedioyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3798: -1.0, Cpd78: 1.0, Cpd3797: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC12DCx')
reaction.name = 'production of dodecanedioyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd3700: -1.0, Cpd184: 1.0, Cpd3796: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC13BRC11BRx')
reaction.name = 'fatty acid beta oxidation(C13br-->C11br)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3799: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3783: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX3 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC13C11m')
reaction.name = 'fatty acid beta oxidation(C13-->C11)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3800: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3784: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC141C121m')
reaction.name = 'fatty acid beta oxidation(C14:1-->C12:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3801: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3786: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC141C121x')
reaction.name = 'fatty acid beta xoidation(C14:1-->C12:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3802: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3787: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC141C141OHm')
reaction.name = 'fatty acid beta oxidation(C14:1-->C14:1OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd3801: -1.0, Cpd240: 1.0, Cpd3705: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC142C122m')
reaction.name = 'fatty acid beta oxidation(C14:2-->C12:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3803: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3789: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC142C122x')
reaction.name = 'fatty acid beta oxidation(C14:2-->C12:2)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3804: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3790: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC142C142OHm')
reaction.name = 'fatty acid beta oxidation(C14:2-->C14:2OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd3805: -1.0, Cpd240: 1.0, Cpd3710: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC143C123m')
reaction.name = 'fatty acid beta oxidation(C14:3-->C12:3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3806: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3794: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC143C123x')
reaction.name = 'fatty acid beta oxidation(C14:3-->C12:3)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3807: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3795: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC14C12m')
reaction.name = 'fatty acid beta oxidation(C14-->C12)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1061: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd2411: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC14C12x')
reaction.name = 'fatty acid beta oxidation(C14-->C12)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd194: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd192: 1.0, Cpd247: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC14C14OHm')
reaction.name = 'fatty acid beta oxidation(C14-->C14OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd1061: -1.0, Cpd240: 1.0, Cpd2460: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC14DCC12DCx')
reaction.name = 'fatty acid beta oxidation(C14DC-->C12DC)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3808: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3700: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC15ATPx')
reaction.name = 'fatty acid activation(C15)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd2025: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd1925: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC15BRC13BRx')
reaction.name = 'fatty acid beta oxidation(C15br-->C13br)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1925: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd247: 1.0, Cpd742: 1.0, Cpd3799: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX3 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC15C13m')
reaction.name = 'fatty acid beta oxidation(C15-->C13)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1062: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3800: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC15NADPx')
reaction.name = 'fatty acid alpha oxidation(2-Hydroxyphytanoyl-CoA)(nadp)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd1946: -1.0, Cpd1517: -1.0, Cpd358: 2.0, Cpd3212: 1.0, Cpd1516: 1.0, Cpd2025: 1.0})
reaction.gene_reaction_rule = '( ALDH3A2 or HACL1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC15NADx')
reaction.name = 'fatty acid alpha oxidation(2-Hydroxyphytanoyl-CoA)(nad)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd261: -1.0, Cpd1946: -1.0, Cpd358: 2.0, Cpd1065: 1.0, Cpd3212: 1.0, Cpd2025: 1.0})
reaction.gene_reaction_rule = '( ALDH3A2 or HACL1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC161C141m')
reaction.name = 'fatty acid beta oxidation(C16:1-->C14:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd599: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3801: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC161C141x')
reaction.name = 'fatty acid beta xoidation(C16:1-->C14:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3809: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3802: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC161C161OHm')
reaction.name = 'production of 3-hydroxyhexadecenoylcoA'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd599: -1.0, Cpd240: 1.0, Cpd3810: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC162C142m')
reaction.name = 'fatty acid beta oxidation(C16:2-->C14:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3811: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3803: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC162C162OHm')
reaction.name = 'fatty acid beta oxidation(C16:2-->C16:2OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd3812: -1.0, Cpd240: 1.0, Cpd3719: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC163C142x')
reaction.name = 'fatty acid beta oxidation(C16:3-->C14:2)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3813: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3804: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC163C143m')
reaction.name = 'fatty acid beta oxidation(C16:3-->C14:3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3814: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3806: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC163C164Gm')
reaction.name = 'fatty acid beta oxidation(C16:3-->C16:4)gm'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3815: -1.0, Cpd240: 1.0, Cpd3816: 1.0})
reaction.gene_reaction_rule = '( ACADVL )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC163C164x')
reaction.name = 'fatty acid beta oxidation(C16:3-->C16:4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3817: -1.0, Cpd247: 1.0, Cpd3818: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC163GC142m')
reaction.name = 'fatty acid beta oxidation(C16:3-->C14:2)gm'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3819: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3803: 1.0})
reaction.gene_reaction_rule = '( HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC163Gm')
reaction.name = 'isomerization(C16:3)gm'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3820: -1.0, Cpd3819: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC163x')
reaction.name = 'isomerization(C16:3)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3821: -1.0, Cpd3813: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164C143m')
reaction.name = 'fatty acid beta oxidation(C16:4-->C14:3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3822: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3806: 1.0})
reaction.gene_reaction_rule = '( HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164C143x')
reaction.name = 'fatty acid beta oxidation(C16:4-->C14:3)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3823: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3807: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164C163x')
reaction.name = 'fatty acid beta oxidation(C16:4-->C16:3)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3818: -1.0, Cpd1517: 1.0, Cpd3821: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164C165m')
reaction.name = 'fatty acid beta oxidation(C16:4-->C16:5)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3824: -1.0, Cpd240: 1.0, Cpd3825: 1.0})
reaction.gene_reaction_rule = '( ACADVL )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164C165x')
reaction.name = 'fatty acid beta oxidation(C16:4-->C16:5)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3826: -1.0, Cpd247: 1.0, Cpd3827: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164GC163m')
reaction.name = 'fatty acid beta oxidation(C16:4-->C16:3)gm'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3816: -1.0, Cpd13: 1.0, Cpd3820: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164m')
reaction.name = 'isomerization(C16:4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3828: -1.0, Cpd3822: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC164x')
reaction.name = 'isomerization(C16:4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3829: -1.0, Cpd3823: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC165C164m')
reaction.name = 'fatty acid beta oxidation (C16:5-->C16:4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3825: -1.0, Cpd13: 1.0, Cpd3828: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC165C164x')
reaction.name = 'fatty acid beta oxidation(C16:5-->C16:4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3827: -1.0, Cpd1517: 1.0, Cpd3829: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16BRx')
reaction.name = 'fatty acid activation(C16br)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd184: -1.0, Cpd940: -1.0, Cpd2026: -1.0, Cpd2283: 1.0, Cpd885: 1.0, Cpd1945: 1.0})
reaction.gene_reaction_rule = '( SLC27A2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16C14m')
reaction.name = 'fatty acid beta oxidation(C16-->C14)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd593: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd1061: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16C14x')
reaction.name = 'fatty acid beta oxidation(C16-->C14)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd196: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd194: 1.0, Cpd247: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16C16OHm')
reaction.name = 'fatty acid beta oxidation(C16-->C16OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd593: -1.0, Cpd240: 1.0, Cpd3830: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16DCC14DCx')
reaction.name = 'fatty acid beta oxidation(C16DC-->C14DC)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3831: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3808: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16DCr')
reaction.name = 'fatty acid activation(C16DC)er'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1957: -1.0, Cpd745: -1.0, Cpd3832: -1.0, Cpd2285: 1.0, Cpd1988: 1.0, Cpd3833: 1.0})
reaction.gene_reaction_rule = '( ACSL5 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC16OHC16r')
reaction.name = 'fatty acid omega oxidation(C16-->w-OHC16)er'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd587: -1.0, Cpd1431: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3834: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC17C15m')
reaction.name = 'fatty acid beta oxidation(C17-->C15)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1066: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd1062: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC181C161m')
reaction.name = 'fatty acid beta oxidation(C18:1-->C16:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3835: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd599: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC181C161x')
reaction.name = 'fatty acid beta xoidation(C18:1-->C16:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3836: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3809: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC181C181OHm')
reaction.name = 'fatty acid beta oxidation(C18:1-->C18:1OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd3837: -1.0, Cpd240: 1.0, Cpd3838: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC182C162m')
reaction.name = 'fatty acid beta oxidation(C18:2-->C16:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1069: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3811: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC182C182OHm')
reaction.name = 'fatty acid beta oxidation(C18:2-->C18:2OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd1069: -1.0, Cpd240: 1.0, Cpd3839: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC183C163Gm')
reaction.name = 'fatty acid beta oxidation(C18:3-->C16:3)gm'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1072: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3815: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC183C163m')
reaction.name = 'fatty acid beta oxidation(C18:3-->C16:3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1071: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3814: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC184C163m')
reaction.name = 'fatty acid beta oxidation (C18:4-->C16:3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3840: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3815: 1.0})
reaction.gene_reaction_rule = '( HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC184C163x')
reaction.name = 'fatty acid beta oxidation(C18:4-->C16:3)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3841: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3817: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC184C164m')
reaction.name = 'fatty acid beta oxidation(C18:4-->C16:4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1074: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3824: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC184C164x')
reaction.name = 'fatty acid beta oxidation(C18:4-->C16:4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1075: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3826: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC184m')
reaction.name = 'isomerization of (C18:4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3842: -1.0, Cpd3840: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC184x')
reaction.name = 'isomerization(C18:4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3843: -1.0, Cpd3841: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC185C164m')
reaction.name = 'fatty acid beta oxidation(C18:5-->C16:4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3844: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3824: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC185m')
reaction.name = 'isomerization(C18:5)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3845: -1.0, Cpd3844: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC18C18OHm')
reaction.name = 'fatty acid beta oxidation(C18-->C18OH)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd238: -1.0, Cpd605: -1.0, Cpd240: 1.0, Cpd3846: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC201C181x')
reaction.name = 'fatty acid beta xoidation(C20:1-->C18:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3847: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3836: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC204C184m')
reaction.name = 'fatty acid beta oxidation(C20:4-->C18:4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3848: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3842: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC204C205x')
reaction.name = 'fatty acid beta oxidation(C20:4-->C20:5)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd3849: -1.0, Cpd247: 1.0, Cpd3850: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC205C184x')
reaction.name = 'fatty acid beta oxidation(C20:5-->C18:4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3850: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3843: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC205C185m')
reaction.name = 'fatty acid beta oxidation(C20:5-->C18:5)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3851: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3845: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC221C201x')
reaction.name = 'fatty acid beta oxidation(C22:1-->C20:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3758: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3847: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC225C204m')
reaction.name = 'fatty acid beta oxidation(C22:5-->C20:4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3852: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3848: 1.0})
reaction.gene_reaction_rule = '( HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC225C204x')
reaction.name = 'fatty acid beta oxidation(C22:5-->C20:4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3853: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3849: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC225C226m')
reaction.name = 'fatty acid beta oxidation(C22:5-->C22:6)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd814: -1.0, Cpd240: 1.0, Cpd3854: 1.0})
reaction.gene_reaction_rule = '( ACADVL )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC225C226x')
reaction.name = 'fatty acid beta oxidation(C22:5-->C22:6)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd246: -1.0, Cpd1081: -1.0, Cpd247: 1.0, Cpd3855: 1.0})
reaction.gene_reaction_rule = '( ACOX1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC225m')
reaction.name = 'isomerization(C22:5)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3856: -1.0, Cpd3852: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC225x')
reaction.name = 'isomerization(C22:5)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3857: -1.0, Cpd3853: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226C205m')
reaction.name = 'fatty acid beta oxidation(C22:6-->C20:5)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3858: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3851: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226C225m')
reaction.name = 'fatty acid beta oxidation(C22:6-->C22:5)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3854: -1.0, Cpd13: 1.0, Cpd3856: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226C225x')
reaction.name = 'fatty acid beta oxidation(C22:6-->C22:5)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd358: -1.0, Cpd1516: -1.0, Cpd3855: -1.0, Cpd1517: 1.0, Cpd3857: 1.0})
reaction.gene_reaction_rule = '( DECR2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226C227m')
reaction.name = 'fatty acid beta oxidation(C22:6-->C22:7)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd617: -1.0, Cpd240: 1.0, Cpd3859: 1.0})
reaction.gene_reaction_rule = '( ACADVL )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC226m')
reaction.name = 'isomerization(C22:6)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3860: -1.0, Cpd3858: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC227C226m')
reaction.name = 'fatty acid beta oxidation(C22:7-->C22:6)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3859: -1.0, Cpd13: 1.0, Cpd3860: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC22C20x')
reaction.name = 'fatty acid beta oxidation(C22-->C20)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3762: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd1076: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC22C22DCHYr')
reaction.name = 'fatty acid omega oxidation by hydroxylases(C22-->C22DC)er'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -3.0, Cpd565: -2.0, Cpd587: -3.0, Cpd3861: -1.0, Cpd437: 4.0, Cpd589: 3.0, Cpd3862: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC22OHC22r')
reaction.name = 'fatty acid omega oxidation(C22-->w-OHC22)er'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd587: -1.0, Cpd3861: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3863: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC241C221x')
reaction.name = 'fatty acid beta oxidation(C24:1-->C22:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1084: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3758: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC24C22x')
reaction.name = 'Fatty acid beta oxidation(C24-->C22)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd1083: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3762: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC3DC')
reaction.name = 'malonylcoa-->malonylcarnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd199: -1.0, Cpd303: -1.0, Cpd78: 1.0, Cpd3864: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC4C2m')
reaction.name = 'fatty acid beta oxidation(C4-->C2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd241: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 2.0, Cpd240: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADS or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC4C4DCc')
reaction.name = 'succ-->C4DCc'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3865: -1.0, Cpd78: 1.0, Cpd3736: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC5C3x')
reaction.name = 'fatty acid beta oxidation(C5-->C3)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3866: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd179: 1.0, Cpd240: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADS or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC5C5DCc')
reaction.name = 'production of glutaryl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd303: -1.0, Cpd3867: -1.0, Cpd78: 1.0, Cpd3868: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC5OHc')
reaction.name = '(3-hydroxyisovalerylcoa-->3-hydroxyisovalerylcarnitine)c'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3869: -1.0, Cpd78: 1.0, Cpd3870: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC61C4m')
reaction.name = 'fatty acid beta oxidation(C6:1-->C4:0)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3871: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd241: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC61C4x')
reaction.name = 'fatty acid beta oxidation(C6:1-->C4:0)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3872: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3739: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC61m')
reaction.name = 'isomerization(C6:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3873: -1.0, Cpd3871: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC61x')
reaction.name = 'isomerization(C6:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3874: -1.0, Cpd3872: 1.0})
reaction.gene_reaction_rule = '( ECI3 or EHHADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC6C4m')
reaction.name = 'fatty acid beta oxidation(C6-->C4)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3875: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd241: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC6C4x')
reaction.name = 'fatty acid beta oxidation(C6-->C4)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3745: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3739: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC6DCC4DCx')
reaction.name = 'fatty acid beta oxidation(C6DC-->C4DC)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3684: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3876: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC7C5m')
reaction.name = 'fatty acid beta oxidation(C7-->C5)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3877: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3866: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC81C61m')
reaction.name = 'fatty acid beta oxidation(C8:1-->C6:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3775: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3873: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC81C61x')
reaction.name = 'fatty acid beta oxidation(C8:1-->C6:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3777: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3874: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC8C6m')
reaction.name = 'fatty acid beta oxidation(C8-->C6)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1063: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3875: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC8C6x')
reaction.name = 'fatty acid beta oxidation(C8-->C6)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd188: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3745: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC8DCC6DCx')
reaction.name = 'fatty acid beta oxidation(C8DC-->C6DC)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd246: -1.0, Cpd261: -1.0, Cpd3782: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd247: 1.0, Cpd3684: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or ACOX1 or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXC9C7m')
reaction.name = 'fatty acid beta oxidation(C9-->C7)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3785: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3877: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ACADM or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXMC10OHMC10r')
reaction.name = 'fatty acid omega oxidation(MC10-->w-OHMC10)er'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd435: -1.0, Cpd565: -1.0, Cpd587: -1.0, Cpd3878: -1.0, Cpd437: 1.0, Cpd589: 1.0, Cpd3879: 1.0})
reaction.gene_reaction_rule = '( CYP4F15 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXOHC16C16DCc')
reaction.name = 'fatty acid omega oxidation(w-OHC16-->C16DC)c'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -2.0, Cpd3880: -1.0, Cpd33: 2.0, Cpd35: 2.0, Cpd3881: 1.0})
reaction.gene_reaction_rule = '( ADH5 or ALDH3A2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXOHC22C22DCc')
reaction.name = 'fatty acid omega oxidation(w-OHC22-->C22DC)c'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -2.0, Cpd3882: -1.0, Cpd33: 2.0, Cpd35: 2.0, Cpd3883: 1.0})
reaction.gene_reaction_rule = '( ADH5 or ALDH3A2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXOHMC10DC10c')
reaction.name = 'fatty acid omega oxidation(w-OHMC10-->DC10)c'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd32: -2.0, Cpd3884: -1.0, Cpd33: 3.0, Cpd35: 2.0, Cpd3885: 1.0})
reaction.gene_reaction_rule = '( ADH5 or ALDH3A2 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXTC101TC102m')
reaction.name = 'fatty acid beta oxidation trans(C10:1-->C10:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd238: -1.0, Cpd3886: -1.0, Cpd240: 1.0, Cpd3887: 1.0})
reaction.gene_reaction_rule = '( ACADM )'
model.add_reactions([reaction])
reaction = Reaction('FAOXTC102C101m')
reaction.name = 'fatty acid beta oxidation trans(C10:2-->C10:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd7: -1.0, Cpd8: -1.0, Cpd3887: -1.0, Cpd13: 1.0, Cpd3768: 1.0})
reaction.gene_reaction_rule = '( DECR1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXTC122TC101m')
reaction.name = 'fatty acid beta oxidation trans(C12:2-->C10:1)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd3888: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd3886: 1.0})
reaction.gene_reaction_rule = '( ACAA2 or ECHS1 or HADH )'
model.add_reactions([reaction])
reaction = Reaction('FAOXTC122m')
reaction.name = 'isomerization trans(C12:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3889: -1.0, Cpd3888: 1.0})
reaction.gene_reaction_rule = '( ECI1 )'
model.add_reactions([reaction])
reaction = Reaction('FAOXTC142TC122m')
reaction.name = 'fatty acid beta oxidation trans(C14:2-->C12:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3805: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3889: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXTC162TC142m')
reaction.name = 'fatty acid beta oxidation trans(C16:2-->C14:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd3812: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3805: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FAOXTC182TC162m')
reaction.name = 'fatty acid beta oxidation trans(C18:2-->C16:2)m'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd12: -1.0, Cpd43: -1.0, Cpd44: -1.0, Cpd238: -1.0, Cpd1070: -1.0, Cpd7: 1.0, Cpd47: 1.0, Cpd178: 1.0, Cpd240: 1.0, Cpd3812: 1.0})
reaction.gene_reaction_rule = '( ACADVL or HADHA or HADHB )'
model.add_reactions([reaction])
reaction = Reaction('FOAXC122C101x')
reaction.name = 'fatty acid beta oxidation(C12:2-->C10:1)x'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1064: -1.0, Cpd184: -1.0, Cpd261: -1.0, Cpd3791: -1.0, Cpd358: 1.0, Cpd1065: 1.0, Cpd182: 1.0, Cpd3765: 1.0})
reaction.gene_reaction_rule = '( ACAA1B or EHHADH or HSD17B4 )'
model.add_reactions([reaction])
reaction = Reaction('GLUTCOAACBP')
reaction.name = 'transport of glutaryl-CoA(5-) from mitochondria into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd46: -1.0, Cpd3867: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HDCACBP')
reaction.name = 'transport of (S)-3-Hydroxydecanoyl-CoA from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3780: -1.0, Cpd3695: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HDDACBP')
reaction.name = 'transport of (S)-3-Hydroxydodecanoyl-CoA from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2427: -1.0, Cpd3702: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HDECAACBP')
reaction.name = 'transport of 3-hydroxyhexadecanoylcoa from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3830: -1.0, Cpd3724: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HDECEACBP')
reaction.name = 'transport of 3-hydroxyhexadecenoylcoa from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3810: -1.0, Cpd3717: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HEXCOAACBP')
reaction.name = 'transport of Hexanoyl-CoA (n-C6:0CoA) from mitochondria into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3875: -1.0, Cpd3743: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HEXDICOAACBP')
reaction.name = 'transport of hexadecanedioyl coa'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3833: -1.0, Cpd3722: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HEXDICOAACBPx')
reaction.name = 'transport of hexadecanedioyl coa'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3833: -1.0, Cpd3831: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HIVCACBP')
reaction.name = 'transport of 3-hydroxyisovalerylcoa from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3890: -1.0, Cpd3869: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HOCDACBP')
reaction.name = 'transport of (S)-3-Hydroxyoctadecanoyl-CoA from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3846: -1.0, Cpd3730: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HOCTDACBP')
reaction.name = 'transport of 3-hydroxyoctadecenoylcoa from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3838: -1.0, Cpd3726: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('HTDCACBP')
reaction.name = 'transport of (S)-3-Hydroxytetradecanoyl-CoA from mitochondria into the cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2460: -1.0, Cpd3715: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('IVCOAACBP')
reaction.name = 'transport of Isovaleryl-CoA from mitochondria into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd242: -1.0, Cpd3740: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('LGNCCOAtcx')
reaction.name = 'transport of lignocericyl coenzyme A from cytosol to peroxisome.'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1041: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1083: 1.0})
reaction.gene_reaction_rule = '( ABCD1 or ABCD2 )'
model.add_reactions([reaction])
reaction = Reaction('NRVNCCOAtxc')
reaction.name = 'transport of nervonylcoa into peroxisomes'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1043: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1084: 1.0})
reaction.gene_reaction_rule = '( ABCD1 or ABCD2 )'
model.add_reactions([reaction])
reaction = Reaction('OCD11COACPT1')
reaction.name = 'transport of 11-octadecenoyl carnitine into the mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3891: -1.0, Cpd78: 1.0, Cpd3892: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('OCD11CRNCACT')
reaction.name = 'transport of 11-octadecenoyl carnitine into the mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3892: -1.0, Cpd3893: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('OCD11CRNCPT2')
reaction.name = 'transport of 11-octadecenoyl carnitine into the mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3893: -1.0, Cpd307: 1.0, Cpd3835: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('OCT11EFATP')
reaction.name = 'uptake of vaccenic acid by cells'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd3894: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd3891: 1.0})
reaction.gene_reaction_rule = '( SLC27A1 or SLC27A2 or SLC27A3 or SLC27A4 or SLC27A5 or SLC27A6 )'
model.add_reactions([reaction])
reaction = Reaction('OCTDEC2ACBP')
reaction.name = 'transport of 3-hydroxyoctadecadienoyl coa from mitochondria into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3839: -1.0, Cpd3728: 1.0})
reaction.gene_reaction_rule = '( DBI )'
model.add_reactions([reaction])
reaction = Reaction('OCTDECCACT')
reaction.name = 'transport of Octadecenoyl-CoA into mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3895: -1.0, Cpd3896: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('OCTDECCPT1')
reaction.name = 'transport of Octadecenoyl-CoA into mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd303: -1.0, Cpd3897: -1.0, Cpd78: 1.0, Cpd3895: 1.0})
reaction.gene_reaction_rule = '( CPT1A or CPT1B or CPT1C )'
model.add_reactions([reaction])
reaction = Reaction('OCTDECCPT2')
reaction.name = 'transport of Octadecenoyl-CoA into mitochondrial matrix'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd43: -1.0, Cpd3896: -1.0, Cpd307: 1.0, Cpd3837: 1.0})
reaction.gene_reaction_rule = '( CPT2 )'
model.add_reactions([reaction])
reaction = Reaction('PRISTCOAtcx')
reaction.name = 'transport of pristanoylcoa from cytosol to peroxisomes.'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1030: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1925: 1.0})
reaction.gene_reaction_rule = '( ABCD3 )'
model.add_reactions([reaction])
reaction = Reaction('SBCOAACOTx')
reaction.name = 'thioesterification of suberyl coa for release into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3782: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3898: 1.0})
reaction.gene_reaction_rule = '( ACOT8 )'
model.add_reactions([reaction])
reaction = Reaction('SCP21cx')
reaction.name = 'transport of phytanoylcoa from cytosol to peroxisomes.'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd1035: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1945: 1.0})
reaction.gene_reaction_rule = '( ABCD3 )'
model.add_reactions([reaction])
reaction = Reaction('SEBCOACROT')
reaction.name = 'production of sebacoyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd3781: -1.0, Cpd184: 1.0, Cpd3692: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('SEBCOAPET')
reaction.name = 'thioesterification of sebacoylcoa for release into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3781: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd3899: 1.0})
reaction.gene_reaction_rule = '( ACOT8 )'
model.add_reactions([reaction])
reaction = Reaction('STCOATxc')
reaction.name = 'transport of Stearoyl-CoA from cytosol to peroxisomes'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd602: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1067: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('STRDNCCOAtxc')
reaction.name = 'transport of stearidonylcoa into peroxisomes.'
reaction.subsystem = 'Transport, peroxisomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd824: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd1075: 1.0})
reaction.gene_reaction_rule = '( ABCD1 )'
model.add_reactions([reaction])
reaction = Reaction('SUBERCACT')
reaction.name = 'transport of suberyl carnitine into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3900: -1.0, Cpd3753: 1.0})
reaction.gene_reaction_rule = '( SLC25A20 )'
model.add_reactions([reaction])
reaction = Reaction('SUBERCROT')
reaction.name = 'production of suberyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd3782: -1.0, Cpd184: 1.0, Cpd3900: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('SUCCCROT')
reaction.name = 'production of succinyl carnitine'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd737: -1.0, Cpd3876: -1.0, Cpd184: 1.0, Cpd3735: 1.0})
reaction.gene_reaction_rule = '( CROT )'
model.add_reactions([reaction])
reaction = Reaction('SUCCOAPET')
reaction.name = 'thioesterification of succinyl coa for release into cytosol'
reaction.subsystem = 'Fatty acid oxidation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd261: -1.0, Cpd3876: -1.0, Cpd358: 1.0, Cpd184: 1.0, Cpd1947: 1.0})
reaction.gene_reaction_rule = '( ACOT4 or ACOT8 )'
model.add_reactions([reaction])
reaction = Reaction('34DHPHELAT1tc')
reaction.name = 'transport of 3,4-Dihydroxy-L-phenylalanine by LAT1 in association with 4F2hc, across the apical surface of the memebranes.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd55: -1.0, Cpd1590: -1.0, Cpd56: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('4OHPROIMINOtc')
reaction.name = 'transport of L-OH-Proline by the apical IMINO amino acid transporters in kidney and intestine'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd3901: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd2402: 1.0})
reaction.gene_reaction_rule = '( SLC6A20B or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('ALAALACNc')
reaction.name = 'ALAALACNc'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3902: -1.0, Cpd394: 2.0})
reaction.gene_reaction_rule = '( CNDP1 or CNDP2 )'
model.add_reactions([reaction])
reaction = Reaction('ALAALAPEPT1tc')
reaction.name = 'transport of Alanylalanine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3903: -1.0, Cpd33: 1.0, Cpd3902: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('ALAATB0tc')
reaction.name = 'transport of L-Alanine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd392: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd394: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('ALAyLATthc')
reaction.name = 'transport of Alanine by y+LAT1 or y+LAT2 with co-transporter of h in small intestine and kidney'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd392: -1.0, Cpd467: -1.0, Cpd33: 1.0, Cpd394: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('ARGATB0tc')
reaction.name = 'transport of L-Arginine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd466: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd467: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('ASCBSVCTtc')
reaction.name = 'transport of L-Ascorbate by SVCT1 or SVCT2 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -2.0, Cpd479: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 2.0, Cpd480: 1.0})
reaction.gene_reaction_rule = '( SLC23A1 or SLC23A2 )'
model.add_reactions([reaction])
reaction = Reaction('ASNATB0tc')
reaction.name = 'transport of L-Asparagine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd395: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd393: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('ASNPHELAT2tc')
reaction.name = 'transport of L-Asparagine into the cell and efflux of L-Phenylalanine out of the cell by LAT2 on the basolateral surfaces of kidney and intestine.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd395: -1.0, Cpd1940: -1.0, Cpd393: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('BALABETAtc')
reaction.name = 'transport of beta alanine into the intestinal cells by beta transport system'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd540: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd483: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A6 )'
model.add_reactions([reaction])
reaction = Reaction('BALAPAT1tc')
reaction.name = 'transport of beta alanine by PAT1 in renal and intestinal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd540: -1.0, Cpd33: 1.0, Cpd483: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('CARPEPT1tc')
reaction.name = 'transport of L-Carnosine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3904: -1.0, Cpd33: 1.0, Cpd1764: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('CARhPTtc')
reaction.name = 'Transport of L-Carnosine by hPT3 or hPT4 peptide transporter.'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3905: -1.0, Cpd1764: 1.0})
reaction.gene_reaction_rule = '( SLC15A3 or SLC15A4 )'
model.add_reactions([reaction])
reaction = Reaction('CBLtle')
reaction.name = 'transport of Adenosylcobalamin into the intestine'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3906: -1.0, Cpd3907: 1.0})
reaction.gene_reaction_rule = '( AMN or CUBN or GIF )'
model.add_reactions([reaction])
reaction = Reaction('CYSATB0tc')
reaction.name = 'transport of L-Cysteine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd397: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd396: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('CYSPHELAT2tc')
reaction.name = 'transport of L-Cysteine into the cell and efflux of L-Phenylalanine out of the cell by LAT2 on the basolateral surfaces of kidney and intestine.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd397: -1.0, Cpd1940: -1.0, Cpd396: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('CYSSNAT5tc')
reaction.name = 'transport of L-Cysteine into the cell coupled with co-transport with Sodium and counter transport with proton by SNAT5 transporter.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -1.0, Cpd397: -1.0, Cpd37: 1.0, Cpd41: 1.0, Cpd396: 1.0})
reaction.gene_reaction_rule = '( SLC38A5 )'
model.add_reactions([reaction])
reaction = Reaction('CYSTALArBATtc')
reaction.name = 'transport of L-Cystine into the cell in exchange for L-Alanine by b0,+AT transporter at the apical surfaces of the membranes of small intestine and renal cells.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd757: -1.0, Cpd392: 1.0, Cpd758: 1.0})
reaction.gene_reaction_rule = '( SLC3A1 or SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('CYSTLEUrBATtc')
reaction.name = 'transport of L-Cystine into the cell in exchange for L-Leucine by b0,+AT transporter at the apical surfaces of the membranes of small intestine and renal cells.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd757: -1.0, Cpd1590: -1.0, Cpd758: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC3A1 or SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('FOLTle')
reaction.name = 'folate uptake into the enterocytes '
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd1119: -1.0, Cpd33: 1.0, Cpd1118: 1.0})
reaction.gene_reaction_rule = '( SLC46A1 )'
model.add_reactions([reaction])
reaction = Reaction('GALSGLT1le')
reaction.name = 'transport of D-Galactose from extracellular space to cytosol of mucosal cells in small intestine'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -2.0, Cpd1266: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 2.0, Cpd1252: 1.0})
reaction.gene_reaction_rule = '( SLC5A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLCSGLT1le')
reaction.name = 'transport of D-glucose from extracellular space to cytosol of mucosal cells in small intestine'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -2.0, Cpd446: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 2.0, Cpd1290: 1.0})
reaction.gene_reaction_rule = '( SLC5A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLNATB0tc')
reaction.name = 'Transport of L-Glutamine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd401: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd399: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('GLNyLATthc')
reaction.name = 'transport of Glutamine by y+LAT1 or y+LAT2 with co-transporter of h in small intestine and kidney'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd401: -1.0, Cpd467: -1.0, Cpd33: 1.0, Cpd399: 1.0, Cpd466: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('GLYGLYCNc')
reaction.name = 'hydrolysis of glycylycine for uptake'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3908: -1.0, Cpd402: 2.0})
reaction.gene_reaction_rule = '( CNDP1 or CNDP2 )'
model.add_reactions([reaction])
reaction = Reaction('GLYGLYPEPT1tc')
reaction.name = 'transport of Glycyl-glycine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3909: -1.0, Cpd3908: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYLEUHYDROc')
reaction.name = 'hydrolysis of Glycylleucine in the small intestine for cellular uptake'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3910: -1.0, Cpd402: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( CNDP2 or DPEP1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYLEUPEPT1tc')
reaction.name = 'transport of Glycylleucine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3911: -1.0, Cpd33: 1.0, Cpd3910: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYPHEPEPT1tc')
reaction.name = 'transport of Glycylphenylalanine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3912: -1.0, Cpd33: 1.0, Cpd3913: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYPROPEPT1tc')
reaction.name = 'transport of Glycylproline by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3914: -1.0, Cpd33: 1.0, Cpd3915: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYPROPRO1c')
reaction.name = 'hydrolysis of Prolylglycine in the small intestine for cellular uptake'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3915: -1.0, Cpd402: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( PEPD )'
model.add_reactions([reaction])
reaction = Reaction('GLYSARCNc')
reaction.name = 'Hydrolysis of glycylsarcosine for uptake'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3916: -1.0, Cpd402: 1.0, Cpd1425: 1.0})
reaction.gene_reaction_rule = '( TPP1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYSARPEPT1tc')
reaction.name = 'Transport of Glycylsarcosine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine, renal cells and brain.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3917: -1.0, Cpd33: 1.0, Cpd3916: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('GLYSNAT5tc')
reaction.name = 'transport of Glycine into the cell coupled with co-transport with Sodium and counter transport with proton by SNAT5 transporter.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -1.0, Cpd403: -1.0, Cpd37: 1.0, Cpd41: 1.0, Cpd402: 1.0})
reaction.gene_reaction_rule = '( SLC38A5 )'
model.add_reactions([reaction])
reaction = Reaction('HISCAT1')
reaction.name = 'Transport of L-Histidine by y+ transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1479: -1.0, Cpd1482: 1.0})
reaction.gene_reaction_rule = '( SLC7A1 )'
model.add_reactions([reaction])
reaction = Reaction('HISSNAT5tc')
reaction.name = 'Transport of L-Histidine into the cell coupled with co-transport with Sodium and counter transport with proton by SNAT5 transporter.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -1.0, Cpd39: -1.0, Cpd1482: -1.0, Cpd37: 1.0, Cpd41: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC38A5 )'
model.add_reactions([reaction])
reaction = Reaction('HIShPTtc')
reaction.name = 'Transport of L-Histidine by hPT3 or hPT4 peptide transporters.'
reaction.subsystem = 'Transport, lysosomal'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd176: -1.0, Cpd3918: -1.0, Cpd33: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC15A3 or SLC15A4 )'
model.add_reactions([reaction])
reaction = Reaction('HISyLATtc')
reaction.name = 'Transport of L-Histidine by y+LAT1 or y+LAT2 transporters in small intestine and kidney'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd467: -1.0, Cpd1482: -1.0, Cpd41: 1.0, Cpd466: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('HISyLATthc')
reaction.name = 'Transport of Histidine by y+LAT1 or y+LAT2 with co-transporter of h in small intestine and kidney'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd467: -1.0, Cpd1482: -1.0, Cpd33: 1.0, Cpd466: 1.0, Cpd1479: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('HPETFABP1tc')
reaction.name = 'transport of 5-HPETE into the enterocytes'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd431: -2.0, Cpd3919: 2.0})
reaction.gene_reaction_rule = '( FABP1 )'
model.add_reactions([reaction])
reaction = Reaction('ILEATB0tc')
reaction.name = 'Transport of L-Isoleucine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd1553: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('ILELAT1tc')
reaction.name = 'transport of L-Isoleucine by LAT1 in association with 4F2hc, across the apical surface of the memebranes.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1553: -1.0, Cpd1590: -1.0, Cpd1589: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('ILEPHELAT2tc')
reaction.name = 'transport of L-Isoleucine into the cell and efflux of L-Phenylalanine out of the cell by LAT2 on the basolateral surfaces of kidney and intestine.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1553: -1.0, Cpd1940: -1.0, Cpd1939: 1.0, Cpd1554: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('LEUATB0tc')
reaction.name = 'Transport of L-Leucine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd1589: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('LEUGLYPEPT1tc')
reaction.name = 'transport of Leucylglycine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3920: -1.0, Cpd33: 1.0, Cpd3921: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('LEULEULAPc')
reaction.name = 'hydrolysis of Leucylleucine in the small intestine for cellular uptake'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3922: -1.0, Cpd1590: 2.0})
reaction.gene_reaction_rule = '( LAP3 )'
model.add_reactions([reaction])
reaction = Reaction('LEULEUPEPT1tc')
reaction.name = 'transport of Leucylleucine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3923: -1.0, Cpd33: 1.0, Cpd3922: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('LEUPHELAT2tc')
reaction.name = 'Transport of L-Leucine into the cell and efflux of L-Phenylalanine out of the cell by LAT2 on the basolateral surfaces of kidney and intestine.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1589: -1.0, Cpd1940: -1.0, Cpd1939: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('LEUyLAThtc')
reaction.name = 'transport of Leucine by y+LAT1 or y+LAT2 with co-transporter of h in small intestine and kidney'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd467: -1.0, Cpd1589: -1.0, Cpd33: 1.0, Cpd466: 1.0, Cpd1590: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('LINOFATPtc')
reaction.name = 'uptake of linoleic acid by the enterocytes'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1605: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd828: 1.0})
reaction.gene_reaction_rule = '( SLC27A4 )'
model.add_reactions([reaction])
reaction = Reaction('LYSATB0tc')
reaction.name = 'transport of L-Lysine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd468: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd465: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('METATB0tc')
reaction.name = 'transport of L-Methionine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd1677: -1.0, Cpd41: 2.0, Cpd554: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('METyLATthc')
reaction.name = 'transport of Methionine by y+LAT1 or y+LAT2 with co-transporter of h in small intestine and kidney'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd467: -1.0, Cpd1677: -1.0, Cpd33: 1.0, Cpd466: 1.0, Cpd554: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('NACHORCTL3le')
reaction.name = 'NACHORCTL3le'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3924: -1.0, Cpd33: 1.0, Cpd1808: 1.0})
reaction.gene_reaction_rule = '( SLC22A13B-PS )'
model.add_reactions([reaction])
reaction = Reaction('OCDEAFABP1tc')
reaction.name = 'Transport of octadecenoyl coA into the enterocytes'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1016: -2.0, Cpd2549: 2.0})
reaction.gene_reaction_rule = '( FABP1 )'
model.add_reactions([reaction])
reaction = Reaction('OLEICFATPtc')
reaction.name = 'uptake of octadecenoate by the enterocytes'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1095: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd606: 1.0})
reaction.gene_reaction_rule = '( SLC27A4 )'
model.add_reactions([reaction])
reaction = Reaction('ORNALArBATtc')
reaction.name = 'transport of L-Ornithine into the cell in exchange for L-Alanine by b0,+AT transporter at the apical surfaces of the membranes of small intestine and renal cells.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd394: -1.0, Cpd1847: -1.0, Cpd252: 1.0, Cpd392: 1.0})
reaction.gene_reaction_rule = '( SLC3A1 or SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('ORNLEUrBATtc')
reaction.name = 'transport of L-Ornithine  into the cell in exchange for L-Leucine by b0,+AT transporter at the apical surfaces of the membranes of small intestine and renal cells.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1847: -1.0, Cpd1590: -1.0, Cpd252: 1.0, Cpd1589: 1.0})
reaction.gene_reaction_rule = '( SLC3A1 or SLC7A9 )'
model.add_reactions([reaction])
reaction = Reaction('PALFATPtc')
reaction.name = 'uptake of Hexadecanoate by the enterocytes'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1094: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd590: 1.0})
reaction.gene_reaction_rule = '( SLC27A4 )'
model.add_reactions([reaction])
reaction = Reaction('PEHSFABPe')
reaction.name = 'transport of lysophosphatidylethanolamine into the enterocytes'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd661: -2.0, Cpd1452: 2.0})
reaction.gene_reaction_rule = '( FABP1 )'
model.add_reactions([reaction])
reaction = Reaction('PHEATB0tc')
reaction.name = 'transport of L-Phenylalanine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd1939: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('PHEyLATthc')
reaction.name = 'transport of Phenylalanine by y+LAT1 or y+LAT2 with co-transporter of h in small intestine and kidney'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd467: -1.0, Cpd1939: -1.0, Cpd33: 1.0, Cpd466: 1.0, Cpd1940: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A6 or SLC7A7 )'
model.add_reactions([reaction])
reaction = Reaction('PMTCOAFABP1tc')
reaction.name = 'transport of palmitoyl coA into the enterocytes'
reaction.subsystem = 'Transport, endoplasmic reticular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd590: -2.0, Cpd2337: 2.0})
reaction.gene_reaction_rule = '( FABP1 )'
model.add_reactions([reaction])
reaction = Reaction('PROGLYPEPT1tc')
reaction.name = 'transport of Prolylglycine by the apical PEPT1 amino acid transporters across the brush border cells of the enterocytes of the intestine and renal cells'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd3925: -1.0, Cpd33: 1.0, Cpd3926: 1.0})
reaction.gene_reaction_rule = '( SLC15A1 )'
model.add_reactions([reaction])
reaction = Reaction('PROGLYPRO1c')
reaction.name = 'hydrolysis of glycylproline in the small intestine for cellular uptake'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd3926: -1.0, Cpd402: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( PEPD )'
model.add_reactions([reaction])
reaction = Reaction('PROIMINOtc')
reaction.name = 'transport of L-Proline by the apical IMINO amino acid transporters in kidney and intestine'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd2013: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( SLC6A20B or TMEM27 )'
model.add_reactions([reaction])
reaction = Reaction('RIBFLVte')
reaction.name = 'release of riboflavin into the portal blood.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd264: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd2051: 1.0})
reaction.gene_reaction_rule = '( SLC52A2 )'
model.add_reactions([reaction])
reaction = Reaction('SERATB0tc')
reaction.name = 'transport of L-Serine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd405: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd404: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('TAUBETAtc')
reaction.name = 'transport of taurine into the intestinal cells by beta transport system'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd2189: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd1586: 1.0})
reaction.gene_reaction_rule = '( SLC6A6 )'
model.add_reactions([reaction])
reaction = Reaction('TAUPAT1c')
reaction.name = 'transport of taurine into the intestinal cells by PAT1'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd2189: -1.0, Cpd33: 1.0, Cpd1586: 1.0})
reaction.gene_reaction_rule = '( SLC36A1 )'
model.add_reactions([reaction])
reaction = Reaction('THRATB0tc')
reaction.name = 'Transport of L-Threonine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd409: -1.0, Cpd698: -1.0, Cpd41: 2.0, Cpd408: 1.0, Cpd700: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('THRPHELAT2tc')
reaction.name = 'transport of L-Threonine into the cell and efflux of L-Phenylalanine out of the cell by LAT2 on the basolateral surfaces of kidney and intestine.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd409: -1.0, Cpd1940: -1.0, Cpd408: 1.0, Cpd1939: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('TRPATB0tc')
reaction.name = 'transport of L-Tryptophan into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd2224: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd1638: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('TYRATB0tc')
reaction.name = 'transport of L-Tyrosine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd2238: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('TYRPHELAT2tc')
reaction.name = 'transport of L-Tyrosine into the cell and efflux of L-Phenylalanine out of the cell by LAT2 on the basolateral surfaces of kidney and intestine.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2238: -1.0, Cpd1940: -1.0, Cpd1939: 1.0, Cpd1544: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('VALATB0tc')
reaction.name = 'transport of L-Valine into the intestinal cells by ATB0 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd698: -1.0, Cpd2274: -1.0, Cpd41: 2.0, Cpd700: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC6A14 )'
model.add_reactions([reaction])
reaction = Reaction('VALLAT1tc')
reaction.name = 'transport of L-Valine by LAT1 in association with 4F2hc, across the apical surface of the memebranes.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2274: -1.0, Cpd1590: -1.0, Cpd1589: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A5 )'
model.add_reactions([reaction])
reaction = Reaction('VALPHELAT2tc')
reaction.name = 'transport of L-Valine into the cell and efflux of L-Phenylalanine out of the cell by LAT2 on the basolateral surfaces of kidney and intestine.'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2274: -1.0, Cpd1940: -1.0, Cpd1939: 1.0, Cpd2275: 1.0})
reaction.gene_reaction_rule = '( SLC3A2 or SLC7A8 )'
model.add_reactions([reaction])
reaction = Reaction('VITEtl')
reaction.name = 'efflux of alpha-Tocopherol into the lyphatics in chylomicrons'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd497: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd496: 1.0})
reaction.gene_reaction_rule = '( ABCA1 )'
model.add_reactions([reaction])
reaction = Reaction('25HVITD3c')
reaction.name = '1-alpha hydroxylation of 25-hydroxy vitamin D'
reaction.subsystem = 'Vitamin D metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -1.0, Cpd17: -1.0, Cpd22: -1.0, Cpd2: 1.0, Cpd25: 1.0, Cpd3927: 1.0})
reaction.gene_reaction_rule = '( CYP27B1 )'
model.add_reactions([reaction])
reaction = Reaction('3HCO3_NAt')
reaction.name = '3HCO3_NAt'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd38: -3.0, Cpd39: -1.0, Cpd40: 3.0, Cpd41: 1.0})
reaction.gene_reaction_rule = '( SLC4A4 or SLC4A5 )'
model.add_reactions([reaction])
reaction = Reaction('4HPROLTASCT1')
reaction.name = 'exit of hydroxy-prloine across the basolateral surface of enterocytes'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd392: -1.0, Cpd2402: -1.0, Cpd41: 1.0, Cpd394: 1.0, Cpd3901: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 )'
model.add_reactions([reaction])
reaction = Reaction('ADNt5le')
reaction.name = 'adenosine transport (1:2 Na/Adn cotransport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd276: -1.0, Cpd41: 2.0, Cpd274: 1.0})
reaction.gene_reaction_rule = '( SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('ASPPROASCT1')
reaction.name = 'aspartate intake by system ASCT-1 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd490: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd90: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 )'
model.add_reactions([reaction])
reaction = Reaction('BIDGLCURr')
reaction.name = 'bilirubin di-glucuronide production'
reaction.subsystem = 'Heme degradation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2253: -1.0, Cpd2244: -1.0, Cpd2255: 1.0, Cpd1769: 1.0})
reaction.gene_reaction_rule = '( UGT1A8 )'
model.add_reactions([reaction])
reaction = Reaction('BTNt3ile')
reaction.name = 'Biotin transport via sodium symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -2.0, Cpd574: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 2.0, Cpd536: 1.0})
reaction.gene_reaction_rule = '( SLC5A6 )'
model.add_reactions([reaction])
reaction = Reaction('CHOLESACATc')
reaction.name = 'esterification of cholesterol to cholesterol ester within enterocytes'
reaction.subsystem = 'Cholesterol metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd843: -1.0, Cpd682: -1.0, Cpd78: 1.0, Cpd2134: 1.0})
reaction.gene_reaction_rule = '( SOAT1 or SOAT2 )'
model.add_reactions([reaction])
reaction = Reaction('CHOLESTle')
reaction.name = 'hydrolysis of cholesterol ester by cholesterol esterase '
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd1582: -1.0, Cpd37: 1.0, Cpd684: 1.0, Cpd1625: 1.0})
reaction.gene_reaction_rule = '( LIPA )'
model.add_reactions([reaction])
reaction = Reaction('CYTDt5le')
reaction.name = 'cytidine transport (1:2 Na/cytd cotransport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd766: -1.0, Cpd41: 2.0, Cpd760: 1.0})
reaction.gene_reaction_rule = '( SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('DPMVDc')
reaction.name = 'diphosphomevalonate decarboxylase, cytosol'
reaction.subsystem = 'Squalene and cholesterol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd3928: -1.0, Cpd107: 1.0, Cpd108: 1.0, Cpd60: 1.0, Cpd880: 1.0})
reaction.gene_reaction_rule = '( MVD )'
model.add_reactions([reaction])
reaction = Reaction('DSREDUCr')
reaction.name = 'Desmosterol reductase'
reaction.subsystem = 'Squalene and cholesterol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd565: -1.0, Cpd587: -1.0, Cpd852: -1.0, Cpd589: 1.0, Cpd670: 1.0})
reaction.gene_reaction_rule = '( DHCR24 )'
model.add_reactions([reaction])
reaction = Reaction('DTMPKm')
reaction.name = 'dTMP kinase in mitochondria'
reaction.subsystem = 'Pyrimidine synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd200: -1.0, Cpd1833: -1.0, Cpd202: 1.0, Cpd898: 1.0})
reaction.gene_reaction_rule = '( DTYMK )'
model.add_reactions([reaction])
reaction = Reaction('FADDPle')
reaction.name = 'FAD diphosphatase'
reaction.subsystem = 'Vitamin B2 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd3929: -1.0, Cpd37: 2.0, Cpd493: 1.0, Cpd3930: 1.0})
reaction.gene_reaction_rule = '( ENPP1 )'
model.add_reactions([reaction])
reaction = Reaction('FADH2ETC')
reaction.name = 'Complex II reaction for respiratory chain'
reaction.subsystem = 'Oxidative phosphorylation'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd240: -1.0, Cpd723: -1.0, Cpd238: 1.0, Cpd752: 1.0})
reaction.gene_reaction_rule = '( ETFA or ETFB or ETFDH )'
model.add_reactions([reaction])
reaction = Reaction('FE2DMT1')
reaction.name = 'uptake of food iron by DMT1 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd37: -1.0, Cpd1114: -1.0, Cpd33: 1.0, Cpd1497: 1.0})
reaction.gene_reaction_rule = '( SLC11A2 )'
model.add_reactions([reaction])
reaction = Reaction('FE3MTP1')
reaction.name = ' transport of ferrous iron into blood'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3931: -1.0, Cpd1113: 1.0})
reaction.gene_reaction_rule = '( SLC40A1 )'
model.add_reactions([reaction])
reaction = Reaction('FERO')
reaction.name = 'ferroxidase'
reaction.subsystem = 'Vitamin B2 metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd3: -1.0, Cpd33: -4.0, Cpd1497: -4.0, Cpd2: 2.0, Cpd3931: 4.0})
reaction.gene_reaction_rule = '( HEPH )'
model.add_reactions([reaction])
reaction = Reaction('FMNALKPle')
reaction.name = 'FMNALKPle'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd114: -1.0, Cpd3930: -1.0, Cpd494: 1.0, Cpd2051: 1.0})
reaction.gene_reaction_rule = '( ACP1 or ALPPL2 )'
model.add_reactions([reaction])
reaction = Reaction('G6PDH2c')
reaction.name = 'glucose 6-phosphate dehydrogenase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -3.0, Cpd1202: -3.0, Cpd33: 3.0, Cpd22: 3.0, Cpd1203: 3.0})
reaction.gene_reaction_rule = '( G6PD2 )'
model.add_reactions([reaction])
reaction = Reaction('GLCGLUT2')
reaction.name = 'Transport of glucose into the portal blood'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1290: -1.0, Cpd446: 1.0})
reaction.gene_reaction_rule = '( SLC2A2 )'
model.add_reactions([reaction])
reaction = Reaction('GLUPROASCT1')
reaction.name = 'glutamate intake by system ASCT-1 transporter'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -1.0, Cpd759: -1.0, Cpd2004: -1.0, Cpd41: 1.0, Cpd74: 1.0, Cpd2013: 1.0})
reaction.gene_reaction_rule = '( SLC1A4 )'
model.add_reactions([reaction])
reaction = Reaction('GNDc')
reaction.name = 'phosphogluconate dehydrogenase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd25: -3.0, Cpd1421: -3.0, Cpd22: 3.0, Cpd60: 3.0, Cpd1422: 3.0})
reaction.gene_reaction_rule = '( PGD )'
model.add_reactions([reaction])
reaction = Reaction('GSNt5le')
reaction.name = 'guanosine transport in via sodium (1:2) symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd1437: -1.0, Cpd41: 2.0, Cpd1438: 1.0})
reaction.gene_reaction_rule = '( SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('HMGCOARc')
reaction.name = 'Hydroxymethylglutaryl CoA reductase (ir) in cytosol'
reaction.subsystem = 'Squalene and cholesterol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd22: -2.0, Cpd1489: -1.0, Cpd25: 2.0, Cpd78: 1.0, Cpd2375: 1.0})
reaction.gene_reaction_rule = '( HMGCR )'
model.add_reactions([reaction])
reaction = Reaction('INSt5le')
reaction.name = 'inosine transport in via sodium (1:2) symport'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd277: -1.0, Cpd41: 2.0, Cpd275: 1.0})
reaction.gene_reaction_rule = '( SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('IPDDI')
reaction.name = 'isopentenyl-diphosphate D-isomerase'
reaction.subsystem = 'Squalene and cholesterol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd880: -1.0, Cpd879: 1.0})
reaction.gene_reaction_rule = '( GM9745 or IDI1 )'
model.add_reactions([reaction])
reaction = Reaction('MEVK1c')
reaction.name = 'mevalonate kinase (atp) cytosol'
reaction.subsystem = 'Squalene and cholesterol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd2375: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd3932: 1.0})
reaction.gene_reaction_rule = '( MVK )'
model.add_reactions([reaction])
reaction = Reaction('MTHFR3')
reaction.name = '5,10-methylenetetrahydrofolatereductase (NADPH)'
reaction.subsystem = 'Folate metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd22: -1.0, Cpd1341: -1.0, Cpd25: 1.0, Cpd127: 1.0})
reaction.gene_reaction_rule = '( MTHFR )'
model.add_reactions([reaction])
reaction = Reaction('P5CR')
reaction.name = 'pyrroline-5-carboxylate reductase'
reaction.subsystem = 'Arginine and Proline Metabolism'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd33: -2.0, Cpd22: -1.0, Cpd2005: -1.0, Cpd25: 1.0, Cpd2004: 1.0})
reaction.gene_reaction_rule = '( PYCR1 or PYCR2 or PYCRL )'
model.add_reactions([reaction])
reaction = Reaction('PGLc')
reaction.name = '6-phosphogluconolactonase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -3.0, Cpd1203: -3.0, Cpd33: 3.0, Cpd1421: 3.0})
reaction.gene_reaction_rule = '( PGLS )'
model.add_reactions([reaction])
reaction = Reaction('PHEMEe')
reaction.name = 'release of heme into the blood'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1495: -1.0, Cpd1938: 1.0})
reaction.gene_reaction_rule = '( MFSD7B or SLC46A1 )'
model.add_reactions([reaction])
reaction = Reaction('PMEVKc')
reaction.name = 'phosphomevalonate kinase, cytosol'
reaction.subsystem = 'Squalene and cholesterol synthesis'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd3932: -1.0, Cpd107: 1.0, Cpd3928: 1.0})
reaction.gene_reaction_rule = '( PMVK )'
model.add_reactions([reaction])
reaction = Reaction('PNTOt5le')
reaction.name = 'Pantothenate sodium symporter  II'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2: -1.0, Cpd105: -1.0, Cpd39: -2.0, Cpd1987: -1.0, Cpd107: 1.0, Cpd33: 1.0, Cpd108: 1.0, Cpd41: 2.0, Cpd1983: 1.0})
reaction.gene_reaction_rule = '( SLC5A6 )'
model.add_reactions([reaction])
reaction = Reaction('PRO1x')
reaction.name = 'L-ProlineNAD+ 5-oxidoreductase'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd32: -1.0, Cpd2004: -1.0, Cpd33: 2.0, Cpd35: 1.0, Cpd2005: 1.0})
reaction.gene_reaction_rule = '( PRODH2 )'
model.add_reactions([reaction])
reaction = Reaction('RPEc')
reaction.name = 'ribulose 5-phosphate 3-epimerase'
reaction.subsystem = 'Pentose phosphate pathway'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd1422: -2.0, Cpd2054: 2.0})
reaction.gene_reaction_rule = '( RPE )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL2FATPc')
reaction.name = 'uptake of Rtotal2 by enterocytes'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1628: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd351: 1.0})
reaction.gene_reaction_rule = '( SLC27A4 )'
model.add_reactions([reaction])
reaction = Reaction('RTOTAL3FATPc')
reaction.name = 'uptake of Rtotal3 by enterocytes'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1632: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd835: 1.0})
reaction.gene_reaction_rule = '( SLC27A4 )'
model.add_reactions([reaction])
reaction = Reaction('RTOTALFATPc')
reaction.name = 'uptake of Rtotal by enterocytes'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd105: -1.0, Cpd78: -1.0, Cpd1625: -1.0, Cpd150: 1.0, Cpd151: 1.0, Cpd843: 1.0})
reaction.gene_reaction_rule = '( SLC27A4 )'
model.add_reactions([reaction])
reaction = Reaction('SPRMTDe')
reaction.name = 'absorption for spermine across the basolateral side'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd2712: -1.0, Cpd2149: 1.0})
reaction.gene_reaction_rule = '( SLC22A1 )'
model.add_reactions([reaction])
reaction = Reaction('THMDt5le')
reaction.name = 'thymidine transport (1:2 Na/Thymd cotransport)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd2202: -1.0, Cpd41: 2.0, Cpd1830: 1.0})
reaction.gene_reaction_rule = '( SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('URIt5le')
reaction.name = 'uridine transport in via sodium symport (1:2)'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd39: -2.0, Cpd1821: -1.0, Cpd41: 2.0, Cpd761: 1.0})
reaction.gene_reaction_rule = '( SLC28A3 )'
model.add_reactions([reaction])
reaction = Reaction('OROTGLUt')
reaction.name = 'OROTGLUt'
reaction.subsystem = 'Transport, extracellular'
reaction.lower_bound = 0.
reaction.upper_bound = 1000.
reaction.add_metabolites({Cpd74: -1.0, Cpd3933: -1.0, Cpd759: 1.0, Cpd865: 1.0})
reaction.gene_reaction_rule = '( SLC22A7 )'
model.add_reactions([reaction])

from cobra.io import load_json_model, save_json_model, load_matlab_model, save_matlab_model, read_sbml_model, write_sbml_model
write_sbml_model(model, 'all_build.xml')
